---
title: Circulating biomarkers for metastatic prostate cancer
abstract: Biomarkers can be assessed for diagnostic, therapy-related or prognostic methods to identify phenotypes, such as a condition or disease, or the stage or progression of a disease. Circulating biomarkers from a bodily fluid can be used in profiling of physiological states or determining phenotypes. These include nucleic acids, protein, and circulating structures such as vesicles. Biomarkers can be used for theranostic purposes to select candidate treatment regimens for diseases, conditions, disease stages, and stages of a condition, and can also be used to determine treatment efficacy. The biomarkers can be circulating biomarkers, including vesicles and microRNA.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09469876&OS=09469876&RS=09469876
owner: Caris Life Sciences Switzerland Holdings GmbH
number: 09469876
owner_city: Basel
owner_country: CH
publication_date: 20110406
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE","BACKGROUND","SUMMARY","INCORPORATION BY REFERENCE","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","Example 1","Purification of Vesicles from Prostate Cancer Cell Lines","Example 2","Purification of Vesicles from VCaP and 22Rv1","Example 3","Plasma Collection and Vesicle Purification","Example 4","Analysis of Vesicles Using Antibody-Coupled Microspheres and Directly Conjugated Antibodies","Example 5","Analysis of Vesicles Using Antibody-Coupled Microspheres and Biotinylated Antibody","Example 6","Antibody Purification and Carbodiimide Coupling to Carboxylated Microspheres","Example 7","Vesicle Concentration from Plasma","Example 8","Determining Bio-Signatures for Prostate Cancer Using Multiplexing","Example 9","Determining Bio-Signatures for Colon Cancer Using Multiplexing","Example 10","Capture of Vesicles Using Magnetic Beads","Example 11","Detection of mRNA Transcripts in Vesicles","Example 12","Obtaining Serum Samples from Subjects","Example 13","RNA Isolation from Human Plasma and Serum Samples","Example 14","Measurement of miRNA Levels in RNA from Plasma and Serum Using qRT-PCR","Example 15","Generation of Standard Curves for Absolute Quantification of miRNAs","Example 16","Extracting microRNA from Vesicles","Example 17","MicroRNA Arrays","Example 18","MicroRNA Profiles in Vesicles","Example 19","MicroRNA Profiles of Magnetic EpCam-Captured Vesicles","Example 20","MicroRNA Profiles of CD9-Captured Vesicles","Example 21","Isolation of Vesicles Using a Filtration Module","Example 22","Multiplex Analysis of Vesicles Isolated with Filters","Example 23","Vesicle Isolation with Filters from Patient Samples","Example 24","Comparison of Vesicles Isolated with Filters Versus with Ultracentrifugation","Example 25","Sample Filter Comparison","Example 26","Flow Cytometry Analysis of Vesicles","Example 27","Antibody Detection of Vesicles","Example 28","Vesicle Reference Values for Prostate Cancer","Example 29","Detection of Prostate Cancer","Example 30","Differentiating BPH from PCa","Example 31","Microsphere Vesicle Prostate Cancer Assay Protocol","Example 32","Microsphere Vesicle PCa Assay Performance","Example 33","Median Fluorescence Intensity (MFI) Threshold Calculations","Example 34","Vesicle PCa Protein Marker Discovery","Example 35","Vesicle Protein Array to Detect Prostate Cancer","Example 36","Distinguishing BPH and PCa on Antibody Arrays","Example 37","Distinguishing BPH and PCa Using miRs","Example 38","miR-145 in Controls and PCa Samples","Example 39","Discovery of miRs Differentially Expressed in Metastatic and Non-Metastatic PCa","Example 40","Detection of miRs Differentially Expressed in PCa","Example 41","miRs Differentially Expressed in PCa","Example 42","miRs to Enhance Vesicle Diagnostic Assay Performance","Example 43","Comparison of miR Expression Patterns in Plasma, Serum and Cell Line Vesicles","Example 44","Isolating Subpopulations of Vesicles and Subsequent miR Profiles","Example 45","MicroRNA Biomarkers in Men with Prostate Cancer and Low PSA","Example 46","Prostate Cancer-Related microRNAs","Example 47","microRNAs Before and after Treatment for PCa","Example 48","Vesicle Isolation and Detection Methods","Example 49","Protocol for Vesicle Concentration from Plasma","Example 50","Microsphere Vesicle Analysis from Concentrated Plasma","Example 51","Analysis of Prostate Cancer (PCa) Vesicles Using Multiplex Assays","Example 52","Analysis of Colorectal Cancer (CRC) Vesicles Using Multiplex Assays","Example 53","Detection of Colorectal Cancer (CRC) with TMEM211 and CD24","Example 54","microRNA Overexpression in Colorectal Cancer Cell Lines","Example 55","microRNA to Detect CRC","Example 56","KRAS Sequencing in CRC Cell Lines and Patient Samples","Example 57","CRC miRs in Vesicle Fractions","Example 58","CRC Marker Combinations","Example 59","Vesicle Biosignatures for Breast Cancer (BCa)","Example 60","Analysis of Breast Cancer (BCa) Vesicles Using Multiplex Assays","Example 61","Plasma-Derived Microvesicles in Breast Cancer","Example 62","Vesicle Biosignatures for Lung Cancer (LCa)","Example 63","Biosignature on Circulating Microvesicles as Tool for Detecting Lung Cancer","Example 64","Circulating Vesicles Compared to Circulating Tumor Cells","Example 65","Depletion of Vesicles from Plasma","Example 66","Tissue Factor as a Vesicle Cancer Marker","Example 67","Selecting a Candidate Treatment for a Cancer","Example 68","Drug Associated Targets","Example 69","Monitoring Treatment Efficacy of Prostate Cancer","Example 70","TMEM211 Epitope Mapping","Example 71","B7H3 Epitope Mapping","Example 72","CD9 Epitope Mapping"],"p":["This application is a U.S. national stage entry of International Application No. PCT\/US2011\/031479 filed on Apr. 6, 2011, which application claims the benefit of U.S. Provisional Patent Application Nos. 61\/321,392, filed Apr. 6, 2010; 61\/332,174, filed May 6, 2010; 61\/348,214, filed May 25, 2010, 61\/348,685, filed May 26, 2010; 61\/354,125, filed Jun. 11, 2010; 61\/355,387, filed Jun. 16, 2010; 61\/357,517, filed Jun. 22, 2010; 61\/362,674, filed Jul. 8, 2010; 61\/364,785, filed Jul. 15, 2010; 61\/413,377, filed Nov. 12, 2010; 61\/322,690, filed Apr. 9, 2010; 61\/334,547, filed May 13, 2010; 61\/370,088, filed Aug. 2, 2010; 61\/391,504, filed Oct. 8, 2010; 61\/393,823, filed Oct. 15, 2010; 61\/416,560, filed Nov. 23, 2010; 61\/321,407, filed Apr. 6, 2010; 61\/356,974, filed Jun. 21, 2010; 61\/379,670, filed Sep. 2, 2010; 61\/381,305, filed Sep. 9, 2010; 61\/383,305, filed Sep. 15, 2010; and 61\/411,890, filed Nov. 9, 2010; each of which applications is incorporated by reference herein in its entirety.","Biomarkers for conditions and diseases such as cancer include biological molecules such as proteins, peptides, lipids, RNAs, DNA and variations and modifications thereof.","The identification of specific biomarkers, such as DNA, RNA and proteins, can provide biosignatures that are used for the diagnosis, prognosis, or theranosis of conditions or diseases. Biomarkers can be detected in bodily fluids, including circulating DNA, RNA, proteins, and vesicles. Circulating biomarkers include proteins such as PSA and CA125, and nucleic acids such as SEPT9 DNA and PCA3 messenger RNA (mRNA). Circulating biomarkers also include circulating vesicles. Vesicles are membrane encapsulated structures that are shed from cells and have been found in a number of bodily fluids, including blood, plasma, serum, breast milk, ascites, bronchoalveolar lavage fluid and urine. Vesicles can take part in the communication between cells as transport vehicles for proteins, RNAs, DNAs, viruses, and prions. MicroRNAs are short RNAs that regulate the transcription and degradation of messenger RNAs. MicroRNAs have been found in bodily fluids and have been observed as a component within vesicles shed from tumor cells. The analysis of circulating biomarkers associated with diseases, including vesicles and\/or microRNA, can aid in detection of disease or severity thereof, determining predisposition to a disease, as well as making treatment decisions.","Vesicles present in a biological sample provide a source of biomarkers, e.g., the markers are present within a vesicle (vesicle payload), or are present on the surface of a vesicle. Characteristics of vesicles (e.g., size, surface antigens, determination of cell-of-origin, payload) can also provide a diagnostic, prognostic or theranostic readout. There remains a need to identify biomarkers that can be used to detect and treat disease. microRNA and other biomarkers associated with vesicles as well as the characteristics of a vesicle can provide a diagnosis, prognosis, or theranosis.","The present invention provides methods and systems for characterizing a phenotype by detecting biomarkers that are indicative of disease or disease progress. The biomarkers can be circulating biomarkers including vesicles and microRNA.","Disclosed herein are methods and compositions for characterizing a phenotype by analyzing a vesicle, such as a vesicle present in a biological sample derived from a subject's cell. Characterizing a phenotype for a subject or individual may include, but is not limited to, the diagnosis of a disease or condition, the prognosis of a disease or condition, the determination of a disease stage or a condition stage, a drug efficacy, a physiological condition, organ distress or organ rejection, disease or condition progression, therapy-related association to a disease or condition, or a specific physiological or biological state.","In an aspect, the invention provides a method of characterizing a prostate disorder comprising, determining a presence or level of one or more biomarker in a biological sample from a subject, identifying a biosignature comprising the presence or level of the one or more biomarker, and comparing the biosignature to a reference, thereby characterizing the prostate disorder. Useful biomarkers are listed in Table 5 or Tables 9-11 herein. The step of comparing the biosignature to the reference can be determining whether any of the one or more biomarker is altered relative to the reference, and thereby providing a prognostic, diagnostic or theranostic determination for the prostate disorder.","In one embodiment, the prostate disorder comprises BPH and the one or more biomarker is selected from the group consisting of BCMA, CEACAM-1, HVEM, IL-1 R4, IL-10 Rb, Trappin-2, p53, hsa-miR-329, hsa-miR-30a, hsa-miR-335, hsa-miR-152, hsa-miR-151-5p, hsa-miR-200a, hsa-miR-145, hsa-miR-29a, hsa-miR-106b, hsa-miR-595, hsa-miR-142-5p, hsa-miR-99a, hsa-miR-20b, hsa-miR-373, hsa-miR-502-5p, hsa-miR-29b, hsa-miR-142-3p, hsa-miR-663, hsa-miR-423-5p, hsa-miR-15a, hsa-miR-888, hsa-miR-361-3p, hsa-miR-365, hsa-miR-10b, hsa-miR-199a-3p, hsa-miR-181a, hsa-miR-19a, hsa-miR-125b, hsa-miR-760, hsa-miR-7a, hsa-miR-671-5p, hsa-miR-7c, hsa-miR-1979, hsa-miR-103 and a combination thereof.","In another embodiment, the prostate disorder comprises prostate cancer and the one or more biomarker is selected from the group consisting of CD9, PSMA, PCSA, CD63, CD81, B7H3, IL6, OPG-13, IL6R, PA2G4, EZH2, RUNX2, SERPINB3, EpCam, hsa-let-7b, hsa-miR-107, hsa-miR-1205, hsa-miR-1270, hsa-miR-130b, hsa-miR-141, hsa-miR-143, hsa-miR-148b*, hsa-miR-150, hsa-miR-154*, hsa-miR-181a*, hsa-miR-181a-2*, hsa-miR-18a*, hsa-miR-19b-1*, hsa-miR-204, hsa-miR-2110, hsa-miR-215, hsa-miR-217, hsa-miR-219-2-3p, hsa-miR-23b*, hsa-miR-299-5p, hsa-miR-301a, hsa-miR-301a, hsa-miR-326, hsa-miR-331-3p, hsa-miR-365*, hsa-miR-373*, hsa-miR-424, hsa-miR-424*, hsa-miR-432, hsa-miR-450a, hsa-miR-451, hsa-miR-484, hsa-miR-497, hsa-miR-517*, hsa-miR-517a, hsa-miR-518f, hsa-miR-574-3p, hsa-miR-595, hsa-miR-617, hsa-miR-625*, hsa-miR-628-5p, hsa-miR-629, hsa-miR-634, hsa-miR-769-5p, hsa-miR-93, hsa-miR-96, and a combination thereof. The prostate disorder can be prostate cancer and the one or more biomarker can be selected from the group consisting of A33, a33 n15, AFP, ALA, ALIX, ALP, AnnexinV, APC, ASCA, ASPH (246-260), ASPH (666-680), ASPH (A-10), ASPH (D01P), ASPH (D03), ASPH (G-20), ASPH(H-300), AURKA, AURKB, B7H3, B7H4, BCA-225, BCNP1, BDNF, BRCA, CA125 (MUC16), CA-19-9, C-Bir, CD1.1, CD10, CD174 (Lewis y), CD24, CD44, CD46, CD59 (MEM-43), CD63, CD66e CEA, CD73, CD81, CD9, CDA, CDAC1 1a2, CEA, C-Erb2, C-erbB2, CRMP-2, CRP, CXCL12, CYFRA21-1, DLL4, DR3, EGFR, Epcam, EphA2, EphA2 (H-77), ER, ErbB4, EZH2, FASL, FRT, FRT c.f23, GDF15, GPCR, GPR30, Gro-alpha, HAP, HBD 1, HBD2, HER 3 (ErbB3), HSP, HSP70, hVEGFR2, iC3b, IL6 Unc, IL-1B, IL6 Unc, IL6R, IL8, IL-8, INSIG-2, KLK2, L1CAM, LAMN, LDH, MACC-1, MAPK4, MART-1, MCP-1, M-CSF, MFG-E8, MIC1, MIF, MIS RII, MMG, MMP26, MMP7, MMP9, MS4A1, MUC1, MUC1 seq1, MUC1 seq11A, MUC17, MUC2, Ncam, NGAL, NPGP\/NPFF2, OPG, OPN, p53, p53, PA2G4, PBP, PCSA, PDGFRB, PGP9.5, PIM1, PR (B), PRL, PSA, PSMA, PSME3, PTEN, R5-CD9 Tube 1, Reg IV, RUNX2, SCRN1, seprase, SERPINB3, SPARC, SPB, SPDEF, SRVN, STAT 3, STEAP1, TF (FL-295), TFF3, TGM2, TIMP-1, TIMP1, TIMP2, TMEM211, TMPRSS2, TNF-alpha, Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK, UNC93A, VEGF A, YPSMA-1, and a combination thereof.","The characterizing may comprise detecting a prostate cancer biosignature in the sample. Characterizing the prostate cancer can comprise identifying the prostate cancer as metastatic or aggressive. In such cases, the one or more biomarker can be selected from the group consisting of hsa-miR-100, hsa-miR-1236, hsa-miR-1296, hsa-miR-141, hsa-miR-146b-5p, hsa-miR-17*, hsa-miR-181a, hsa-miR-200b, hsa-miR-20a*, hsa-miR-23a*, hsa-miR-331-3p, hsa-miR-375, hsa-miR-452, hsa-miR-572, hsa-miR-574-3p, hsa-miR-577, hsa-miR-582-3p, hsa-miR-937, miR-10a, miR-134, miR-141, miR-200b, miR-30a, miR-32, miR-375, miR-495, miR-564, miR-570, miR-574-3p, miR-885-3p, and a combination thereof.","The characterizing can also comprise determining whether the subject is responding to a therapeutic treatment, or whether the subject is likely to respond or not respond to a therapeutic treatment. In an embodiment, the therapeutic treatment is selected from Tables 8-10 herein. For example, the therapeutic treatment comprises radical prostatectomy and\/or radiation therapy. The therapeutic treatment can be selected from a standard of care for prostate cancer, including without limitation one ore more of watchful waiting, surgical pelvic lymphadenectomy, radical prostatectomy, transurethral resection of the prostate (TURP), orchiectomy, radiation therapy, external-beam radiation therapy (EBRT), iodine I 125, palladium, iridium, hormone therapy, luteinizing hormone-releasing hormone agonists, leuprolide, goserelin, buserelin, antiandrogens, flutamide, bicalutamide, megestrol acetate, nilutamide, ketoconazole, aminoglutethimide, gonadotropin-releasing hormone (GnRH), estrogen, cryotherapy, chemotherapy, biologic therapy, ultrasound, and proton beam radiation. The biosignature for monitoring the response to treatment can include one or more of hsa-miR-1974, hsa-miR-27b, hsa-miR-103, hsa-miR-146a, hsa-miR-22, hsa-miR-382, hsa-miR-23a, hsa-miR-376c, hsa-miR-335, hsa-miR-142-5p, hsa-miR-221, hsa-miR-142-3p, hsa-miR-151-3p, hsa-miR-21, and hsa-miR-16.","In some embodiments, of the invention, the one or more biomarker comprises one or more of 5T4, ACTG1, ADAM10, ADAM15, ALDOA, ANXA2, ANXA6, APOA1, ATP1A1, BASP1, C1orf58, C20orf114, C8B, CAPZA1, CAV1, CD151, CD2AP, CD59, CD9, CD9, CFL1, CFP, CHMP4B, CLTC, COTL1, CTNND1, CTSB, CTSZ, CYCS, DPP4, EEF1A1, EHD1, ENO1, F11R, F2, F5, FAM125A, FNBP1L, FOLH1, GAPDH, GLB1, GPX3, HIST1H1C, HIST1H2AB, HSP90AB1, HSPA1B, HSPA8, IGSF8, ITGB1, ITIH3, JUP, LDHA, LDHB, LUM, LYZ, MFGE8, MGAM, MMP9, MYH2, MYL6B, NME1, NME2, PABPC1, PABPC4, PACSIN2, PCBP2, PDCD6IP, PRDX2, PSA, PSMA, PSMA1, PSMA2, PSMA4, PSMA6, PSMA7, PSMB1, PSMB2, PSMB3, PSMB4, PSMB5, PSMB6, PSMB8, PTGFRN, RPS27A, SDCBP, SERINC5, SH3GL1, SLC3A2, SMPDL3B, SNX9, TACSTD1, TCN2, THBS1, TPI1, TSG101, TUBB, VDAC2, VPS37B, YWHAG, YWHAQ, and YWHAZ. The one or more biomarker can also include one or more of CD9, CD63, CD81, PSMA, PCSA, B7H3 and EpCam. Each of these biomarkers can be assessed in association with a vesicle membrane.","In another aspect, the invention provides a method of characterizing benign prostate hyperplasia (BPH) comprising, determining a presence or level of one or more biomarker listed in Table 5 herein in a biological sample from a subject, identifying a biosignature comprising the presence or level of the one or more biomarker, and comparing the biosignature to a reference. The invention also provides a method of characterizing a prostate cancer comprising, determining a presence or level of one or more biomarker listed in Table 5 herein in a biological sample from a subject, identifying a biosignature comprising the presence or level of the one or more biomarker, and comparing the biosignature to a reference. Still further, the invention provides method of characterizing an aggressiveness of a prostate cancer comprising, determining a presence or level of one or more biomarker listed in Table 5 herein in a biological sample from a subject, identifying a biosignature comprising the presence or level of the one or more, and comparing the biosignature to a reference.","In embodiments of the subject methods, the biological sample comprises a bodily fluid. The bodily fluid can comprise peripheral blood, sera, plasma, ascites, urine, cerebrospinal fluid (CSF), sputum, saliva, bone marrow, synovial fluid, aqueous humor, amniotic fluid, cerumen, breast milk, broncheoalveolar lavage fluid, semen, prostatic fluid, cowper's fluid or pre-ejaculatory fluid, female ejaculate, sweat, fecal matter, hair, tears, cyst fluid, pleural and peritoneal fluid, pericardial fluid, lymph, chyme, chyle, bile, interstitial fluid, menses, pus, sebum, vomit, vaginal secretions, mucosal secretion, stool water, pancreatic juice, lavage fluids from sinus cavities, bronchopulmonary aspirates, blastocyl cavity fluid, or umbilical cord blood. The biological sample can be, e.g., urine, blood or blood derivatives. Blood derivatives include without limitation plasma and serum.","In some embodiments, the biological sample comprises one or more microvesicle. The one or more biomarker included in the biosignature can be associated with the one or more microvesicle. When the biosignature comprises multiple markers, some markers can be associated with vesicles whereas other markers are not. Alternately, all markers in the biosignature can be associated with the one or more microvesicle.","The one or more microvesicle can have a diameter between 20 nm and 800 nm, e.g., between 20 nm and 200 nm, between 20 nm and 100 nm, or between 100 nm and 500 nm.","The one or more microvesicle can be subjected to size exclusion chromatography, density gradient centrifugation, differential centrifugation, nanomembrane ultrafiltration, immunoabsorbent capture, affinity purification, affinity capture, immunoassay, microfluidic separation, flow cytometry or combinations thereof. Such methods can be useful for isolating the one or more microvesicle, in whole or in part, from other constituents in the sample.","In some embodiments, the one or more microvesicle is contacted with one or more binding agent. The one or more binding agent may comprise a nucleic acid, DNA molecule, RNA molecule, antibody, antibody fragment, aptamer, peptoid, zDNA, peptide nucleic acid (PNA), locked nucleic acid (LNA), lectin, peptide, dendrimer, membrane protein labeling agent, chemical compound, or a combination thereof. The one or more binding agent can be used to capture and\/or detect the one or more microvesicle. For example, the one or more binding agent can bind to one or more surface antigen on the one or more microvesicle. The one or more binding agent can be used to capture the one or more microvesicle, e.g., wherein the binding agent is tethered to a substrate. The one or more binding agent can also be used to detect the one or more microvesicle, e.g., wherein the binding agent is labeled or can also bind to a labeled moiety.","The one or more surface antigen recognized by the binding agent can be one or more protein, e.g. a surface protein on the microvesicle. The one or more protein may comprise one or more of CD9, CD63, CD81, PSMA, PCSA, B7H3 and EpCam. The one or more protein may comprise one or more of a tetraspanin, CD9, CD63, CD81, CD63, CD9, CD81, CD82, CD37, CD53, Rab-5b, Annexin V, MFG-E8, or a protein in Table 3.","In addition to surface antigens, the one or more biomarker may comprise payload within the one or more microvesicle. For example, a surface antigen can be used to capture the one or more microvesicle, then the payload within one or more captured microvesicle is assessed.","The microvesicle payload may comprise one or more nucleic acid, peptide, protein, lipid, antigen, carbohydrate, and\/or proteoglycan. The nucleic acid may comprise one or more DNA, mRNA, microRNA, snoRNA, snRNA, rRNA, tRNA, siRNA, hnRNA, or shRNA. In an embodiment, the nucleic acid comprises one or more microRNA selected from the group consisting of hsa-miR-200b, hsa-miR-375, hsa-miR-141, hsa-miR-331-3p, hsa-miR-181a, hsa-miR-574-3p and a combination thereof. The nucleic acid may also comprise one or more microRNA selected from Tables 27-41 herein.","In another aspect, the invention provides a method of characterizing a colorectal cancer comprising, determining the presence or levels of one or more biomarker in a biological sample from a subject, identifying a biosignature comprising the presence or levels of one or more biomarker in the biological sample, and comparing the biosignature to a reference, thereby characterizing the colorectal cancer. Useful biomarkers are listed in Table 6 and Tables 9-11 herein. The step of comparing the biosignature to the reference may comprise determining whether any of the one or more biomarker is altered relative to the reference, and thereby providing a prognostic, diagnostic or theranostic determination for the colorectal cancer.","In an embodiment, the one or more biomarker is selected from the group consisting of miR-92, miR-21, miR-9, miR-491, hsa-miR-376c, hsa-miR-215, hsa-miR-652, hsa-miR-582-5p, hsa-miR-324-5p, hsa-miR-1296, hsa-miR-28-5p, hsa-miR-190, hsa-miR-590-5p, hsa-miR-202, hsa-miR-195, and a combination thereof. In another embodiment, the one or more biomarker comprises one or more of Muc1, GPCR 110, TMEM211 and CD24. In still another embodiment, the one or more biomarker comprises one or more of A33, AFP, ALIX, ALX4, ANCA, APC, ASCA, AURKA, AURKB, B7H3, BANK1, BCNP1, BDNF, CA-19-9, CCSA-2, CCSA-3&4, CD10, CD24, CD44, CD63, CD66 CEA, CD66e CEA, CD81, CD9, CDA, C-Erb2, CRMP-2, CRP, CRTN, CXCL12, CYFRA21-1, DcR3, DLL4, DR3, EGFR, Epcam, EphA2, FASL, FRT, GAL3, GDF15, GPCR (GPR110), GPR30, GRO-1, HBD 1, HBD2, HNP1-3, IL-1B, IL8, IMP3, L1CAM, LAMN, MACC-1, MGC20553, MCP-1, M-CSF, MIC1, MIF, MMP7, MMP9, MS4A1, MUC1, MUC17, MUC2, Ncam, NGAL, NNMT, OPN, p53, PCSA, PDGFRB, PRL, PSMA, PSME3, Reg IV, SCRN1, Sept-9, SPARC, SPON2, SPR, SRVN, TFF3, TGM2, TIMP-1, TMEM211, TNF-alpha, TPA, TPS, Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK, UNC93A, and VEGFA. The one or more biomarker may comprise one or more of CD9, EGFR, NGAL, CD81, STEAP, CD24, A33, CD66E, EPHA2, Ferritin, GPR30, GPR110, MMP9, OPN, p53, TMEM211, TROP2, TGM2, TIMP, EGFR, DR3, UNC93A, MUC17, EpCAM, MUC1, MUC2, TSG101, CD63, B7H3, CD24, and TETS.","The characterizing may comprise detecting a colorectal cancer biosignature in the sample. The characterizing may comprise identifying the colorectal cancer as metastatic or aggressive.","The characterizing may also comprise determining whether the subject is responding to a therapeutic treatment, or whether the subject is likely to respond or not respond to a therapeutic treatment. The therapeutic treatment can be selected from Tables 8-10 herein. The therapeutic treatment can be a standard of care for the colorectal cancer, including without limitation one or more of primary surgical therapy, local excision, resection and anastomosis of primary lesion and removal of surrounding lymph nodes, adjuvant therapy, fluorouracil (5-FU), capecitabine, leucovorin, oxaliplatin, erlotinib, irinotecan, aspirin, mitomycin C, suntinib, cetuximab, bevacizumab, pegfilgrastim, panitumumab, ramucirumab, celecoxib, combination therapy, FOLFOX4 regimen, FOLFOX6 regimen, FOLFIRI regimen, FUFOX regimen, FUOX regimen, IFL regimen, XELOX regimen, 5-FU and levamisole regimens, German AIO regimen, CAPDX regimen, Douillard regimen, and radiation therapy.","In an embodiment, the one or more biomarker comprises comprise one or more of A26C1A, A26C1B, A2M, ACAA2, ACE, ACOT7, ACP1, ACTA1, ACTA2, ACTB, ACTBL2, ACTBL3, ACTC1, ACTG1, ACTG2, ACTN1, ACTN2, ACTN4, ACTR3, ADAM10, ADSL, AGR2, AGR3, AGRN, AHCY, AHNAK, AKR1B10, ALB, ALDH16A1, ALDH1A1, ALDOA, ANXA1, ANXA11, ANXA2, ANXA2P2, ANXA4, ANXA5, ANXA6, AP2A1, AP2A2, APOA1, ARF1, ARF3, ARF4, ARF5, ARF6, ARHGDIA, ARPC3, ARPC5L, ARRDC1, ARVCF, ASCC3L1, ASNS, ATP1A1, ATP1A2, ATP1A3, ATP1B1, ATP4A, ATP5A1, ATP5B, ATP5I, ATP5L, ATP50, ATP6AP2, B2M, BAIAP2, BAIAP2L1, BRI3BP, BSG, BUB3, C1orf58, C5orf32, CAD, CALM1, CALM2, CALM3, CAND1, CANX, CAPZA1, CBR1, CBR3, CCT2, CCT3, CCT4, CCT5, CCT6A, CCT7, CCT8, CD44, CD46, CD55, CD59, CD63, CD81, CD82, CD9, CDC42, CDH1, CDH17, CEACAM5, CFL1, CFL2, CHMP1A, CHMP2A, CHMP4B, CKB, CLDN3, CLDN4, CLDN7, CLIC1, CLIC4, CLSTN1, CLTC, CLTCL1, CLU, COL12A1, COPB1, COPB2, CORO1C, COX411, COX5B, CRYZ, CSPG4, CSRP1, CST3, CTNNA1, CTNNB1, CTNND1, CTTN, CYFIP1, DCD, DERA, DIP2A, DIP2B, DIP2C, DMBT1, DPEP1, DPP4, DYNC1H1, EDIL3, EEF1A1, EEF1A2, EEF1AL3, EEF1G, EEF2, EFNB1, EGFR, EHD1, EHD4, EIF3EIP, EIF3I, EIF4A1, EIF4A2, ENO1, ENO2, ENO3, EPHA2, EPHA5, EPHB1, EPHB2, EPHB3, EPHB4, EPPK1, ESD, EZR, F11R, F5, F7, FAM125A, FAM125B, FAM129B, FASLG, FASN, FAT, FCGBP, FER1L3, FKBP1A, FLNA, FLNB, FLOT1, FLOT2, G6PD, GAPDH, GARS, GCN1L1, GDI2, GK, GMDS, GNA13, GNAI2, GNAI3, GNAS, GNB1, GNB2, GNB2L1, GNB3, GNB4, GNG12, GOLGA7, GPA33, GPI, GPRC5A, GSN, GSTP1, H2AFJ, HADHA, hCG_1757335, HEPH, HIST1H2AB, HIST1H2AE, HIST1H2AJ, HIST1H2AK, HIST1H4A, HIST1H4B, HIST1H4C, HIST1H4D, HIST1H4E, HIST1H4F, HIST1H4H, HIST1H4I, HIST1H4J, HIST1H4K, HIST1H4L, HIST2H2AC, HIST2H4A, HIST2H4B, HIST3H2A, HIST4H4, HLA-A, HLA-A29.1, HLA-B, HLA-C, HLA-E, HLA-H, HNRNPA2B1, HNRNPH2, HPCAL1, HRAS, HSD17B4, HSP90AA1, HSP90AA2, HSP90AA4P, HSP90AB1, HSP90AB2P, HSP90AB3P, HSP90B1, HSPA1A, HSPA1B, HSPA1L, HSPA2, HSPA4, HSPA5, HSPA6, HSPA7, HSPA8, HSPA9, HSPD1, HSPE1, HSPG2, HYOU1, IDH1, IFITM1, IFITM2, IFITM3, IGH@, IGHG1, IGHG2, IGHG3, IGHG4, IGHM, IGHV4-31, IGK@, IGKC, IGKV1-5, IGKV2-24, IGKV3-20, IGSF3, IGSF8, IQGAP1, IQGAP2, ITGA2, ITGA3, ITGA6, ITGAV, ITGB1, ITGB4, JUP, KIAA0174, KIAA1199, KPNB1, KRAS, KRT1, KRT10, KRT13, KRT14, KRT15, KRT16, KRT17, KRT18, KRT19, KRT2, KRT20, KRT24, KRT25, KRT27, KRT28, KRT3, KRT4, KRT5, KRT6A, KRT6B, KRT6C, KRT7, KRT75, KRT76, KRT77, KRT79, KRT8, KRT9, LAMAS, LAMP1, LDHA, LDHB, LFNG, LGALS3, LGALS3BP, LGALS4, LIMA1, LIN7A, LIN7C, LOC100128936, LOC100130553, LOC100133382, LOC100133739, LOC284889, LOC388524, LOC388720, LOC442497, LOC653269, LRP4, LRPPRC, LRSAM1, LSR, LYZ, MAN1A1, MAP4K4, MARCKS, MARCKSL1, METRNL, MFGE8, MICA, MIF, MINK1, MITD1, MMP1, MOBKL1A, MSN, MTCH2, MUC13, MYADM, MYH10, MYH11, MYH14, MYH9, MYL6, MYL6B, MYO1C, MYO1D, NARS, NCALD, NCSTN, NEDD4, NEDD4L, NME1, NME2, NOTCH1, NQO1, NRAS, P4HB, PCBP1, PCNA, PCSK9, PDCD6, PDCD6IP, PDIA3, PDXK, PEBP1, PFN1, PGK1, PHB, PHB2, PKM2, PLEC1, PLEKHB2, PLSCR3, PLXNA1, PLXNB2, PPIA, PPIB, PPP2R1A, PRDX1, PRDX2, PRDX3, PRDX5, PRDX6, PRKAR2A, PRKDC, PRSS23, PSMA2, PSMC6, PSMD11, PSMD3, PSME3, PTGFRN, PTPRF, PYGB, QPCT, QSOX1, RAB10, RAB11A, RAB11B, RAB13, RAB14, RAB15, RAB1A, RAB1B, RAB2A, RAB33B, RAB35, RAB43, RAB4B, RAB5A, RAB5B, RAB5C, RAB6A, RAB6B, RAB7A, RAB8A, RAB8B, RAC1, RAC3, RALA, RALB, RAN, RANP1, RAP1A, RAP1B, RAP2A, RAP2B, RAP2C, RDX, REG4, RHOA, RHOC, RHOG, ROCK2, RP11-631M21.2, RPL10A, RPL12, RPL6, RPL8, RPLP0, RPLP0-like, RPLP1, RPLP2, RPN1, RPS13, RPS14, RPS15A, RPS16, RPS18, RPS20, RPS21, RPS27A, RPS3, RPS4X, RPS4Y1, RPS4Y2, RPS7, RPS8, RPSA, RPSAP15, RRAS, RRAS2, RUVBL1, RUVBL2, S100A10, S100A11, S100A14, S100A16, S100A6, S100P, SDC1, SDC4, SDCBP, SDCBP2, SERINC1, SERINC5, SERPINA1, SERPINF1, SETD4, SFN, SLC12A2, SLC12A7, SLC16A1, SLC1A5, SLC25A4, SLC25A5, SLC25A6, SLC29A1, SLC2A1, SLC3A2, SLC44A1, SLC7A5, SLC9A3R1, SMPDL3B, SNAP23, SND1, SOD1, SORT1, SPTAN1, SPTBN1, SSBP1, SSR4, TACSTD1, TAGLN2, TBCA, TCEB1, TCP1, TF, TFRC, THBS1, TJP2, TKT, TMED2, TNFSF10, TNIK, TNKS1BP1, TNPO3, TOLLIP, TOMM22, TPI1, TPM1, TRAP1, TSG101, TSPAN1, TSPAN14, TSPAN15, TSPAN6, TSPAN8, TSTA3, TTYH3, TUBA1A, TUBA1B, TUBA1C, TUBA3C, TUBA3D, TUBA3E, TUBA4A, TUBA4B, TUBA8, TUBB, TUBB2A, TUBB2B, TUBB2C, TUBB3, TUBB4, TUBB4Q, TUBB6, TUFM, TXN, UBA1, UBA52, UBB, UBC, UBE2N, UBE2V2, UGDH, UQCRC2, VAMP1, VAMP3, VAMPS, VCP, VIL1, VPS25, VPS28, VPS35, VPS36, VPS37B, VPS37C, WDR1, YWHAB, YWHAE, YWHAG, YWHAH, YWHAQ, and YWHAZ. The biomarkers can be associated with the membrane of a vesicle.","In embodiments, the biological sample comprises a bodily fluid. The bodily fluid can comprise peripheral blood, sera, plasma, ascites, urine, cerebrospinal fluid (CSF), sputum, saliva, bone marrow, synovial fluid, aqueous humor, amniotic fluid, cerumen, breast milk, broncheoalveolar lavage fluid, semen, prostatic fluid, cowper's fluid or pre-ejaculatory fluid, female ejaculate, sweat, fecal matter, hair, tears, cyst fluid, pleural and peritoneal fluid, pericardial fluid, lymph, chyme, chyle, bile, interstitial fluid, menses, pus, sebum, vomit, vaginal secretions, mucosal secretion, stool water, pancreatic juice, lavage fluids from sinus cavities, bronchopulmonary aspirates, blastocyl cavity fluid, or umbilical cord blood. The biological sample can be, e.g., stool, blood or blood derivatives. Blood derivatives include without limitation plasma and serum.","In some embodiments, the biological sample comprises one or more microvesicle. The one or more biomarker included in the biosignature can be associated with the one or more microvesicle. When the biosignature comprises multiple markers, some markers can be associated with vesicles whereas other markers are not. Alternately, all markers in the biosignature can be associated with the one or more microvesicle.","The one or more microvesicle can have a diameter between 20 nm and 800 nm, e.g., between 20 nm and 200 nm, between 20 nm and 100 nm, or between 100 nm and 500 nm.","The one or more microvesicle can be subjected to size exclusion chromatography, density gradient centrifugation, differential centrifugation, nanomembrane ultrafiltration, immunoabsorbent capture, affinity purification, affinity capture, immunoassay, microfluidic separation, flow cytometry or combinations thereof. Such methods can be useful for isolating the one or more microvesicle, in whole or in part, from other constituents in the sample.","In some embodiments, the one or more microvesicle is contacted with one or more binding agent. The one or more binding agent may comprise a nucleic acid, DNA molecule, RNA molecule, antibody, antibody fragment, aptamer, peptoid, zDNA, peptide nucleic acid (PNA), locked nucleic acid (LNA), lectin, peptide, dendrimer, membrane protein labeling agent, chemical compound, or a combination thereof. The one or more binding agent can be used to capture and\/or detect the one or more microvesicle. For example, the one or more binding agent can bind to one or more surface antigen on the one or more microvesicle. The one or more binding agent can be used to capture the one or more microvesicle, e.g., wherein the binding agent is tethered to a substrate. The one or more binding agent can also be used to detect the one or more microvesicle, e.g., wherein the binding agent is labeled or can also bind to a labeled moiety.","The one or more surface antigen recognized by the binding agent can be one or more protein, e.g. a surface protein on the microvesicle. The one or more protein may comprise one or more of a tetraspanin, CD9, CD63, CD81, CD63, CD9, CD81, CD82, CD37, CD53, Rab-5b, Annexin V, MFG-E8, or a protein in Table 3 herein.","In addition to surface antigens, the one or more biomarker may comprise payload within the one or more microvesicle. For example, a surface antigen can be used to capture the one or more microvesicle, then the payload within one or more captured microvesicle is assessed.","The microvesicle payload may comprise one or more nucleic acid, peptide, protein, lipid, antigen, carbohydrate, and\/or proteoglycan. The nucleic acid may comprise one or more DNA, mRNA, microRNA, snoRNA, snRNA, rRNA, tRNA, siRNA, hnRNA, or shRNA. In an embodiment, the nucleic acid comprises one or more microRNA selected from the Table 6 herein, or a combination thereof.","The various methods of the invention can be performed in vitro.","In another aspect, the invention provides the use of a reagent to carry out any of the methods of the invention. The reagent can be used for the diagnosis, prognosis or theranosis of a disease or disorder, e.g., a prostate disorder or a colorectal disorder. In a related aspect, the invention also provides a kit comprising a reagent to carry out any of the methods of the invention.","In still another aspect, the invention provides a composition comprising an isolated vesicle. In an embodiment, the isolated vesicle comprises one or more biomarker selected from Table 5 herein. In another embodiment, the isolated vesicle comprises one or more biomarker selected from Table 6 herein. In still other embodiments, the isolated vesicle comprises one or more biomarker selected from Tables 9-11 herein.","All publications, patents and patent applications mentioned in this specification are herein incorporated by reference to the same extent as if each individual publication, patent or patent application was specifically and individually indicated to be incorporated by reference.","Disclosed herein are methods and systems for characterizing a phenotype of a biological sample, e.g., a sample from a cell culture, an organism, or a subject. The phenotype can be characterized by assessing one or more biomarkers. The biomarkers can be associated with a vesicle or vesicle population, either presented vesicle surface antigens or vesicle payload. As used herein, vesicle payload comprises entities encapsulated within a vesicle. Vesicle associated biomarkers can comprise both membrane bound and soluble biomarkers. The biomarkers can also be circulating biomarkers, such as microRNA or protein assessed in a bodily fluid. Unless otherwise specified, the terms \u201cpurified\u201d or \u201cisolated\u201d as used herein in reference to vesicles or biomarker components mean partial or complete purification or isolation of such components from a cell or organism. Furthermore, unless otherwise specified, reference to vesicle isolation using a binding agent includes binding a vesicle with the binding agent whether or not such binding results in complete isolation of the vesicle apart from other biological entities in the starting material.","A method of characterizing a phenotype by analyzing a circulating biomarker, e.g., a nucleic acid biomarker, is depicted in scheme A of , as a non-limiting illustrative example. In a first step , a biological sample is obtained, e.g., a bodily fluid, tissue sample or cell culture. Nucleic acids are isolated from the sample . The nucleic acid can be DNA or RNA, e.g., microRNA. Assessment of such nucleic acids can provide a biosignature for a phenotype. By sampling the nucleic acids associated with target phenotype (e.g., disease versus healthy, pre- and post-treatment), one or more nucleic acid markers that are indicative of the phenotype can be determined. Various aspects of the present invention are directed to biosignatures determined by assessing one or more nucleic acid molecules (e.g., microRNA) present in the sample , where the biosignature corresponds to a predetermined phenotype .  illustrates a scheme B of using vesicles to isolate the nucleic acid molecules. In one example, a biological sample is obtained , and one or more vesicles, e.g., vesicles from a particular cell-of-origin and\/or vesicles associated with a particular disease state, are isolated from the sample . The vesicles are analyzed  by characterizing surface antigens associated with the vesicles and\/or determining the presence or levels of components present within the vesicles (\u201cpayload\u201d). Unless specified otherwise, the term \u201cantigen\u201d as used herein refers generally to a biomarker that can be bound by a binding agent, whether the binding agent is an antibody, aptamer, lectin, or other binding agent for the biomarker and regardless of whether such biomarker illicits an immune response in a host. Vesicle payload may be protein, including peptides and polypeptides, and\/or nucleic acids such as DNA and RNAs. RNA payload includes messenger RNA (mRNA) and microRNA (also referred to herein as miRNA or miR). A phenotype is characterized based on the biosignature of the vesicles . In another illustrative method of the invention, schemes A and B are performed together to characterize a phenotype. In such a scheme, vesicles and nucleic acids, e.g., microRNA, are assessed, thereby characterizing the phenotype.","In a related aspect, methods are provided herein for the discovery of biomarkers comprising assessing vesicle surface markers or payload markers in one sample and comparing the markers to another sample. Markers that distinguish between the samples can be used as biomarkers according to the invention. Such samples can be from a subject or group of subjects. For example, the groups can be, e.g., known responders and non-responders to a given treatment for a given disease or disorder. Biomarkers discovered to distinguish the known responders and non-responders provide a biosignature of whether a subject is likely to respond to a treatment such as a therapeutic agent, e.g., a drug or biologic.","Phenotypes","Disclosed herein are products and processes for characterizing a phenotype of an individual by analyzing a vesicle such as a membrane vesicle. A phenotype can be any observable characteristic or trait of a subject, such as a disease or condition, a disease stage or condition stage, susceptibility to a disease or condition, prognosis of a disease stage or condition, a physiological state, or response to therapeutics. A phenotype can result from a subject's gene expression as well as the influence of environmental factors and the interactions between the two, as well as from epigenetic modifications to nucleic acid sequences.","A phenotype in a subject can be characterized by obtaining a biological sample from a subject and analyzing one or more vesicles from the sample. For example, characterizing a phenotype for a subject or individual may include detecting a disease or condition (including pre-symptomatic early stage detecting), determining the prognosis, diagnosis, or theranosis of a disease or condition, or determining the stage or progression of a disease or condition. Characterizing a phenotype can also include identifying appropriate treatments or treatment efficacy for specific diseases, conditions, disease stages and condition stages, predictions and likelihood analysis of disease progression, particularly disease recurrence, metastatic spread or disease relapse. A phenotype can also be a clinically distinct type or subtype of a condition or disease, such as a cancer or tumor. Phenotype determination can also be a determination of a physiological condition, or an assessment of organ distress or organ rejection, such as post-transplantation. The products and processes described herein allow assessment of a subject on an individual basis, which can provide benefits of more efficient and economical decisions in treatment.","In an aspect, the invention relates to the analysis of vesicles to provide a biosignature to predict whether a subject is likely to respond to a treatment for a disease or disorder. Characterizating a phenotype includes predicting the responder\/non-responder status of the subject, wherein a responder responds to a treatment for a disease and a non-responder does not respond to the treatment. Vesicles can be analyzed in the subject and compared to vesicle analysis of previous subjects that were known to respond or not to a treatment. If the vesicle biosignature in a subject more closely aligns with that of previous subjects that were known to respond to the treatment, the subject can be characterized, or predicted, as a responder to the treatment. Similarly, if the vesicle biosignature in the subject more closely aligns with that of previous subjects that did not respond to the treatment, the subject can be characterized, or predicted as a non-responder to the treatment. The treatment can be for any appropriate disease, disorder or other condition. The method can be used in any disease setting where a vesicle biosignature that correlates with responder\/non-responder status is known.","The term \u201cphenotype\u201d as used herein can mean any trait or characteristic that is attributed to a vesicle biosignature that is identified utilizing methods of the invention. For example, a phenotype can be the identification of a subject as likely to respond to a treatment, or more broadly, it can be a diagnostic, prognostic or theranostic determination based on a characterized biosignature for a sample obtained from a subject.","In some embodiments, the phenotype comprises a disease or condition such as those listed in Table 1. For example, the phenotype can comprise the presence of or likelihood of developing a tumor, neoplasm, or cancer. A cancer detected or assessed by products or processes described herein includes, but is not limited to, breast cancer, ovarian cancer, lung cancer, colon cancer, hyperplastic polyp, adenoma, colorectal cancer, high grade dysplasia, low grade dysplasia, prostatic hyperplasia, prostate cancer, melanoma, pancreatic cancer, brain cancer (such as a glioblastoma), hematological malignancy, hepatocellular carcinoma, cervical cancer, endometrial cancer, head and neck cancer, esophageal cancer, gastrointestinal stromal tumor (GIST), renal cell carcinoma (RCC) or gastric cancer. The colorectal cancer can be CRC Dukes B or Dukes C-D. The hematological malignancy can be B-Cell Chronic Lymphocytic Leukemia, B-Cell Lymphoma-DLBCL, B-Cell Lymphoma-DLBCL-germinal center-like, B-Cell Lymphoma-DLBCL-activated B-cell-like, and Burkitt's lymphoma.","The phenotype can be a premalignant condition, such as actinic keratosis, atrophic gastritis, leukoplakia, erythroplasia, Lymphomatoid Granulomatosis, preleukemia, fibrosis, cervical dysplasia, uterine cervical dysplasia, xeroderma pigmentosum, Barrett's Esophagus, colorectal polyp, or other abnormal tissue growth or lesion that is likely to develop into a malignant tumor. Transformative viral infections such as HIV and HPV also present phenotypes that can be assessed according to the invention.","The cancer characterized by the methods of the invention can comprise, without limitation, a carcinoma, a sarcoma, a lymphoma or leukemia, a germ cell tumor, a blastoma, or other cancers. Carcinomas include without limitation epithelial neoplasms, squamous cell neoplasms squamous cell carcinoma, basal cell neoplasms basal cell carcinoma, transitional cell papillomas and carcinomas, adenomas and adenocarcinomas (glands), adenoma, adenocarcinoma, linitis plastica insulinoma, glucagonoma, gastrinoma, vipoma, cholangiocarcinoma, hepatocellular carcinoma, adenoid cystic carcinoma, carcinoid tumor of appendix, prolactinoma, oncocytoma, hurthle cell adenoma, renal cell carcinoma, grawitz tumor, multiple endocrine adenomas, endometrioid adenoma, adnexal and skin appendage neoplasms, mucoepidermoid neoplasms, cystic, mucinous and serous neoplasms, cystadenoma, pseudomyxoma peritonei, ductal, lobular and medullary neoplasms, acinar cell neoplasms, complex epithelial neoplasms, warthin's tumor, thymoma, specialized gonadal neoplasms, sex cord stromal tumor, thecoma, granulosa cell tumor, arrhenoblastoma, sertoli leydig cell tumor, glomus tumors, paraganglioma, pheochromocytoma, glomus tumor, nevi and melanomas, melanocytic nevus, malignant melanoma, melanoma, nodular melanoma, dysplastic nevus, lentigo maligna melanoma, superficial spreading melanoma, and malignant acral lentiginous melanoma. Sarcoma includes without limitation Askin's tumor, botryodies, chondrosarcoma, Ewing's sarcoma, malignant hemangio endothelioma, malignant schwannoma, osteosarcoma, soft tissue sarcomas including: alveolar soft part sarcoma, angiosarcoma, cystosarcoma phyllodes, dermatofibrosarcoma, desmoid tumor, desmoplastic small round cell tumor, epithelioid sarcoma, extraskeletal chondrosarcoma, extraskeletal osteosarcoma, fibrosarcoma, hemangiopericytoma, hemangiosarcoma, kaposi's sarcoma, leiomyosarcoma, liposarcoma, lymphangiosarcoma, lymphosarcoma, malignant fibrous histiocytoma, neurofibrosarcoma, rhabdomyosarcoma, and synovialsarcoma. Lymphoma and leukemia include without limitation chronic lymphocytic leukemia\/small lymphocytic lymphoma, B-cell prolymphocytic leukemia, lymphoplasmacytic lymphoma (such as waldenstrom macroglobulinemia), splenic marginal zone lymphoma, plasma cell myeloma, plasmacytoma, monoclonal immunoglobulin deposition diseases, heavy chain diseases, extranodal marginal zone B cell lymphoma, also called malt lymphoma, nodal marginal zone B cell lymphoma (nmzl), follicular lymphoma, mantle cell lymphoma, diffuse large B cell lymphoma, mediastinal (thymic) large B cell lymphoma, intravascular large B cell lymphoma, primary effusion lymphoma, burkitt lymphoma\/leukemia, T cell prolymphocytic leukemia, T cell large granular lymphocytic leukemia, aggressive NK cell leukemia, adult T cell leukemia\/lymphoma, extranodal NK\/T cell lymphoma, nasal type, enteropathy-type T cell lymphoma, hepatosplenic T cell lymphoma, blastic NK cell lymphoma, mycosis fungoides\/sezary syndrome, primary cutaneous CD30-positive T cell lymphoproliferative disorders, primary cutaneous anaplastic large cell lymphoma, lymphomatoid papulosis, angioimmunoblastic T cell lymphoma, peripheral T cell lymphoma, unspecified, anaplastic large cell lymphoma, classical hodgkin lymphomas (nodular sclerosis, mixed cellularity, lymphocyte-rich, lymphocyte depleted or not depleted), and nodular lymphocyte-predominant hodgkin lymphoma. Germ cell tumors include without limitation germinoma, dysgerminoma, seminoma, nongerminomatous germ cell tumor, embryonal carcinoma, endodermal sinus turmor, choriocarcinoma, teratoma, polyembryoma, and gonadoblastoma. Blastoma includes without limitation nephroblastoma, medulloblastoma, and retinoblastoma. Other cancers include without limitation labial carcinoma, larynx carcinoma, hypopharynx carcinoma, tongue carcinoma, salivary gland carcinoma, gastric carcinoma, adenocarcinoma, thyroid cancer (medullary and papillary thyroid carcinoma), renal carcinoma, kidney parenchyma carcinoma, cervix carcinoma, uterine corpus carcinoma, endometrium carcinoma, chorion carcinoma, testis carcinoma, urinary carcinoma, melanoma, brain tumors such as glioblastoma, astrocytoma, meningioma, medulloblastoma and peripheral neuroectodermal tumors, gall bladder carcinoma, bronchial carcinoma, multiple myeloma, basalioma, teratoma, retinoblastoma, choroidea melanoma, seminoma, rhabdomyosarcoma, craniopharyngeoma, osteosarcoma, chondrosarcoma, myosarcoma, liposarcoma, fibrosarcoma, Ewing sarcoma, and plasmocytoma.","In a further embodiment, the cancer under analysis may be a lung cancer including non-small cell lung cancer and small cell lung cancer (including small cell carcinoma (oat cell cancer), mixed small cell\/large cell carcinoma, and combined small cell carcinoma), colon cancer, breast cancer, prostate cancer, liver cancer, pancreas cancer, brain cancer, kidney cancer, ovarian cancer, stomach cancer, skin cancer, bone cancer, gastric cancer, breast cancer, pancreatic cancer, glioma, glioblastoma, hepatocellular carcinoma, papillary renal carcinoma, head and neck squamous cell carcinoma, leukemia, lymphoma, myeloma, or a solid tumor.","In embodiments, the cancer comprises an acute lymphoblastic leukemia; acute myeloid leukemia; adrenocortical carcinoma; AIDS-related cancers; AIDS-related lymphoma; anal cancer; appendix cancer; astrocytomas; atypical teratoid\/rhabdoid tumor; basal cell carcinoma; bladder cancer; brain stem glioma; brain tumor (including brain stem glioma, central nervous system atypical teratoid\/rhabdoid tumor, central nervous system embryonal tumors, astrocytomas, craniopharyngioma, ependymoblastoma, ependymoma, medulloblastoma, medulloepithelioma, pineal parenchymal tumors of intermediate differentiation, supratentorial primitive neuroectodermal tumors and pineoblastoma); breast cancer; bronchial tumors; Burkitt lymphoma; cancer of unknown primary site; carcinoid tumor; carcinoma of unknown primary site; central nervous system atypical teratoid\/rhabdoid tumor; central nervous system embryonal tumors; cervical cancer; childhood cancers; chordoma; chronic lymphocytic leukemia; chronic myelogenous leukemia; chronic myeloproliferative disorders; colon cancer; colorectal cancer; craniopharyngioma; cutaneous T-cell lymphoma; endocrine pancreas islet cell tumors; endometrial cancer; ependymoblastoma; ependymoma; esophageal cancer; esthesioneuroblastoma; Ewing sarcoma; extracranial germ cell tumor; extragonadal germ cell tumor; extrahepatic bile duct cancer; gallbladder cancer; gastric (stomach) cancer; gastrointestinal carcinoid tumor; gastrointestinal stromal cell tumor; gastrointestinal stromal tumor (GIST); gestational trophoblastic tumor; glioma; hairy cell leukemia; head and neck cancer; heart cancer; Hodgkin lymphoma; hypopharyngeal cancer; intraocular melanoma; islet cell tumors; Kaposi sarcoma; kidney cancer; Langerhans cell histiocytosis; laryngeal cancer; lip cancer; liver cancer; malignant fibrous histiocytoma bone cancer; medulloblastoma; medulloepithelioma; melanoma; Merkel cell carcinoma; Merkel cell skin carcinoma; mesothelioma; metastatic squamous neck cancer with occult primary; mouth cancer; multiple endocrine neoplasia syndromes; multiple myeloma; multiple myeloma\/plasma cell neoplasm; mycosis fungoides; myelodysplastic syndromes; myeloproliferative neoplasms; nasal cavity cancer; nasopharyngeal cancer; neuroblastoma; Non-Hodgkin lymphoma; nonmelanoma skin cancer; non-small cell lung cancer; oral cancer; oral cavity cancer; oropharyngeal cancer; osteosarcoma; other brain and spinal cord tumors; ovarian cancer; ovarian epithelial cancer; ovarian germ cell tumor; ovarian low malignant potential tumor; pancreatic cancer; papillomatosis; paranasal sinus cancer; parathyroid cancer; pelvic cancer; penile cancer; pharyngeal cancer; pineal parenchymal tumors of intermediate differentiation; pineoblastoma; pituitary tumor; plasma cell neoplasm\/multiple myeloma; pleuropulmonary blastoma; primary central nervous system (CNS) lymphoma; primary hepatocellular liver cancer; prostate cancer; rectal cancer; renal cancer; renal cell (kidney) cancer; renal cell cancer; respiratory tract cancer; retinoblastoma; rhabdomyosarcoma; salivary gland cancer; S\u00e9zary syndrome; small cell lung cancer; small intestine cancer; soft tissue sarcoma; squamous cell carcinoma; squamous neck cancer; stomach (gastric) cancer; supratentorial primitive neuroectodermal tumors; T-cell lymphoma; testicular cancer; throat cancer; thymic carcinoma; thymoma; thyroid cancer; transitional cell cancer; transitional cell cancer of the renal pelvis and ureter; trophoblastic tumor; ureter cancer; urethral cancer; uterine cancer; uterine sarcoma; vaginal cancer; vulvar cancer; Waldenstr\u00f6m macroglobulinemia; or Wilm's tumor. The methods of the invention can be used to characterize these and other cancers. Thus, characterizing a phenotype can be providing a diagnosis, prognosis or theranosis of one of the cancers disclosed herein.","The phenotype can also be an inflammatory disease, immune disease, or autoimmune disease. For example, the disease may be inflammatory bowel disease (IBD), Crohn's disease (CD), ulcerative colitis (UC), pelvic inflammation, vasculitis, psoriasis, diabetes, autoimmune hepatitis, Multiple Sclerosis, Myasthenia Gravis, Type I diabetes, Rheumatoid Arthritis, Psoriasis, Systemic Lupus Erythematosis (SLE), Hashimoto's Thyroiditis, Grave's disease, Ankylosing Spondylitis Sjogrens Disease, CREST syndrome, Scleroderma, Rheumatic Disease, organ rejection, Primary Sclerosing Cholangitis, or sepsis.","The phenotype can also comprise a cardiovascular disease, such as atherosclerosis, congestive heart failure, vulnerable plaque, stroke, or ischemia. The cardiovascular disease or condition can be high blood pressure, stenosis, vessel occlusion or a thrombotic event.","The phenotype can also comprise a neurological disease, such as Multiple Sclerosis (MS), Parkinson's Disease (PD), Alzheimer's Disease (AD), schizophrenia, bipolar disorder, depression, autism, Prion Disease, Pick's disease, dementia, Huntington disease (HD), Down's syndrome, cerebrovascular disease, Rasmussen's encephalitis, viral meningitis, neurospsychiatric systemic lupus erythematosus (NPSLE), amyotrophic lateral sclerosis, Creutzfeldt-Jacob disease, Gerstmann-Straussler-Scheinker disease, transmissible spongiform encephalopathy, ischemic reperfusion damage (e.g. stroke), brain trauma, microbial infection, or chronic fatigue syndrome. The phenotype may also be a condition such as fibromyalgia, chronic neuropathic pain, or peripheral neuropathic pain.","The phenotype may also comprise an infectious disease, such as a bacterial, viral or yeast infection. For example, the disease or condition may be Whipple's Disease, Prion Disease, cirrhosis, methicillin-resistant , HIV, hepatitis, syphilis, meningitis, malaria, tuberculosis, or influenza. Viral proteins, such as HIV or HCV-like particles can be assessed in a vesicle, to characterize a viral condition.","The phenotype can also comprise a perinatal or pregnancy related condition (e.g. preeclampsia or preterm birth), metabolic disease or condition, such as a metabolic disease or condition associated with iron metabolism. For example, hepcidin can be assayed in a vesicle to characterize an iron deficiency. The metabolic disease or condition can also be diabetes, inflammation, or a perinatal condition.","The methods of the invention can be used to characterize these and other diseases and disorders that can be assessed via biomarkers. Thus, characterizing a phenotype can be providing a diagnosis, prognosis or theranosis of one of the diseases and disorders disclosed herein.","Subject","One or more phenotypes of a subject can be determined by analyzing one or more vesicles, such as vesicles, in a biological sample obtained from the subject. A subject or patient can include, but is not limited to, mammals such as bovine, avian, canine, equine, feline, ovine, porcine, or primate animals (including humans and non-human primates). A subject can also include a mammal of importance due to being endangered, such as a Siberian tiger; or economic importance, such as an animal raised on a farm for consumption by humans, or an animal of social importance to humans, such as an animal kept as a pet or in a zoo. Examples of such animals include, but are not limited to, carnivores such as cats and dogs; swine including pigs, hogs and wild boars; ruminants or ungulates such as cattle, oxen, sheep, giraffes, deer, goats, bison, camels or horses. Also included are birds that are endangered or kept in zoos, as well as fowl and more particularly domesticated fowl, i.e. poultry, such as turkeys and chickens, ducks, geese, guinea fowl. Also included are domesticated swine and horses (including race horses). In addition, any animal species connected to commercial activities are also included such as those animals connected to agriculture and aquaculture and other activities in which disease monitoring, diagnosis, and therapy selection are routine practice in husbandry for economic productivity and\/or safety of the food chain.","The subject can have a pre-existing disease or condition, such as cancer. Alternatively, the subject may not have any known pre-existing condition. The subject may also be non-responsive to an existing or past treatment, such as a treatment for cancer.","Samples","The biological sample obtained from the subject can be any bodily fluid. For example, the biological sample can be peripheral blood, sera, plasma, ascites, urine, cerebrospinal fluid (CSF), sputum, saliva, bone marrow, synovial fluid, aqueous humor, amniotic fluid, cerumen, breast milk, broncheoalveolar lavage fluid, semen (including prostatic fluid), Cowper's fluid or pre-ejaculatory fluid, female ejaculate, sweat, fecal matter, hair, tears, cyst fluid, pleural and peritoneal fluid, pericardial fluid, lymph, chyme, chyle, bile, interstitial fluid, menses, pus, sebum, vomit, vaginal secretions, mucosal secretion, stool water, pancreatic juice, lavage fluids from sinus cavities, bronchopulmonary aspirates or other lavage fluids. A biological sample may also include the blastocyl cavity, umbilical cord blood, or maternal circulation which may be of fetal or maternal origin. The biological sample may also be a tissue sample or biopsy from which vesicles and other circulating biomarkers may be obtained. For example, cells from the sample can be cultured and vesicles isolated from the culture (see for example, Example 1). In various embodiments, biomarkers or more particularly biosignatures disclosed herein can be assessed directly from such biological samples (e.g., identification of presence or levels of nucleic acid or polypeptide biomarkers or functional fragments thereof) utilizing various methods, such as extraction of nucleic acid molecules from blood, plasma, serum or any of the foregoing biological samples, use of protein or antibody arrays to identify polypeptide (or functional fragment) biomarker(s), as well as other array, sequencing, PCR and proteomic techniques known in the art for identification and assessment of nucleic acid and polypeptide molecules.","Table 1 lists illustrative examples of diseases, conditions, or biological states and a corresponding list of biological samples from which vesicles may be analyzed.",{"@attributes":{"id":"p-0184","num":"0183"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"322pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Examples of Biological Samples for Vesicle Analysis for"},{"entry":"Various Diseases, Conditions, or Biological States"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Illustrative Disease, Condition or Biological State","Illustrative Biological Samples"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Cancers\/neoplasms affecting the following tissue","Blood, serum, plasma, cerebrospinal fluid (CSF),"]},{"entry":["types\/bodily systems: breast, lung, ovarian, colon,","urine, sputum, ascites, synovial fluid, semen, nipple"]},{"entry":["rectal, prostate, pancreatic, brain, bone, connective","aspirates, saliva, bronchoalveolar lavage fluid, tears,"]},{"entry":["tissue, glands, skin, lymph, nervous system, endocrine,","oropharyngeal washes, feces, peritoneal fluids, pleural"]},{"entry":["germ cell, genitourinary, hematologic\/blood, bone","effusion, sweat, tears, aqueous humor, pericardial"]},{"entry":["marrow, muscle, eye, esophageal, fat tissue, thyroid,","fluid, lymph, chyme, chyle, bile, stool water, amniotic"]},{"entry":["pituitary, spinal cord, bile duct, heart, gall bladder,","fluid, breast milk, pancreatic juice, cerumen, Cowper's"]},{"entry":["bladder, testes, cervical, endometrial, renal, ovarian,","fluid or pre-ejaculatory fluid, female ejaculate,"]},{"entry":["digestive\/gastrointestinal, stomach, head and neck,","interstitial fluid, menses, mucus, pus, sebum, vaginal"]},{"entry":["liver, leukemia, respiratory\/thorasic, cancers of","lubrication, vomit"]},{"entry":"unknown primary (CUP)"},{"entry":["Neurodegenerative\/neurological disorders:","Blood, serum, plasma, CSF, urine"]},{"entry":"Parkinson's disease, Alzheimer's Disease and multiple"},{"entry":"sclerosis, Schizophrenia, and bipolar disorder,"},{"entry":"spasticity disorders, epilepsy"},{"entry":["Cardiovascular Disease: atherosclerosis,","Blood, serum, plasma, CSF, urine"]},{"entry":"cardiomyopathy, endocarditis, vunerable plaques,"},{"entry":"infection"},{"entry":["Stroke: ischemic, intracerebral hemorrhage,","Blood, serum, plasma, CSF, urine"]},{"entry":"subarachnoid hemorrhage, transient ischemic attacks"},{"entry":"(TIA)"},{"entry":["Pain disorders: peripheral neuropathic pain and","Blood, serum, plasma, CSF, urine"]},{"entry":"chronic neuropathic pain, and fibromyalgia,"},{"entry":["Autoimmune disease: systemic and localized diseases,","Blood, serum, plasma, CSF, urine, synovial fluid"]},{"entry":"rheumatic disease, Lupus, Sjogren's syndrome"},{"entry":["Digestive system abnormalities: Barrett's esophagus,","Blood, serum, plasma, CSF, urine"]},{"entry":"irritable bowel syndrome, ulcerative colitis, Crohn's"},{"entry":"disease, Diverticulosis and Diverticulitis, Celiac"},{"entry":"Disease"},{"entry":["Endocrine disorders: diabetes mellitus, various forms","Blood, serum, plasma, CSF, urine"]},{"entry":"of Thyroiditis,, adrenal disorders, pituitary disorders"},{"entry":["Diseases and disorders of the skin: psoriasis","Blood, serum, plasma, CSF, urine, synovial fluid, tears"]},{"entry":["Urological disorders: benign prostatic hypertrophy","Blood, serum, plasma, urine"]},{"entry":"(BPH), polycystic kidney disease, interstitial cystitis"},{"entry":["Hepatic disease\/injury: Cirrhosis, induced","Blood, serum, plasma, urine"]},{"entry":"hepatotoxicity (due to exposure to natural or synthetic"},{"entry":"chemical sources)"},{"entry":["Kidney disease\/injury: acute, sub-acute, chronic","Blood, serum, plasma, urine"]},{"entry":"conditions, Podocyte injury, focal segmental"},{"entry":"glomerulosclerosis"},{"entry":["Endometriosis","Blood, serum, plasma, urine, vaginal fluids"]},{"entry":["Osteoporosis","Blood, serum, plasma, urine, synovial fluid"]},{"entry":["Pancreatitis","Blood, serum, plasma, urine, pancreatic juice"]},{"entry":["Asthma","Blood, serum, plasma, urine, sputum, bronchiolar"]},{"entry":[{},"lavage fluid"]},{"entry":["Allergies","Blood, serum, plasma, urine, sputum, bronchiolar"]},{"entry":[{},"lavage fluid"]},{"entry":["Prion-related diseases","Blood, serum, plasma, CSF, urine"]},{"entry":["Viral Infections: HIV\/AIDS","Blood, serum, plasma, urine"]},{"entry":["Sepsis","Blood, serum, plasma, urine, tears, nasal lavage"]},{"entry":["Organ rejection\/transplantation","Blood, serum, plasma, urine, various lavage fluids"]},{"entry":["Differentiating conditions: adenoma versus","Blood, serum, plasma, urine, sputum, feces, colonic"]},{"entry":["hyperplastic polyp, irritable bowel syndrome (IBS)","lavage fluid"]},{"entry":"versus normal, classifying Dukes stages A, B, C,"},{"entry":"and\/or D of colon cancer, adenoma with low-grade"},{"entry":"hyperplasia versus high-grade hyperplasia, adenoma"},{"entry":"versus normal, colorectal cancer versus normal, IBS"},{"entry":"versus. ulcerative colitis (UC) versus Crohn's disease"},{"entry":"(CD),"},{"entry":["Pregnancy related physiological states, conditions, or","Maternal serum, plasma, amniotic fluid, cord blood"]},{"entry":"affiliated diseases: genetic risk, adverse pregnancy"},{"entry":"outcomes"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The methods of the invention can be used to characterize a phenotype using a blood sample or blood derivative. Blood derivatives include plasma and serum. Blood plasma is the liquid component of whole blood, and makes up approximately 55% of the total blood volume. It is composed primarily of water with small amounts of minerals, salts, ions, nutrients, and proteins in solution. In whole blood, red blood cells, leukocytes, and platelets are suspended within the plasma. Blood serum refers to blood plasma without fibrinogen or other clotting factors (i.e., whole blood minus both the cells and the clotting factors).","The biological sample may be obtained through a third party, such as a party not performing the analysis of the biomarkers, whether direct assessment of a biological sample or by profiling one or more vesicles obtained from the biological sample. For example, the sample may be obtained through a clinician, physician, or other health care manager of a subject from which the sample is derived. Alternatively, the biological sample may obtained by the same party analyzing the vesicle. In addition, biological samples be assayed, are archived (e.g., frozen) or ortherwise stored in under preservative conditions.","The volume of the biological sample used for biomarker analysis can be in the range of between 0.1-20 mL, such as less than about 20, 15, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1 or 0.1 mL.","A sample of bodily fluid can be used as a sample for characterizing a phenotype. For example, biomarkers in the sample can be assessed to provide a diagnosis, prognosis and\/or theranosis of a disease. The biomarkers can be circulating biomarkers, such as circulating proteins or nucleic acids. The biomarkers can also be associated with a vesicle or vesicle population. Methods of the invention can be applied to assess one or more vesicles, as well as one or more different vesicle populations that may be present in a biological sample or in a subject. Analysis of one or more biomarkers in a biological sample can be used to determine whether an additional biological sample should be obtained for analysis. For example, analysis of one or more vesicles in a sample of bodily fluid can aid in determining whether a tissue biopsy should be obtained.","A sample from a patient can be collected under conditions that preserve the circulating biomarkers and other entities of interest contained therein for subsequent analysis. In an embodiment, the samples are processed using one or more of CellSave Preservative Tubes (Veridex, North Raritan, N.J.), PAXgene Blood DNA Tubes (QIAGEN GmbH, Germany), and RNAlater (QIAGEN GmbH, Germany).","CellSave Preservative Tubes (CellSave tubes) are sterile evacuated blood collection tubes. Each tube contains a solution that contains Na2EDTA and a cell preservative. The EDTA absorbs calcium ions, which can reduce or eliminate blood clotting. The preservative preserves the morphology and cell surface antigen expression of epithelial and other cells. The collection and processing can be performed as described in a protocol provided by the manufacturer. Each tube is evacuated to withdraw venous whole blood following standard phlebotomy procedures as known to those of skill in the art. CellSave tubes are disclosed in U.S. Pat. Nos. 5,466,574; 5,512,332; 5,597,531; 5,698,271; 5,985,153; 5,993,665; 6,120,856; 6,136,182; 6,365,362; 6,551,843; 6,620,627; 6,623,982; 6,645,731; 6,660,159; 6,790,366; 6,861,259; 6,890,426; 7,011,794; 7,282,350; 7,332,288; 5,849,517 and 5,459,073, each of which is incorporated by reference in its entirety herein.","The PAXgene Blood DNA Tube (PAXgene tube) is a plastic, evacuated tube for the collection of whole blood for the isolation of nucleic acids. The tubes can be used for blood collection, transport and storage of whole blood specimens and isolation of nucleic acids contained therein, e.g., DNA or RNA. Blood is collected under a standard phlebotomy protocol into an evacuated tube that contains an additive. The collection and processing can be performed as described in a protocol provided by the manufacturer. PAXgene tubes are disclosed in U.S. Pat. Nos. 5,906,744; 4,741,446; 4,991,104, each of which is incorporated by reference in its entirety herein.","The RNAlater RNA Stabilization Reagent (RNAlater) is used for immediate stabilization of RNA in tissues. RNA can be unstable in harvested samples. The aqueous RNAlater reagent permeates tissues and other biological samples, thereby stabilizing and protecting the RNA contained therein. Such protection helps ensure that downstream analyses reflect the expression profile of the RNA in the tissue or other sample. The samples are submerged in an appropriate volume of RNAlater reagent immediately after harvesting. The collection and processing can be performed as described in a protocol provided by the manufacturer. According to the manufacturer, the reagent preserves RNA for up to 1 day at 37\u00b0 C., 7 days at 18-25\u00b0 C., or 4 weeks at 2-8\u00b0 C., allowing processing, transportation, storage, and shipping of samples without liquid nitrogen or dry ice. The samples can also be placed at \u221220\u00b0 C. or \u221280\u00b0 C., e.g., for archival storage. The preserved samples can be used to analyze any type of RNA, including without limitation total RNA, mRNA, and microRNA. RNAlater can also be useful for collecting samples for DNA, RNA and protein analysis. RNAlater is disclosed in U.S. Pat. No. 5,346,994, each of which is incorporated by reference in its entirety herein.","Vesicles","Methods of the invention can include assessing one or more vesicles, including assessing vesicle populations. A vesicle, as used herein, is a membrane vesicle that is shed from cells. Vesicles or membrane vesicles include without limitation: circulating microvesicles (cMVs), microvesicle, exosome, nanovesicle, dexosome, bleb, blebby, prostasome, microparticle, intralumenal vesicle, membrane fragment, intralumenal endosomal vesicle, endosomal-like vesicle, exocytosis vehicle, endosome vesicle, endosomal vesicle, apoptotic body, multivesicular body, secretory vesicle, phospholipid vesicle, liposomal vesicle, argosome, texasome, secresome, tolerosome, melanosome, oncosome, or exocytosed vehicle. Furthermore, although vesicles may be produced by different cellular processes, the methods of the invention are not limited to or reliant on any one mechanism, insofar as such vesicles are present in a biological sample and are capable of being characterized by the methods disclosed herein. Unless otherwise specified, methods that make use of a species of vesicle can be applied to other types of vesicles. Vesicles comprise spherical structures with a lipid bilayer similar to cell membranes which surrounds an inner compartment which can contain soluble components, sometimes referred to as the payload. In some embodiments, the methods of the invention make use of exosomes, which are small secreted vesicles of about 40-100 nm in diameter. For a review of membrane vesicles, including types and characterizations, see Thery et al., 2009 August; 9(8): 581-93. Some properties of different types of vesicles include those in Table 2:",{"@attributes":{"id":"p-0194","num":"0193"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"336pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Vesicle Properties"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"7"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"7","colwidth":"49pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"Membrane","Exosome-","Apoptotic"]},{"entry":["Feature","Exosomes","Microvesicles","Ectosomes","particles","like vesicles","vesicles"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":["Size","\u2009\u200250-100 nm","100-1,000 nm","50-200 nm","\u2009\u200250-80 nm","20-50 nm","\u2009\u200250-500 nm"]},{"entry":["Density in ","1.13-1.19 g\/ml",{},{},"1.04-1.07 g\/ml","1.1 g\/ml","1.16-1.28 g\/ml"]},{"entry":["sucrose",{},{},{},{},{},{}]},{"entry":["EM ","Cup shape","Irregular shape, ","Bilamellar ","Round","Irregular ","Heterogeneous"]},{"entry":["appearance",{},"electron dense","round ",{},"shape",{}]},{"entry":[{},{},{},"structures",{},{},{}]},{"entry":["Sedimentation","100,000 g","10,000 g","160,000-","100,000-","175,000 g","\u20031,200 g, "]},{"entry":[{},{},{},"200,000 g","200,000 g",{},"\u200210,000 g, "]},{"entry":[{},{},{},{},{},{},"100,000 g"]},{"entry":["Lipid ","Enriched in ","Expose PPS","Enriched in ",{},"No lipid ",{}]},{"entry":["composition","cholesterol, ",{},"cholesterol and ",{},"rafts",{}]},{"entry":[{},"sphingomyelin ",{},"diacylglycerol;",{},{},{}]},{"entry":[{},"and ceramide; ",{},"expose PPS",{},{},{}]},{"entry":[{},"contains lipid ",{},{},{},{},{}]},{"entry":[{},"rafts; expose ",{},{},{},{},{}]},{"entry":[{},"PPS",{},{},{},{},{}]},{"entry":["Major protein ","Tetraspanins ","Integrins, ","CR1 and ","CD133; no ","TNFRI","Histones"]},{"entry":["markers","(e.g., CD63, ","selectins and ","proteolytic ","CD63",{},{}]},{"entry":[{},"CD9), Alix, ","CD40 ligand","enzymes; no ",{},{},{}]},{"entry":[{},"TSG101",{},"CD63",{},{},{}]},{"entry":["Intracellular ","Internal ","Plasma ","Plasma ","Plasma ",{},{}]},{"entry":["origin","compartments ","membrane","membrane","membrane",{},{}]},{"entry":[{},"(endosomes)"]},{"entry":{"@attributes":{"namest":"1","nameend":"7","align":"center","rowsep":"1"}}},{"entry":"Abbreviations: phosphatidylserine (PPS); electron microscopy (EM)"}]}}]}}},"Vesicles include shed membrane bound particles, or \u201cmicroparticles,\u201d that are derived from either the plasma membrane or an internal membrane. Vesicles can be released into the extracellular environment from cells. Cells releasing vesicles include without limitation cells that originate from, or are derived from, the ectoderm, endoderm, or mesoderm. The cells may have undergone genetic, environmental, and\/or any other variations or alterations. For example, the cell can be tumor cells. A vesicle can reflect any changes in the source cell, and thereby reflect changes in the originating cells, e.g., cells having various genetic mutations. In one mechanism, a vesicle is generated intracellularly when a segment of the cell membrane spontaneously invaginates and is ultimately exocytosed (see for example, Keller et al., 107 (2): 102-8 (2006)). Vesicles also include cell-derived structures bounded by a lipid bilayer membrane arising from both herniated evagination (blebbing) separation and sealing of portions of the plasma membrane or from the export of any intracellular membrane-bounded vesicular structure containing various membrane-associated proteins of tumor origin, including surface-bound molecules derived from the host circulation that bind selectively to the tumor-derived proteins together with molecules contained in the vesicle lumen, including but not limited to tumor-derived microRNAs or intracellular proteins. Blebs and blebbing are further described in Charras et al., , Vol. 9, No. 11, p. 730-736 (2008). A vesicle shed into circulation or bodily fluids from tumor cells may be referred to as a \u201ccirculating tumor-derived vesicle.\u201d When such vesicle is an exosome, it may be referred to as a circulating-tumor derived exosome (CTE). In some instances, a vesicle can be derived from a specific cell of origin. CTE, as with a cell-of-origin specific vesicle, typically have one or more unique biomarkers that permit isolation of the CTE or cell-of-origin specific vesicle, e.g., from a bodily fluid and sometimes in a specific manner. For example, a cell or tissue specific markers are utilized to identify the cell of origin. Examples of such cell or tissue specific markers are disclosed herein and can further be accessed in the Tissue-specific Gene Expression and Regulation (TiGER) Database, available at bioinfo.wilmer.jhu.edu\/tiger\/; Liu et al. (2008) TiGER: a database for tissue-specific gene expression and regulation. BMC Bioinformatics. 9:271; TissueDistributionDBs, available at genome dkfz-heidelberg.de\/menu\/tissue_db\/index.html.","A vesicle can have a diameter of greater than about 10 nm, 20 nm, or 30 nm. A vesicle can have a diameter of greater than 40 nm, 50 nm, 100 nm, 200 nm, 500 nm, 1000 nm or greater than 10,000 nm. A vesicle can have a diameter of about 30-1000 nm, about 30-800 nm, about 30-200 nm, or about 30-100 nm. In some embodiments, the vesicle has a diameter of less than 10,000 nm, 1000 nm, 800 nm, 500 nm, 200 nm, 100 nm, 50 nm, 40 nm, 30 nm, 20 nm or less than 10 nm. As used herein the term \u201cabout\u201d in reference to a numerical value means that variations of 10% above or below the numerical value are within the range ascribed to the specified value. Typical sizes for various types of vesicles are shown in Table 2. Vesicles can be assessed to measure the diameter of a single vesicle or any number of vesicles. For example, the range of diameters of a vesicle population or an average diameter of a vesicle population can be determined. Vesicle diameter can be assessed using methods known in the art, e.g., imaging technologies such as electron microscopy. In an embodiment, a diameter of one or more vesicles is determined using optical particle detection. See, e.g., U.S. Pat. No. 7,751,053, entitled \u201cOptical Detection and Analysis of Particles\u201d and issued Jul. 6, 2010; and U.S. Pat. No. 7,399,600, entitled \u201cOptical Detection and Analysis of Particles\u201d and issued Jul. 15, 2010.","In some embodiments, vesicles are directly assayed from a biological sample without prior isolation, purification, or concentration from the biological sample. For example, the amount of vesicles in the sample can by itself provide a biosignature that provides a diagnostic, prognostic or theranostic determination. Alternatively, the vesicle in the sample may be isolated, captured, purified, or concentrated from a sample prior to analysis. As noted, isolation, capture or purification as used herein comprises partial isolation, partial capture or partial purification apart from other components in the sample. Vesicle isolation can be performed using various techniques as described herein, e.g., chromatography, filtration, centrifugation, flow cytometry, affinity capture (e.g., to a planar surface or bead), and\/or using microfluidics.","Vesicles such as exosomes can be assessed to provide a phenotypic characterization by comparing vesicle characteristics to a reference. In some embodiments, surface antigens on a vesicle are assessed. The surface antigens can provide an indication of the anatomical origin and\/or cellular of the vesicles and other phenotypic information, e.g., tumor status. For example, wherein vesicles found in a patient sample, e.g., a bodily fluid such as blood, serum or plasma, are assessed for surface antigens indicative of colorectal origin and the presence of cancer. The surface antigens may comprise any informative biological entity that can be detected on the vesicle membrane surface, including without limitation surface proteins, lipids, carbohydrates, and other membrane components. For example, positive detection of colon derived vesicles expressing tumor antigens can indicate that the patient has colorectal cancer. As such, methods of the invention can be used to characterize any disease or condition associated with an anatomical or cellular origin, by assessing, for example, disease-specific and cell-specific biomarkers of one or more vesicles obtained from a subject.","In another embodiment, one or more vesicle payloads are assessed to provide a phenotypic characterization. The payload with a vesicle comprises any informative biological entity that can be detected as encapsulated within the vesicle, including without limitation proteins and nucleic acids, e.g., genomic or cDNA, mRNA, or functional fragments thereof, as well as microRNAs (miRs). In addition, methods of the invention are directed to detecting vesicle surface antigens (in addition or exclusive to vesicle payload) to provide a phenotypic characterization. For example, vesicles can be characterized by using binding agents (e.g., antibodies or aptamers) that are specific to vesicle surface antigens, and the bound vesicles can be further assessed to identify one or more payload components disclosed therein. As described herein, the levels of vesicles with surface antigens of interest or with payload of interest can be compared to a reference to characterize a phenotype. For example, overexpression in a sample of cancer-related surface antigens or vesicle payload, e.g., a tumor associated mRNA or microRNA, as compared to a reference, can indicate the presence of cancer in the sample. The biomarkers assessed can be present or absent, increased or reduced based on the selection of the desired target sample and comparison of the target sample to the desired reference sample. Non-limiting examples of target samples include: disease; treated\/not-treated; different time points, such as a in a longitudinal study; and non-limiting examples of reference sample: non-disease; normal; different time points; and sensitive or resistant to candidate treatment(s).","MicroRNA","Various biomarker molecules can be assessed in biological samples or vesicles obtained from such biological samples. MicroRNAs comprise one class biomarkers assessed via methods of the invention. MicroRNAs, also referred to herein as miRNAs or miRs, are short RNA strands approximately 21-23 nucleotides in length. MiRNAs are encoded by genes that are transcribed from DNA but are not translated into protein and thus comprise non-coding RNA. The miRs are processed from primary transcripts known as pri-miRNA to short stem-loop structures called pre-miRNA and finally to the resulting single strand miRNA. The pre-miRNA typically forms a structure that folds back on itself in self-complementary regions. These structures are then processed by the nuclease Dicer in animals or DCL1 in plants. Mature miRNA molecules are partially complementary to one or more messenger RNA (mRNA) molecules and can function to regulate translation of proteins. Identified sequences of miRNA can be accessed at publicly available databases, such as www.microRNA.org, www.mirbase.org, or www.mirz.unibas.ch\/cgi\/miRNA.cgi.","miRNAs are generally assigned a number according to the naming convention \u201cmir-[number].\u201d The number of a miRNA is assigned according to its order of discovery relative to previously identified miRNA species. For example, if the last published miRNA was mir-121, the next discovered miRNA will be named mir-122, etc. When a miRNA is discovered that is homologous to a known miRNA from a different organism, the name can be given an optional organism identifier, of the form [organism identifier]-mir-[number]. Identifiers include hsa for and mmu for . For example, a human homolog to mir-121 might be referred to as hsa-mir-121 whereas the mouse homolog can be referred to as mmu-mir-121.","Mature microRNA is commonly designated with the prefix \u201cmiR\u201d whereas the gene or precursor miRNA is designated with the prefix \u201cmir.\u201d For example, mir-121 is a precursor for miR-121. When differing miRNA genes or precursors are processed into identical mature miRNAs, the genes\/precursors can be delineated by a numbered suffix. For example, mir-121-1 and mir-121-2 can refer to distinct genes or precursors that are processed into miR-121. Lettered suffixes are used to indicate closely related mature sequences. For example, mir-121a and mir-121b can be processed to closely related miRNAs miR-121a and miR-121b, respectively. In the context of the invention, any microRNA (miRNA or miR) designated herein with the prefix mir-* or miR-* is understood to encompass both the precursor and\/or mature species, unless otherwise explicitly stated otherwise.","Sometimes it is observed that two mature miRNA sequences originate from the same precursor. When one of the sequences is more abundant that the other, a \u201c*\u201d suffix can be used to designate the less common variant. For example, miR-121 would be the predominant product whereas miR-121* is the less common variant found on the opposite arm of the precursor. If the predominant variant is not identified, the miRs can be distinguished by the suffix \u201c5p\u201d for the variant from the 5\u2032 arm of the precursor and the suffix \u201c3p\u201d for the variant from the 3\u2032 arm. For example, miR-121-5p originates from the 5\u2032 arm of the precursor whereas miR-121-3p originates from the 3\u2032 arm. Less commonly, the 5p and 3p variants are referred to as the sense (\u201cs\u201d) and anti-sense (\u201cas\u201d) forms, respectively. For example, miR-121-5p may be referred to as miR-121-s whereas miR-121-3p may be referred to as miR-121-as.","The above naming conventions have evolved over time and are general guidelines rather than absolute rules. For example, the let\u2014 and lin\u2014 families of miRNAs continue to be referred to by these monikers. The mir\/miR convention for precursor\/mature forms is also a guideline and context should be taken into account to determine which form is referred to. Further details of miR naming can be found at www.mirbase.org or Ambros et al., A uniform system for microRNA annotation, RNA 9:277-279 (2003).","Plant miRNAs follow a different naming convention as described in Meyers et al., Plant Cell. 2008 20(12):3186-3190.","A number of miRNAs are involved in gene regulation, and miRNAs are part of a growing class of non-coding RNAs that is now recognized as a major tier of gene control. In some cases, miRNAs can interrupt translation by binding to regulatory sites embedded in the 3\u2032-UTRs of their target mRNAs, leading to the repression of translation. Target recognition involves complementary base pairing of the target site with the miRNA's seed region (positions 2-8 at the miRNA's 5\u2032 end), although the exact extent of seed complementarity is not precisely determined and can be modified by 3\u2032 pairing. In other cases, miRNAs function like small interfering RNAs (siRNA) and bind to perfectly complementary mRNA sequences to destroy the target transcript.","Characterization of a number of miRNAs indicates that they influence a variety of processes, including early development, cell proliferation and cell death, apoptosis and fat metabolism. For example, some miRNAs, such as lin-4, let-7, mir-14, mir-23, and bantam, have been shown to play critical roles in cell differentiation and tissue development. Others are believed to have similarly important roles because of their differential spatial and temporal expression patterns.","The miRNA database available at miRBase (www.mirbase.org) comprises a searchable database of published miRNA sequences and annotation. Further information about miRBase can be found in the following articles, each of which is incorporated by reference in its entirety herein: Griffiths-Jones et al., miRBase: tools for microRNA genomics. NAR 2008 36(Database Issue):D154-D158; Griffiths-Jones et al., miRBase: microRNA sequences, targets and gene nomenclature. NAR 2006 34(Database Issue):D140-D144; and Griffiths-Jones, S. The microRNA Registry. NAR 2004 32(Database Issue):D109-D111. Representative miRNAs contained in Release 16 of miRBase, made available September 2010.","As described herein, microRNAs are known to be involved in cancer and other diseases and can be assessed in order to characterize a phenotype in a sample. See, e.g., Ferracin et al., Micromarkers: miRNAs in cancer diagnosis and prognosis, Exp Rev Mol Diag, April 2010, Vol. 10, No. 3, Pages 297-308; Fabbri, miRNAs as molecular biomarkers of cancer, Exp Rev Mol Diag, May 2010, Vol. 10, No. 4, Pages 435-444. Techniques to isolate and characterize vesicles and miRs are known to those of skill in the art. In addition to the methodology presented herein, additional methods can be found in U.S. Pat. No. 7,888,035, entitled \u201cMETHODS FOR ASSESSING RNA PATTERNS\u201d and issued Feb. 15, 2011; and International Patent Application Nos. PCT\/US2010\/058461, entitled \u201cMETHODS AND SYSTEMS FOR ISOLATING, STORING, AND ANALYZING VESICLES\u201d and filed Nov. 30, 2010; and PCT\/US2011\/021160, entitled \u201cDETECTION OF GASTROINTESTINAL DISORDERS\u201d and filed Jan. 13, 2011; each of which applications are incorporated by reference herein in their entirety.","Circulating Biomarkers","Circulating biomarkers include biomarkers that are detectable in body fluids, such as blood, plasma, serum. Examples of circulating cancer biomarkers include cardiac troponin T (cTnT), prostate specific antigen (PSA) for prostate cancer and CA125 for ovarian cancer. Circulating biomarkers according to the invention include any appropriate biomarker that can be detected in bodily fluid, including without limitation protein, nucleic acids, e.g., DNA, mRNA and microRNA, lipids, carbohydrates and metabolites. Circulating biomarkers can include biomarkers that are not associated with cells, such as biomarkers that are membrane associated, embedded in membrane fragments, part of a biological complex, or free in solution. In one embodiment, circulating biomarkers are biomarkers that are associated with one or more vesicles present in the biological fluid of a subject.","Circulating biomarkers have been identified for use in characterization of various phenotypes. See, e.g., Ahmed N, et al., Proteomic-based identification of haptoglobin-1 precursor as a novel circulating biomarker of ovarian cancer. Br. J. Cancer 2004; Mathelin et al., Circulating proteinic biomarkers and breast cancer, Gynecol Obstet Fertil. 2006 July-August; 34(7-8):638-46. Epub 2006 Jul. 28; Ye et al., Recent technical strategies to identify diagnostic biomarkers for ovarian cancer. Expert Rev Proteomics. 2007 February; 4(1):121-31; Carney, Circulating oncoproteins HER2\/neu, EGFR and CAIX (MN) as novel cancer biomarkers. Expert Rev Mol Diagn. 2007 May; 7(3):309-19; Gagnon, Discovery and application of protein biomarkers for ovarian cancer, Curr Opin Obstet. Gynecol. 2008 February; 20(1):9-13; Pasterkamp et al., Immune regulatory cells: circulating biomarker factories in cardiovascular disease. Clin Sci (Load). 2008 August; 115(4):129-31; Fabbri, miRNAs as molecular biomarkers of cancer, Exp Rev Mol Diag, May 2010, Vol. 10, No. 4, Pages 435-444; PCT Patent Publication WO\/2007\/088537; U.S. Pat. Nos. 7,745,150 and 7,655,479; U.S. Patent Publications 20110008808, 20100330683, 20100248290, 20100222230, 20100203566, 20100173788, 20090291932, 20090239246, 20090226937, 20090111121, 20090004687, 20080261258, 20080213907, 20060003465, 20050124071, and 20040096915, each of which publication is incorporated herein by reference in its entirety.","Vesicle Enrichment","A vesicle or a population of vesicles may be isolated, purified, concentrated or otherwise enriched prior to and\/or during analysis. Unless otherwise specified, the terms \u201cpurified,\u201d \u201cisolated,\u201d \u201c \u201d as used herein in reference to vesicles or biomarker components include partial or complete purification or isolation of such components from a cell or organism. Analysis of a vesicle can include quantitiating the amount one or more vesicle populations of a biological sample. For example, a heterogeneous population of vesicles can be quantitated, or a homogeneous population of vesicles, such as a population of vesicles with a particular biomarker profile, a particular biosignature, or derived from a particular cell type can be isolated from a heterogeneous population of vesicles and quantitated. Analysis of a vesicle can also include detecting, quantitatively or qualitatively, one or more particular biomarker profile or biosignature of a vesicle, as described herein.","A vesicle can be stored and archived, such as in a bio-fluid bank and retrieved for analysis as necessary. A vesicle may also be isolated from a biological sample that has been previously harvested and stored from a living or deceased subject. In addition, a vesicle may be isolated from a biological sample which has been collected as described in King et al., 7(5): 198-204 (2005). A vesicle can be isolated from an archived or stored sample. Alternatively, a vesicle may be isolated from a biological sample and analyzed without storing or archiving of the sample. Furthermore, a third party may obtain or store the biological sample, or obtain or store the vesicle for analysis.","An enriched population of vesicles can be obtained from a biological sample. For example, vesicles may be concentrated or isolated from a biological sample using size exclusion chromatography, density gradient centrifugation, differential centrifugation, nanomembrane ultrafiltration, immunoabsorbent capture, affinity purification, microfluidic separation, or combinations thereof.","Size exclusion chromatography, such as gel permeation columns, centrifugation or density gradient centrifugation, and filtration methods can be used. For example, a vesicle can be isolated by differential centrifugation, anion exchange and\/or gel permeation chromatography (for example, as described in U.S. Pat. Nos. 6,899,863 and 6,812,023), sucrose density gradients, organelle electrophoresis (for example, as described in U.S. Pat. No. 7,198,923), magnetic activated cell sorting (MACS), or with a nanomembrane ultrafiltration concentrator. Various combinations of isolation or concentration methods can be used.","Highly abundant proteins, such as albumin and immunoglobulin, may hinder isolation of vesicles from a biological sample. For example, a vesicle can be isolated from a biological sample using a system that utilizes multiple antibodies that are specific to the most abundant proteins found in a biological sample, such as blood. Such a system can remove up to several proteins at once, thus unveiling the lower abundance species such as cell-of-origin specific vesicles.","This type of system can be used for isolation of vesicles from biological samples such as blood, cerebrospinal fluid or urine. The isolation of vesicles from a biological sample may also be enhanced by high abundant protein removal methods as described in Chromy et al. 2004; 3:1120-1127. In another embodiment, the isolation of vesicles from a biological sample may also be enhanced by removing serum proteins using glycopeptide capture as described in Zhang et al, 2005; 4:144-155. In addition, vesicles from a biological sample such as urine may be isolated by differential centrifugation followed by contact with antibodies directed to cytoplasmic or anti-cytoplasmic epitopes as described in Pisitkun et al., 2004; 101:13368-13373.","Isolation or enrichment of a vesicle from a biological sample can also be enhanced by use of sonication (for example, by applying ultrasound), detergents, other membrane-activating agents, or any combination thereof. For example, ultrasonic energy can be applied to a potential tumor site, and without being bound by theory, release of vesicles from a tissue can be increased, allowing an enriched population of vesicles that can be analyzed or assessed from a biological sample using one or more methods disclosed herein.","Sample Handling","With methods of detecting circulating biomarkers as described here, e.g., antibody affinity isolation, the consistency of the results can be optimized as necessary using various concentration or isolation procedures. Such steps can include agitation such as shaking or vortexing, different isolation techniques such as polymer based isolation, e.g., with PEG, and concentration to different levels during filtration or other steps. It will be understood by those in the art that such treatments can be applied at various stages of testing the vesicle containing sample. In one embodiment, the sample itself, e.g., a bodily fluid such as plasma or serum, is vortexed. In some embodiments, the sample is vortexed after one or more sample treatment step, e.g., vesicle isolation, has occurred. Agitation can occur at some or all appropriate sample treatment steps as desired. Additives can be introduced at the various steps to improve the process, e.g., to control aggregation or degradation of the biomarkers of interest.","The results can also be optimized as desirable by treating the sample with various agents. Such agents include additives to control aggregation and\/or additives to adjust pH or ionic strength. Additives that control aggregation include blocking agents such as bovine serum albumin (BSA) and milk, chaotropic agents such as guanidium hydro chloride, and detergents or surfactants. Useful ionic detergents include sodium dodecyl sulfate (SDS, sodium lauryl sulfate (SLS)), sodium laureth sulfate (SLS, sodium lauryl ether sulfate (SLES)), ammonium lauryl sulfate (ALS), cetrimonium bromide, cetrimonium chloride, cetrimonium stearate, and the like. Useful non-ionic (zwitterionic) detergents include polyoxyethylene glycols, polysorbate 20 (also known as Tween 20), other polysorbates (e.g., 40, 60, 65, 80, etc), Triton-X (e.g., X100, X114), 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonate (CHAPS), CHAPSO, deoxycholic acid, sodium deoxycholate, NP-40, glycosides, octyl-thio-glucosides, maltosides, and the like. In some embodiments, Pluronic F-68, a surfactant shown to reduce platelet aggregation, is used to treat samples containing vesicles during isolation and\/or detection. F68 can be used from a 0.1% to 10% concentration, e.g., a 1%, 2.5% or 5% concentration. The pH and\/or ionic strength of the solution can be adjusted with various acids, bases, buffers or salts, including without limitation sodium chloride (NaCl), phosphate-buffered saline (PBS), tris-buffered saline (TBS), sodium phosphate, potassium chloride, potassium phosphate, sodium citrate and saline-sodium citrate (SSC) buffer. In some embodiments, NaCl is added at a concentration of 0.1% to 10%, e.g., 1%, 2.5% or 5% final concentration. In some embodiments, Tween 20 is added to 0.005 to 2% concentration, e.g., 0.05%, 0.25% or 0.5% final concentration. Blocking agents for use with the invention comprise inert proteins, e.g., milk proteins, non-fat dry milk protein, albumin, BSA, casein, or serum such as newborn calf serum (NBCS), goat serum, rabbit serum or salmon serum. The proteins can be added at a 0.1% to 10% concentration, e.g., 1%, 2%, 3%, 3.5%, 4%, 5%, 6%, 7%, 8%, 9% or 10% concentration. In some embodiments, BSA is added to 0.1% to 10% concentration, e.g., 1%, 2%, 3%, 3.5%, 4%, 5%, 6%, 7%, 8%, 9% or 10% concentration. In an embodiment, the sample is treated according to the methodology presented in U.S. patent application Ser. No. 11\/632,946, filed Jul. 13, 2005, which application is incorporated herein by reference in its entirety. Commercially available blockers may be used, such as SuperBlock, StartingBlock, Protein-Free from Pierce (a division of Thermo Fisher Scientific, Rockford, Ill.). In some embodiments, SSC\/detergent (e.g., 20\u00d7SSC with 0.5% Tween 20 or 0.1% Triton-X 100) is added to 0.1% to 10% concentration, e.g., at 1.0% or 5.0% concentration.","The methods of detecting vesicles and other circulating biomarkers can be optimized as desired with various combinations of protocols and treatments as described herein. A detection protocol can be optimized by various combinations of agitation, isolation methods, and additives. In some embodiments, the patient sample is vortexed before and after isolation steps, and the sample is treated with blocking agents including BSA and\/or F68. Such treatments may reduce the formation of large aggregates or protein or other biological debris and thus provide a more consistent detection reading.","Filters","A vesicle can be isolated from a biological sample by filtering a biological sample from a subject through a filtration module and collecting from the filtration module a retentate comprising the vesicle, thereby isolating the vesicle from the biological sample. The method can comprise filtering a biological sample from a subject through a filtration module comprising a filter; and collecting from the filtration module a retentate comprising the vesicle, thereby isolating the vesicle from the biological sample. In one embodiment, the filter retains molecules greater than about 100 kiloDaltons.","The method can further comprise determining a biosignature of the vesicle. The method can also further comprise applying the retentate to a plurality of substrates, wherein each substrate is coupled to one or more capture agents, and each subset of the plurality of substrates comprises a different capture agent or combination of capture agents than another subset of the plurality of substrates.","Also provided herein is a method of determining a biosignature of a vesicle in a sample comprising: filtering a biological sample from a subject with a disorder through a filtration module, collecting from the filtration module a retentate comprising one or more vesicles, and determining a biosignature of the one or more vesicles. In one embodiment, the filtration module comprises a filter that retains molecules greater than about 100 or 150 kiloDaltons.","The method disclosed herein can further comprise characterizing a phenotype in a subject by filtering a biological sample from a subject through a filtration module, collecting from the filtration module a retentate comprising one or more vesicles; detecting a biosignature of the one or more vesicles; and characterizing a phenotype in the subject based on the biosignature, wherein characterizing is with at least 70% sensitivity. In some embodiments, characterizing comprises determining an amount of one or more vesicle having the biosignature. Furthermore, the characterizing can be from about 80% to 100% sensitivity.","Also provided herein is a method for multiplex analysis of a plurality of vesicles. In some embodiments, the method comprises filtering a biological sample from a subject through a filtration module; collecting from the filtration module a retentate comprising the plurality of vesicles, applying the plurality of vesicles to a plurality of capture agents, wherein the plurality of capture agents is coupled to a plurality of substrates, and each subset of the plurality of substrates is differentially labeled from another subset of the plurality of substrates; capturing at least a subset of the plurality of vesicles; and determining a biosignature for at least a subset of the captured vesicles. In one embodiment, each substrate is coupled to one or more capture agents, and each subset of the plurality of substrates comprises a different capture agent or combination of capture agents as compared to another subset of the plurality of substrates. In some embodiments, at least a subset of the plurality of substrates is intrinsically labeled, such as comprising one or more labels. The substrate can be a particle or bead, or any combination thereof. In one embodiment, the filtration module comprises a filter that retains molecules greater than about 100 or 150 kiloDaltons.","In some embodiments, the method for multiplex analysis of a plurality of vesicles comprises filtering a biological sample from a subject through a filtration module, wherein the filtration module comprises a filter that retains molecules greater than about 100 kiloDaltons; collecting from the filtration module a retentate comprising the plurality of vesicles; applying the plurality of vesicles to a plurality of capture agents, wherein the plurality of capture agents is coupled to a microarray; capturing at least a subset of the plurality of vesicles on the microarray; and determining a biosignature for at least a subset of the captured vesicles. In one embodiment, the filtration module comprises a filter that retains molecules greater than about 100 or 150 kiloDaltons.","The biological sample can be clarified prior to isolation by filtration. For example, non-vesicle components such as cellular debris can be removed. The clarification can be by low-speed centrifugation, such as at about 5,000\u00d7g, 4,000\u00d7g, 3,000\u00d7g, 2,000\u00d7g, 1,000\u00d7g, or less. The supernatant, or clarified biological sample, containing the vesicle can then be collected and filtered to isolate the vesicle from the clarified biological sample. In some embodiments, the biological sample is not clarified prior to isolation of a vesicle by filtration.","In some embodiments, isolation of a vesicle from a sample does not use high-speed centrifugation, such as ultracentrifugation. For example, isolation may not require the use of centrifugal speeds, such as about 100,000\u00d7g or more. In some embodiments, isolation of a vesicle from a sample uses speeds of less than 50,000\u00d7g, 40,000\u00d7g, 30,000\u00d7g, 20,000\u00d7g, 15,000\u00d7g, 12,000\u00d7g, or 10,000\u00d7g.","The filtration module utilized to isolate the vesicle from the biological sample can be a fiber-based filtration cartridge. For example, the fiber can be a hollow polymeric fiber, such as a polypropylene hollow fiber. A biological sample can be introduced into the filtration module by pumping the sample fluid, such as a biological fluid as disclosed herein, into the module with a pump device, such as a peristaltic pump. The pump flow rate can vary, such as at about 0.25, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5, 5, 6, 7, 8, 9, or 10 mL\/minute.","The filtration module can be a membrane filtration module. For example, the membrane filtration module can comprise a filter disc membrane, such as a hydrophilic polyvinylidene difluoride (PVDF) filter disc membrane housed in a stirred cell apparatus (e.g., comprising a magnetic stirrer). In some embodiments, the sample moves through the filter as a result of a pressure gradient established on either side of the filter membrane.","The filter can comprise a material having low hydrophobic absorptivity and\/or high hydrophilic properties. For example, the filter can have an average pore size for vesicle retention and permeation of most proteins as well as a surface that is hydrophilic, thereby limiting protein adsorption. For example, the filter can comprise a material selected from the group consisting of polypropylene, PVDF, polyethylene, polyfluoroethylene, cellulose, secondary cellulose acetate, polyvinylalcohol, and ethylenevinyl alcohol (EVAL\u00ae, Kuraray Co., Okayama, Japan). Additional materials that can be used in a filter include, but are not limited to, polysulfone and polyethersulfone.","The filtration module can have a filter that retains molecules greater than about 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160, 170, 180, 190, 200, 250, 300, 400, or 500 kiloDaltons (kDa), such as a filter that has a MWCO (molecular weight cut off) of about 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160, 170, 180, 190, 200, 250, 300, 400, or 500. In some embodiments, the filter within the filtration module has an average pore diameter of about 0.01 \u03bcm to about 0.15 \u03bcm, and in some embodiments from about 0.05 \u03bcm to about 0.12 \u03bcm. In some embodiments, the filter has an average pore diameter of about 0.06 \u03bcm, 0.07. \u03bcm, 0.08 \u03bcm, 0.09 \u03bcm, 0.1 \u03bcm, or 0.11 \u03bcm.","The filtration module can be a commerically available column, such as a column typically used for concentrating proteins or for isoatling proteins. Examples include, but are not limited to, columns from Millpore (Billerica, Mass.), such as Amicon\u00ae centrifugal filters, or from Pierce\u00ae (Rockford, Ill.), such as Pierce Concentrator filter devices. Useful columns from Pierce include disposable ultrafiltration centrifugal devices with a MWCO of 9 kDa, 20 kDa and\/or 150 kDa. These concentrators consist of a high-performance regenerated cellulose membrane welded to a conical device. The filters can be as described in U.S. Pat. Nos. 6,269,957 or 6,357,601, both of which applications are incorporated by reference in their entirety herein.","The retentate comprising the isolated vesicle can be collected from the filtration module. The retentate can be collected by flushing the retentate from the filter. Selection of a filter composition having hydrophilic surface properties, thereby limiting protein adsorption, can be used, without being bound by theory, for easier collection of the retentate and minimize use of harsh or time-consuming collection techniques.","The collected retentate can then be used subsequent analysis, such as assessing a biosignature of one or more vesicles in the retentate, as further described herein. The analysis can be directly performed on the collected retentate. Alternatively, the collected retentate can be further concentrated or purified, prior to analysis of one or more vesicles. For example, the retentate can be further concentrated or vesicles further isolated from the retentate using size exclusion chromatography, density gradient centrifugation, differential centrifugation, immunoabsorbent capture, affinity purification, microfluidic separation, or combinations thereof, such as described herein. In some embodiments, the retentate can undergo another step of filtration. Alternatively, prior to isolation of a vesicle using a filter, the vesicle is concentrated or isolated using size exclusion chromatography, density gradient centrifugation, differential centrifugation, immunoabsorbent capture, affinity purification, microfluidic separation, or combinations thereof.","For example, prior to filtering a biological sample through a filtration module with a filter that retains molecules greater than about 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160, 170, 180, 190, 200, 250, 300, 400, or 500 kiloDaltons (kDa), such as a filter that has a MWCO (molecular weight cut off) of about 50, 60, 70, 80, 90, 100, 110, 120, 130, 140, 150, 160, 170, 180, 190, 200, 250, 300, 400, or 500, the biological sample may first be filtered through a filter having a porosity or pore size of between about 0.01 \u03bcm to about 2 \u03bcm, about 0.05 \u03bcm to about 1.5 \u03bcm, In some embodiments, the filter has a pore size of about 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9 or 2.0 \u03bcm. The filter may be a syringe filter. Thus, in one embodiment, the method comprises filtering the biological sample through a filter, such as a syringe filter, wherein the syringe filter has a porosity of greater than about 1 \u03bcm, prior to filtering the sample through a filtration module comprising a filter that retains molecules greater than about 100 or 150 kiloDaltons. In an embodiment, the filter is 1.2 \u03bcM filter and the filtration is followed by passage of the sample through a 7 ml or 20 ml concentrator column with a 150 kDa cutoff.","The filtration module can be a component of a microfluidic device. Microfluidic devices, which may also be referred to as \u201clab-on-a-chip\u201d systems, biomedical micro-electro-mechanical systems (bioMEMs), or multicomponent integrated systems, can be used for isolating, and analyzing, vesicles. Such systems miniaturize and compartmentalize processes that allow for binding of vesicles, detection of biomarkers, and other processes, such as further described herein","A microfluidic device can also be used for isolation of a vesicle by comprising a filtration module. For example, a microfluidic device can use one more channels for isolating a vesicle from a biological sample based on size from a biological sample. A biological sample can be introduced into one or more microfluidic channels, which selectively allows the passage of vesicles. The microfluidic device can further comprise binding agents, or more than one filtration module to select vesicles based on a property of the vesicles, for example, size, shape, deformability, biomarker profile, or biosignature.","Binding Agents","A binding agent is an agent that binds to a circulating biomarker, such as a vesicle or a component of a vesicle. The binding agent can be used as a capture agent and\/or a detection agent. A capture agent can bind and capture a circulating biomarker, such as by binding a component or biomarker of a vesicle. For example, the capture agent can be a capture antibody or capture antigen that binds to an antigen on a vesicle. A detection agent can bind to a circulating biomarker thereby facilitating detection of the biomarker. For example, a capture agent comprising an antigen or aptamer that is sequestered to a substrate can be used to capture a vesicle in a sample, and a detection agent comprising an antigen or aptamer that carries a label can be used to detect the captured vesicle via detection of the detection agent's label. In some embodiments, a vesicle is assessed using capture and detection agents that recognize the same vesicle biomarkers. For example, a vesicle population can be captured using a tetraspanin such as by using an anti-CD9 antibody bound to a substrate, and the captured vesicles can be detected using a fluorescently labeled anti-CD9 antibody to label the captured vesicles. In other embodiments, a vesicle is assessed using capture and detection agents that recognize different vesicle biomarkers. For example, a vesicle population can be captured using a cell-specific marker such as by using an anti-PCSA antibody bound to a substrate, and the captured vesicles can be detected using a fluorescently labeled anti-CD9 antibody to label the captured vesicles Similarly, the vesicle population can be captured using a general vesicle marker such as by using an anti-CD9 antibody bound to a substate, and the captured vesicles can be detected using a fluorescently labeled antibody to a cell-specific or disease specific marker to label the captured vesicles.","In one embodiment, a vesicle is captured using a capture agent that binds to a biomarker on a vesicle. The capture agent can be coupled to a substrate and used to isolate a vesicle, as further described herein. In one embodiment, a capture agent is used for affinity capture or isolation of a vesicle present in a substance or sample.","A binding agent can be used after a vesicle is concentrated or isolated from a biological sample. For example, a vesicle can first be isolated from a biological sample before a vesicle with a specific biosignature is isolated or detected. The vesicle with a specific biosignature can be isolated or detected using a binding agent for the biomarker. A vesicle with the specific biomarker can be isolated or detected from a heterogeneous population of vesicles. Alternatively, a binding agent may be used on a biological sample comprising vesicles without a prior isolation or concentration step. For example, a binding agent is used to isolate or detect a vesicle with a specific biosignature directly from a biological sample.","A binding agent can be a nucleic acid, protein, or other molecule that can bind to a component of a vesicle. The binding agent can comprise DNA, RNA, monoclonal antibodies, polyclonal antibodies, Fabs, Fab\u2032, single chain antibodies, synthetic antibodies, aptamers (DNA\/RNA), peptoids, zDNA, peptide nucleic acids (PNAs), locked nucleic acids (LNAs), lectins, synthetic or naturally occurring chemical compounds (including but not limited to drugs, labeling reagents), dendrimers, or a combination thereof. For example, the binding agent can be a capture antibody. In embodiments of the invention, the binding agent comprises a membrane protein labeling agent. See, e.g., the membrane protein labeling agents disclosed in Alroy et al., US. Patent Publication US 2005\/0158708. In an embodiment, vesicles are isolated or captured as described herein, and one or more membrane protein labeling agent is used to detect the vesicles.","In some instances, a single binding agent can be employed to isolate or detect a vesicle. In other instances, a combination of different binding agents may be employed to isolate or detect a vesicle. For example, at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 different binding agents may be used to isolate or detect a vesicle from a biological sample. Furthermore, the one or more different binding agents for a vesicle can form a biosignature of a vesicle, as further described below.","Different binding agents can also be used for multiplexing. For example, isolation or detection of more than one population of vesicles can be performed by isolating or detecting each vesicle population with a different binding agent. Different binding agents can be bound to different particles, wherein the different particles are labeled. In another embodiment, an array comprising different binding agents can be used for multiplex analysis, wherein the different binding agents are differentially labeled or can be ascertained based on the location of the binding agent on the array. Multiplexing can be accomplished up to the resolution capability of the labels or detection method, such as described below. The binding agents can be used to detect the vesicles, such as for detecting cell-of-origin specific vesicles. A binding agent or multiple binding agents can themselves form a binding agent profile that provides a biosignature for a vesicle. One or more binding agents can be selected from . For example, if a vesicle population is detected or isolated using two, three, four or more binding agents in a differential detection or isolation of a vesicle from a heterogeneous population of vesicles, the particular binding agent profile for the vesicle population provides a biosignature for the particular vesicle population. The vesicle can be detected using any number of binding agents in a multiplex fashion. Thus, the binding agent can also be used to form a biosignature for a vesicle. The biosignature can be used to characterize a phenotype.","The binding agent can be a lectin. Lectins are proteins that bind selectively to polysaccharides and glycoproteins and are widely distributed in plants and animals. For example, lectins such as those derived from in the form of agglutinin (\u201cGNA\u201d), in the form of agglutinin (\u201cNPA\u201d) and the blue green algae called \u201ccyanovirin\u201d (Boyd et al. 41(7): 15211530, 1997; Hammar et al. 724: 166 169, 1994; Kaku et al. 279(2): 298 304, 1990) can be used to isolate a vesicle. These lectins can bind to glycoproteins having a high mannose content (Chervenak et al. 34(16): 5685 5695, 1995). High mannose glycoprotein refers to glycoproteins having mannose-mannose linkages in the form of \u03b1-1\u21923 or \u03b1-1\u21926 mannose-mannose linkages.","The binding agent can be an agent that binds one or more lectins. Lectin capture can be applied to the isolation of the biomarker cathepsin D since it is a glycosylated protein capable of binding the lectins agglutinin (GNA) and concanavalin A (ConA).","Methods and devices for using lectins to capture vesicles are described in International Patent Applications PCT\/US2010\/058461, entitled \u201cMETHODS AND SYSTEMS FOR ISOLATING, STORING, AND ANALYZING VESICLES\u201d and filed Nov. 30, 2010; PCT\/US2009\/066626, entitled \u201cAFFINITY CAPTURE OF CIRCULATING BIOMARKERS\u201d and filed Dec. 3, 2009; PCT\/US2010\/037467, entitled \u201cMETHODS AND MATERIALS FOR ISOLATING EXOSOMES\u201d and filed Jun. 4, 2010; and PCT\/US2007\/006101, entitled \u201cEXTRACORPOREAL REMOVAL OF MICROVESICULAR PARTICLES\u201d and filed Mar. 9, 2007, each of which applications is incorporated by reference herein in its entirety.","The binding agent can be an antibody. For example, a vesicle may be isolated using one or more antibodies specific for one or more antigens present on the vesicle. For example, a vesicle can have CD63 on its surface, and an antibody, or capture antibody, for CD63 can be used to isolate the vesicle. Alternatively, a vesicle derived from a tumor cell can express EpCam, the vesicle can be isolated using an antibody for EpCam and CD63. Other antibodies for isolating vesicles can include an antibody, or capture antibody, to CD9, PSCA, TNFR, CD63, B7H3, MFG-E8, EpCam, Rab, CD81, STEAP, PCSA, PSMA, or 5T4. Other antibodies for isolating vesicles can include an antibody, or capture antibody, to DR3, STEAP, epha2, TMEM211, MFG-E8, Tissue Factor (TF), unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, or TETS.","In some embodiments, the capture agent is an antibody to CD9, CD63, CD81, PSMA, PCSA, B7H3, EpCam, PSCA, ICAM, STEAP, or EGFR. The capture agent can also be used to identify a biomarker of a vesicle. For example, a capture agent such as an antibody to CD9 would identify CD9 as a biomarker of the vesicle. In some embodiments, a plurality of capture agents can be used, such as in multiplex analysis. The plurality of captures agents can comprise binding agents to one or more of: CD9, CD63, CD81, PSMA, PCSA, B7H3, EpCam, PSCA, ICAM, STEAP, and EGFR. In some embodiments, the plurality of capture agents comprise binding agents to CD9, CD63, CD81, PSMA, PCSA, B7H3, MFG-E8, and\/or EpCam. In yet other embodiments, the plurality of capture agents comprises binding agents to CD9, CD63, CD81, PSMA, PCSA, B7H3, EpCam, PSCA, ICAM, STEAP, and\/or EGFR. The plurality of capture agents comprises binding agents to TMEM211, MFG-E8, Tissue Factor (TF), and\/or CD24.","The antibodies referenced herein can be immunoglobulin molecules or immunologically active portions of immunoglobulin molecules, i.e., molecules that contain an antigen binding site that specifically binds an antigen and synthetic antibodies. The immunoglobulin molecules can be of any class (e.g., IgG, IgE, IgM, IgD or IgA) or subclass of immunoglobulin molecule. Antibodies include, but are not limited to, polyclonal, monoclonal, bispecific, synthetic, humanized and chimeric antibodies, single chain antibodies, Fab fragments and F(ab\u2032)fragments, Fv or Fv\u2032 portions, fragments produced by a Fab expression library, anti-idiotypic (anti-Id) antibodies, or epitope-binding fragments of any of the above. An antibody, or generally any molecule, \u201cbinds specifically\u201d to an antigen (or other molecule) if the antibody binds preferentially to the antigen, and, e.g., has less than about 30%, 20%, 10%, 5% or 1% cross-reactivity with another molecule.","The binding agent can also be a polypeptide or peptide. Polypeptide is used in its broadest sense and may include a sequence of subunit amino acids, amino acid analogs, or peptidomimetics. The subunits may be linked by peptide bonds. The polypeptides may be naturally occurring, processed forms of naturally occurring polypeptides (such as by enzymatic digestion), chemically synthesized or recombinantly expressed. The polypeptides for use in the methods of the present invention may be chemically synthesized using standard techniques. The polypeptides may comprise D-amino acids (which are resistant to L-amino acid-specific proteases), a combination of D- and L-amino acids, \u03b2 amino acids, or various other designer or non-naturally occurring amino acids (e.g., \u03b2-methyl amino acids, C\u03b1-methyl amino acids, and N\u03b1-methyl amino acids, etc.) to convey special properties. Synthetic amino acids may include ornithine for lysine, and norleucine for leucine or isoleucine. In addition, the polypeptides can have peptidomimetic bonds, such as ester bonds, to prepare polypeptides with novel properties. For example, a polypeptide may be generated that incorporates a reduced peptide bond, i.e., R\u2014CH\u2014NH\u2014R, where Rand Rare amino acid residues or sequences. A reduced peptide bond may be introduced as a dipeptide subunit. Such a polypeptide would be resistant to protease activity, and would possess an extended half-live in vivo. Polypeptides can also include peptoids (N-substituted glycines), in which the side chains are appended to nitrogen atoms along the molecule's backbone, rather than to the \u03b1-carbons, as in amino acids. Polypeptides and peptides are intended to be used interchangeably throughout this application, i.e. where the term peptide is used, it may also include polypeptides and where the term polypeptides is used, it may also include peptides.","A vesicle may be isolated, captured or detected using a binding agent. The binding agent can be an agent that binds a vesicle \u201chousekeeping protein,\u201d or general vesicle biomarker. The biomarker can be CD63, CD9, CD81, CD82, CD37, CD53, Rab-5b, Annexin V or MFG-E8. Tetraspanins, a family of membrane proteins with four transmembrane domains, can be used as general vesicle markers. The tetraspanins include CD151, CD53, CD37, CD82, CD81, CD9 and CD63. There have been over 30 tetraspanins identified in mammals, including the TSPAN1 (TSP-1), TSPAN2 (TSP-2), TSPAN3 (TSP-3), TSPAN4 (TSP-4, NAG-2), TSPAN5 (TSP-5), TSPAN6 (TSP-6), TSPAN7 (CD231, TALLA-1, A15), TSPAN8 (CO-029), TSPAN9 (NET-5), TSPAN10 (Oculospanin), TSPAN11 (CD151-like), TSPAN12 (NET-2), TSPAN13 (NET-6), TSPAN14, TSPAN15 (NET-7), TSPAN16 (TM4-B), TSPAN17, TSPAN18, TSPAN19, TSPAN20 (UP1b, UPK1B), TSPAN21 (UP1a, UPK1A), TSPAN22 (RDS, PRPH2), TSPAN23 (ROM1), TSPAN24 (CD151), TSPAN25 (CD53), TSPAN26 (CD37), TSPAN27 (CD82), TSPAN28 (CD81), TSPAN29 (CD9), TSPAN30 (CD63), TSPAN31 (SAS), TSPAN32 (TSSC6), TSPAN33, and TSPAN34. Other commonly observed vesicle markers include those listed in Table 3. Any of these proteins can be used as vesicle markers.",{"@attributes":{"id":"p-0257","num":"0256"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Proteins Observed in Vesicles from Multiple Cell Types"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"147pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Class","Protein"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Antigen Presentation","MHC class I, MHC class II, Integrins, Alpha 4"]},{"entry":[{},"beta 1, Alpha M beta 2, Beta 2"]},{"entry":["Immunoglobulin","ICAM1\/CD54, P-selection"]},{"entry":"family"},{"entry":["Cell-surface peptidases","Dipeptidylpeptidase IV\/CD26, Aminopeptidase"]},{"entry":[{},"n\/CD13"]},{"entry":["Tetraspanins","CD151, CD53, CD37, CD82, CD81, CD9 and"]},{"entry":[{},"CD63"]},{"entry":["Heat-shock proteins","Hsp70, Hsp84\/90"]},{"entry":["Cytoskeletal proteins","Actin, Actin-binding proteins, Tubulin"]},{"entry":["Membrane transport","Annexin I, Annexin II, Annexin IV, Annexin V,"]},{"entry":["and fusion","Annexin VI, RAB7\/RAP1B\/RADGDI"]},{"entry":["Signal transduction","Gi2alpha\/14-3-3, CBL\/LCK"]},{"entry":["Abundant membrane","CD63, GAPDH, CD9, CD81, ANXA2, ENO1,"]},{"entry":["proteins","SDCBP, MSN, MFGE8, EZR, GK, ANXA1,"]},{"entry":[{},"LAMP2, DPP4, TSG101, HSPA1A,"]},{"entry":[{},"GDI2, CLTC, LAMP1, Cd86, ANPEP,"]},{"entry":[{},"TFRC, SLC3A2, RDX, RAP1B,"]},{"entry":[{},"RAB5C, RAB5B, MYH9, ICAM1,"]},{"entry":[{},"FN1, RAB11B, PIGR, LGALS3,"]},{"entry":[{},"ITGB1, EHD1, CLIC1, ATP1A1,"]},{"entry":[{},"ARF1, RAP1A, P4HB, MUC1, KRT10,"]},{"entry":[{},"HLA-A, FLOT1, CD59, C1orf58,"]},{"entry":[{},"BASP1, TACSTD1, STOM"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The binding agent can also be an agent that binds to a vesicle derived from a specific cell type, such as a tumor cell (e.g. binding agent for Tissue factor, EpCam, B7H3 or CD24) or a specific cell-of-origin. The binding agent used to isolate or detect a vesicle can be a binding agent for an antigen selected from . The binding agent for a vesicle can also be selected from those listed in . The binding agent can be for an antigen such as a tetraspanin, MFG-E8, Annexin V, 5T4, B7H3, caveolin, CD63, CD9, E-Cadherin, Tissue factor, MFG-E8, TMEM211, CD24, PSCA, PCSA, PSMA, Rab-5B, STEAP, TNFR1, CD81, EpCam, CD59, CD81, ICAM, EGFR, or CD66. A binding agent for a platelet can be a glycoprotein such as GpIa-IIa, GpIIb-IIIa, GpIIIb, GpIb, or GpIX. One or more binding agents, such as one or more binding agents for two or more of the antigens, can be used for isolating or detecting a vesicle. The binding agent used can be selected based on the desire of isolating or detecting a vesicle derived from a particular cell type or cell-of-origin specific vesicle.","A binding agent can also be linked directly or indirectly to a solid surface or substrate. A solid surface or substrate can be any physically separable solid to which a binding agent can be directly or indirectly attached including, but not limited to, surfaces provided by microarrays and wells, particles such as beads, columns, optical fibers, wipes, glass and modified or functionalized glass, quartz, mica, diazotized membranes (paper or nylon), polyformaldehyde, cellulose, cellulose acetate, paper, ceramics, metals, metalloids, semiconductive materials, quantum dots, coated beads or particles, other chromatographic materials, magnetic particles; plastics (including acrylics, polystyrene, copolymers of styrene or other materials, polypropylene, polyethylene, polybutylene, polyurethanes, TEFLON\u2122, etc.), polysaccharides, nylon or nitrocellulose, resins, silica or silica-based materials including silicon and modified silicon, carbon, metals, inorganic glasses, plastics, ceramics, conducting polymers (including polymers such as polypyrole and polyindole); micro or nanostructured surfaces such as nucleic acid tiling arrays, nanotube, nanowire, or nanoparticulate decorated surfaces; or porous surfaces or gels such as methacrylates, acrylamides, sugar polymers, cellulose, silicates, or other fibrous or stranded polymers. In addition, as is known the art, the substrate may be coated using passive or chemically-derivatized coatings with any number of materials, including polymers, such as dextrans, acrylamides, gelatins or agarose. Such coatings can facilitate the use of the array with a biological sample.","For example, an antibody used to isolate a vesicle can be bound to a solid substrate such as a well, such as commercially available plates (e.g. from Nunc, Milan Italy). Each well can be coated with the antibody. In some embodiments, the antibody used to isolate a vesicle is bound to a solid substrate such as an array. The array can have a predetermined spatial arrangement of molecule interactions, binding islands, biomolecules, zones, domains or spatial arrangements of binding islands or binding agents deposited within discrete boundaries. Further, the term array may be used herein to refer to multiple arrays arranged on a surface, such as would be the case where a surface bore multiple copies of an array. Such surfaces bearing multiple arrays may also be referred to as multiple arrays or repeating arrays.","A binding agent can also be bound to particles such as beads or microspheres. For example, an antibody specific for a component of a vesicle can be bound to a particle, and the antibody-bound particle is used to isolate a vesicle from a biological sample. In some embodiments, the microspheres may be magnetic or fluorescently labeled. In addition, a binding agent for isolating vesicles can be a solid substrate itself. For example, latex beads, such as aldehyde\/sulfate beads (Interfacial Dynamics, Portland, Oreg.) can be used.","A binding agent bound to a magnetic bead can also be used to isolate a vesicle. For example, a biological sample such as serum from a patient can be collected for colon cancer screening. The sample can be incubated with anti-CCSA-3 (Colon Cancer-Specific Antigen) coupled to magnetic microbeads. A low-density microcolumn can be placed in the magnetic field of a MACS Separator and the column is then washed with a buffer solution such as Tris-buffered saline. The magnetic immune complexes can then be applied to the column and unbound, non-specific material can be discarded. The CCSA-3 selected vesicle can be recovered by removing the column from the separator and placing it on a collection tube. A buffer can be added to the column and the magnetically labeled vesicle can be released by applying the plunger supplied with the column. The isolated vesicle can be diluted in IgG elution buffer and the complex can then be centrifuged to separate the microbeads from the vesicle. The pelleted isolated cell-of-origin specific vesicle can be resuspended in buffer such as phosphate-buffered saline and quantitated. Alternatively, due to the strong adhesion force between the antibody captured cell-of-origin specific vesicle and the magnetic microbeads, a proteolytic enzyme such as trypsin can be used for the release of captured vesicles without the need for centrifugation. The proteolytic enzyme can be incubated with the antibody captured cell-of-origin specific vesicles for at least a time sufficient to release the vesicles.","A binding agent, such as an antibody, for isolating vesicles is preferably contacted with the biological sample comprising the vesicles of interest for at least a time sufficient for the binding agent to bind to a component of the vesicle. For example, an antibody may be contacted with a biological sample for various intervals ranging from seconds days, including but not limited to, about 10 minutes, 30 minutes, 1 hour, 3 hours, 5 hours, 7 hours, 10 hours, 15 hours, 1 day, 3 days, 7 days or 10 days.","A binding agent, such as an antibody specific to an antigen listed in , or a binding agent listed in , can be labeled with, including but not limited to, a magnetic label, a fluorescent moiety, an enzyme, a chemiluminescent probe, a metal particle, a non-metal colloidal particle, a polymeric dye particle, a pigment molecule, a pigment particle, an electrochemically active species, semiconductor nanocrystal or other nanoparticles including quantum dots or gold particles. The label can be, but not be limited to, fluorophores, quantum dots, or radioactive labels. For example, the label can be a radioisotope (radionuclides), such as H, C, C, F, F, S, Cu, Ga, Y, Te, In, I, I, I, I, Xe, Lu, At, or Bi. The label can be a fluorescent label, such as a rare earth chelate (europium chelate), fluorescein type, such as, but not limited to, FITC, 5-carboxyfluorescein, 6-carboxy fluorescein; a rhodamine type, such as, but not limited to, TAMRA; dansyl; Lissamine; cyanines; phycoerythrins; Texas Red; and analogs thereof. The fluorescent label can be one or more of FAM, dRHO, 5-FAM, 6FAM, dR6G, JOE, HEX, VIC, TET, dTAMRA, TAMRA, NED, dROX, PET, BHQ, Gold540 and LIZ.","A binding agent can be directly or indirectly labeled, e.g., the label is attached to the antibody through biotin-streptavidin. Alternatively, an antibody is not labeled, but is later contacted with a second antibody that is labeled after the first antibody is bound to an antigen of interest.","For example, various enzyme-substrate labels are available or disclosed (see for example, U.S. Pat. No. 4,275,149). The enzyme generally catalyzes a chemical alteration of a chromogenic substrate that can be measured using various techniques. For example, the enzyme may catalyze a color change in a substrate, which can be measured spectrophotometrically. Alternatively, the enzyme may alter the fluorescence or chemiluminescence of the substrate. Examples of enzymatic labels include luciferases (e.g., firefly luciferase and bacterial luciferase; U.S. Pat. No. 4,737,456), luciferin, 2,3-dihydrophthalazinediones, malate dehydrogenase, urease, peroxidase such as horseradish peroxidase (HRP), alkaline phosphatase (AP), \u03b2-galactosidase, glucoamylase, lysozyme, saccharide oxidases (e.g., glucose oxidase, galactose oxidase, and glucose-6-phosphate dehydrogenase), heterocyclic oxidases (such as uricase and xanthine oxidase), lactoperoxidase, microperoxidase, and the like. Examples of enzyme-substrate combinations include, but are not limited to, horseradish peroxidase (HRP) with hydrogen peroxidase as a substrate, wherein the hydrogen peroxidase oxidizes a dye precursor (e.g., orthophenylene diamine (OPD) or 3,3\u2032,5,5\u2032-tetramethylbenzidine hydrochloride (TMB)); alkaline phosphatase (AP) with para-nitrophenyl phosphate as chromogenic substrate; and \u03b2-D-galactosidase (\u03b2-D-Gal) with a chromogenic substrate (e.g., p-nitrophenyl-\u03b2-D-galactosidase) or fluorogenic substrate 4-methylumbelliferyl-\u03b2-D-galactosidase.","Depending on the method of isolation or detection used, the binding agent may be linked to a solid surface or substrate, such as arrays, particles, wells and other substrates described above. Methods for direct chemical coupling of antibodies, to the cell surface are known in the art, and may include, for example, coupling using glutaraldehyde or maleimide activated antibodies. Methods for chemical coupling using multiple step procedures include biotinylation, coupling of trinitrophenol (TNP) or digoxigenin using for example succinimide esters of these compounds. Biotinylation can be accomplished by, for example, the use of D-biotinyl-N-hydroxysuccinimide. Succinimide groups react effectively with amino groups at pH values above 7, and preferentially between about pH 8.0 and about pH 8.5. Biotinylation can be accomplished by, for example, treating the cells with dithiothreitol followed by the addition of biotin maleimide.","Flow Cytometry","Isolation or detection of a vesicle using a particle such as a bead or microsphere can also be performed using flow cytometry. Flow cytometry can be used for sorting microscopic particles suspended in a stream of fluid. As particles pass through they can be selectively charged and on their exit can be deflected into separate paths of flow. It is therefore possible to separate populations from an original mix, such as a biological sample, with a high degree of accuracy and speed. Flow cytometry allows simultaneous multiparametric analysis of the physical and\/or chemical characteristics of single cells flowing through an optical\/electronic detection apparatus. A beam of light, usually laser light, of a single frequency (color) is directed onto a hydrodynamically focused stream of fluid. A number of detectors are aimed at the point where the stream passes through the light beam; one in line with the light beam (Forward Scatter or FSC) and several perpendicular to it (Side Scatter or SSC) and one or more fluorescent detectors.","Each suspended particle passing through the beam scatters the light in some way, and fluorescent chemicals in the particle may be excited into emitting light at a lower frequency than the light source. This combination of scattered and fluorescent light is picked up by the detectors, and by analyzing fluctuations in brightness at each detector (one for each fluorescent emission peak), it is possible to deduce various facts about the physical and chemical structure of each individual particle. FSC correlates with the cell size and SSC depends on the inner complexity of the particle, such as shape of the nucleus, the amount and type of cytoplasmic granules or the membrane roughness. Some flow cytometers have eliminated the need for fluorescence and use only light scatter for measurement.","Flow cytometers can analyze several thousand particles every second in \u201creal time\u201d and can actively separate out and isolate particles having specified properties. They offer high-throughput automated quantification, and separation, of the set parameters for a high number of single cells during each analysis session. Flow cytomers can have multiple lasers and fluorescence detectors, allowing multiple labels to be used to more precisely specify a target population by their phenotype. Thus, a flow cytometer, such as a multicolor flow cytometer, can be used to detect one or more vesicles with multiple fluorescent labels or colors. In some embodiments, the flow cytometer can also sort or isolate different vesicle populations, such as by size or by different markers.","The flow cytometer may have one or more lasers, such as 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or more lasers. In some embodiments, the flow cytometer can detect more than one color or fluorescent label, such as at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, or 20 different colors or fluorescent labels. For example, the flow cytometer can have at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, or 20 fluorescence detectors.","Examples of commerically available flow cytometers that can be used to detect or analyze one or more vesicles, to sort or separate different populations of vesicles, include, but are not limited to the MoFlo\u2122 XDP Cell Sorter (Beckman Coulter, Brea, Calif.), MoFlo\u2122 Legacy Cell Sorter (Beckman Coulter, Brea, Calif.), BD FACSAria\u2122 Cell Sorter (BD Biosciences, San Jose, Calif.), BD\u2122 LSRII (BD Biosciences, San Jose, Calif.), and BD FACSCalibur\u2122 (BD Biosciences, San Jose, Calif.). Use of multicolor or multi-fluor cytometers can be used in multiplex analysis of vesicles, as further described below. In some embodiments, the flow cytometer can sort, and thereby collect or sort more than one population of vesicles based one or more characteristics. For example, two populations of vesicles differ in size, such that the vesicles within each population have a similar size range and can be differentially detected or sorted. In another embodiment, two different populations of vesicles are differentially labeled.","The data resulting from flow-cytometers can be plotted in 1 dimension to produce histograms or seen in 2 dimensions as dot plots or in 3 dimensions with newer software. The regions on these plots can be sequentially separated by a series of subset extractions which are termed gates. Specific gating protocols exist for diagnostic and clinical purposes especially in relation to hematology. The plots are often made on logarithmic scales. Because different fluorescent dye's emission spectra overlap, signals at the detectors have to be compensated electronically as well as computationally. Fluorophores for labeling biomarkers may include those described in Ormerod, 2., Springer-Verlag, New York (1999), and in Nida et al., 2005; 4 889-894 which is incorporated herein by reference.","Multiplexing","Multiplex experiments comprise experiments that can simultaneously measure multiple analytes in a single assay. Vesicles and associated biomarkers can be assessed in a multiplex fashion. Different binding agents can be used for multiplexing different vesicle populations. Different vesicle populations can be isolated or detected using different binding agents. Each population in a biological sample can be labeled with a different signaling label, such as a fluorophore, quantum dot, or radioactive label, such as described above. The label can be directly conjugated to a binding agent or indirectly used to detect a binding agent that binds a vesicle. The number of populations detected in a multiplexing assay is dependent on the resolution capability of the labels and the summation of signals, as more than two differentially labeled vesicle populations that bind two or more affinity elements can produce summed signals.","Multiplexing of at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 different vesicle populations may be performed. For example, one population of vesicles specific to a cell-of-origin can be assayed along with a second population of vesicles specific to a different cell-of-origin, where each population is labeled with a different label. Alternatively, a population of vesicles with a particular biomarker or biosignature can be assayed along with a second population of vesicles with a different biomarker or biosignature. In some cases, hundreds or thousands of vesicles are assessed in a single assay.","In one embodiment, multiplex analysis is performed by applying a plurality of vesicles comprising more than one population of vesicles to a plurality of substrates, such as beads. Each bead is coupled to one or more capture agents. The plurality of beads is divided into subsets, where beads with the same capture agent or combination of capture agents form a subset of beads, such that each subset of beads has a different capture agent or combination of capture agents than another subset of beads. The beads can then be used to capture vesicles that comprise a component that binds to the capture agent. The different subsets can be used to capture different populations of vesicles. The captured vesicles can then be analyzed by detecting one or more biomarkers.","Flow cytometry can be used in combination with a particle-based or bead based assay. Multiparametric immunoassays or other high throughput detection assays using bead coatings with cognate ligands and reporter molecules with specific activities consistent with high sensitivity automation can be used. For example, beads in each subset can be differentially labeled from another subset. In a particle based assay system, a binding agent or capture agent for a vesicle, such as a capture antibody, can be immobilized on addressable beads or microspheres. Each binding agent for each individual binding assay (such as an immunoassay when the binding agent is an antibody) can be coupled to a distinct type of microsphere (i.e., microbead) and the binding assay reaction takes place on the surface of the microspheres. Microspheres can be distinguished by different labels, for example, a microsphere with a specific capture agent would have a different signaling label as compared to another microsphere with a different capture agent. For example, microspheres can be dyed with discrete fluorescence intensities such that the fluorescence intensity of a microsphere with a specific binding agent is different than that of another microsphere with a different binding agent.","The microsphere can be labeled or dyed with at least 2 different labels or dyes. In some embodiments, the microsphere is labeled with at least 3, 4, 5, 6, 7, 8, 9, or 10 different labels. Different microspheres in a plurality of microspheres can have more than one label or dye, wherein various subsets of the microspheres have various ratios and combinations of the labels or dyes permitting detection of different microspheres with different binding agents. For example, the various ratios and combinations of labels and dyes can permit different fluorescent intensities. Alternatively, the various ratios and combinations maybe used to generate different detection patters to identify the binding agent. The microspheres can be labeled or dyed externally or may have intrinsic fluorescence or signaling labels. Beads can be loaded separately with their appropriate binding agents and thus, different vesicle populations can be isolated based on the different binding agents on the differentially labeled microspheres to which the different binding agents are coupled.","In another embodiment, multiplex analysis can be performed using a planar substrate, wherein the substrate comprises a plurality of capture agents. The plurality of capture agents can capture one or more populations of vesicles, and one or more biomarkers of the captured vesicles detected. The planar substrate can be a microarray or other substrate as further described herein.","Novel Binding Agents","A vesicle may be isolated or detected using a binding agent for a novel component of a vesicle, such as an antibody for a novel antigen specific to a vesicle of interest. Novel antigens that are specific to a vesicle of interest may be isolated or identified using different test compounds of known composition bound to a substrate, such as an array or a plurality of particles, which can allow a large amount of chemical\/structural space to be adequately sampled using only a small fraction of the space. The novel antigen identified can also serve as a biomarker for the vesicle. For example, a novel antigen identified for a cell-of-origin specific vesicle can be a useful biomarker.","A binding agent can be identified by screening either a homogeneous or heterogeneous vesicle population against test compounds. Since the composition of each test compound on the substrate surface is known, this constitutes a screen for affinity elements. For example, a test compound array comprises test compounds at specific locations on the substrate addressable locations, and can be used to identify one or more binding agents for a vesicle. The test compounds can all be unrelated or related based on minor variations of a core sequence or structure. The different test compounds may include variants of a given test compound (such as polypeptide isoforms), test compounds that are structurally or compositionally unrelated, or a combination thereof.","A test compound can be a peptoid, polysaccharide, organic compound, inorganic compound, polymer, lipids, nucleic acid, polypeptide, antibody, protein, polysaccharide, or other compound. The test compound can be natural or synthetic. The test compound can comprise or consist of linear or branched heteropolymeric compounds based on any of a number of linkages or combinations of linkages (e.g., amide, ester, ether, thiol, radical additions, metal coordination, etc.), dendritic structures, circular structures, cavity structures or other structures with multiple nearby sites of attachment that serve as scaffolds upon which specific additions are made. Thes test compound can be spotted on a substrate or synthesized in situ, using standard methods in the art. In addition, the test compound can be spotted or synthesized in situ in combinations in order to detect useful interactions, such as cooperative binding.","The test compound can be a polypeptide with known amino acid sequence, thus, detection of a test compound binding with a vesicle can lead to identification of a polypeptide of known amino sequence that can be used as a binding agent. For example, a homogenous population of vesicles can be applied to a spotted array on a slide containing between a few and 1,000,000 test polypeptides having a length of variable amino acids. The polypeptides can be attached to the surface through the C-terminus. The sequence of the polypeptides can be generated randomly from 19 amino acids, excluding cysteine. The binding reaction can include a non-specific competitor, such as excess bacterial proteins labeled with another dye such that the specificity ratio for each polypeptide binding target can be determined. The polypeptides with the highest specificity and binding can be selected. The identity of the polypeptide on each spot is known, and thus can be readily identified. Once the novel antigens specific to the homogeneous vesicle population, such as a cell-of-origin specific vesicle is identified, such cell-of-origin specific vesicles may subsequently be isolated using such antigens in methods described hereafter.","An array can also be used for identifying an antibody as a binding agent for a vesicle. Test antibodies can be attached to an array and screened against a heterogeneous population of vesicles to identify antibodies that can be used to isolate or identify a vesicle. A homogeneous population of vesicles such as cell-of-origin specific vesicles can also be screened with an antibody array. Other than identifying antibodies to isolate or detect a homogeneous population of vesicles, one or more protein biomarkers specific to the homogenous population can be identified. Commercially available platforms with test antibodies pre-selected or custom selection of test antibodies attached to the array can be used. For example, an antibody array from Full Moon Biosystems can be screened using prostate cancer cell derived vesicles identifying antibodies to Bcl-XL, ERCC1, Keratin 15, CD81\/TAPA-1, CD9, Epithelial Specific Antigen (ESA), and Mast Cell Chymase as binding agents (see for example, ), and the proteins identified can be used as biomarkers for the vesicles.","An antibody or synthetic antibody to be used as a binding agent can also be identified through a peptide array. Another method is the use of synthetic antibody generation through antibody phage display. M13 bacteriophage libraries of antibodies (e.g. Fabs) are displayed on the surfaces of phage particles as fusions to a coat protein. Each phage particle displays a unique antibody and also encapsulates a vector that contains the encoding DNA. Highly diverse libraries can be constructed and represented as phage pools, which can be used in antibody selection for binding to immobilized antigens. Antigen-binding phages are retained by the immobilized antigen, and the nonbinding phages are removed by washing. The retained phage pool can be amplified by infection of an host and the amplified pool can be used for additional rounds of selection to eventually obtain a population that is dominated by antigen-binding clones. At this stage, individual phase clones can be isolated and subjected to DNA sequencing to decode the sequences of the displayed antibodies. Through the use of phase display and other methods known in the art, high affinity designer antibodies for vesicles can be generated.","Bead-based assays can also be used to identify novel binding agents to isolate or detect a vesicle. A test antibody or peptide can be conjugated to a particle. For example, a bead can be conjugated to an antibody or peptide and used to detect and quantify the proteins expressed on the surface of a population of vesicles in order to discover and specifically select for novel antibodies that can target vesicles from specific tissue or tumor types. Any molecule of organic origin can be successfully conjugated to a polystyrene bead through use of a commercially available kit according to manufacturer's instructions. Each bead set can be colored a certain detectable wavelength and each can be linked to a known antibody or peptide which can be used to specifically measure which beads are linked to exosomal proteins matching the epitope of previously conjugated antibodies or peptides. The beads can be dyed with discrete fluorescence intensities such that each bead with a different intensity has a different binding agent as described above.","For example, a purified vesicle preparation can be diluted in assay buffer to an appropriate concentration according to empirically determined dynamic range of assay. A sufficient volume of coupled beads can be prepared and approximately 1 \u03bcl of the antibody-coupled beads can be aliqouted into a well and adjusted to a final volume of approximately 50 \u03bcl. Once the antibody-conjugated beads have been added to a vacuum compatible plate, the beads can be washed to ensure proper binding conditions. An appropriate volume of vesicle preparation can then be added to each well being tested and the mixture incubated, such as for 15-18 hours. A sufficient volume of detection antibodies using detection antibody diluent solution can be prepared and incubated with the mixture for 1 hour or for as long as necessary. The beads can then be washed before the addition of detection antibody (biotin expressing) mixture composed of streptavidin phycoereythin. The beads can then be washed and vacuum aspirated several times before analysis on a suspension array system using software provided with an instrument. The identity of antigens that can be used to selectively extract the vesicles can then be elucidated from the analysis.","Assays using imaging systems can be utilized to detect and quantify proteins expressed on the surface of a vesicle in order to discover and specifically select for and enrich vesicles from specific tissue, cell or tumor types. Antibodies, peptides or cells conjugated to multiple well multiplex carbon coated plates can be used. Simultaneous measurement of many analytes in a well can be achieved through the use of capture antibodies arrayed on the patterned carbon working surface. Analytes can then be detected with antibodies labeled with reagents in electrode wells with an enhanced electro-chemiluminescent plate. Any molecule of organic origin can be successfully conjugated to the carbon coated plate. Proteins expressed on the surface of vesicles can be identified from this assay and can be used as targets to specifically select for and enrich vesicles from specific tissue or tumor types.","The binding agent can also be an aptamer, which refers to nucleic acids that can bond molecules other than their complementary sequence. An aptamer typically contains 30-80 nucleic acids and can have a high affinity towards a certain target molecule (K's reported are between 10-10mole\/1). An aptamer for a target can be identified using systematic evolution of ligands by exponential enrichment (SELEX) (Tuerk & Gold, 249:505-510, 1990; Ellington & Szostak, 346:818-822, 1990), such as described in U.S. Pat. Nos. 5,270,163, 6,482, 594, 6,291, 184, 6,376, 190 and 6,458,539. A library of nucleic acids can be contacted with a target vesicle, and those nucleic acids specifically bound to the target are partitioned from the remainder of nucleic acids in the library which do not specifically bind the target. The partitioned nucleic acids are amplified to yield a ligand-enriched pool. Multiple cycles of binding, partitioning, and amplifying (i.e., selection) result in identification of one or more aptamers with the desired activity. Another method for identifying an aptamer to isolate vesicles is described in U.S. Pat. No. 6,376,190, which describes increasing or decreasing frequency of nucleic acids in a library by their binding to a chemically synthesized peptide. Modified methods, such as Laser SELEX or deSELEX as described in U.S. Patent Publication No. 20090264508 can also be used.","Microfluidics","The methods for isolating or identifying vesicles can be used in combination with microfluidic devices. The methods of isolating or detecting a vesicle, such as described herien, can be performed using a microfluidic device. Microfluidic devices, which may also be referred to as \u201clab-on-a-chip\u201d systems, biomedical micro-electro-mechanical systems (bioMEMs), or multicomponent integrated systems, can be used for isolating and analyzing a vesicle. Such systems miniaturize and compartmentalize processes that allow for binding of vesicles, detection of biosignatures, and other processes.","A microfluidic device can also be used for isolation of a vesicle through size differential or affinity selection. For example, a microfluidic device can use one more channels for isolating a vesicle from a biological sample based on size or by using one or more binding agents for isolating a vesicle from a biological sample. A biological sample can be introduced into one or more microfluidic channels, which selectively allows the passage of a vesicle. The selection can be based on a property of the vesicle, such as the size, shape, deformability, or biosignature of the vesicle.","In one embodiment, a heterogeneous population of vesicles can be introduced into a microfluidic device, and one or more different homogeneous populations of vesicles can be obtained. For example, different channels can have different size selections or binding agents to select for different vesicle populations. Thus, a microfluidic device can isolate a plurality of vesicles wherein at least a subset of the plurality of vesicles comprises a different biosignature from another subset of the plurality of vesicles. For example, the microfluidic device can isolate at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, or 100 different subsets of vesicles, wherein each subset of vesicles comprises a different biosignature.","In some embodiments, the microfluidic device can comprise one or more channels that permit further enrichment or selection of a vesicle. A population of vesicles that has been enriched after passage through a first channel can be introduced into a second channel, which allows the passage of the desired vesicle or vesicle population to be further enriched, such as through one or more binding agents present in the second channel.","Array-based assays and bead-based assays can be used with microfluidic device. For example, the binding agent can be coupled to beads and the binding reaction between the beads and vesicle can be performed in a microfluidic device. Multiplexing can also be performed using a microfluidic device. Different compartments can comprise different binding agents for different populations of vesicles, where each population is of a different cell-of-origin specific vesicle population. In one embodiment, each population has a different biosignature. The hybridization reaction between the microsphere and vesicle can be performed in a microfluidic device and the reaction mixture can be delivered to a detection device. The detection device, such as a dual or multiple laser detection system can be part of the microfluidic system and can use a laser to identify each bead or microsphere by its color-coding, and another laser can detect the hybridization signal associated with each bead.","Examples of microfluidic devices that may be used, or adapted for use with vesicles, include but are not limited to those described in U.S. Pat. Nos. 7,591,936, 7,581,429, 7,579,136, 7,575,722, 7,568,399, 7,552,741, 7,544,506, 7,541,578, 7,518,726, 7,488,596, 7,485,214, 7,467,928, 7,452,713, 7,452,509, 7,449,096, 7,431,887, 7,422,725, 7,422,669, 7,419,822, 7,419,639, 7,413,709, 7,411,184, 7,402,229, 7,390,463, 7,381,471, 7,357,864, 7,351,592, 7,351,380, 7,338,637, 7,329,391, 7,323,140, 7,261,824, 7,258,837, 7,253,003, 7,238,324, 7,238,255, 7,233,865, 7,229,538, 7,201,881, 7,195,986, 7,189,581, 7,189,580, 7,189,368, 7,141,978, 7,138,062, 7,135,147, 7,125,711, 7,118,910, 7,118,661, 7,640,947, 7,666,361, 7,704,735; and International Patent Publication WO 2010\/072410; each of which patents or applications are incorporated herein by reference in their entirety. Another example for use with methods disclosed herein is described in Chen et al., \u201c, Dec. 8, 2009 DOI: 10. 1039\/b916199f.","In one embodiment, a microfluidic device for isolating or detecting a vesicle comprises a channel of less than about 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 35, 40, 45, 50, 55, of 60 mm in width, or between about 2-60, 3-50, 3-40, 3-30, 3-20, or 4-20 mm in width. The microchannel can have a depth of less than about 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 45, 50, 55, 60, 65 or 70 \u03bcm, or between about 10-70, 10-40, 15-35, or 20-30 \u03bcm. Furthermore, the microchannel can have a length of less than about 1, 2, 3, 3.5, 4, 4.5, 5, 5.5, 6, 6.5, 7, 7.5, 8, 8.5, 9, 9.5 or 10 cm. The microfluidic device can have grooves on its ceiling that are less than about 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 6, 65, 70, 75, or 80 \u03bcm wide, or between about 40-80, 40-70, 40-60 or 45-55 \u03bcm wide. The grooves can be less than about 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30, 35, 40, 45, or 50 \u03bcm deep, such as between about 1-50, 5-40, 5-30, 3-20 or 5-15 \u03bcm.","The microfluidic device can have one or more binding agents attached to a surface in a channel, or present in a channel. For example, the microchannel can have one or more capture agents, such as a capture agent for EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR. In one embodiment, a microchannel surface is treated with avidin and a capture agent, such as an antibody, that is biotinylated can be injected into the channel to bind the avidin.","A biological sample can be flowed into the microfluidic device, or a microchannel, at rates such as at least about 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30, 35, 40, 45, or 50 \u03bcl per minute, such as between about 1-50, 5-40, 5-30, 3-20 or 5-15 \u03bcl per minute. One or more vesicles can be captured and directly detected in the microfludic device. Alternatively, the captured vesicle may be released and exit the microfluidic device prior to analysis. In another embodiment, one or more captured vesicles are lysed in the microchannel and the lysate can be analyzed. Lysis buffer can be flowed through the channel and lyse the captured vesicles. For example, the lysis buffer can be flowed into the device or microchannel at rates such as at least about a, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 26, 27, 28, 29, 30, 35, 40, 45, or 50 \u03bcl per minute, such as between about 1-50, 5-40, 10-30, 5-30 or 10-35 \u03bcl per minute. The lysate can be collected and analyzed, such as performing RT-PCR, PCR, mass spectrometry, Western blotting, or other assays, to detect one or more biomarkers of the vesicle.","Cell-of-Origin and Disease-Specific Vesicles","The bindings agent disclosed herein can be used to isolate or detect a vesicle, such as a cell-of-origin vesicle or vesicle with a specific biosignature. The binding agent can be used to isolate or detect a heterogeneous population of vesicles from a sample or can be used to isolate or detect a homogeneous population of vesicles, such as cell-of-origin specific vesicles with specific biosignatures, from a heterogeneous population of vesicles.","A homogeneous population of vesicles, such as cell-of-origin specific vesicles, can be analyzed and used to characterize a phenotype for a subject. Cell-of-origin specific vesicles are esicles derived from specific cell types, which can include, but are not limited to, cells of a specific tissue, cells from a specific tumor of interest or a diseased tissue of interest, circulating tumor cells, or cells of maternal or fetal origin. The vesicles may be derived from tumor cells or lung, pancreas, stomach, intestine, bladder, kidney, ovary, testis, skin, colorectal, breast, prostate, brain, esophagus, liver, placenta, or fetal cells. The isolated vesicle can also be from a particular sample type, such as urinary vesicle.","A cell-of-origin specific vesicle from a biological sample can be isolated using one or more binding agents that are specific to a cell-of-origin. Vesicles for analysis of a disease or condition can be isolated using one or more binding agent specific for biomarkers for that disease or condition.","A vesicle can be concentrated prior to isolation or detection of a cell-of-origin specific vesicle, such as through centrifugation, chromatography, or filtration, as described above, to produce a heterogeneous population of vesicles prior to isolation of cell-of-origin specific vesicles. Alternatively, the vesicle is not concentrated, or the biological sample is not enriched for a vesicle, prior to isolation of a cell-of-origin vesicle.",{"@attributes":{"id":"p-0304","num":"0303"},"figref":"FIG. 61B","b":["6100","6102","6104","6106","6108","6104"]},"A cell-of-origin specific vesicle can be isolated from a biological sample of a subject by employing one or more binding agents that bind with high specificity to the cell-of-origin specific vesicle. In some instances, a single binding agent can be employed to isolate a cell-of-origin specific vesicle. In other instances, a combination of binding agents may be employed to isolate a cell-of-origin specific vesicle. For example, at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75, or 100 different binding agents may be used to isolate a cell-of-origin vesicle. Therefore, a vesicle population (e.g., vesicles having the same binding agent profile) can be identified by utilizing a single or a plurality of binding agents.","One or more binding agents can be selected based on their specificity for a target antigen(s) that is specific to a cell-of-origin, e.g., a cell-of-origin that is related to a tumor, autoimmune disease, cardiovascular disease, neurological disease, infection or other disease or disorder. The cell-of-origin can be from a cell that is informative for a diagnosis, prognosis, disease stratification, theranosis, prediction of responder\/non-responder status, disease monitoring, treatment monitoring and the like as related to such diseases and disorders. The cell-of-origin can also be from a cell useful to discover biomarkers for use thereto. Non-limiting examples of antigens which may be used singularly, or in combination, to isolate a cell-of-origin specific vesicle, disease specific vesicle, or tumor specific vesicle, are shown in  and are also described herein. The antigen can comprise membrane bound antigens which are accessible to binding agents. The antigen can be a biomarker related to characterizing a phenotype.","One of skill will appreciate that any applicable antigen that can be used to isolate an informative vesicle is contemplated by the invention. Binding agents, e.g., antibodies, aptamers and lectins, can be chosen that recognize surface antigens and\/or fragments thereof, as outlined herein. The binding agents can recognize antigens specific to the desired cell type or location and\/or recognize biomarkers associated with the desired cells. The cells can be, e.g., tumor cells, other diseased cells, cells that serve as markers of disease such as activated immune cells, etc. One of skill will appreciate that binding agents for any cells of interest can be useful for isolating vesicles associated with those cells. One of skill will further appreciate that the binding agents disclosed herein can be used for detecting vesicles of interest. As a non-limiting example, a binding agent to a vesicle biomarker can be labeled directly or indirectly in order to detect vesicles bound by one of more of the same or different binding agents.","A number of targets for binding agents useful for binding to vesicles associated with cancer, autoimmune diseases, cardiovascular diseases, neurological diseases, infection or other disease or disorders are presented in Table 4. A vesicle derived from a cell associated with one of the listed disorders can be characterized using one of the antigens in the table. The binding agent, e.g., an antibody or aptamer, can recognize an epitope of the listed antigens, a fragment thereof, or binding agents can be used against any appropriate combination. Other antigens associated with the disease or disorder can be recognized as well in order to characterize the vesicle. One of skill will appreciate that any applicable antigen that can be used to assess an informative vesicle is contemplated by the invention for isolation, capture or detection in order to characterize a vesicle.",{"@attributes":{"id":"p-0309","num":"0308"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"308pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Illustrative Antigens for Use in Characterizing Various Diseases and Disorders"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Disease or disorder","Target"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Breast cancer, e.g., glandular or stromal cells","BCA-225, hsp70, MART1, ER, VEGFA, Class III b-"]},{"entry":[{},"tubulin, HER2\/neu (for Her2+ breast cancer), GPR30,"]},{"entry":[{},"ErbB4 (JM) isoform, MPR8, MISIIR"]},{"entry":["Ovarian Cancer","CA125, VEGFR2, HER2, MISIIR, VEGFA, CD24"]},{"entry":["Lung Cancer","CYFRA21-1, TPA-M, TPS, CEA, SCC-Ag, XAGE-"]},{"entry":[{},"1b, HLA Class 1, TA-MUC1, KRAS, hENT1, kinin"]},{"entry":[{},"B1 receptor, kinin B2 receptor, TSC403, HTI56, DC-"]},{"entry":[{},"LAMP"]},{"entry":["Colon Cancer","CEA, MUC2, GPA33, CEACAM5, ENFB1, CCSA-3,"]},{"entry":[{},"CCSA-4, ADAM10, CD44, NG2, ephrin B1,"]},{"entry":[{},"plakoglobin, galectin 4, RACK1, tetraspanin-8, FASL,"]},{"entry":[{},"A33, CEA, EGFR, dipeptidase 1, PTEN, Na(+)-"]},{"entry":[{},"dependent glucose transporter, UDP-"]},{"entry":[{},"glucuronosyltransferase 1A"]},{"entry":["Prostate Cancer","PSA, TMPRSS2, FASLG, TNFSF10, PSMA, NGEP,"]},{"entry":[{},"I1-7RI, CSCR4, CysLT1R, TRPM8, Kv1.3, TRPV6,"]},{"entry":[{},"TRPM8, PSGR, MISIIR, galectin-3, PCA3,"]},{"entry":[{},"TMPRSS2:ERG"]},{"entry":["Brain Cancer","PRMT8, BDNF, EGFR, DPPX, Elk, Densin-180,"]},{"entry":[{},"BAI2, BAI3"]},{"entry":["Blood Cancer (hematological malignancy)","CD44, CD58, CD31, CD11a, CD49d, GARP, BTS,"]},{"entry":[{},"Raftlin"]},{"entry":["Melanoma","DUSP1, TYRP1, SILV, MLANA, MCAM, CD63,"]},{"entry":[{},"Alix, hsp70, meosin, p120 catenin, PGRL, syntaxin"]},{"entry":[{},"binding protein 1 & 2, caveolin"]},{"entry":["Liver Cancer (hepatocellular carcinoma)","HBxAg, HBsAg, NLT"]},{"entry":["Cervical Cancer","MCT-1, MCT-2, MCT-4"]},{"entry":["Endometrial Cancer","Alpha V Beta 6 integrin"]},{"entry":["Psoriasis","flt-1, VPF receptors, kdr"]},{"entry":["Autoimmune Disease","Tim-2"]},{"entry":["Irritable Bowel Disease (IBD or Syndrome (IBS)","IL-16, IL-1beta, IL-12, TNF-alpha, interferon-gamma,"]},{"entry":[{},"IL-6, Rantes, II-12, MCP-1, 5HT"]},{"entry":["Diabetes, e.g., pancreatic cells","IL-6, CRP, RBP4"]},{"entry":["Barrett's Esophagus","p53, MUC1, MUC6"]},{"entry":["Fibromyalgia","neopterin, gp130"]},{"entry":["Benign Prostatic Hyperplasia (BPH)","KIA1, intact fibronectin"]},{"entry":["Multiple Sclerosis","B7, B7-2, CD-95 (fas), Apo-1\/Fas"]},{"entry":["Parkinson's Disease","PARK2, ceruloplasmin, VDBP, tau, DJ-1"]},{"entry":["Rheumatic Disease","Citrulinated fibrin a-chain, CD5 antigen-like"]},{"entry":[{},"fibrinogen fragment D, CD5 antigen-like fibrinogen"]},{"entry":[{},"fragment B, TNF alpha"]},{"entry":["Alzheimer's Disease","APP695, APP751 or APP770, BACE1, cystatin C,"]},{"entry":[{},"amyloid \u03b2, T-tau, complement factor H, alpha-2-"]},{"entry":[{},"macroglobulin"]},{"entry":["Head and Neck Cancer","EGFR, EphB4, Ephrin B2"]},{"entry":["Gastrointestinal Stromal Tumor (GIST)","c-kit PDGFRA, NHE-3"]},{"entry":["Renal Cell Carcinoma","c PDGFRA, VEGF, HIF 1 alpha"]},{"entry":["Schizophrenia","ATP5B, ATP5H, ATP6V1B, DNM1"]},{"entry":["Peripheral Neuropathic Pain","OX42, ED9"]},{"entry":["Chronic Neuropathic Pain","chemokine receptor (CCR2\/4)"]},{"entry":["Prion Disease","PrPSc, 14-3-3 zeta, S-100, AQP4"]},{"entry":["Stroke","S-100, neuron specific enolase, PARK7, NDKA,"]},{"entry":[{},"ApoC-I, ApoC-III, SAA or AT-III fragment, Lp-"]},{"entry":[{},"PLA2, hs-CRP"]},{"entry":["Cardiovascular Disease","FATP6"]},{"entry":["Esophageal Cancer","CaSR"]},{"entry":["Tuberculosis","antigen 60, HSP, Lipoarabinomannan, Sulfolipid,"]},{"entry":[{},"antigen of acylated trehalose family, DAT, TAT,"]},{"entry":[{},"Trehalose 6,6-dimycolate (cord-factor) antigen"]},{"entry":["HIV","gp41, gp120"]},{"entry":["Autism","VIP, PACAP, CGRP, NT3"]},{"entry":["Asthma","YKL-40, S-nitrosothiols, SSCA2, PAI, amphiregulin,"]},{"entry":[{},"periostin"]},{"entry":["Lupus","TNFR"]},{"entry":["Cirrhosis","NLT, HBsAg"]},{"entry":["Influenza","hemagglutinin, neurominidase"]},{"entry":["Vulnerable Plaque","Alpha v. Beta 3 integrin, MMP9"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"A cell-of-origin specific vesicle may be isolated using novel binding agents, using methods as described herein. Furthermore, a cell-of-origin specific vesicle can also be isolated from a biological sample using isolation methods based on cellular binding partners or binding agents of such vesicles. Such cellular binding partners can include but are not limited to peptides, proteins, RNA, DNA, apatmers, cells or serum-associated proteins that only bind to such vesicles when one or more specific biomarkers are present. Isolation or deteciton of a cell-of-origin specific vesicle can be carried out with a single binding partner or binding agent, or a combination of binding partners or binding agents whose singular application or combined application results in cell-of-origin specific isolation or detection. Non-limiting examples of such binding agents are provided in . For example, a vesicle for characterizing breast cancer can be isolated with one or more binding agents including, but not limited to, estrogen, progesterone, Herceptin (Trastuzumab), CCND1, MYC PNA, IGF-1 PNA, MYC PNA, SC4 aptamer (Ku), AII-7 aptamer (ERB2), Galectin-3, mucin-type O-glycans, L-PHA, Galectin-9, or any combination thereof.","A binding agent may also be used for isolating or detecting a cell-of-origin specific vesicle based on: i) the presence of antigens specific for cell-of-origin specific vesicles; ii) the absence of markers specific for cell-of-origin specific vesicles; or iii) expression levels of biomarkers specific for cell-of-origin specific vesicles. A heterogeneous population of vesicles can be applied to a surface coated with specific binding agents designed to rule out or identify the cell-of-origin characteristics of the vesicles. Various binding agents, such as antibodies, can be arrayed on a solid surface or substrate and the heterogeneous population of vesicles is allowed to contact the solid surface or substrate for a sufficient time to allow interactions to take place. Specific binding or non-binding to given antibody locations on the array surface or substrate can then serve to identify antigen specific characteristics of the vesicle population that are specific to a given cell-of-origin.","A cell-of-origin specific vesicle can be enriched or isolated using one or more binding agents using a magnetic capture method, fluorescence activated cell sorting (FACS) or laser cytometry as described above. Magnetic capture methods can include, but are not limited to, the use of magnetically activated cell sorter (MACS) microbeads or magnetic columns. Examples of immunoaffinity and magnetic particle methods that can be used are described in U.S. Pat. No. 4,551,435, 4,795,698, 4,925,788, 5,108,933, 5,186,827, 5,200,084 or 5,158,871. A cell-of-origin specific vesicle can also be isolated following the general methods described in U.S. Pat. No. 7,399,632, by using combination of antigens specific to a vesicle.","Any other appropriate method for isolating or otherwise enriching the cell-of-origin specific vesicles with respect to a biological sample may also be used in combination with the present invention. For example, size exclusion chromatography such as gel permeation columns, centrifugation or density gradient centrifugation, and filtration methods can be used in combination with the antigen selection methods described herein. The cell-of-origin specific vesicles may also be isolated following the methods described in Koga et al., 25:3703-3708 (2005), Taylor et al., 110:13-21 (2008), Nanjee et al., 2000; 46:207-223 or U.S. Pat. No. 7,232,653.","Accordingly, vesicles can be isolated that are isolated from cells derived from a tumor, or site of autoimmune disease, cardiovascular disease, neurological disease, infection or other disease or disorder. In some embodiments, the isolated vesicles are derived from cells related to such diseases and disorders, e.g., immune cells that play a role in the etiology of the disease and whose analysis is informative for a diagnosis, prognosis, disease stratification, theranosis, prediction of responder\/non-responder status, disease monitoring, treatment monitoring and the like as relates to such diseases and disorders. The vesicles are further useful to discover biomarkers. The isolated vesicles can then be assessed for characterizing a phenotype as described herein.","Vesicle Assessment","A phenotype can be characterized for a subject by analyzing a biological sample from the subject and determining the level, amount, or concentration of one or more populations of vesicles in the sample. A vesicle can be purified or concentrated prior to determining the amount of vesicles. Alternatively, the amount of vesicles can be directly assayed from a sample, without prior purification or concentration. The vesicles can be cell-of-origin specific vesicles or vesicles with a specific biosignature. The amount of vesicles can be used when characterizing a phenotype, such as a diagnosis, prognosis, theranosis, or prediction of responder\/non-responder status. In some embodiments, the amount is used to determine a physiological or biological state, such as pregnancy or the stage of pregnancy. The amount of vesicles can also be used to determine treatment efficacy, stage of a disease or condition, or progression of a disease or condition. For example, the amount of vesicles can be proportional or inversely proportional to an increase in disease stage or progression. The amount of vesicles can also be used to monitor progression of a disease or condition or to monitor a subject's response to a treatment.","The vesicles can be evaluated by comparing the level of vesicles with a reference level or value of vesicles. The reference value can be particular to physical or temporal endpoint. For example, the reference value can be from the same subject from whom a sample is assessed, or the reference value can be from a representative population of samples (e.g., samples from normal subjects not exhibiting a symptom of disease). Therefore, a reference value can provide a threshold measurement which is compared to a subject sample's readout for a vesicle population assayed in a given sample. Such reference values may be set according to data pooled from groups of sample corresponding to a particular cohort, including but not limited to age (e.g., newborns, infants, adolescents, young, middle-aged adults, seniors and adults of varied ages), racial\/ethnic groups, normal versus diseased subjects, smoker v. non-smoker, subject receiving therapy versus untreated subject, different time points of treatment for a particular individual or group of subjects similarly diagnosed or treated or combinations thereof. Furthermore, by determining vesicle levels at different timepoints of treatment for a particular individual, the individual's response to the treatment or progression of a disease or condition for which the individual is being treated for, can be monitored.","A reference value may be based on samples assessed from the same subject so to provide individualized tracking. Frequent testing of a patient may provide better comparisons to the reference values previously established for a particular patient and would allow a physician to more accurately assess the patient's disease stage or progression, and to inform a better decision for treatment. The reduced intraindividual variance of vesicle levels can allow a more specific and individualized threshold to be defined for the patient. Temporal intrasubject variation allows each individual to serve as a longitudinal control for optimum analysis of disease or physiological state.","Reference values can be established for unaffected individuals (of varying ages, ethnic backgrounds and sexes) without a particular phenotype by determining the amount of vesicles in an unaffected individual. For example, a reference value for a reference population can be used as a baseline for detection of one or more vesicle populations in a test subject. If a sample from a subject has a level or value that is similar to the reference, the subject can be identified to not have the disease, or of having a low likelihood of developing a disease.","Alternatively, reference values or levels can be established for individuals with a particular phenotype by determining the amount of one or more populations of vesicles in an individual with the phenotype. In addition, an index of values can be generated for a particular phenotype. For example, different disease stages can have different values, such as obtained from individuals with the different disease stages. A subject's value can be compared to the index and a diagnosis or prognosis of the disease can be determined, such as the disease stage or progression. In other embodiments, an index of values is generated for therapeutic efficacies. For example, the level of vesicles of individuals with a particular disease can be generated and noted what treatments were effective for the individual. The levels can be used to generate values of which is a subject's value is compared, and a treatment or therapy can be selected for the individual, e.g., by predicting from the levels whether the subject is likely to be a responder or non-responder for a treatment.","In some embodiments, a reference value is determined for individuals unaffected with a particular cancer, by isolating or detecting vesicles with an antigen that specifically targets biomarkers for the particular cancer. As a non-limiting example, individuals with varying stages of colorectal cancer and noncancerous polyps can be surveyed using the same techniques described for unaffected individuals and the levels of circulating vesicles for each group can be determined. In some embodiments, the levels are defined as means\u00b1standard deviations from at least two separate experiments performed in at least triplicate. Comparisons between these groups can be made using statistical tests to determine statistical significance of distinguishing biomarkers observed. In some embodiments, statistical significance is determined using a parametric statistical test. The parametric statistical test can comprise, without limitation, a fractional factorial design, analysis of variance (ANOVA), a t-test, least squares, a Pearson correlation, simple linear regression, nonlinear regression, multiple linear regression, or multiple nonlinear regression. Alternatively, the parametric statistical test can comprise a one-way analysis of variance, two-way analysis of variance, or repeated measures analysis of variance. In other embodiments, statistical significance is determined using a nonparametric statistical test. Examples include, but are not limited to, a Wilcoxon signed-rank test, a Mann-Whitney test, a Kruskal-Wallis test, a Friedman test, a Spearman ranked order correlation coefficient, a Kendall Tau analysis, and a nonparametric regression test. In some embodiments, statistical significance is determined at a p-value of less than 0.05, 0.01, 0.005, 0.001, 0.0005, or 0.0001. The p-values can also be corrected for multiple comparisons, e.g., using a Bonferroni correction, a modification thereof, or other technique known to those in the art, e.g., the Hochberg correction, Holm-Bonferroni correction, {hacek over (S)}id\u00e1k correction, Dunnett's correction or Tukey's multiple comparisons. In some embodiments, an ANOVA is followed by Tukey's correction for post-test comparing of the biomarkers from each population.","Reference values can also be established for disease recurrence monitoring (or exacerbation phase in MS), for therapeutic response monitoring, or for predicting responder\/non-responder status.","In some embodiments, a reference value for microRNA obtained from vesicles is determined using an artificial vesicle, also referred to herein as a synthetic vesicle. Methods for manufacturing artificial vesicles are known to those of skill in the art, e.g., using liposomes. Artificial vesicles can be manufactured using methods disclosed in US20060222654 and U.S. Pat. No. 4,448,765, which are incorporated herein by reference in its entirety. Artificial vesicles can be constructed with known markers to facilitate capture and\/or detection. In some embodiments, artificial vesicles are spiked into a bodily sample prior to processing. The level of intact synthetic vesicle can be tracked during processing, e.g., using filtration or other isolation methods disclosed herein, to provide a control for the amount of vesicles in the initial versus processed sample. Similarly, artificial vesicles can be spiked into a sample before or after any processing steps. In some embodiments, artificial vesicles are used to calibrate equipment used for isolation and detection of vesicles.","Artificial vesicles can be produced and used a control to test the viability of an assay, such as a bead-based assay. The artificial vesicle can bind to both the beads and to the detection antibodies. Thus, the artificial vesicle contains the amino acid sequence\/conformation that each of the antibodies binds. The artificial vesicle can comprise a purified protein or a synthetic peptide sequence to which the antibody binds. The artificial vesicle could be a bead, e.g., a polystyrene bead, that is capable of having biological molecules attached thereto. If the bead has an available carboxyl group, then the protein or peptide could be attached to the bead via an available amine group, such as using carbodiimide coupling.","In another embodiment, the artificial vesicle can be a polystyrene bead coated with avidin and a biotin is placed on the protein or peptide of choice either at the time of synthesis or via a biotin-maleimide chemistry. The proteins\/peptides to be on the bead can be mixed together in ratio specific to the application the artificial vesicle is being used for, and then conjugated to the bead. These artificial vesicles can then serve as a link between the capture beads and the detection antibodies, thereby providing a control to show that the components of the assay are working properly.","The value can be a quantitative or qualitative value. The value can be a direct measurement of the level of vesicles (example, mass per volume), or an indirect measure, such as the amount of a specific biomarker. The value can be a quantitative, such as a numerical value. In other embodiments, the value is qualitiative, such as no vesicles, low level of vesicles, medium level, high level of vesicles, or variations thereof.","The reference value can be stored in a database and used as a reference for the diagnosis, prognosis, theranosis, disease stratification, disease monitoring, treatment monitoring or prediction of non-responder\/responder status of a disease or condition based on the level or amount of microRNA, such as total amount of microRNA, or the amount of a specific population of microRNA, such as cell-of-origin specific microRNA or microRNA from vesicles with a specific biosignature. In an illustrative example, consider a method of determining a diagnosis for a cancer. MicroRNA from reference subjects with and without the cancer are assessed and stored in the database. The reference subjects provide biosignature indicative of the cancer or of another state, e.g., a healthy state. A sample from a test subject is then assayed and the microRNA biosignature is compared against those in the database. If the subject's biosignature correlates more closely with reference values indicative of cancer, a diagnosis of cancer may be made. Conversely, if the subject's biosignature correlates more closely with reference values indicative of a healthy state, the subject may be determined to not have the disease. One of skill will appreciate that this example is non-limiting and can be expanded for assessing other phenotypes, e.g., other diseases, prognosis, theranosis, disease stratification, disease monitoring, treatment monitoring or prediction of non-responder\/responder status, and the like.","A biosignature for characterizing a phenotype can be determined by detecting microRNA and\/or vesicles. The microRNA can be assessed within a vesicle. Alternately, the microRNA and vesicles in a sample are analyzed to characterize the phenotype without isolating the microRNA from the vesicles. Many analytical techniques are available to assess vesicles. In some embodiments, vesicle levels are characterized using mass spectrometry, flow cytometry, immunocytochemical staining, Western blotting, electrophoresis, chromatography or x-ray crystallography in accordance with procedures known in the art. For example, vesicles can be characterized and quantitatively measured using flow cytometry as described in Clayton et al., 2001; 163-174, which is herein incorporated by reference in its entirety. Vesicle levels may be determined using binding agents as described above. For example, a binding agent to vesicles can be labeled and the label detected and used to determine the amount of vesicles in a sample. The binding agent can be bound to a substrate, such as arrays or particles, such as described above. Alternatively, the vesicles may be labeled directly.","Electrophoretic tags or eTags can be used to determine the amount of vesicles. eTags are small fluorescent molecules linked to nucleic acids or antibodies and are designed to bind one specific nucleic acid sequence or protein, respectively. After the eTag binds its target, an enzyme is used to cleave the bound eTag from the target. The signal generated from the released eTag, called a \u201creporter,\u201d is proportional to the amount of target nucleic acid or protein in the sample. The eTag reporters can be identified by capillary electrophoresis. The unique charge-to-mass ratio of each eTag reporter\u2014that is, its electrical charge divided by its molecular weight\u2014makes it show up as a specific peak on the capillary electrophoresis readout. Thus by targeting a specific biomarker of a vesicle with an eTag, the amount or level of vesicles can be determined","The vesicle level can determined from a heterogeneous population of vesicles, such as the total population of vesicles in a sample. Alternatively, the vesicles level is determined from a homogenous population, or substantially homogenous population of vesicles, such as the level of specific cell-of-origin vesicles, such as vesicles from prostate cancer cells. In yet other embodiments, the level is determined for vesicles with a particular biomarker or combination of biomarkers, such as a biomarker specific for prostate cancer. Determining the level vesicles can be performed in conjunction with determining the biomarker or combination of biomarkers of a vesicle. Alternatively, determining the amount of vesicle may be performed prior to or subsequent to determining the biomarker or combination of biomarkers of the vesicles.","Determining the amount of vesicles can be assayed in a multiplexed manner. For example, determining the amount of more than one population of vesicles, such as different cell-of-origin specific vesicles with different biomarkers or combination of biomarkers, can be performed, such as those disclosed herein.","Performance of a diagnostic or related test is typically assessed using statistical measures. The performance of the characterization can be assessed by measuring sensitivity, specificity and related measures. For example, a level of microRNAs of interest can be assayed to characterize a phenotype, such as detecting a disease. The sensitivity and specificity of the assay to detect the disease is determined.","A true positive is a subject with a characteristic, e.g., a disease or disorder, correctly identified as having the characteristic. A false positive is a subject without the characteristic that the test improperly identifies as having the characteristic. A true negative is a subject without the characteristic that the test correctly identifies as not having the characteristic. A false negative is a person with the characteristic that the test improperly identifies as not having the characteristic. The ability of the test to distinguish between these classes provides a measure of test performance.","The specificity of a test is defined as the number of true negatives divided by the number of actual negatives (i.e., sum of true negatives and false positives). Specificity is a measure of how many subjects are correctly identified as negatives. A specificity of 100% means that the test recognizes all actual negatives\u2014for example, all healthy people will be recognized as healthy. A lower specificity indicates that more negatives will be determined as positive.","The sensitivity of a test is defined as the number of true positives divided by the number of actual positives (i.e., sum of true positives and false negatives). Specificity is a measure of how many subjects are correctly identified as positives. A sensitivity of 100% means that the test recognizes all actual positives\u2014for example, all sick people will be recognized as sick. A lower sensitivity indicates that more positives will be missed by being determined as negative.","The accuracy of a test is defined as the number of true positives and true negatives divided by the sum of all true and false positives and all true and false negatives. It provides one number that combines sensitivity and specificity measurements.","Sensitivity, specificity and accuracy are determined at a particular discrimination threshold value. For example, a common threshold for prostate cancer (PCa) detection is 4 ng\/mL of prostate specific antigen (PSA) in serum. A level of PSA equal to or above the threshold is considered positive for PCa and any level below is considered negative. As the threshold is varied, the sensitivity and specificity will also vary. For example, as the threshold for detecting cancer is increased, the specificity will increase because it is harder to call a subject positive, resulting in fewer false positives. At the same time, the sensitivity will decrease. A receiver operating characteristic curve (ROC curve) is a graphical plot of the true positive rate (i.e., sensitivity) versus the false positive rate (i.e., 1\u2212specificity) for a binary classifier system as its discrimination threshold is varied. The ROC curve shows how sensitivity and specificity change as the threshold is varied. The Area Under the Curve (AUC) of an ROC curve provides a summary value indicative of a test's performance over the entire range of thresholds. The AUC is equal to the probability that a classifier will rank a randomly chosen positive sample higher than a randomly chosen negative sample. An AUC of 0.5 indicates that the test has a 50% chance of proper ranking, which is equivalent to no discriminatory power (a coin flip also has a 50% chance of proper ranking). An AUC of 1.0 means that the test properly ranks (classifies) all subjects. The AUC is equivalent to the Wilcoxon test of ranks.","A biosignature according to the invention can be used to characterize a phenotype with at least 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, or 70% sensitivity, such as with at least 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, or 87% sensitivity. In some embodiments, the phenotype is characterized with at least 87.1, 87.2, 87.3, 87.4, 87.5, 87.6, 87.7, 87.8, 87.9, 88.0, or 89% sensitivity, such as at least 90% sensitivity. The phenotype can be characterized with at least 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100% sensitivity.","A biosignature according to the invention can be used to characterize a phenotype of a subject with at least 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, or 97% specificity, such as with at least 97.1, 97.2, 97.3, 97.4, 97.5, 97.6, 97.7, 97.8, 97.8, 97.9, 98.0, 98.1, 98.2, 98.3, 98.4, 98.5, 98.6, 98.7, 98.8, 98.9, 99.0, 99.1, 99.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99.8, 99.9 or 100% specificity.","A biosignature according to the invention can be used to characterize a phenotype of a subject, e.g., based on microRNA level or other characteristic, with at least 50% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 55% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 60% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 65% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 70% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 75% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 80% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 85% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 86% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 87% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 88% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 89% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 90% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 91% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 92% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 93% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 94% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 95% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 96% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 97% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 98% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; at least 99% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity; or substantially 100% sensitivity and at least 60, 65, 70, 75, 80, 85, 90, 95, 99, or 100% specificity.","A biosignature according to the invention can be used to characterize a phenotype of a subject with at least 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, or 97% accuracy, such as with at least 97.1, 97.2, 97.3, 97.4, 97.5, 97.6, 97.7, 97.8, 97.8, 97.9, 98.0, 98.1, 98.2, 98.3, 98.4, 98.5, 98.6, 98.7, 98.8, 98.9, 99.0, 99.1, 99.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99.8, 99.9 or 100% accuracy.","In some embodiments, a biosignature according to the invention is used to characterize a phenotype of a subject with an AUC of at least 0.60, 0.61, 0.62, 0.63, 0.64, 0.65, 0.66, 0.67, 0.68, 0.69, 0.70, 0.71, 0.72, 0.73, 0.74, 0.75, 0.76, 0.77, 0.78, 0.79, 0.80, 0.81, 0.82, 0.83, 0.84, 0.85, 0.86, 0.87, 0.88, 0.89, 0.90, 0.91, 0.92, 0.93, 0.94, 0.95, 0.96, or 0.97, such as with at least 0.971, 0.972, 0.973, 0.974, 0.975, 0.976, 0.977, 0.978, 0.978, 0.979, 0.980, 0.981, 0.982, 0.983, 0.984, 0.985, 0.986, 0.987, 0.988, 0.989, 0.99, 0.991, 0.992, 0.993, 0.994, 0.995, 0.996, 0.997, 0.998, 0.999 or 1.00.","Furthermore, the confidence level for determining the specificity, sensitivity, accuracy or AUC, may be determined with at least 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, or 99% confidence.","Other related performance measures include positive and negative likelihood ratios [positive LR=sensitivity\/(1\u2212specificity); negative LR=(1\u2212sensitivity)\/specificity]. Such measures can also be used to gauge test performance according to the methods of the invention.","Classification","Biosignature according to the invention can be used to classify a sample. Techniques for discriminate analysis are known to those of skill in the art. For example, a sample can be classified as, or predicted to be, a responder or non-responder to a given treatment for a given disease or disorder. Many statistical classification techniques are known to those of skill in the art. In supervised learning approaches, a group of samples from two or more groups are analyzed with a statistical classification method. Biomarkers can be discovered that can be used to build a classifier that differentiates between the two or more groups. A new sample can then be analyzed so that the classifier can associate the new with one of the two or more groups. Commonly used supervised classifiers include without limitation the neural network (multi-layer perceptron), support vector machines, k-nearest neighbors, Gaussian mixture model, Gaussian, naive Bayes, decision tree and radial basis function (RBF) classifiers. Linear classification methods include Fisher's linear discriminant, logistic regression, naive Bayes classifier, perceptron, and support vector machines (SVMs). Other classifiers for use with the invention include quadratic classifiers, k-nearest neighbor, boosting, decision trees, random forests, neural networks, pattern recognition, Bayesian networks and Hidden Markov models. One of skill will appreciate that these or other classifiers, including improvements of any of these, are contemplated within the scope of the invention.","Classification using supervised methods is generally performed by the following methodology:","In order to solve a given problem of supervised learning (e.g. learning to recognize handwriting) one has to consider various steps:","1. Gather a training set. These can include, for example, samples that are from a subject with or without a disease or disorder, subjects that are known to respond or not respond to a treatment, subjects whose disease progresses or does not progress, etc. The training samples are used to \u201ctrain\u201d the classifier.","2. Determine the input \u201cfeature\u201d representation of the learned function. The accuracy of the learned function depends on how the input object is represented. Typically, the input object is transformed into a feature vector, which contains a number of features that are descriptive of the object. The number of features should not be too large, because of the curse of dimensionality; but should be large enough to accurately predict the output. The features might include a set of biomarkers such as those derived from vesicles as described herein.","3. Determine the structure of the learned function and corresponding learning algorithm. A learning algorithm is chosen, e.g., artificial neural networks, decision trees, Bayes classifiers or support vector machines. The learning algorithm is used to build the classifier.","4. Build the classifier. The learning algorithm is run the gathered training set. Parameters of the learning algorithm may be adjusted by optimizing performance on a subset (called a validation set) of the training set, or via cross-validation. After parameter adjustment and learning, the performance of the algorithm may be measured on a test set of naive samples that is separate from the training set.","Once the classifier is determined as described above, it can be used to classify a sample, e.g., that of a subject who is being analyzed by the methods of the invention. As an example, a classifier can be built using data for levels of microRNA of interest in reference subjects with and without a disease as the training and test sets. MicroRNA levels found in a sample from a test subject are assessed and the classifier is used to classify the subject as with or without the disease. As another example, a classifier can be built using data for levels of vesicle biomarkers of interest in reference subjects that have been found to respond or not respond to certain diseases as the training and test sets. The vesicle biomarker levels found in a sample from a test subject are assessed and the classifier is used to classify the subject as with or without the disease.","Unsupervised learning approaches can also be used with the invention. Clustering is an unsupervised learning approach wherein a clustering algorithm correlates a series of samples without the use the labels. The most similar samples are sorted into \u201cclusters.\u201d A new sample could be sorted into a cluster and thereby classified with other members that it most closely associates. Many clustering algorithms well known to those of skill in the art can be used with the invention, such as hierarchical clustering.","Biosignatures","A biosignature can be obtained according to the invention by assessing a vesicle population, including surface and payload vesicle associated biomarkers, and\/or circulating biomarkers including microRNA and protein. A biosignature derived from a subject can be used to characterize a phenotype of the subject. A biosignature can further include the level of one or more additional biomarkers, e.g., circulating biomarkers or biomarkers associated with a vesicle of interest. A biosignature of a vesicle of interest can include particular antigens or biomarkers that are present on the vesicle. The biosignature can also include one or more antigens or biomarkers that are carried as payload within the vesicle, including the microRNA under examination. The biosignature can comprise a combination of one or more antigens or biomarkers that are present on the vesicle with one or more biomarkers that are detected in the vesicle. The biosignature can further comprise other information about a vesicle aside from its biomarkers. Such information can include vesicle size, circulating half-life, metabolic half-life, and specific activity in vivo or in vitro. The biosignature can comprise the biomarkers or other characteristics used to build a classifier.","In some embodiments, the microRNA is detected directly in a biological sample. For example, RNA in a bodily fluid can be isolated using commercially available kits such as mirVana kits (Applied Biosystems\/Ambion, Austin, Tex.), MagMAX\u2122 RNA Isolation Kit (Applied Biosystems\/Ambion, Austin, Tex.), and QIAzol Lysis Reagent and RNeasy Midi Kit (Qiagen Inc., Valencia Calif.). Particular species of microRNAs can be determined using array or PCR techniques as described below.","In some embodiments, the microRNA payload with vesicles is assessed in order to characterize a phenotype. The vesicles can be purified or concentrated prior to determining the biosignature. For example, a cell-of-origin specific vesicle can be isolated and its biosignature determined. Alternatively, the biosignature of the vesicle can be directly assayed from a sample, without prior purification or concentration. The biosignature of the invention can be used to determine a diagnosis, prognosis, or theranosis of a disease or condition or similar measures described herein. A biosignature can also be used to determine treatment efficacy, stage of a disease or condition, or progression of a disease or condition, or responder\/non-responder status. Furthermore, a biosignature may be used to determine a physiological state, such as pregnancy.","A characteristic of a vesicle in and of itself can be assessed to determine a biosignature. The characteristic can be used to diagnose, detect or determine a disease stage or progression, the therapeutic implications of a disease or condition, or characterize a physiological state. Such characteristics include without limitation the level or amount of vesicles, vesicle size, temporal evaluation of the variation in vesicle half-life, circulating vesicle half-life, metabolic half-life of a vesicle, or activity of a vesicle.","Biomarkers that can be included in a biosignature include one or more proteins or peptides (e.g., providing a protein signature), nucleic acids (e.g. RNA signature as described, or a DNA signature), lipids (e.g. lipid signature), or combinations thereof. In some embodiments, the biosignature can also comprise the type or amount of drug or drug metabolite present in a vesicle, (e.g., providing a drug signature), as such drug may be taken by a subject from which the biological sample is obtained, resulting in a vesicle carrying the drug or metabolites of the drug.","A biosignature can also include an expression level, presence, absence, mutation, variant, copy number variation, truncation, duplication, modification, or molecular association of one or more biomarkers. A genetic variant, or nucleotide variant, refers to changes or alterations to a gene or cDNA sequence at a particular locus, including, but not limited to, nucleotide base deletions, insertions, inversions, and substitutions in the coding and non-coding regions. Deletions may be of a single nucleotide base, a portion or a region of the nucleotide sequence of the gene, or of the entire gene sequence. Insertions may be of one or more nucleotide bases. The genetic variant may occur in transcriptional regulatory regions, untranslated regions of mRNA, exons, introns, or exon\/intron junctions. The genetic variant may or may not result in stop codons, frame shifts, deletions of amino acids, altered gene transcript splice forms or altered amino acid sequence.","In an embodiment, nucleic acid biomarkers, including nucleic acid payload within a vesicle, is assessed for nucleotide variants. The nucleic acid biomarker may comprise one or more RNA species, e.g., mRNA, miRNA, snoRNA, snRNA, rRNAs, tRNAs, siRNA, hnRNA, shRNA, or a combination thereof. Similarly, DNA payload can be assessed to form a DNA signature.","An RNA signature or DNA signature can also include a mutational, epigenetic modification, or genetic variant analysis of the RNA or DNA present in the vesicle. Epigenetic modifications include patterns of DNA methylation. See, e.g., Lesche R. and Eckhardt F., DNA methylation markers: a versatile diagnostic tool for routine clinical use. Curr Opin Mol Ther. 2007 June; 9(3):222-30, which is incorporated herein by reference in its entirety. Thus, a biomarker can be the methylation status of a segment of DNA.","A biosignature can comprise one or more miRNA signatures combined with one or more additional signatures including, but not limited to, an mRNA signature, DNA signature, protein signature, peptide signature, antigen signature, or any combination thereof. For example, the biosignature can comprise one or more miRNA biomarkers with one or more DNA biomarkers, one or more mRNA biomarkers, one or more snoRNA biomarkers, one or more protein biomarkers, one or more peptide biomarkers, one or more antigen biomarkers, one or more antigen biomarkers, one or more lipid biomarkers, or any combination thereof.","A biosignature can comprise a combination of one or more antigens or binding agents (such as ability to bind one or more binding agents), such as listed in , respectively, or those described elsewhere herein. The biosignature can further comprise one or more other biomarkers, such as, but not limited to, miRNA, DNA (e.g. single stranded DNA, complementary DNA, or noncoding DNA), or mRNA. The biosignature of a vesicle can comprise a combination of one or more antigens, such as shown in , one or more binding agents, such as shown in , and one or more biomarkers for a condition or disease, such as listed in . The biosignature can comprise one or more biomarkers, for example miRNA, with one or more antigens specific for a cancer cell (for example, as shown in ).","In some embodiments, a vesicle used in the subject methods has a biosignature that is specific to the cell-of-origin and is used to derive disease-specific or biological state specific diagnostic, prognostic or therapy-related biosignatures representative of the cell-of-origin. In other embodiments, a vesicle has a biosignature that is specific to a given disease or physiological condition that is different from the biosignature of the cell-of-origin for use in the diagnosis, prognosis, staging, therapy-related determinations or physiological state characterization. Biosignatures can also comprise a combination of cell-of-origin specific and non-specific vesicles.","Biosignatures can be used to evaluate diagnostic criteria such as presence of disease, disease staging, disease monitoring, disease stratification, or surveillance for detection, metastasis or recurrence or progression of disease. A biosignature can also be used clinically in making decisions concerning treatment modalities including therapeutic intervention. A biosignature can further be used clinically to make treatment decisions, including whether to perform surgery or what treatment standards should be utilized along with surgery (e.g., either pre-surgery or post-surgery). As an illustrative example, a microRNA (miRNA) biosignature that indicates an aggressive form of cancer may call for a more aggressive surgical procedure and\/or more aggressive therapeutic regimen to treat the patient.","A biosignature can be used in therapy related diagnostics to provide tests useful to diagnose a disease or choose the correct treatment regimen, such as provide a theranosis. Theranostics includes diagnostic testing that provides the ability to affect therapy or treatment of a diseased state. Theranostics testing provides a theranosis in a similar manner that diagnostics or prognostic testing provides a diagnosis or prognosis, respectively. As used herein, theranostics encompasses any desired form of therapy related testing, including predictive medicine, personalized medicine, integrated medicine, pharmacodiagnostics and Dx\/Rx partnering. Therapy related tests can be used to predict and assess drug response in individual subjects, i.e., to provide personalized medicine. Predicting a drug response can be determining whether a subject is a likely responder or a likely non-responder to a candidate therapeutic agent, e.g., before the subject has been exposed or otherwise treated with the treatment. Assessing a drug response can be monitoring a response to a drug, e.g., monitoring the subject's improvement or lack thereof over a time course after initiating the treatment. Therapy related tests are useful to select a subject for treatment who is particularly likely to benefit from the treatment or to provide an early and objective indication of treatment efficacy in an individual subject. Thus, a biosignature as disclosed herein may indicate that treatment should be altered to select a more promising treatment, thereby avoiding the great expense of delaying beneficial treatment and avoiding the financial and morbidity costs of administering an ineffective drug(s).","Therapy related diagnostics are also useful in clinical diagnosis and management of a variety of diseases and disorders, which include, but are not limited to cardiovascular disease, cancer, infectious diseases, sepsis, neurological diseases, central nervous system related diseases, endovascular related diseases, and autoimmune related diseases. Therapy related diagnostics also aid in the prediction of drug toxicity, drug resistance or drug response. Therapy related tests may be developed in any suitable diagnostic testing format, which include, but are not limited to, e.g., immunohistochemical tests, clinical chemistry, immunoassay, cell-based technologies, nucleic acid tests or body imaging methods. Therapy related tests can further include but are not limited to, testing that aids in the determination of therapy, testing that monitors for therapeutic toxicity, or response to therapy testing. Thus, a biosignature can be used to predict or monitor a subject's response to a treatment. A biosignature can be determined at different time points for a subject after initiating, removing, or altering a particular treatment.","In some embodiments, a determination or prediction as to whether a subject is responding to a treatment is made based on a change in the amount of one or more components of a biosignature (i.e., the microRNA, vesicles and\/or biomarkers of interest), an amount of one or more components of a particular biosignature, or the biosignature detected for the components. In another embodiment, a subject's condition is monitored by determining a biosignature at different time points. The progression, regression, or recurrence of a condition is determined. Response to therapy can also be measured over a time course. Thus, the invention provides a method of monitoring a status of a disease or other medical condition in a subject, comprising isolating or detecting a biosignature from a biological sample from the subject, detecting the overall amount of the components of a particular biosignature, or detecting the biosignature of one or more components (such as the presence, absence, or expression level of a biomarker). The biosignatures are used to monitor the status of the disease or condition.","In some embodiments, a biosignature is used to determine whether a particular disease or condition is resistant to a drug. If a subject is drug resistant, a physician need not waste valuable time with such drug treatment. To obtain early validation of a drug choice or treatment regimen, a biosignature is determined for a sample obtained from a subject. The biosignature is used to assess whether the particular subject's disease has the biomarker associated with drug resistance. Such a determination enables doctors to devote critical time as well as the patient's financial resources to effective treatments.","Moreover, biosignature may be used to assess whether a subject is afflicted with disease, is at risk for developing disease or to assess the stage or progression of the disease. For example, a biosignature can be used to assess whether a subject has prostate cancer (for example, ) or colon cancer (for example, ). Furthermore, a biosignature can be used to determine a stage of a disease or condition, such as colon cancer (for example, ).","Furthermore, determining the amount of vesicles, such a heterogeneous population of vesicles, and the amount of one or more homogeneous population of vesicles, such as a population of vesicles with the same biosignature, can be used to characterize a phenotype. For example, determination of the total amount of vesicles in a sample (i.e. not cell-type specific) and determining the presence of one or more different cell-of-origin specific vesicles can be used to characterize a phenotype. Threshold values, or reference values or amounts can be determined based on comparisons of normal subjects and subjects with the phenotype of interest, as further described below, and criteria based on the threshold or reference values determined The different criteria can be used to characterize a phenotype.","One criterion can be based on the amount of a heterogeneous population of vesicles in a sample. In one embodiment, general vesicle markers, such as CD9, CD81, and CD63 can be used to determine the amount of vesicles in a sample. The expression level of CD9, CD81, CD63, or a combination thereof can be detected and if the level is greater than a threshold level, the criterion is met. In another embodiment, the criterion is met if if level of CD9, CD81, CD63, or a combination thereof is lower than a threshold value or reference value. In another embodiment, the criterion can be based on whether the amount of vesicles is higher than a threshold or reference value. Another criterion can be based on the amount of vesicles with a specific biosignature. If the amount of vesicles with the specific biosignature is lower than a threshold or reference value, the criterion is met. In another embodiment, if the amount of vesicles with the specific biosignature is higher than a threshold or reference value, the criterion is met. A criterion can also be based on the amount of vesicles derived from a particular cell type. If the amount is lower than a threshold or reference value, the criterion is met. In another embodiment, if the amount is higher than a threshold value, the criterion is met.","In a non-limiting example, consider that vesicles from prostate cells are determined by detecting the biomarker PCSA or PSCA, and that a criterion is met if the level of detected PCSA or PSCA is greater than a threshold level. The threshold can be the level of the same markers in a sample from a control cell line or control subject. Another criterion can be based on whether the amount of vesicles derived from a cancer cell or comprising one or more cancer specific biomarkers. For example, the biomarkers B7H3, EpCam, or both, can be determined and a criterion met if the level of detected B7H3 and\/or EpCam is greater than a threshold level or within a pre-determined range. If the amount is lower, or higher, than a threshold or reference value, the criterion is met. A criterion can also be the reliability of the result, such as meeting a quality control measure or value. A detected amount of B7H3 and\/or EpCam in a test sample that is above the amount of these markers in a control sample may indicate the presence of a cancer in the test sample.","As described, analysis of multiple markers can be combined to assess whether a criterion is met. In an illustrative example, a biosignature is used to assess whether a subject has prostate cancer by detecting one or more of the general vesicle markers CD9, CD63 and CD81; one or more prostate epithelial markers including PCSA or PSMA; and one or more cancer markers such as B7H3 and\/or EpCam. Higher levels of the markers in a sample from a subject than in a control individual without prostate cancer indicates the presence of the prostate cancer in the subject. In some embodiments, the multiple markers are assessed in a multiplex fashion.","One of skill will understand that such rules based on meeting criterion as described can be applied to any appropriate biomarker. For example, the criterion can be applied to vesicle characteristics such as amount of vesicles present, amount of vesicles with a particular biosignature present, amount of vesicle payload biomarkers present, amount of microRNA or other circulating biomarkers present, and the like. The ratios of appropriate biomarkers can be determined. As illustrative examples, the criterion could be a ratio of an vesicle surface protein to another vesicle surface protein, a ratio of an vesicle surface protein to a microRNA, a ratio of one vesicle population to another vesicle population, a ratio of one circulating biomarker to another circulating biomarker, etc.","A phenotype for a subject can be characterized based on meeting any number of useful criteria. In some embodiments, at least one criterion is used for each biomarker. In some embodiments, at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 50, 60, 70, 80, 90 or at least 100 criteria are used. For example, for the characterizing of a cancer, a number of different criteria can be used when the subject is diagnosed with a cancer: 1) if the amount of microRNA in a sample from a subject is higher than a reference value; 2) if the amount of a microRNA within cell type specific vesicles (i.e. vesicles derived from a specific tissue or organ) is higher than a reference value; or 3) if the amount of microRNA within vesicles with one or more cancer specific biomarkers is higher than a reference value. Similar rules can apply if the amount of microRNA is less than or the same as the reference. The method can further include a quality control measure, such that the results are provided for the subject if the samples meet the quality control measure. In some embodiments, if the criteria are met but the quality control is questionable, the subject is reassessed.","In other embodiments, a single measure is determined for assessment of multiple biomarkers, and the measure is compared to a reference. For illustration, a test for prostate cancer might comprise multiplying the level of PSA against the level of miR-141 in a blood sample. The criterion is met if the product of the levels is above a threshold, indicating the presense of the cancer. As another illustration, a number of binding agents to general vesicle markers can carry the same label, e.g., the same fluorophore. The level of the detected label can be compared to a threshold.","Criterion can be applied to multiple types of biomarkers in addition to multiple biomarkers of the same type. For example, the levels of one or more circulating biomarkers (e.g., RNA, DNA, peptides), vesicles, mutations, etc, can be compared to a reference. Different components of a biosignature can have different criteria. As a non-limiting example, a biosignature used to diagnose a cancer can include overexpression of one miR species as compared to a reference and underexpression of a vesicle surface antigen as compared to another reference.","A biosignature can be determined by comparing the amount of vesicles, the structure of a vesicle, or any other informative characteristic of a vesicle. Vesicle structure can be assessed using transmission electron microscopy, see for example, Hansen et al., 311: 134-134(1) (1998), or scanning electron microscopy. Various combinations of methods and techniques or analyzing one or more vesicles can be used to determine a phenotype for a subject.","A biosignature can include without limitation the presence or absence, copy number, expression level, or activity level of a biomarker. Other useful components of a biosignature include the presence of a mutation (e.g., mutations which affect activity of a transcription or translation product, such as substitution, deletion, or insertion mutations), variant, or post-translation modification of a biomarker. Post-translational modification of a protein biomarker include without limitation acylation, acetylation, phosphorylation, ubiquitination, deacetylation, alkylation, methylation, amidation, biotinylation, gamma-carboxylation, glutamylation, glycosylation, glycyation, hydroxylation, covalent attachment of heme moiety, iodination, isoprenylation, lipoylation, prenylation, GPI anchor formation, myristoylation, farnesylation, geranylgeranylation, covalent attachment of nucleotides or derivatives thereof, ADP-ribosylation, flavin attachment, oxidation, palmitoylation, pegylation, covalent attachment of phosphatidylinositol, phosphopantetheinylation, polysialylation, pyroglutamate formation, racemization of proline by prolyl isomerase, tRNA-mediation addition of amino acids such as arginylation, sulfation, the addition of a sulfate group to a tyrosine, or selenoylation of the biomarker.","The methods described herein can be used to identify a biosignature that is associated with a disease, condition or physiological state. The biosignature can also be utilized to determine if a subject is afflicted with cancer or is at risk for developing cancer. A subject at risk of developing cancer can include those who may be predisposed or who have pre-symptomatic early stage disease.","A biosignature can also be utilized to provide a diagnostic or theranostic determination for other diseases including but not limited to autoimmune diseases, inflammatory bowel diseases, Alzheimer's disease, Parkinson's disease, Multiple Sclerosis, sepsis or pancreatitis or any disease, conditions or symptoms listed in .","The biosignature can also be used to identify a given pregnancy state from the peripheral blood, umbilical cord blood, or amniotic fluid (e.g. miRNA signature specific to Downs Syndrome) or adverse pregnancy outcome such as pre-eclampsia, pre-term birth, premature rupture of membranes, intrauterine growth restriction or recurrent pregnancy loss. The biosignature can also be used to indicate the health of the mother, the fetus at all developmental stages, the pre-implantation embryo or a newborn.","A biosignature can be utilized for pre-symptomatic diagnosis. Furthermore, the biosignature can be utilized to detect disease, determine disease stage or progression, determine the recurrence of disease, identify treatment protocols, determine efficacy of treatment protocols or evaluate the physiological status of individuals related to age and environmental exposure.","Monitoring a biosignature of a vesicle can also be used to identify toxic exposures in a subject including, but not limited to, situations of early exposure or exposure to an unknown or unidentified toxic agent. Without being bound by any one specific theory for mechanism of action, vesicles can shed from damaged cells and in the process compartmentalize specific contents of the cell including both membrane components and engulfed cytoplasmic contents. Cells exposed to toxic agents\/chemicals may increase vesicle shedding to expel toxic agents or metabolites thereof, thus resulting in increased vesicle levels. Thus, monitoring vesicle levels, vesicle biosignature, or both, allows assessment of an individual's response to potential toxic agent(s).","A vesicle and\/or other biomarkers of the invention can be used to identify states of drug-induced toxicity or the organ injured, by detecting one or more specific antigen, binding agent, biomarker, or any combination thereof. The level of vesicles, changes in the biosignature of a vesicle, or both, can be used to monitor an individual for acute, chronic, or occupational exposures to any number of toxic agents including, but not limited to, drugs, antibiotics, industrial chemicals, toxic antibiotic metabolites, herbs, household chemicals, and chemicals produced by other organisms, either naturally occurring or synthetic in nature. In addition, a biosignature can be used to identify conditions or diseases, including cancers of unknown origin, also known as cancers of unknown primary (CUP).","A vesicle may be isolated from a biological sample as previously described to arrive at a heterogeneous population of vesicles. The heterogeneous population of vesicles can then be contacted with substrates coated with specific binding agents designed to rule out or identify antigen specific characteristics of the vesicle population that are specific to a given cell-of-origin. Further, as described above, the biosignature of a vesicle can correlate with the cancerous state of cells. Compounds that inhibit cancer in a subject may cause a change, e.g., a change in biosignature of a vesicle, which can be monitored by serial isolation of vesicles over time and treatment course. The level of vesicles or changes in the level of vesicles with a specific biosignature can be monitored.","In one aspect, the present invention relates to biomarker discovery and biosignature discovery. In an embodiment, one or more subjects that respond to a therapy (responders) and one or more subjects that do not respond to the same therapy (non-responders) can have their vesicles interrogated. Interrogation can be performed to identify the presence of one or more biomarkers, including any of the biomarkers described herein. In one aspect, the presence, quantity, and payload of a miR are assayed. The payload of a miR can be, for example, and surface or internal protein, nucleic acid, lipid or carbohydrate.","The presence or absence of a biosignature in responders but not in the non-responders can be used for theranosis. A sample from responders may be analyzed for one or more of the following: amount of vesicles, amount of a unique subset or species of vesicles, biomarkers in such vesicles, biosignature of such vesicles, etc. In one instance, vesicles such as microvesicles or exosomes from responders and non-responders are analyzed for the presence and\/or quantity of one or more miRNAs, such as miRNA 122, miR-548c-5p, miR-362-3p, miR-422a, miR-597, miR-429, miR-200a, and\/or miR-200b. A difference in biosignatures between responders and non-responders can be used for theranosis. In another embodiment, vesicles are obtained from subjects having a disease or condition. Vesicles are also obtained from subjects free of such disease or condition. The vesicles from both groups of subjects are assayed for unique biosignatures that are associated with all subjects in that group but not in subjects from the other group. Such biosignatures or biomarkers can then used as a diagnostic for the presence or absence of the condition or disease, or to classify the subject as belonging on one of the groups (those with\/without disease, aggressive\/non-aggressive disease, responder\/non-responder, etc).","In a further example, vesicles are assayed from patients having a stage I cancer and patients having stage II or III of the same cancer. A difference in biosignatures or biomarkers between vesicles from each group of patient is identified (e.g., vesicles from stage III cancer may have an increased expression of one or more genes or miR's), thereby identifying a biosignature or biomarker that distinguishes different stages of a disease. Such biosignature can then be used to prognose patients having the disease.","In some instances, a biosignature is determined by assaying vesicles from a subject over a period of time (e.g., every day, week, month, or year). Thus, responders and non-responders or patients in phase I and phase II\/III can have their vesicles interrogated over time (e.g., every month). The payload or physical attributes of the vesicles in each point in time can be compared. A temporal pattern can thus form a biosignature that can then be used for theranosis, diagnosis, prognosis, disease stratification, treatment monitoring, disease monitoring or making a prediction of responder\/non-responder status. As a non-limiting example, an increasing amount of a biomarker (e.g., miR 122) in vesiscles over a time course can be associated with metastatic cancer, as opposed to a stagnant amounts of the biomarker in vesiscles over the time course can be associated with non-metastatic cancer. A time course may last over at least 1 week, 2 weeks, 3 weeks, 4 weeks, 1 month, 6 weeks, 8 weeks, 2 months, 10 weeks, 12 weeks, 3 months, 4 months, 5 months, 6 months, 7 months, 8 months, 9 months, 10 months, 11 months, or at least 12 months.","The level of vesicles, level of vesicles with a specific biosignature, or a biosignature of a vesicle can also be used to assess the efficacy of a therapy for a condition. For example, the level of vesicles, level of vesicles with a specific biosignature, or a biosignature of a vesicle can be used to assess the efficacy of a cancer treatment, e.g., chemotherapy, radiation therapy, surgery, or any other therapeutic approach useful for inhibiting cancer in a subject. In addition, a biosignature can be used in a screening assay to identify candidate or test compounds or agents (e.g., proteins, peptides, peptidomimetics, peptoids, small molecules or other drugs) that have a modulatory effect on the biosignature of a vesicle. Compounds identified via such screening assays may be useful, for example, for modulating, e.g., inhibiting, ameliorating, treating, or preventing conditions or diseases.","For example, a biosignature for a vesicle can be obtained from a patient who is undergoing successful treatment for a particular cancer. Cells from a cancer patient not being treated with the same drug can be cultured and vesicles from the cultures obtained for determining biosignatures. The cells can be treated with test compounds and the biosignature of the vesicles from the cultures can be compared to the biosignature of the vesicles obtained from the patient undergoing successful treatment. The test compounds that results in biosignatures that are similar to those of the patient undergoing successful treatment can be selected for further studies.","The biosignature of a vesicle can also be used to monitor the influence of an agent (e.g., drug compounds) on the biosignature in clinical trials. Monitoring the level of vesicles, changes in the biosignature of a vesicle, or both, can also be used in a method of assessing the efficacy of a test compound, such as a test compound for inhibiting cancer cells.","The level of vesicles, the biosignature of a vesicle, or both, can also be used to determine the effectiveness of a particular therapeutic intervention (pharmaceutical or non-pharmaceutical) and to alter the intervention to 1) reduce the risk of developing adverse outcomes, 2) enhance the effectiveness of the intervention or 3) identify resistant states. Thus, in addition to diagnosing or confirming the presence of or risk for developing a disease, condition or a syndrome, the methods and compositions disclosed herein also provide a system for optimizing the treatment of a subject having such a disease, condition or syndrome. For example, a therapy-related approach to treating a disease, condition or syndrome by integrating diagnostics and therapeutics to improve the real-time treatment of a subject can be determined by identifying the biosignature of a vesicle.","Tests that identify the level of vesicles, the biosignature of a vesicle, or both, can be used to identify which patients are most suited to a particular therapy, and provide feedback on how well a drug is working, so as to optimize treatment regimens. For example, in pregnancy-induced hypertension and associated conditions, therapy-related diagnostics can flexibly monitor changes in important parameters (e.g., cytokine and\/or growth factor levels) over time, to optimize treatment.","Within the clinical trial setting of investigational agents as defined by the FDA, MDA, EMA, USDA, and EMEA, therapy-related diagnostics as determined by a biosignature disclosed herein, can provide key information to optimize trial design, monitor efficacy, and enhance drug safety. For instance, for trial design, therapy-related diagnostics can be used for patient stratification, determination of patient eligibility (inclusion\/exclusion), creation of homogeneous treatment groups, and selection of patient samples that are optimized to a matched case control cohort. Such therapy-related diagnostic can therefore provide the means for patient efficacy enrichment, thereby minimizing the number of individuals needed for trial recruitment. For example, for efficacy, therapy-related diagnostics are useful for monitoring therapy and assessing efficacy criteria. Alternatively, for safety, therapy-related diagnostics can be used to prevent adverse drug reactions or avoid medication error and monitor compliance with the therapeutic regimen.","In some embodiments, the invention provides a method of identifying responder and non-responders to a treatment undergoing clinical trials, comprising detecting biosignatures comprising microRNA in subjects enrolled in the clinical trial, and identifying biosignatures that distinguish between responders and non-responders. In a further embodiment, the biosignatures are measured in a drug naive subject and used to predict whether the subject will be a responder or non-responder. The prediction can be based upon whether the biosignatures of the drug naive subject correlate more closely with the clinical trial subjects identified as responders, thereby predicting that the drug naive subject will be a responder. Conversely, if the biosignatures of the drug naive subject correlate more closely with the clinical trial subjects identified as non-responders, the methods of the invention can predict that the drug naive subject will be a non-responder. The prediction can therefore be used to stratify potential responders and non-responders to the treatment. In some embodiments, the prediction is used to guide a course of treatment, e.g., by helping treating physicians decide whether to administer the drug. In some embodiments, the prediction is used to guide selection of patients for enrollment in further clinical trials. In a non-limiting example, biosignatures that predict responder\/non-responder status in Phase II trials can be used to select patients for a Phase III trial, thereby increasing the likelihood of response in the Phase III patient population. One of skill will appreciate that the method can be adapted to identify biosignatures to stratify subjects on criteria other than responder\/non-responder status. In one embodiment, the criterion is treatment safety. Therefore the method is followed as above to identify subjects who are likely or not to have adverse events to the treatment. In a non-limiting example, biosignatures that predict safety profile in Phase II trials can be used to select patients for a Phase III trial, thereby increasing the treatment safety profile in the Phase III patient population.","Therefore, the level of vesicles, the biosignature of a vesicle, or both, can be used to monitor drug efficacy, determine response or resistance to a given drug, or both, thereby enhancing drug safety. For example, in colon cancer, vesicles are typically shed from colon cancer cells and can be isolated from the peripheral blood and used to isolate one or more biomarkers e.g., KRAS mRNA which can then be sequenced to detect KRAS mutations. In the case of mRNA biomarkers, the mRNA can be reverse transcribed into cDNA and sequenced (e.g., by Sanger sequencing, pyrosequencing, NextGen sequencing, RT-PCR assays) to determine if there are mutations present that confer resistance to a drug (e.g., cetuximab or panitumimab). In another example, vesicles that are specifically shed from lung cancer cells are isolated from a biological sample and used to isolate a lung cancer biomarker, e.g., EGFR mRNA. The EGFR mRNA is processed to cDNA and sequenced to determine if there are EGFR mutations present that show resistance or response to specific drugs or treatments for lung cancer.","One or more biosignatures can be grouped so that information obtained about the set of biosignatures in a particular group provides a reasonable basis for making a clinically relevant decision, such as but not limited to a diagnosis, prognosis, or management of treatment, such as treatment selection.","As with most diagnostic markers, it is often desirable to use the fewest number of markers sufficient to make a correct medical judgment. This prevents a delay in treatment pending further analysis as well inappropriate use of time and resources.","Also disclosed herein are methods of conducting retrospective analysis on samples (e.g., serum and tissue biobanks) for the purpose of correlating qualitative and quantitative properties, such as biosignatures of vesicles, with clinical outcomes in terms of disease state, disease stage, progression, prognosis; therapeutic efficacy or selection; or physiological conditions. Furthermore, methods and compositions disclosed herein are utilized for conducting prospective analysis on a sample (e.g., serum and\/or tissue collected from individuals in a clinical trial) for the purpose of correlating qualitative and quantitative biosignatures of vesicles with clinical outcomes in terms of disease state, disease stage, progression, prognosis; therapeutic efficacy or selection; or physiological conditions can also be performed. As used herein, a biosignature for a vesicle can be used to identify a cell-of-origin specific vesicle. Furthermore, a biosignature can be determined based on a surface marker profile of a vesicle or contents of a vesicle.","The biosignatures used to characterize a phenotype according to the invention can comprise multiple components (e.g., microRNA, vesicles or other biomarkers) or characteristics (e.g., vesicle size or morphology). The biosignatures can comprise at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30, 40, 50, 75, or 100 components or characteristics. A biosignature with more than one component or characteristic, such as at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 30, 40, 50, 75, or 100 components, may provide higher sensitivity and\/or specificity in characterizing a phenotype. In some embodiments, assessing a plurality of components or characteristics provides increased sensitivity and\/or specificity as compared to assessing fewer components or characteristics. On the other hand, it is often desirable to use the fewest number of components or characteristics sufficient to make a correct medical judgment. Fewer markers can avoid statistical overfitting of a classifier and can prevent a delay in treatment pending further analysis as well inappropriate use of time and resources. Thus, the methods of the invention comprise determining an optimal number of components or characteristics.","A biosignature according to the invention can be used to characterize a phenotype with a sensitivity, specificity, accuracy, or similar performance metric as described above. The biosignatures can also be used to build a classifier to classify a sample as belonging to a group, such as belonging to a group having a disease or not, a group having an aggressive disease or not, or a group of responders or non-responders. In one embodiment, a classifier is used to determine whether a subject has an aggressive or non-aggressive cancer. In the illustrative case of prostate cancer, this can help a physician to determine whether to watch the cancer, i.e., prescribe \u201cwatchful waiting,\u201d or perform a prostatectomy. In another embodiment, a classifier is used to determine whether a breast cancer patient is likely to respond or not to tamoxifen, thereby helping the physician to determine whether or not to treat the patient with tamoxifen or another drug.","Biomarkers","A biosignature used to characterize a phenotype can comprise one or more biomarkers. The biomarker can be a circulating marker, a membrane associated marker, or a component present within a vesicle or on a vesicle's surface. These biomarkers include without limitation a nucleic acid (e.g. RNA (mRNA, miRNA, etc.) or DNA), protein, peptide, polypeptide, antigen, lipid, carbohydrate, or proteoglycan.","The biosignature can include the presence or absence, expression level, mutational state, genetic variant state, or any modification (such as epigentic modification, post-translation modification) of a biomarker (e.g. any one or more biomarker listed in ). The expression level of a biomarker can be compared to a control or reference, to determine the overexpression or underexpression (or upregulation or downregulation) of a biomarker in a sample. In some embodiments, the control or reference level comprises the amount of a same biomarker, such as a miRNA, in a control sample from a subject that does not have or exhibit the condition or disease. In another embodiment, the control of reference levels comprises that of a housekeeping marker whose level is minimally affected, if at all, in different biological settings such as diseased versus non-diseased states. In yet another embodiment, the control or reference level comprises that of the level of the same marker in the same subject but in a sample taken at a different time point. Other types of controls are described herein.","Nucleic acid biomarkers include various RNA or DNA species. For example, the biomarker can be mRNA, microRNA (miRNA), small nucleolar RNAs (snoRNA), small nuclear RNAs (snRNA), ribosomal RNAs (rRNA), heterogeneous nuclear RNA (hnRNA), ribosomal RNAS (rRNA), siRNA, transfer RNAs (tRNA), or shRNA. The DNA can be double-stranded DNA, single stranded DNA, complementary DNA, or noncoding DNA. miRNAs are short ribonucleic acid (RNA) molecules which average about 22 nucleotides long. miRNAs act as post-transcriptional regulators that bind to complementary sequences in the three prime untranslated regions (3\u2032 UTRs) of target messenger RNA transcripts (mRNAs), which can result in gene silencing. One miRNA may act upon 1000s of mRNAs. miRNAs play multiple roles in negative regulation, e.g., transcript degradation and sequestering, translational suppression, and may also have a role in positive regulation, e.g., transcriptional and translational activation. By affecting gene regulation, miRNAs can influence many biologic processes. Different sets of expressed miRNAs are found in different cell types and tissues.","Biomarkers for use with the invention further include peptides, polypeptides, or proteins, which terms are used interchangeably throughout unless otherwise noted. In some embodiments, the protein biomarker comprises its modification state, truncations, mutations, expression level (such as overexpression or underexpression as compared to a reference level), and\/or post-translational modifications, such as described above. In a non-limiting example, a biosignature for a disease can include a protein having a certain post-translational modification that is more prevalent in a sample associated with the disease than without.","A biosignature may include a number of the same type of biomarkers (e.g., two different microRNA species) or one or more of different types of biomarkers (e.g. mRNAs, miRNAs, proteins, peptides, ligands, and antigens).","One or more biosignatures can comprise at least one biomarker selected from those listed in . A specific cell-of-origin biosignature may include one or more biomarkers.  depict tables which lists a number of disease or condition specific biomarkers that can be derived and analyzed from a vesicle. The biomarker can also be CD24, midkine, hepcidin, TMPRSS2-ERG, PCA-3, PSA, EGFR, EGFRvIII, BRAF variant, MET, cKit, PDGFR, Wnt, beta-catenin, K-ras, H-ras, N-ras, Raf, N-myc, c-myc, IGFR, PI3K, Akt, BRCA1, BRCA2, PTEN, VEGFR-2, VEGFR-1, Tie-2, TEM-1, CD276, HER-2, HER-3, or HER-4. The biomarker can also be annexin V, CD63, Rab-5b, or caveolin, or a miRNA, such as let-7a; miR-15b; miR-16; miR-19b; miR-21; miR-26a; miR-27a; miR-92; miR-93; miR-320 or miR-20. The biomarker can also be of any gene or fragment thereof as disclosed in PCT Publication No. WO2009\/100029, such as those listed in Tables 3-15 therein.","Other biomarkers useful for assessment in methods and compositions disclosed herein include those associated with conditions or physiological states as disclosed in U.S. Pat. Nos. 6,329,179 and 7,625,573; U.S. Patent Publication Nos. 2002\/106684, 2004\/005596, 2005\/0159378, 2005\/0064470, 2006\/116321, 2007\/0161004, 2007\/0077553, 2007\/104738, 2007\/0298118, 2007\/0172900, 2008\/0268429, 2010\/0062450, 2007\/0298118, 2009\/0220944 and 2010\/0196426; U.S. patent application Ser. Nos. 12\/524,432, 12\/524,398, 12\/524,462; Canadian Patent CA 2453198; and International PCT Patent Publication Nos. WO1994022018, WO2001036601, WO2003063690, WO2003044166, WO2003076603, WO2005121369, WO2005118806, WO\/2005\/078124, WO2007126386, WO2007088537, WO2007103572, WO2009019215, WO2009021322, WO2009036236, WO2009100029, WO2009015357, WO2009155505, WO 2010\/065968 and WO 2010\/070276; each of which patent or application is incorporated herein by reference in their entirety. The biomarkers disclosed in these patents and applications, including vesicle biomarkers and microRNAs, can be assessed as part of a signature for characterizing a phenotype, such as providing a diagnosis, prognosis or theranosis of a cancer or other disease. Furthermore, the methods and techniques disclosed therein can be used to assess biomarkers, including vesicle biomarkers and microRNAs.","Another group of useful biomarkers for assessment in methods and compositions disclosed herein include those associated with cancer diagnostics, prognostics and theranostics as disclosed in U.S. Pat. Nos. 6,692,916, 6,960,439, 6,964,850, 7,074,586; U.S. patent application Ser. Nos. 11\/159,376, 11\/804,175, 12\/594,128, 12\/514,686, 12\/514,775, 12\/594,675, 12\/594,911, 12\/594,679, 12\/741,787, 12\/312,390; and International PCT Patent Application Nos. PCT\/US2009\/049935, PCT\/US2009\/063138, PCT\/US2010\/000037; each of which patent or application is incorporated herein by reference in their entirety. Usefule biomarkers further include those described in U.S. patent application Ser. No. 10\/703,143 and U.S. Ser. No. 10\/701,391 for inflammatory disease; Ser. No. 11\/529,010 for rheumatoid arthritis; Ser. No. 11\/454,553 and Ser. No. 11\/827,892 for multiple sclerosis; Ser. No. 11\/897,160 for transplant rejection; Ser. No. 12\/524,677 for lupus; PCT\/US2009\/048684 for osteoarthritis; Ser. No. 10\/742,458 for infectious disease and sepsis; Ser. No. 12\/520,675 for sepsis; each of which patent or application is incorporated herein by reference in their entirety. The biomarkers disclosed in these patents and applications, including mRNAs, can be assessed as part of a signature for characterizing a phenotype, such as providing a diagnosis, prognosis or theranosis of a cancer or other disease. Furthermore, the methods and techniques disclosed therein can be used to assess biomarkers, including vesicle biomarkers and microRNAs.","Still other biomarkers useful for assessment in methods and compositions disclosed herein include those associated with conditions or physiological states as disclosed in Wieczorek et al., Isolation and characterization of an RNA-proteolipid complex associated with the malignant state in humans, Proc Natl Acad Sci USA. 1985 May; 82(10):3455-9; Wieczorek et al., Diagnostic and prognostic value of RNA-proteolipid in sera of patients with malignant disorders following therapy: first clinical evaluation of a novel tumor marker, Cancer Res. 1987 Dec. 1; 47(23):6407-12; Escola et al. -. J. Biol. Chem. (1998) 273:20121-27; Pileri et al. 81 Science, (1998) 282:938-41); Kopreski et al. , Clin. Cancer Res. (1999) 5:1961-1965; Carr et al. , Cancer Research, (1985) 45:5944-51; Weichert et al. Cytoplasmic CD24 expression in colorectal cancer independently correlates with shortened patient survival. Clinical Cancer Research, 2005, 11:6574-81); Iorio et al. . Cancer Res (2005) 65:7065-70; Taylor et al. ---British J Cancer (2005) 92:305-11; Valadi et al. -Nature Cell Biol (2007) 9:654-59; Taylor et al. -J Immunol (2006) 176:1534-42; Koga et al. -Anticancer Res (2005) 25:3703-08; Seligson et al. () Clin Cancer Res (2004) 10:2659-69; Clayton et al. (--55 59. Eur J Immunol (2003) 33:522-31); Simak et al. Trans Med Reviews (2006) 20:1-26; Choi et al. J Proteome Res (2007) 6:4646-4655; Iero et al. -Cell Death Diff (2008) 15:80-88; Baj-Krzyworzeka et al. -Cencer Immunol Immunother (2006) 55:808-18; Admyre et al. --2-J Allergy Clin Immunol (2007) 120:1418-1424; Aoki et al. 33-1 --8-Endocrinol (2007) 148:3850-3862; Baj-Krzyworzeka et al. -Cencer Immunol Immunother (2006) 55:808-18; Skog et al. Nature Cell Biol (2008) 10:1470-76; El-Hefnawy et al. Clin Chem (2004) 50:564-573; Pisitkun et al., 2004; 101:13368-13373; Mitchell et al., Can urinary exosomes act as treatment response markers in Prostate Cancer?, Journal of Translational Medicine 2009, 7:4; Clayton et al., Human Tumor-Derived Exosomes Selectively Impair Lymphocyte Responses to Interleukin-2, Cancer Res 2007; 67: (15). Aug. 1, 2007; Rabesandratana et al. -(55) (59) . Blood 91:2573-2580 (1998); Lamparski et al. . J Immunol Methods 270:211-226 (2002); Keller et al. 24 . Kidney Int'l 72:1095-1102 (2007); Runz et al. -24 . Gyn Oncol 107:563-571 (2007); Redman et al. 29:73-77 (2008); Gutwein et al. 1 . Clin Cancer Res 11:2492-2501 (2005); Kristiansen et al., CD24 is an independent prognostic marker of survival in nonsmall cell lung cancer patients, Brit J Cancer 88:231-236 (2003); Lim and Oh, The Role of CD24 in Various Human Epithelial Neoplasias, Pathol Res Pract 201:479-86 (2005); Matutes et al., The Immunophenotype of Splenic Lymphoma with Villous Lymphocytes and its Relevance to the Differential Diagnosis With Other B-Cell Disorders, Blood 83:1558-1562 (1994); Pirruccello and Lang, Differential Expression of CD24-Related Epitopes in Mycosis Fungoides\/Sezary Syndrome: A Potential Marker for Circulating Sezary Cells, Blood 76:2343-2347 (1990). The biomarkers disclosed in these publications, including vesicle biomarkers and microRNAs, can be assessed as part of a signature for characterizing a phenotype, such as providing a diagnosis, prognosis or theranosis of a cancer or other disease. Furthermore, the methods and techniques disclosed therein can be used to assess biomarkers, including vesicle biomarkers and microRNAs.","Still other biomarkers useful for assessment in methods and compositions disclosed herein include those associated with conditions or physiological states as disclosed in Rajendran et al., 2006; 103:11172-11177, Taylor et al., 2008; 110:13-21, Zhou et al., 2008; 74:613-621, Buning et al., 2008, Prado et al. 2008; 181:1519-1525, Vella et al. (2008) 124(3-4): 385-93, Gould et al. (2003). 100(19): 10592-7, Fang et al. (2007). 5(6): e158, Chen, B. J and R. A. Lamb (2008). 372(2): 221-32, Bhatnagar, S, and J. S. Schorey (2007). 282(35): 25779-89, Bhatnagar et al. (2007) 110(9): 3234-44, Yuyama, et al. (2008). 105(1): 217-24, Gomes et al. (2007). 428(1): 43-6, Nagahama et al. (2003). 36(3): 125-31, Taylor, D. D., S. Akyol, et al. (2006). 176(3): 1534-42, Peche, et al. (2006). 6(7): 1541-50, Zero, M., M. Valenti, et al. (2008). 15: 80-88, Gesierich, S., I. Berezoversuskiy, et al. (2006), 66(14): 7083-94, Clayton, A., A. Turkes, et al. (2004). 18(9): 977-9, Skriner., K. Adolph, et al. (2006). 54(12): 3809-14, Brouwer, R., G. J Pruijn, et al. (2001). 3(2): 102-6, Kim, S. H, N Bianco, et al. (2006). 13(2): 289-300, Evans, C. H, S. C. Ghivizzani, et al. (2000). (379 Suppl): S300-7, Zhang, H G., C. Liu, et al. (2006). 176(12): 7385-93, Van Niel, G., J. Mallegol, et al. (2004). 52: 1690-1697, Fiasse, R. and O. Dewit (2007). 17(12): 1423-1441(19). The biomarkers disclosed in these publications, including vesicle biomarkers and microRNAs, can be assessed as part of a signature for characterizing a phenotype, such as providing a diagnosis, prognosis or theranosis of a cancer or other disease. Furthermore, the methods and techniques disclosed therein can be used to assess biomarkers, including vesicle biomarkers and microRNAs.","A biomarker that can be derived and analyzed from a vesicle is miRNA (miR), miRNA*nonsense (miR*), and other RNAs (including, but not limited to, mRNA, preRNA, priRNA, hnRNA, snRNA, siRNA, shRNA). A miRNA biomarker includes not only its miRNA and microRNA* nonsense, but its precursor molecules: pri-microRNAs (pri-miRs) and pre-microRNAs (pre-miRs). The sequence of a miRNA can be obtained from publicly available databases such as http:\/\/www.mirbase.org\/, http:\/\/www.microrna.org\/, or any others available. The biomarker can also be a nucleic acid molecule (e.g. DNA), protein, or peptide. The presence or absence, expression level, mutations (for example genetic mutations, such as deletions, translocations, duplications, nucleotide or amino acid substitutions, and the like) can be determined for the biomarker. Any epigenetic modulation or copy number variation of a biomarker can also be analyzed.","The one or more biomarkers analyzed can be indicative of a particular tissue or cell of origin, disease, or physiological state. Furthermore, the presence, absence or expression level of one or more of the biomarkers described herein can be correlated to a phenotype of a subject, including a disease, condition, prognosis or drug efficacy. The specific biomarker and biosignature set forth below constitute non-inclusive examples for each of the diseases, condition comparisons, conditions, and\/or physiological states. Furthermore, the one or more biomarker assessed for a phenotype can be a cell-of-origin specific vesicle.","The one or more miRNAs used to characterize a phenotype may be selected from those disclosed in PCT Publication No. WO2009\/036236. For example, one or more miRNAs listed in Tables I-VI () therein can be used to characterize colon adenocarcinoma, colorectal cancer, prostate cancer, lung cancer, breast cancer, b-cell lymphoma, pancreatic cancer, diffuse large BCL cancer, CLL, bladder cancer, renal cancer, hypoxia-tumor, uterine leiomyomas, ovarian cancer, hepatitis C virus-associated hepatocellular carcinoma, ALL, Alzheimer's disease, myelofibrosis, myelofibrosis, polycythemia vera, thrombocythemia, HIV, or HIV-I latency, as further described herein.","The one or more miRNAs can be detected in a vesicle. The one or more miRNAs can be miR-223, miR-484, miR-191, miR-146a, miR-016, miR-026a, miR-222, miR-024, miR-126, and miR-32. One or more miRNAs can also be detected in PBMC. The one or more miRNAs can be miR-223, miR-150, miR-146b, miR-016, miR-484, miR-146a, miR-191, miR-026a, miR-019b, or miR-020a. The one or more miRNAs can be used to characterize a particular disease or condition. For example, for the disease bladder cancer, one or more miRNAs can be detected, such as miR-223, miR-26b, miR-221, miR-103-1, miR-185, miR-23b, miR-203, miR-17-5p, miR-23a, miR-205 or any combination thereof. The one or more miRNAs may be upregulated or overexpressed.","In some embodiments, the one or more miRNAs is used to characterize hypoxia-tumor. The one or more miRNA may be miR-23, miR-24, miR-26, miR-27, miR-103, miR-107, miR-181, miR-210, or miR-213, and may be upregulated. One or more miRNAs can also be used to characterize uterine leiomyomas. For example, the one or more miRNAs used to characterize a uterine leiomyoma may be a let-7 family member, miR-21, miR-23b, miR-29b, or miR-197. The miRNA can be upregulated.","Myelofibrosis can also be characterized by one or more miRNAs, such as miR-190, which can be upregulated; miR-31, miR-150 and miR-95, which can be downregulated, or any combination thereof. Furthermore, myelofibrosis, polycythemia vera or thrombocythemia can also be characterized by detecting one or more miRNAs, such as, but not limited to, miR-34a, miR-342, miR-326, miR-105, miR-149, miR-147, or any combination thereof. The one or more miRNAs may be down-regulated.","Other examples of phenotypes that can be characterized by assessing a vesicle for one or more biomarkers are further described herein.","The one or more biomarkers can be detected using a probe. A probe can comprise an oligonucleotide, such as DNA or RNA, an aptamer, monoclonal antibody, polyclonal antibody, Fabs, Fab\u2032, single chain antibody, synthetic antibody, peptoid, zDNA, peptide nucleic acid (PNA), locked nucleic acid (LNA), lectin, synthetic or naturally occurring chemical compound (including but not limited to a drug or labeling reagent), dendrimer, or a combination thereof. The probe can be directly detected, for example by being directly labeled, or be indirectly detected, such as through a labeling reagent. The probe can selectively recognize a biomarker. For example, a probe that is an oligonucleotide can selectively hybridize to a miRNA biomarker.","In aspects, the invention provides for the diagnosis, theranosis, prognosis, disease stratification, disease staging, treatment monitoring or predicting responder\/non-responder status of a disease or disorder in a subject. The invention comprises assessing vesicles from a subject, including assessing biomarkers present on the vesicles and\/or assessing payload within the vesicles, such as protein, nucleic acid or other biological molecules. Any appropriate biomarker that can be assessed using a vesicle and that relates to a disease or disorder can be used the carry out the methods of the invention. Furthermore, any appropriate technique to assess a vesicle as described herein can be used. Exemplary biomarkers for specific diseases that can be assessed according to the methods of the invention include the following:","Breast Cancer","Breast cancer specific biomarkers can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNA, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in .","One or more breast cancer specific biomarker can be assessed to provide a breast cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, including but not limited to, miR-21, miR-155, miR-206, miR-122a, miR-210, miR-21, miR-21, miR-155, miR-206, miR-122a, miR-210, or miR-21, or any combination thereof.","The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, let-7, miR-10b, miR-125a, miR-125b, miR-145, miR-143, miR-145, miR-16, let-7, let-7, let-7, miR-10b, miR-125a, miR-125b, or miR-145, or any combination thereof.","The mRNAs that may be analyzed can include, but are not limited to, ER, PR, HER2, MUC1, or EGFR, or any combination thereof. Mutations including, but not limited to, those related to KRAS, B-Raf, or CYP2D6, or any combination thereof can also be used as specific biomarkers from a vesicle for breast cancer. In addition, a protein, ligand, or peptide that can be used as biomarkers from a vesicle that is specific to breast cancer includes, but are not limited to, hsp70, MART-1, TRP, HER2, hsp70, MART-1, TRP, HER2, ER, PR, Class III b-tubulin, or VEGFA, or any combination thereof. Furthermore the snoRNA that can be used as an exosomal biomarker for breast cancer include, but are not limited to, GAS5. The gene fusion ETV6-NTRK3 can also be used a biomarker for breast cancer.","The invention also provides an isolated vesicle comprising one or more breast cancer specific biomarkers, such as ETV6-NTRK3, or biomarkers listed in  and in  for breast cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more breast cancer specific biomarkers, such as ETV6-NTRK3, or biomarkers listed in  and in  for breast cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for breast cancer specific vesicles or vesicles comprising one or more breast cancer specific biomarkers, such as ETV6-NTRK3, or biomarkers listed in  and in  for breast cancer.","One or more breast cancer specific biomarkers, such as ETV6-NTRK3, or biomarkers listed in  and in  for breast cancer can also be detected by one or more systems disclosed herein, for characterizing a breast cancer. For example, a detection system can comprise one or more probes to detect one or more breast cancer specific biomarkers, such as ETV6-NTRK3, or biomarkers listed in  and in  for breast cancer, of one or more vesicles of a biological sample.","Ovarian Cancer","Ovarian cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a ovarian cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-200a, miR-141, miR-200c, miR-200b, miR-21, miR-141, miR-200a, miR-200b, miR-200c, miR-203, miR-205, miR-214, miR-199*, or miR-215, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-199a, miR-140, miR-145, miR-100, miR-let-7 cluster, or miR-125b-1, or any combination thereof. The one or more mRNAs that may be analyzed can include without limitation ERCC1, ER, TOPO1, TOP2A, AR, PTEN, HER2\/neu, CD24 or EGFR, or any combination thereof.","A biomarker mutation for ovarian cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of KRAS, mutation of B-Raf, or any combination of mutations specific for ovarian cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, VEGFA, VEGFR2, or HER2, or any combination thereof. Furthermore, a vesicle isolated or assayed can be ovarian cancer cell specific, or derived from ovarian cancer cells.","The invention also provides an isolated vesicle comprising one or more ovarian cancer specific biomarkers, such as CD24, those listed in  and in  for ovarian cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more ovarian cancer specific biomarkers, such as CD24, those listed in  and in  for ovarian cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for ovarian cancer specific vesicles or vesicles comprising one or more ovarian cancer specific biomarkers, such as CD24, those listed in  and in  for ovarian cancer.","One or more ovarian cancer specific biomarkers, such as CD24, those listed in  and in  for ovarian cancer can also be detected by one or more systems disclosed herein, for characterizing an ovarian cancer. For example, a detection system can comprise one or more probes to detect one or more ovarian cancer specific biomarkers, such as CD24, those listed in  and in  for ovarian cancer, of one or more vesicles of a biological sample.","Lung Cancer","Lung cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a lung cancer specific biosignature.","The biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-21, miR-205, miR-221 (protective), let-7a (protective), miR-137 (risky), miR-372 (risky), or miR-122a (risky), or any combination thereof. The biosignature can comprise one or more upregulated or overexpressed miRNAs, such as miR-17-92, miR-19a, miR-21, miR-92, miR-155, miR-191, miR-205 or miR-210; one or more downregulated or underexpressed miRNAs, such as miR-let-7, or any combination thereof. The one or more biomarker may be miR-92a-2*, miR-147, miR-574-5p, such as for small cell lung cancer.","The one or more mRNAs that may be analyzed can include, but are not limited to, EGFR, PTEN, RRM1, RRM2, ABCB1, ABCG2, LRP, VEGFR2, VEGFR3, class III b-tubulin, or any combination thereof.","A biomarker mutation for lung cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of EGFR, KRAS, B-Raf, UGT1A1, or any combination of mutations specific for lung cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, KRAS, hENT1, or any combination thereof.","The biomarker can also be midkine (MK or MDK). Furthermore, a vesicle isolated or assayed can be lung cancer cell specific, or derived from lung cancer cells.","The invention also provides an isolated vesicle comprising one or more lung cancer specific biomarkers, such as RLF-MYCL1, TGF-ALK, or CD74-ROS1, or those listed in  and in  for lung cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more lung cancer specific biomarkers, such as RLF-MYCL1, TGF-ALK, or CD74-ROS1, or those listed in  and in  for lung cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for lung cancer specific vesicles or vesicles comprising one or more lung cancer specific biomarkers, such as RLF-MYCL1, TGF-ALK, or CD74-ROS1, or those listed in  and in  for lung cancer.","One or more lung cancer specific biomarkers, such as RLF-MYCL1, TGF-ALK, or CD74-ROS1, or those listed in  and in  for lung cancer can also be detected by one or more systems disclosed herein, for characterizing a lung cancer. For example, a detection system can comprise one or more probes to detect one or more lung cancer specific biomarkers, such as RLF-MYCL1, TGF-ALK, or CD74-ROS1, or those listed in  and in  for lung cancer, of one or more vesicles of a biological sample.","Colon Cancer","Colon cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a colon cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-24-1, miR-29b-2, miR-20a, miR-10a, miR-32, miR-203, miR-106a, miR-17-5p, miR-30c, miR-223, miR-126, miR-128b, miR-21, miR-24-2, miR-99b, miR-155, miR-213, miR-150, miR-107, miR-191, miR-221, miR-20a, miR-510, miR-92, miR-513, miR-19a, miR-21, miR-20, miR-183, miR-96, miR-135b, miR-31, miR-21, miR-92, miR-222, miR-181b, miR-210, miR-20a, miR-106a, miR-93, miR-335, miR-338, miR-133b, miR-346, miR-106b, miR-153a, miR-219, miR-34a, miR-99b, miR-185, miR-223, miR-211, miR-135a, miR-127, miR-203, miR-212, miR-95, or miR-17-5p, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as miR-143, miR-145, miR-143, miR-126, miR-34b, miR-34c, let-7, miR-9-3, miR-34a, miR-145, miR-455, miR-484, miR-101, miR-145, miR-133b, miR-129, miR-124a, miR-30-3p, miR-328, miR-106a, miR-17-5p, miR-342, miR-192, miR-1, miR-34b, miR-215, miR-192, miR-301, miR-324-5p, miR-30a-3p, miR-34c, miR-331, miR-548c-5p, miR-362-3p, miR-422a, or miR-148b, or any combination thereof.","The one or more biomarker can be an upregulated or overexpressed miRNA, such as miR-20a, miR-21, miR-106a, miR-181b or miR-203, for characterizing a colon adenocarcinoma. The one or more biomarker can be used to characterize a colorectal cancer, such as an upregulated or overexpressed miRNA selected from the group consisting of: miR-19a, miR-21, miR-127, miR-31, miR-96, miR-135b and miR-183, a downregulated or underexpressed miRNA, such as miR-30c, miR-133a, mir143, miR-133b or miR-145, or any combination thereof. The one or more biomarker can be used to characterize a colorectal cancer, such as an upregulated or overexpressed miRNA selected from the group consisting of: miR-548c-5p, miR-362-3p, miR-422a, miR-597, miR-429, miR-200a, and miR-200b, or any combination thereof.","The one or more mRNAs that may be analyzed can include, but are not limited to, EFNB1, ERCC1, HER2, VEGF, or EGFR, or any combination thereof. A biomarker mutation for colon cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of EGFR, KRAS, VEGFA, B-Raf, APC, or p53, or any combination of mutations specific for colon cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, AFRs, Rabs, ADAM10, CD44, NG2, ephrin-B1, MIF, b-catenin, Junction, plakoglobin, glalectin-4, RACK1, tetrspanin-8, FasL, TRAIL, A33, CEA, EGFR, dipeptidase 1, hsc-70, tetraspanins, ESCRT, TS, PTEN, or TOPO1, or any combination thereof. Furthermore, a vesicle isolated or assayed can be colon cancer cell specific, or derived from colon cancer cells.","The invention also provides an isolated vesicle comprising one or more colon cancer specific biomarkers, such as listed in  and in  for colon cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more colon cancer specific biomarkers, such as listed in  and in  for colon cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for colon cancer specific vesicles or vesicles comprising one or more colon cancer specific biomarkers, such as listed in  and in  for colon cancer.","One or more colon cancer specific biomarkers, such as listed in  and in  for colon cancer can also be detected by one or more systems disclosed herein, for characterizing a colon cancer. For example, a detection system can comprise one or more probes to detect one or more colon cancer specific biomarkers, such as listed in  and in  for colon cancer, of one or more vesicles of a biological sample.","Adenoma Versus Hyperplastic Polyp","Adenoma versus hyperplastic polyp specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, or any combination thereof, such as listed in , and can be used to create an adenoma versus hyperplastic polyp specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, ABCA8, KIAA1199, GCG, MAMDC2, C2orf32, 229670_at, IGF1, PCDH7, PRDX6, PCNA, COX2, or MUC6, or any combination thereof.","A biomarker mutation to distinguish for adenoma versus hyperplastic polyp that can be assessed in a vesicle includes, but is not limited to, a mutation of KRAS, mutation of B-Raf, or any combination of mutations specific for distinguishing between adenoma versus hyperplastic polyp. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, hTERT.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between an adenoma and a hyperplastic polyp, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between an adenoma and a hyperplastic polyp, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between an adenoma and a hyperplastic polyp, such as listed in .","One or more specific biomarkers for distinguishing between an adenoma and a hyperplastic polyp, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between an adenoma and a hyperplastic polyp. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between an adenoma and a hyperplastic polyp, such as listed in , of one or more vesicles of a biological sample.","Bladder Cancer","Biomarkers for bladder cancer can be used to assess a bladder cancer according to the methods of the invention. The biomarkers can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof. Biomarkers for bladder cancer include without limitation one or more of miR-223, miR-26b, miR-221, miR-103-1, miR-185, miR-23b, miR-203, miR-17-5p, miR-23a, miR-205 or any combination thereof. Further biomarkers for bladder cancer include FGFR3, EGFR, pRB (retinoblastoma protein), 5T4, p53, Ki-67, VEGF, CK20, COX2, p21, Cyclin D1, p14, p15, p16, Her-2, MAPK (mitogen-activated protein kinase), Bax\/Bcl-2, PI3K (phosphoinositide-3-kinase), CDKs (cyclin-dependent kinases), CD40, TSP-1, HA-ase, telomerase, survivin, NMP22, TNF, Cyclin E1, p27, caspase, survivin, NMP22 (Nuclear matrix protein 22), BCLA-4, Cytokeratins (8, 18, 19 and 20), CYFRA 21-1, IL-2, and complement factor H-related protein. In an embodiment, non-receptor tyrosine kinase ETK\/BMX and\/or Carbonic Anhydrase IX is used as a marker of bladder cancer for diagnostic, prognostic and therapeutic purposes. See Guo et al., Tyrosine Kinase ETK\/BMX Is Up-Regulated in Bladder Cancer and Predicts Poor Prognosis in Patients with Cystectomy. PLoS One. 2011 Mar. 7; 6(3):e17778.; Klatte et al., Carbonic anhydrase IX in bladder cancer: a diagnostic, prognostic, and therapeutic molecular marker. Cancer. 2009 Apr. 1; 115(7):1448-58. The biomarker can be one or more vesicle biomarker associated with bladder cancer as described in Pisitkun et al., Discovery of urinary biomarkers. Mol Cell Proteomics. 2006 October; 5(10):1760-71; Welton et al, Proteomics analysis of bladder cancer exosomes. Mol Cell Proteomics. 2010 June; 9(6):1324-38. These biomarkers can be used for assessing a bladder cancer. The markers can be associated with a vesicle or vesicle population.","Irritable Bowel Disease (IBD)","IBD versus normal biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a IBD versus normal specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, REG1A, MMP3, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between IBD and a normal sample, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between IBD and a normal sample, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between IBD and a normal sample, such as listed in .","One or more specific biomarkers for distinguishing between IBD and a normal sample, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between IBD and a normal sample. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between IBD and a normal sample, such as listed in , of one or more vesicles of a biological sample.","Adenoma Versus Colorectal Cancer (CRC)","Adenoma versus CRC specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a Adenoma versus CRC specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, GREM1, DDR2, GUCY1A3, TNS1, ADAMTS1, FBLN1, FLJ38028, RDX, FAM129A, ASPN, FRMD6, MCC, RBMS1, SNAI2, MEIS1, DOCK10, PLEKHC1, FAM126A, TBC1D9, VWF, DCN, ROBO1, MSRB3, LATS2, MEF2C, IGFBP3, GNB4, RCN3, AKAP12, RFTN1, 226834_at, COL5A1, GNG2, NR3C1*, SPARCL1, MAB21L2, AXIN2, 236894_at, AEBP1, AP1S2, C10orf56, LPHN2, AKT3, FRMD6, COL15A1, CRYAB, COL14A1, LOC286167, QKI, WWTR1, GNG11, PAPPA, or ELDT1, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between an adenoma and a CRC, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between an adenoma and a CRC, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between an adenoma and a CRC, such as listed in .","One or more specific biomarkers for distinguishing between an adenoma and a CRC, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between an adenoma and a CRC. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between an adenoma and a CRC, such as listed in , of one or more vesicles of a biological sample.","IBD Versus CRC","IBD versus CRC specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a IBD versus CRC specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, 227458_at, INDO, CXCL9, CCR2, CD38, RARRES3, CXCL10, FAM26F, TNIP3, NOS2A, CCRL1, TLR8, IL18BP, FCRL5, SAMD9L, ECGF1, TNFSF13B, GBP5, or GBP1, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between IBD and a CRC, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between IBD and a CRC, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between IBD and a CRC, such as listed in .","One or more specific biomarkers for distinguishing between IBD and a CRC, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between IBD and a CRC. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between IBD and a CRC, such as listed in , of one or more vesicles of a biological sample.","CRC Dukes B Versus Dukes C-D","CRC Dukes B versus Dukes C-D specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a CRC D-B versus C-D specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, TMEM37*, IL33, CA4, CCDC58, CLIC6, VERSUSNL1, ESPN, APCDD1, C13orf18, CYP4X1, ATP2A3, LOC646627, MUPCDH, ANPEP, C1orf115, HSD3B2, GBA3, GABRB2, GYLTL1B, LYZ, SPC25, CDKN2B, FAM89A, MOGAT2, SEMA6D, 229376_at, TSPAN5, IL6R, or SLC26A2, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between CRC Dukes B and a CRC Dukes C-D, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between CRC Dukes B and a CRC Dukes C-D, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between CRC Dukes B and a CRC Dukes C-D, such as listed in .","One or more specific biomarkers for distinguishing between CRC Dukes B and a CRC Dukes C-D, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between CRC Dukes B and a CRC Dukes C-D. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between CRC Dukes B and a CRC Dukes C-D, such as listed in , of one or more vesicles of a biological sample.","Adenoma with Low Grade Dysplasia Versus Adenoma with High Grade Dysplasia","Adenoma with low grade dysplasia versus adenoma with high grade dysplasia specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create an adenoma low grade dysplasia versus adenoma high grade dysplasia specific biosignature. For example, the one or mRNAs that may be analyzed can include, but are not limited to, SI, DMBT1, CFI*, AQP1, APOD, TNFRSF17, CXCL10, CTSE, IGHA1, SLC9A3, SLC7A1, BATF2, SOCS1, DOCK2, NOS2A, HK2, CXCL2, IL15RA, POU2AF1, CLEC3B, ANI3BP, MGC13057, LCK*, C4BPA, HOXC6, GOLT1A, C2orf32, IL10RA, 240856_at, SOC3, MEIS3P1, HIPK1, GLS, CPLX1, 236045_x_at, GALC, AMN, CCDC69, CCL28, CPA3, TRIB2, HMGA2, PLCL2, NR3C1, EIF5A, LARP4, RP5-1022P6.2, PHLDB2, FKBP1B, INDO, CLDN8, CNTN3, PBEF1, SLC16A9, CDC25B, TPSB2, PBEF1, ID4, GJB5, CHN2, LIMCH1, or CXCL9, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and adenoma with high grade dysplasia, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and adenoma with high grade dysplasia, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and adenoma with high grade dysplasia, such as listed in .","One or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and adenoma with high grade dysplasia, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between adenoma with low grade dysplasia and adenoma with high grade dysplasia. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and adenoma with high grade dysplasia, such as listed in , of one or more vesicles of a biological sample.","Ulcerative Colitis (UC) Versus Crohn's Disease (CD)","Ulcerative colitis (UC) versus Crohn's disease (CD) specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a UC versus CD specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, IFITM1, IFITM3, STAT1, STAT3, TAP1, PSME2, PSMB8, HNF4G, KLF5, AQP8, APT2B1, SLC16A, MFAP4, CCNG2, SLC44A4, DDAH1, TOB1, 231152_at, MKNK1, CEACAM7*, 1562836_at, CDC42SE2, PSD3, 231169_at, IGL@*, GSN, GPM6B, CDV3*, PDPK1, ANP32E, ADAM9, CDH1, NLRP2, 215777_at, OSBPL1, VNN1, RABGAP1L, PHACTR2, ASH1L, 213710_s_at, CDH1, NLRP2, 215777_at, OSBPL1, VNN1, RABGAP1L, PHACTR2, ASH1, 213710_s_at, ZNF3, FUT2, IGHA1, EDEM1, GPR171, 229713_at, LOC643187, FLVCR1, SNAP23*, ETNK1, LOC728411, POSTN, MUC12, HOXA5, SIGLEC1, LARP5, PIGR, SPTBN1, UFM1, C6orf62, WDR90, ALDH1A3, F2RL1, IGHV1-69, DUOX2, RAB5A, or CP, or any combination thereof can also be used as specific biomarkers from a vesicle for UC versus CD.","A biomarker mutation for distinguishing UC versus CD that can be assessed in a vesicle includes, but is not limited to, a mutation of CARD15, or any combination of mutations specific for distinguishing UC versus CD. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, (P) ASCA.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between UC and CD, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between UC and CD, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between UC and CD, such as listed in .","One or more specific biomarkers for distinguishing between UC and CD, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between UC and CD. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between UC and CD, such as listed in , of one or more vesicles of a biological sample.","Hyperplastic Polyp","Hyperplastic polyp versus normal specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a hyperplastic polyp versus normal specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, SLC6A14, ARHGEF10, ALS2, IL1RN, SPRY4, PTGER3, TRIM29, SERPINB5, 1560327_at, ZAK, BAG4, TRIB3, TTL, FOXQ1, or any combination.","The invention also provides an isolated vesicle comprising one or more hyperplastic polyp specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more hyperplastic polyp specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for hyperplastic polyp specific vesicles or vesicles comprising one or more hyperplastic polyp specific biomarkers, such as listed in .","One or more hyperplastic polyp specific biomarkers, such as listed in  can also be detected by one or more systems disclosed herein, for characterizing a hyperplastic polyp. For example, a detection system can comprise one or more probes to detect one or more listed in . One or more hyperplastic specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Adenoma with Low Grade Dysplasia Versus Normal","Adenoma with low grade dysplasia versus normal specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create an adenoma low grade dysplasia versus normal specific biosignature. For example, the RNAs that may be analyzed can include, but are not limited to, UGT2A3, KLK11, KIAA1199, FOXQ1, CLDN8, ABCA8, or PYY, or any combination thereof and can be used as specific biomarkers from a vesicle for Adenoma low grade dysplasia versus normal. Furthermore, the snoRNA that can be used as an exosomal biomarker for adenoma low grade dysplasia versus normal can include, but is not limited to, GAS5.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and normal, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and normal, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and normal, such as listed in .","One or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and normal, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between adenoma with low grade dysplasia and normal. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between adenoma with low grade dysplasia and normal, such as listed in , of one or more vesicles of a biological sample.","Adenoma Versus Normal","Adenoma versus normal specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create an Adenoma versus normal specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, KIAA1199, FOXQ1, or CA7, or any combination thereof. The protein, ligand, or peptide that can be used as a biomarker from a vesicle that is specific to adenoma versus. normal can include, but is not limited to, Clusterin.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between adenoma and normal, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between adenoma and normal, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between adenoma and normal, such as listed in .","One or more specific biomarkers for distinguishing between adenoma and normal, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between adenoma and normal. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between adenoma and normal, such as listed in , of one or more vesicles of a biological sample.","CRC Versus Normal","CRC versus normal specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a CRC versus normal specific biosignature. For example, the one or mRNAs that may be analyzed can include, but are not limited to, VWF, IL8, CHI3L1, S100A8, GREM1, or ODC, or any combination thereof and can be used as specific biomarkers from a vesicle for CRC versus normal.","A biomarker mutation for CRC versus normal that can be assessed in a vesicle includes, but is not limited to, a mutation of KRAS, BRAF, APC, MSH2, or MLH1, or any combination of mutations specific for distinguishing between CRC versus normal. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, cytokeratin 13, calcineurin, CHK1, clathrin light chain, phospho-ERK, phospho-PTK2, or MDM2, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more specific biomarkers for distinguishing between CRC and normal, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more specific biomarkers for distinguishing between CRC and normal, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for having one or more specific biomarkers for distinguishing between CRC and normal, such as listed in .","One or more specific biomarkers for distinguishing between CRC and normal, such as listed in  can also be detected by one or more systems disclosed herein, for distinguishing between CRC and normal. For example, a detection system can comprise one or more probes to detect one or more specific biomarkers for distinguishing between CRC and normal, such as listed in , of one or more vesicles of a biological sample.","Benign Prostatic Hyperplasia (BPH)","Benign prostatic hyperplasia (BPH) specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a BPH specific biosignature. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, intact fibronectin.","The invention also provides an isolated vesicle comprising one or more BPH specific biomarkers, such as listed in  and in  for BPH. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more BPH specific biomarkers, such as listed in  and in  for BPH. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for BPH specific vesicles or vesicles comprising one or more BPH specific biomarkers, such as listed in  and in  for BPH.","One or more BPH specific biomarkers, such as listed in  and in  for BPH, can also be detected by one or more systems disclosed herein, for characterizing a BPH. For example, a detection system can comprise one or more probes to detect one or more BPH specific biomarkers, such as listed in  and in  for BPH, of one or more vesicles of a biological sample.","Prostate Cancer","Prostate cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a prostate cancer specific biosignature. For example, a biosignature for prostate cancer can comprise miR-9, miR-21, miR-141, miR-370, miR-200b, miR-210, miR-155, or miR-196a. In some embodiments, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-202, miR-210, miR-296, miR-320, miR-370, miR-373, miR-498, miR-503, miR-184, miR-198, miR-302c, miR-345, miR-491, miR-513, miR-32, miR-182, miR-31, miR-26a-1\/2, miR-200c, miR-375, miR-196a-1\/2, miR-370, miR-425, miR-425, miR-194-1\/2, miR-181a-1\/2, miR-34b, let-7i, miR-188, miR-25, miR-106b, miR-449, miR-99b, miR-93, miR-92-1\/2, miR-125a, or miR-141, or any combination thereof.","The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, let-7a, let-7b, let-7c, let-7d, let-7g, miR-16, miR-23a, miR-23b, miR-26a, miR-92, miR-99a, miR-103, miR-125a, miR-125b, miR-143, miR-145, miR-195, miR-199, miR-221, miR-222, miR-497, let-7f, miR-19b, miR-22, miR-26b, miR-27a, miR-27b, miR-29a, miR-29b, miR-305p, miR-30c, miR-100, miR-141, miR-148a, miR-205, miR-520h, miR-494, miR-490, miR-133a-1, miR-1-2, miR-218-2, miR-220, miR-128a, miR-221, miR-499, miR-329, miR-340, miR-345, miR-410, miR-126, miR-205, miR-7-1\/2, miR-145, miR-34a, miR-487, or let-7b, or any combination thereof. The biosignature can comprise upregulated or overexpressed miR-21, downregulated or underexpressed miR-15a, miR-16-1, miR-143 or miR-145, or any combination there.","The one or more mRNAs that may be analyzed can include, but are not limited to, AR, PCA3, or any combination thereof and can be used as specific biomarkers from a vesicle for prostate cancer.","The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, FASLG or TNFSF10 or any combination thereof. Furthermore, a vesicle isolated or assayed can be prostate cancer cell specific, or derived from prostate cancer cells. Furthermore, the snoRNA that can be used as an exosomal biomarker for prostate cancer can include, but is not limited to, U50. Examples of prostate cancer biosignatures are further described below.","The invention also provides an isolated vesicle comprising one or more prostate cancer specific biomarkers, such as ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1, TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4, or those listed in  and in  for prostate cancer. In some embodiments, the isolated vesicle is EpCam+, CK+, CD45\u2212. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more prostate cancer specific biomarkers such as ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1, TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4, or those listed in  and in  for prostate cancer. In some embodiments, the composition comprises a population of vesicles that are EpCam+, CK+, CD45\u2212. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for prostate cancer specific vesicles or vesicles comprising one or more prostate cancer specific biomarkers, such as ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1, TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4, or those listed in  and in  for prostate cancer. In one embodiment, the composition can comprise a substantially enriched population of vesicles that are EpCam+, CK+, CD45\u2212.","One or more prostate cancer specific biomarkers, such as ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1, TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4, or those listed in  and in  for prostate cancer can also be detected by one or more systems disclosed herein, for characterizing a prostate cancer. In some embodiments, the biomarkers EpCam, CK (cytokeratin), and CD45 are detected by one or more of systems disclosed herein, for characterizing prostate cancer, such as determining the prognosis for a subject's prostate cancer, or the therapy-resistance of a subject. For example, a detection system can comprise one or more probes to detect one or more prostate cancer specific biomarkers, such as ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1, TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4, or those listed in  and in  for prostate cancer, of one or more vesicles of a biological sample. In one embodiment, the detection system can comprise one or more probes to detect EpCam, CK, CD45, or a combination thereof.","Melanoma","Melanoma specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a melanoma specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-19a, miR-144, miR-200c, miR-211, miR-324-5p, miR-331, or miR-374, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-9, miR-15a, miR-17-3p, miR-23b, miR-27a, miR-28, miR-29b, miR-30b, miR-31, miR-34b, miR-34c, miR-95, miR-96, miR-100, miR-104, miR-105, miR-106a, miR-107, miR-122a, miR-124a, miR-125b, miR-127, miR-128a, miR-128b, miR-129, miR-135a, miR-135b, miR-137, miR-138, miR-139, miR-140, miR-141, miR-149, miR-154, miR-154#3, miR-181a, miR-182, miR-183, miR-184, miR-185, miR-189, miR-190, miR-199, miR-199b, miR-200a, miR-200b, miR-204, miR-213, miR-215, miR-216, miR-219, miR-222, miR-224, miR-299, miR-302a, miR-302b, miR-302c, miR-302d, miR-323, miR-325, let-7a, let-7b, let-7d, let-7e, or let-7g, or any combination thereof.","The one or more mRNAs that may be analyzed can include, but are not limited to, MUM-1, beta-catenin, or Nop\/5\/Sik, or any combination thereof and can be used as specific biomarkers from a vesicle for melanoma.","A biomarker mutation for melanoma that can be assessed in a vesicle includes, but is not limited to, a mutation of CDK4 or any combination of mutations specific for melanoma. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, DUSP-1, Alix, hsp70, Gib2, Gia, moesin, GAPDH, malate dehydrogenase, p120 catenin, PGRL, syntaxin-binding protein 1 & 2, septin-2, or WD-repeat containing protein 1, or any combination thereof. The snoRNA that can be used as an exosomal biomarker for melanoma include, but are not limited to, H\/ACA (U107f), SNORA11D, or any combination thereof. Furthermore, a vesicle isolated or assayed can be melanoma cell specific, or derived from melanoma cells.","The invention also provides an isolated vesicle comprising one or more melanoma specific biomarkers, such as listed in  and in  for melanoma. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more melanoma specific biomarkers, such as listed in  and in  for melanoma. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for melanoma specific vesicles or vesicles comprising one or more melanoma specific biomarkers, such as listed in  and in  for melanoma.","One or more melanoma specific biomarkers, such as listed in  and in  for melanoma can also be detected by one or more systems disclosed herein, for characterizing a melanoma. For example, a detection system can comprise one or more probes to detect one or more cancer specific biomarkers, such as listed in  and in  for melanoma, of one or more vesicles of a biological sample.","Pancreatic Cancer","Pancreatic cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a pancreatic cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-221, miR-181a, miR-155, miR-210, miR-213, miR-181b, miR-222, miR-181b-2, miR-21, miR-181b-1, miR-220, miR-181d, miR-223, miR-100-1\/2, miR-125a, miR-143, miR-10a, miR-146, miR-99, miR-100, miR-199a-1, miR-10b, miR-199a-2, miR-221, miR-181a, miR-155, miR-210, miR-213, miR-181b, miR-222, miR-181b-2, miR-21, miR-181b-1, miR-181c, miR-220, miR-181d, miR-223, miR-100-1\/2, miR-125a, miR-143, miR-10a, miR-146, miR-99, miR-100, miR-199a-1, miR-10b, miR-199a-2, miR-107, miR-103, miR-103-2, miR-125b-1, miR-205, miR-23a, miR-221, miR-424, miR-301, miR-100, miR-376a, miR-125b-1, miR-21, miR-16-1, miR-181a, miR-181c, miR-92, miR-15, miR-155, let-7f-1, miR-212, miR-107, miR-024-1\/2, miR-18a, miR-31, miR-93, miR-224, or let-7d, or any combination thereof.","The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-148a, miR-148b, miR-375, miR-345, miR-142, miR-133a, miR-216, miR-217 or miR-139, or any combination thereof. The one or more mRNAs that may be analyzed can include, but are not limited to, PSCA, Mesothelin, or Osteopontin, or any combination thereof and can be used as specific biomarkers from a vesicle for pancreatic cancer.","A biomarker mutation for pancreatic cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of KRAS, CTNNLB1, AKT, NCOA3, or B-RAF, or any combination of mutations specific for pancreatic cancer. The biomarker can also be BRCA2, PALB2, or p16. Furthermore, a vesicle isolated or assayed can be pancreatic cancer cell specific, or derived from pancreatic cancer cells.","The invention also provides an isolated vesicle comprising one or more pancreatic cancer specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more pancreatic cancer specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for pancreatic cancer specific vesicles or vesicles comprising one or more pancreatic cancer specific biomarkers, such as listed in .","One or more pancreatic cancer specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a pancreatic cancer. For example, a detection system can comprise one or more probes to detect one or more pancreatic cancer specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Brain Cancer","Brain cancer (including, but not limited to, gliomas, glioblastomas, meinigiomas, acoustic neuroma\/schwannomas, medulloblastoma) specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a brain cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to miR-21, miR-10b, miR-130a, miR-221, miR-125b-1, miR-125b-2, miR-9-2, miR-21, miR-25, or miR-123, or any combination thereof.","The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-128a, miR-181c, miR-181a, or miR-181b, or any combination thereof. The one or more mRNAs that may be analyzed include, but are not limited to, MGMT, which can be used as specific biomarker from a vesicle for brain cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, EGFR.","The invention also provides an isolated vesicle comprising one or more brain cancer specific biomarkers, such as GOPC-ROS1, or those listed in  and in  for brain cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more brain cancer specific biomarkers, such as GOPC-ROS1, or those listed in  and in  for brain cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for brain cancer specific vesicles or vesicles comprising one or more brain cancer specific biomarkers, such as GOPC-ROS1, or those listed in  and in  for brain cancer.","One or more brain cancer specific biomarkers, such as listed in  and in  for brain cancer, can also be detected by one or more systems disclosed herein, for characterizing a brain cancer. For example, a detection system can comprise one or more probes to detect one or more brain cancer specific biomarkers, such as GOPC-ROS1, or those listed in  and in  for brain cancer, of one or more vesicles of a biological sample.","Psoriasis","Psoriasis specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a psoriasis specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-146b, miR-20a, miR-146a, miR-31, miR-200a, miR-17-5p, miR-30e-5p, miR-141, miR-203, miR-142-3p, miR-21, or miR-106a, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such a, but not limited to, miR-125b, miR-99b, miR-122a, miR-197, miR-100, miR-381, miR-518b, miR-524, let-7e, miR-30c, miR-365, miR-133b, miR-10a, miR-133a, miR-22, miR-326, or miR-215, or any combination thereof.","The oneor more mRNAs that may be analyzed can include, but are not limited to, IL-20, VEGFR-1, VEGFR-2, VEGFR-3, or EGR1, or any combination thereof and can be used as specific biomarkers from a vesicle for psoriasis. A biomarker mutation for psoriasis that can be assessed in a vesicle includes, but is not limited to, a mutation of MGST2, or any combination of mutations specific for psoriasis.","The invention also provides an isolated vesicle comprising one or more psoriasis specific biomarkers, such as listed in  and in  for psoriasis. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more psoriasis specific biomarkers, such as listed in  and in  for psoriasis. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for psoriasis specific vesicles or vesicles comprising one or more psoriasis specific biomarkers, such as listed in  and in  for psoriasis.","One or more psoriasis specific biomarkers, such as listed in  and in  for psoriasis, can also be detected by one or more systems disclosed herein, for characterizing psoriasis. For example, a detection system can comprise one or more probes to detect one or more psoriasis specific biomarkers, such as listed in  and in  for psoriasis, of one or more vesicles of a biological sample.","Cardiovascular Disease (CVD)","CVD specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a CVD specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-195, miR-208, miR-214, let-7b, let-7c, let-7e, miR-15b, miR-23a, miR-24, miR-27a, miR-27b, miR-93, miR-99b, miR-100, miR-103, miR-125b, miR-140, miR-145, miR-181a, miR-191, miR-195, miR-199a, miR-320, miR-342, miR-451, or miR-499, or any combination thereof.","The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-1, miR-10a, miR-17-5p, miR-19a, miR-19b, miR-20a, miR-20b, miR-26b, miR-28, miR-30e-5p, miR-101, miR-106a, miR-126, miR-222, miR-374, miR-422b, or miR-423, or any combination thereof. The mRNAs that may be analyzed can include, but are not limited to, MRP14, CD69, or any combination thereof and can be used as specific biomarkers from a vesicle for CVD.","A biomarker mutation for CVD that can be assessed in a vesicle includes, but is not limited to, a mutation of MYH7, SCN5A, or CHRM2, or any combination of mutations specific for CVD.","The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, CK-MB, cTnI (cardiac troponin), CRP, BPN, IL-6, MCSF, CD40, CD40L, or any combination thereof. Furthermore, a vesicle isolated or assayed can be a CVD cell specific, or derived from cardiac cells.","The invention also provides an isolated vesicle comprising one or more CVD specific biomarkers, such as listed in  and in  for CVD. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more CVD specific biomarkers, such as listed in  and in  for CVD. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for CVD specific vesicles or vesicles comprising one or more CVD specific biomarkers, such as listed in  and in  for CVD.","One or more CVD specific biomarkers, such as listed in  and in  for CVD, can also be detected by one or more systems disclosed herein, for characterizing a CVD. For example, a detection system can comprise one or more probes to detect one or more CVD specific biomarkers, such as listed in  and in  for CVD, of one or more vesicles of a biological sample.","An increase in an miRNA or combination or miRNA, such as miR-21, miR-129, miR-212, miR-214, miR-134, or a combination thereof (as disclosed in US Publication No. 2010\/0010073), can be used to diagnose an increased risk of development or already the existence of cardiac hypertrophy and\/or heart failure. A downregulation of miR-182, miR-290, or a combination thereof can be used to diagnose an increased risk of development or already the existence of cardiac hypertrophy and\/or heart failure. An increased expression of miR-21, miR-129, miR-212, miR-214, miR-134, or a combination thereof with a reduced expression of miR-182, miR-290, or a combination thereof, may be used to diagnose an increased risk of development or the existence of cardiac hypertrophy and\/or heart failure.","Blood Cancers","Hematological malignancies specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a hematological malignancies specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, HOX11, TAL1, LY1, LMO1, or LMO2, or any combination thereof and can be used as specific biomarkers from a vesicle for hematological malignancies.","A biomarker mutation for a blood cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of c-kit, PDGFR, or ABL, or any combination of mutations specific for hematological malignancies.","The invention also provides an isolated vesicle comprising one or more blood cancer specific biomarkers, such as listed in  and in  for blood cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more blood cancer specific biomarkers, such as listed in  and in  for blood cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for blood cancer specific vesicles or vesicles comprising one or more blood cancer specific biomarkers, such as listed in  and in  for blood cancer.","One or more blood cancer specific biomarkers, such as listed in  and in  for blood cancer, can also be detected by one or more systems disclosed herein, for characterizing a blood cancer. For example, a detection system can comprise one or more probes to detect one or more blood cancer specific biomarkers, such as listed in  and in  for blood cancer, of one or more vesicles of a biological sample.","The one or more blood cancer specific biomarkers can also be a gene fusion selected from the group consisting of: TTL-ETV6, CDK6-MLL, CDK6-TLX3, ETV6-FLT3, ETV6-RUNX1, ETV6-TTL, MLL-AFF1, MLL-AFF3, MLL-AFF4, MLL-GAS7, TCBA1-ETV6, TCF3-PBX1 or TCF3-TFPT, for acute lymphocytic leukemia (ALL); BCL11B-TLX3, IL2-TNFRFS17, NUP214-ABL1, NUP98-CCDC28A, TAL1-STIL, or ETV6-ABL2, for T-cell acute lymphocytic leukemia (T-ALL); ATIC-ALK, KIAA1618-ALK, MSN-ALK, MYH9-ALK, NPM1-ALK, TGF-ALK or TPM3-ALK, for anaplastic large cell lymphoma (ALCL); BCR-ABL1, BCR-JAK2, ETV6-EVI1, ETV6-MN1 or ETV6-TCBA1, for chronic myelogenous leukemia (CML); CBFB-MYH11, CHIC2-ETV6, ETV6-ABL1, ETV6-ABL2, ETV6-ARNT, ETV6-CDX2, ETV6-HLXB9, ETV6-PER1, MEF2D-DAZAP1, AML-AFF1, MLL-ARHGAP26, MLL-ARHGEF12, MLL-CASC5, MLL-CBL, MLL-CREBBP, MLL-DAB21P, MLL-ELL, MLL-EP300, MLL-EPS15, MLL-FNBP1, MLL-FOXO3A, MLL-GMPS, MLL-GPHN, MLL-MLLT1, MLL-MLLT11, MLL-MLLT3, MLL-MLLT6, MLL-MYO1F, MLL-PICALM, MLL-SEPT2, MLL-SEPT6, MLL-SORBS2, MYST3-SORBS2, MYST-CREBBP, NPM1-MLF1, NUP98-HOXA13, PRDM16-EVI1, RABEP1-PDGFRB, RUNX1-EVI1, RUNX1-MDS1, RUNX1-RPL22, RUNX1-RUNX1T1, RUNX1-SH3D19, RUNX1-USP42, RUNX1-YTHDF2, RUNX1-ZNF687, or TAF15-ZNF-384, for AML; CCND1-FSTL3, for chronic lymphocytic leukemia (CLL); and FLIP1-PDGFRA, FLT3-ETV6, KIAA1509-PDGFRA, PDE4DIP-PDGFRB, NIN-PDGFRB, TP53BP1-PDGFRB, or TPM3-PDGFRB, for hyper eosinophilia\/chronic eosinophilia.","The one or more biomarkers for CLL can also include one or more of the following upregulated or overexpressed miRNAs, such as miR-23b, miR-24-1, miR-146, miR-155, miR-195, miR-221, miR-331, miR-29a, miR-195, miR-34a, or miR-29c; one or more of the following downregulated or underexpressed miRs, such as miR-15a, miR-16-1, miR-29 or miR-223, or any combination thereof.","The one or more biomarkers for ALL can also include one or more of the following upregulated or overexpressed miRNAs, such as miR-128b, miR-204, miR-218, miR-331, miR-181b-1, miR-17-92; or any combination thereof.","B-Cell Chronic Lymphocytic Leukemia (B-CLL)","B-CLL specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a B-CLL specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-183-prec, miR-190, miR-24-1-prec, miR-33, miR-19a, miR-140, miR-123, miR-10b, miR-15b-prec, miR-92-1, miR-188, miR-154, miR-217, miR-101, miR-141-prec, miR-153-prec, miR-196-2, miR-134, miR-141, miR-132, miR-192, or miR-181b-prec, or any combination thereof.","The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-213, miR-220, or any combination thereof. The one or more mRNAs that may be analyzed can include, but are not limited to, ZAP70, AdipoR1, or any combination thereof and can be used as specific biomarkers from a vesicle for B-CLL. A biomarker mutation for B-CLL that can be assessed in a vesicle includes, but is not limited to, a mutation of IGHV, P53, ATM, or any combination of mutations specific for B-CLL.","The invention also provides an isolated vesicle comprising one or more B-CLL specific biomarkers, such as BCL3-MYC, MYC-BTG1, BCL7A-MYC, BRWD3-ARHGAP20 or BTG1-MYC, or those listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more B-CLL specific biomarkers, such as BCL3-MYC, MYC-BTG1, BCL7A-MYC, BRWD3-ARHGAP20 or BTG1-MYC, or those listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for B-CLL specific vesicles or vesicles comprising one or more B-CLL specific biomarkers, such as BCL3-MYC, MYC-BTG1, BCL7A-MYC, BRWD3-ARHGAP20 or BTG1-MYC, or those listed in .","One or more B-CLL specific biomarkers, such as BCL3-MYC, MYC-BTG1, BCL7A-MYC, BRWD3-ARHGAP20 or BTG1-MYC, or those listed in , can also be detected by one or more systems disclosed herein, for characterizing a B-CLL. For example, a detection system can comprise one or more probes to detect one or more B-CLL specific biomarkers, such as BCL3-MYC, MYC-BTG1, BCL7A-MYC, BRWD3-ARHGAP20 or BTG1-MYC, or those listed in , of one or more vesicles of a biological sample.","B-Cell Lymphoma","B-cell lymphome specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a B-cell lymphoma specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-17-92 polycistron, miR-155, miR-210, or miR-21, miR-19a, miR-92, miR-142 miR-155, miR-221 miR-17-92, miR-21, miR-191, miR-205, or any combination thereof. Furthermore the snoRNA that can be used as an exosomal biomarker for B-cell lymphoma can include, but is not limited to, U50.","The invention also provides an isolated vesicle comprising one or more B-cell lymphoma specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more B-cell lymphoma specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for B-cell lymphoma specific vesicles or vesicles comprising one or more B-cell lymphoma specific biomarkers, such as listed in .","One or more B-cell lymphoma specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a B-cell lymphoma. For example, a detection system can comprise one or more probes to detect one or more B-cell lymphoma specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Diffuse Large B-Cell Lymphoma (DLBCL)","DLBCL specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a DLBCL specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-17-92, miR-155, miR-210, or miR-21, or any combination thereof. The one or more mRNAs that may be analyzed can include, but are not limited to, A-myb, LMO2, JNK3, CD10, bcl-6, Cyclin D2, IRF4, Flip, or CD44, or any combination thereof and can be used as specific biomarkers from a vesicle for DLBCL.","The invention also provides an isolated vesicle comprising one or more DLBCL specific biomarkers, such as CITTA-BCL6, CLTC-ALK, IL21R-BCL6, PIM1-BCL6, TFCR-BCL6, IKZF1-BCL6 or SEC31A-ALK, or those listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more DLBCL specific biomarkers, such as CITTA-BCL6, CLTC-ALK, IL21R-BCL6, PIM1-BCL6, TFCR-BCL6, IKZF1-BCL6 or SEC31A-ALK, or those listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for DLBCL specific vesicles or vesicles comprising one or more DLBCL specific biomarkers, such as CITTA-BCL6, CLTC-ALK, IL21R-BCL6, PIM1-BCL6, TFCR-BCL6, IKZF1-BCL6 or SEC31A-ALK, or those listed in .","One or more DLBCL specific biomarkers, such as CITTA-BCL6, CLTC-ALK, IL21R-BCL6, PIM1-BCL6, TFCR-BCL6, IKZF1-BCL6 or SEC31A-ALK, or those listed in , can also be detected by one or more systems disclosed herein, for characterizing a DLBCL. For example, a detection system can comprise one or more probes to detect one or more DLBCL specific biomarkers, such as CITTA-BCL6, CLTC-ALK, IL21R-BCL6, PIM1-BCL6, TFCR-BCL6, IKZF1-BCL6 or SEC31A-ALK, or those listed in , of one or more vesicles of a biological sample.","Burkitt's Lymphoma","Burkitt's lymphoma specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a Burkitt's lymphoma specific biosignature. For example, the biosignature can also comprise one or more underexpressed miRs such as, but not limited to, pri-miR-155, or any combination thereof. The one or more mRNAs that may be analyzed can include, but are not limited to, MYC, TERT, NS, NP, MAZ, RCF3, BYSL, IDE3, CDC7, TCL1A, AUTS2, MYBL1, BMP7, ITPR3, CDC2, BACK2, TTK, MME, ALOX5, or TOP1, or any combination thereof and can be used as specific biomarkers from a vesicle for Burkitt's lymphoma. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, BCL6, KI-67, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more Burkitt's lymphoma specific biomarkers, such as IGH-MYC, LCP1-BCL6, or those listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more Burkitt's lymphoma specific biomarkers, such as IGH-MYC, LCP1-BCL6, or those listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for Burkitt's lymphoma specific vesicles or vesicles comprising one or more Burkitt's lymphoma specific biomarkers, such as IGH-MYC, LCP1-BCL6, or those listed in .","One or more Burkitt's lymphoma specific biomarkers, such as IGH-MYC, LCP1-BCL6, or those listed in , can also be detected by one or more systems disclosed herein, for characterizing a Burkitt's lymphoma. For example, a detection system can comprise one or more probes to detect one or more Burkitt's lymphoma specific biomarkers, such as IGH-MYC, LCP1-BCL6, or those listed in , of one or more vesicles of a biological sample.","Hepatocellular Carcinoma","Hepatocellular carcinoma specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in  and can be used to create a hepatocellular carcinoma specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-221. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, let-7a-1, let-7a-2, let-7a-3, let-7b, let-7c, let-7d, let-7e, let-7f-2, let-fg, miR-122a, miR-124a-2, miR-130a, miR-132, miR-136, miR-141, miR-142, miR-143, miR-145, miR-146, miR-150, miR-155(BIC), miR-181a-1, miR-181a-2, miR-181c, miR-195, miR-199a-1-5p, miR-199a-2-5p, miR-199b, miR-200b, miR-214, miR-223, or pre-miR-594, or any combination thereof. The one or more mRNAs that may be analyzed can include, but are not limited to, FAT10.","The one or more biomarkers of a biosignature can also be used to characterize hepatitis C virus-associated hepatocellular carcinoma. The one or more biomarkers can be a miRNA, such as an overexpressed or underexpressed miRNA. For example, the upregulated or overexpressed miRNA can be miR-122, miR-100, or miR-10a and the downregulated miRNA can be miR-198 or miR-145.","The invention also provides an isolated vesicle comprising one or more hepatocellular carcinoma specific biomarkers, such as listed in  and in  for hepatocellular carcinoma. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more hepatocellular carcinoma specific biomarkers, such as listed in  and in  for hepatocellular carcinoma. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for hepatocellular carcinoma specific vesicles or vesicles comprising one or more hepatocellular carcinoma specific biomarkers, such as listed in  and in  for hepatocellular carcinoma.","One or more hepatocellular carcinoma specific biomarkers, such as listed in  and in  for hepatocellular carcinoma, can also be detected by one or more systems disclosed herein, for characterizing a hepatocellular carcinoma. For example, a detection system can comprise one or more probes to detect one or more hepatocellular carcinoma specific biomarkers, such as listed in  and in  for hepatocellular carcinoma, of one or more vesicles of a biological sample.","Cervical Cancer","Cervical cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a cervical cancer specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, HPV E6, HPV E7, or p53, or any combination thereof and can be used as specific biomarkers from a vesicle for cervical cancer.","The invention also provides an isolated vesicle comprising one or more cervical cancer specific biomarkers, such as listed in  and in  for cervical cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more cervical cancer specific biomarkers, such as listed in  and in  for cervical cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for cervical cancer specific vesicles or vesicles comprising one or more cervical cancer specific biomarkers, such as listed in  and in  for cervical cancer.","One or more cervical cancer specific biomarkers, such as listed in  and in  for cervical cancer, can also be detected by one or more systems disclosed herein, for characterizing a cervical cancer. For example, a detection system can comprise one or more probes to detect one or more cervical cancer specific biomarkers, such as listed in  and in  for cervical cancer, of one or more vesicles of a biological sample.","Endometrial Cancer","Endometrial cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in  and can be used to create a endometrial cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-185, miR-106a, miR-181a, miR-210, miR-423, miR-103, miR-107, or let-7c, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-7i, miR-221, miR-193, miR-152, or miR-30c, or any combination thereof.","A biomarker mutation for endometrial cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of PTEN, K-RAS, B-catenin, p53, Her2\/neu, or any combination of mutations specific for endometrial cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, NLRP7, AlphaV Beta6 integrin, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more endometrial cancer specific biomarkers, such as listed in  and in  for endometrial cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more endometrial cancer specific biomarkers, such as listed in  and in  for endometrial cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for endometrial cancer specific vesicles or vesicles comprising one or more endometrial cancer specific biomarkers, such as listed in  and in  for endometrial cancer.","One or more endometrial cancer specific biomarkers, such as listed in  and in  for endometrial cancer, can also be detected by one or more systems disclosed herein, for characterizing a endometrial cancer. For example, a detection system can comprise one or more probes to detect one or more endometrial cancer specific biomarkers, such as listed in  and in  for endometrial cancer, of one or more vesicles of a biological sample.","Head and Neck Cancer","Head and neck cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a head and neck cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-21, let-7, miR-18, miR-29c, miR-142-3p, miR-155, miR-146b, miR-205, or miR-21, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-494. The one or more mRNAs that may be analyzed include, but are not limited to, HPV E6, HPV E7, p53, IL-8, SAT, H3FA3, or EGFR, or any combination thereof and can be used as specific biomarkers from a vesicle for head and neck cancer.","A biomarker mutation for head and neck cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of GSTM1, GSTT1, GSTP1, OGG1, XRCC1, XPD, RAD51, EGFR, p53, or any combination of mutations specific for head and neck cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, EGFR, EphB4, or EphB2, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more head and neck cancer specific biomarkers, such as CHCHD7-PLAG1, CTNNB1-PLAG1, FHIT-HMGA2, HMGA2-NFIB, LIFR-PLAG1, or TCEA1-PLAG1, or those listed in  and in  for head and neck cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more head and neck cancer specific biomarkers, such as CHCHD7-PLAG1, CTNNB1-PLAG1, FHIT-HMGA2, HMGA2-NFIB, LIFR-PLAG1, or TCEA1-PLAG1, or those listed in  and in  for head and neck cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for head and neck cancer specific vesicles or vesicles comprising one or more head and neck cancer specific biomarkers, such as CHCHD7-PLAG1, CTNNB1-PLAG1, FHIT-HMGA2, HMGA2-NFIB, LIFR-PLAG1, or TCEA1-PLAG1, or those listed in  and in  for head and neck cancer.","One or more head and neck cancer specific biomarkers, such as listed in  and in  for head and neck cancer, can also be detected by one or more systems disclosed herein, for characterizing a head and neck cancer. For example, a detection system can comprise one or more probes to detect one or more head and neck cancer specific biomarkers, such as CHCHD7-PLAG1, CTNNB1-PLAG1, FHIT-HMGA2, HMGA2-NFIB, LIFR-PLAG1, or TCEA1-PLAG1, or those listed in  and in  for head and neck cancer, of one or more vesicles of a biological sample.","Inflammatory Bowel Disease (IBD)","IBD specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a IBD specific biosignature. The one or more mRNAs that may be analyzed can include, but are not limited to, Trypsinogen IV, SERT, or any combination thereof and can be used as specific biomarkers from a vesicle for IBD.","A biomarker mutation for IBD that can be assessed in a vesicle can include, but is not limited to, a mutation of CARD15 or any combination of mutations specific for IBD. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, II-16, II-1beta, II-12, TNF-alpha, interferon gamma, II-6, Rantes, MCP-1, Resistin, or 5-HT, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more IBD specific biomarkers, such as listed in  and in  for IBD. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more IBD specific biomarkers, such as listed in  and in  for IBD. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for IBD specific vesicles or vesicles comprising one or more IBD specific biomarkers, such as listed in  and in  for IBD.","One or more IBD specific biomarkers, such as listed in  and in  for IBD, can also be detected by one or more systems disclosed herein, for characterizing a IBD. For example, a detection system can comprise one or more probes to detect one or more IBD specific biomarkers, such as listed in  and in  for IBD, of one or more vesicles of a biological sample.","Diabetes","Diabetes specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a diabetes specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, Il-8, CTSS, ITGB2, HLA-DRA, CD53, PLAG27, or MMP9, or any combination thereof and can be used as specific biomarkers from a vesicle for diabetes. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, RBP4.","The invention also provides an isolated vesicle comprising one or more diabetes specific biomarkers, such as listed in  and in  for diabetes. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more diabetes specific biomarkers, such as listed in  and in  for diabetes. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for diabetes specific vesicles or vesicles comprising one or more diabetes specific biomarkers, such as listed in  and in  for diabetes.","One or more diabetes specific biomarkers, such as listed in  and in  for diabetes, can also be detected by one or more systems disclosed herein, for characterizing diabetes. For example, a detection system can comprise one or more probes to detect one or more diabetes specific biomarkers, such as listed in  and in  for diabetes, of one or more vesicles of a biological sample.","Barrett's Esophagus","Barrett's Esophagus specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a Barrett's Esophagus specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-21, miR-143, miR-145, miR-194, or miR-215, or any combination thereof. The one or more mRNAs that may be analyzed include, but are not limited to, S100A2, S100A4, or any combination thereof and can be used as specific biomarkers from a vesicle for Barrett's Esophagus.","A biomarker mutation for Barrett's Esophagus that can be assessed in a vesicle includes, but is not limited to, a mutation of p53 or any combination of mutations specific for Barrett's Esophagus. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, p53, MUC1, MUC2, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more Barrett's Esophagus specific biomarkers, such as listed in  and in  for Barrett's Esophagus. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more Barrett's Esophagus specific biomarkers, such as listed in  and in  for Barrett's Esophagus. The composition can comprise a substantially enriched population of vesicles,","wherein the population of vesicles is substantially homogeneous for Barrett's Esophagus specific vesicles or vesicles comprising one or more Barrett's Esophagus specific biomarkers, such as listed in  and in  for Barrett's Esophagus.","One or more Barrett's Esophagus specific biomarkers, such as listed in  and in  for Barrett's Esophagus, can also be detected by one or more systems disclosed herein, for characterizing a Barrett's Esophagus. For example, a detection system can comprise one or more probes to detect one or more Barrett's Esophagus specific biomarkers, such as listed in  and in  for Barrett's Esophagus, of one or more vesicles of a biological sample.","Fibromyalgia","Fibromyalgia specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a fibromyalgia specific biosignature. The one or more mRNAs that may be analyzed can include, but are not limited to, NR2D which can be used as a specific biomarker from a vesicle for fibromyalgia.","The invention also provides an isolated vesicle comprising one or more fibromyalgia specific biomarkers, such as listed in  and in  for fibromyalgia. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more fibromyalgia specific biomarkers, such as listed in  and in  for fibromyalgia. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for fibromyalgia specific vesicles or vesicles comprising one or more fibromyalgia specific biomarkers, such as listed in  and in  for fibromyalgia.","One or more fibromyalgia specific biomarkers, such as listed in  and in  for fibromyalgia, can also be detected by one or more systems disclosed herein, for characterizing a fibromyalgia. For example, a detection system can comprise one or more probes to detect one or more fibromyalgia specific biomarkers, such as listed in  and in  for fibromyalgia, of one or more vesicles of a biological sample.","Stroke","Stroke specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a stroke specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, MMP9, S100-P, S100A12, S100A9, coag factor V, ArginaseI, CA-IV, monocarboxylic acid transporter, ets-2, EIF2alpha, cytoskeleton associated protein 4, N-formylpeptide receptor, Ribonuclease2, N-acetylneuraminate pyruvate lyase, BCL-6, or Glycogen phosphorylase, or any combination thereof and can be used as specific biomarkers from a vesicle for stroke.","The invention also provides an isolated vesicle comprising one or more stroke specific biomarkers, such as listed in  and in  for stroke. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more stroke specific biomarkers, such as listed in  and in  for stroke. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for stroke specific vesicles or vesicles comprising one or more stroke specific biomarkers, such as listed in  and in  for stroke.","One or more stroke specific biomarkers, such as listed in  and in  for stroke, can also be detected by one or more systems disclosed herein, for characterizing a stroke. For example, a detection system can comprise one or more probes to detect one or more stroke specific biomarkers, such as listed in  and in  for stroke, of one or more vesicles of a biological sample.","Multiple Sclerosis (MS)","MS specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a MS specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, IL-6, IL-17, PAR-3, IL-17, T1\/ST2, JunD, 5-LO, LTA4H, MBP, PLP, or alpha-beta crystallin, or any combination thereof and can be used as specific biomarkers from a vesicle for MS.","The invention also provides an isolated vesicle comprising one or more MS specific biomarkers, such as listed in  and in  for MS. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more MS specific biomarkers, such as listed in  and in  for MS. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for MS specific vesicles or vesicles comprising one or more MS specific biomarkers, such as listed in  and in  for MS.","One or more MS specific biomarkers, such as listed in  and in  for MS, can also be detected by one or more systems disclosed herein, for characterizing a MS. For example, a detection system can comprise one or more probes to detect one or more MS specific biomarkers, such as listed in  and in  for MS, of one or more vesicles of a biological sample.","Parkinson's Disease","Parkinson's disease specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a Parkinson's disease specific biosignature. For example, the biosignature can include, but is not limited to, one or more underexpressed miRs such as miR-133b. The one or more mRNAs that may be analyzed can include, but are not limited to Nurr1, BDNF, TrkB, gstm1, or S100 beta, or any combination thereof and can be used as specific biomarkers from a vesicle for Parkinson's disease.","A biomarker mutation for Parkinson's disease that can be assessed in a vesicle includes, but is not limited to, a mutation of FGF20, alpha-synuclein, FGF20, NDUFV2, FGF2, CALB1, B2M, or any combination of mutations specific for Parkinson's disease. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, apo-H, Ceruloplasmin, BDNF, IL-8, Beta2-microglobulin, apoAII, tau, ABeta1-42, DJ-1, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more Parkinson's disease specific biomarkers, such as listed in  and in  for Parkinson's disease A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more Parkinson's disease specific biomarkers, such as listed in  and in  for Parkinson's disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for Parkinson's disease specific vesicles or vesicles comprising one or more Parkinson's disease specific biomarkers, such as listed in  and in  for Parkinson's disease.","One or more Parkinson's disease specific biomarkers, such as listed in  and in  for Parkinson's disease, can also be detected by one or more systems disclosed herein, for characterizing a Parkinson's disease. For example, a detection system can comprise one or more probes to detect one or more Parkinson's disease specific biomarkers, such as listed in  and in  for Parkinson's disease, of one or more vesicles of a biological sample.","Rheumatic Disease","Rheumatic disease specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a rheumatic disease specific biosignature. For example, the biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-146a, miR-155, miR-132, miR-16, or miR-181, or any combination thereof. The one or more mRNAs that may be analyzed can include, but are not limited to, HOXD10, HOXD11, HOXD13, CCL8, LIM homeobox2, or CENP-E, or any combination thereof and can be used as specific biomarkers from a vesicle for rheumatic disease. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, TNF\u03b1.","The invention also provides an isolated vesicle comprising one or more rheumatic disease specific biomarkers, such as listed in  and in  for rheumatic disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more rheumatic disease specific biomarkers, such as listed in  and in  for rheumatic disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for rheumatic disease specific vesicles or vesicles comprising one or more rheumatic disease specific biomarkers, such as listed in  and in  for rheumatic disease.","One or more rheumatic disease specific biomarkers, such as listed in  and in  for rheumatic disease, can also be detected by one or more systems disclosed herein, for characterizing a rheumatic disease. For example, a detection system can comprise one or more probes to detect one or more rheumatic disease specific biomarkers, such as listed in  and in  for rheumatic disease, of one or more vesicles of a biological sample.","Alzheimer's Disease","Alzheimer's disease specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a Alzheimers disease specific biosignature. For example, the biosignature can also comprise one or more underexpressed miRs such as miR-107, miR-29a, miR-29b-1, or miR-9, or any combination thereof. The biosignature can also comprise one or more overexpressed miRs such as miR-128 or any combination thereof.","The one or more mRNAs that may be analyzed can include, but are not limited to, HIF-1\u03b1, BACE1, Reelin, CHRNA7, or 3Rtau\/4Rtau, or any combination thereof and can be used as specific biomarkers from a vesicle for Alzheimer's disease.","A biomarker mutation for Alzheimer's disease that can be assessed in a vesicle includes, but is not limited to, a mutation of APP, presenilin1, presenilin2, APOE4, or any combination of mutations specific for Alzheimer's disease. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, BACE1, Reelin, Cystatin C, Truncated Cystatin C, Amyloid Beta, C3a, t-Tau, Complement factor H, or alpha-2-macroglobulin, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more Alzheimer's disease specific biomarkers, such as listed in  and in  for Alzheimer's disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more Alzheimer's disease specific biomarkers, such as listed in  and in  for Alzheimer's disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for Alzheimer's disease specific vesicles or vesicles comprising one or more Alzheimer's disease specific biomarkers, such as listed in  and in  for Alzheimer's disease.","One or more Alzheimer's disease specific biomarkers, such as listed in  and in  for Alzheimer's disease, can also be detected by one or more systems disclosed herein, for characterizing a Alzheimer's disease. For example, a detection system can comprise one or more probes to detect one or more Alzheimer's disease specific biomarkers, such as listed in  and in  for Alzheimer's disease, of one or more vesicles of a biological sample.","Prion Disease","Prion specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a prion specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, Amyloid B4, App, IL-1R1, or SOD1, or any combination thereof and can be used as specific biomarkers from a vesicle for a prion. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, PrP(c), 14-3-3, NSE, S-100, Tau, AQP-4, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more prion disease specific biomarkers, such as listed in  and in  for prion disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more prion disease specific biomarkers, such as listed in  and in  for prion disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for prion disease specific vesicles or vesicles comprising one or more prion disease specific biomarkers, such as listed in  and in  for prion disease.","One or more prion disease specific biomarkers, such as listed in  and in  for prion disease, can also be detected by one or more systems disclosed herein, for characterizing a prion disease. For example, a detection system can comprise one or more probes to detect one or more prion disease specific biomarkers, such as listed in  and in  for prion disease, of one or more vesicles of a biological sample.","Sepsis","Sepsis specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a sepsis specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, 15-Hydroxy-PG dehydrogenase (up), LAIR1 (up), NFKB1A (up), TLR2, PGLYPR1, TLR4, MD2, TLR5, IFNAR2, IRAK2, IRAK3, IRAK4, PI3K, PI3KCB, MAP2K6, MAPK14, NFKB1A, NFKB1, IL1R1, MAP2K1IP1, MKNK1, FAS, CASP4, GADD45B, SOCS3, TNFSF10, TNFSF13B, OSM, HGF, or IL18R1, or any combination thereof and can be used as specific biomarkers from a vesicle for sepsis.","The invention also provides an isolated vesicle comprising one or more sepsis specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more sepsis specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for sepsis specific vesicles or vesicles comprising one or more sepsis specific biomarkers, such as listed in .","One or more sepsis specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a sepsis. For example, a detection system can comprise one or more probes to detect one or more sepsis specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Chronic Neuropathic Pain","Chronic neuropathic pain (CNP) specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a CNP specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, ICAM-1 (rodent), CGRP (rodent), TIMP-1 (rodent), CLR-1 (rodent), HSP-27 (rodent), FABP (rodent), or apolipoprotein D (rodent), or any combination thereof and can be used as specific biomarkers from a vesicle for CNP. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, chemokines, chemokine receptors (CCR2\/4), or any combination thereof.","The invention also provides an isolated vesicle comprising one or more chronic neuropathic pain specific biomarkers, such as listed in  and in  for chronic neuropathic pain. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more chronic neuropathic pain specific biomarkers, such as listed in  and in  for chronic neuropathic pain. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for chronic neuropathic pain specific vesicles or vesicles comprising one or more chronic neuropathic pain specific biomarkers, such as listed in  and in  for chronic neuropathic pain.","One or more chronic neuropathic pain specific biomarkers, such as listed in  and in  for chronic neuropathic pain, can also be detected by one or more systems disclosed herein, for characterizing a chronic neuropathic pain. For example, a detection system can comprise one or more probes to detect one or more chronic neuropathic pain specific biomarkers, such as listed in  and in  for chronic neuropathic pain, of one or more vesicles of a biological sample.","Peripheral Neuropathic Pain","Peripheral neuropathic pain (PNP) specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a PNP specific biosignature. For example, the protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, OX42, ED9, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more peripheral neuropathic pain specific biomarkers, such as listed in  and in  for peripheral neuropathic pain. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more peripheral neuropathic pain specific biomarkers, such as listed in  and in  for peripheral neuropathic pain. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for peripheral neuropathic pain specific vesicles or vesicles comprising one or more peripheral neuropathic pain specific biomarkers, such as listed in  and in  for peripheral neuropathic pain.","One or more peripheral neuropathic pain specific biomarkers, such as listed in  and in  for peripheral neuropathic pain, can also be detected by one or more systems disclosed herein, for characterizing a peripheral neuropathic pain. For example, a detection system can comprise one or more probes to detect one or more peripheral neuropathic pain specific biomarkers, such as listed in  and in  for peripheral neuropathic pain, of one or more vesicles of a biological sample.","Schizophrenia","Schizophrenia specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a schizophrenia specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-181b. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-7, miR-24, miR-26b, miR-29b, miR-30b, miR-30e, miR-92, or miR-195, or any combination thereof.","The one or more mRNAs that may be analyzed can include, but are not limited to, IFITM3, SERPINA3, GLS, or ALDH7A1BASP1, or any combination thereof and can be used as specific biomarkers from a vesicle for schizophrenia. A biomarker mutation for schizophrenia that can be assessed in a vesicle includes, but is not limited to, a mutation of to DISC1, dysbindin, neuregulin-1, seratonin 2a receptor, NURR1, or any combination of mutations specific for schizophrenia.","The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, ATP5B, ATP5H, ATP6V1B, DNM1, NDUFV2, NSF, PDHB, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more schizophrenia specific biomarkers, such as listed in  and in  for schizophrenia. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more schizophrenia specific biomarkers, such as listed in  and in  for schizophrenia. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for schizophrenia specific vesicles or vesicles comprising one or more schizophrenia specific biomarkers, such as listed in  and in  for schizophrenia.","One or more schizophrenia specific biomarkers, such as listed in  and in  for schizophrenia, can also be detected by one or more systems disclosed herein, for characterizing a schizophrenia. For example, a detection system can comprise one or more probes to detect one or more schizophrenia specific biomarkers, such as listed in  and in  for schizophrenia, of one or more vesicles of a biological sample.","Bipolar Disease","Bipolar disease specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a bipolar disease specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, FGF2, ALDH7A1, AGXT2L1, AQP4, or PCNT2, or any combination thereof and can be used as specific biomarkers from a vesicle for bipolar disease. A biomarker mutation for bipolar disease that can be assessed in a vesicle includes, but is not limited to, a mutation of Dysbindin, DAOA\/G30, DISC1, neuregulin-1, or any combination of mutations specific for bipolar disease.","The invention also provides an isolated vesicle comprising one or more bipolar disease specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more bipolar disease specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for bipolar disease specific vesicles or vesicles comprising one or more bipolar disease specific biomarkers, such as listed in .","One or more bipolar disease specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a bipolar disease. For example, a detection system can comprise one or more probes to detect one or more bipolar disease specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Depression","Depression specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a depression specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, FGFR1, FGFR2, FGFR3, or AQP4, or any combination thereof can also be used as specific biomarkers from a vesicle for depression.","The invention also provides an isolated vesicle comprising one or more depression specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more depression specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for depression specific vesicles or vesicles comprising one or more depression specific biomarkers, such as listed in .","One or more depression specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a depression. For example, a detection system can comprise one or more probes to detect one or more depression specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Gastrointestinal Stromal Tumor (GIST)","GIST specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a GIST specific biosignature. For example, the one or more mRNAs that may be analyzed can include, but are not limited to, DOG-1, PKC-theta, KIT, GPR20, PRKCQ, KCNK3, KCNH2, SCG2, TNFRSF6B, or CD34, or any combination thereof and can be used as specific biomarkers from a vesicle for GIST.","A biomarker mutation for GIST that can be assessed in a vesicle includes, but is not limited to, a mutation of PKC-theta or any combination of mutations specific for GIST. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, PDGFRA, c-kit, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more GIST specific biomarkers, such as listed in  and in  for GIST. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more GIST specific biomarkers, such as listed in  and in  for GIST. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for GIST specific vesicles or vesicles comprising one or more GIST specific biomarkers, such as listed in  and in  for GIST.","One or more GIST specific biomarkers, such as listed in  and in  for GIST, can also be detected by one or more systems disclosed herein, for characterizing a GIST. For example, a detection system can comprise one or more probes to detect one or more GIST specific biomarkers, such as listed in  and in  for GIST, of one or more vesicles of a biological sample.","Renal Cell Carcinoma","Renal cell carcinoma specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a renal cell carcinoma specific biosignature. For example, the biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-141, miR-200c, or any combination thereof. The one or more upregulated or overexpressed miRNA can be miR-28, miR-185, miR-27, miR-let-7f-2, or any combination thereof.","The one or more mRNAs that may be analyzed can include, but are not limited to, laminin receptor 1, betaig-h3, Galectin-1, a-2 Macroglobulin, Adipophilin, Angiopoietin 2, Caldesmon 1, Class II MHC-associated invariant chain (CD74), Collagen IV-al, Complement component, Complement component 3, Cytochrome P450, subfamily Ill polypeptide 2, Delta sleep-inducing peptide, Fc g receptor IIIa (CD16), HLA-B, HLA-DRa, HLA-DRb, HLA-SB, IFN-induced transmembrane protein 3, IFN-induced transmembrane protein 1, or Lysyl Oxidase, or any combination thereof and can be used as specific biomarkers from a vesicle for renal cell carcinoma.","A biomarker mutation for renal cell carcinoma that can be assessed in a vesicle includes, but is not limited to, a mutation of VHL or any combination of mutations specific renal cell carcinoma.","The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, IF1alpha, VEGF, PDGFRA, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more RCC specific biomarkers, such as ALPHA-TFEB, NONO-TFE3, PRCC-TFE3, SFPQ-TFE3, CLTC-TFE3, or MALAT1-TFEB, or those listed in  and in  for RCC. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more RCC specific biomarkers, such as ALPHA-TFEB, NONO-TFE3, PRCC-TFE3, SFPQ-TFE3, CLTC-TFE3, or MALAT1-TFE, or those listed in  and in  for RCC. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for RCC specific vesicles or vesicles comprising one or more RCC specific biomarkers, such as ALPHA-TFEB, NONO-TFE3, PRCC-TFE3, SFPQ-TFE3, CLTC-TFE3, or MALAT1-TFE, or those listed in  and in  for RCC.","One or more RCC specific biomarkers, such as ALPHA-TFEB, NONO-TFE3, PRCC-TFE3, SFPQ-TFE3, CLTC-TFE3, or MALAT1-TFE, or those listed in  and in  for RCC, can also be detected by one or more systems disclosed herein, for characterizing a RCC. For example, a detection system can comprise one or more probes to detect one or more RCC specific biomarkers, such as ALPHA-TFEB, NONO-TFE3, PRCC-TFE3, SFPQ-TFE3, CLTC-TFE3, or MALAT1-TFE, or those listed in  and in  for RCC, of one or more vesicles of a biological sample.","Cirrhosis","Cirrhosis specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a cirrhosis specific biosignature. The one or more mRNAs that may be analyzed include, but are not limited to, NLT, which can be used as aspecific biomarker from a vesicle for cirrhosis.","The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, NLT, HBsAG, AST, YKL-40, Hyaluronic acid, TIMP-1, alpha 2 macroglobulin, a-1-antitrypsin P1Z allele, haptoglobin, or acid phosphatase ACP AC, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more cirrhosis specific biomarkers, such as those listed in  and in  for cirrhosis. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more cirrhosis specific biomarkers, such as those listed in  and in  for cirrhosis. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for cirrhosis specific vesicles or vesicles comprising one or more cirrhosis specific biomarkers, such as those listed in  and in  for cirrhosis.","One or more cirrhosis specific biomarkers, such as those listed in  and in  for cirrhosis, can also be detected by one or more systems disclosed herein, for characterizing cirrhosis. For example, a detection system can comprise one or more probes to detect one or more cirrhosis specific biomarkers, such as those listed in  and in  for cirrhosis, of one or more vesicles of a biological sample.","Esophageal Cancer","Esophageal cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a esophageal cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-192, miR-194, miR-21, miR-200c, miR-93, miR-342, miR-152, miR-93, miR-25, miR-424, or miR-151, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-27b, miR-205, miR-203, miR-342, let-7c, miR-125b, miR-100, miR-152, miR-192, miR-194, miR-27b, miR-205, miR-203, miR-200c, miR-99a, miR-29c, miR-140, miR-103, or miR-107, or any combination thereof. The one or more mRNAs that may be analyzed include, but are not limited to, MTHFR and can be used as specific biomarkers from a vesicle for esophageal cancer.","The invention also provides an isolated vesicle comprising one or more esophageal cancer specific biomarkers, such as listed in  and in  for esophageal cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more esophageal cancer specific biomarkers, such as listed in  and in  for esophageal cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for esophageal cancer specific vesicles or vesicles comprising one or more esophageal cancer specific biomarkers, such as listed in  and in  for esophageal cancer.","One or more esophageal cancer specific biomarkers, such as listed in  and in  for esophageal cancer, can also be detected by one or more systems disclosed herein, for characterizing a esophageal cancer. For example, a detection system can comprise one or more probes to detect one or more esophageal cancer specific biomarkers, such as listed in  and in  for esophageal cancer, of one or more vesicles of a biological sample.","Gastric Cancer","Gastric cancer specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a gastric cancer specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-106a, miR-21, miR-191, miR-223, miR-24-1, miR-24-2, miR-107, miR-92-2, miR-214, miR-25, or miR-221, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, let-7a.","The one or more mRNAs that may be analyzed include, but are not limited to, RRM2, EphA4, or survivin, or any combination thereof and can be used as specific biomarkers from a vesicle for gastric cancer. A biomarker mutation for gastric cancer that can be assessed in a vesicle includes, but is not limited to, a mutation of APC or any combination of mutations specific for gastric cancer. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to EphA4.","The invention also provides an isolated vesicle comprising one or more gastric cancer specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more gastric cancer specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for gastric cancer specific vesicles or vesicles comprising one or more gastric cancer specific biomarkers, such as listed in .","One or more gastric cancer specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a gastric cancer. For example, a detection system can comprise one or more probes to detect one or more gastric cancer specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Autism","Autism specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create an autism specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-484, miR-21, miR-212, miR-23a, miR-598, miR-95, miR-129, miR-431, miR-7, miR-15a, miR-27a, miR-15b, miR-148b, miR-132, or miR-128, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-93, miR-106a, miR-539, miR-652, miR-550, miR-432, miR-193b, miR-181d, miR-146b, miR-140, miR-381, miR-320a, or miR-106b, or any combination thereof. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, GM1, GD1a, GD1b, or GT1b, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more autism specific biomarkers, such as listed in  and in  for autism. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more autism specific biomarkers, such as listed in  and in  for autism. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for autism specific vesicles or vesicles comprising one or more autism specific biomarkers, such as listed in  and in  for autism.","One or more autism specific biomarkers, such as listed in  and in  for autism, can also be detected by one or more systems disclosed herein, for characterizing a autism. For example, a detection system can comprise one or more probes to detect one or more autism specific biomarkers, such as listed in  and in  for autism, of one or more vesicles of a biological sample.","Organ Rejection","Organ rejection specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create an organ rejection specific biosignature. For example, the biosignature can comprise one or more overexpressed miRs, such as, but not limited to, miR-658, miR-125a, miR-320, miR-381, miR-628, miR-602, miR-629, or miR-125a, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as, but not limited to, miR-324-3p, miR-611, miR-654, miR-330_MM1, miR-524, miR-17-3p_MM1, miR-483, miR-663, miR-516-5p, miR-326, miR-197_MM2, or miR-346, or any combination thereof. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, matix metalloprotein-9, proteinase 3, or HNP, or any combinations thereof. The biomarker can be a member of the matrix metalloproteinases.","The invention also provides an isolated vesicle comprising one or more organ rejection specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more organ rejection specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for organ rejection specific vesicles or vesicles comprising one or more organ rejection specific biomarkers, such as listed in .","One or more organ rejection specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a organ rejection. For example, a detection system can comprise one or more probes to detect one or more organ rejection specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Methicillin-Resistant ","Methicillin-resistant specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a methicillin-resistant specific biosignature.","The one or more mRNAs that may be analyzed include, but are not limited to, TSST-1 which can be used as a specific biomarker from a vesicle for methicillin-resistant . A biomarker mutation for methicillin-resistant that can be assessed in a vesicle includes, but is not limited to, a mutation of mecA, Protein A SNPs, or any combination of mutations specific for methicillin-resistant . The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, ETA, ETB, TSST-1, or leukocidins, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more methicillin-resistant specific biomarkers, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more methicillin-resistant specific biomarkers, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for methicillin-resistant specific vesicles or vesicles comprising one or more methicillin-resistant specific biomarkers, such as listed in .","One or more methicillin-resistant specific biomarkers, such as listed in , can also be detected by one or more systems disclosed herein, for characterizing a methicillin-resistant . For example, a detection system can comprise one or more probes to detect one or more methicillin-resistant specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","Vulnerable Plaque","Vulnerable plaque specific biomarkers from a vesicle can include one or more (for example, 2, 3, 4, 5, 6, 7, 8, or more) overexpressed miRs, underexpressed miRs, mRNAs, genetic mutations, proteins, ligands, peptides, snoRNA, or any combination thereof, such as listed in , and can be used to create a vulnerable plaque specific biosignature. The protein, ligand, or peptide that can be assessed in a vesicle can include, but is not limited to, IL-6, MMP-9, PAPP-A, D-dimer, fibrinogen, Lp-PLA2, SCD40L, Il-18, oxLDL, GPx-1, MCP-1, PIGF, or CRP, or any combination thereof.","The invention also provides an isolated vesicle comprising one or more vulnerable plaque specific biomarkers, such as listed in  and in  for vulnerable plaque. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more vulnerable plaque specific biomarkers, such as listed in  and in  for vulnerable plaque. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vulnerable plaque specific vesicles or vesicles comprising one or more vulnerable plaque specific biomarkers, such as listed in  and in  for vulnerable plaque.","One or more vulnerable plaque specific biomarkers, such as listed in  and in  for vulnerable plaque, can also be detected by one or more systems disclosed herein, for characterizing a vulnerable plaque. For example, a detection system can comprise one or more probes to detect one or more vulnerable plaque specific biomarkers, such as listed in  and in  for vulnerable plaque, of one or more vesicles of a biological sample.","Autoimmune Disease","The invention also provides an isolated vesicle comprising one or more autoimmune disease specific biomarkers, such as listed in  for autoimmune disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more autoimmune disease specific biomarkers, such as listed in  for autoimmune disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for autoimmune disease specific vesicles or vesicles comprising one or more autoimmune disease specific biomarkers, such as listed in  for autoimmune disease.","One or more autoimmune disease specific biomarkers, such as listed in  for autoimmune disease, can also be detected by one or more systems disclosed herein, for characterizing a autoimmune disease. For example, a detection system can comprise one or more probes to detect one or more autoimmune disease specific biomarkers, such as listed in  for autoimmune disease, of one or more vesicles of a biological sample.","Tuberculosis (TB)","The invention also provides an isolated vesicle comprising one or more TB disease specific biomarkers, such as listed in  for TB disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more TB disease specific biomarkers, such as listed in  for TB disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for TB disease specific vesicles or vesicles comprising one or more TB disease specific biomarkers, such as listed in  for TB disease.","One or more TB disease specific biomarkers, such as listed in  for TB disease, can also be detected by one or more systems disclosed herein, for characterizing a TB disease. For example, a detection system can comprise one or more probes to detect one or more TB disease specific biomarkers, such as listed in  for TB disease, of one or more vesicles of a biological sample.","HIV","The invention also provides an isolated vesicle comprising one or more HIV disease specific biomarkers, such as listed in  for HIV disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more HIV disease specific biomarkers, such as listed in  for HIV disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for HIV disease specific vesicles or vesicles comprising one or more HIV disease specific biomarkers, such as listed in  for HIV disease.","One or more HIV disease specific biomarkers, such as listed in  for HIV disease, can also be detected by one or more systems disclosed herein, for characterizing a HIV disease. For example, a detection system can comprise one or more probes to detect one or more HIV disease specific biomarkers, such as listed in  for HIV disease, of one or more vesicles of a biological sample.","The one or more biomarker can also be a miRNA, such as an upregulated or overexpressed miRNA. The upregulated miRNA can be miR-29a, miR-29b, miR-149, miR-378 or miR-324-5p. One or more biomarkers can also be used to characterize HIV-1 latency, such as by assessing one or more miRNAs. The miRNA can be miR-28, miR-125b, miR-150, miR-223 and miR-382, and upregulated.","Asthma","The invention also provides an isolated vesicle comprising one or more asthma disease specific biomarkers, such as listed in  for asthma disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more asthma disease specific biomarkers, such as listed in  for asthma disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for asthma disease specific vesicles or vesicles comprising one or more asthma disease specific biomarkers, such as listed in  for asthma disease.","One or more asthma disease specific biomarkers, such as listed in  for asthma disease, can also be detected by one or more systems disclosed herein, for characterizing a asthma disease. For example, a detection system can comprise one or more probes to detect one or more asthma disease specific biomarkers, such as listed in  for asthma disease, of one or more vesicles of a biological sample.","Lupus","The invention also provides an isolated vesicle comprising one or more lupus disease specific biomarkers, such as listed in  for lupus disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more lupus disease specific biomarkers, such as listed in  for lupus disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for lupus disease specific vesicles or vesicles comprising one or more lupus disease specific biomarkers, such as listed in  for lupus disease.","One or more lupus disease specific biomarkers, such as listed in  for lupus disease, can also be detected by one or more systems disclosed herein, for characterizing a lupus disease. For example, a detection system can comprise one or more probes to detect one or more lupus disease specific biomarkers, such as listed in  for lupus disease, of one or more vesicles of a biological sample.","Influenza","The invention also provides an isolated vesicle comprising one or more influenza disease specific biomarkers, such as listed in  for influenza disease. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more influenza disease specific biomarkers, such as listed in  for influenza disease. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for influenza disease specific vesicles or vesicles comprising one or more influenza disease specific biomarkers, such as listed in  for influenza disease.","One or more influenza disease specific biomarkers, such as listed in  for influenza disease, can also be detected by one or more systems disclosed herein, for characterizing a influenza disease. For example, a detection system can comprise one or more probes to detect one or more influenza disease specific biomarkers, such as listed in  for influenza disease, of one or more vesicles of a biological sample.","Thyroid Cancer","The invention also provides an isolated vesicle comprising one or more thyroid cancer specific biomarkers, such as AKAP9-BRAF, CCDC6-RET, ERC1-RETM, GOLGA5-RET, HOOK3-RET, HRH4-RET, KTN1-RET, NCOA4-RET, PCM1-RET, PRKARA1A-RET, RFG-RET, RFG9-RET, Ria-RET, TGF-NTRK1, TPM3-NTRK1, TPM3-TPR, TPR-MET, TPR-NTRK1, TRIM24-RET, TRIM27-RET or TRIM33-RET, characteristic of papillary thyroid carcinoma; or PAX8-PPARy, characteristic of follicular thyroid cancer. A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more thyroid cancer specific biomarkers, such as listed in  for thyroid cancer. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for thyroid cancer specific vesicles or vesicles comprising one or more thyroid cancer specific biomarkers, such as listed in  for thyroid cancer.","One or more thyroid cancer specific biomarkers, such as listed in  for thyroid cancer, can also be detected by one or more systems disclosed herein, for characterizing a thyroid cancer. For example, a detection system can comprise one or more probes to detect one or more thyroid cancer specific biomarkers, such as listed in  for thyroid cancer, of one or more vesicles of a biological sample.","Gene Fusions","The one or more biomarkers assessed of vesicle, can be a gene fusion, such as one or more listed in . A fusion gene is a hybrid gene created by the juxtaposition of two previously separate genes. This can occur by chromosomal translocation or inversion, deletion or via trans-splicing. The resulting fusion gene can cause abnormal temporal and spatial expression of genes, such as leading to abnormal expression of cell growth factors, angiogenesis factors, tumor promoters or other factors contributing to the neoplastic transformation of the cell and the creation of a tumor. Such fusion genes can be oncogenic due to the juxtaposition of: 1) a strong promoter region of one gene next to the coding region of a cell growth factor, tumor promoter or other gene promoting oncogenesis leading to elevated gene expression, or 2) due to the fusion of coding regions of two different genes, giving rise to a chimeric gene and thus a chimeric protein with abnormal activity.","An example of a fusion gene is BCR-ABL, a characteristic molecular aberration in \u02dc90% of chronic myelogenous leukemia (CML) and in a subset of acute leukemias (Kurzrock et al., 2003; 138(10):819-830). The BCR-ABL results from a translocation between chromosomes 9 and 22. The translocation brings together the 5\u2032 region of the BCR gene and the 3\u2032 region of ABL1, generating a chimeric BCR-ABL1 gene, which encodes a protein with constitutively active tyrosine kinase activity (Mittleman et al., Nature 2007; 7(4):233-245). The aberrant tyrosine kinase activity leads to de-regulated cell signaling, cell growth and cell survival, apoptosis resistance and growth factor independence, all of which contribute to the pathophysiology of leukemia (Kurzrock et al., 2003; 138(10):819-830).","Another fusion gene is IGH-MYC, a defining feature of \u02dc80% of Burkitt's lymphoma (Ferry et al. 2006; 11(4):375-83). The causal event for this is a translocation between chromosomes 8 and 14, bringing the c-Myc oncogene adjacent to the strong promoter of the immunoglobin heavy chain gene, causing c-myc overexpression (Mittleman et al., 2007; 7(4):233-245). The c-myc rearrangement is a pivotal event in lymphomagenesis as it results in a perpetually proliferative state. It has wide ranging effects on progression through the cell cycle, cellular differentiation, apoptosis, and cell adhesion (Ferry et al. 2006; 11(4):375-83).","A number of recurrent fusion genes have been catalogued in the Mittleman database (cgap.nci.nih.gov\/Chromosomes\/Mitelman) and can be assessed in a vesicle, and used to characterize a phenotype. The gene fusion can be used to characterize a hematological malignancy or epithelial tumor. For example, TMPRSS2-ERG, TMPRSS2-ETV and SLC45A3-ELK4 fusions can be detected and used to characterize prostate cancer; and ETV6-NTRK3 and ODZ4-NRG1 for breast cancer.","Furthermore, assessing the presence or absence, or expression level of a fusion gene can be used to diagnosis a phenotype such as a cancer as well as a monitoring a therapeutic response to selecting a treatment. For example, the presence of the BCR-ABL fusion gene is a characteristic not only for the diagnosis of CML, but is also the target of the Novartis drug Imatinib mesylate (Gleevec), a receptor tyrosine kinase inhibitor, for the treatment of CML. Imatinib treatment has led to molecular responses (disappearance of BCR-ABL+blood cells) and improved progression-free survival in BCR-ABL+CML patients (Kantarjian et al., 2007; 13(4):1089-1097).","Assessing a vesicle for the presence, absence, or expression level of a gene fusion can be of by assessing a heterogeneous population of vesicles for the presence, absence, or expression level of a gene fusion. Alternatively, the vesicle that is assessed can be derived from a specific cell type, such as cell-of-origin specific vesicle, as described above. Illustrative examples of use of fusions that can be assessed to characterize a phenotype include the following:","Breast Cancer","To characterize a breast cancer, a vesicle can be assessed for one or more breast cancer specific fusions, including, but not limited to, ETV6-NTRK3. The vesicle can be derived from a breast cancer cell.","Lung Cancer","To characterize a lung cancer, a vesicle can be assessed for one or more lung cancer specific fusions, including, but not limited to, RLF-MYCL1, TGF-ALK, or CD74-ROS1. The vesicle can be derived from a lung cancer cell.","Prostate Cancer","To characterize a prostate cancer, a vesicle can be assessed for one or more prostate cancer specific fusions, including, but not limited to, ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1,TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4. The vesicle can be derived from a prostate cancer cell.","Brain Cancer","To characterize a brain cancer, a vesicle can be assessed for one or more brain cancer specific fusions, including, but not limited to, GOPC-ROS 1. The vesicle can be derived from a brain cancer cell.","Head and Neck Cancer","To characterize a head and neck cancer, a vesicle can be assessed for one or more head and neck cancer specific fusions, including, but not limited to, CHCHD7-PLAG1, CTNNB1-PLAG1, FHIT-HMGA2, HMGA2-NFIB, LIFR-PLAG1, or TCEA1-PLAG1. The vesicle can be derived from a head and\/or neck cancer cell.","Renal Cell Carcinoma (RCC)","To characterize a RCC, a vesicle can be assessed for one or more RCC specific fusions, including, but not limited to, ALPHA-TFEB, NONO-TFE3, PRCC-TFE3, SFPQ-TFE3, CLTC-TFE3, or MALAT1-TFEB. The vesicle can be derived from a RCC cell.","Thyroid Cancer","To characterize a thyroid cancer, a vesicle can be assessed for one or more thyroid cancer specific fusions, including, but not limited to, AKAP9-BRAF, CCDC6-RET, ERC1-RETM, GOLGA5-RET, HOOK3-RET, HRH4-RET, KTN1-RET, NCOA4-RET, PCM1-RET, PRKARA1A-RET, RFG-RET, RFG9-RET, Ria-RET, TGF-NTRK1, TPM3-NTRK1, TPM3-TPR, TPR-MET, TPR-NTRK1, TRIM24-RET, TRIM27-RET or TRIM33-RET, characteristic of papillary thyroid carcinoma; or PAX8-PPARy, characteristic of follicular thyroid cancer. The vesicle can be derived from a thyroid cancer cell.","Blood Cancers","To characterize a blood cancer, a vesicle can be assessed for one or more blood cancer specific fusions, including, but not limited to, TTL-ETV6, CDK6-MLL, CDK6-TLX3, ETV6-FLT3, ETV6-RUNX1, ETV6-TTL, MLL-AFF1, MLL-AFF3, MLL-AFF4, MLL-GAS7, TCBA1-ETV6, TCF3-PBX1 or TCF3-TFPT, characteristic of acute lymphocytic leukemia (ALL); BCL11B-TLX3, IL2-TNFRFS17, NUP214-ABL1, NUP98-CCDC28A, TAL1-STIL, or ETV6-ABL2, characteristic of T-cell acute lymphocytic leukemia (T-ALL); ATIC-ALK, KIAA1618-ALK, MSN-ALK, MYH9-ALK, NPM1-ALK, TGF-ALK or TPM3-ALK, characteristic of anaplastic large cell lymphoma (ALCL); BCR-ABL1, BCR-JAK2, ETV6-EVI1, ETV6-MN1 or ETV6-TCBA1, characteristic of chronic myelogenous leukemia (CML); CBFB-MYH11, CHIC2-ETV6, ETV6-ABL1, ETV6-ABL2, ETV6-ARNT, ETV6-CDX2, ETV6-HLXB9, ETV6-PER1, MEF2D-DAZAP1, AML-AFF1, MLL-ARHGAP26, MLL-ARHGEF12, MLL-CASC5, MLL-CBL, MLL-CREBBP, MLL-DAB21P, MLL-ELL, MLL-EP300, MLL-EPS15, MLL-FNBP1, MLL-FOXO3A, MLL-GMPS, MLL-GPHN, MLL-MLLT1, MLL-MLLT11, MLL-MLLT3, MLL-MLLT6, MLL-MYO1F, MLL-PICALM, MLL-SEPT2, MLL-SEPT6, MLL-SORBS2, MYST3-SORBS2, MYST-CREBBP, NPM1-MLF1, NUP98-HOXA13, PRDM16-EVI1, RABEP1-PDGFRB, RUNX1-EVI1, RUNX1-MDS1, RUNX1-RPL22, RUNX1-RUNX1T1, RUNX1-SH3D19, RUNX1-USP42, RUNX1-YTHDF2, RUNX1-ZNF687, or TAF15-ZNF-384, characteristic of AML; CCND1-FSTL3, characteristic of chronic lymphocytic leukemia (CLL); BCL3-MYC, MYC-BTG1, BCL7A-MYC, BRWD3-ARHGAP20 or BTG1-MYC, characteristic of B-cell chronic lymphocytic leukemia (B-CLL); CITTA-BCL6, CLTC-ALK, IL21R-BCL6, PIM1-BCL6, TFCR-BCL6, IKZF1-BCL6 or SEC31A-ALK, characteristic of diffuse large B-cell lymphomas (DLBCL); FLIP1-PDGFRA, FLT3-ETV6, KIAA1509-PDGFRA, PDE4DIP-PDGFRB, NIN-PDGFRB, TP53BP1-PDGFRB, or TPM3-PDGFRB, characteristic of hyper eosinophilia\/chronic eosinophilia; IGH-MYC or LCP1-BCL6, characteristic of Burkitt's lymphoma. The vesicle can be derived from a blood cancer cell.","The invention also provides an isolated vesicle comprising one or more gene fusions as disclosed herein, such as listed in . A composition comprising the isolated vesicle is also provided. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more gene fusions, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more gene fusions, such as listed in .","Also provided herein is a detection system for detecting one or more gene fusions, such as gene fusions listed in . For example, a detection system can comprise one or more probes to detect one or more gene fusions listed in . Detection of the one or more gene fusions can be used to charcaterize a cancer.","Gene-Associated MiRNA Biomarkers","The one or more biomarkers assessed can also include one or more genes selected from the group consisting of PFKFB3, RHAMM (HMMR), cDNA FLJ42103, ASPM, CENPF, NCAPG, Androgen Receptor, EGFR, HSP90, SPARC, DNMT3B, GART, MGMT, SSTR3, and TOP2B. The microRNA that interacts with the one or more genes can also be a biomarker (see for example, ). Furthermore, the one or more biomarkers can be used to characterize prostate cancer.","The invention also provides an isolated vesicle comprising one or more one or more biomarkers consisting of PFKFB3, RHAMM (HMMR), cDNA FLJ42103, ASPM, CENPF, NCAPG, Androgen Receptor, EGFR, HSP90, SPARC, DNMT3B, GART, MGMT, SSTR3, and TOP2B; or the microRNA that interacts with the one or more genes (see for example, ). The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of PFKFB3, RHAMM (HMMR), cDNA FLJ42103, ASPM, CENPF, NCAPG, Androgen Receptor, EGFR, HSP90, SPARC, DNMT3B, GART, MGMT, SSTR3, and TOP2B; or the microRNA that interacts with the one or more genes, such as listed in . The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more biomarkers consisting of PFKFB3, RHAMM (HMMR), cDNA FLJ42103, ASPM, CENPF, NCAPG, Androgen Receptor, EGFR, HSP90, SPARC, DNMT3B, GART, MGMT, SSTR3, and TOP2B; or the microRNA that interacts with the one or more genes, such as listed in .","One or more prostate cancer specific biomarkers, such as listed in  can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more prostate cancer specific biomarkers, such as listed in , of one or more vesicles of a biological sample.","The miRNA that interacts with PFKFB3 can be miR-513a-3p, miR-128, miR-488, miR-539, miR-658, miR-524-5p, miR-1258, miR-150, miR-216b, miR-377, miR-135a, miR-26a, miR-548a-5p, miR-26b, miR-520d-5p, miR-224, miR-1297, miR-1197, miR-182, miR-452, miR-509-3-5p, miR-548m, miR-625, miR-509-5p, miR-1266, miR-135b, miR-190b, miR-496, miR-616, miR-621, miR-650, miR-105, miR-19a, miR-346, miR-620, miR-637, miR-651, miR-1283, miR-590-3p, miR-942, miR-1185, miR-577, miR-602, miR-1305, miR-220c, miR-1270, miR-1282, miR-432, miR-491-5p, miR-548n, miR-765, miR-768-3p or miR-924, and can be used as a biomarker.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with PFKFB3. Also provided herein is a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with PFKFB3. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with PFKFB3. Furthermore, the one or more miRNA that interacts with PFKFB3 can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with PFKFB3 of one or more vesicles of a biological sample.","The miRNA that interacts with RHAMM can be miR-936, miR-656, miR-105, miR-361-5p, miR-194, miR-374a, miR-590-3p, miR-186, miR-769-5p, miR-892a, miR-380, miR-875-3p, miR-208a, miR-208b, miR-586, miR-125a-3p, miR-630, miR-374b, miR-411, miR-629, miR-1286, miR-1185, miR-16, miR-200b, miR-671-5p, miR-95, miR-421, miR-496, miR-633, miR-1243, miR-127-5p, miR-143, miR-15b, miR-200c, miR-24 or miR-34c-3p.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with RHAMM. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with RHAMM. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with RHAMM. Furthermore, the one or more miRNA that interacts with RHAMM can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with RHAMM of one or more vesicles of a biological sample.","The miRNA that interacts with CENPF can be miR-30c, miR-30b, miR-190, miR-508-3p, miR-384, miR-512-5p, miR-548p, miR-297, miR-520f, miR-376a, miR-1184, miR-577, miR-708, miR-205, miR-376b, miR-520g, miR-520h, miR-519d, miR-596, miR-768-3p, miR-340, miR-620, miR-539, miR-567, miR-671-5p, miR-1183, miR-129-3p, miR-636, miR-106a, miR-1301, miR-17, miR-20a, miR-570, miR-656, miR-1263, miR-1324, miR-142-5p, miR-28-5p, miR-302b, miR-452, miR-520d-3p, miR-548o, miR-892b, miR-302d, miR-875-3p, miR-106b, miR-1266, miR-1323, miR-20b, miR-221, miR-520e, miR-664, miR-920, miR-922, miR-93, miR-1228, miR-1271, miR-30e, miR-483-3p, miR-509-3-5p, miR-515-3p, miR-519e, miR-520b, miR-520c-3p or miR-582-3p.","Also provided herein is a vesicle comprising one or more one or more miRNA that interacts with CENPF. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with CENPF. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with CENPF. Furthermore, the one or more miRNA that interacts with CENPF can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with CENPF of one or more vesicles of a biological sample.","The miRNA that interacts with NCAPG can be miR-876-5p, miR-1260, miR-1246, miR-548c-3p, miR-1224-3p, miR-619, miR-605, miR-490-5p, miR-186, miR-448, miR-129-5p, miR-188-3p, miR-516b, miR-342-3p, miR-1270, miR-548k, miR-654-3p, miR-1290, miR-656, miR-34b, miR-520g, miR-1231, miR-1289, miR-1229, miR-23a, miR-23b, miR-616 or miR-620.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with NCAPG. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with NCAPG. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with NCAPG. Furthermore, the one or more miRNA that interacts with NCAPG can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with NCAPG of one or more vesicles of a biological sample.",", The miRNA that interacts with Androgen Receptor can be miR-124a, miR-130a, miR-130b, miR-143, miR-149, miR-194, miR-29b, miR-29c, miR-301, miR-30a-5p, miR-30d, miR-30e-5p, miR-337, miR-342, miR-368, miR-488, miR-493-5p, miR-506, miR-512-5p, miR-644, miR-768-5p or miR-801.","The miRNA that interacts with EGFR can be miR-105, miR-128a, miR-128b, miR-140, miR-141, miR-146a, miR-146b, miR-27a, miR-27b, miR-302a, miR-302d, miR-370, miR-548c, miR-574, miR-587 or miR-7.","The invention also provides an isolated vesicle, comprising one or more one or more miRNA that interacts with AR. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with AR. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with AR. Furthermore, the one or more miRNA that interacts with AR can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with AR of one or more vesicles of a biological sample.","The miRNA that interacts with HSP90 can be miR-1, miR-513a-3p, miR-548d-3p, miR-642, miR-206, miR-450b-3p, miR-152, miR-148a, miR-148b, miR-188-3p, miR-23a, miR-23b, miR-578, miR-653, miR-1206, miR-192, miR-215, miR-181b, miR-181d, miR-223, miR-613, miR-769-3p, miR-99a, miR-100, miR-454, miR-548n, miR-640, miR-99b, miR-150, miR-181a, miR-181c, miR-522, miR-624, miR-130a, miR-130b, miR-146, miR-148a, miR-148b, miR-152, miR-181a, miR-181b, miR-181c, miR-204, miR-206, miR-211, miR-212, miR-215, miR-223, miR-23a, miR-23b, miR-301, miR-31, miR-325, miR-363, miR-566, miR-9 or miR-99b.","The invention also provides an isolated vesicle, comprising one or more one or more miRNA that interacts with HSP90. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with HSP90. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with HSP90. Furthermore, the one or more miRNA that interacts with HSP90 can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with HSP90 of one or more vesicles of a biological sample.","The miRNA that interacts with SPARC can be miR-768-5p, miR-203, miR-196a, miR-569, miR-187, miR-641, miR-1275, miR-432, miR-622, miR-296-3p, miR-646, miR-196b, miR-499-5p, miR-590-5p, miR-495, miR-625, miR-1244, miR-512-5p, miR-1206, miR-1303, miR-186, miR-302d, miR-494, miR-562, miR-573, miR-10a, miR-203, miR-204, miR-211, miR-29, miR-29b, miR-29c, miR-339, miR-433, miR-452, miR-515-5p, miR-517a, miR-517b, miR-517c, miR-592 or miR-96.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with SPARC. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with SPARC. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with SPARC. Furthermore, the one or more miRNA that interacts with SPARC can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with SPARC of one or more vesicles of a biological sample.","The miRNA that interacts with DNMT3B can be miR-618, miR-1253, miR-765, miR-561, miR-330-5p, miR-326, miR-188, miR-203, miR-221, miR-222, miR-26a, miR-26b, miR-29a, miR-29b, miR-29c, miR-370, miR-379, miR-429, miR-519e, miR-598, miR-618 or miR-635.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with DNMT3B. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with DNMT3B. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with DNMT3B. Furthermore, the one or more miRNA that interacts with DNMT3B can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with DNMT3B of one or more vesicles of a biological sample.","The miRNA that interacts with GART can be miR-101, miR-141, miR-144, miR-182, miR-189, miR-199a, miR-199b, miR-200a, miR-200b, miR-202, miR-203, miR-223, miR-329, miR-383, miR-429, miR-433, miR-485-5p, miR-493-5p, miR-499, miR-519a, miR-519b, miR-519c, miR-569, miR-591, miR-607, miR-627, miR-635, miR-636 or miR-659.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with GART. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with GART. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with GART. Furthermore, the one or more miRNA that interacts with GART can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with GART of one or more vesicles of a biological sample.","The miRNA that interacts with MGMT can be miR-122a, miR-142-3p, miR-17-3p, miR-181a, miR-181b, miR-181c, miR-181d, miR-199b, miR-200a, miR-217, miR-302b, miR-32, miR-324-3p, miR-34a, miR-371, miR-425-5p, miR-496, miR-514, miR-515-3p, miR-516-3p, miR-574, miR-597, miR-603, miR-653, miR-655, miR-92, miR-92b or miR-99a.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with MGMT. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with MGMT. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with MGMT. Furthermore, the one or more miRNA that interacts with MGMT can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with MGMT of one or more vesicles of a biological sample.","The miRNA that interacts with SSTR3 can be miR-125a, miR-125b, miR-133a, miR-133b, miR-136, miR-150, miR-21, miR-380-5p, miR-504, miR-550, miR-671, miR-766 or miR-767-3p.","The invention also provides an isolated vesicle comprising one or more one or more miRNA that interacts with SSTR3. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with SSTR3. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with SSTR3. Furthermore, the one or more miRNA that interacts with SSTR3 can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with SSTR3 of one or more vesicles of a biological sample.","The miRNA that interacts with TOP2B can be miR-548f, miR-548a-3p, miR-548g, miR-513a-3p, miR-548c-3p, miR-101, miR-653, miR-548d-3p, miR-575, miR-297, miR-576-3p, miR-548b-3p, miR-624, miR-548n, miR-758, miR-1253, miR-1324, miR-23b, miR-320a, miR-320b, miR-1183, miR-1244, miR-23a, miR-451, miR-568, miR-1276, miR-548e, miR-590-3p, miR-1, miR-101, miR-126, miR-129, miR-136, miR-140, miR-141, miR-144, miR-147, miR-149, miR-18, miR-181b, miR-181c, miR-182, miR-184, miR-186, miR-189, miR-191, miR-19a, miR-19b, miR-200a, miR-206, miR-210, miR-218, miR-223, miR-23a, miR-23b, miR-24, miR-27a, miR-302, miR-30a, miR-31, miR-320, miR-323, miR-362, miR-374, miR-383, miR-409-3p, miR-451, miR-489, miR-493-3p, miR-514, miR-542-3p, miR-544, miR-548a, miR-548b, miR-548c, miR-548d, miR-559, miR-568, miR-575, miR-579, miR-585, miR-591, miR-598, miR-613, miR-649, miR-651, miR-758, miR-768-3p or miR-9.","Also provided herein is a vesicle comprising one or more one or more miRNA that interacts with TOP2B. The invention further provides a composition comprising the isolated vesicle. Accordingly, in some embodiments, the composition comprises a population of vesicles comprising one or more biomarkers consisting of miRNA that interacts with TOP2B. The composition can comprise a substantially enriched population of vesicles, wherein the population of vesicles is substantially homogeneous for vesicles comprising one or more miRNA that interacts with TOP2B. Furthermore, the one or more miRNA that interacts with TOP2B can also be detected by one or more systems disclosed herein. For example, a detection system can comprise one or more probes to detect one or more one or more miRNA that interacts with TOP2B of one or more vesicles of a biological sample.","Other MicroRNA Biomarkers","Other microRNAs that can be detected or assessed in a vesicle and used to characterize a phenotype include, but are not limited to, hsa-let-7a, hsa-let-7b, hsa-let-7c, hsa-let-7d, hsa-let-7e, hsa-let-7f, hsa-miR-15a, hsa-miR-16, hsa-miR-17-5p, hsa-miR-17-3p, hsa-miR-18a, hsa-miR-19a, hsa-miR-19b, hsa-miR-20a, hsa-miR-21, hsa-miR-22, hsa-miR-23a, hsa-miR-189, hsa-miR-24, hsa-miR-25, hsa-miR-26a, hsa-miR-26b, hsa-miR-27a, hsa-miR-28, hsa-miR-29a, hsa-miR-30a-5p, hsa-miR-30a-3p, hsa-miR-31, hsa-miR-32, hsa-miR-33, hsa-miR-92, hsa-miR-93, hsa-miR-95, hsa-miR-96, hsa-miR-98, hsa-miR-99a, hsa-miR-100, hsa-miR-101, hsa-miR-29b, hsa-miR-103, hsa-miR-105, hsa-miR-106a, hsa-miR-107, hsa-miR-192, hsa-miR-196a, hsa-miR-197, hsa-miR-198, hsa-miR-199a, hsa-miR-199a*, hsa-miR-208, hsa-miR-129, hsa-miR-148a, hsa-miR-30c, hsa-miR-30d, hsa-miR-139, hsa-miR-147, hsa-miR-7, hsa-miR-10a, hsa-miR-10b, hsa-miR-34a, hsa-miR-181a, hsa-miR-181b, hsa-miR-181c, hsa-miR-182, hsa-miR-182*, hsa-miR-183, hsa-miR-187, hsa-miR-199b, hsa-miR-203, hsa-miR-204, hsa-miR-205, hsa-miR-210, hsa-miR-211, hsa-miR-212, hsa-miR-181a*, hsa-miR-214, hsa-miR-215, hsa-miR-216, hsa-miR-217, hsa-miR-218, hsa-miR-219, hsa-miR-220, hsa-miR-221, hsa-miR-222, hsa-miR-223, hsa-miR-224, hsa-miR-200b, hsa-let-7g, hsa-let-7i, hsa-miR-1, hsa-miR-15b, hsa-miR-23b, hsa-miR-27b, hsa-miR-30b, hsa-miR-122a, hsa-miR-124a, hsa-miR-125b, hsa-miR-128a, hsa-miR-130a, hsa-miR-132, hsa-miR-133a, hsa-miR-135a, hsa-miR-137, hsa-miR-138, hsa-miR-140, hsa-miR-141, hsa-miR-142-5p, hsa-miR-142-3p, hsa-miR-143, hsa-miR-144, hsa-miR-145, hsa-miR-152, hsa-miR-153, hsa-miR-191, hsa-miR-9, hsa-miR-9*, hsa-miR-125a, hsa-miR-126*, hsa-miR-126, hsa-miR-127, hsa-miR-134, hsa-miR-136, hsa-miR-146a, hsa-miR-149, hsa-miR-150, hsa-miR-154, hsa-miR-154*, hsa-miR-184, hsa-miR-185, hsa-miR-186, hsa-miR-188, hsa-miR-190, hsa-miR-193a, hsa-miR-194, hsa-miR-195, hsa-miR-206, hsa-miR-320, hsa-miR-200c, hsa-miR-155, hsa-miR-128b, hsa-miR-106b, hsa-miR-29c, hsa-miR-200a, hsa-miR-302a*, hsa-miR-302a, hsa-miR-34b, hsa-miR-34c, hsa-miR-299-3p, hsa-miR-301, hsa-miR-99b, hsa-miR-296, hsa-miR-130b, hsa-miR-30e-5p, hsa-miR-30e-3p, hsa-miR-361, hsa-miR-362, hsa-miR-363, hsa-miR-365, hsa-mir-302b*, hsa-miR-302b, hsa-miR-302c*, hsa-miR-302c, hsa-miR-302d, hsa-miR-367, hsa-miR-368, hsa-miR-369-3p, hsa-miR-370, hsa-miR-371, hsa-miR-372, hsa-miR-373*, hsa-miR-373, hsa-miR-374, hsa-miR-375, hsa-miR-376a, hsa-miR-377, hsa-miR-378, hsa-miR-422b, hsa-miR-379, hsa-miR-380-5p, hsa-miR-380-3p, hsa-miR-381, hsa-miR-382, hsa-miR-383, hsa-miR-340, hsa-miR-330, hsa-miR-328, hsa-miR-342, hsa-miR-337, hsa-miR-323, hsa-miR-326, hsa-miR-151, hsa-miR-135b, hsa-miR-148b, hsa-miR-331, hsa-miR-324-5p, hsa-miR-324-3p, hsa-miR-338, hsa-miR-339, hsa-miR-335, hsa-miR-133b, hsa-miR-325, hsa-miR-345, hsa-miR-346, ebv-miR-BHRF1-1, ebv-miR-BHRF1-2*, ebv-miR-BHRF1-2, ebv-miR-BHRF1-3, ebv-miR-BART1-5p, ebv-miR-BART2, hsa-miR-384, hsa-miR-196b, hsa-miR-422a, hsa-miR-423, hsa-miR-424, hsa-miR-425-3p, hsa-miR-18b, hsa-miR-20b, hsa-miR-448, hsa-miR-429, hsa-miR-449, hsa-miR-450, hcmv-miR-UL22A, hcmv-miR-UL22A*, hcmv-miR-UL36, hcmv-miR-UL112, hcmv-miR-UL148D, hcmv-miR-US5-1, hcmv-miR-US5-2, hcmv-miR-US25-1, hcmv-miR-US25-2-5p, hcmv-miR-US25-2-3p, hcmv-miR-US33, hsa-miR-191*, hsa-miR-200a*, hsa-miR-369-5p, hsa-miR-431, hsa-miR-433, hsa-miR-329, hsa-miR-453, hsa-miR-451, hsa-miR-452, hsa-miR-452*, hsa-miR-409-5p, hsa-miR-409-3p, hsa-miR-412, hsa-miR-410, hsa-miR-376b, hsa-miR-483, hsa-miR-484, hsa-miR-485-5p, hsa-miR-485-3p, hsa-miR-486, hsa-miR-487a, kshv-miR-K12-10a, kshv-miR-K12-10b, kshv-miR-K12-11, kshv-miR-K12-1, kshv-miR-K12-2, kshv-miR-K12-9*, kshv-miR-K12-9, kshv-miR-K12-8, kshv-miR-K12-7, kshv-miR-K12-6-5p, kshv-miR-K12-6-3p, kshv-miR-K12-5, kshv-miR-K12-4-5p, kshv-miR-K12-4-3p, kshv-miR-K12-3, kshv-miR-K12-3*, hsa-miR-488, hsa-miR-489, hsa-miR-490, hsa-miR-491, hsa-miR-511, hsa-miR-146b, hsa-miR-202*, hsa-miR-202, hsa-miR-492, hsa-miR-493-5p, hsa-miR-432, hsa-miR-432*, hsa-miR-494, hsa-miR-495, hsa-miR-496, hsa-miR-193b, hsa-miR-497, hsa-miR-181d, hsa-miR-512-5p, hsa-miR-512-3p, hsa-miR-498, hsa-miR-520e, hsa-miR-515-5p, hsa-miR-515-3p, hsa-miR-519e*, hsa-miR-519e, hsa-miR-520f, hsa-miR-526c, hsa-miR-519c, hsa-miR-520a*, hsa-miR-520a, hsa-miR-526b, hsa-miR-526b*, hsa-miR-519b, hsa-miR-525, hsa-miR-525*, hsa-miR-523, hsa-miR-518% hsa-miR-518f, hsa-miR-520b, hsa-miR-518b, hsa-miR-526a, hsa-miR-520c, hsa-miR-518c*, hsa-miR-518c, hsa-miR-524*, hsa-miR-524, hsa-miR-517*, hsa-miR-517a, hsa-miR-519d, hsa-miR-521, hsa-miR-520d*, hsa-miR-520d, hsa-miR-517b, hsa-miR-520g, hsa-miR-516-5p, hsa-miR-516-3p, hsa-miR-518e, hsa-miR-527, hsa-miR-518a, hsa-miR-518d, hsa-miR-517c, hsa-miR-520h, hsa-miR-522, hsa-miR-519a, hsa-miR-499, hsa-miR-500, hsa-miR-501, hsa-miR-502, hsa-miR-503, hsa-miR-504, hsa-miR-505, hsa-miR-513, hsa-miR-506, hsa-miR-507, hsa-miR-508, hsa-miR-509, hsa-miR-510, hsa-miR-514, hsa-miR-532, hsa-miR-299-5p, hsa-miR-18a*, hsa-miR-455, hsa-miR-493-3p, hsa-miR-539, hsa-miR-544, hsa-miR-545, hsa-miR-487b, hsa-miR-551a, hsa-miR-552, hsa-miR-553, hsa-miR-554, hsa-miR-92b, hsa-miR-555, hsa-miR-556, hsa-miR-557, hsa-miR-558, hsa-miR-559, hsa-miR-560, hsa-miR-561, hsa-miR-562, hsa-miR-563, hsa-miR-564, hsa-miR-565, hsa-miR-566, hsa-miR-567, hsa-miR-568, hsa-miR-551b, hsa-miR-569, hsa-miR-570, hsa-miR-571, hsa-miR-572, hsa-miR-573, hsa-miR-574, hsa-miR-575, hsa-miR-576, hsa-miR-577, hsa-miR-578, hsa-miR-579, hsa-miR-580, hsa-miR-581, hsa-miR-582, hsa-miR-583, hsa-miR-584, hsa-miR-585, hsa-miR-548a, hsa-miR-586, hsa-miR-587, hsa-miR-548b, hsa-miR-588, hsa-miR-589, hsa-miR-550, hsa-miR-590, hsa-miR-591, hsa-miR-592, hsa-miR-593, hsa-miR-595, hsa-miR-596, hsa-miR-597, hsa-miR-598, hsa-miR-599, hsa-miR-600, hsa-miR-601, hsa-miR-602, hsa-miR-603, hsa-miR-604, hsa-miR-605, hsa-miR-606, hsa-miR-607, hsa-miR-608, hsa-miR-609, hsa-miR-610, hsa-miR-611, hsa-miR-612, hsa-miR-613, hsa-miR-614, hsa-miR-615, hsa-miR-616, hsa-miR-548c, hsa-miR-617, hsa-miR-618, hsa-miR-619, hsa-miR-620, hsa-miR-621, hsa-miR-622, hsa-miR-623, hsa-miR-624, hsa-miR-625, hsa-miR-626, hsa-miR-627, hsa-miR-628, hsa-miR-629, hsa-miR-630, hsa-miR-631, hsa-miR-33b, hsa-miR-632, hsa-miR-633, hsa-miR-634, hsa-miR-635, hsa-miR-636, hsa-miR-637, hsa-miR-638, hsa-miR-639, hsa-miR-640, hsa-miR-641, hsa-miR-642, hsa-miR-643, hsa-miR-644, hsa-miR-645, hsa-miR-646, hsa-miR-647, hsa-miR-648, hsa-miR-649, hsa-miR-650, hsa-miR-651, hsa-miR-652, hsa-miR-548d, hsa-miR-661, hsa-miR-662, hsa-miR-663, hsa-miR-449b, hsa-miR-653, hsa-miR-411, hsa-miR-654, hsa-miR-655, hsa-miR-656, hsa-miR-549, hsa-miR-657, hsa-miR-658, hsa-miR-659, hsa-miR-660, hsa-miR-421, hsa-miR-542-5p, hcmv-miR-US4, hcmv-miR-UL70-5p, hcmv-miR-UL70-3p, hsa-miR-363*, hsa-miR-376a*, hsa-miR-542-3p, ebv-miR-BART1-3p, hsa-miR-425-5p, ebv-miR-BART3-5p, ebv-miR-BART3-3p, ebv-miR-BART4, ebv-miR-BART5, ebv-miR-BART6-5p, ebv-miR-BART6-3p, ebv-miR-BART7, ebv-miR-BART8-5p, ebv-miR-BART8-3p, ebv-miR-BART9, ebv-miR-BART10, ebv-miR-BART11-5p, ebv-miR-BART11-3p, ebv-miR-BART12, ebv-miR-BART13, ebv-miR-BART14-5p, ebv-miR-BART14-3p, kshv-miR-K12-12, ebv-miR-BART15, ebv-miR-BART16, ebv-miR-BART17-5p, ebv-miR-BART17-3p, ebv-miR-BART18, ebv-miR-BART19, ebv-miR-BART20-5p, ebv-miR-BART20-3p, hsv1-miR-H1, hsa-miR-758, hsa-miR-671, hsa-miR-668, hsa-miR-767-5p, hsa-miR-767-3p, hsa-miR-454-5p, hsa-miR-454-3p, hsa-miR-769-5p, hsa-miR-769-3p, hsa-miR-766, hsa-miR-765, hsa-miR-768-5p, hsa-miR-768-3p, hsa-miR-770-5p, hsa-miR-802, hsa-miR-801, and hsa-miR-675.","For example, without being bound by theory, miR-128A5 miR-129 and miR-128B are highly enriched in brain; miR-194, miR-148 and miR-192 are highly enriched in liver; mIR-96, miR-150, miR-205, miR-182 and miR-183 are highly enriched in the thymus; miR-204, miR-10B5 miR-154 and miR134 are highly enriched in testes; and miR-122, miR-210, miR-221, miR-141, miR-23A, miR-200C and miR-136 are highly enriched in the placenta. The biosignature comprising one or more of the aforementioned miRs can be used to distinguish positive and negative lymph nodes from a subject with cervical, colon or breast cancer.","In another embodiment, a biosignature can comprise one or more of the following miRs: miR-125b-1, miR125b-2, miR-145, miR-21, miR-155, miR-10b, miR-009-1 (miR131-1), miR-34 (miR-170), miR-102 (miR-29b), miR-123 (miR-126), miR-140-as, miR-125a, miR-125b-1, miR-125b-2, miR-194, miR-204, miR-213, let-7a-2, let-7a-3, let-7d (let-7d-v1), let-7f-2, let-71 (let-7d-v2), miR-101-1, miR-122a, miR-128b, miR-136, miR-143, miR-149, miR-191, miR-196-1, miR-196-2, miR-202, miR-203, miR-206, and miR-210, which can be used to characterize breast cancer.","In another embodiment, miR-375 expression is detected in a vesicle and used to characterize pancreatic insular or acinar tumors.","In yet another embodiment, one or more of the following miRs can be detected in a vesicle: miR-103-2, miR-107, miR-103-1, miR-342, miR-100, miR-24-2, miR-23a, miR-125a, miR-26a-1, miR-24-1, miR-191, miR-15a, miR-368, miR-26b, miR-125b-2, miR-125b-1, miR-26a-2, miR-335, miR-126. miR-1-2, miR-21, miR-25, miR-92-2, miR-130a, miR-93, miR-16-1, miR-145, miR-17, miR-99b, miR-181b-1, miR-146, miR-181b-2, miR-16-2, miR-99a, miR-197, miR-10a, miR-224, miR-92-1, miR-27a, miR-221, miR-320, miR-7-1, miR-29b-2, miR-150, miR-30d, miR-29a, miR-23b, miR-135a-2, miR-223, miR-3p21-v, miR-128b, miR-30b, miR-29b-1, miR-106b, miR-132, miR-214, miR-7-3, miR-29c, miR-367, miR-30c-2, miR-27b, miR-140, miR-10b, miR-20, miR-129-1, miR-340, miR-30a, miR-30c-1, miR-106a, miR-32, miR-95, miR-222, miR-30e, miR-129-2, miR-345, miR-143, miR-182, miR-1-1, miR-133a-1, miR-200c, miR-194-1, miR-210, miR-181c, miR-192, miR-220, miR-213, miR-323, and miR-375, wherein high expression or overexpression of the one or more miRs can be used to characterize pancreatic cancer.","Expression of one or more of the following miRs: miR-101, miR-126, miR-99a, miR-99-prec, miR-106, miR-339, miR-99b, miR-149, miR-33, miR-135 and miR-20 can be detected in a vesicle and used to characterize megakaryocytopoiesis.","It is believed cell proliferation has been correlated with the expression of miR-31, miR-92, miR-99a, miR-100, miR-125a, miR-129, miR-130a, miR-150, miR-187, miR-190, miR-191, miR-193, miR 204, miR-210, miR-21 1, miR-212, miR-213, miR-215, miR-216, miR-217, miR 218, miR-224, miR-292, miR-294, miR-320, miR-324, miR-325, miR-326, miR-330, miR-331, miR-338, miR-341, miR-369, miR-370, et-7a, Let-7b, Let-7c, Let-7d, Let-7g, miR-7, miR-9, miR-10a, miR-10b, miR-15a, miR-18, miR-19a, miR-17-3p, miR-20, miR-23b, miR-25, miR-26a, miR-26a, miR-30e-5p, miR-31, miR-32, miR-92, miR-93, miR-100, miR-125a, miR-125b, miR-126, miR-127, miR-128, miR-129, miR-130a, miR-135, miR-138, miR-139, miR-140, miR-141, miR-143, miR-145, miR-146, miR-150, miR-154, miR-155, miR-181a, miR-182, miR-186, miR-187, miR-188, miR-190, miR-191, miR-193, miR-194, miR-196, miR-197, miR-198, miR-199, miR-201, miR-204, miR-216, miR-218, miR-223, miR-293, miR-291-3p, miR-294, miR-295, miR-322, miR-333, miR-335, miR-338, miR-341, miR-350, miR-369, miR-373, miR-410, and miR-412. Detection one or more of the above miRs can be used to characterize a cancer.","Other examples of miRs that detected in a vesicle and used to characterize cancer is disclosed in U.S. Pat. No. 7,642,348, describing identification of 3,765 unique nucleic acid sequences correlated with prostate cancer), and U.S. Pat. No. 7,592,441, which describes microRNAs related to liver cancer.","Other microRNAs that are expressed commonly in solid cancer, such as colon cancer, lung cancer, breast cancer, stomach cancer, prostate cancer, and pancreatic cancer, can also be detected in a vesicle and used to characterize a cancer. For example, one or more of the following miRs: miR-21, miR-17-5p, miR-191, miR-29b-2, miR-223, miR-128b, miR-199a-1, miR-24-1, miR-24-2, miR-146, miR-155, miR-181b-1, miR-20a, miR-107, miR-32, miR-92-2, miR-214, miR-30c, miR-25, miR-221, and miR-106a, can be detected in a vesicle and used to characterize a solid cancer.","Other examples of microRNAs that can be detected in a vesicle are disclosed in PCT Publication Nos. WO2006126040, WO2006033020, WO2005116250, and WO2005111211, US Publications Nos. US20070042982 and US20080318210; and EP Publication Nos. EP1784501A2 and EP1751311A2, each of which is incorporated by reference.","Biomarker Detection","A biosignature can be detected qualitatively or quantitatively by detecting a presence, level or concentration of a microRNA, vesicle or other biomarkers, as disclosed herein. These biosignature components can be detected using a number of techniques known to those of skill in the art. For example, a biomarker can be detected by microarray analysis, polymerase chain reaction (PCR) (including PCR-based methods such as real time polymerase chain reaction (RT-PCR), quantitative real time polymerase chain reaction (Q-PCR\/qPCR) and the like), hybridization with allele-specific probes, enzymatic mutation detection, ligation chain reaction (LCR), oligonucleotide ligation assay (OLA), flow-cytometric heteroduplex analysis, chemical cleavage of mismatches, mass spectrometry, nucleic acid sequencing, single strand conformation polymorphism (SSCP), denaturing gradient gel electrophoresis (DGGE), temperature gradient gel electrophoresis (TGGE), restriction fragment polymorphisms, serial analysis of gene expression (SAGE), or combinations thereof. A biomarker, such as a nucleic acid, can be amplified prior to detection. A biomarker can also be detected by immunoassay, immunoblot, immunoprecipitation, enzyme-linked immunosorbent assay (ELISA; EIA), radioimmunoassay (RIA), flow cytometry, or electron microscopy (EM).","Biosignatures can be detected using capture agents and detection agents, as described herein. A capture agent can comprise an antibody, aptamer or other entity which recognizes a biomarker and can be used for capturing the biomarker. Biomarkers that can be captured include circulating biomarkers, e.g., a protein, nucleic acid, lipid or biological complex in solution in a bodily fluid. Similarly, the capture agent can be used for capturing a vesicle. A detection agent can comprise an antibody or other entity which recognizes a biomarker and can be used for detecting the biomarker vesicle, or which recognizes a vesicle and is useful for detecting a vesicle. In some embodiments, the detection agent is labeled and the label is detected, thereby detecting the biomarker or vesicle. The detection agent can be a binding agent, e.g., an antibody or aptamer. In other embodiments, the detection agent comprises a small molecule such as a membrane protein labeling agent. See, e.g., the membrane protein labeling agents disclosed in Alroy et al., US. Patent Publication US 2005\/0158708. In an embodiment, vesicles are isolated or captured as described herein, and one or more membrane protein labeling agent is used to detect the vesicles. In many cases, the antigen or other vesicle-moiety that is recognized by the capture and detection agents are interchangeable. As a non-limiting example, consider a vesicle having a cell-of-origin specific antigen on its surface and a cancer-specific antigen on its surface. In one instance, the vesicle can be captured using an antibody to the cell-of-origin specific antigen, e.g., by tethering the capture antibody to a substrate, and then the vesicle is detected using an antibody to the cancer-specific antigen, e.g., by labeling the detection antibody with a fluorescent dye and detecting the fluorescent radiation emitted by the dye. In another instance, the vesicle can be captured using an antibody to the cancer specific antigen, e.g., by tethering the capture antibody to a substrate, and then the vesicle is detected using an antibody to the cell-of-origin specific antigen, e.g., by labeling the detection antibody with a fluorescent dye and detecting the fluorescent radiation emitted by the dye.","In some embodiments, a same biomarker is recognized by both a capture agent and a detection agent. This scheme can be used depending on the setting. In one embodiment, the biomarker is sufficient to detect a vesicle of interest, e.g., to capture cell-of-origin specific vesicles. In other embodiments, the biomarker is multifunctional, e.g., having both cell-of-origin specific and cancer specific properties. The biomarker can be used in concert with other biomarkers for capture and detection as well.","One method of detecting a biomarker comprises purifying or isolating a heterogeneous population of vesicles from a biological sample, as described above, and performing a sandwich assay. A vesicle in the population can be captured with a capture agent. The capture agent can be a capture antibody, such as a primary antibody. The capture antibody can be bound to a substrate, for example an array, well, or particle. The captured or bound vesicle can be detected with a detection agent, such as a detection antibody. For example, the detection antibody can be for an antigen of the vesicle. The detection antibody can be directly labeled and detected. Alternatively, the detection agent can be indirectly labeled and detected, such as through an enzyme linked secondary antibody that can react with the detection agent. A detection reagent or detection substrate can be added and the reaction detected, such as described in PCT Publication No. WO2009092386. In an illustrative example wherein the capture agent binds Rab-5b and the detection agent binds or detects CD63 or caveolin-1, the capture agent can be an anti-Rab 5b antibody and the detection agent can be an anti-CD63 or anti-caveolin-1 antibody. In some embodiments, the capture agent binds CD9, PSCA, TNFR, CD63, B7H3, MFG-E8, EpCam, Rab, CD81, STEAP, PCSA, PSMA, or 5T4. For example, the capture agent can be an antibody to CD9, PSCA, TNFR, CD63, B7H3, MFG-E8, EpCam, Rab, CD81, STEAP, PCSA, PSMA, or 5T4. The capture agent can also be an antibody to MFG-E8, Annexin V, Tissue Factor, DR3, STEAP, epha2, TMEM211, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, or TETS. The detection agent can be an agent that binds or detects CD63, CD9, CD81, B7H3, or EpCam, such as a detection antibody to CD63, CD9, CD81, B7H3, or EpCam. Various combinations of capture and\/or detection agents can be used in concert. In an embodiment, the capture agents comprise PCSA, PSMA, B7H3 and optionally EpCam, and the detection agents comprise one or more tetraspanin such as CD9, CD63 and CD81. In another embodiment, the capture agents comprise TMEM211 and CD24, and the detection agents comprise one or more tetraspanin such as CD9, CD63 and CD81. In another embodiment, the capture agents comprise CD66 and EpCam, and the detection agents comprise one or more tetraspanin such as CD9, CD63 and CD81. Increasing numbers of such tetraspanins and\/or other general vesicle markers can improve the detection signal in some cases. Proteins or other circulating biomarkers can also be detected using sandwich approaches. The captured vesicles can be collected and used to analyze the payload contained therein, e.g., mRNA, microRNAs, DNA and soluble protein.","In some embodiments, the capture agent binds or targets EpCam, B7H3 or CD24, and the one or more biomarkers detected on the vesicle are CD9 and\/or CD63. In one embodiment, the capture agent binds or targets EpCam, and the one or more biomarkers detected on the vesicle are CD9, EpCam and\/or CD81. The single capture agent can be selected from CD9, PSCA, TNFR, CD63, B7H3, MFG-E8, EpCam, Rab, CD81, STEAP, PCSA, PSMA, or 5T4. The single capture agent can also be an antibody to DR3, STEAP, epha2, TMEM211, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, MFG-E8, TF, Annexin V or TETS. In some embodiments, the single capture agent is selected from PCSA, PSMA, B7H3, CD81, CD9 and CD63.","In other embodiments, the capture agent targets PCSA, and the one or more biomarkers detected on the captured vesicle are B7H3 and\/or PSMA. In other embodiments, the capture agent targets PSMA, and the one or more biomarkers detected on the captured vesicle are B7H3 and\/or PCSA. In other embodiments, the capture agent targets B7H3, and the one or more biomarkers detected on the captured vesicle are PSMA and\/or PCSA. In yet other embodiments, the capture agent targets CD63 and the one or more biomarkers detected on the vesicle are CD81, CD83, CD9 and\/or CD63. The different capture agent and biomarker combinations disclosed herein can be used to characterize a phenotype, such as detecting, diagnosing or prognosing a disease, e.g., a cancer. In some embodiments, vesicles are analyzed to characterize prostate cancer using a capture agent targeting EpCam and detection of CD9 and CD63; a capture agent targeting PCSA and detection of B7H3 and PSMA; or a capture agent of CD63 and detection of CD81. In other embodiments, vesicles are used to characterize colon cancer using capture agent targeting CD63 and detection of CD63, or a capture agent targeting CD9 coupled with detection of CD63. One of skill will appreciate that targets of capture agents and detection agents can be used interchangeably. In an illustrative example, consider a capture agent targeting PCSA and detection agents targeting B7H3 and PSMA. Because all of these markers are useful for detecting PCa derived vesicles, B7H3 or PSMA could be targeted by the capture agent and PCSA could be recognized by a detection agent. For example, in some embodiments, the detection agent targets PCSA, and one or more biomarkers used to capture the vesicle comprise B7H3 and\/or PSMA. In other embodiments, the detection agent targets PSMA, and the one or more biomarkers used to capture the vesicle comprise B7H3 and\/or PCSA. In other embodiments, the detection agent targets B7H3, and the one or more biomarkers used to capture the vesicle comprise PSMA and\/or PCSA. In some embodiments, the invention provides a method of detecting prostate cancer cells in bodily fluid using capture agents and\/or detection agents to PSMA, B7H3 and\/or PCSA. The bodily fluid can comprise blood, including serum or plasma. The bodily fluid can comprise ejaculate or sperm. In further embodiments, the methods of detecting prostate cancer further use capture agents and\/or detection agents to CD81, CD83, CD9 and\/or CD63. The method further provides a method of characterizing a GI disorder, comprising capturing vesicles with one or more of DR3, STEAP, epha2, TMEM211, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, and TETS, and detecting the captured vesicles with one or more general vesicle antigen, such as CD81, CD63 and\/or CD9. Additional agents can improve the test performance, e.g., improving test accuracy or AUC, either by providing additional biological discriminatory power and\/or by reducing experimental noise.","Techniques of detecting biomarkers for use with the invention include the use of a planar substrate such as an array (e.g., biochip or microarray), with molecules immobilized to the substrate as capture agents that facilitate the detection of a particular biosignature. The array can be provided as part of a kit for assaying one or more biomarkers or vesicles. A molecule that identifies the biomarkers described above and shown in , as well as antigens in , can be included in an array for detection and diagnosis of diseases including presymptomatic diseases. In some embodiments, an array comprises a custom array comprising biomolecules selected to specifically identify biomarkers of interest. Customized arrays can be modified to detect biomarkers that increase statistical performance, e.g., additional biomolecules that identifies a biosignature which lead to improved cross-validated error rates in multivariate prediction models (e.g., logistic regression, discriminant analysis, or regression tree models). In some embodiments, customized array(s) are constructed to study the biology of a disease, condition or syndrome and profile biosignatures in defined physiological states. Markers for inclusion on the customized array be chosen based upon statistical criteria, e.g., having a desired level of statistical significance in differentiating between phenotypes or physiological states. In some embodiments, standard significance of p-value=0.05 is chosen to exclude or include biomolecules on the microarray. The p-values can be corrected for multiple comparisons. As an illustrative example, nucleic acids extracted from samples from a subject with or without a disease can be hybridized to a high density microarray that binds to thousands of gene sequences. Nucleic acids whose levels are significantly different between the samples with or without the disease can be selected as biomarkers to distinguish samples as having the disease or not. A customized array can be constructed to detect the selected biomarkers. In some embodiments, customized arrays comprise low density microarrays, which refer to arrays with lower number of addressable binding agents, e.g., tens or hundreds instead of thousands. Low density arrays can be formed on a substrate. In some embodiments, customizable low density arrays use PCR amplification in plate wells, e.g., TaqMan\u00ae Gene Expression Assays (Applied Biosystems by Life Technologies Corporation, Carlsbad, Calif.).","A planar array generally contains addressable locations (e.g., pads, addresses, or micro-locations) of biomolecules in an array format. The size of the array will depend on the composition and end use of the array. Arrays can be made containing from 2 different molecules to many thousands. Generally, the array comprises from two to as many as 100,000 or more molecules, depending on the end use of the array and the method of manufacture. A microarray for use with the invention comprises at least one biomolecule that identifies or captures a biomarker present in a biosignature of interest, e.g., a microRNA or other biomolecule or vesicle that makes up the biosignature. In some arrays, multiple substrates are used, either of different or identical compositions. Accordingly, planar arrays may comprise a plurality of smaller substrates.","The present invention can make use of many types of arrays for detecting a biomarker, e.g., a biomarker associated with a biosignature of interest. Useful arrays or microarrays include without limitation DNA microarrays, such as cDNA microarrays, oligonucleotide microarrays and SNP microarrays, microRNA arrays, protein microarrays, antibody microarrays, tissue microarrays, cellular microarrays (also called transfection microarrays), chemical compound microarrays, and carbohydrate arrays (glycoarrays). These arrays are described in more detail above. In some embodiments, microarrays comprise biochips that provide high-density immobilized arrays of recognition molecules (e.g., antibodies), where biomarker binding is monitored indirectly (e.g., via fluorescence).  shows an illustrative configuration in which capture antibodies against a vesicle antigen of interest are tethered to a surface. The captured vesicles are then detected using detector antibodies against the same or different vesicle antigens of interest. The capture antibodies can be substituted with tethered aptamers as available and desirable. Fluorescent detectors are shown. Other detectors can be used similarly, e.g., enzymatic reaction, detectable nanoparticles, radiolabels, and the like. In other embodiments, an array comprises a format that involves the capture of proteins by biochemical or intermolecular interaction, coupled with detection by mass spectrometry (MS). The vesicles can be eluted from the surface and the payload therein, e.g., microRNA, can be analyzed.","An array or microarray that can be used to detect one or more biomarkers of a biosignature can be made according to the methods described in U.S. Pat. Nos. 6,329,209; 6,365,418; 6,406,921; 6,475,808; and 6,475,809, and U.S. patent application Ser. No. 10\/884,269, each of which is herein incorporated by reference in its entirety. Custom arrays to detect specific selections of sets of biomarkers described herein can be made using the methods described in these patents. Commercially available microarrays can also be used to carry out the methods of the invention, including without limitation those from Affymetrix (Santa Clara, Calif.), Illumina (San Diego, Calif.), Agilent (Santa Clara, Calif.), Exiqon (Denmark), or Invitrogen (Carlsbad, Calif.). Custom and\/or commercial arrays include arrays for detection proteins, nucleic acids, and other biological molecules and entities (e.g., cells, vesicles, virii) as described herein.","In some embodiments, molecules to be immobilized on an array comprise proteins or peptides. One or more types of proteins may be immobilized on a surface. In certain embodiments, the proteins are immobilized using methods and materials that minimize the denaturing of the proteins, that minimize alterations in the activity of the proteins, or that minimize interactions between the protein and the surface on which they are immobilized.","Array surfaces useful may be of any desired shape, form, or size. Non-limiting examples of surfaces include chips, continuous surfaces, curved surfaces, flexible surfaces, films, plates, sheets, or tubes. Surfaces can have areas ranging from approximately a square micron to approximately 500 cm. The area, length, and width of surfaces may be varied according to the requirements of the assay to be performed. Considerations may include, for example, ease of handling, limitations of the material(s) of which the surface is formed, requirements of detection systems, requirements of deposition systems (e.g., arrayers), or the like.","In certain embodiments, it is desirable to employ a physical means for separating groups or arrays of binding islands or immobilized biomolecules: such physical separation facilitates exposure of different groups or arrays to different solutions of interest. Therefore, in certain embodiments, arrays are situated within microwell plates having any number of wells. In such embodiments, the bottoms of the wells may serve as surfaces for the formation of arrays, or arrays may be formed on other surfaces and then placed into wells. In certain embodiments, such as where a surface without wells is used, binding islands may be formed or molecules may be immobilized on a surface and a gasket having holes spatially arranged so that they correspond to the islands or biomolecules may be placed on the surface. Such a gasket is preferably liquid tight. A gasket may be placed on a surface at any time during the process of making the array and may be removed if separation of groups or arrays is no longer necessary.","In some embodiments, the immobilized molecules can bind to one or more biomarkers or vesicles present in a biological sample contacting the immobilized molecules. In some embodiments, the immobilized molecules modify or are modified by molecules present in the one or more vesicles contacting the immobilized molecules. Contacting the sample typically comprises overlaying the sample upon the array.","Modifications or binding of molecules in solution or immobilized on an array can be detected using detection techniques known in the art. Examples of such techniques include immunological techniques such as competitive binding assays and sandwich assays; fluorescence detection using instruments such as confocal scanners, confocal microscopes, or CCD-based systems and techniques such as fluorescence, fluorescence polarization (FP), fluorescence resonant energy transfer (FRET), total internal reflection fluorescence (TIRF), fluorescence correlation spectroscopy (FCS); colorimetric\/spectrometric techniques; surface plasmon resonance, by which changes in mass of materials adsorbed at surfaces are measured; techniques using radioisotopes, including conventional radioisotope binding and scintillation proximity assays (SPA); mass spectroscopy, such as matrix-assisted laser desorption\/ionization mass spectroscopy (MALDI) and MALDI-time of flight (TOF) mass spectroscopy; ellipsometry, which is an optical method of measuring thickness of protein films; quartz crystal microbalance (QCM), a very sensitive method for measuring mass of materials adsorbing to surfaces; scanning probe microscopies, such as atomic force microscopy (AFM), scanning force microscopy (SFM) or scanning electron microscopy (SEM); and techniques such as electrochemical, impedance, acoustic, microwave, and IR\/Raman detection. See, e.g., Mere L, et al., \u201c--4(8):363-369 (1999), and references cited therein; Lakowicz JR, 2(1999), or Jain K K: V, ed., ed. 1: Totowa, N.J.: Humana Press, 2007, each of which is herein incorporated by reference in its entirety.","Microarray technology can be combined with mass spectroscopy (MS) analysis and other tools. Electrospray interface to a mass spectrometer can be integrated with a capillary in a microfluidics device. For example, one commercially available system contains eTag reporters that are fluorescent labels with unique and well-defined electrophoretic mobilities; each label is coupled to biological or chemical probes via cleavable linkages. The distinct mobility address of each eTag reporter allows mixtures of these tags to be rapidly deconvoluted and quantitated by capillary electrophoresis. This system allows concurrent gene expression, protein expression, and protein function analyses from the same sample Jain K K: , ed., ed. 1: Totowa, N.J.: 2007, which is herein incorporated by reference in its entirety.","A biochip can include components for a microfluidic or nanofluidic assay. A microfluidic device can be used for isolating or analyzing biomarkers, such as determining a biosignature. Microfluidic systems allow for the miniaturization and compartmentalization of one or more processes for isolating, capturing or detecting a vesicle, detecting a microRNA, detecting a circulating biomarker, detecting a biosignature, and other processes. The microfluidic devices can use one or more detection reagents in at least one aspect of the system, and such a detection reagent can be used to detect one or more biomarkers. In one embodiment, the device detects a biomarker on an isolated or bound vesicle. Various probes, antibodies, proteins, or other binding agents can be used to detect a biomarker within the microfluidic system. The detection agents may be immobilized in different compartments of the microfluidic device or be entered into a hybridization or detection reaction through various channels of the device.","A vesicle in a microfluidic device can be lysed and its contents detected within the microfluidic device, such as proteins or nucleic acids, e.g., DNA or RNA such as miRNA or mRNA. The nucleic acid may be amplified prior to detection, or directly detected, within the microfluidic device. Thus microfluidic system can also be used for multiplexing detection of various biomarkers. In an embodiment, vesicles are captured within the microfluidic device, the captured vesicles are lysed, and a biosignature of microRNA from the vesicle payload is determined. The biosignature can further comprise the capture agent used to capture the vesicle.","Novel nanofabrication techniques are opening up the possibilities for biosensing applications that rely on fabrication of high-density, precision arrays, e.g., nucleotide-based chips and protein arrays otherwise know as heterogeneous nanoarrays. Nanofluidics allows a further reduction in the quantity of fluid analyte in a microchip to nanoliter levels, and the chips used here are referred to as nanochips. (See, e.g., Unger M et al., 1999; 27(5):1008-14, Kartalov E P et al., 2006; 40(1):85-90, each of which are herein incorporated by reference in their entireties.) Commercially available nanochips currently provide simple one step assays such as total cholesterol, total protein or glucose assays that can be run by combining sample and reagents, mixing and monitoring of the reaction. Gel-free analytical approaches based on liquid chromatography (LC) and nanoLC separations (Cutillas et al. 2005; 5:101-112 and Cutillas et al., 2005; 4:1038-1051, each of which is herein incorporated by reference in its entirety) can be used in combination with the nanochips.","Further provided herein is a rapid detection device that facilitates the detection of a particular biosignature in a biological sample. The device can integrate biological sample preparation with polymerase chain reaction (PCR) on a chip. The device can facilitate the detection of a particular biosignature of a vesicle in a biological sample, and an example is provided as described in Pipper et al., 47(21), p. 3900-3904 (2008), which is herein incorporated by reference in its entirety. A biosignature can be incorporated using micro-\/nano-electrochemical system (MEMS\/NEMS) sensors and oral fluid for diagnostic applications as described in Li et al., 18(1): 3-5 (2005), which is herein incorporated by reference in its entirety.","As an alternative to planar arrays, assays using particles, such as bead based assays as described herein, can be used in combination with flow cytometry. Multiparametric assays or other high throughput detection assays using bead coatings with cognate ligands and reporter molecules with specific activities consistent with high sensitivity automation can be used. In a bead based assay system, a binding agent for a biomarker or vesicle, such as a capture agent (e.g. capture antibody), can be immobilized on an addressable microsphere. Each binding agent for each individual binding assay can be coupled to a distinct type of microsphere (i.e., microbead) and the assay reaction takes place on the surface of the microsphere, such as depicted in . A binding agent for a vesicle can be a capture antibody coupled to a bead. Dyed microspheres with discrete fluorescence intensities are loaded separately with their appropriate binding agent or capture probes. The different bead sets carrying different binding agents can be pooled as necessary to generate custom bead arrays. Bead arrays are then incubated with the sample in a single reaction vessel to perform the assay. Examples of microfluidic devices that may be used, or adapted for use with the invention, include but are not limited to those described herein.","Product formation of the biomarker with an immobilized capture molecule or binding agent can be detected with a fluorescence based reporter system (see for example, ). The biomarker can either be labeled directly by a fluorophore or detected by a second fluorescently labeled capture biomolecule. The signal intensities derived from captured biomarkers can be measured in a flow cytometer. The flow cytometer can first identify each microsphere by its individual color code. For example, distinct beads can be dyed with discrete fluorescence intensities such that each bead with a different intensity has a different binding agent. The beads can be labeled or dyed with at least 2 different labels or dyes. In some embodiments, the beads are labeled with at least 3, 4, 5, 6, 7, 8, 9, or 10 different labels. The beads with more than one label or dye can also have various ratios and combinations of the labels or dyes. The beads can be labeled or dyed externally or may have intrinsic fluorescence or signaling labels.","The amount of captured biomarkers on each individual bead can be measured by the second color fluorescence specific for the bound target. This allows multiplexed quantitation of multiple targets from a single sample within the same experiment. Sensitivity, reliability and accuracy are compared or can be improved to standard microtiter ELISA procedures. An advantage of a bead-based system is the individual coupling of the capture biomolecule or binding agent for a vesicle to distinct microspheres provides multiplexing capabilities. For example, as depicted in , a combination of 5 different biomarkers to be detected (detected by antibodies to antigens such as CD63, CD9, CD81, B7H3, and EpCam) and 20 biomarkers for which to capture a vesicle, (using capture antibodies, such as antibodies to CD9, PSCA, TNFR, CD63, B7H3, MFG-E8, EpCam, Rab, CD81, STEAP, PCSA, PSMA, 5T4, and\/or CD24) can result in approximately 100 combinations to be detected. As shown in  as \u201cEpCam 2\u00d7,\u201d \u201cCD63 2\u00d7,\u201d multiple antibodies to a single target can be used to probe detection against various epitopes. In another example, multiplex analysis comprises capturing a vesicle using a binding agent to CD24 and detecting the captured vesicle using a binding agent for CD9, CD63, and\/or CD81. The captured vesicles can be detected using a detection agent such as an antibody. The detection agents can be labeled directly or indirectly, as described herein.","Multiplexing of at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 different biomarkers may be performed. For example, an assay of a heterogeneous population of vesicles can be performed with a plurality of particles that are differentially labeled. There can be at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 differentially labeled particles. The particles may be externally labeled, such as with a tag, or they may be intrinsically labeled. Each differentially labeled particle can be coupled to a capture agent, such as a binding agent, for a vesicle, resulting in capture of a vesicle. The multiple capture agents can be selected to characterize a phenotype of interest, including capture agents against general vesicle biomarkers, cell-of-origin specific biomarkers, and disease biomarkers. One or more biomarkers of the captured vesicle can then be detected by a plurality of binding agents. The binding agent can be directly labeled to facilitate detection. Alternatively, the binding agent is labeled by a secondary agent. For example, the binding agent may be an antibody for a biomarker on the vesicle. The binding agent is linked to biotin. A secondary agent comprises streptavidin linked to a reporter and can be added to detect the biomarker. In some embodiments, the captured vesicle is assayed for at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 different biomarkers. For example, multiple detectors, i.e., detection of multiple biomarkers of a captured vesicle or population of vesicles, can increase the signal obtained, permitted increased sensitivity, specificity, or both, and the use of smaller amounts of samples. For example, detection with more than one general vesicle marker can improve the signal as compared to using a lesser number of detection markers, such as a single marker. To illustrate, detection of vesicles with labeled binding agents to two or three of CD9, CD63 and CD81 can improve the signal compared to detection with any one of the tetraspanins individually.","An immunoassay based method or sandwich assay can also be used to detect a biomarker of a vesicle. An example includes ELISA. A binding agent or capture agent can be bound to a well. For example an antibody to an antigen of a vesicle can be attached to a well. A biomarker on the captured vesicle can be detected based on the methods described herein.  shows an illustrative schematic for a sandwich-type of immunoassay. The capture antibody can be against a vesicle antigen of interest, e.g., a general vesicle biomarker, a cell-of-origin marker, or a disease marker. In the figure, the captured vesicles are detected using fluorescently labeled antibodies against vesicle antigens of interest. Multiple capture antibodies can be used, e.g., in distinguishable addresses on an array or different wells of an immunoassay plate. The detection antibodies can be against the same antigen as the capture antibody, or can be directed against other markers. The capture antibodies can be substituted with alternate binding agents, such as tethered aptamers or lectins, and\/or the detector antibodies can be similarly substituted, e.g., with detectable (e.g., labeled) aptamers, lectins or other binding proteins or entities. In an embodiment, one or more capture agents to a general vesicle biomarker, a cell-of-origin marker, and\/or a disease marker are used along with detection agents against general vesicle biomarker, such as tetraspanin molecules including without limitation one or more of CD9, CD63 and CD81.",{"@attributes":{"id":"p-0810","num":"0809"},"figref":["FIG. 63D","FIG. 63D"],"b":["6300","6301","6302","6301","6303","6300","6301","6302","6303","6300","6302","6301","6302","6303","6300","6301","6302"]},"In the scheme shown in  ii), a population of vesicles is captured with one or more capture agents against cell-of-origin biomarkers () and\/or disease biomarkers (). The captured vesicles are then detected using detectors against general vesicle biomarkers (). If only cell-of-origin capture agents are used (), the biosignature used to characterize the phenotype () can include the cell-of-origin biomarkers () and the general vesicle markers (). If only disease biomarker capture agents are used (), the biosignature used to characterize the phenotype () can include the disease biomarkers () and the general vesicle biomarkers (). Alternately, capture agents to one or more cell-of-origin biomarkers () and one or more disease specific biomarkers () are used to capture vesicles. In this case, the biosignature used to characterize the phenotype () can include the cell-of-origin biomarkers (), the disease biomarkers (), and the general vesicle markers (). The biomarkers combinations are selected to characterize the phenotype of interest and can be selected from the biomarkers and phenotypes described herein.","Biomarkers comprising vesicle payload can be analyzed to characterize a phenotype. Payload comprises the biological entities contained within a vesicle membrane. These entities include without limitation nucleic acids, e.g., mRNA, microRNA, or DNA fragments; protein, e.g., soluble and membrane associated proteins; carbohydrates; lipids; metabolites; and various small molecules, e.g., hormones. The payload can be part of the cellular milieu that is encapsulated as a vesicle is formed in the cellular environment. In some embodiments of the invention, the payload is analyzed in addition to detecting vesicle surface antigens. Specific populations of vesicles can be captured as described above then the payload in the captured vesicles can be used to characterize a phenotype. For example, vesicles captured on a substrate can be further isolated to assess the payload therein. Alternately, the vesicles in a sample are detected and sorted without capture. The vesicles so detected can be further isolated to assess the payload therein. In an embodiment, vesicle populations are sorted by flow cytometry and the payload in the sorted vesicles is analyzed. In the scheme shown in  iii), a population of vesicles is captured and\/or detected () using one or more of cell-of-origin biomarkers (), disease biomarkers (), and general vesicle markers (). The payload of the isolated vesicles is assessed (). A biosignature detected within the payload can be used to characterize a phenotype (). In a non-limiting example, a vesicle population can be analyzed in a plasma sample from a patient using antibodies against one or more vesicle antigens of interest. The antibodies can be capture antibodies which are tethered to a substrate to isolate a desired vesicle population. Alternately, the antibodies can be directly labeled and the labeled vesicles isolated by sorting with flow cytometry. The presence or level of microRNA or mRNA extracted from the isolated vesicle population can be used to detect a biosignature. The biosignature is then used to diagnose, prognose or theranose the patient.","In other embodiments, vesicle payload is analyzed in a vesicle population without first capturing or detected subpopulations of vesicles. For example, vesicles can be generally isolated from a sample using centrifugation, filtration, chromatography, or other techniques as described herein. The payload of the isolated vesicles can be analyzed thereafter to detect a biosignature and characterize a phenotype. In the scheme shown in  iv), a population of vesicles is isolated () and the payload of the isolated vesicles is assessed (). A biosignature detected within the payload can be used to characterize a phenotype (). In a non-limiting example, a vesicle population is isolated from a plasma sample from a patient using size exclusion and membrane filtration. The presence or level of microRNA or mRNA extracted from the vesicle population is used to detect a biosignature. The biosignature is then used to diagnose, prognose or theranose the patient.","A peptide or protein biomarker can be analyzed by mass spectrometry or flow cytometry. Proteomic analysis of a vesicle may be carried out by immunocytochemical staining, Western blotting, electrophoresis, SDS-PAGE, chromatography, x-ray crystallography or other protein analysis techniques in accordance with procedures well known in the art. In other embodiments, the protein biosignature of a vesicle may be analyzed using 2 D differential gel electrophoresis as described in, Chromy et al. 2004; 3:1120-1127, which is herein incorporated by reference in its entirety, or with liquid chromatography mass spectrometry as described in Zhang et al. 2005; 4:144-155, which is herein incorporated by reference in its entirety. A vesicle may be subjected to activity-based protein profiling described for example, in Berger et al., 2004; 4:371-381, which is in incorporated by reference in its entirety. In other embodiments, a vesicle may be profiled using nanospray liquid chromatography-tandem mass spectrometry as described in Pisitkun et al., 2004; 101:13368-13373, which is herein incorporated by reference in its entirety. In another embodiment, the vesicle may be profiled using tandem mass spectrometry (MS) such as liquid chromatography\/MS\/MS (LC-MS\/MS) using for example a LTQ and LTQ-FT ion trap mass spectrometer. Protein identification can be determined and relative quantitation can be assessed by comparing spectral counts as described in Smalley et al., 2008; 7:2088-2096, which is herein incorporated by reference in its entirety.","The expression of circulating protein biomarkers or protein payload within a vesicle can also be identified. The latter analysis can optionally follow the isolation of specific vesicles using capture agents to capture populations of interest. In an embodiment, immunocytochemical staining is used to analyze protein expression. The sample can be resuspended in buffer, centrifuged at 100\u00d7g for example, for 3 minutes using a cytocentrifuge on adhesive slides in preparation for immunocytochemical staining. The cytospins can be air-dried overnight and stored at \u221280\u00b0 C. until staining. Slides can then be fixed and blocked with serum-free blocking reagent. The slides can then be incubated with a specific antibody to detect the expression of a protein of interest. In some embodiments, the vesicles are not purified, isolated or concentrated prior to protein expression analysis.","Biosignatures comprising vesicle payload can be characterized by analysis of a metabolite marker or metabolite within the vesicle. Various metabolite-oriented approaches have been described such as metabolite target analyses, metabolite profiling, or metabolic fingerprinting, see for example, Denkert et al., 2008; 7: 4598-4617, Ellis et al., 2006; 8: 875-885, Kuhn et al., 2007; 24: 7401-7406, Fiehn O., 2001; 2:155-168, Fancy et al., 20(15): 2271-80 (2006), Lindon et al., 23(6): 1075-88 (2006), Holmes et al., 2007 Apr. 1; 79(7):2629-40. Epub 2007 Feb. 27. Erratum in: 2008 Aug. 1; 80(15):6142-3, Stanley et al., 2005 Aug. 15; 343(2): 195-202., Lehtim\u00e4ki et al., 2003 Nov. 14; 278(46):45915-23, each of which is herein incorporated by reference in its entirety.","Peptides can be analyzed by systems described in Jain K K: . Volume 1: Totowa, N.J.: 2007, which is herein incorporated by reference in its entirety. This system can generate sensitive molecular fingerprints of proteins present in a body fluid as well as in vesicles. Commercial applications which include the use of chromatography\/mass spectroscopy and reference libraries of all stable metabolites in the human body, for example Paradigm Genetic's Human Metabolome Project, may be used to determine a metabolite biosignature. Other methods for analyzing a metabolic profile can include methods and devices described in U.S. Pat. No. 6,683,455 (Metabometrix), U.S. Patent Application Publication Nos. 20070003965 and 20070004044 (Biocrates Life Science), each of which is herein incorporated by reference in its entirety. Other proteomic profiling techniques are described in Kennedy, 120:379-384 (2001), Berven et al., 7(3): 147-58 (2006), Conrads et al., 2(5): 693-703, Decramer et al., 25(5): 457-65 (2007), Decramer et al., 7(10): 1850-62 (2008), Decramer et al., 160: 127-41 (2008), Diamandis, 5(9): 2079-82 (2006), Immler et al., 6(10): 2947-58 (2006), Khan et al., 5(10): 2824-38 (2006), Kumar et al., 11(5): 385-405 (2006), Noble et al., 104(2): 191-6 (2007), Omenn, 20(3): 131-4 (2004), Powell et al., 3(1): 63-74 (2006), Rai et al., 126(12): 1518-26 (2002), Ramstrom et al., 3(2): 184-90 (2003), Tammen et al., 79(1): 83-93 (2003), Theodorescu et al., 7(3): 230-40 (2006), or Zurbig et al., 27(11): 2111-25 (2006).","For analysis of mRNAs, miRNAs or other small RNAs, the total RNA can be isolated using any known methods for isolating nucleic acids such as methods described in U.S. Patent Application Publication No. 2008132694, which is herein incorporated by reference in its entirety. These include, but are not limited to, kits for performing membrane based RNA purification, which are commercially available. Generally, kits are available for the small-scale (30 mg or less) preparation of RNA from cells and tissues, for the medium scale (250 mg tissue) preparation of RNA from cells and tissues, and for the large scale (1 g maximum) preparation of RNA from cells and tissues. Other commercially available kits for effective isolation of small RNA-containing total RNA are available. Such methods can be used to isolate nucleic acids from vesicles.","Alternatively, RNA can be isolated using the method described in U.S. Pat. No. 7,267,950, which is herein incorporated by reference in its entirety. U.S. Pat. No. 7,267,950 describes a method of extracting RNA from biological systems (cells, cell fragments, organelles, tissues, organs, or organisms) in which a solution containing RNA is contacted with a substrate to which RNA can bind and RNA is withdrawn from the substrate by applying negative pressure. Alternatively, RNA may be isolated using the method described in U.S. Patent Application No. 20050059024, which is herein incorporated by reference in its entirety, which describes the isolation of small RNA molecules. Other methods are described in U.S. Patent Application Nos. 20050208510, 20050277121, 20070238118, each of which is incorporated by reference in its entirety.","In one embodiment, mRNA expression analysis can be carried out on mRNAs from a vesicle isolated from a sample. In some embodiments, the vesicle is a cell-of-origin specific vesicle. An expression pattern generated from a vesicle can be indicative of a given disease state, disease stage, therapy related signature, or physiological condition.","In one embodiment, once the total RNA has been isolated, cDNA can be synthesized and either qRT-PCR assays (e.g. Applied Biosystem's Taqman\u00ae assays) for specific mRNA targets can be performed according to manufacturer's protocol, or an expression microarray can be performed to look at highly multiplexed sets of expression markers in one experiment. Methods for establishing gene expression profiles include determining the amount of RNA that is produced by a gene that can code for a protein or peptide. This can be accomplished by quantitative reverse transcriptase PCR (qRT-PCR), competitive RT-PCR, real time RT-PCR, differential display RT-PCR, Northern Blot analysis or other related tests. While it is possible to conduct these techniques using individual PCR reactions, it is also possible to amplify complementary DNA (cDNA) or complementary RNA (cRNA) produced from mRNA and analyze it via microarray.","The level of a miRNA product in a sample can be measured using any appropriate technique that is suitable for detecting mRNA expression levels in a biological sample, including but not limited to Northern blot analysis, RT-PCR, qRT-PCR, in situ hybridization or microarray analysis. For example, using gene specific primers and target cDNA, qRT-PCR enables sensitive and quantitative miRNA measurements of either a small number of target miRNAs (via singleplex and multiplex analysis) or the platform can be adopted to conduct high throughput measurements using 96-well or 384-well plate formats. See for example, Ross J S et al, 2008 May; 13(5):477-93, which is herein incorporated by reference in its entirety. A number of different array configurations and methods for microarray production are known to those of skill in the art and are described in U.S. patents such as: U.S. Pat. Nos. 5,445,934; 5,532,128; 5,556,752; 5,242,974; 5,384,261; 5,405,783; 5,412,087; 5,424,186; 5,429,807; 5,436,327; 5,472,672; 5,527,681; 5,529,756; 5,545,531; 5,554,501; 5,561,071; 5,571,639; 5,593,839; 5,599,695; 5,624,711; 5,658,734; or 5,700,637; each of which is herein incorporated by reference in its entirety. Other methods of profiling miRNAs are described in Taylor et al., 2008 July; 110(1): 13-21, Gilad et al, 2008 Sep. 5; 3(9):e3148, Lee et al., 2008 Sep. 25 and Mitchell et al, 2008 Jul. 29; 105(30):10513-8, Shen R et al, 2004 Dec. 14; 5(1):94, Mina L et al, 2007 June; 103(2):197-208, Zhang L et al, 2008 May 13; 105(19):7004-9, Ross J S et al, 2008 May; 13(5):477-93, Schetter A J et al, 2008 Jan. 30; 299(4):425-36, Staudt L M, 2003; 348:1777-85, Mulligan G et al, 2007 Apr. 15; 109(8):3177-88. Epub 2006 Dec. 21, McLendon R et al, 2008 Oct. 23; 455(7216):1061-8, and U.S. Pat. Nos. 5,538,848, 5,723,591, 5,876,930, 6,030,787, 6,258,569, and 5,804,375, each of which is herein incorporated by reference. In some embodiments, arrays of microRNA panels are use to simultaneously query the expression of multiple miRs. The Exiqon mIRCURY LNA microRNA PCR system panel (Exiqon, Inc., Woburn, Mass.) or the TaqMan\u00ae MicroRNA Assays and Arrays systems from Applied Biosystems (Foster City, Calif.) can be used for such purposes.","Microarray technology allows for the measurement of the steady-state mRNA or miRNA levels of thousands of transcripts or miRNAs simultaneously thereby presenting a powerful tool for identifying effects such as the onset, arrest, or modulation of uncontrolled cell proliferation. Two microarray technologies, such as cDNA arrays and oligonucleotide arrays can be used. The product of these analyses are typically measurements of the intensity of the signal received from a labeled probe used to detect a cDNA sequence from the sample that hybridizes to a nucleic acid sequence at a known location on the microarray. Typically, the intensity of the signal is proportional to the quantity of cDNA, and thus mRNA or miRNA, expressed in the sample cells. A large number of such techniques are available and useful. Methods for determining gene expression can be found in U.S. Pat. No. 6,271,002 to Linsley, et al.; U.S. Pat. No. 6,218,122 to Friend, et al.; U.S. Pat. No. 6,218,114 to Peck et al.; or U.S. Pat. No. 6,004,755 to Wang, et al., each of which is herein incorporated by reference in its entirety.","Analysis of an expression level can be conducted by comparing such intensities. This can be performed by generating a ratio matrix of the expression intensities of genes in a test sample versus those in a control sample. The control sample may be used as a reference, and different references to account for age, ethnicity and sex may be used. Different references can be used for different conditions or diseases, as well as different stages of diseases or conditions, as well as for determining therapeutic efficacy.","For instance, the gene expression intensities of mRNA or miRNAs derived from a diseased tissue, including those isolated from vesicles, can be compared with the expression intensities of the same entities in normal tissue of the same type (e.g., diseased breast tissue sample versus normal breast tissue sample). A ratio of these expression intensities indicates the fold-change in gene expression between the test and control samples. Alternatively, if vesicles are not normally present in from normal tissues (e.g. breast) then absolute quantitation methods, as is known in the art, can be used to define the number of miRNA molecules present without the requirement of miRNA or mRNA isolated from vesicles derived from normal tissue.","Gene expression profiles can also be displayed in a number of ways. A common method is to arrange raw fluorescence intensities or ratio matrix into a graphical dendogram where columns indicate test samples and rows indicate genes. The data is arranged so genes that have similar expression profiles are proximal to each other. The expression ratio for each gene is visualized as a color. For example, a ratio less than one (indicating down-regulation) may appear in the blue portion of the spectrum while a ratio greater than one (indicating upregulation) may appear as a color in the red portion of the spectrum. Commercially available computer software programs are available to display such data.","mRNAs or miRNAs that are considered differentially expressed can be either over expressed or under expressed in patients with a disease relative to disease free individuals. Over and under expression are relative terms meaning that a detectable difference (beyond the contribution of noise in the system used to measure it) is found in the amount of expression of the mRNAs or miRNAs relative to some baseline. In this case, the baseline is the measured mRNA\/miRNA expression of a non-diseased individual. The mRNA\/miRNA of interest in the diseased cells can then be either over or under expressed relative to the baseline level using the same measurement method. Diseased, in this context, refers to an alteration of the state of a body that interrupts or disturbs, or has the potential to disturb, proper performance of bodily functions as occurs with the uncontrolled proliferation of cells. Someone is diagnosed with a disease when some aspect of that person's genotype or phenotype is consistent with the presence of the disease. However, the act of conducting a diagnosis or prognosis includes the determination of disease\/status issues such as determining the likelihood of relapse or metastasis and therapy monitoring. In therapy monitoring, clinical judgments are made regarding the effect of a given course of therapy by comparing the expression of genes over time to determine whether the mRNA\/miRNA expression profiles have changed or are changing to patterns more consistent with normal tissue.","Levels of over and under expression are distinguished based on fold changes of the intensity measurements of hybridized microarray probes. A 2\u00d7 difference is preferred for making such distinctions or a p-value less than 0.05. That is, before an mRNA\/miRNA is the to be differentially expressed in diseased\/relapsing versus normal\/non-relapsing cells, the diseased cell is found to yield at least 2 times more, or 2 times less intensity than the normal cells. The greater the fold difference, the more preferred is use of the gene as a diagnostic or prognostic tool. mRNA\/miRNAs selected for the expression profiles of the instant invention have expression levels that result in the generation of a signal that is distinguishable from those of the normal or non-modulated genes by an amount that exceeds background using clinical laboratory instrumentation.","Statistical values can be used to confidently distinguish modulated from non-modulated mRNA\/miRNA and noise. Statistical tests find the mRNA\/miRNA most significantly different between diverse groups of samples. The Student's t-test is an example of a robust statistical test that can be used to find significant differences between two groups. The lower the p-value, the more compelling the evidence that the gene shows a difference between the different groups. Nevertheless, since microarrays measure more than one mRNA\/miRNA at a time, tens of thousands of statistical tests may be performed at one time. Because of this, one is unlikely to see small p-values just by chance and adjustments for this using a Sidak correction as well as a randomization\/permutation experiment can be made. A p-value less than 0.05 by the t-test is evidence that the gene is significantly different. More compelling evidence is a p-value less then 0.05 after the Sidak correction is factored in. For a large number of samples in each group, a p-value less than 0.05 after the randomization\/permutation test is the most compelling evidence of a significant difference.","In one embodiment, a method of generating a posterior probability score to enable diagnostic, prognostic, therapy-related, or physiological state specific biosignature scores can be arrived at by obtaining mRNA or miRNA (biomarker) expression data from a statistically significant number of patients; applying linear discrimination analysis to the data to obtain selected biomarkers; and applying weighted expression levels to the selected biomarkers with discriminate function factor to obtain a prediction model that can be applied as a posterior probability score. Other analytical tools can also be used to answer the same question such as, logistic regression and neural network approaches.","For instance, the following can be used for linear discriminant analysis:","where,\n\n","Numerous other well-known methods of pattern recognition are available. The following references provide some examples: : Golub et al. (1999); : Su et al. (2001); and Ramaswamy et al. (2001); -(2001); and : van 't Veer et al. (2002), all of which are herein incorporated by reference in their entireties.","A biosignature portfolio, further described below, can be established such that the combination of biomarkers in the portfolio exhibit improved sensitivity and specificity relative to individual biomarkers or randomly selected combinations of biomarkers. In one embodiment, the sensitivity of the biosignature portfolio can be reflected in the fold differences, for example, exhibited by a transcript's expression in the diseased state relative to the normal state. Specificity can be reflected in statistical measurements of the correlation of the signaling of transcript expression with the condition of interest. For example, standard deviation can be a used as such a measurement. In considering a group of biomarkers for inclusion in a biosignature portfolio, a small standard deviation in expression measurements correlates with greater specificity. Other measurements of variation such as correlation coefficients can also be used in this capacity.","Another parameter that can be used to select mRNA\/miRNA that generate a signal that is greater than that of the non-modulated mRNA\/miRNA or noise is the use of a measurement of absolute signal difference. The signal generated by the modulated mRNA\/miRNA expression is at least 20% different than those of the normal or non-modulated gene (on an absolute basis). It is even more preferred that such mRNA\/miRNA produce expression patterns that are at least 30% different than those of normal or non-modulated mRNA\/miRNA.","MiRNA can also be detected and measured by amplification from a biological sample and measured using methods described in U.S. Pat. No. 7,250,496, U.S. Application Publication Nos. 20070292878, 20070042380 or 20050222399 and references cited therein, each of which is herein incorporated by reference in its entirety. The microRNA can be assessed as in U.S. Pat. No. 7,888,035, entitled \u201cMETHODS FOR ASSESSING RNA PATTERNS,\u201d issued Feb. 15, 2011, which application is incorporated by reference herein in its entirety.","Peptide nucleic acids (PNAs) which are a new class of synthetic nucleic acid analogs in which the phosphate-sugar polynucleotide backbone is replaced by a flexible pseudo-peptide polymer may be utilized in analysis of a biosignature. PNAs are capable of hybridizing with high affinity and specificity to complementary RNA and DNA sequences and are highly resistant to degradation by nucleases and proteinases. Peptide nucleic acids (PNAs) are an attractive new class of probes with applications in cytogenetics for the rapid in situ identification of human chromosomes and the detection of copy number variation (CNV). Multicolor peptide nucleic acid-fluorescence in situ hybridization (PNA-FISH) protocols have been described for the identification of several human CNV-related disorders and infectious diseases. PNAs can also be utilized as molecular diagnostic tools to non-invasively measure oncogene mRNAs with tumor targeted radionuclide-PNA-peptide chimeras. Methods of using PNAs are described further in Pellestor F et al, 2008; 14(24):2439-44, Tian X et al, 2005 November; 1059:106-44, Paulasova P and Pellestor F, 47 (2004) 349-358, Stender H. 2003 September; 3(5):649-55. , Vigneault et al., 5(9), 777-779 (2008), each reference is herein incorporated by reference in its entirety. These methods can be used to screen the genetic materials isolated from a vesicle. When applying these techniques to a cell-of-origin specific vesicle, they can be used to identify a given molecular signal that directly pertains to the cell of origin.","Mutational analysis may be carried out for mRNAs and DNA, including those that are identified from a vesicle. For mutational analysis of a target or biomarker that is of RNA origin, the RNA (mRNA, miRNA or other) can be reverse transcribed into cDNA and subsequently sequenced or assayed, such as for known SNPs (by Taqman SNP assays, for example) or single nucleotide mutations, as well as using sequencing to look for insertions or deletions to determine mutations present in the cell-of-origin. Multiplexed ligation dependent probe amplification (MLPA) could alternatively be used for the purpose of identifying CNV in small and specific areas of interest. For example, once the total RNA has been obtained from isolated colon cancer-specific vesicles, cDNA can be synthesized and primers specific for exons 2 and 3 of the KRAS gene can be used to amplify these two exons containing codons 12, 13 and 61 of the KRAS gene. The same primers used for PCR amplification can be used for Big Dye Terminator sequence analysis on the ABI 3730 to identify mutations in exons 2 and 3 of KRAS. Mutations in these codons are known to confer resistance to drugs such as Cetuximab and Panitumimab. Methods of conducting mutational analysis are described in Maheswaran S et al, Jul. 2, 2008 (10.1056\/NEJMoa0800668) and Orita, M et al, 1989, (86): 2766-70, each of which is herein incorporated by reference in its entirety.","Other methods of conducting mutational analysis include miRNA sequencing. Applications for identifying and profiling miRNAs can be done by cloning techniques and the use of capillary DNA sequencing or \u201cnext-generation\u201d sequencing technologies. The new sequencing technologies currently available allow the identification of low-abundance miRNAs or those exhibiting modest expression differences between samples, which may not be detected by hybridization-based methods. Such new sequencing technologies include the massively parallel signature sequencing (MPSS) methodology described in Nakano et al. 20062006; 34:D731-D735. doi: 10.1093\/nar\/gkj077, the Roche\/454 platform described in Margulies et al. 20052005; 437:376-380 or the Illumina sequencing platform described in Berezikov et al. 2006b; 38:1375-1377, each of which is incorporated by reference in its entirety.","Additional methods to determine a biosignature includes assaying a biomarker by allele-specific PCR, which includes specific primers to amplify and discriminate between two alleles of a gene simultaneously, single-strand conformation polymorphism (SSCP), which involves the electrophoretic separation of single-stranded nucleic acids based on subtle differences in sequence, and DNA and RNA aptamers. DNA and RNA aptamers are short oligonucleotide sequences that can be selected from random pools based on their ability to bind a particular molecule with high affinity. Methods of using aptamers are described in Ulrich H et al, Comb Chem High Throughput Screen. 2006 September; 9(8):619-32, Ferreira C S et al, Anal Bioanal Chem. 2008 February; 390(4):1039-50, Ferreira C S et al, Tumour Biol. 2006; 27(6):289-301, each of which is herein incorporated by reference in its entirety.","Biomarkers can also be detected using fluorescence in situ hybridization (FISH). Methods of using FISH to detect and localize specific DNA sequences, localize specific mRNAs within tissue samples or identify chromosomal abnormalities are described in Shaffer D R et al, 2007 Apr. 1; 13(7):2023-9, Cappuzo F et al, , Volume 2, Number 5, May 2007, Moroni M et al, 2005 May; 6(5):279-86, each of which is herein incorporated by reference in its entirety.","An illustrative schematic for analyzing a population of vesicles for their payload is presented in . In an embodiment, the methods of the invention include characterizing a phenotype by capturing vesicles () and determining a level of microRNA species contained therein (), thereby characterizing the phenotype ().","A biosignature comprising a circulating biomarker or vesicle can comprise a binding agent thereto. The binding agent can be a DNA, RNA, aptamer, monoclonal antibody, polyclonal antibody, Fabs, Fab\u2032, single chain antibody, synthetic antibody, aptamer (DNA\/RNA), peptoid, zDNA, peptide nucleic acid (PNA), locked nucleic acid (LNA), lectin, synthetic or naturally occurring chemical compounds (including but not limited to drugs and labeling reagents).","A binding agent can used to isolate or detect a vesicle by binding to a component of the vesicle, as described above. The binding agent can be used to detect a vesicle, such as for detecting a cell-of-origin specific vesicle. A binding agent or multiple binding agents can themselves form a binding agent profile that provides a biosignature for a vesicle. One or more binding agents can be selected from . For example, if a vesicle population is detected or isolated using two, three or four binding agents in a differential detection or isolation of a vesicle from a heterogeneous population of vesicles, the particular binding agent profile for the vesicle population provides a biosignature for the particular vesicle population.","As an illustrative example, a vesicle for characterizing a cancer can be detected with one or more binding agents including, but not limited to, PSA, PSMA, PCSA, PSCA, B7H3, EpCam, TMPRSS2, mAB 5D4, XPSM-A9, XPSM-A10, Galectin-3, E-selectin, Galectin-1, or E4 (IgG2a kappa), or any combination thereof.","The binding agent can also be for a general vesicle biomarker, such as a \u201chousekeeping protein\u201d or antigen. The biomarker can be CD9, CD63, or CD81. For example, the binding agent can be an antibody for CD9, CD63, or CD81. The binding agent can also be for other proteins, such as for tissue specific or cancer specific vesicles. The binding agent can be for PCSA, PSMA, EpCam, B7H3, or STEAP. The binding agent can be for DR3, STEAP, epha2, TMEM211, MFG-E8, Annexin V, TF, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, or TETS. For example, the binding agent can be an antibody or aptamer for PCSA, PSMA, EpCam, B7H3, DR3, STEAP, epha2, TMEM211, MFG-E8, Annexin V, TF, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, or TETS.","Various proteins are not typically distributed evenly or uniformly on a vesicle shell. See, e.g., , which illustrates a schematic of protein expression patterns. Vesicle-specific proteins are typically more common, while cancer-specific proteins are less common. In some embodiments, capture of a vesicle is accomplished using a more common, less cancer-specific protein, such as one or more housekeeping proteins or antigen or general vesicle antigen (e.g., a tetraspanin), and one or more cancer-specific biomarkers and\/or one or more cell-of-origin specific biomarkers is used in the detection phase. In another embodiment, one or more cancer-specific biomarkers and\/or one or more cell-of-origin specific biomarkers are used for capture, and one or more housekeeping proteins or antigen or general vesicle antigen (e.g., a tetraspanin) is used for detection. In embodiments, the same biomarker is used for both capture and detection. Different binding agents for the same biomarker can be used, such as antibodies or aptamers that bind different epitopes of an antigen.","Additional cellular binding partners or binding agents may be identified by any conventional methods known in the art, or as described herein, and may additionally be used as a diagnostic, prognostic or therapy-related marker.","As an illustrative example, a vesicle for analysis for lung cancer can be detected with one or more binding agents including, but not limited to, SCLC specific aptamer HCA 12, SCLC specific aptamer HCC03, SCLC specific aptamer HCH07, SCLC specific aptamer HCH01, A-p50 aptamer (NF-KB), Cetuximab, Panitumumab, Bevacizumab, L19 Ab, F16 Ab, anti-CD45 (anti-ICAM-1, aka UV3), or L2G7 Ab (anti-HGF), or any combination thereof.","A vesicle for characterizing colon cancer can be detected with one or more binding agents including, but not limited to, angiopoietin 2 specific aptamer, beta-catenin aptamer, TCF1 aptamer, anti-Derlin1 ab, anti-RAGE, mAbgb3.1, Galectin-3, Cetuximab, Panitumumab, Matuzumab, Bevacizumab, or Mac-2, or any combination thereof.","A vesicle for characterizing adenoma versus colorectal cancer (CRC) can be detected with one or more binding agents including, but not limited to, Complement C3, histidine-rich glycoprotein, kininogen-1, or Galectin-3, or any combination thereof.","A vesicle for characterizing adenoma with low grade hyperplasia versus adenoma with high grade hyperplasia can be detected with a binding agent such as, but not limited to, Galectin-3 or any combination of binding agents specific for this comparison.","A vesicle for characterizing CRC versus normal state can be detected with one or more binding agents including, but not limited to, anti-ODC mAb, anti-CEA mAb, or Mac-2, or any combination thereof.","A vesicle for characterizing prostate cancer can be detected with one or more binding agents including, but not limited to, PSA, PSMA, TMPRSS2, mAB 5D4, XPSM-A9, XPSM-A10, Galectin-3, E-selectin, Galectin-1, or E4 (IgG2a kappa), or any combination thereof.","A vesicle for characterizing melanoma can be detected with one or more binding agents including, but not limited to, Tremelimumab (anti-CTLA4), Ipilimumumab (anti-CTLA4), CTLA-4 aptamers, STAT-3 peptide aptamers, Galectin-1, Galectin-3, or PNA, or any combination thereof.","A vesicle for characterizing pancreatic cancer can be detected with one or more binding agents including, but not limited to, H38-15 (anti-HGF) aptamer, H38-21(anti-HGF) aptamer, Matuzumab, Cetuximanb, or Bevacizumab, or any combination thereof.","A vesicle for characterizing brain cancer can be detected with one or more binding agents including, but not limited to, aptamer III.1 (pigpen) and\/or TTA1 (Tenascin-C) aptamer, or any combination thereof.","A vesicle for characterizing psoriasis can be detected with one or more binding agents including, but not limited to, E-selectin, ICAM-1, VLA-4, VCAM-1, alphaEbeta7, or any combination thereof.","A vesicle for characterizing cardiovascular disease (CVD) can be detected with one or more binding agents including, but not limited to, RB007 (factor IXA aptamer), ARC1779 (anti VWF) aptamer, or LOX1, or any combination thereof.","A vesicle for characterizing hematological malignancies can be detected with one or more binding agents including, but not limited to, anti-CD20 and\/or anti-CD52, or any combination thereof.","A vesicle for characterizing B-cell chronic lymphocytic leukemias can be detected with one or more binding agents including, but not limited to, Rituximab, Alemtuzumab, Apt48 (BCL6), R0-60, or D-R15-8, or any combination thereof.","A vesicle for characterizing B-cell lymphoma can be detected with one or more binding agents including, but not limited to, Ibritumomab, Tositumomab, Anti-CD20 Antibodies, Alemtuzumab, Galiximab, Anti-CD40 Antibodies, Epratuzumab, Lumiliximab, Hu1D10, Galectin-3, or Apt48, or any combination thereof.","A vesicle for characterizing Burkitt's lymphoma can be detected with one or more binding agents including, but not limited to, TD05 aptamer, IgM mAB (38-13), or any combination thereof.","A vesicle for characterizing cervical cancer can be detected with one or more binding agents including, but not limited to, Galectin-9 and\/or HPVE7 aptamer, or any combination thereof.","A vesicle for characterizing endometrial cancer can be detected with one or more binding agents including, but not limited to, Galectin-1 or any combinations of binding agents specific for endometrial cancer.","A vesicle for characterizing head and neck cancer can be detected with one or more binding agents including, but not limited to, (111) In-cMAb U36, anti-LOXL4, U36, BIWA-1, BIWA-2, BIWA-4, or BIWA-8, or any combination thereof.","A vesicle for characterizing IBD can be detected with one or more binding agents including, but not limited to, ACCA (anti-glycan Ab), ALCA (anti-glycan Ab), or AMCA (anti-glycan Ab), or any combination thereof.","A vesicle for characterizing diabetes can be detected with one or more binding agents including, but not limited to, RBP4 aptamer or any combination of binding agents specific for diabetes.","A vesicle for characterizing fibromyalgia can be detected with one or more binding agents including, but not limited to, L-selectin or any combination of binding agents specific for fibromyalgia.","A vesicle for characterizing multiple sclerosis (MS) can be detected with one or more binding agents including, but not limited to, Natalizumab (Tysabri) or any combination of binding agents specific for MS.","In addition, a vesicle for characterizing rheumatic disease can be detected with one or more binding agents including, but not limited to, Rituximab (anti-CD20 Ab) and\/or Keliximab (anti-CD4 Ab), or any combination of binding agents specific for rheumatic disease.","A vesicle for characterizing Alzheimer disease can be detected with one or more binding agents including, but not limited to, TH14-BACE1 aptapers, S10-BACE1 aptapers, anti-Abeta, Bapineuzumab (AAB-001)-Elan, LY2062430 (anti-amyloid beta Ab)-Eli Lilly, or BACE1-Anti sense, or any combination thereof.","A vesicle for characterizing Prion specific diseases can be detected with one or more binding agents including, but not limited to, rhuPrP(c) aptamer, DP7 aptamer, Thioaptamer 97, SAF-93 aptamer, 15B3 (anti-PrPSc Ab), monoclonal anti PrPSc antibody P1:1, 1.5D7, 1.6F4 Abs, mab 14D3, mab 4F2, mab 8G8, or mab 12F10, or any combination thereof.","A vesicle for characterizing sepsis can be detected with one or more binding agents including, but not limited to, HA-1A mAb, E-5 mAb, TNF-alpha MAb, Afelimomab, or E-selectin, or any combination thereof.","A vesicle for characterizing schizophrenia can be detected with one or more binding agents including, but not limited to, L-selectin and\/or N-CAM, or any combination of binding agents specific for schizophrenia.","A vesicle for characterizing depression can be detected with one or more binding agents including, but not limited to, GPIb or any combination of binding agents specific for depression.","A vesicle for characterizing GIST can be detected with one or more binding agents including, but not limited to, ANTI-DOG1 Ab or any combination of binding agents specific for GIST.","A vesicle for characterizing esophageal cancer can be detected with one or more binding agents including, but not limited to, CaSR binding agent or any combination of binding agents specific for esophageal cancer.","A vesicle for characterizing gastric cancer can be detected with one or more binding agents including, but not limited to, Calpain nCL-2 binding agent and\/or drebrin binding agent, or any combination of binding agents specific for gastric cancer.","A vesicle for characterizing COPD can be detected with one or more binding agents including, but not limited to, CXCR3 binding agent, CCR5 binding agent, or CXCR6 binding agent, or any combination of binding agents specific for COPD.","A vesicle for characterizing asthma can be detected with one or more binding agents including, but not limited to, VIP binding agent, PACAP binding agent, CGRP binding agent, NT3 binding agent, YKL-40 binding agent, S-nitrosothiols, SCCA2 binding agent, PAI binding agent, amphiregulin binding agent, or Periostin binding agent, or any combination of binding agents specific for asthma.","A vesicle for characterizing vulnerable plaque can be detected with one or more binding agents including, but not limited to, Gd-DTPA-g-mimRGD (Alpha v Beta 3 integrin binding peptide), or MMP-9 binding agent, or any combination of binding agents specific for vulnerable plaque.","A vesicle for characterizing ovarian cancer can be detected with one or more binding agents including, but not limited to, (90) Y-muHMFG1 binding agent and\/or OC125 (anti-CA125 antibody), or any combination of binding agents specific for ovarian cancer.","The binding agent can also be for a general vesicle biomarker, such as a \u201chousekeeping protein\u201d or antigen. The biomarker can be CD9, CD63, or CD81. For example, the binding agent can be an antibody for CD9, CD63, or CD81. The binding agent can also be for other proteins, such as for prostate specific or cancer specific vesicles. The binding agent can be for PCSA, PSMA, EpCam, B7H3, or STEAP. For example, the binding agent can be an antibody for PCSA, PSMA, EpCam, B7H3, or STEAP.","Furthermore, additional cellular binding partners or binding agents may be identified by any conventional methods known in the art, or as described herein, and may additionally be used as a diagnostic, prognostic or therapy-related marker.","Biosignatures for Prostate Cancer, GI Cancer and Ovarian Cancer","As described herein, biosignatures comprising circulating biomarkers can be used to characterize a cancer. This Section presents a non-exclusive list of biomarkers that can be used as part of a biosignature, e.g., for prostate, GI, or ovarian cancer. In some embodiments, the circulating biomarkers are associated with a vesicle or with a population of vesicles. For example, circulating biomarkers associated with vesicles can be used to capture and\/or to detect a vesicle or a vesicle population. This Section presents a non-exclusive list of biomarkers that can be used as part of a biosignature, e.g., for prostate, GI, or ovarian cancer.","It will be appreciated that the biomarkers presented herein may be useful in biosignatures for other diseases, e.g., other proliferative disorders and cancers of other cellular or tissue origins. For example, transformation in various cell types can be due to common events, e.g., mutation in p53 or other tumor suppressor. A biosignature comprising cell-of-origin biomarkers and cancer biomarkers can be used to further assess the nature of the cancer. Biomarkers for metastatic cancer may be used with cell-of-origin biomarkers to assess a metastatic cancer. Such biomarkers for use with the invention include those in Dawood, Novel biomarkers of metastatic cancer, Exp Rev Mol Diag July 2010, Vol. 10, No. 5, Pages 581-590, which publication is incorporated herein by reference in its entirety.","The biosignatures of the invention may comprise markers that are upregulated, downregulated, or have no change, depending on the reference. Solely for illustration, if the reference is a normal sample, the biosignature may indicate that the subject is normal if the subject's biosignature is not changed compared to the reference. Alternately, the biosignature may comprise a mutated nucleic acid or amino acid sequence so that the levels of the components in the biosignature are the same between a normal reference and a diseased sample. In another case, the reference can be a cancer sample, such that the subject's biosignature indicates cancer if the subject's biosignature is substantially similar to the reference. The biosignature of the subject can comprise components that are both upregulated and downregulated compared to the reference. Solely for illustration, if the reference is a normal sample, a cancer biosignature can comprise both upregulated oncogenes and downregulated tumor suppressors. Vesicle markers can also be differentially expressed in various settings. For example, tetraspanins may be overexpressed in cancer vesicles compared to non-cancer vesicles, whereas MFG-E8 can be overexpressed in non-cancer vesicles as compared to cancer vesicles.","Prostate Cancer","Prostate-specific antigen (PSA) is a protein produced by the cells of the prostate gland. PSA is present in small quantities in the serum of normal men, and is often elevated in the presence of prostate cancer (PCa) and in other prostate disorders. A blood test to measure PSA is currently used for the screening of prostate cancer, but this effectiveness has also been questioned. For example, PSA levels can be increased by prostate infection, irritation, benign prostatic hyperplasia (BPH), digital rectal examination (DRE) and recent ejaculation, producing a false positive result that can lead to unnecessary prostate biopsy and concomitant morbidities. BPH is a common cause of elevated PSA levels. PSA may indicate whether there is something wrong with the prostate, but it cannot effectively differentiate between BPH and PCa. PCA3, a transcript found to be overexpressed by prostate cancer cells, is thought to be slightly more specific for PCa, but this depends on the cutoffs used for PSA and PCA3, as well as the populations studied.","The invention provides circulating biomarkers can be used to distinguish BPH and PCa. A biomarker panel is assessed to distinguish BPH from PCa. The panel can be used to detect vesicles displaying certain surface markers. In some embodiments, the surface markers comprise one or more of BCMA, CEACAM-1, HVEM, IL-1 R4, IL-10 Rb and Trappin-2. The levels of the biomarkers in vesicles derived from blood samples can be assayed and then used to distinguish BPH from PCa.","In another aspect, microRNAs (miRs) are used to differentiate between BPH and prostate cancer. The miRs can be isolated directly from a patient sample, and\/or vesicles derived from patient samples can be analyzed for miR payload contained within the vesicles. The sample can be a bodily fluid, including semen, urine, blood, serum or plasma. The sample can also comprise a tissue or biopsy sample. A number of different methodologies are available for detecting miRs as described herein. In some embodiments, arrays of miR panels are use to simultaneously query the expression of multiple miRs. For example, the Exiqon mIRCURY LNA microRNA PCR system panel (Exiqon, Inc., Woburn, Mass.) can be used for such purposes. miRs that distinguish BPH and PCa can be overexpressed in BPH samples as compared to PCa samples, including without limitation one or more of: hsa-miR-329, hsa-miR-30a, hsa-miR-335, hsa-miR-152, hsa-miR-151-5p, hsa-miR-200a and hsa-miR-145. Alternately, miRs that distinguish BPH and PCa can be overexpressed in PCa samples versus BPH samples, including without limitation one or more of: hsa-miR-29a, hsa-miR-106b, hsa-miR-595, hsa-miR-142-5p, hsa-miR-99a, hsa-miR-20b, hsa-miR-373, hsa-miR-502-5p, hsa-miR-29b, hsa-miR-142-3p, hsa-miR-663, hsa-miR-423-5p, hsa-miR-15a, hsa-miR-888, hsa-miR-361-3p, hsa-miR-365, hsa-miR-10b, hsa-miR-199a-3p, hsa-miR-181a, hsa-miR-19a, hsa-miR-125b, hsa-miR-760, hsa-miR-7a, hsa-miR-671-5p, hsa-miR-7c, hsa-miR-1979, and hsa-miR-103.","The expression levels of one or more of the above miRs can be assessed and compared to reference levels to detect miRs that are differentially expressed, thereby providing a diagnostic, prognostic or theranostic readout. The reference levels can be those of the miRs in exosomes derived from normal patients, e.g., patients without prostate disease. Thus, differential expression of one or more miRs from the reference levels can indicate that the sample differs from normal, e.g., comprises BPH or PCa. The reference levels can be those of the miRs in exosomes derived from BPH patients. Thus, differential expression of one or more miRs from the reference levels can indicate that the sample differs from BPH, e.g., comprises normal or PCa. The reference levels can be those of the miRs in exosomes derived from PCa patients. Thus, differential expression of one or more miRs from the reference levels can indicate that the sample differs from PCa, e.g., comprises normal or BPH.","In some embodiments, the level of one or more miR in the test sample are correlated with the level of the same miRs in a reference sample, thereby providing a diagnostic, prognostic or theranostic readout. The reference sample can comprise the miR levels of one or more samples with BPH, PCa, or can be from normals without BPH or PCa. When the level of one or more miR in the test sample correlates most closely with that of the normal reference levels, the test sample can be classified as normal. When the level of one or more miR in the test sample correlates most closely with that of the BPH reference levels, the test sample can be classified as BPH. When the level of one or more miR in the test sample correlates most closely with that of the PCa reference levels, the test sample can be classified as PCa.","A biosignature can be used to characterize prostate cancer. As described above, a biosignature for prostate cancer can comprise a binding agent associated with prostate cancer (for example, as shown in ), and one or more additional biomarkers, such as shown in . For example, a biosignature for prostate cancer can comprise a binding agent to PSA, PSMA, TMPRSS2, mAB 5D4, XPSM-A9, XPSM-A10, Galectin-3, E-selectin, Galectin-1, E4 (IgG2a kappa), or any combination thereof, with one or more additional biomarkers, such as one or more miRNA, one or more DNA, one or more additional peptide, protein, or antigen associated with prostate cancer, such as, but not limited to, those shown in .","A biosignature for prostate cancer can comprise an antigen associated with prostate cancer (for example, as shown in ), and one or more additional biomarkers, such as shown in . A biosignature for prostate cancer can comprise one or more antigens associated with prostate cancer, such as, but not limited to, KIA1, intact fibronectin, PSA, TMPRSS2, FASLG, TNFSF10, PSMA, NGEP, IL-7RI, CSCR4, CysLT1R, TRPM8, Kv1.3, TRPV6, TRPM8, PSGR, MISIIR, or any combination thereof. The biosignature for prostate cancer can comprise one or more of the aforementioned antigens and one or more additional biomarkers, such as, but not limited to miRNA, mRNA, DNA, or any combination thereof.","A biosignature for prostate cancer can also comprise one or more antigens associated with prostate cancer, such as, but not limited to, KIA1, intact fibronectin, PSA, PCA3, TMPRSS2, TMPRSS2-ERG, FASLG, TNFSF10, PSMA, NGEP, IL-7RI, CSCR4, CysLT1R, TRPM8, Kv1.3, TRPV6, TRPM8, PSGR, MISIIR, or any combination thereof, and one or more miRNA biomarkers, such as, but not limited to, miR-202, miR-210, miR-296, miR-320, miR-370, miR-373, miR-498, miR-503, miR-184, miR-198, miR-302c, miR-345, miR-491, miR-513, miR-32, miR-182, miR-31, miR-26a-1\/2, miR-200c, miR-375, miR-196a-1\/2, miR-370, miR-425, miR-425, miR-194-1\/2, miR-181a-1\/2, miR-34b, let-7i, miR-188, miR-25, miR-106b, miR-449, miR-99b, miR-93, miR-92-1\/2, miR-125a, miR-141, let-7a, let-7b, let-7c, let-7d, let-7g, miR-16, miR-23a, miR-23b, miR-26a, miR-92, miR-99a, miR-103, miR-125a, miR-125b, miR-143, miR-145, miR-195, miR-199, miR-221, miR-222, miR-497, let-7f, miR-19b, miR-22, miR-26b, miR-27a, miR-27b, miR-29a, miR-29b, miR-30_5p, miR-30c, miR-100, miR-141, miR-148a, miR-205, miR-520h, miR-494, miR-490, miR-133a-1, miR-1-2, miR-218-2, miR-220, miR-128a, miR-221, miR-499, miR-329, miR-340, miR-345, miR-410, miR-126, miR-205, miR-7-1\/2, miR-145, miR-34a, miR-487, or let-7b, or any combination thereof.","A biosignature for prostate cancer can also comprise one or more circulating biomarkers, such as microRNAs associated with prostate cancer, including those described in Brase et al., Circulating miRNAs are correlated with tumor progression in prostate cancer. Int J Cancer. 2011 Feb. 1; 128(3):608-16; Wach et al., MiRNA profiles of prostate carcinoma detected by multi-platform miRNA screening. Int J Cancer. 2011 Mar. 11. doi: 10.1002\/ijc.26064; Gordanpour et al., miR-221 Is Down-regulated in TMPRSS2:ERG Fusion-positive Prostate Cancer. Anticancer Res. 2011 February; 31(2):403-10; Hagman et al., miR-34c is downregulated in prostate cancer and exerts tumor suppressive functions. Int J Cancer. 2010 Dec. 15; 127(12):2768-76; Sun et al., miR-99 Family of MicroRNAs Suppresses the Expression of Prostate-Specific Antigen and Prostate Cancer Cell Proliferation. Cancer Res. 2011 Feb. 15; 71(4):1313-24; Bao et al., Polymorphisms inside MicroRNAs and MicroRNA Target Sites Predict Clinical Outcomes in Prostate Cancer Patients Receiving Androgen-Deprivation Therapy. Clin Cancer Res. 2011 Feb. 15; 17(4):928-936; Moltzahn et al., Microfluidic-based multiplex qRT-PCR identifies diagnostic and prognostic microRNA signatures in the sera of prostate cancer patients. Cancer Res. 2011 Jan. 15; 71(2):550-60; Carlsson et al., Validation of suitable endogenous control genes for expression studies of miRNA in prostate cancer tissues. Cancer Genet Cytogenet. 2010 Oct. 15; 202(2):71-75; Zhang et al., Serum miRNA-21: elevated levels in patients with metastatic hormone-refractory prostate cancer and potential predictive factor for the efficacy of docetaxel-based chemotherapy. Prostate. 2011 Feb. 15; 71(3):326-31; Majid et al., MicroRNA-205-directed transcriptional activation of tumor suppressor genes in prostate cancer. Cancer. 2010 Dec. 15; 116(24):5637-49; Kojima et al., MiR-34a attenuates paclitaxel-resistance of hormone-refractory prostate cancer PC3 cells through direct and indirect mechanisms. Prostate. 2010 Oct. 1; 70(14):1501-12; Lewinshtein et al., Genomic predictors of prostate cancer therapy outcomes. Expert Rev Mol Diagn. 2010 July; 10(5):619-36; each of which publication is hereby incorporated by reference in its entirety.","Furthermore, the miRNA for a prostate cancer biosignature can be a miRNA that interacts with PFKFB3, RHAMM (HMMR), cDNA FLJ42103, ASPM, CENPF, NCAPG, Androgen Receptor, EGFR, HSP90, SPARC, DNMT3B, GART, MGMT, SSTR3, TOP2B, or any combination thereof, such as those described herein and depicted in . The miRNA can also be miR-9, miR-629, miR-141, miR-671-3p, miR-491, miR-182, miR-125a-3p, miR-324-5p, miR-148B, miR-222, or any combination thereof.","The biosignature for prostate cancer can comprise one or more antigens associated with prostate cancer, such as, but not limited to, KIA1, intact fibronectin, PSA, TMPRSS2, FASLG, TNFSF10, PSMA, NGEP, IL-7RI, CSCR4, CysLT1R, TRPM8, Kv1.3, TRPV6, TRPM8, PSGR, MISIIR, or any combination thereof, and one or more additional biomarkers such as, but not limited to, the aforementioned miRNAs, mRNAs (such as, but not limited to, AR or PCA3), snoRNA (such as, but not limited to, U50) or any combination thereof.","The biosignature can also comprise one or more gene fusions, such as ACSL3-ETV1, C15ORF21-ETV1, FLJ35294-ETV1, HERV-ETV1, TMPRSS2-ERG, TMPRSS2-ETV1\/4\/5, TMPRSS2-ETV4\/5, SLC5A3-ERG, SLC5A3-ETV1, SLC5A3-ETV5 or KLK2-ETV4.","A vesicle can be isolated, assayed, or both, for one or more miRNA and one or more antigens associated with prostate cancer to provide a diagnostic, prognostic or theranostic profile, such as the stage of the cancer, the efficacy of the cancer, or other characteristics of the cancer. Alternatively, the vesicle can be directly assayed from a sample, such that the vesicle is not purified or concentrated prior to assaying for one or more miRNA or antigens associated with prostate cancer.","As depicted in , a prostate cancer biosignature can comprise assaying EpCam, CD63, CD81, CD9, or any combination thereof, of a vesicle. The prostate cancer biosignature can comprise detection of EpCam, CD9, CD63, CD81, PCSA or any combination thereof. For example, the prostate cancer biosignature can comprise EpCam, CD9, CD63 and CD81 or PCSA, CD9, CD63 and CD81 (see for example, ). The prostate cancer biosignature can also comprise PCSA, PSMA, B7H3, or any combination thereof (see for example, ).","Furthermore, assessing a plurality of biomarkers can provide increased sensitivity, specificity, or signal intensity, as compared to assessing less than a plurality of biomarkers. For example, assessing PSMA and B7H3 can provide increased sensitivity in detection as compared to assessing PSMA or B7H3 alone. Assessing CD9 and CD63 can provide increased sensitivity in detection as compared to assessing CD9 or CD63 alone. In one embodiment, one or more of the following biomarkers are detected: EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR. In another embodiment, EpCam+, CK+, CD45\u2212 vesicles are detected.","Prostate cancer can also be characterized based on meeting at least 1, 2, 3, 4, 5, 6, 7, 8, 9, or 10 criteria. For example, a number of different criteria can be used: 1) if the amount of vesicles in a sample from a subject is higher than a reference value; 2) if the amount of prostate cell derived vesicles is higher than a reference value; and 3) if the amount of vesicles with one or more cancer specific biomarkers is higher than a reference value, the subject is diagnosed with prostate cancer. The method can further include a quality control measure.","In another embodiment, one or more biosignatures of a vesicle is used for the diagnosis between normal prostate and prostate cancer, or between normal prostate, BPH and PCa. Any appropriate biomarker disclosed herein can be used to distinguish PCa. In some embodiments, one or more general capture agents to a biomarker (or capture biomarker, a biomarker that is detected or bound by a capture agent) can be used to capture one or more vesicles from a sample from a subject.","Prostate specific biomarkers can be used to identify prostate specific vesicles. Cancer biomarkers can be used to identify cancer specific vesicles. In some embodiments, one or more of CD9, CD81 and CD63 are used as capture biomarkers. In some embodiments, PCSA is used as a prostate biomarker. In some embodiments, the one or more cancer biomarkers comprise one or more of EpCam and B7H3. Additional biomarkers that can distinguish PCa from normal include ICAM1, EGFR, STEAP1 and PSCA.","In some embodiments, the method of identifying prostate cancer in a subject comprises: (a) capturing a population of vesicles in a sample from the subject using a capture agent; (b) determining a level of one or more cancer biomarkers in the population of vesicles; (c) determining a level of one or more prostate biomarkers in the population of vesicles; and (d) identifying the subject as having prostate cancer if the level of the one or more cancer biomarkers and the level of one or more prostate biomarkers meet a predetermined threshold value. In some embodiments, the capture agent comprises one or more binding agents for CD9, CD81 and CD63. In some embodiments, the one or more prostate biomarker comprises PCSA and\/or PSMA. In some embodiments, the one or more cancer biomarkers comprise one or more of EpCam and B7H3. In other embodiments, binding agents to the one or more prostate and\/or cancer biomarkers are used as capture agents and binding agents to the one or more general vesicle markers are used as detection agents. In some embodiments, the predetermined threshold value comprises a measured value of a detectable label. For example, the detectable label can be a fluorescent moiety and the value can be a luminscence value of the moeity.","In another embodiment, the prognosis of prostate cancer is determined by detecting EpCam, CK (cytokeratin), and\/or CD45 expression. In an embodiment, a poor prognosis is provided when EpCam and CK are detected or detected at a high levels, and detection of CD45 is low or absent (ie. a vesicle that is EpCam+, CK+, CD45\u2212).","The methods of the invention can be used to distinguish a stage or grade of a prostate cancer. Prostate cancer staging is a process of categorizing the risk of cancer spread beyond the prostate. Such spread is related to the probability of being cured with local therapies such as surgery or radiation. The information considered in such prognostic classification is based on clinical and pathological factors, including physical examination, imaging studies, blood tests and\/or biopsy examination.","The most common scheme used to stage prostate cancer is promulgated by the American Joint Committee on Cancer, and is referred to as the TNM system. The TNM system evaluates the size of the tumor, the extent of involved lymph nodes, metastasis and also takes into account cancer grade. As with many other cancers, the cancers are often grouped by stage, e.g., stages I-IV). Generally, Stage I disease is cancer that is found incidentally in a small part of the sample when prostate tissue was removed for other reasons, such as benign prostatic hypertrophy, and the cells closely resemble normal cells and the gland feels normal to the examining finger. In Stage II more of the prostate is involved and a lump can be felt within the gland. In Stage III, the tumor has spread through the prostatic capsule and the lump can be felt on the surface of the gland. In Stage IV disease, the tumor has invaded nearby structures, or has spread to lymph nodes or other organs.","The Whitmore-Jewett stage is another staging scheme that is now used less often. The Gleason Grading System is based on cellular content and tissue architecture from biopsies, which provides an estimate of the destructive potential and ultimate prognosis of the disease.","The TNM tumor classification system can be used to describe the extent of cancer in a subject's body. T describes the size of the tumor and whether it has invaded nearby tissue, N describes regional lymph nodes that are involved, and M describes distant metastasis. TNM is maintained by the International Union Against Cancer (UICC) and is used by the American Joint Committee on Cancer (AJCC) and the International Federation of Gynecology and Obstetrics (FIGO). Those of skill in the art understand that not all tumors have TNM classifications such as, e.g., brain tumors. Generally, T (a,is,(0), 1-4) is measured as the size or direct extent of the primary tumor. N (0-3) refers to the degree of spread to regional lymph nodes: N0 means that tumor cells are absent from regional lymph nodes, N1 means that tumor cells spread to the closest or small numbers of regional lymph nodes, N2 means that tumor cells spread to an extent between N1 and N3; N3 means that tumor cells spread to most distant or numerous regional lymph nodes. M (0\/1) refers to the presence of metastasis: MX means that distant metastasis was not assessed; M0 means that no distant metastasis are present; M1 means that metastasis has occurred to distant organs (beyond regional lymph nodes). M1 can be further delineated as follows: M1a indicates that the cancer has spread to lymph nodes beyond the regional ones; M1b indicates that the cancer has spread to bone; and M1c indicates that the cancer has spread to other sites (regardless of bone involvement). Other parameters may also be assessed. G (1-4) refers to the grade of cancer cells (i.e., they are low grade if they appear similar to normal cells, and high grade if they appear poorly differentiated). R (0\/1\/2) refers to the completeness of an operation (i.e., resection-boundaries free of cancer cells or not). L (0\/1) refers to invasion into lymphatic vessels. V (0\/1) refers to invasion into vein. C (1-4) refers to a modifier of the certainty (quality) of V.","Prostate tumors are often assessed using the Gleason scoring system. The Gleason scoring system is based on microscopic tumor patterns assessed by a pathologist while interpreting a biopsy specimen. When prostate cancer is present in the biopsy, the Gleason score is based upon the degree of loss of the normal glandular tissue architecture (i.e. shape, size and differentiation of the glands). The classic Gleason scoring system has five basic tissue patterns that are technically referred to as tumor \u201cgrades.\u201d The microscopic determination of this loss of normal glandular structure caused by the cancer is represented by a grade, a number ranging from 1 to 5, with 5 being the worst grade. Grade 1 is typically where the cancerous prostate closely resembles normal prostate tissue. The glands are small, well-formed, and closely packed. At Grade 2 the tissue still has well-formed glands, but they are larger and have more tissue between them, whereas at Grade 3 the tissue still has recognizable glands, but the cells are darker. At high magnification, some of these cells in a Grade 3 sample have left the glands and are beginning to invade the surrounding tissue. Grade 4 samples have tissue with few recognizable glands and many cells are invading the surrounding tissue. For Grade 5 samples, the tissue does not have recognizable glands, and are often sheets of cells throughout the surrounding tissue.","miRs that distinguish metastatic and non-metastatic prostate cancer can be overexpressed in metastatic samples versus non-metastatic. Alternately, miRs that distinguish metastatic and non-metastatic prostate cancer can be overexpressed in non-metastatic samples versus metastatic. Useful miRs for distinguishing metastatic prostate cancer include one or more of miR-495, miR-10a, miR-30a, miR-570, miR-32, miR-885-3p, miR-564, and miR-134. In another embodiment, miRs for distinguishing metastatic prostate cancer include one or more of hsa-miR-375, hsa-miR-452, hsa-miR-200b, hsa-miR-146b-5p, hsa-miR-1296, hsa-miR-17*, hsa-miR-100, hsa-miR-574-3p, hsa-miR-20a*, hsa-miR-572, hsa-miR-1236, hsa-miR-181a, hsa-miR-937, and hsa-miR-23a*. In still another embodiment, useful miRs for distinguishing metastatic prostate cancer include one or more of hsa-miR-200b, hsa-miR-375, hsa-miR-582-3p, hsa-miR-17*, hsa-miR-1296, hsa-miR-20a*, hsa-miR-100, hsa-miR-452, and hsa-miR-577. The miRs for distinguishing metastatic prostate cancer can be one or more of miR-141, miR-375, miR-200b and miR-574-3p.","In another aspect, microRNAs (miRs) are used to differentiate between cancer and non-cancer samples. Useful miRs for distinguishing cancer from non-cancer include one or more of hsa-miR-574-3p, hsa-miR-331-3p, hsa-miR-326, hsa-miR-181a-2*, hsa-miR-130b, hsa-miR-301a, hsa-miR-141, hsa-miR-432, hsa-miR-107, hsa-miR-628-5p, hsa-miR-625*, hsa-miR-497, and hsa-miR-484. In another embodiment, useful miRs for distinguishing cancer from non-cancer include one or more of hsa-miR-574-3p, hsa-miR-141, hsa-miR-331-3p, hsa-miR-432, hsa-miR-326, hsa-miR-2110, hsa-miR-107, hsa-miR-130b, hsa-miR-301a, and hsa-miR-625*. In still another embodiment, the useful miRs for distinguishing cancer from non-cancer include one or more of hsa-miR-107, hsa-miR-326, hsa-miR-432, hsa-miR-574-3p, hsa-miR-625*, hsa-miR-2110, hsa-miR-301a, hsa-miR-141 or hsa-miR-373*. The biosignature for distinguishing cancer from non-cancer can comprise one or more of miR-148a, miR-122, miR-146a, miR-22, and miR-24.","The biosignatures of the invention can comprise multiple markers. For example, multiple protein markers and miRs can be used to distinguish prostate cancer from normal, BPH and PCa, or metastatic versus non-metastatic disease. In this manner, improved sensitivity, specificity, and\/or accuracy can be obtained. In some embodiments, the levels of one or more of hsa-miR-432, hsa-miR-143, hsa-miR-424, hsa-miR-204, hsa-miR-581f and hsa-miR-451 are detected in a patient sample to assess the presence of prostate cancer. Any of these miRs can be elevated in patients with PCa but having serum PSA<4.0 ng\/ml. In an embodiment, the invention provides a method of assessing a prostate cancer, comprising determining a level of one or more of hsa-miR-432, hsa-miR-143, hsa-miR-424, hsa-miR-204, hsa-miR-581f and hsa-miR-451 in a sample from a subject. The sample can be a bodily fluid, e.g., blood, plasma or serum. The miRs can be isolated in vesicles isolated from the sample. The subject can have a PSA level less than some threshold, such as 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, 3.6, 3.8, 4.0, 4.2, 4.4, 4.6, 4.8, 5.0, 5.2, 5.4, 5.6, 5.8, or 6.0 ng\/ml in a blood sample. Higher levels of the miRs than in a reference sample can indicate the presence of PCa in the sample. In some embodiments, the reference comprises a level of the one or more miRs in control samples from subjects without PCa. In some embodiments, the reference comprises a level of the one or more miRs in control samples from subject with PCa and PSA levels\u2267some threshold, such as 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, 3.6, 3.8, 4.0, 4.2, 4.4, 4.6, 4.8, 5.0, 5.2, 5.4, 5.6, 5.8, or 6.0 ng\/ml. The threshold can be 4.0 ng\/ml.","The invention provides for assessing a prostate disorder comprising detecting a presence or level of one or more circulating biomarker selected from the biomarkers listed above. The one or more circulating biomarker can also be selected from BCMA, CEACAM-1, HVEM, IL-1 R4, IL-10 Rb, Trappin-2, p53, hsa-miR-103, hsa-miR-106b, hsa-miR-10b, hsa-miR-125b, hsa-miR-142-3p, hsa-miR-142-5p, hsa-miR-145, hsa-miR-151-5p, hsa-miR-152, hsa-miR-15a, hsa-miR-181a, hsa-miR-1979, hsa-miR-199a-3p, hsa-miR-19a, hsa-miR-200a, hsa-miR-20b, hsa-miR-29a, hsa-miR-29b, hsa-miR-30a, hsa-miR-329, hsa-miR-335, hsa-miR-361-3p, hsa-miR-365, hsa-miR-373, hsa-miR-423-5p, hsa-miR-502-5p, hsa-miR-595, hsa-miR-663, hsa-miR-671-5p, hsa-miR-760, hsa-miR-7a, hsa-miR-7c, hsa-miR-888, hsa-miR-99a, and a combination thereof. The one or more circulating biomarkers can be selected from the following: hsa-miR-100, hsa-miR-1236, hsa-miR-1296, hsa-miR-141, hsa-miR-146b-5p, hsa-miR-17*, hsa-miR-181a, hsa-miR-200b, hsa-miR-20a*, hsa-miR-23a*, hsa-miR-331-3p, hsa-miR-375, hsa-miR-452, hsa-miR-572, hsa-miR-574-3p, hsa-miR-577, hsa-miR-582-3p, hsa-miR-937, miR-10a, miR-134, miR-141, miR-200b, miR-30a, miR-32, miR-375, miR-495, miR-564, miR-570, miR-574-3p, miR-885-3p, and a combination thereof. Further still, the one or more circulating biomarkers can be selected from the following: hsa-let-7b, hsa-miR-107, hsa-miR-1205, hsa-miR-1270, hsa-miR-130b, hsa-miR-141, hsa-miR-143, hsa-miR-148b*, hsa-miR-150, hsa-miR-154*, hsa-miR-181a*, hsa-miR-181a-2*, hsa-miR-18a*, hsa-miR-19b-1*, hsa-miR-204, hsa-miR-2110, hsa-miR-215, hsa-miR-217, hsa-miR-219-2-3p, hsa-miR-23b*, hsa-miR-299-5p, hsa-miR-301a, hsa-miR-301a, hsa-miR-326, hsa-miR-331-3p, hsa-miR-365*, hsa-miR-373*, hsa-miR-424, hsa-miR-424*, hsa-miR-432, hsa-miR-450a, hsa-miR-451, hsa-miR-484, hsa-miR-497, hsa-miR-517*, hsa-miR-517a, hsa-miR-518f, hsa-miR-574-3p, hsa-miR-595, hsa-miR-617, hsa-miR-625*, hsa-miR-628-5p, hsa-miR-629, hsa-miR-634, hsa-miR-769-5p, hsa-miR-93, hsa-miR-96. The circulating biomarkers can be one or more of hsa-miR-1974, hsa-miR-27b, hsa-miR-103, hsa-miR-146a, hsa-miR-22, hsa-miR-382, hsa-miR-23a, hsa-miR-376c, hsa-miR-335, hsa-miR-142-5p, hsa-miR-221, hsa-miR-142-3p, hsa-miR-151-3p, hsa-miR-21 and hsa-miR-16. In an embodiment, the circulating biomarkers comprise one or more of CD9, PSMA, PCSA, CD63, CD81, B7H3, IL 6, OPG-13, IL6R, PA2G4, EZH2, RUNX2, SERPINB3, and EpCam. The biomarkers can comprise one or more of FOX01A, SOX9, CLNS1A, PTGDS, XPO1, LETMD1, RAD23B, ABCC3, APC, CHES1, EDNRA, FRZB, HSPG2, and TMPRSS2_ETV1 fusion. See WO2010056993, which application is incorporated by reference herein in its entirety. In another embodiment, the circulating biomarkers comprise one or more of A33, a33 n15, AFP, ALA, ALIX, ALP, AnnexinV, APC, ASCA, ASPH (246-260), ASPH (666-680), ASPH (A-10), ASPH (D01P), ASPH (D03), ASPH (G-20), ASPH(H-300), AURKA, AURKB, B7H3, B7H4, BCA-225, BCNP1, BDNF, BRCA, CA125 (MUC16), CA-19-9, C-Bir, CD1.1, CD10, CD174 (Lewis y), CD24, CD44, CD46, CD59 (MEM-43), CD63, CD66e CEA, CD73, CD81, CD9, CDA, CDAC1 1a2, CEA, C-Erb2, C-erbB2, CRMP-2, CRP, CXCL12, CYFRA21-1, DLL4, DR3, EGFR, Epcam, EphA2, EphA2 (H-77), ER, ErbB4, EZH2, FASL, FRT, FRT c.f23, GDF15, GPCR, GPR30, Gro-alpha, HAP, HBD 1, HBD2, HER 3 (ErbB3), HSP, HSP70, hVEGFR2, iC3b, IL6 Unc, IL-1B, IL6 Unc, IL6R, IL8, IL-8, INSIG-2, KLK2, L1CAM, LAMN, LDH, MACC-1, MAPK4, MART-1, MCP-1, M-CSF, MFG-E8, MIC1, MIF, MIS RII, MMG, MMP26, MMP7, MMP9, MS4A1, MUC1, MUC1 seq1, MUC1 seq11A, MUC17, MUC2, Ncam, NGAL, NPGP\/NPFF2, OPG, OPN, p53, p53, PA2G4, PBP, PCSA, PDGFRB, PGP9.5, PIM1, PR (B), PRL, PSA, PSMA, PSME3, PTEN, R5-CD9 Tube 1, Reg IV, RUNX2, SCRN1, seprase, SERPINB3, SPARC, SPB, SPDEF, SRVN, STAT 3, STEAP1, TF (FL-295), TFF3, TGM2, TIMP-1, TIMP1, TIMP2, TMEM211, TMPRSS2, TNF-alpha, Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK, UNC93A, VEGF A, and YPSMA-1. Any combination of these markers can be used in a biosignature to assess a prostate cancer. The circulating biomarkers can be associated with vesicles, e.g., vesicle surface markers or vesicle payload. The prostate disorders include without limitation a benign disorder such as BPH, or prostate cancer, including cancers of various stages and grades. See, e.g., Table 5.",{"@attributes":{"id":"p-0919","num":"0921"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"329pt","align":"center"}},"thead":{"row":{"entry":"TABLE 5"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Biomarkers for Prostate Disorders"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Illustrative Disorder","Biomarkers"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Benign Prostate","BCMA, CEACAM-1, HVEM, IL-1 R4, IL-10 Rb, Trappin-2, p53, hsa-miR-329,"]},{"entry":["Hyperplasia (BPH)","hsa-miR-30a, hsa-miR-335, hsa-miR-152, hsa-miR-151-5p, hsa-miR-200a, hsa-"]},{"entry":[{},"miR-145, hsa-miR-29a, hsa-miR-106b, hsa-miR-595, hsa-miR-142-5p, hsa-miR-"]},{"entry":[{},"99a, hsa-miR-20b, hsa-miR-373, hsa-miR-502-5p, hsa-miR-29b, hsa-miR-142-3p,"]},{"entry":[{},"hsa-miR-663, hsa-miR-423-5p, hsa-miR-15a, hsa-miR-888, hsa-miR-361-3p, hsa-"]},{"entry":[{},"miR-365, hsa-miR-10b, hsa-miR-199a-3p, hsa-miR-181a, hsa-miR-19a, hsa-miR-"]},{"entry":[{},"125b, hsa-miR-760, hsa-miR-7a, hsa-miR-671-5p, hsa-miR-7c, hsa-miR-1979,"]},{"entry":[{},"hsa-miR-103"]},{"entry":["Metastatic Prostate","hsa-miR-100, hsa-miR-1236, hsa-miR-1296, hsa-miR-141, hsa-miR-146b-5p, hsa-"]},{"entry":["Cancer","miR-17*, hsa-miR-181a, hsa-miR-200b, hsa-miR-20a*, hsa-miR-23a*, hsa-miR-"]},{"entry":[{},"331-3p, hsa-miR-375, hsa-miR-452, hsa-miR-572, hsa-miR-574-3p, hsa-miR-577,"]},{"entry":[{},"hsa-miR-582-3p, hsa-miR-937, miR-10a, miR-134, miR-141, miR-200b, miR-30a,"]},{"entry":[{},"miR-32, miR-375, miR-495, miR-564, miR-570, miR-574-3p, miR-885-3p"]},{"entry":["Metastatic Prostate","hsa-miR-200b, hsa-miR-375, hsa-miR-141, hsa-miR-331-3p, hsa-miR-181a, hsa-"]},{"entry":["Cancer","miR-574-3p"]},{"entry":["Metastatic Prostate","FOX01A, SOX9, CLNS1A, PTGDS, XPO1, LETMD1, RAD23B, ABCC3, APC,"]},{"entry":["Cancer","CHES1, EDNRA, FRZB, HSPG2, TMPRSS2_ETV1 fusion"]},{"entry":["Prostate Cancer","hsa-let-7b, hsa-miR-107, hsa-miR-1205, hsa-miR-1270, hsa-miR-130b, hsa-miR-"]},{"entry":[{},"141, hsa-miR-143, hsa-miR-148b*, hsa-miR-150, hsa-miR-154*, hsa-miR-181a*,"]},{"entry":[{},"hsa-miR-181a-2*, hsa-miR-18a*, hsa-miR-19b-1*, hsa-miR-204, hsa-miR-2110,"]},{"entry":[{},"hsa-miR-215, hsa-miR-217, hsa-miR-219-2-3p, hsa-miR-23b*, hsa-miR-299-5p,"]},{"entry":[{},"hsa-miR-301a, hsa-miR-301a, hsa-miR-326, hsa-miR-331-3p, hsa-miR-365*, hsa-"]},{"entry":[{},"miR-373*, hsa-miR-424, hsa-miR-424*, hsa-miR-432, hsa-miR-450a, hsa-miR-"]},{"entry":[{},"451, hsa-miR-484, hsa-miR-497, hsa-miR-517*, hsa-miR-517a, hsa-miR-518f,"]},{"entry":[{},"hsa-miR-574-3p, hsa-miR-595, hsa-miR-617, hsa-miR-625*, hsa-miR-628-5p,"]},{"entry":[{},"hsa-miR-629, hsa-miR-634, hsa-miR-769-5p, hsa-miR-93, hsa-miR-96"]},{"entry":["Prostate Cancer","CD9, PSMA, PCSA, CD63, CD81, B7H3, IL 6, OPG-13, IL6R, PA2G4, EZH2,"]},{"entry":[{},"RUNX2, SERPINB3, EpCam"]},{"entry":["Prostate Cancer","A33, a33 n15, AFP, ALA, ALIX, ALP, AnnexinV, APC, ASCA, ASPH (246-"]},{"entry":[{},"260), ASPH (666-680), ASPH (A-10), ASPH (D01P), ASPH (D03), ASPH (G-"]},{"entry":[{},"20), ASPH (H-300), AURKA, AURKB, B7H3, B7H4, BCA-225, BCNP1, BDNF,"]},{"entry":[{},"BRCA, CA125 (MUC16), CA-19-9, C-Bir, CD1.1, CD10, CD174 (Lewis y),"]},{"entry":[{},"CD24, CD44, CD46, CD59 (MEM-43), CD63, CD66e CEA, CD73, CD81, CD9,"]},{"entry":[{},"CDA, CDAC1 1a2, CEA, C-Erb2, C-erbB2, CRMP-2, CRP, CXCL12, CYFRA21-"]},{"entry":[{},"1, DLL4, DR3, EGFR, Epcam, EphA2, EphA2 (H-77), ER, ErbB4, EZH2, FASL,"]},{"entry":[{},"FRT, FRT c.f23, GDF15, GPCR, GPR30, Gro-alpha, HAP, HBD 1, HBD2, HER 3"]},{"entry":[{},"(ErbB3), HSP, HSP70, hVEGFR2, iC3b, IL 6 Unc, IL-1B, IL6 Unc, IL6R, IL8,"]},{"entry":[{},"IL-8, INSIG-2, KLK2, L1CAM, LAMN, LDH, MACC-1, MAPK4, MART-1,"]},{"entry":[{},"MCP-1, M-CSF, MFG-E8, MIC1, MIF, MIS RII, MMG, MMP26, MMP7, MMP9,"]},{"entry":[{},"MS4A1, MUC1, MUC1 seq1, MUC1 seq11A, MUC17, MUC2, Ncam, NGAL,"]},{"entry":[{},"NPGP\/NPFF2, OPG, OPN, p53, p53, PA2G4, PBP, PCSA, PDGFRB, PGP9.5,"]},{"entry":[{},"PIM1, PR (B), PRL, PSA, PSMA, PSME3, PTEN, R5-CD9 Tube 1, Reg IV,"]},{"entry":[{},"RUNX2, SCRN1, seprase, SERPINB3, SPARC, SPB, SPDEF, SRVN, STAT 3,"]},{"entry":[{},"STEAP1, TF (FL-295), TFF3, TGM2, TIMP-1, TIMP1, TIMP2, TMEM211,"]},{"entry":[{},"TMPRSS2, TNF-alpha, Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK,"]},{"entry":[{},"UNC93A, VEGF A, YPSMA-1"]},{"entry":["Prostate Cancer Vesicle","5T4, ACTG1, ADAM10, ADAM15, ALDOA, ANXA2, ANXA6, APOA1,"]},{"entry":["Markers","ATP1A1, BASP1, C1orf58, C20orf114, C8B, CAPZA1, CAV1, CD151, CD2AP,"]},{"entry":[{},"CD59, CD9, CD9, CFL1, CFP, CHMP4B, CLTC, COTL1, CTNND1, CTSB,"]},{"entry":[{},"CTSZ, CYCS, DPP4, EEF1A1, EHD1, ENO1, F11R, F2, F5, FAM125A,"]},{"entry":[{},"FNBP1L, FOLH1, GAPDH, GLB1, GPX3, HIST1H1C, HIST1H2AB,"]},{"entry":[{},"HSP90AB1, HSPA1B, HSPA8, IGSF8, ITGB1, ITIH3, JUP, LDHA, LDHB,"]},{"entry":[{},"LUM, LYZ, MFGE8, MGAM, MMP9, MYH2, MYL6B, NME1, NME2,"]},{"entry":[{},"PABPC1, PABPC4, PACSIN2, PCBP2, PDCD6IP, PRDX2, PSA, PSMA,"]},{"entry":[{},"PSMA1, PSMA2, PSMA4, PSMA6, PSMA7, PSMB1, PSMB2, PSMB3, PSMB4,"]},{"entry":[{},"PSMB5, PSMB6, PSMB8, PTGFRN, RPS27A, SDCBP, SERINC5, SH3GL1,"]},{"entry":[{},"SLC3A2, SMPDL3B, SNX9, TACSTD1, TCN2, THBS1, TPI1, TSG101, TUBB,"]},{"entry":[{},"VDAC2, VPS37B, YWHAG, YWHAQ, YWHAZ"]},{"entry":["Prostate Cancer Treatment","hsa-miR-1974, hsa-miR-27b, hsa-miR-103, hsa-miR-146a, hsa-miR-22, hsa-miR-"]},{"entry":[{},"382, hsa-miR-23a, hsa-miR-376c, hsa-miR-335, hsa-miR-142-5p, hsa-miR-221,"]},{"entry":[{},"hsa-miR-142-3p, hsa-miR-151-3p, hsa-miR-21, hsa-miR-16"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"Any combination of these markers can be used in a biosignature to assess a prostate disorder, such as BPH and prostate cancer. The biosignature can also be used to assess the stage or grade of the prostate cancer.","The prostate cancer can be characterizing using one or more processes disclosed herein with at least 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, or 70% sensitivity. The prostate cancer can be characterized with at least 80, 81, 82, 83, 84, 85, 86, or 87% sensitivity. For example, the prostate cancer can be characterized with at least 87.1, 87.2, 87.3, 87.4, 87.5, 87.6, 87.7, 87.8, 87.9, 88.0, or 89% sensitivity, such as with at least 90% sensitivity, such as at least 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100% sensitivity.","The prostate cancer of a subject can also be characterized with at least 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, or 97% specificity, such as with at least 97.1, 97.2, 97.3, 97.4, 97.5, 97.6, 97.7, 97.8, 97.8, 97.9, 98.0, 98.1, 98.2, 98.3, 98.4, 98.5, 98.6, 98.7, 98.8, 98.9, 99.0, 99.1, 99.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99.8, 99.9 or 100% specificity.","The prostate cancer can also be characterized with at least 70% sensitivity and at least 80, 90, 95, 99, or 100% specificity; at least 80% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 85% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 86% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 87% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 88% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 89% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 90% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 95% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 99% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; or at least 100% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity.","In some embodiments, the biosignature characterizes a phenotype of a subject with at least 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, or 97% accuracy, such as with at least 97.1, 97.2, 97.3, 97.4, 97.5, 97.6, 97.7, 97.8, 97.8, 97.9, 98.0, 98.1, 98.2, 98.3, 98.4, 98.5, 98.6, 98.7, 98.8, 98.9, 99.0, 99.1, 99.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99.8, 99.9 or 100% accuracy.","In some embodiments, the biosignature characterizes a phenotype of a subject with an AUC of at least 0.70, 0.71, 0.72, 0.73, 0.74, 0.75, 0.76, 0.77, 0.78, 0.79, 0.80, 0.81, 0.82, 0.83, 0.84, 0.85, 0.86, 0.87, 0.88, 0.89, 0.90, 0.91, 0.92, 0.93, 0.94, 0.95, 0.96, or 0.97, such as with at least 0.971, 0.972, 0.973, 0.974, 0.975, 0.976, 0.977, 0.978, 0.978, 0.979, 0.980, 0.981, 0.982, 0.983, 0.984, 0.985, 0.986, 0.987, 0.988, 0.989, 0.99, 0.991, 0.992, 0.993, 0.994, 0.995, 0.996, 0.997, 0.998, 0.999 or 1.00.","Furthermore, the confidence level for determining the specificity, sensitivity, accuracy and\/or AUC can be determined with at least 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, or 99% confidence.","Gastrointestinal Cancer","The gastrointestinal (GI) tract includes without limitation the oral cavity, gums, pharynx, tongue, salivary glands, esophagus, pancreas, liver, gallbladder, small intestine (duodenum, jejunum, ileum), bile duct, stomach, large intestine (cecum, colon, rectum), appendix and anus. The biosignature can be used to detect or characterize cancers of such components, e.g., colorectal cancer (CRC), stomach cancer, intestinal cancer, liver cancer or esophageal cancer. A gastrointestinal (GI) tract bio-signature can comprise any one or more antigens for as listed in , any one or more binding agents associated with isolating a vesicle for characterizing colon cancer (for example, as shown in ), any one or more additional biomarkers, such as shown in .","Colon Cancer","Although colonoscopy is the gold standard to screen and identify colorectal cancer (CRC), it is estimated half of patients who are recommended for colonoscopy are not compliant. Often the lack of compliance is because many perceive a colonoscopy as an uncomfortable and invasive procedure. A less invasive diagnostic test that could identify those patients that have a blood-based biosignature indicative of the need for detection and biopsy by colonoscopy could improve compliance. This strategy would result in cancers being identified earlier and prevent disease-free individuals from undergoing an unnecessary invasive procedure. Current blood-based tests rely on increased levels of either carcinoembryonic antigen (CEA) or carbohydrate antigenic determinant (CA 19-9). Unfortunately, CEA and CA 19-9 are neither organ-specific nor tumor-specific. The present invention improves upon these markers using vesicle-based detection assays.","The present invention provides methods to identify subjects likely to have or having CRC using a biosignature derived from a sample from the subject. The sample can be bodily fluids such as blood, plasma or serum, or stool. The biosignature can contain circulating biomarkers, including biomarkers associated with vesicles. The biosignature may contain mutations detected by sequencing nucleic acids, e.g., RNAs contained within vesicles.","In one aspect of the invention, biosignatures are derived from vesicles isolated from plasma of patients with and without CRC. Vesicle surface proteins are used in a multiplex assay to capture and detect vesicles. The quantity of vesicles with significant concentrations of these surface proteins leads to the development of a vesicle-specific biosignature that can differentiate CRC samples from normal. Such vesicles present in blood plasma of CRC patients provide a signature by which CRC can be diagnosed as early as histological grade 1. In some embodiments, vesicles are captured with antibodies to various surface proteins. The capture vesicles can be detected with vesicle specific markers, e.g., one or more of CD9, CD81, and CD63. In some embodiments, the vesicle-based biosignature comprises measuring the level of one or more of CD9, CD81, CD63, EpCam, EGFR, and STEAP. In some embodiments, one or more of the following markers are used to capture and\/or detect vesicles: CD9, NGAL, CD81, STEAP, CD24, A33, CD66E, EPHA2, TMEM211, TROP2, TROP2, EGFR, DR3, UNC93A, MUC17, EpCAM, MUC17, CD63, B7H3. In some embodiments, one or more of the following markers are used to capture and\/or detect vesicles: TMEM211, MUC1, GPR110 (GPCR 110), CD24, CD9, CD81, and CD63. In some embodiments, one or more of the following markers are used to capture and\/or detect vesicles: DR3, STEAP, epha2, TMEM211, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, and TETS. In some embodiments, TMEM211 is used to capture and\/or detect vesicles. In some embodiments, MUC1 is used to capture and\/or detect vesicles. In some embodiments, GPR110 is used to capture and\/or detect vesicles. In some embodiments, CD24 is used to capture and\/or detect vesicles. In some embodiments, one or more of TMEM211, MUC1, GPR110 (GPCR 110), and CD24 are used to capture vesicles and one or more general vesicle marker is used to detection the captured vesicles.","In another aspect of the invention, microRNAs (miRs) associated with vesicles are used to determine a biosignature. The miRs can be derived from vesicles, e.g., exosomes, isolated from a patient sample, e.g., blood. In some embodiments, one or more of the following miRs are used to derive a CRC biosignature: miR 92, miR 21, miR 9 and miR 491.","In still another aspect of the invention, the payload within a vesicle is assessed. KRAS and BRAF mutation screening can be used for colon cancer monitoring from tumor samples. As shown in Example 4, KRAS mutations are found in RNA derived from colon cell line vesicles. Example 5 shows that KRAS can be sequenced in RNA vesicles derived from plasma samples. In some embodiments, sequencing of KRAS and\/or BRAF nucleic acid within vesicles can be used to detect CRC. The nucleic acid can be RNA, e.g., mRNA. A CRC biosignature can comprise sequencing of KRAS and BRAF RNA isolated from vesicles.","A colon cancer biosignature can comprise any one or more antigens for colon cancer as listed in , any one or more binding agents associated with isolating or detecting a vesicle for characterizing colon cancer (for example, as shown in ), any one or more additional biomarkers, such as shown in .","The biosignature can comprise one or more miRNA selected from the group consisting of miR-24-1, miR-29b-2, miR-20a, miR-10a, miR-32, miR-203, miR-106a, miR-17-5p, miR-30c, miR-223, miR-126, miR-128b, miR-21, miR-24-2, miR-99b, miR-155, miR-213, miR-150, miR-107, miR-191, miR-221, miR-20a, miR-510, miR-92, miR-513, miR-19a, miR-21, miR-20, miR-183, miR-96, miR-135b, miR-31, miR-21, miR-92, miR-222, miR-181b, miR-210, miR-20a, miR-106a, miR-93, miR-335, miR-338, miR-133b, miR-346, miR-106b, miR-153a, miR-219, miR-34a, miR-99b, miR-185, miR-223, miR-211, miR-135a, miR-127, miR-203, miR-212, miR-95, or miR-17-5p, or any combination thereof. The biosignature can also comprise one or more underexpressed miRs such as miR-143, miR-145, miR-143, miR-126, miR-34b, miR-34c, let-7, miR-9-3, miR-34a, miR-145, miR-455, miR-484, miR-101, miR-145, miR-133b, miR-129, miR-124a, miR-30-3p, miR-328, miR-106a, miR-17-5p, miR-342, miR-192, miR-1, miR-34b, miR-215, miR-192, miR-301, miR-324-5p, miR-30a-3p, miR-34c, miR-331, miR-148b, miR-548c-5p, miR-362-3p and miR422a.","The biosignature can comprise assessing one or more genes, such as EFNB1, ERCC1, HER2, VEGF, and EGFR. A biomarker mutation for colon cancer that can be assessed in a vesicle can also include one or more mutations of EGFR, KRAS, VEGFA, B-Raf, APC, or p53. The biosignature can also comprise one or more proteins, ligands, or peptides that can be assessed of a vesicle, such as AFRs, Rabs, ADAM10, CD44, NG2, ephrin-B1, MIF, b-catenin, Junction, plakoglobin, glalectin-4, RACK1, tetrspanin-8, FasL, TRAIL, A33, CEA, EGFR, dipeptidase 1, hsc-70, tetraspanins, ESCRT, TS, PTEN, or TOPO1.","A vesicle can be isolated and assayed for to provide a diagnostic, prognostic or theranostic profile, such as the stage of the cancer, the efficacy of the cancer, or other characteristics of the cancer. Alternatively, the esicle can be directly assayed from a sample, such that the vesicles are not purified or concentrated prior to assaying for a biosignature associated with colon cancer.","As depicted in , a GI cancer, such as colon cancer, a biosignature can comprise detection of EpCam, CD63, CD81, CD9, CD66, or any combination thereof, of a vesicle. Furthermore, a colon cancer-biosignature for various stages of cancer can comprise CD63, CD9, EpCam, or any combination thereof (see for example, ). For example, the biosignature can comprise CD9 and EpCam. In some embodiments, the GI cancer biosignature comprises one or more miRNA selected from the group consisting of miR-548c-5p, miR-362-3p, miR-422a, miR-597, miR-429, miR-200a, and miR-200b. These miRNAs can be overexpressed in GI cancers, as shown in . The miRNA signature can be combined with the biomarkers listed above. The biosignatures can provide a diagnostic, prognostic or theranostic profile, such as the stage of the cancer, the efficacy of the cancer, or other characteristics of the cancer.","The invention provides for assessing a gastrointestinal disorder comprising detecting a presence or level of one or more circulating biomarker selected from the biomarkers listed above. The one or more circulating biomarker can also be selected from CD9, EGFR, NGAL, CD81, STEAP, CD24, A33, CD66E, EPHA2, Ferritin, GPR30, GPR110, MMP9, OPN, p53, TMEM211, TROP2, TGM2, TIMP, EGFR, DR3, UNC93A, MUC17, EpCAM, MUC1, MUC2, TSG101, CD63, B7H3, and a combination thereof. The one or more circulating biomarkers can be selected from the following: DR3, STEAP, epha2, TMEM211, unc93A, A33, CD24, NGAL, EpCam, MUC17, TROP2, TETS, and a combination thereof. Further still, the one or more circulating biomarkers can be selected from the following: A33, AFP, ALIX, ALX4, ANCA, APC, ASCA, AURKA, AURKB, B7H3, BANK1, BCNP1, BDNF, CA-19-9, CCSA-2, CCSA-3&4, CD10, CD24, CD44, CD63, CD66 CEA, CD66e CEA, CD81, CD9, CDA, C-Erb2, CRMP-2, CRP, CRTN, CXCL12, CYFRA21-1, DcR3, DLL4, DR3, EGFR, Epcam, EphA2, FASL, FRT, GAL3, GDF15, GPCR (GPR110), GPR30, GRO-1, HBD 1, HBD2, HNP1-3, IL-1B, IL8, IMP3, L1CAM, LAMN, MACC-1, MGC20553, MCP-1, M-CSF, MIC1, MIF, MMP7, MMP9, MS4A1, MUC1, MUC17, MUC2, Ncam, NGAL, NNMT, OPN, p53, PCSA, PDGFRB, PRL, PSMA, PSME3, Reg IV, SCRN1, Sept-9, SPARC, SPON2, SPR, SRVN, TFF3, TGM2, TIMP-1, TMEM211, TNF-alpha, TPA, TPS, Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK, UNC93A, and VEGFA, and a combination thereof. In an embodiment, the circulating biomarkers comprise one or more of miR 92, miR 21, miR 9 and miR 491, and\/or one or more of hsa-miR-376c, hsa-miR-215, hsa-miR-652, hsa-miR-582-5p, hsa-miR-324-5p, hsa-miR-1296, hsa-miR-28-5p, hsa-miR-190, hsa-miR-590-5p, hsa-miR-202, and hsa-miR-195. In another embodiment, the circulating biomarkers comprise one or more of TMEM211, MUC1, CD24 and\/or GPR110 (GPCR 110). The circulating biomarkers can be associated with vesicles, e.g., vesicle surface markers or vesicle payload. The gastrointestinal disorders include without limitation a benign disorder such as benign polyps, or a cancer such as colorectal cancer, including cancers of various stages and grades. See, e.g., Table 6.",{"@attributes":{"id":"p-0941","num":"0943"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"315pt","align":"center"}},"thead":{"row":{"entry":"TABLE 6"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Biomarkers for Gastrointestinal Disorders"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"252pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Illustrative Disorder","Biomarkers"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Colorectal cancer","CD9, EGFR, NGAL, CD81, STEAP, CD24, A33, CD66E, EPHA2, Ferritin, GPR30,"]},{"entry":[{},"GPR110, MMP9, OPN, p53, TMEM211, TROP2, TGM2, TIMP, EGFR, DR3,"]},{"entry":[{},"UNC93A, MUC17, EpCAM, MUC1, MUC2, TSG101, CD63, B7H3"]},{"entry":["Colorectal cancer","DR3, STEAP, epha2, TMEM211, unc93A, A33, CD24, NGAL, EpCam, MUC17,"]},{"entry":[{},"TROP2, TETS"]},{"entry":["Colorectal cancer","A33, AFP, ALIX, ALX4, ANCA, APC, ASCA, AURKA, AURKB, B7H3, BANK1,"]},{"entry":[{},"BCNP1, BDNF, CA-19-9, CCSA-2, CCSA-3&4, CD10, CD24, CD44, CD63, CD66"]},{"entry":[{},"CEA, CD66e CEA, CD81, CD9, CDA, C-Erb2, CRMP-2, CRP, CRTN, CXCL12,"]},{"entry":[{},"CYFRA21-1, DcR3, DLL4, DR3, EGFR, Epcam, EphA2, FASL, FRT, GAL3,"]},{"entry":[{},"GDF15, GPCR (GPR110), GPR30, GRO-1, HBD 1, HBD2, HNP1-3, IL-1B, IL8,"]},{"entry":[{},"IMP3, L1CAM, LAMN, MACC-1, MGC20553, MCP-1, M-CSF, MIC1, MIF,"]},{"entry":[{},"MMP7, MMP9, MS4A1, MUC1, MUC17, MUC2, Ncam, NGAL, NNMT, OPN,"]},{"entry":[{},"p53, PCSA, PDGFRB, PRL, PSMA, PSME3, Reg IV, SCRN1, Sept-9, SPARC,"]},{"entry":[{},"SPON2, SPR, SRVN, TFF3, TGM2, TIMP-1, TMEM211, TNF-alpha, TPA, TPS,"]},{"entry":[{},"Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK, UNC93A, VEGFA"]},{"entry":["Colorectal cancer","miR 92, miR 21, miR 9 and miR 491"]},{"entry":["Colorectal cancer","TMEM211, MUC1, CD24 and\/or GPR110 (GPCR 110)"]},{"entry":["Colorectal cancer","hsa-miR-376c, hsa-miR-215, hsa-miR-652, hsa-miR-582-5p, hsa-miR-324-5p, hsa-"]},{"entry":[{},"miR-1296, hsa-miR-28-5p, hsa-miR-190, hsa-miR-590-5p, hsa-miR-202, hsa-miR-"]},{"entry":[{},"195"]},{"entry":["Colorectal cancer","A26C1A, A26C1B, A2M, ACAA2, ACE, ACOT7, ACP1, ACTA1, ACTA2, ACTB,"]},{"entry":["vesicle markers","ACTBL2, ACTBL3, ACTC1, ACTG1, ACTG2, ACTN1, ACTN2, ACTN4,"]},{"entry":[{},"ACTR3, ADAM10, ADSL, AGR2, AGR3, AGRN, AHCY, AHNAK, AKR1B10,"]},{"entry":[{},"ALB, ALDH16A1, ALDH1A1, ALDOA, ANXA1, ANXA11, ANXA2, ANXA2P2,"]},{"entry":[{},"ANXA4, ANXA5, ANXA6, AP2A1, AP2A2, APOA1, ARF1, ARF3, ARF4, ARF5,"]},{"entry":[{},"ARF6, ARHGDIA, ARPC3, ARPC5L, ARRDC1, ARVCF, ASCC3L1, ASNS,"]},{"entry":[{},"ATP1A1, ATP1A2, ATP1A3, ATP1B1, ATP4A, ATP5A1, ATP5B, ATP5I, ATP5L,"]},{"entry":[{},"ATP5O, ATP6AP2, B2M, BAIAP2, BAIAP2L1, BRI3BP, BSG, BUB3, C1orf58,"]},{"entry":[{},"C5orf32, CAD, CALM1, CALM2, CALM3, CAND1, CANX, CAPZA1, CBR1,"]},{"entry":[{},"CBR3, CCT2, CCT3, CCT4, CCT5, CCT6A, CCT7, CCT8, CD44, CD46, CD55,"]},{"entry":[{},"CD59, CD63, CD81, CD82, CD9, CDC42, CDH1, CDH17, CEACAM5, CFL1,"]},{"entry":[{},"CFL2, CHMP1A, CHMP2A, CHMP4B, CKB, CLDN3, CLDN4, CLDN7, CLIC1,"]},{"entry":[{},"CLIC4, CLSTN1, CLTC, CLTCL1, CLU, COL12A1, COPB1, COPB2, CORO1C,"]},{"entry":[{},"COX4I1, COX5B, CRYZ, CSPG4, CSRP1, CST3, CTNNA1, CTNNB1, CTNND1,"]},{"entry":[{},"CTTN, CYFIP1, DCD, DERA, DIP2A, DIP2B, DIP2C, DMBT1, DPEP1, DPP4,"]},{"entry":[{},"DYNC1H1, EDIL3, EEF1A1, EEF1A2, EEF1AL3, EEF1G, EEF2, EFNB1, EGFR,"]},{"entry":[{},"EHD1, EHD4, EIF3EIP, EIF3I, EIF4A1, EIF4A2, ENO1, ENO2, ENO3, EPHA2,"]},{"entry":[{},"EPHA5, EPHB1, EPHB2, EPHB3, EPHB4, EPPK1, ESD, EZR, F11R, F5, F7,"]},{"entry":[{},"FAM125A, FAM125B, FAM129B, FASLG, FASN, FAT, FCGBP, FER1L3,"]},{"entry":[{},"FKBP1A, FLNA, FLNB, FLOT1, FLOT2, G6PD, GAPDH, GARS, GCN1L1,"]},{"entry":[{},"GDI2, GK, GMDS, GNA13, GNAI2, GNAI3, GNAS, GNB1, GNB2, GNB2L1,"]},{"entry":[{},"GNB3, GNB4, GNG12, GOLGA7, GPA33, GPI, GPRC5A, GSN, GSTP1, H2AFJ,"]},{"entry":[{},"HADHA, hCG_1757335, HEPH, HIST1H2AB, HIST1H2AE, HIST1H2AJ,"]},{"entry":[{},"HIST1H2AK, HIST1H4A, HIST1H4B, HIST1H4C, HIST1H4D, HIST1H4E,"]},{"entry":[{},"HIST1H4F, HIST1H4H, HIST1H4I, HIST1H4J, HIST1H4K, HIST1H4L,"]},{"entry":[{},"HIST2H2AC, HIST2H4A, HIST2H4B, HIST3H2A, HIST4H4, HLA-A, HLA-"]},{"entry":[{},"A29.1, HLA-B, HLA-C, HLA-E, HLA-H, HNRNPA2B1, HNRNPH2, HPCAL1,"]},{"entry":[{},"HRAS, HSD17B4, HSP90AA1, HSP90AA2, HSP90AA4P, HSP90AB1,"]},{"entry":[{},"HSP90AB2P, HSP90AB3P, HSP90B1, HSPA1A, HSPA1B, HSPA1L, HSPA2,"]},{"entry":[{},"HSPA4, HSPA5, HSPA6, HSPA7, HSPA8, HSPA9, HSPD1, HSPE1, HSPG2,"]},{"entry":[{},"HYOU1, IDH1, IFITM1, IFITM2, IFITM3, IGH@, IGHG1, IGHG2, IGHG3,"]},{"entry":[{},"IGHG4, IGHM, IGHV4-31, IGK@, IGKC, IGKV1-5, IGKV2-24, IGKV3-20,"]},{"entry":[{},"IGSF3, IGSF8, IQGAP1, IQGAP2, ITGA2, ITGA3, ITGA6, ITGAV, ITGB1,"]},{"entry":[{},"ITGB4, JUP, KIAA0174, KIAA1199, KPNB1, KRAS, KRT1, KRT10, KRT13,"]},{"entry":[{},"KRT14, KRT15, KRT16, KRT17, KRT18, KRT19, KRT2, KRT20, KRT24,"]},{"entry":[{},"KRT25, KRT27, KRT28, KRT3, KRT4, KRT5, KRT6A, KRT6B, KRT6C, KRT7,"]},{"entry":[{},"KRT75, KRT76, KRT77, KRT79, KRT8, KRT9, LAMA5, LAMP1, LDHA, LDHB,"]},{"entry":[{},"LFNG, LGALS3, LGALS3BP, LGALS4, LIMA1, LIN7A, LIN7C, LOC100128936,"]},{"entry":[{},"LOC100130553, LOC100133382, LOC100133739, LOC284889, LOC388524,"]},{"entry":[{},"LOC388720, LOC442497, LOC653269, LRP4, LRPPRC, LRSAM1, LSR, LYZ,"]},{"entry":[{},"MAN1A1, MAP4K4, MARCKS, MARCKSL1, METRNL, MFGE8, MICA, MIF,"]},{"entry":[{},"MINK1, MITD1, MMP7, MOBKL1A, MSN, MTCH2, MUC13, MYADM,"]},{"entry":[{},"MYH10, MYH11, MYH14, MYH9, MYL6, MYL6B, MYO1C, MYO1D, NARS,"]},{"entry":[{},"NCALD, NCSTN, NEDD4, NEDD4L, NME1, NME2, NOTCH1, NQO1, NRAS,"]},{"entry":[{},"P4HB, PCBP1, PCNA, PCSK9, PDCD6, PDCD6IP, PDIA3, PDXK, PEBP1, PFN1,"]},{"entry":[{},"PGK1, PHB, PHB2, PKM2, PLEC1, PLEKHB2, PLSCR3, PLXNA1, PLXNB2,"]},{"entry":[{},"PPIA, PPIB, PPP2R1A, PRDX1, PRDX2, PRDX3, PRDX5, PRDX6, PRKAR2A,"]},{"entry":[{},"PRKDC, PRSS23, PSMA2, PSMC6, PSMD11, PSMD3, PSME3, PTGFRN, PTPRF,"]},{"entry":[{},"PYGB, QPCT, QSOX1, RAB10, RAB11A, RAB11B, RAB13, RAB14, RAB15,"]},{"entry":[{},"RAB1A, RAB1B, RAB2A, RAB33B, RAB35, RAB43, RAB4B, RAB5A, RAB5B,"]},{"entry":[{},"RAB5C, RAB6A, RAB6B, RAB7A, RAB8A, RAB8B, RAC1, RAC3, RALA,"]},{"entry":[{},"RALB, RAN, RANP1, RAP1A, RAP1B, RAP2A, RAP2B, RAP2C, RDX, REG4,"]},{"entry":[{},"RHOA, RHOC, RHOG, ROCK2, RP11-631M21.2, RPL10A, RPL12, RPL6, RPL8,"]},{"entry":[{},"RPLP0, RPLP0-like, RPLP1, RPLP2, RPN1, RPS13, RPS14, RPS15A, RPS16,"]},{"entry":[{},"RPS18, RPS20, RPS21, RPS27A, RPS3, RPS4X, RPS4Y1, RPS4Y2, RPS7, RPS8,"]},{"entry":[{},"RPSA, RPSAP15, RRAS, RRAS2, RUVBL1, RUVBL2, S100A10, S100A11,"]},{"entry":[{},"S100A14, S100A16, S100A6, S100P, SDC1, SDC4, SDCBP, SDCBP2, SERINC1,"]},{"entry":[{},"SERINC5, SERPINA1, SERPINF1, SETD4, SFN, SLC12A2, SLC12A7, SLC16A1,"]},{"entry":[{},"SLC1A5, SLC25A4, SLC25A5, SLC25A6, SLC29A1, SLC2A1, SLC3A2,"]},{"entry":[{},"SLC44A1, SLC7A5, SLC9A3R1, SMPDL3B, SNAP23, SND1, SOD1, SORT1,"]},{"entry":[{},"SPTAN1, SPTBN1, SSBP1, SSR4, TACSTD1, TAGLN2, TBCA, TCEB1, TCP1,"]},{"entry":[{},"TF, TFRC, THBS1, TJP2, TKT, TMED2, TNFSF10, TNIK, TNKS1BP1, TNPO3,"]},{"entry":[{},"TOLLIP, TOMM22, TPI1, TPM1, TRAP1, TSG101, TSPAN1, TSPAN14,"]},{"entry":[{},"TSPAN15, TSPAN6, TSPAN8, TSTA3, TTYH3, TUBA1A, TUBA1B, TUBA1C,"]},{"entry":[{},"TUBA3C, TUBA3D, TUBA3E, TUBA4A, TUBA4B, TUBA8, TUBB, TUBB2A,"]},{"entry":[{},"TUBB2B, TUBB2C, TUBB3, TUBB4, TUBB4Q, TUBB6, TUFM, TXN, UBA1,"]},{"entry":[{},"UBA52, UBB, UBC, UBE2N, UBE2V2, UGDH, UQCRC2, VAMP1, VAMP3,"]},{"entry":[{},"VAMP8, VCP, VIL1, VPS25, VPS28, VPS35, VPS36, VPS37B, VPS37C, WDR1,"]},{"entry":[{},"YWHAB, YWHAE, YWHAG, YWHAH, YWHAQ, YWHAZ"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"The colon cancer can be characterized using one or more processes disclosed herein with at least 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, or 70% sensitivity. The colon cancer can be characterized with at least 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, or 87% sensitivity. For example, the colon cancer can be characterized with at least 87.1, 87.2, 87.3, 87.4, 87.5, 87.6, 87.7, 87.8, 87.9, 88.0, or 89% sensitivity, such as with at least 90% sensitivity, such as at least 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100% sensitivity.","The colon cancer of a subject can also be characterized with at least 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, or 97% specificity, such as with at least 97.1, 97.2, 97.3, 97.4, 97.5, 97.6, 97.7, 97.8, 97.8, 97.9, 98.0, 98.1, 98.2, 98.3, 98.4, 98.5, 98.6, 98.7, 98.8, 98.9, 99.0, 99.1, 99.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99.8, 99.9 or 100% specificity.","The colon cancer can also be characterized with at least 70% sensitivity and at least 80, 90, 95, 99, or 100% specificity; at least 80% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 85% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 86% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 87% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 88% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 89% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 90% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 95% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; at least 99% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity; or at least 100% sensitivity and at least 80, 85, 90, 95, 99, or 100% specificity.","Furthermore, the confidence level for determining the specificity, sensitivity, and\/or other statistical performance measures may be with at least 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, or 99% confidence.","Ovarian Cancer","A biosignature for characterizing ovarian cancer can comprise an antigen associated with ovarian cancer (for example, as shown in ), and one or more additional biomarkers, such as shown in . In one embodiment, a biosignature for ovarian cancer can comprise one or more antigens associated with ovarian cancer, such as, but not limited to, CD24, CA125, VEGF1, VEGFR2, HER2, MISIIR, or any combination thereof. The biosignature for ovarian cancer can comprise one or more of the aforementioned antigens and one or more additional biomarker, such as, but not limited to miRNA, mRNA, DNA, or any combination thereof. The biosignature for ovarian cancer can comprise one or more antigens associated with ovarian cancer, such as, but not limited to, CD24, CA125, VEGF1, VEGFR2, HER2, MISIIR, or any combination thereof, with one or more miRNA biomarkers, such as, but not limited to, miR-200a, miR-141, miR-200c, miR-200b, miR-21, miR-141, miR-200a, miR-200b, miR-200c, miR-203, miR-205, miR-214, miR-215, miR-199a, miR-140, miR-145, miR-125b-1, or any combination thereof.","A biosignature for ovarian cancer can comprise one or more antigens associated with ovarian cancer, such as, but not limited to, CD24, CA125, VEGF1, VEGFR2, HER2, MISIIR, or any combination thereof, with one or more miRNA biomarkers (such as the aforementioned miRNA), mRNAs (such as, but not limited to, ERCC1, ER, TOPO1, TOP2A, AR, PTEN, HER2\/neu, EGFR), mutations (including, but not limited to, those relating to KRAS and\/or B-Raf) or any combination thereof.","A vesicle can be isolated, assayed or both, for one or more miRNA and one or more antigens associated with ovarian cancer to provide a diagnostic, prognostic or theranostic profile. Alternatively, the vesicle can be directly assayed from a sample, such that the vesicle is not purified or concentrated prior to assaying for one or more miRNA or antigens associated with ovarian cancer.","Organ Transplant Rejection and Autoimmune Conditions","A vesicle can also be used for determining phenotypes such as organ distress and\/or organ transplant rejection. As used herein organ transplant includes partial organ or tissue transplant. The presence, absence or levels of one or more biomarkers present in a vesicle can be assessed to monitor organ rejection or success. The level or amount of vesicles in the sample can also be used to assess organ rejection or success. The assessment can be determined with at least 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, or 99% specificity, sensitivity, or both. For example, the assessment can be determined with at least 97.5, 97.6, 97.7, 97.8, 97.8, 97.9, 98.0, 98.1, 98.2, 98.3, 98.4, 98.5, 98.6, 98.7, 98.8, 98.9, 99.0, 99.1, 998.2, 99.3, 99.4, 99.5, 99.6, 99.7, 99.8, 99.9% sensitivity, specificity, or both","The vesicle can be purified or concentrated prior to analysis. Alternatively, the level, or amount, of vesicles can be directly assayed from a sample, without prior purification or concentration. The vesicle can be quantitated, su. For example, a cell or tissue-specific vesicle can be isolated using one or more binding agents specific for a particular organ. The cell-of-origin specific vesicle can be assessed for one or more molecular features, such as one or more biomarkers associated with organ distress or organ transplant rejection. The presence, absence or levels of one or more biomarkers present, can be assessed to monitor organ rejection or success.","One or more vesicles can be analyzed for the assessment, detection or diagnosis of the rejection of a tissue or organ transplant by a subject. The tissue or organ transplant rejection can be hyperacute, acute, or chronic rejection. The vesicle can also be analyzed for the assessment, detection or diagnosis of graft versus host disease in a subject. The subject can be the recipient of an autogenic, allogenic or xenogenic tissue or organ transplant.","The vesicle can also be analyzed to detect the rejection of a tissue or organ transplant. The vesicle may be produced by the tissue or organ transplant. Such tissues or organs include, but are not limited to, a heart, lung, pancreas, kidney, eye, cornea, muscle, bone marrow, skin, cartilage, bone, appendages, hair, face, tendon, stomach, intestine, vein, artery, differentiated cells, partially differentiated cells or stem cells.","The vesicle can comprise at least one biomarker which is used to assess, diagnose or determine the probability or occurrence of rejection of a tissue or organ transplant by a subject. A biomarker can also be used to assess, diagnose or detect graft versus host disease in a subject. The biomarker can be a protein, a polysaccharide, a fatty acid or a nucleic acid (such as DNA or RNA). The biomarker can be associated with the rejection of a specific tissue or organ or systemic organ failure. More than one biomarker can be analyzed, for example, one or more proteins marker can be analyzed in combination with one or more nucleic acid markers. The biomarker may be an intracellular or extracellular marker.","The vesicle can also be analyzed for at least one marker for the assessment, detection or diagnosis of cell apoptosis or necrosis associated with, or the causation of, rejection of a tissue or organ transplant by a subject.","The presence of a biomarker can be indicative of the rejection of a tissue or an organ by a subject, wherein the biomarker includes, but is not limited to, CD40, CD40 ligand, N-acetylmuramoyl-L-alanine amidase precursor, adiponectin, AMBP protein precursor, C4b-binding protein a-chain precursor, ceruloplasmin precursor, complement C3 precursor, complement component C9 precursor, complement factor D precursor, alpha1-B-glycoprotein, beta2-glycoprotein I precursor, heparin cofactor II precursor, Immunoglobulin mu chain C region protein, Leucine-rich alpha2-glycoprotein precursor, pigment epithelium-derived factor precursor, plasma retinol-binding protein precursor, translation initiation factor 3 subunit 10, ribosomal protein L7, beta-transducin, 1-TRAF, or lysyl-tRNA synthetase.","Rejection of a kidney by a subject can also be detected by analyzing vesicles for the presence of beta-transducin. Rejection of transplanted tissue can also be detected by isolating a cell-of-origin specific vesicles from CD40-expressing cells and detecting for the increase of Bcl-2 or TNFalpha.","Rejection of a liver transplant by a subject can be detected by analyzing the vesicles for the presence of an F1 antigen marker. The F1 antigen is, without being bound to theory, specific to liver to and can be used to detect an increase in liver cell-of-origin specific vesicles. This increase can be used as an early indication of organ distress\/rejection.","Bronchiolitis obliterans due to bone marrow and\/or lung transplantation or other causes, or graft atherosclerosis\/graft phlebosclerosis can also be diagnosed by the analysis of a vesicle.","A vesicle can also be analyzed for the detection, diagnosis or assessment of an autoimmune or other immunological reaction-related phenotype in a subject. Examples of such a disorder include, but are not limited to, systemic lupus erythematosus (SLE), discoid lupus, lupus nephritis, sarcoidosis, inflammatory arthritis, including juvenile arthritis, rheumatoid arthritis, psoriatic arthritis, Reiter's syndrome, ankylosing spondylitis, and gouty arthritis, multiple sclerosis, hyper IgE syndrome, polyarteritis nodosa, primary biliary cirrhosis, inflammatory bowel disease, Crohn's disease, celiac's disease (gluten-sensitive enteropathy), autoimmune hepatitis, pernicious anemia, autoimmune hemolytic anemia, psoriasis, scleroderma, myasthenia gravis, autoimmune thrombocytopenic purpura, autoimmune thyroiditis, Grave's disease, Hasimoto's thyroiditis, immune complex disease, chronic fatigue immune dysfunction syndrome (CFIDS), polymyositis and dermatomyositis, cryoglobulinemia, thrombolysis, cardiomyopathy, pemphigus vulgaris, pulmonary interstitial fibrosis, asthma, Churg-Strauss syndrome (allergic granulomatosis), atopic dermatitis, allergic and irritant contact dermatitis, urtecaria, IgE-mediated allergy, atherosclerosis, vasculitis, idiopathic inflammatory myopathies, hemolytic disease, Alzheimer's disease, chronic inflammatory demyelinating polyneuropathy and AIDs.","One or more biomarkers from the vesicles can be used to assess, diagnose or determine the probability of the occurrence of an autoimmune or other immunological reaction-related disorder in a subject. The biomarker can be a protein, a polysaccharide, a fatty acid or a nucleic acid (such as DNA or RNA). The biomarker can be associated with a specific autoimmune disorder, a systemic autoimmune disorder, or other immunological reaction-related disorder. More than one biomarker can be analyzed. For example one or more protein markers can be analyzed in combination with one or more nucleic acid markers. The biomarker can be an intracellular or extracellular marker. The biomarker can also be used to detect, diagnose or assess inflammation.","Analysis of vesicles from subjects can be used identify subjects with inflammation associated with asthma, sarcoidosis, emphysema, cystic fibrosis, idiopathic pulmonary fibrosis, chronic bronchitis, allergic rhinitis and allergic diseases of the lung such as hypersensitivity pneumonitis, eosinophilic pneumonia, as well as pulmonary fibrosis resulting from collagen, vascular, and autoimmune diseases such as rheumatoid arthritis.","Theranosis","As disclosed herein, methods are disclosed for characterizing a phenotype for a subject by assessing one or more biomarkers, including vesicle biomarkers and\/or circulating biomarkers. The biomarkers can be assessed using methods for multiplexed analysis of vesicle biomarkers disclosed herein. Characterizing a phenotype can include providing a theranosis for a subject, such as determining if a subject is predicted to respond to a treatment or is predicted to be non-responsive to a treatment. A subject that responds to a treatment can be termed a responder whereas a subject that does not respond can be termed a non-responder. A subject suffering from a condition can be considered to be a responder for a treatment based on, but not limited to, an improvement of one or more symptoms of the condition; a decrease in one or more side effects of an existing treatment; an increased improvement, or rate of improvement, in one or more symptoms as compared to a previous or other treatment; or prolonged survival as compared to without treatment or a previous or other treatment. For example, a subject suffering from a condition can be considered to be a responder to a treatment based on the beneficial or desired clinical results including, but are not limited to, alleviation or amelioration of one or more symptoms, diminishment of extent of disease, stabilized (i.e., not worsening) state of disease, preventing spread of disease, delay or slowing of disease progression, amelioration or palliation of the disease state, and remission (whether partial or total), whether detectable or undetectable. Treatment also includes prolonging survival as compared to expected survival if not receiving treatment or if receiving a different treatment.","The systems and methods disclosed herein can be used to select a candidate treatment for a subject in need thereof. Selection of a therapy can be based on one or more characteristics of a vesicle, such as the biosignature of a vesicle, the amount of vesicles, or both. Vesicle typing or profiling, such as the identification of the biosignature of a vesicle, the amount of vesicles, or both, can be used to identify one or more candidate therapeutic agents for an individual suffering from a condition. For example, vesicle profiling can be used to determine if a subject is a non-responder or responder to a particular therapeutic, such as a cancer therapeutic if the subject is suffering from a cancer.","Vesicle profiling can be used to provide a diagnosis or prognosis for a subject, and a therapy can be selected based on the diagnosis or prognosis. Alternatively, therapy selection can be directly based on a subject's vesicle profile. Furthermore, a subject's vesicle profile can be used to follow the evolution of a disease, to evaluate the efficacy of a medication, adapt an existing treatment for a subject suffering from a disease or condition, or select a new treatment for a subject suffering from a disease or condition.","A subject's response to a treatment can be assessed using biomarkers, including vesicles, microRNA, and other circulating biomarkers. In one embodiment, a subject is determined, classified, or identified as a non-responder or responder based on the subject's vesicle profile assessed prior to any treatment. During pretreatment, a subject can be classifed as a non-responder or responder, thereby reducing unnecessary treatment options, and avoidance of possible side effects from ineffective therapeutics. Furthermore, the subject can be identified as a responder to a particular treatment, and thus vesicle profiling can be used to prolong survival of a subject, improve the subject's symptoms or condition, or both, by providing personalized treatment options. Thus, a subject suffering from a condition can have a biosignature generated from vesicles and other circulating biomarkers using one or more systems and methods disclosed herein, and the profile can then be used to determine whether a subject is a likely non-responder or responder to a particular treatment for the condition. Based on use of the biosignature to predict whether the subject is a non-responder or responder to the initially contemplated treatment, a particular treatment contemplated for treating the subject's condition can be selected for the subject, or another potentially more optimal treatment can be selected.","In one embodiment, a subject suffering from a condition is currently being treated with a therapeutic. A sample can be obtained from the subject before treatment and at one or more timepoints during treatment. A biosignature including vesicles or other biomarkers from the samples can be assessed and used to determine the subject's response to the drug, such as based on a change in the biosignature over time. If the subject is not responding to the treatment, e.g., the biosignature does not indicate that the patient is responding, the subject can be classified as being non-responsive to the treatment, or a non-responder. Similarly, one or more biomarkers associated with a worsening condition may be detected such that the biosignature is indicative of patient's failure to respond favorably to the treatment. In another example, one or more biomarkers associated with the condition remain the same despite treatment, indicating that the condition is not improving. Thus, based on the biosignature, a treatment regimen for the subject can be changed or adapted, including selection of a different therapeutic.","Alternatively, the subject can be determined to be responding to the treatment, and the subject can be classified as being responsive to the treatment, or a responder. For example, one or more biomarkers associated with an improvement in the condition or disorder may be detected. In another example, one or more biomarkers associated with the condition changes, thus indicating an improvement. Thus, the existing treatment can be continued. In another embodiment, even when there is an indiciation of improvement, the existing treatment may be adapted or changed if the biosignature indicates that another line of treatment may be more effective. The existing treatment may be combined with another therapeutic, the dosage of the current therapeutic may be increased, or a different candidate treatment or therapeutic may be selected. Criteria for selecting the different candidate treatment can depend on the setting. In one embodiment, the candidate treatment may have been known to be effective for subjects with success on the existing treatment. In another embodiment, the candidate treatment may have been known to be effective for other subjects with a similar biosignature.","In some embodiments, the subject is undergoing a second, third or more line of treatment, such as cancer treatment. A biosignature according to the invention can be determined for the subject prior to a second, third or more line of treatment, to determine whether a subject would be a responder or non-resonder to the second, third or more line of treatment. In another embodiment, a biosignature is determined for the subject during the second, third or more line of treatment, to determine if the subject is responding to the second, third or more line of treatment.","The methods and systems described herein for assessing one or more vesicles can be used to determine if a subject suffering from a condition is responsive to a treatment, and thus can be used to select a treatment that improves one or more symptoms of the condition; decreases one or more side effects of an existing treatment; increases the improvement, or rate of improvement, in one or more symptoms as compared to a previous or other treatment; or prolongs survival as compared to without treatment or a previous or other treatment. Thus, the methods described herein can be used to prolong survival of a subject by providing personalized treatment options, and\/or may reduce unnecessary treatment options and unnecessary side effects for a subject.","The prolonged survival can be an increased progression-free survival (PFS), which denotes the chances of staying free of disease progression for an individual or a group of individuals suffering from a disease, e.g., a cancer, after initiating a course of treatment. It can refer to the percentage of individuals in the group whose disease is likely to remain stable (e.g., not show signs of progression) after a specified duration of time. Progression-free survival rates are an indication of the effectiveness of a particular treatment. In other embodiments, the prolonged survival is disease-free survival (DFS), which denotes the chances of staying free of disease after initiating a particular treatment for an individual or a group of individuals suffering from a cancer. It can refer to the percentage of individuals in the group who are likely to be free of disease after a specified duration of time. Disease-free survival rates are an indication of the effectiveness of a particular treatment. Two treatment strategies can be compared on the basis of the disease-free survival that is achieved in similar groups of patients. Disease-free survival is often used with the term overall survival when cancer survival is described.","The candidate treatment selected by vesicle profiling as described herein can be compared to a non-vesicle profiling selected treatment by comparing the progression free survival (PFS) using therapy selected by vesicle profiling (period B) with PFS for the most recent therapy on which the subject has just progressed (period A). In one setting, a PFSB\/PFSA ratio\u22671.3 is used to indicate that the vesicle profiling selected therapy provides benefit for subject (see for example, Robert Temple, . Edited by Wu Ningano and G. T. Thicker John Wiley and Sons Ltd. 1995; Von Hoff D. D. 4: 1079, 1999: Dhani et al. 15: 118-123, 2009).","Other methods of comparing the treatment selected by vesicle profiling can be compared to a non-vesicle profiling selected treatment by determine response rate (RECIST) and percent of subjects without progression or death at 4 months. The term \u201cabout\u201d as used in the context of a numerical value for PFS means a variation of +\/\u2212 ten percent (10%) relative to the numerical value. The PFS from a treatment selected by vesicle profiling can be extended by at least 10%, 15%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or at least 90% as compared to a non-vesicle profiling selected treatment. In some embodiments, the PFS from a treatment selected by vesicle profiling can be extended by at least 100%, 150%, 200%, 300%, 400%, 500%, 600%, 700%, 800%, 900%, or at least about 1000% as compared to a non-vesicle profiling selected treatment. In yet other embodiments, the PFS ratio (PFS on vesicle profiling selected therapy or new treatment\/PFS on prior therapy or treatment) is at least about 1.3. In yet other embodiments, the PFS ratio is at least about 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, or 2.0. In yet other embodiments, the PFS ratio is at least about 3, 4, 5, 6, 7, 8, 9 or 10.","Similarly, the DFS can be compared in subjects whose treatment is selected with or without determining a biosignature according to the invention. The DFS from a treatment selected by vesicle profiling can be extended by at least 10%, 15%, 20%, 30%, 40%, 50%, 60%, 70%, 80%, or at least 90% as compared to a non-vesicle profiling selected treatment. In some embodiments, the DFS from a treatment selected by vesicle profiling can be extended by at least 100%, 150%, 200%, 300%, 400%, 500%, 600%, 700%, 800%, 900%, or at least about 1000% as compared to a non-vesicle profiling selected treatment. In yet other embodiments, the DFS ratio (DFS on vesicle profiling selected therapy or new treatment\/DFS on prior therapy or treatment) is at least about 1.3. In yet other embodiments, the DFS ratio is at least about 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, or 2.0. In yet other embodiments, the DFS ratio is at least about 3, 4, 5, 6, 7, 8, 9 or 10.","In some embodiments, the candidate treatment selected by microvescile profiling does not increase the PFS ratio or the DFS ratio in the subject; nevertheless vesicle profiling provides subject benefit. For example, in some embodiments no known treatment is available for the subject. In such cases, vesicle profiling provides a method to identify a candidate treatment where none is currently identified. The vesicle profiling may extend PFS, DFS or lifespan by at least 1 week, 2 weeks, 3 weeks, 4 weeks, 1 month, 5 weeks, 6 weeks, 7 weeks, 8 weeks, 2 months, 9 weeks, 10 weeks, 11 weeks, 12 weeks, 3 months, 4 months, 5 months, 6 months, 7 months, 8 months, 9 months, 10 months, 11 months, 12 months, 13 months, 14 months, 15 months, 16 months, 17 months, 18 months, 19 months, 20 months, 21 months, 22 months, 23 months, 24 months or 2 years. The vesicle profiling may extend PFS, DFS or lifespan by at least 2\u00bd years, 3 years, 4 years, 5 years, or more. In some embodiments, the methods of the invention improve outcome so that subject is in remission.","The effectiveness of a treatment can be monitored by other measures. A complete response (CR) comprises a complete disappearance of the disease: no disease is evident on examination, scans or other tests. A partial response (PR) refers to some disease remaining in the body, but there has been a decrease in size or number of the lesions by 30% or more. Stable disease (SD) refers to a disease that has remained relatively unchanged in size and number of lesions. Generally, less than a 50% decrease or a slight increase in size would be described as stable disease. Progressive disease (PD) means that the disease has increased in size or number on treatment. In some embodiments, vesicle profiling according to the invention results in a complete response or partial response. In some embodiments, the methods of the invention result in stable disease. In some embodiments, the invention is able to achieve stable disease where non-vesicle profiling results in progressive disease.","The theranosis based on a biosignature of the invention can be for a phenotype including without limitation those listed herein. Characterizing a phenotype includes determining a theranosis for a subject, such as predicting whether a subject is likely to respond to a treatment (\u201cresponder\u201d) or be non-responsive to a treatment (\u201cnon-responder\u201d). As used herein, identifying a subject as a \u201cresponder\u201d to a treatment or as a \u201cnon-responder\u201d to the treatment comprises identifying the subject as either likely to respond to the treatment or likely to not respond to the treatment, respectively, and does not require determining a definitive prediction of the subject's response. One or more vesicles, or populations of vesicles, obtained from subject are used to determine if a subject is a non-responder or responder to a particular therapeutic, by assessing biomarkers disclosed herein, e.g., those listed in Table 7. Detection of a high or low expression level of a biomarker, or a mutation of a biomarker, can be used to select a candidate treatment, such as a pharmaceutical intervention, for a subject with a condtion. Table 7 contains illustrative conditions and pharmaceutical interventions for those conditions. The table lists biomarkers that affect the efficacy of the intervention. The biomarkers can be assessed using the methods of the invention, e.g., as circulating biomarkers or in association with a vesicle.",{"@attributes":{"id":"p-0978","num":"0980"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"center"}},"thead":{"row":{"entry":"TABLE 7"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Examples of Biomarkers and Pharmaceutical Intervention for a Condition"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Condition","Pharmaceutial intervention","Biomarker"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Peripheral Arterial Disease","Atorvastatin","C-reactive protein(CRP)"]},{"entry":[{},"Simvastatin","serum Amylyoid A (SAA)"]},{"entry":[{},"Rosuvastatin","interleukin-6"]},{"entry":[{},"Pravastatin","intracellular adhesion molecule"]},{"entry":[{},"Fluvastatin","(ICAM)"]},{"entry":[{},"Lovastatin","vascular adhesion molecule"]},{"entry":[{},{},"(VCAM)"]},{"entry":[{},{},"CD40L"]},{"entry":[{},{},"fibrinogen"]},{"entry":[{},{},"fibrin D-dimer"]},{"entry":[{},{},"fibrinopeptide A"]},{"entry":[{},{},"von Willibrand factor"]},{"entry":[{},{},"tissue plasminogen activator"]},{"entry":[{},{},"antigen (t-PA)"]},{"entry":[{},{},"factor VII"]},{"entry":[{},{},"prothrombin fragment 1"]},{"entry":[{},{},"oxidized low density lipoprotein"]},{"entry":[{},{},"(oxLDL)"]},{"entry":[{},{},"lipoprotein A"]},{"entry":["Non-Small Cell Lung Cancer","Erlotinib","EGFR"]},{"entry":[{},"Carboplatin","excision repair cross-"]},{"entry":[{},"Paclitaxel","complementation group 1 (ERCC1)"]},{"entry":[{},"Gefitinib","p53"]},{"entry":[{},{},"Ras"]},{"entry":[{},{},"p27"]},{"entry":[{},{},"class III beta tubulin"]},{"entry":[{},{},"breast cancer gene 1 (BRCA1)"]},{"entry":[{},{},"breast cancer gene 1 (BRCA2)"]},{"entry":[{},{},"ribonucleotide reductase messenger"]},{"entry":[{},{},"1 (RRM1)"]},{"entry":["Colorectal Cancer","Panitumumab","K-ras"]},{"entry":[{},"Cetuximab"]},{"entry":["Breast Cancer","Trastuzumab","HER2"]},{"entry":[{},"Anthracyclines","toposiomerase IIalpha"]},{"entry":[{},"Taxane","estrogen receptor"]},{"entry":[{},"Methotrexate","progesterone receptor"]},{"entry":[{},"fluorouracil"]},{"entry":["Alzheimer's Disease","Donepezil","beta-amyloid protein"]},{"entry":[{},"Galantamine","amyloid precursor protein (APP)"]},{"entry":[{},"Memantine","APP670\/671"]},{"entry":[{},"Rivastigmine","APP693"]},{"entry":[{},"Tacrine","APP692"]},{"entry":[{},{},"APP715"]},{"entry":[{},{},"APP716"]},{"entry":[{},{},"APP717"]},{"entry":[{},{},"APP723"]},{"entry":[{},{},"presenilin 1"]},{"entry":[{},{},"presenilin 2"]},{"entry":[{},{},"cerebrospinal fluid amyloid beta"]},{"entry":[{},{},"protein 42 (CSF-Abeta42)"]},{"entry":[{},{},"cerebrospinal fluid amyloid beta"]},{"entry":[{},{},"protein 40 (CSF-Abeta40)"]},{"entry":[{},{},"F2 isoprostane"]},{"entry":[{},{},"4-hydroxynonenal"]},{"entry":[{},{},"F4 neuroprostane"]},{"entry":[{},{},"acrolein"]},{"entry":["Arrhythmia","Disopyramide","SERCA"]},{"entry":[{},"Flecainide","AAP"]},{"entry":[{},"Lidocaine","Connexin 40"]},{"entry":[{},"Mexiletine","Connexin 43"]},{"entry":[{},"Moricizine","ATP-sensitive potassium channel"]},{"entry":[{},"Procainamide","Kv1.5 channel"]},{"entry":[{},"Propafenone","acetylcholine-activated posassium"]},{"entry":[{},"Quinidine","channel"]},{"entry":[{},"Tocainide"]},{"entry":[{},"Acebutolol"]},{"entry":[{},"Atenolol"]},{"entry":[{},"Betaxolol"]},{"entry":[{},"Bisoprolol"]},{"entry":[{},"Carvedilol"]},{"entry":[{},"Esmolol"]},{"entry":[{},"Metoprolol"]},{"entry":[{},"Nadolol"]},{"entry":[{},"Propranolol"]},{"entry":[{},"Sotalol"]},{"entry":[{},"Timolol"]},{"entry":[{},"Amiodarone"]},{"entry":[{},"Azimilide"]},{"entry":[{},"Bepridil"]},{"entry":[{},"Dofetilide"]},{"entry":[{},"Ibutilide"]},{"entry":[{},"Tedisamil"]},{"entry":[{},"Diltiazem"]},{"entry":[{},"Verapamil"]},{"entry":[{},"Azimilide"]},{"entry":[{},"Dronedarone"]},{"entry":[{},"Amiodarone"]},{"entry":[{},"PM101"]},{"entry":[{},"ATI-2042"]},{"entry":[{},"Tedisamil"]},{"entry":[{},"Nifekalant"]},{"entry":[{},"Ambasilide"]},{"entry":[{},"Ersentilide"]},{"entry":[{},"Trecetilide"]},{"entry":[{},"Almokalant"]},{"entry":[{},"D-sotalol"]},{"entry":[{},"BRL-32872"]},{"entry":[{},"HMR1556"]},{"entry":[{},"L768673"]},{"entry":[{},"Vernakalant"]},{"entry":[{},"AZD70009"]},{"entry":[{},"AVE0118"]},{"entry":[{},"S9947"]},{"entry":[{},"NIP-141\/142"]},{"entry":[{},"XEN-D0101\/2"]},{"entry":[{},"Ranolazine"]},{"entry":[{},"Pilsicainide"]},{"entry":[{},"JTV519"]},{"entry":[{},"Rotigaptide"]},{"entry":[{},"GAP-134"]},{"entry":["Rheumatoid arthritis","Methotrexate","677CC\/1298AA MTHFR"]},{"entry":[{},"infliximab","677CT\/1298AC MTHFR"]},{"entry":[{},"adalimumab","677CT MTHFR"]},{"entry":[{},"etanercept","G80AA RFC-1"]},{"entry":[{},"sulfasalazine","3435TT MDR1 (ABCB1)"]},{"entry":[{},{},"3435TT ABCB1"]},{"entry":[{},{},"AMPD1\/ATIC\/ITPA"]},{"entry":[{},{},"IL1-RN3"]},{"entry":[{},{},"HLA-DRB103"]},{"entry":[{},{},"CRP"]},{"entry":[{},{},"HLA-D4"]},{"entry":[{},{},"HLA DRB-1"]},{"entry":[{},{},"anti-citrulline epitope containing"]},{"entry":[{},{},"peptides"]},{"entry":[{},{},"anti-A1\/RA33"]},{"entry":[{},{},"Erythrocyte sedimentation rate"]},{"entry":[{},{},"(ESR)"]},{"entry":[{},{},"C-reactive protein (CRP)"]},{"entry":[{},{},"SAA (serum amyloid-associated"]},{"entry":[{},{},"protein)"]},{"entry":[{},{},"rheumatoid factor"]},{"entry":[{},{},"IL-1"]},{"entry":[{},{},"TNF"]},{"entry":[{},{},"IL-6"]},{"entry":[{},{},"IL-8"]},{"entry":[{},{},"IL-1Ra"]},{"entry":[{},{},"Hyaluronic acid"]},{"entry":[{},{},"Aggrecan"]},{"entry":[{},{},"Glc-Gal-PYD"]},{"entry":[{},{},"osteoprotegerin"]},{"entry":[{},{},"RNAKL"]},{"entry":[{},{},"carilage oligomeric matrix protein"]},{"entry":[{},{},"(COMP)"]},{"entry":[{},{},"calprotectin"]},{"entry":["Arterial Fibrillation","warfarin","F1.2"]},{"entry":[{},"aspirin","TAT"]},{"entry":[{},"anticoagulants","FPA"]},{"entry":[{},"heparin","beta-throboglobulin"]},{"entry":[{},"ximelagatran","platelet factor 4"]},{"entry":[{},{},"soluble P-selectin"]},{"entry":[{},{},"IL-6"]},{"entry":[{},{},"CRP"]},{"entry":["HIV Infection","Zidovudine","HIV p24 antigen"]},{"entry":[{},"Didanosine","TNF-alpha"]},{"entry":[{},"Zalcitabine","TNFR-II"]},{"entry":[{},"Stavudine","CD3"]},{"entry":[{},"Lamivudine","CD14"]},{"entry":[{},"Saquinavir","CD25"]},{"entry":[{},"Ritonavir","CD27"]},{"entry":[{},"Indinavir","Fas"]},{"entry":[{},"Nevirane","FasL"]},{"entry":[{},"Nelfinavir","beta2 microglobulin"]},{"entry":[{},"Delavirdine","neopterin"]},{"entry":[{},"Stavudine","HIV RNA"]},{"entry":[{},"Efavirenz","HLA-B *5701"]},{"entry":[{},"Etravirine"]},{"entry":[{},"Enfuvirtide"]},{"entry":[{},"Darunavir"]},{"entry":[{},"Abacavir"]},{"entry":[{},"Amprenavir"]},{"entry":[{},"Lonavir\/Ritonavirc"]},{"entry":[{},"Tenofovir"]},{"entry":[{},"Tipranavir"]},{"entry":["Cardiovascular Disease","lisinopril","ACE inhibitor"]},{"entry":[{},"candesartan","angiotensin"]},{"entry":[{},"enalapril"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"Cancer","Vesicle biosignatures can be used in the theranosis of a cancer, such as identifying whether a subject suffering from cancer is a likely responder or non-responder to a particular cancer treatment. The subject methods can be used to theranose cancers including those listed herein, e.g., in the \u201cPhenotype\u201d section above. These include without limitation lung cancer, non-small cell lung cancerm small cell lung cancer (including small cell carcinoma (oat cell cancer), mixed small cell\/large cell carcinoma, and combined small cell carcinoma), colon cancer, breast cancer, prostate cancer, liver cancer, pancreatic cancer, brain cancer, kidney cancer, ovarian cancer, stomach cancer, melanoma, bone cancer, gastric cancer, breast cancer, glioma, gliobastoma, hepatocellular carcinoma, papillary renal carcinoma, head and neck squamous cell carcinoma, leukemia, lymphoma, myeloma, or other solid tumors.","Cancer: Biosignatures","A biosignature can be determined to provide a theranosis for a subject. The biosignature of a vesicle can comprise one or more biomarkers such as, but not limited to, any one or more biomarkers as described herein, such as, but not limited to, those listed in .","The invention provides numerous methods of identifying a biosignature for characterizing a cancer. Further provided herein are biomarkers that are assessed to identify the biosignature. In one embodiment, a biosignature for prostate cancer comprises one or more of the following biomarkers: EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR. In another embodiment, a biosignature for classifying a prostate cancer as being castration-resistant comprises EpCam+, CK+, CD45\u2212 vesicles. In another embodiment, a vesicle biosignature for theranosing small cell lung cancer comprises miR-451, miR-92a-2*, miR-147, and\/or miR-574-5p. In yet another embodiment, a biosignature for the theranosis of colorectal cancer comprises one or more miRs selected from the group consisting of: miR-548c-5p, miR-362-3p, miR-422a, miR-597, miR-429, miR-200a, and miR-200b. The biosignature for theranosing a cancer can comprise one or more of CA IX, CMET, VEGFR2, VEGF, Vimentin, CD44v6, Ckit, Axl, RET (ret proto-oncogene), E Cadherin, and VE Cadherin.","Cancer: Standard of Care","A biosignature of circulating biomarkers, including markers associated with vesicle, in a sample from a subject suffering from a cancer can be used select a candidate treatment for the subject. The biosignature can be determined according to the methods of the invention presented herein. In some embodiments, the candidate treatment comprises a standard of care for the cancer. The biosignature can be used to determine if a subject is a non-responder or responder to a particular treatment or standard of care. The treatment can be a cancer treatment such as radiation, surgery, chemotherapy or a combination thereof. The cancer treatment can be a therapeutic such as anti-cancer agents and chemotherapeutic regimens. Cancer treatments for use with the methods of the invention include without limitation those listed in Table 8:",{"@attributes":{"id":"p-0986","num":"0988"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"336pt","align":"center"}},"thead":{"row":{"entry":"TABLE 8"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Cancer Treatments"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"273pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Treatment or Agent"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Cancer therapies","Radiation, Surgery, Chemotherapy, Biologic therapy, Neo-adjuvant therapy, Adjuvant"]},{"entry":[{},"therapy, Palliative therapy, Watchful waiting"]},{"entry":["Anti-cancer agents","13-cis-Retinoic Acid, 2-CdA, 2-Chlorodeoxyadenosine, 5-Azacitidine, 5-Fluorouracil,"]},{"entry":["(chemotherapies and","5-FU, 6-Mercaptopurine, 6-MP, 6-TG, 6-Thioguanine, Abraxane, Accutane\u2009\u00ae,"]},{"entry":["biologics)","Actinomycin-D, Adriamycin\u2009\u00ae, Adrucil\u2009\u00ae, Afinitor\u2009\u00ae, Agrylin\u2009\u00ae, Ala-Cort\u2009\u00ae,"]},{"entry":[{},"Aldesleukin, Alemtuzumab, ALIMTA, Alitretinoin, Alkaban-AQ\u2009\u00ae, Alkeran\u2009\u00ae, All-"]},{"entry":[{},"transretinoic Acid, Alpha Interferon, Altretamine, Amethopterin, Amifostine,"]},{"entry":[{},"Aminoglutethimide, Anagrelide, Anandron\u2009\u00ae, Anastrozole, Arabinosylcytosine, Ara-C,"]},{"entry":[{},"Aranesp\u2009\u00ae, Aredia\u2009\u00ae, Arimidex\u2009\u00ae, Aromasin\u2009\u00ae, Arranon\u2009\u00ae, Arsenic Trioxide,"]},{"entry":[{},"Asparaginase, ATRA, Avastin\u2009\u00ae, Azacitidine, BCG, BCNU, Bendamustine,"]},{"entry":[{},"Bevacizumab, Bexarotene, BEXXAR\u2009\u00ae, Bicalutamide, BiCNU, Blenoxane\u2009\u00ae,"]},{"entry":[{},"Bleomycin, Bortezomib, Busulfan, Busulfex\u2009\u00ae, C225, Calcium Leucovorin, Campath\u2009\u00ae,"]},{"entry":[{},"Camptosar\u2009\u00ae, Camptothecin-11, Capecitabine, Carac\u2009\u2122, Carboplatin, Carmustine,"]},{"entry":[{},"Carmustine Wafer, Casodex\u2009\u00ae, CC-5013, CCI-779, CCNU, CDDP, CeeNU,"]},{"entry":[{},"Cerubidine\u2009\u00ae, Cetuximab, Chlorambucil, Cisplatin, Citrovorum Factor, Cladribine,"]},{"entry":[{},"Cortisone, Cosmegen\u2009\u00ae, CPT-11, Cyclophosphamide, Cytadren\u2009\u00ae, Cytarabine,"]},{"entry":[{},"Cytarabine Liposomal, Cytosar-U\u2009\u00ae, Cytoxan\u2009\u00ae, Dacarbazine, Dacogen, Dactinomycin,"]},{"entry":[{},"Darbepoetin Alfa, Dasatinib, Daunomycin Daunorubicin, Daunorubicin Hydrochloride,"]},{"entry":[{},"Daunorubicin Liposomal, DaunoXome\u2009\u00ae, Decadron, Decitabine, Delta-Cortef\u2009\u00ae,"]},{"entry":[{},"Deltasone\u2009\u00ae, Denileukin, Diftitox, DepoCyt\u2009\u2122, Dexamethasone, Dexamethasone Acetate"]},{"entry":[{},"Dexamethasone Sodium Phosphate, Dexasone, Dexrazoxane, DHAD, DIC, Diodex"]},{"entry":[{},"Docetaxel, Doxil\u2009\u00ae, Doxorubicin, Doxorubicin Liposomal, Droxia\u2009\u2122, DTIC, DTIC-"]},{"entry":[{},"Dome\u2009\u00ae, Duralone\u2009\u00ae, Efudex\u2009\u00ae, Eligard\u2009\u2122, Ellence\u2009\u2122, Eloxatin\u2009\u2122, Elspar\u2009\u00ae, Emcyt\u2009\u00ae,"]},{"entry":[{},"Epirubicin, Epoetin Alfa, Erbitux, Erlotinib, Erwinia L-asparaginase, Estramustine,"]},{"entry":[{},"Ethyol Etopophos\u2009\u00ae, Etoposide, Etoposide Phosphate, Eulexin\u2009\u00ae, Everolimus, Evista\u2009\u00ae,"]},{"entry":[{},"Exemestane, Fareston\u2009\u00ae, Faslodex\u2009\u00ae, Femara\u2009\u00ae, Filgrastim, Floxuridine, Fludara\u2009\u00ae,"]},{"entry":[{},"Fludarabine, Fluoroplex\u2009\u00ae, Fluorouracil, Fluorouracil (cream), Fluoxymesterone,"]},{"entry":[{},"Flutamide, Folinic Acid, FUDR\u2009\u00ae, Fulvestrant, G-CSF, Gefitinib, Gemcitabine,"]},{"entry":[{},"Gemtuzumab ozogamicin, Gemzar, Gleevec\u2009\u2122, Gliadel\u2009\u00ae Wafer, GM-CSF, Goserelin,"]},{"entry":[{},"Granulocyte - Colony Stimulating Factor, Granulocyte Macrophage Colony Stimulating"]},{"entry":[{},"Factor, Halotestin\u2009\u00ae, Herceptin\u2009\u00ae, Hexadrol, Hexalen\u2009\u00ae, Hexamethylmelamine, HMM,"]},{"entry":[{},"Hycamtin\u2009\u00ae, Hydrea\u2009\u00ae, Hydrocort Acetate\u2009\u00ae, Hydrocortisone, Hydrocortisone Sodium"]},{"entry":[{},"Phosphate, Hydrocortisone Sodium Succinate, Hydrocortone Phosphate, Hydroxyurea,"]},{"entry":[{},"Ibritumomab, Ibritumomab, Tiuxetan, Idamycin\u2009\u00ae, Idarubicin, Ifex\u2009\u00ae, IFN-alpha,"]},{"entry":[{},"Ifosfamide, IL-11, IL-2, Imatinib mesylate, Imidazole Carboxamide, Interferon alfa,"]},{"entry":[{},"Interferon Alfa-2b (PEG Conjugate), Interleukin-2, Interleukin-11, Intron A\u2009\u00ae"]},{"entry":[{},"(interferon alfa-2b), Iressa\u2009\u00ae, Irinotecan, Isotretinoin, Ixabepilone, Ixempra\u2009\u2122, Kidrolase"]},{"entry":[{},"(t), Lanacort\u2009\u00ae, Lapatinib, L-asparaginase, LCR, Lenalidomide, Letrozole, Leucovorin,"]},{"entry":[{},"Leukeran, Leukine\u2009\u2122, Leuprolide, Leurocristine, Leustatin\u2009\u2122, Liposomal Ara-C Liquid"]},{"entry":[{},"Pred\u2009\u00ae, Lomustine, L-PAM, L-Sarcolysin, Lupron\u2009\u00ae, Lupron Depot\u2009\u00ae, Matulane\u2009\u00ae,"]},{"entry":[{},"Maxidex, Mechlorethamine, Mechlorethamine Hydrochloride, Medralone\u2009\u00ae, Medrol\u2009\u00ae,"]},{"entry":[{},"Megace\u2009\u00ae, Megestrol, Megestrol Acetate, Melphalan, Mercaptopurine, Mesna,"]},{"entry":[{},"Mesnex\u2009\u2122, Methotrexate, Methotrexate Sodium, Methylprednisolone, Meticorten\u2009\u00ae,"]},{"entry":[{},"Mitomycin, Mitomycin-C, Mitoxantrone, M-Prednisol\u2009\u00ae, MTC, MTX, Mustargen\u2009\u00ae,"]},{"entry":[{},"Mustine, Mutamycin\u2009\u00ae, Myleran\u2009\u00ae, Mylocel\u2009\u2122, Mylotarg\u2009\u00ae, Navelbine\u2009\u00ae, Nelarabine,"]},{"entry":[{},"Neosar\u2009\u00ae, Neulasta\u2009\u2122, Neumega\u2009\u00ae, Neupogen\u2009\u00ae, Nexavar\u2009\u00ae, Nilandron\u2009\u00ae, Nilutamide,"]},{"entry":[{},"Nipent\u2009\u00ae, Nitrogen Mustard, Novaldex\u2009\u00ae, Novantrone\u2009\u00ae, Octreotide, Octreotide acetate,"]},{"entry":[{},"Oncospar\u2009\u00ae, Oncovin\u2009\u00ae, Ontak\u2009\u00ae, Onxal\u2009\u2122, Oprevelkin, Orapred\u2009\u00ae, Orasone\u2009\u00ae,"]},{"entry":[{},"Oxaliplatin, Paclitaxel, Paclitaxel Protein-bound, Pamidronate, Panitumumab,"]},{"entry":[{},"Panretin\u2009\u00ae, Paraplatin\u2009\u00ae, Pediapred\u2009\u00ae, PEG Interferon, Pegaspargase, Pegfilgrastim,"]},{"entry":[{},"PEG-INTRON\u2009\u2122, PEG-L-asparaginase, PEMETREXED, Pentostatin, Phenylalanine"]},{"entry":[{},"Mustard, Platinol\u2009\u00ae, Platinol-AQ\u2009\u00ae, Prednisolone, Prednisone, Prelone\u2009\u00ae, Procarbazine,"]},{"entry":[{},"PROCRIT\u2009\u00ae, Proleukin\u2009\u00ae, Prolifeprospan 20 with Carmustine Implant, Purinethol\u2009\u00ae,"]},{"entry":[{},"Raloxifene, Revlimid\u2009\u00ae, Rheumatrex\u2009\u00ae, Rituxan\u2009\u00ae, Rituximab, Roferon-A\u2009\u00ae (Interferon"]},{"entry":[{},"Alfa-2a), Rubex\u2009\u00ae, Rubidomycin hydrochloride, Sandostatin\u2009\u00ae, Sandostatin LAR\u2009\u00ae,"]},{"entry":[{},"Sargramostim, Solu-Cortef\u2009\u00ae, Solu-Medrol\u2009\u00ae, Sorafenib, SPRYCEL\u2009\u2122, STI-571,"]},{"entry":[{},"Streptozocin, SU11248, Sunitinib, Sutent\u2009\u00ae, Tamoxifen, Tarceva\u2009\u00ae, Targretin\u2009\u00ae, Taxol\u2009\u00ae,"]},{"entry":[{},"Taxotere\u2009\u00ae, Temodar\u2009\u00ae, Temozolomide, Temsirolimus, Teniposide, TESPA,"]},{"entry":[{},"Thalidomide, Thalomid\u2009\u00ae, TheraCys\u2009\u00ae, Thioguanine, Thioguanine Tabloid\u2009\u00ae,"]},{"entry":[{},"Thiophosphoamide, Thioplex\u2009\u00ae, Thiotepa, TICE\u2009\u00ae, Toposar\u2009\u00ae, Topotecan, Toremifene,"]},{"entry":[{},"Torisel\u2009\u00ae, Tositumomab, Trastuzumab, Treanda\u2009\u00ae, Tretinoin, Trexall\u2009\u2122, Trisenox\u2009\u00ae,"]},{"entry":[{},"TSPA, TYKERB\u2009\u00ae, VCR, Vectibix\u2009\u2122, Velban\u2009\u00ae, Velcade\u2009\u00ae, VePesid\u2009\u00ae, Vesanoid\u2009\u00ae,"]},{"entry":[{},"Viadur\u2009\u2122, Vidaza\u2009\u00ae, Vinblastine, Vinblastine Sulfate, Vincasar Pfs\u2009\u00ae, Vincristine,"]},{"entry":[{},"Vinorelbine, Vinorelbine tartrate, VLB, VM-26, Vorinostat, VP-16, Vumon\u2009\u00ae, Xeloda\u2009\u00ae,"]},{"entry":[{},"Zanosar\u2009\u00ae, Zevalin\u2009\u2122, Zinecard\u2009\u00ae, Zoladex\u2009\u00ae, Zoledronic acid, Zolinza, Zometa\u2009\u00ae"]},{"entry":["Combination","CHOP (cyclophosphamide, doxorubicin, vincristine, and prednisone); CVP"]},{"entry":["Therapies","(cyclophosphamide, vincristine, and prednisone); RCVP (Rituximab + CVP); RCHOP"]},{"entry":[{},"(Rituximab + CHOP); RICE (Rituximab + ifosamide, carboplatin, etoposide); RDHAP,"]},{"entry":[{},"(Rituximab + dexamethasone, cytarabine, cisplatin); RESHAP (Rituximab + etoposide,"]},{"entry":[{},"methylprednisolone, cytarabine, cisplatin); combination treatment with vincristine,"]},{"entry":[{},"prednisone, and anthracycline, with or without asparaginase; combination treatment with"]},{"entry":[{},"daunorubicin, vincristine, prednisone, and asparaginase; combination treatment with"]},{"entry":[{},"teniposide and Ara-C (cytarabine); combination treatment with methotrexate and"]},{"entry":[{},"leucovorin; combination treatment with bleomycin, doxorubicin, etoposide,"]},{"entry":[{},"mechlorethamine, prednisone, vinblastine, and vincristine; FOLFOX4 regimen"]},{"entry":[{},"(oxaliplatin, leucovorin, and fluorouracil [5-FU]); FOLFIRI regimen (Irinotecan"]},{"entry":[{},"Hydrochloride, Fluorouracil, and Leucovorin Calcium); Levamisole regimen (5-FU and"]},{"entry":[{},"levamisole); NCCTG regimen (5-FU and low-dose leucovorin); NSABP regimen (5-FU"]},{"entry":[{},"and high-dose leucovorin); XAD (Xelox (Capecitabine + Oxaliplatin) + Bevacizumab +"]},{"entry":[{},"Dasatinib); FOLFOX\/Bevacizumab\/Hydroxychloroquine; German AIO regimen (folic"]},{"entry":[{},"acid, 5-FU, and irinotecan); Douillard regimen (folic acid, 5-FU, and irinotecan);"]},{"entry":[{},"CAPOX regimen (Capecitabine, oxaliplatin); FOLFOX6 regimen (oxaliplatin,"]},{"entry":[{},"leucovorin, and 5-FU); FOLFIRI regimen (folic acid, 5-FU, and irinotecan); FUFOX"]},{"entry":[{},"regimen (oxaliplatin, leucovorin, and 5-FU); FUOX regimen (oxaliplatin and 5-FU);"]},{"entry":[{},"IFL regimen (irinotecan, 5-FU, and leucovorin); XELOX regimen (capecitabine"]},{"entry":[{},"oxaliplatin); KHAD-L (ketoconazole, hydrocortisone, dutasteride and lapatinib);"]},{"entry":["Biologics","anti-CD52 antibodies (e.g., Alemtuzumab), anti-CD20 antibodies (e.g., Rituximab),"]},{"entry":[{},"anti-CD40 antibodies (e.g., SGN40)"]},{"entry":["Classes of","Anthracyclines and related substances, Anti-androgens, Anti-estrogens, Antigrowth"]},{"entry":["Treatments","hormones (e.g., Somatostatin analogs), Combination therapy (e.g., vincristine, bcnu,"]},{"entry":[{},"melphalan, cyclophosphamide, prednisone (VBMCP)), DNA methyltransferase"]},{"entry":[{},"inhibitors, Endocrine therapy - Enzyme inhibitor, Endocrine therapy - other hormone"]},{"entry":[{},"antagonists and related agents, Folic acid analogs (e.g., methotrexate), Folic acid"]},{"entry":[{},"analogs (e.g., pemetrexed), Gonadotropin releasing hormone analogs, Gonadotropin-"]},{"entry":[{},"releasing hormones, Monoclonal antibodies (EGFR-Targeted - e.g., panitumumab,"]},{"entry":[{},"cetuximab), Monoclonal antibodies (Her2-Targeted - e.g., trastuzumab), Monoclonal"]},{"entry":[{},"antibodies (Multi-Targeted - e.g., alemtuzumab), Other alkylating agents, Antineoplastic"]},{"entry":[{},"agents (e.g., asparaginase, ATRA, bexarotene, celecoxib, gemcitabine, hydroxyurea,"]},{"entry":[{},"irinotecan, topotecan, pentostatin), Cytotoxic antibiotics, Platinum compounds,"]},{"entry":[{},"Podophyllotoxin derivatives (e.g., etoposide), Progestogens, Protein kinase inhibitors"]},{"entry":[{},"(EGFR-Targeted), Protein kinase inhibitors (Her2 targeted therapy - e.g., lapatinib),"]},{"entry":[{},"Pyrimidine analogs (e.g., cytarabine), Pyrimidine analogs (e.g., fluoropyrimidines),"]},{"entry":[{},"Salicylic acid and derivatives (e.g., aspirin), Src-family protein tyrosine kinase inhibitors"]},{"entry":[{},"(e.g., dasatinib), Taxanes (e.g., nab-paclitaxel), Vinca Alkaloids and analogs, Vitamin D"]},{"entry":[{},"and analogs, Monoclonal antibodies (Multi-Targeted - e.g., bevacizumab), Protein"]},{"entry":[{},"kinase inhibitors (e.g., imatinib, sorafenib, sunitinib)"]},{"entry":["Prostate Cancer","Watchful waiting (i.e., monitor without treatment); Surgery (e.g., Pelvic"]},{"entry":["Treatments","lymphadenectomy, Radical prostatectomy, Transurethral resection of the prostate"]},{"entry":[{},"(TURP); Orchiectomy); Radiation therapy (e.g., external-beam radiation therapy"]},{"entry":[{},"(EBRT), Proton beam radiation; implantation of radioisotopes (i.e., iodine I 125,"]},{"entry":[{},"palladium, and iridium)); Hormone therapy (e.g., Luteinizing hormone-releasing"]},{"entry":[{},"hormone agonists such as leuprolide, goserelin, buserelin or ozarelix; Antiandrogens"]},{"entry":[{},"such as flutamide, 2-hydroxyflutamide, bicalutamide, megestrol acetate, nilutamide,"]},{"entry":[{},"ketoconazole, aminoglutethimide; calcitriol, gonadotropin-releasing hormone (GnRH),"]},{"entry":[{},"estrogens (DES, chlorotrianisene, ethinyl estradiol, conjugated estrogens USP, and DES-"]},{"entry":[{},"diphosphate), triptorelin, finasteride, cyproterone acetate, ASP3550);"]},{"entry":[{},"Cryosurgery\/cryotherapy; Chemotherapy and Biologic therapy (dutasteride, zoledronate,"]},{"entry":[{},"azacitidine, docetaxel, prednisolone, celecoxib, atorvastatin, AMT2003, soy protein,"]},{"entry":[{},"LHRH agonist, PD-103, pomegranate extract, soy extract, taxotere, I-125, zoledronic"]},{"entry":[{},"acid, dasatinib, vitamin C, vitamin D, vitamin D3, vitamin E, gemcitabine, cisplatin,"]},{"entry":[{},"lenalidomide, prednisone, degarelix, OGX-011, OGX-427, MDV3100, tasquinimod,"]},{"entry":[{},"cabazitaxel, TOOKAD\u2009\u00ae, lanreotide, PROSTVAC, GM-CSF, lenalidomide, samarium"]},{"entry":[{},"Sm-153 lexidronam, N-Methyl-D-Aspartate (NMDA)-Receptor Antagonist, sorafenib,"]},{"entry":[{},"sorafenib tosylate, mitoxantrone, ABI-008, hydrocortisone, panobinostat, soy-tomato"]},{"entry":[{},"extract, KHAD-L, TOK-001, cixutumumab, temsirolimus, ixabepilone, TAK-700,"]},{"entry":[{},"TAK-448, TRC105, cyclophosphamide, lenalidomide, MLN8237, GDC-0449,"]},{"entry":[{},"Alpharadin\u2009\u00ae, ARN-509, PX-866, ISIS EIF4E Rx, AEZS-108, 131I-F16SIP Monoclonal"]},{"entry":[{},"Antibody, anti-OX40 antibody, Muscadine Plus, ODM-201, BBI608, ZD4054, erlotinib,"]},{"entry":[{},"rIL-2, epirubicin, estramustine phosphate, HuJ591-GS monoclonal (177Lu-J591),"]},{"entry":[{},"abraxane, IVIG, fermented wheat germ nutriment (FWGE), 153Sm-EDTMP,"]},{"entry":[{},"estramustine, mitoxantrone, vinblastine, carboplatin, paclitaxel, pazopanib, cytarabine,"]},{"entry":[{},"testosterone replacement, Zoledronic Acid, Strontium Chloride Sr 89, paricalcitol,"]},{"entry":[{},"satraplatin, RAD001 (everolimus), valproic acid, tea extract, Hamsa-1,"]},{"entry":[{},"hydroxychloroquine, sipuleucel-T, selenomethionine, selenium, lycopene, sunitinib,"]},{"entry":[{},"vandetanib, IMC-A12 antibody, monoclonal antibody IMC-3G3, ixabepilone,"]},{"entry":[{},"diindolylmethane, metformin, efavirenz, dasatinib, nilutamide, abiraterone, cabozantinib"]},{"entry":[{},"(XL184), isoflavines, cinacalcet hydrochloride, SB939, LY2523355, KX2-391, olaparib,"]},{"entry":[{},"genestein, digoxin, RO4929097, ipilimumab, bafetinib, cediranib maleate, MK2206,"]},{"entry":[{},"phenelzine sulfate, triptorelin pamoate, saracatinib, STA-9090, tesetaxel, pasireotide,"]},{"entry":[{},"afatinib, GTx 758, lonafarnib, satraplatin, radiolabeled antibody 7E11,"]},{"entry":[{},"FP253\/fludarabine, Coxsackie A21 (CVA21) virus, ARRY-380, ARRY-382, anti-"]},{"entry":[{},"PSMA designer T cells, pemetrexed disodium, bortezomib, MDX-1106, white button"]},{"entry":[{},"mushroom extract, SU011248, MLN9708, BMTP-11, ABT-888, CX-4945, 4SC-205,"]},{"entry":[{},"temozolomide, MGAH22, vinorelbine ditartrate, Sodium Selenite, vorinostat, Ad-"]},{"entry":[{},"REIC\/Dkk-3, ASG-5ME, IMF-001, PROHIBITIN-TP01, DSTP3086S, ridaforolimus,"]},{"entry":[{},"MK-2206, MK-0752, polyunsaturated fatty acids, I-125, statins, cholecalciferol, omega-"]},{"entry":[{},"3 fatty acids, raloxifene, etoposide, POMELLA\u2009\u2122 extract, Lucrin depot); Cancer"]},{"entry":[{},"vaccines (e.g., DNA vaccines, peptide vaccines, dendritic cell vaccines, PEP223,"]},{"entry":[{},"PSA\/TRICOM, PROSTVAC-V\/TRICOM, PROSTVAC-F\/TRICOM, PSA vaccine,"]},{"entry":[{},"TroVax\u2009\u00ae, GI-6207, PSMA and TARP Peptide Vaccine); Ultrasound; Proton beam"]},{"entry":[{},"radiation"]},{"entry":["Colorectal Cancer","Primary Surgical Therapy (e.g., local excision; resection and anastomosis of primary"]},{"entry":["Treatments","lesion and removal of surrounding lymph nodes); Adjuvant Therapy (e.g., fluorouracil"]},{"entry":[{},"(5-FU), capecitabine, leucovorin, oxaliplatin, erlotinib, irinotecan, aspirin, mitomycin C,"]},{"entry":[{},"suntinib, cetuximab, bevacizumab, pegfilgrastim, panitumumab, ramucirumab,"]},{"entry":[{},"curcumin, celecoxib, FOLFOX4 regimen, FOLFOX6 regimen, FOLFIRI regimen,"]},{"entry":[{},"FUFOX regimen, FUOX regimen, IFL regimen, XELOX regimen, 5-FU and levamisole"]},{"entry":[{},"regimens, German AIO regimen, CAPOX regimen, Douillard regimen, XAD, RAD001"]},{"entry":[{},"(everolimus), ARQ 197, BMS-908662, JI-101, hydroxychloroquine (HCQ), Yttrium"]},{"entry":[{},"Microspheres, EZN-2208, CS-7017, IMC-1121B, IMC-18F1, docetaxel, lonafarnib,"]},{"entry":[{},"Maytansinoid DM4-Conjugated Humanized Monoclonal Antibody huC242, paclitaxel,"]},{"entry":[{},"ARRY-380, ARRY-382, IMO-2055, MDX1105-01, CX-4945, Pazopanib, Ixabepilone,"]},{"entry":[{},"OSI-906, NPC-1C Chimeric Monoclonal Antibody, brivanib, Poly-ADP Ribose (PARP)"]},{"entry":[{},"Inhibitor, RO4929097, Anti-cancer vaccine, CEA vaccine, cyclophosphamide, yttrium"]},{"entry":[{},"Y 90 DOTA anti-CEA monoclonal antibody M5A, MEHD7945A, ABT-806, ABT-888,"]},{"entry":[{},"MEDI-565, LY2801653, AZD6244, PRI-724, BKM120, tivozanib, floxuridine,"]},{"entry":[{},"dexamethosone, NKTR-102, perifosine, regorafenib, EP0906, Celebrex, PHY906,"]},{"entry":[{},"KRN330, imatinib mesylate, azacitidine, entinostat, PX-866, ABX-EGF, BAY 43-9006,"]},{"entry":[{},"ESO-1 Lymphocytes and Aldesleukin, LBH589, olaparib, fostamatinib, PD 0332991,"]},{"entry":[{},"STA-9090, cholecalciferol, GI-4000, IL-12, AMG 706, temsirolimus, dulanermin,"]},{"entry":[{},"bortezomib, ursodiol, ridaforolimus, veliparib, NK012, Dalotuzumab, MK-2206, MK-"]},{"entry":[{},"0752, lenalidomide, REOLYSIN\u2009\u00ae, AUY922, PRI-724, BKM120, avastin, dasatinib);"]},{"entry":[{},"Adjuvant Radiation Therapy (particularly for rectal cancer)"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"As shown in Table 8, cancer treatments include various surgical and therapeutic treatments. Anti-cancer agents include drugs such as small molecules and biologicals. The methods of the invention can be used to identify a biosignature comprising circulating biomarkers that can then be used for theranostic purposes such as monitoring a treatment efficacy, classifying a subject as a responder or non-responder to a treatment, or selecting a candidate therapeutic agent. The invention can be used to provide a theranosis for any cancer treatments, including without limitation thernosis involving the cancer treatments in Tables 8-10. Cancer therapies that can be identified as candidate treatments by the methods of the invention include without limitation the chemotherapeutic agents listed in Tables 8-10 and any appropriate combinations thereof. In one embodiment, the treatments are specific for a specific type of cancer, such as the treatments listed for prostate cancer, colorectal cancer, breast cancer and lung cancer in Table 8. In other embodiments, the treatments are specific for a tumor regardless of its origin but that displays a certain biosignature, such as a biosignature comprising a marker listed in Tables 9-10, or a drug associated marker listed in Table 11.","The invention provides methods of monitoring a cancer treatment comprising identifying a series of biosignatures in a subject over a time course, such as before and after a treatment, or over time after the treatment. The biosignatures are compared to a reference to determine the efficacy of the treatment. In an embodiment, the treatment is selected from Tables 8-10, such as radiation, surgery, chemotherapy, biologic therapy, neo-adjuvant therapy, adjuvant therapy, or watchful waiting. The reference can be from another individual or group of individuals or from the same subject. For example, a subject with a biosignature indicative of a cancer pre-treatment may have a biosignature indicative of a healthy state after a successful treatment. Conversely, the subject may have a biosignature indicative of cancer after an unsuccessful treatment. The biosignatures can be compared over time to determine whether the subject's biosignatures indicate an improvement, worsening of the condition, or no change. Additional treatments may be called for if the cancer is worsening or there is no change over time. For example, hormone therapy may be used in addition to surgery or radiation therapy to treat more aggressive prostate cancers. One or more of the following miRs can be used in a biosignature for monitoring an efficacy of prostate cancer treatment: hsa-miR-1974, hsa-miR-27b, hsa-miR-103, hsa-miR-146a, hsa-miR-22, hsa-miR-382, hsa-miR-23a, hsa-miR-376c, hsa-miR-335, hsa-miR-142-5p, hsa-miR-221, hsa-miR-142-3p, hsa-miR-151-3p, hsa-miR-21, hsa-miR-16. One or more miRs listed in the following publication can be used in a biosignature for monitoring treatment of a cancer of the GI tract: Albulescu et al., Tissular and soluble miRNAs for diagnostic and therapy improvement in digestive tract cancers, Exp Rev Mol Diag, 11:1, 101-120.","In some embodiments, the invention provides a method of identifying a biosignature in a sample from a subject in order to select a candidate therapeutic. For example, the biosignature may indicate that a drug-associated target is mutated or differentially expressed, thereby indicating that the subject is likely to respond or not respond to certain treatments. The candidate treatments can be chosen from the anti-cancer agents or classes of therapeutic agents identified in Tables 8-10. In some embodiments, the candidate treatments identified according to the subject methods are chosen from at least the groups of treatments consisting of 5-fluorouracil, abarelix, alemtuzumab, aminoglutethimide, anastrozole, asparaginase, aspirin, ATRA, azacitidine, bevacizumab, bexarotene, bicalutamide, calcitriol, capecitabine, carboplatin, celecoxib, cetuximab, chemotherapy, cholecalciferol, cisplatin, cytarabine, dasatinib, daunorubicin, decitabine, doxorubicin, epirubicin, erlotinib, etoposide, exemestane, flutamide, fulvestrant, gefitinib, gemcitabine, gonadorelin, goserelin, hydroxyurea, imatinib, irinotecan, lapatinib, letrozole, leuprolide, liposomal-doxorubicin, medroxyprogesterone, megestrol, megestrol acetate, methotrexate, mitomycin, nab-paclitaxel, octreotide, oxaliplatin, paclitaxel, panitumumab, pegaspargase, pemetrexed, pentostatin, sorafenib, sunitinib, tamoxifen, taxanes, temozolomide, toremifene, trastuzumab, VBMCP, and vincristine.","Similar to selecting a candidate treatment, the invention also provides a method of determining whether to treat a cancer at all. For example, prostate cancer can be a non-aggressive disease that is unlikely to substantially harm the subject. Radiation therapy with androgen ablation (hormone reduction) is the standard method of treating locally advanced prostate cancer. Morbidities of hormone therapy include impotence, hot flashes, and loss of libido. In addition, a treatment such as prostatectomy can have morbidities such as impotence or incontinence. Therefore, the invention provides biosignatures that indicate aggressiveness or a progression (e.g., stage or grade) of the cancer. A non-aggressive cancer or localized cancer might not require immediate treatment but rather be watched, e.g., \u201cwatchful waiting\u201d of a prostate cancer. Whereas an aggressive or advanced stage lesion would require a concomitantly more aggressive treatment regimen.","Examples of biomarkers that can be detected, and treatment agents that can be selected or possibly avoided are listed in Table 9. For example, a biosignature is identified for a subject with a prostate cancer, wherein the biosignature comprises levels of androgen receptor (AR). Overexpression or overproduction of AR, such as high levels of mRNA levels or protein levels in a vesicle, provides an identification of candidate treatments for the subject. Such treatments include agents for treating the subject such as Bicalutamide, Flutamide, Leuprolide, or Goserelin. The subject is accordingly identified as a responder to Bicalutamide, Flutamide, Leuprolide, or Goserelin. In another illustrative example, BCRP mRNA, protein, or both is detected at high levels in a vesicle from a subject suffering from NSCLC. The subject may then be classified as a non-responder to the agents Cisplatin and Carboplatin, or the agents are considered to be less effective than other agents for treating NSCLC in the subject and not selected for use in treating the subject. Any of the following biomarkers can be assessed in a vesicle obtained from a subject, and the biomarker can be in the form including but not limited to one or more of a nucleic acid, polypeptide, peptide or peptide mimetic. In yet another illustrative example, a mutation in one or more of KRAS, BRAF, PIK3CA, and\/or c-kit can be used to select a candidate treatment. For example, a mutation in KRAS or BRAF in a patient may indicate that cetuximab and\/or panitumumab are likely to be less effective in treating the patient.",{"@attributes":{"id":"p-0992","num":"0994"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"350pt","align":"center"}},"thead":{"row":{"entry":"TABLE 9"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Examples of Biomarkers, Lineage and Agents"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Possibly Less Effective","Possible Agents to"]},{"entry":["Biomarker","Lineage","Agents","Consider"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["AR (high expression)","Prostate",{},"Bicalutamide, Flutamide,"]},{"entry":[{},{},{},"Leuprolide, Goserelin"]},{"entry":["AR (high expression)","default",{},"Bicaluamide, Flutamide,"]},{"entry":[{},{},{},"Leuprolide, Goserelin"]},{"entry":["BCRP (high","Non-small cell lung cancer","Cisplatin, Carboplatin"]},{"entry":["expression)","(NSCLC)"]},{"entry":["BCRP (low","Non-small cell lung cancer",{},"Cisplatin, Carboplatin"]},{"entry":["expression)","(NSCLC)"]},{"entry":["BCRP (high","default","Cisplatin, Carboplatin"]},{"entry":"expression)"},{"entry":["BCRP (low","default",{},"Cisplatin, Carboplatin"]},{"entry":"expression)"},{"entry":["BRAF V600E","Colorectal","Cetuximab, Panitumumab"]},{"entry":"(mutation positive)"},{"entry":["BRAF V600E","Colorectal",{},"Cetuximab, Panitumumab"]},{"entry":"(mutation negative)"},{"entry":["BRAF V600E","All other","Cetuximab, Panitumumab"]},{"entry":"(mutation positive)"},{"entry":["BRAF V600E","All other",{},"Cetuximab, Panitumumab"]},{"entry":"(mutation negative)"},{"entry":["BRAF V600E","default","Cetuximab, Panitumumab"]},{"entry":"(mutation positive)"},{"entry":["BRAF V600E","default",{},"Cetuximab, Panitumumab"]},{"entry":"(mutation negative)"},{"entry":["CD52 (high","Leukemia",{},"Alemtuzumab"]},{"entry":"expression)"},{"entry":["CD52 (low","Leukemia","Alemtuzumab"]},{"entry":"expression)"},{"entry":["CD52 (high","default (Hematologic",{},"Alemtuzumab"]},{"entry":["expression)","malignancies only)"]},{"entry":["CD52 (low","default (Hematologic","Alemtuzumab"]},{"entry":["expression)","malignancies only)"]},{"entry":["c-kit","Uveal Melanoma"]},{"entry":["c-kit (high expression)","Gastrointestinal Stromal",{},"Imatinib"]},{"entry":[{},"Tumors [GIST]; cKIT will"]},{"entry":[{},"not be performed on Uveal"]},{"entry":[{},"Melanoma as imatinib is"]},{"entry":[{},"not useful in the setting of"]},{"entry":[{},"WT cKIT positive uveal"]},{"entry":[{},"melanoma (see Hofmann"]},{"entry":[{},"et al. 2009)"]},{"entry":["c-kit (high expression)","Extrahepatic Bile Duct",{},"Imatinib"]},{"entry":[{},"Tumors; cKIT will not be"]},{"entry":[{},"performed on Uveal"]},{"entry":[{},"Melanoma as imatinib is"]},{"entry":[{},"not useful in the setting of"]},{"entry":[{},"WT cKIT positive uveal"]},{"entry":[{},"melanoma (see Hofmann"]},{"entry":[{},"et al. 2009)"]},{"entry":["c-kit (high expression)","Acute myeloid leukemia",{},"Imatinib"]},{"entry":[{},"(AML)"]},{"entry":["c-kit (high expression)","default; cKIT will not be",{},"Imatinib"]},{"entry":[{},"performed on Uveal"]},{"entry":[{},"Melanoma as imatinib is"]},{"entry":[{},"not useful in the setting of"]},{"entry":[{},"WT cKIT positive uveal"]},{"entry":[{},"melanoma (see Hofmann"]},{"entry":[{},"et al. 2009)"]},{"entry":["EGFR (high copy","Head and neck squamous",{},"Erlotinib, Gefitinib"]},{"entry":["number)","cell carcinoma (HNSCC)"]},{"entry":["EGFR","Head and neck squamous","Erlotinib, Gefitinib"]},{"entry":[{},"cell carcinoma (HNSCC)"]},{"entry":["EGFR (high copy","Non-small cell lung cancer",{},"Erlotinib, Gefitinib"]},{"entry":["number)","(NSCLC)"]},{"entry":["EGFR (low copy","Non-small cell lung cancer","Erlotinib, Gefitinib"]},{"entry":["number)","(NSCLC)"]},{"entry":["EGFR (high copy","default",{},"Cetuxumab, Panitumumab,"]},{"entry":["number)",{},{},"Erlotinib, Gefitinib"]},{"entry":["EGFR (low copy","default","Cetuxumab, Panitumumab,"]},{"entry":["number)",{},"Erlotinib, Gefitinib"]},{"entry":["ER (high expression)","Breast","Ixabepilone","Tamoxifen-based treatment,"]},{"entry":[{},{},{},"aromatase inhibitors"]},{"entry":[{},{},{},"(anastrazole, letrozole)"]},{"entry":["ER (low expression)","Breast",{},"Ixabepilone"]},{"entry":["ER (high expression)","Ovarian",{},"Tamoxifen-based treatment,"]},{"entry":[{},{},{},"aromatase inhibitors"]},{"entry":[{},{},{},"(anastrazole, letrozole)"]},{"entry":["ER (high expression)","default",{},"Tamoxifen-based treatment,"]},{"entry":[{},{},{},"aromatase inhibitors"]},{"entry":[{},{},{},"(anastrazole, letrozole)"]},{"entry":["ERCC1 (high","Non-small cell lung cancer","Carboplatin, Cisplatin"]},{"entry":["expression)","(NSCLC)"]},{"entry":["ERCC1 (low","Non-small cell lung cancer",{},"Carboplatin, Cisplatin"]},{"entry":["expression)","(NSCLC)"]},{"entry":["ERCC1 (high","Small Cell Lung Cancer","Carboplatin, Cisplatin"]},{"entry":["expression)","(SCLC)"]},{"entry":["ERCC1 (low","Small Cell Lung Cancer",{},"Carboplatin, Cisplatin"]},{"entry":["expression)","(SCLC)"]},{"entry":["ERCC1 (high","Gastric","Oxaliplatin"]},{"entry":"expression)"},{"entry":["ERCC1 (low","Gastric",{},"Oxaliplatin"]},{"entry":"expression)"},{"entry":["ERCC1 (high","default","Carboplatin, Cisplatin,"]},{"entry":["expression)",{},"Oxaliplatin"]},{"entry":["ERCC1 (low","default",{},"Carboplatin, Cisplatin,"]},{"entry":["expression)",{},{},"Oxaliplatin"]},{"entry":["HER-2 (high","Breast",{},"Lapatinib, Trastuzumab"]},{"entry":"expression)"},{"entry":["HER-2 (high","default",{},"Lapatinib, Trastuzumab"]},{"entry":"expression)"},{"entry":["KRAS (mutation","Colorectal cancer","Cetuximab, Panitumumab"]},{"entry":"positive)"},{"entry":["KRAS (mutation","Colorectal cancer",{},"Cetuximab, Panitumumab"]},{"entry":"negative)"},{"entry":["KRAS (mutation","Non-small cell lung cancer","Erlotinib, Gefitinib"]},{"entry":["positive)","(NSCLC)"]},{"entry":["KRAS (mutation","Non-small cell lung cancer",{},"Erlotinib, Gefitinib"]},{"entry":["negative)","(NSCLC)"]},{"entry":["KRAS (mutation","Bronchioloalveolar","Erlotinib"]},{"entry":["positive)","carcinoma (BAC) or"]},{"entry":[{},"adenocarcinoma (BAC"]},{"entry":[{},"subtype)"]},{"entry":["KRAS (mutation","Bronchioloalveolar",{},"Erlotinib"]},{"entry":["negative)","carcinoma (BAC) or"]},{"entry":[{},"adenocarcinoma (BAC"]},{"entry":[{},"subtype)"]},{"entry":["KRAS (mutation","Multiple myeloma","VBMCP\/Cyclophosphamide"]},{"entry":"positive)"},{"entry":["KRAS (mutation","Multiple myeloma",{},"VBMCP\/Cyclophosphamide"]},{"entry":"negative)"},{"entry":["KRAS (mutation","default","Cetuximab, Panitumumab"]},{"entry":"positive)"},{"entry":["KRAS (mutation","default",{},"Cetuximab, panitumumab"]},{"entry":"negative)"},{"entry":["KRAS (mutation","default","Cetuximab, Erlotinib,"]},{"entry":["positive)",{},"Panitumumab, Gefitinib"]},{"entry":["KRAS (mutation","default",{},"Cetuximab, Erlotinib,"]},{"entry":["negative)",{},{},"Panitumumab, Gefitinib"]},{"entry":["MGMT (high","Pituitary tumors,","Temozolomide"]},{"entry":["expression)","oligodendroglioma"]},{"entry":["MGMT (low","Pituitary tumors,",{},"Temozolomide"]},{"entry":["expression)","oligodendroglioma"]},{"entry":["MGMT (high","Neuroendocrine tumors","Temozolomide"]},{"entry":"expression)"},{"entry":["MGMT (low","Neuroendocrine tumors",{},"Temozolomide"]},{"entry":"expression)"},{"entry":["MGMT (high","default","Temozolomide"]},{"entry":"expression)"},{"entry":["MGMT (low","default",{},"Temozolomide"]},{"entry":"expression)"},{"entry":["MRP1 (high","Breast","Cyclophosphamide"]},{"entry":"expression)"},{"entry":["MRP1 (low","Breast",{},"Cyclophosphamide"]},{"entry":"expression)"},{"entry":["MRP1 (high","Small Cell Lung Cancer","Etoposide"]},{"entry":["expression)","(SCLC)"]},{"entry":["MRP1 (low","Small Cell Lung Cancer",{},"Etoposide"]},{"entry":["expression)","(SCLC)"]},{"entry":["MRP1 (high","Nodal Diffuse Large B-","Cyclophosphamide\/Vincristine"]},{"entry":["expression)","Cell Lymphoma"]},{"entry":["MRP1 (low","Nodal Diffuse Large B-",{},"Cyclophosphamide\/Vincristine"]},{"entry":["expression)","Cell Lymphoma"]},{"entry":["MRP1 (high","default","Cyclophosphamide,"]},{"entry":["expression)",{},"Etoposide, Vincristine"]},{"entry":["MRP1 (low","default",{},"Cyclophosphamide,"]},{"entry":["expression)",{},{},"Etoposide, Vincristine"]},{"entry":["PDGFRA (high","Malignant Solitary Fibrous",{},"Imatinib"]},{"entry":["expression)","Tumor of the Pleura"]},{"entry":[{},"(MSFT)"]},{"entry":["PDGFRA (high","Gastrointestinal stromal",{},"Imatinib"]},{"entry":["expression)","tumor (GIST)"]},{"entry":["PDGFRA (high","Default",{},"Imatinib"]},{"entry":"expression)"},{"entry":["p-glycoprotein (high","Acute myeloid leukemia","Etoposide"]},{"entry":["expression)","(AML)"]},{"entry":["p-glycoprotein (low","Acute myeloid leukemia",{},"Etoposide"]},{"entry":["expression)","(AML)"]},{"entry":["p-glycoprotein (high","Diffuse Large B-cell","Doxorubicin"]},{"entry":["expression)","Lymphoma (DLBCL)"]},{"entry":["p-glycoprotein (low","Diffuse Large B-cell",{},"Doxorubicin"]},{"entry":["expression)","Lymphoma (DLBCL)"]},{"entry":["p-glycoprotein (high","Lung","Etoposide"]},{"entry":"expression)"},{"entry":["p-glycoprotein (low","Lung",{},"Etoposide"]},{"entry":"expression)"},{"entry":["p-glycoprotein (high","Breast","Doxorubicin"]},{"entry":"expression)"},{"entry":["p-glycoprotein (low","Breast",{},"Doxorubicin"]},{"entry":"expression)"},{"entry":["p-glycoprotein (high","Ovarian","Paclitaxel"]},{"entry":"expression)"},{"entry":["p-glycoprotein (low","Ovarian",{},"Paclitaxel"]},{"entry":"expression)"},{"entry":["p-glycoprotein (high","Head and neck squamous","Vincristine"]},{"entry":["expression)","cell carcinoma (HNSCC)"]},{"entry":["p-glycoprotein (low","Head and neck squamous",{},"Vincristine"]},{"entry":["expression)","cell carcinoma (HNSCC)"]},{"entry":["p-glycoprotein (high","default","Vincristine, Etoposide,"]},{"entry":["expression)",{},"Doxorubicin, Paclitaxel"]},{"entry":["p-glycoprotein (low","default",{},"Vincristine, Etoposide,"]},{"entry":["expression)",{},{},"Doxorubicin, Paclitaxel"]},{"entry":["PR (high expression)","Breast","Chemoendocrine therapy","Tamoxifen, Anastrazole,"]},{"entry":[{},{},{},"Letrozole"]},{"entry":["PR (low expression)","default","Chemoendocrine therapy","Tamoxifen, Anastrazole,"]},{"entry":[{},{},{},"Letrozole"]},{"entry":["PTEN (high","Breast",{},"Trastuzumab"]},{"entry":"expression)"},{"entry":["PTEN (low","Breast","Trastuzumab"]},{"entry":"expression)"},{"entry":["PTEN (high","Non-small cell Lung",{},"Gefitinib"]},{"entry":["expression)","Cancer (NSCLC)"]},{"entry":["PTEN (low","Non-small cell Lung","Gefitinib"]},{"entry":["expression)","Cancer (NSCLC)"]},{"entry":["PTEN (high","Colorectal",{},"Cetuximab, Panitumumab"]},{"entry":"expression)"},{"entry":["PTEN (low","Colorectal","Cetuximab, Panitumumab"]},{"entry":"expression)"},{"entry":["PTEN (high","Glioblastoma",{},"Erlotinib, Gefitinib"]},{"entry":"expression)"},{"entry":["PTEN (low","Glioblastoma","Erlotinib, Gefitinib"]},{"entry":"expression)"},{"entry":["PTEN (high","default",{},"Cetuximab, Panitumumab,"]},{"entry":["expression)",{},{},"Erlotinib, Gefitinib and"]},{"entry":[{},{},{},"Trastuzumab"]},{"entry":["PTEN (low","default","Cetuximab, Panitumumab,"]},{"entry":["expression)",{},"Erlotinib, Gefitinib and"]},{"entry":[{},{},"Trastuzumab"]},{"entry":["RRM1 (high","Non-small cell lung cancer","Gemcitabine"]},{"entry":["experssion)","(NSCLC)"]},{"entry":["RRM1 (low","Non-small cell lung cancer",{},"Gemcitabine"]},{"entry":["expression)","(NSCLC)"]},{"entry":["RRM1 (high","Pancreas","Gemcitabine"]},{"entry":"experssion)"},{"entry":["RRM1 (low","Pancreas",{},"Gemcitabine"]},{"entry":"expression)"},{"entry":["RRM1 (high","default","Gemcitabine"]},{"entry":"experssion)"},{"entry":["RRM1 (low","default",{},"Gemcitabine"]},{"entry":"expression)"},{"entry":["SPARC (high","Breast",{},"nab-paclitaxel"]},{"entry":"expression)"},{"entry":["SPARC (high","default",{},"nab-paclitaxel"]},{"entry":"expression)"},{"entry":["TS (high expression)","Colorectal","fluoropyrimidines"]},{"entry":["TS (low expression)","Colorectal",{},"fluoropyrimidines"]},{"entry":["TS (high expression)","Pancreas","fluoropyrimidines"]},{"entry":["TS (low expression)","Pancreas",{},"fluoropyrimidines"]},{"entry":["TS (high expression)","Head and Neck Cancer","fluoropyrimidines"]},{"entry":["TS (low expression)","Head and Neck Cancer",{},"fluoropyrimidines"]},{"entry":["TS (high expression)","Gastric","fluoropyrimidines"]},{"entry":["TS (low expression)","Gastric",{},"fluoropyrimidines"]},{"entry":["TS (high expression)","Non-small cell lung cancer","fluoropyrimidines"]},{"entry":[{},"(NSCLC)"]},{"entry":["TS (low expression)","Non-small cell lung cancer",{},"fluoropyrimidines"]},{"entry":[{},"(NSCLC)"]},{"entry":["TS (high expression)","Liver","fluoropyrimidines"]},{"entry":["TS (low expression)","Liver",{},"fluoropyrimidines"]},{"entry":["TS (high expression)","default","fluoropyrimidines"]},{"entry":["TS (low expression)","default",{},"fluoropyrimidines"]},{"entry":["TOPO1 (high","Colorectal",{},"Irinotecan"]},{"entry":"expression)"},{"entry":["TOPO1 (low","Colorectal","Irinotecan"]},{"entry":"expression)"},{"entry":["TOPO1 (high","Ovarian",{},"Irinotecan"]},{"entry":"expression)"},{"entry":["TOPO1 (low","Ovarian","Irinotecan"]},{"entry":"expression)"},{"entry":["TOPO1 (high","default",{},"Irinotecan"]},{"entry":"expression)"},{"entry":["TOPO1 (low","default","Irinotecan"]},{"entry":"expression)"},{"entry":["TopoIIa (high","Breast",{},"Doxorubicin, liposomal-"]},{"entry":["epxression)",{},{},"Doxorubicin, Epirubicin"]},{"entry":["TopoIIa (low","Breast","Doxorubicin, liposomal-"]},{"entry":["expression)",{},"Doxorubicin, Epirubicin"]},{"entry":["TopoIIa (high","default",{},"Doxorubicin, liposomal-"]},{"entry":["epxression)",{},{},"Doxorubicin, Epirubicin"]},{"entry":["TopoIIa (low","default","Doxorubicin, liposomal-"]},{"entry":["expression)",{},"Doxorubicin, Epirubicin"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Other examples of biomarkers that can be detected and the treatment agents that can be selected or possibly avoided based on the biomarker signatures are listed in Table 10. For example, for a subject suffering from cancer, detecting overexpression of ADA in vesicles from a subject is used to classify the subject as a responder to pentostatin, or pentostatin identified as an agent to use for treating the subject. In another example, for a subject suffering from cancer, detecting overexpression of BCRP in vesicles from the subject is used to classify the subject as a non-responder to cisplatin, carboplatin, irinotecan, and topotecan, meaning that cisplatin, carboplatin, irinotecan, and topotecan are identified as agents that are suboptimal for treating the subject.",{"@attributes":{"id":"p-0994","num":"0996"},"tables":{"@attributes":{"id":"TABLE-US-00010","num":"00010"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"315pt","align":"center"}},"thead":{"row":{"entry":"TABLE 10"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Examples of Biomarkers, Agents and Resistance"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Gene Name","Expression Status","Candidate Agent(s)","Possible Resistance"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["ADA","Overexpressed","pentostatin",{}]},{"entry":["ADA","Underexpressed",{},"cytarabine"]},{"entry":["AR","Overexpressed","abarelix, bicalutamide,"]},{"entry":[{},{},"flutamide, gonadorelin,"]},{"entry":[{},{},"goserelin, leuprolide"]},{"entry":["ASNS","Underexpressed","asparaginase, pegaspargase"]},{"entry":["BCRP (ABCG2)","Overexpressed",{},"cisplatin, carboplatin,"]},{"entry":[{},{},{},"irinotecan, topotecan"]},{"entry":["BRCA1","Underexpressed","mitomycin"]},{"entry":["BRCA2","Underexpressed","mitomycin"]},{"entry":["CD52","Overexpressed","alemtuzumab"]},{"entry":["CDA","Overexpressed",{},"cytarabine"]},{"entry":["c-erbB2","High levels of","Trastuzumab, c-erbB2"]},{"entry":[{},"phosphorylation in","kinase inhibitor, lapatinib"]},{"entry":[{},"epithelial cells"]},{"entry":["CES2","Overexpressed","irinotecan"]},{"entry":["c-kit","Overexpressed","sorafenib, sunitinib,"]},{"entry":[{},{},"imatinib"]},{"entry":["COX-2","Overexpressed","celecoxib"]},{"entry":["DCK","Overexpressed","gemcitabine","cytarabine"]},{"entry":["DHFR","Underexpressed","methotrexate, pemetrexed"]},{"entry":["DHFR","Overexpressed",{},"methotrexate"]},{"entry":["DNMT1","Overexpressed","azacitidine, decitabine"]},{"entry":["DNMT3A","Overexpressed","azacitidine, decitabine"]},{"entry":["DNMT3B","Overexpressed","azacitidine, decitabine"]},{"entry":["EGFR","Overexpressed","erlotinib, gefitinib,"]},{"entry":[{},{},"cetuximab, panitumumab"]},{"entry":["EML4-ALK","Overexpressed (present)","crizotinib"]},{"entry":["EPHA2","Overexpressed","dasatinib"]},{"entry":["ER","Overexpressed","anastrazole, exemestane,"]},{"entry":[{},{},"fulvestrant, letrozole,"]},{"entry":[{},{},"megestrol, tamoxifen,"]},{"entry":[{},{},"medroxyprogesterone,"]},{"entry":[{},{},"toremifene,"]},{"entry":[{},{},"aminoglutethimide"]},{"entry":["ERCC1","Overexpressed",{},"carboplatin, cisplatin"]},{"entry":["GART","Underexpressed","pemetrexed"]},{"entry":["GRN (PCDGF, PGRN)","Overexpressed",{},"anti-oestrogen therapy,"]},{"entry":[{},{},{},"tamoxifen, faslodex,"]},{"entry":[{},{},{},"letrozole, herceptin in"]},{"entry":[{},{},{},"Her-2 overexpressing"]},{"entry":[{},{},{},"cells, doxorubicin"]},{"entry":["HER-2 (ERBB2)","Overexpressed","trastuzumab, lapatinib"]},{"entry":["HIF-1\u03b1","Overexpressed","sorafenib, sunitinib,"]},{"entry":[{},{},"bevacizumab"]},{"entry":["I\u03baB-\u03b1","Overexpressed","bortezomib"]},{"entry":["MGMT","Underexpressed","temozolomide"]},{"entry":["MGMT","Overexpressed",{},"temozolomide"]},{"entry":["MRP1 (ABCC1)","Overexpressed",{},"etoposide, paclitaxel,"]},{"entry":[{},{},{},"docetaxel, vinblastine,"]},{"entry":[{},{},{},"vinorelbine, topotecan,"]},{"entry":[{},{},{},"teniposide"]},{"entry":["P-gp (ABCB1)","Overexpressed",{},"doxorubicin, etoposide,"]},{"entry":[{},{},{},"epirubicin, paclitaxel,"]},{"entry":[{},{},{},"docetaxel, vinblastine,"]},{"entry":[{},{},{},"vinorelbine, topotecan,"]},{"entry":[{},{},{},"teniposide, liposomal"]},{"entry":[{},{},{},"doxorubicin"]},{"entry":["PDGFR-\u03b1","Overexpressed","sorafenib, sunitinib,"]},{"entry":[{},{},"imatinib"]},{"entry":["PDGFR-\u03b2","Overexpressed","sorafenib, sunitinib,"]},{"entry":[{},{},"imatinib"]},{"entry":["PR","Overexpressed","exemestane, fulvestrant,"]},{"entry":[{},{},"gonadorelin, goserelin,"]},{"entry":[{},{},"medroxyprogesterone,"]},{"entry":[{},{},"megestrol, tamoxifen,"]},{"entry":[{},{},"toremifene"]},{"entry":["RARA","Overexpressed","ATRA"]},{"entry":["RRM1","Underexpressed","gemcitabine, hydroxyurea"]},{"entry":["RRM2","Underexpressed","gemcitabine, hydroxyurea"]},{"entry":["RRM2B","Underexpressed","gemcitabine, hydroxyurea"]},{"entry":["RXR-\u03b1","Overexpressed","bexarotene"]},{"entry":["RXR-\u03b2","Overexpressed","bexarotene"]},{"entry":["SPARC","Overexpressed","nab-paclitaxel"]},{"entry":["SRC","Overexpressed","dasatinib"]},{"entry":["SSTR2","Overexpressed","octreotide"]},{"entry":["SSTR5","Overexpressed","octreotide"]},{"entry":["TOPO I","Overexpressed","irinotecan, topotecan"]},{"entry":["TOPO II\u03b1","Overexpressed","doxorubicin, epirubicin,"]},{"entry":[{},{},"liposomal-doxorubicin"]},{"entry":["TOPO II\u03b2","Overexpressed","doxorubicin, epirubicin,"]},{"entry":[{},{},"liposomal-doxorubicin"]},{"entry":["TS","Underexpressed","capecitabine, 5-"]},{"entry":[{},{},"fluorouracil, pemetrexed"]},{"entry":["TS","Overexpressed",{},"capecitabine, 5-"]},{"entry":[{},{},{},"fluorouracil"]},{"entry":["VDR","Overexpressed","calcitriol, cholecalciferol"]},{"entry":["VEGFR1 (Flt1)","Overexpressed","sorafenib, sunitinib,"]},{"entry":[{},{},"bevacizumab"]},{"entry":["VEGFR2","Overexpressed","sorafenib, sunitinib,"]},{"entry":[{},{},"bevacizumab"]},{"entry":["VHL","Underexpressed","sorafenib, sunitinib"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Further drug associations and rules that are used in embodiments of the invention are found in U.S. patent application Ser. No. 12\/658,770, filed Feb. 12, 2010; International PCT Patent Application PCT\/US2010\/000407, filed Feb. 11, 2010; International PCT Patent Application PCT\/US2010\/54366, filed Oct. 27, 2010; and U.S. Provisional Patent Application 61\/427,788, filed Dec. 28, 2010; all of which applications are incorporated by reference herein in their entirety. See, e.g., \u201cTable 4: Rules Summary for Treatment Selection\u201d of PCT\/US2010\/54366.","Any drug-associated target can be part of a biosignature for providing a theranosis. A \u201cdruggable target\u201d comprising a target that can be modulated with a therapeutic agent such as a small molecule or biologic, is a candidate for inclusion in the biosignature of the invention. Drug-associated targets also include biomarkers that can confer resistance to a treatment, such as shown in Tables 9 and 10. The biosignature can be based on either the gene, e.g., DNA sequence, and\/or gene product, e.g., mRNA or protein, or the drug-associated target. Such nucleic acid and\/or polypeptide can be profiled as applicable as to presence or absence, level or amount, activity, mutation, sequence, haplotype, rearrangement, copy number, or other measurable characteristic. The gene or gene product can be associated with a vesicle population, e.g., as a vesicle surface marker or as vesicle payload. In an embodiment, the invention provides a method of theranosing a cancer, comprising identifying a biosignature that comprises a presence or level of one or more drug-associated target, and selecting a candidate therapeutic based on the biosignature. The drug-associated target can be a circulating biomarker, a vesicle, or a vesicle associated biomarker. Because drug-associated targets can be independent of the tissue or cell-of-origin, biosignatures comprising drug-associated targets can be used to provide a theranosis for any proliferative disease, such as cancers from various anatomical origins, including cancers of unknown origin such as CUPS.","The drug-associated targets assessed using the methods of the invention comprise without limitation ABCC1, ABCG2, ACE2, ADA, ADH1C, ADH4, AGT, AR, AREG, ASNS, BCL2, BCRP, BDCA1, beta III tubulin, BIRC5, B-RAF, BRCA1, BRCA2, CA2, caveolin, CD20, CD25, CD33, CD52, CDA, CDKN2A, CDKN1A, CDKN1B, CDK2, CDW52, CES2, CK 14, CK 17, CK 5\/6, c-KIT, c-Met, c-Myc, COX-2, Cyclin D1, DCK, DHFR, DNMT1, DNMT3A, DNMT3B, E-Cadherin, ECGF1, EGFR, EML4-ALK fusion, EPHA2, Epiregulin, ER, ERBR2, ERCC1, ERCC3, EREG, ESR1, FLT1, folate receptor, FOLR1, FOLR2, FSHB, FSHPRH1, FSHR, FYN, GART, GNRH1, GNRHR1, GSTP1, HCK, HDAC1, hENT-1, Her2\/Neu, HGF, HIF1A, HIG1, HSP90, HSP90AA1, HSPCA, IGF-1R, IGFRBP, IGFRBP3, IGFRBP4, IGFRBP5, IL13RA1, IL2RA, KDR, Ki67, KIT, K-RAS, LCK, LTB, Lymphotoxin Beta Receptor, LYN, MET, MGMT, MLH1, MMR, MRP1, MS4A1, MSH2, MSH5, Myc, NFKB1, NFKB2, NFKBIA, ODC1, OGFR, p16, p21, p27, p53, p95, PARP-1, PDGFC, PDGFR, PDGFRA, PDGFRB, PGP, PGR, PI3K, POLA, POLA1, PPARG, PPARGC1, PR, PTEN, PTGS2, RAF1, RARA, RRM1, RRM2, RRM2B, RXRB, RXRG, SPARC, SRC, SSTR1, SSTR2, SSTR3, SSTR4, SSTR5, Survivin, TK1, TLE3, TNF, TOP1, TOP2A, TOP2B, TS, TXN, TXNRD1, TYMS, VDR, VEGF, VEGFA, VEGFC, VHL, YES 1, ZAP70, or any combination thereof. A biosignature including one or combination of these markers can be used to characterize a phenotype according to the invention, such as providing a theranosis. These markers are known to play a role in the efficacy of various chemotherapeutic agents against proliferative diseases. Accordingly, the markers can be assessed to select a candidate treatment for the cancer independent of the origin or type of cancer. In an embodiment, the invention provides a method of selecting a candidate therapeutic for a cancer, comprising identifying a biosignature comprising a level or presence of one or more drug associated target, and selecting the candidate therapeutic based on its predicted efficacy for a patient with the biosignature. The one or more drug-associated target can be one of the targets listed above, or in Tables 9-11. In some embodiments, at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 15, 20, 25, 30, 35, 40, 45, or at least 50 of the one or more drug-associated targets are assessed. The one or more drug-associated target can be associated with a vesicle, e.g., as a vesicle surface marker or as vesicle payload as either nucleic acid (e.g., DNA, mRNA) or protein. In some embodiments, the presence or level of a microRNA known to interact with the one or more drug-associated target is assessed, wherein a high level of microRNA known to suppress the one or more drug-associated target can indicate a lower expression of the one or more drug-associated target and thus a lower likelihood of response to a treatment against the drug-associated target. The one or more drug-associated target can be circulating biomarkers. The one or more drug-associated target can be assessed in a tissue sample. The predicted efficacy can be determined by comparing the presence or level of the one or more drug-associated target to a reference value, wherein a higher level that the reference indicates that the subject is a likely responder. The predicted efficacy can be determined using a classifier algorithm, wherein the classifier was trained by comparing the biosignature of the one or more drug-associated target in subjects that are known to be responders or non-responders to the candidate treatment. Molecular associations of the one or more drug-associated target with appropriate candidate targets are displayed in Tables 9-10 herein and U.S. patent application Ser. No. 12\/658,770, filed Feb. 12, 2010; International PCT Patent Application PCT\/US2010\/000407, filed Feb. 11, 2010; International PCT Patent Application PCT\/US2010\/54366, filed Oct. 27, 2010; and U.S. Provisional Patent Application 61\/427,788, filed Dec. 28, 2010; all of which applications are incorporated by reference herein in their entirety.","Table 11 provides a listing of gene and corresponding protein symbols and names of many of the theranostic targets that are analyzed according to the methods of the invention. As understood by those of skill in the art, genes and proteins have developed a number of alternative names in the scientific literature. Thus, the listing in Table 11 comprises an illustrative but not exhaustive compilation. A further listing of gene aliases and descriptions can be found using a variety of online databases, including GeneCards\u00ae (www.genecards.org), HUGO Gene Nomenclature (www.genenames.org), Entrez Gene (www.ncbi.nlm.nih.gov\/entrez\/query.fcgi?db=gene), UniProtKB\/Swiss-Prot (www.uniprot.org), UniProtKB\/TrEMBL (www.uniprot.org), OMIM (www.ncbi.nlm.nih.gov\/entrez\/query.fcgi?db=OMIM), GeneLoc (genecards.weizmann.ac.il\/geneloc\/), and Ensembl (www.ensembl.org). Generally, gene symbols and names below correspond to those approved by HUGO, and protein names are those recommended by UniProtKB\/Swiss-Prot. Common alternatives are provided as well. Where a protein name indicates a precursor, the mature protein is also implied. Throughout the application, gene and protein symbols may be used interchangeably and the meaning can be derived from context as necessary.",{"@attributes":{"id":"p-0999","num":"1001"},"tables":{"@attributes":{"id":"TABLE-US-00011","num":"00011"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"336pt","align":"center"}},"thead":{"row":{"entry":"TABLE 11"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Genes and Related Proteins for Cancer Theranostics"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"133pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Gene",{},"Protein",{}]},{"entry":["Symbol","Gene Name","Symbol","Protein Name"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["ABCB1,","ATP-binding cassette, sub-family B","ABCB1,","Multidrug resistance protein 1; P-"]},{"entry":["PGP","(MDR\/TAP), member 1","MDR1, PGP","glycoprotein"]},{"entry":["ABCC1,","ATP-binding cassette, sub-family C","MRP1,","Multidrug resistance-associated protein 1"]},{"entry":["MRP1","(CFTR\/MRP), member 1","ABCC1"]},{"entry":["ABCG2,","ATP-binding cassette, sub-family G","ABCG2","ATP-binding cassette sub-family G"]},{"entry":["BCRP","(WHITE), member 2",{},"member 2"]},{"entry":["ACE2","angiotensin I converting enzyme","ACE2","Angiotensin-converting enzyme 2"]},{"entry":[{},"(peptidyl-dipeptidase A) 2",{},"precursor"]},{"entry":["ADA","adenosine deaminase","ADA","Adenosine deaminase"]},{"entry":["ADH1C","alcohol dehydrogenase 1C (class I),","ADH1G","Alcohol dehydrogenase 1C"]},{"entry":[{},"gamma polypeptide"]},{"entry":["ADH4","alcohol dehydrogenase 4 (class II), pi","ADH4","Alcohol dehydrogenase 4"]},{"entry":[{},"polypeptide"]},{"entry":["AGT","angiotensinogen (serpin peptidase","ANGT,","Angiotensinogen precursor"]},{"entry":[{},"inhibitor, clade A, member 8)","AGT"]},{"entry":["ALK","anaplastic lymphoma receptor tyrosine","ALK","ALK tyrosine kinase receptor precursor"]},{"entry":[{},"kinase"]},{"entry":["AR","androgen receptor","AR","Androgen receptor"]},{"entry":["AREG","amphiregulin","AREG","Amphiregulin precursor"]},{"entry":["ASNS","asparagine synthetase","ASNS","Asparagine synthetase [glutamine-"]},{"entry":[{},{},{},"hydrolyzing]"]},{"entry":["BCL2","B-cell CLL\/lymphoma 2","BCL2","Apoptosis regulator Bcl-2"]},{"entry":["BDCA1,","CD1c molecule","CD1C","T-cell surface glycoprotein CD1c precursor"]},{"entry":"CD1C"},{"entry":["BIRC5","baculoviral IAP repeat-containing 5","BIRC5,","Baculoviral IAP repeat-containing protein"]},{"entry":[{},{},"Survivin","5; Survivin"]},{"entry":["BRAF","v-raf murine sarcoma viral oncogene","B-RAF,","Serine\/threonine-protein kinase B-raf"]},{"entry":[{},"homolog B1","BRAF"]},{"entry":["BRCA1","breast cancer 1, early onset","BRCA1","Breast cancer type 1 susceptibility protein"]},{"entry":["BRCA2","breast cancer 2, early onset","BRCA2","Breast cancer type 2 susceptibility protein"]},{"entry":["CA2","carbonic anhydrase II","CA2","Carbonic anhydrase 2"]},{"entry":["CAV1","caveolin 1, caveolae protein, 22 kDa","CAV1","Caveolin-1"]},{"entry":["CCND1","cyclin D1","CCND1,","G1\/S-specific cyclin-D1"]},{"entry":[{},{},"Cyclin D1,"]},{"entry":[{},{},"BCL-1"]},{"entry":["CD20,","membrane-spanning 4-domains,","CD20","B-lymphocyte antigen CD20"]},{"entry":["MS4A1","subfamily A, member 1"]},{"entry":["CD25,","interleukin 2 receptor, alpha","CD25","Interleukin-2 receptor subunit alpha"]},{"entry":["IL2RA",{},{},"precursor"]},{"entry":["CD33","CD33 molecule","CD33","Myeloid cell surface antigen CD33"]},{"entry":[{},{},{},"precursor"]},{"entry":["CD52,","CD52 molecule","CD52","CAMPATH-1 antigen precursor"]},{"entry":"CDW52"},{"entry":["CDA","cytidine deaminase","CDA","Cytidine deaminase"]},{"entry":["CDH1,","cadherin 1, type 1, E-cadherin","E-Cad","Cadherin-1 precursor (E-cadherin)"]},{"entry":["ECAD","(epithelial)"]},{"entry":["CDK2","cyclin-dependent kinase 2","CDK2","Cell division protein kinase 2"]},{"entry":["CDKN1A,","cyclin-dependent kinase inhibitor 1A","CDKN1A,","Cyclin-dependent kinase inhibitor 1"]},{"entry":["P21","(p21, Cip1)","p21"]},{"entry":["CDKN1B","cyclin-dependent kinase inhibitor 1B","CDKN1B,","Cyclin-dependent kinase inhibitor 1B"]},{"entry":[{},"(p27, Kip1)","p27"]},{"entry":["CDKN2A,","cyclin-dependent kinase inhibitor 2A","CD21A, p16","Cyclin-dependent kinase inhibitor 2A,"]},{"entry":["P16","(melanoma, p16, inhibits CDK4)",{},"isoforms 1\/2\/3"]},{"entry":["CES2","carboxylesterase 2 (intestine, liver)","CES2, EST2","Carboxylesterase 2 precursor"]},{"entry":["CK 5\/6","cytokeratin 5\/cytokeratin 6","CK 5\/6","Keratin, type II cytoskeletal 5; Keratin,"]},{"entry":[{},{},{},"type II cytoskeletal 6"]},{"entry":["CK14,","keratin 14","CK14","Keratin, type I cytoskeletal 14"]},{"entry":"KRT14"},{"entry":["CK17,","keratin 17","CK17","Keratin, type I cytoskeletal 17"]},{"entry":"KRT17"},{"entry":["COX2,","prostaglandin-endoperoxide synthase 2","COX-2,","Prostaglandin G\/H synthase 2 precursor"]},{"entry":["PTGS2","(prostaglandin G\/H synthase and","PTGS2"]},{"entry":[{},"cyclooxygenase)"]},{"entry":["DCK","deoxycytidine kinase","DCK","Deoxycytidine kinase"]},{"entry":["DHFR","dihydrofolate reductase","DHFR","Dihydrofolate reductase"]},{"entry":["DNMT1","DNA (cytosine-5-)-methyltransferase 1","DNMT1","DNA (cytosine-5)-methyltransferase 1"]},{"entry":["DNMT3A","DNA (cytosine-5-)-methyltransferase 3","DNMT3A","DNA (cytosine-5)-methyltransferase 3A"]},{"entry":[{},"alpha"]},{"entry":["DNMT3B","DNA (cytosine-5-)-methyltransferase 3","DNMT3B","DNA (cytosine-5)-methyltransferase 3B"]},{"entry":[{},"beta"]},{"entry":["ECGF1,","thymidine phosphorylase","TYMP, PD-","Thymidine phosphorylase precursor"]},{"entry":["TYMP",{},"ECGF,"]},{"entry":[{},{},"ECDF1"]},{"entry":["EGFR,","epidermal growth factor receptor","EGFR,","Epidermal growth factor receptor"]},{"entry":["ERBB1,","(erythroblastic leukemia viral (v-erb-b)","ERBB1,","precursor"]},{"entry":["HER1","oncogene homolog, avian)","HER1"]},{"entry":["EML4","echinoderm microtubule associated","EML4","Echinoderm microtubule-associated"]},{"entry":[{},"protein like 4",{},"protein-like 4"]},{"entry":["EPHA2","EPH receptor A2","EPHA2","Ephrin type-A receptor 2 precursor"]},{"entry":["ER, ESR1","estrogen receptor 1","ER, ESR1","Estrogen receptor"]},{"entry":["ERBB2,","v-erb-b2 erythroblastic leukemia viral","ERBB2,","Receptor tyrosine-protein kinase erbB-2"]},{"entry":["HER2\/NEU","oncogene homolog 2,","HER2,","precursor"]},{"entry":[{},"neuro\/glioblastoma derived oncogene","HER-2\/neu"]},{"entry":[{},"homolog (avian)"]},{"entry":["ERCC1","excision repair cross-complementing","ERCC1","DNA excision repair protein ERCC-1"]},{"entry":[{},"rodent repair deficiency,"]},{"entry":[{},"complementation group 1 (includes"]},{"entry":[{},"overlapping antisense sequence)"]},{"entry":["ERCC3","excision repair cross-complementing","ERCC3","TFIIH basal transcription factor complex"]},{"entry":[{},"rodent repair deficiency,",{},"helicase XPB subunit"]},{"entry":[{},"complementation group 3 (xeroderma"]},{"entry":[{},"pigmentosum group B complementing)"]},{"entry":["EREG","Epiregulin","EREG","Proepiregulin precursor"]},{"entry":["FLT1","fms-related tyrosine kinase 1 (vascular","FLT-1,","Vascular endothelial growth factor receptor"]},{"entry":[{},"endothelial growth factor\/vascular","VEGFR1","1 precursor"]},{"entry":[{},"permeability factor receptor)"]},{"entry":["FOLR1","folate receptor 1 (adult)","FOLR1","Folate receptor alpha precursor"]},{"entry":["FOLR2","folate receptor 2 (fetal)","FOLR2","Folate receptor beta precursor"]},{"entry":["FSHB","follicle stimulating hormone, beta","FSHB","Follitropin subunit beta precursor"]},{"entry":[{},"polypeptide"]},{"entry":["FSHPRH1,","centromere protein I","FSHPRH1,","Centromere protein I"]},{"entry":["CENP1",{},"CENP1"]},{"entry":["FSHR","follicle stimulating hormone receptor","FSHR","Follicle-stimulating hormone receptor"]},{"entry":[{},{},{},"precursor"]},{"entry":["FYN","FYN oncogene related to SRC, FGR,","FYN","Tyrosine-protein kinase Fyn"]},{"entry":[{},"YES"]},{"entry":["GART","phosphoribosylglycinamide","GART,","Trifunctional purine biosynthetic protein"]},{"entry":[{},"formyltransferase,","PUR2","adenosine-3"]},{"entry":[{},"phosphoribosylglycinamide synthetase,"]},{"entry":[{},"phosphoribosylaminoimidazole"]},{"entry":[{},"synthetase"]},{"entry":["GNRH1","gonadotropin-releasing hormone 1","GNRH1,","Progonadoliberin-1 precursor"]},{"entry":[{},"(luteinizing-releasing hormone)","GON1"]},{"entry":["GNRHR1,","gonadotropin-releasing hormone","GNRHR1","Gonadotropin-releasing hormone receptor"]},{"entry":["GNRHR","receptor"]},{"entry":["GSTP1","glutathione S-transferase pi 1","GSTP1","Glutathione S-transferase P"]},{"entry":["HCK","hemopoietic cell kinase","HCK","Tyrosine-protein kinase HCK"]},{"entry":["HDAC1","histone deacetylase 1","HDAC1","Histone deacetylase 1"]},{"entry":["HGF","hepatocyte growth factor (hepapoietin","HGF","Hepatocyte growth factor precursor"]},{"entry":[{},"A; scatter factor)"]},{"entry":["HIF1A","hypoxia inducible factor 1, alpha","HIF1A","Hypoxia-inducible factor 1-alpha"]},{"entry":[{},"subunit (basic helix-loop-helix"]},{"entry":[{},"transcription factor)"]},{"entry":["HIG1,","HIG1 hypoxia inducible domain","HIG1,","HIG1 domain family member 1A"]},{"entry":["HIGD1A,","family, member 1A","HIGD1A,"]},{"entry":["HIG1A",{},"HIG1A"]},{"entry":["HSP90AA1,","heat shock protein 90 kDa alpha","HSP90,","Heat shock protein HSP 90-alpha"]},{"entry":["HSP90,","(cytosolic), class A member 1","HSP90A"]},{"entry":"HSPCA"},{"entry":["IGF1R","insulin-like growth factor 1 receptor","IGF-1R","Insulin-like growth factor 1 receptor"]},{"entry":[{},{},{},"precursor"]},{"entry":["IGFBP3,","insulin-like growth factor binding","IGFBP-3,","Insulin-like growth factor-binding protein"]},{"entry":["IGFRBP3","protein 3","IBP-3","3 precursor"]},{"entry":["IGFBP4,","insulin-like growth factor binding","IGFBP-4,","Insulin-like growth factor-binding protein"]},{"entry":["IGFRBP4","protein 4","IBP-4","4 precursor"]},{"entry":["IGFBP5,","insulin-like growth factor binding","IGFBP-5,","Insulin-like growth factor-binding protein"]},{"entry":["IGFRBP5","protein 5","IBP-5","5 precursor"]},{"entry":["IL13RA1","interleukin 13 receptor, alpha 1","IL-13RA1","Interleukin-13 receptor subunit alpha-1"]},{"entry":[{},{},{},"precursor"]},{"entry":["KDR","kinase insert domain receptor (a type","KDR,","Vascular endothelial growth factor receptor"]},{"entry":[{},"III receptor tyrosine kinase)","VEGFR2","2 precursor"]},{"entry":["KIT, c-KIT","v-kit Hardy-Zuckerman 4 feline","KIT, c-KIT","Mast\/stem cell growth factor receptor"]},{"entry":[{},"sarcoma viral oncogene homolog",{},"precursor"]},{"entry":["KRAS","v-Ki-ras2 Kirsten rat sarcoma viral","K-RAS","GTPase KRas precursor"]},{"entry":[{},"oncogene homolog"]},{"entry":["LCK","lymphocyte-specific protein tyrosine","LCK","Tyrosine-protein kinase Lck"]},{"entry":[{},"kinase"]},{"entry":["LTB","lymphotoxin beta (TNF superfamily,","LTB, TNF3","Lymphotoxin-beta"]},{"entry":[{},"member 3)"]},{"entry":["LTBR","lymphotoxin beta receptor (TNFR","LTBR,","Tumor necrosis factor receptor superfamily"]},{"entry":[{},"superfamily, member 3)","LTBR3,","member 3 precursor"]},{"entry":[{},{},"TNFR"]},{"entry":["LYN","v-yes-1 Yamaguchi sarcoma viral","LYN","Tyrosine-protein kinase Lyn"]},{"entry":[{},"related oncogene homolog"]},{"entry":["MET, c-","met proto-oncogene (hepatocyte","MET, c-","Hepatocyte growth factor receptor"]},{"entry":["MET","growth factor receptor)","MET","precursor"]},{"entry":["MGMT","O-6-methylguanine-DNA","MGMT","Methylated-DNA--protein-cysteine"]},{"entry":[{},"methyltransferase",{},"methyltransferase"]},{"entry":["MKI67,","antigen identified by monoclonal","Ki67, Ki-67","Antigen KI-67"]},{"entry":["KI67","antibody Ki-67"]},{"entry":["MLH1","mutL homolog 1, colon cancer,","MLH1","DNA mismatch repair protein Mlh1"]},{"entry":[{},"nonpolyposis type 2 ()"]},{"entry":["MMR","mismatch repair (refers to MLH1,"]},{"entry":[{},"MSH2, MSH5)"]},{"entry":["MSH2","mutS homolog 2, colon cancer,","MSH2","DNA mismatch repair protein Msh2"]},{"entry":[{},"nonpolyposis type 1 ()"]},{"entry":["MSH5","mutS homolog 5 ()","MSH5,","MutS protein homolog 5"]},{"entry":[{},{},"hMSH5"]},{"entry":["MYC, c-","v-myc myelocytomatosis viral","MYC, c-","Myc proto-oncogene protein"]},{"entry":["MYC","oncogene homolog (avian)","MYC"]},{"entry":["NBN, P95","nibrin","NBN, p95","Nibrin"]},{"entry":["NDGR1","N-myc downstream regulated 1","NDGR1","Protein NDGR1"]},{"entry":["NFKB1","nuclear factor of kappa light","NFKB1","Nuclear factor NF-kappa-B p105 subunit"]},{"entry":[{},"polypeptide gene enhancer in B-cells 1"]},{"entry":["NFKB2","nuclear factor of kappa light","NFKB2","Nuclear factor NF-kappa-B p100 subunit"]},{"entry":[{},"polypeptide gene enhancer in B-cells 2"]},{"entry":[{},"(p49\/p100)"]},{"entry":["NFKBIA","nuclear factor of kappa light","NFKBIA","NF-kappa-B inhibitor alpha"]},{"entry":[{},"polypeptide gene enhancer in B-cells"]},{"entry":[{},"inhibitor, alpha"]},{"entry":["ODC1","ornithine decarboxylase 1","ODC","Ornithine decarboxylase"]},{"entry":["OGFR","opioid growth factor receptor","OGFR","Opioid growth factor receptor"]},{"entry":["PARP1","poly (ADP-ribose) polymerase 1","PARP-1","Poly [ADP-ribose] polymerase 1"]},{"entry":["PDGFC","platelet derived growth factor C","PDGF-C,","Platelet-derived growth factor C precursor"]},{"entry":[{},{},"VEGF-E"]},{"entry":["PDGFR","platelet-derived growth factor receptor","PDGFR","Platelet-derived growth factor receptor"]},{"entry":["PDGFRA","platelet-derived growth factor receptor,","PDGFRA,","Alpha-type platelet-derived growth factor"]},{"entry":[{},"alpha polypeptide","PDGFR2,","receptor precursor"]},{"entry":[{},{},"CD140 A"]},{"entry":["PDGFRB","platelet-derived growth factor receptor,","PDGFRB,","Beta-type platelet-derived growth factor"]},{"entry":[{},"beta polypeptide","PDGFR,","receptor precursor"]},{"entry":[{},{},"PDGFR1,"]},{"entry":[{},{},"CD140 B"]},{"entry":["PIK3CA","phosphoinositide-3-kinase, catalytic,","PI3K","phosphoinositide-3-kinase, catalytic, alpha"]},{"entry":[{},"alpha polypeptide","subunit","polypeptide"]},{"entry":[{},{},"p110\u03b1"]},{"entry":["PSMD9,","proteasome (prosome, macropain) 26S","p27","26S proteasome non-ATPase regulatory"]},{"entry":["P27","subunit, non-ATPase, 9",{},"subunit 9"]},{"entry":["PTEN","phosphatase and tensin homolog"]},{"entry":["RRM1","ribonucleotide reductase M1","RRM1, RR1","Ribonucleoside-diphosphate reductase"]},{"entry":[{},{},{},"large subunit"]},{"entry":["RRM2","ribonucleotide reductase M2","RRM2,","Ribonucleoside-diphosphate reductase"]},{"entry":[{},{},"RR2M, RR2","subunit M2"]},{"entry":["RRM2B","ribonucleotide reductase M2 B (TP53","RRM2B,","Ribonucleoside-diphosphate reductase"]},{"entry":[{},"inducible)","P53R2","subunit M2 B"]},{"entry":["RXRB","retinoid X receptor, beta","RXRB","Retinoic acid receptor RXR-beta"]},{"entry":["RXRG","retinoid X receptor, gamma","RXRG,","Retinoic acid receptor RXR-gamma"]},{"entry":[{},{},"RXRC"]},{"entry":["SIK2","salt-inducible kinase 2","SIK2,","Salt-inducible protein kinase 2;"]},{"entry":[{},{},"Q9H0K1","Serine\/threonine-protein kinase SIK2"]},{"entry":["SLC29A1","solute carrier family 29 (nucleoside","ENT-1","Equilibrative nucleoside transporter 1"]},{"entry":[{},"transporters), member 1"]},{"entry":["SPARC","secreted protein, acidic, cysteine-rich","SPARC","SPARC precursor; Osteonectin"]},{"entry":[{},"(osteonectin)"]},{"entry":["SRC","v-src sarcoma (Schmidt-Ruppin A-2)","SRC","Proto-oncogene tyrosine-protein kinase"]},{"entry":[{},"viral oncogene homolog (avian)",{},"Src"]},{"entry":["SSTR1","somatostatin receptor 1","SSTR1,","Somatostatin receptor type 1"]},{"entry":[{},{},"SSR1, SS1R"]},{"entry":["SSTR2","somatostatin receptor 2","SSTR2,","Somatostatin receptor type 2"]},{"entry":[{},{},"SSR2, SS2R"]},{"entry":["SSTR3","somatostatin receptor 3","SSTR3,","Somatostatin receptor type 3"]},{"entry":[{},{},"SSR3, SS3R"]},{"entry":["SSTR4","somatostatin receptor 4","SSTR4,","Somatostatin receptor type 4"]},{"entry":[{},{},"SSR4, SS4R"]},{"entry":["SSTR5","somatostatin receptor 5","SSTR5,","Somatostatin receptor type 5"]},{"entry":[{},{},"SSR5, SS5R"]},{"entry":["TK1","thymidine kinase 1, soluble","TK1, KITH","Thymidine kinase, cytosolic"]},{"entry":["TLE3","transducin-like enhancer of split 3","TLE3","Transducin-like enhancer protein 3"]},{"entry":[{},"(E(sp1) homolog, )"]},{"entry":["TNF","tumor necrosis factor (TNF","TNF, TNF-","Tumor necrosis factor precursor"]},{"entry":[{},"superfamily, member 2)","alpha, TNF-a"]},{"entry":["TOP1,","topoisomerase (DNA) I","TOP1,","DNA topoisomerase 1"]},{"entry":["TOPO1",{},"TOPO1"]},{"entry":["TOP2A,","topoisomerase (DNA) II alpha 170 kDa","TOP2A,","DNA topoisomerase 2-alpha;"]},{"entry":["TOPO2A",{},"TOP2,","Topoisomerase II alpha"]},{"entry":[{},{},"TOPO2A"]},{"entry":["TOP2B,","topoisomerase (DNA) II beta 180 kDa","TOP2B,","DNA topoisomerase 2-beta;"]},{"entry":["TOPO2B",{},"TOPO2B","Topoisomerase II beta"]},{"entry":["TP53","tumor protein p53","p53","Cellular tumor antigen p53"]},{"entry":["TUBB3","tubulin, beta 3","Beta III","Tubulin beta-3 chain"]},{"entry":[{},{},"tubulin,"]},{"entry":[{},{},"TUBB3,"]},{"entry":[{},{},"TUBB4"]},{"entry":["TXN","thioredoxin","TXN, TRX,","Thioredoxin"]},{"entry":[{},{},"TRX-1"]},{"entry":["TXNRD1","thioredoxin reductase 1","TXNRD1,","Thioredoxin reductase 1, cytoplasmic;"]},{"entry":[{},{},"TXNR","Oxidoreductase"]},{"entry":["TYMS, TS","thymidylate synthetase","TYMS, TS","Thymidylate synthase"]},{"entry":["VDR","vitamin D (1,25-dihydroxyvitamin D3)","VDR","Vitamin D3 receptor"]},{"entry":[{},"receptor"]},{"entry":["VEGFA,","vascular endothelial growth factor A","VEGF-A,","Vascular endothelial growth factor A"]},{"entry":["VEGF",{},"VEGF","precursor"]},{"entry":["VEGFC","vascular endothelial growth factor C","VEGF-C","Vascular endothelial growth factor C"]},{"entry":[{},{},{},"precursor"]},{"entry":["VHL","von Hippel-Lindau tumor suppressor","VHL","Von Hippel-Lindau disease tumor"]},{"entry":[{},{},{},"suppressor"]},{"entry":["YES1","v-yes-1 Yamaguchi sarcoma viral","YES1, Yes,","Proto-oncogene tyrosine-protein kinase"]},{"entry":[{},"oncogene homolog 1","p61-Yes","Yes"]},{"entry":["ZAP70","zeta-chain (TCR) associated protein","ZAP-70","Tyrosine-protein kinase ZAP-70"]},{"entry":[{},"kinase 70 kDa"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Genes and gene products that are known to play a role in cancer and can be included in a biosignature of the invention include without limitation 2AR, A DISINTEGRIN, ACTIVATOR OF THYROID AND RETINOIC ACID RECEPTOR (ACTR), ADAM 11, ADIPOGENESIS INHIBITORY FACTOR (ADIF), ALPHA 6 INTEGRIN SUBUNIT, ALPHA V INTEGRIN SUBUNIT, ALPHA-CATENIN, AMPLIFIED IN BREAST CANCER 1 (AIB1), AMPLIFIED IN BREAST CANCER 3 (AIB3), AMPLIFIED IN BREAST CANCER 4 (AIB4), AMYLOID PRECURSOR PROTEIN SECRETASE (APPS), AP-2 GAMMA, APPS, ATP-BINDING CASSETTE TRANSPORTER (ABCT), PLACENTA-SPECIFIC (ABCP), ATP-BINDING CASSETTE SUBFAMILY C MEMBER (ABCC1), BAG-1, BASIGIN (BSG), BCEI, B-CELL DIFFERENTIATION FACTOR (BCDF), B-CELL LEUKEMIA 2 (BCL-2), B-CELL STIMULATORY FACTOR-2 (BSF-2), BCL-1, BCL-2-ASSOCIATED X PROTEIN (BAX), BCRP, BETA 1 INTEGRIN SUBUNIT, BETA 3 INTEGRIN SUBUNIT, BETA 5 INTEGRIN SUBUNIT, BETA-2 INTERFERON, BETA-CATENIN, BETA-CATENIN, BONE SIALOPROTEIN (BSP), BREAST CANCER ESTROGEN-INDUCIBLE SEQUENCE (BCEI), BREAST CANCER RESISTANCE PROTEIN (BCRP), BREAST CANCER TYPE 1 (BRCA1), BREAST CANCER TYPE 2 (BRCA2), BREAST CARCINOMA AMPLIFIED SEQUENCE 2 (BCAS2), CADHERIN, EPITHELIAL CADHERIN-11, CADHERIN-ASSOCIATED PROTEIN, CALCITONIN RECEPTOR (CTR), CALCIUM PLACENTAL PROTEIN (CAPL), CALCYCLIN, CALLA, CAM5, CAPL, CARCINOEMBRYONIC ANTIGEN (CEA), CATENIN, ALPHA 1, CATHEPSIN B, CATHEPSIN D, CATHEPSIN K, CATHEPSIN L2, CATHEPSIN O, CATHEPSIN O1, CATHEPSIN V, CD10, CD146, CD147, CD24, CD29, CD44, CD51, CD54, CD61, CD66e, CD82, CD87, CD9, CEA, CELLULAR RETINOL-BINDING PROTEIN 1 (CRBP1), c-ERBB-2, CK7, CK8, CK18, CK19, CK20, CLAUDIN-7, c-MET, COLLAGENASE, FIBROBLAST, COLLAGENASE, INTERSTITIAL, COLLAGENASE-3, COMMON ACUTE LYMPHOCYTIC LEUKEMIA ANTIGEN (CALLA), CONNEXIN 26 (Cx26), CONNEXIN 43 (Cx43), CORTACTIN, COX-2, CTLA-8, CTR, CTSD, CYCLIN D1, CYCLOOXYGENASE-2, CYTOKERATIN 18, CYTOKERATIN 19, CYTOKERATIN 8, CYTOTOXIC T-LYMPHOCYTE-ASSOCIATED SERINE ESTERASE 8 (CTLA-8), DIFFERENTIATION-INHIBITING ACTIVITY (DIA), DNA AMPLIFIED IN MAMMARY CARCINOMA 1 (DAM1), DNA TOPOISOMERASE II ALPHA, DR-NM23, E-CADHERIN, EMMPRIN, EMS1, ENDOTHELIAL CELL GROWTH FACTOR (ECGR), PLATELET-DERIVED (PD-ECGF), ENKEPHALINASE, EPIDERMAL GROWTH FACTOR RECEPTOR (EGFR), EPISIALIN, EPITHELIAL MEMBRANE ANTIGEN (EMA), ER-ALPHA, ERBB2, ERBB4, ER-BETA, ERF-1, ERYTHROID-POTENTIATING ACTIVITY (EPA), ESR1, ESTROGEN RECEPTOR-ALPHA, ESTROGEN RECEPTOR-BETA, ETS-1, EXTRACELLULAR MATRIX METALLOPROTEINASE INDUCER (EMMPRIN), FIBRONECTIN RECEPTOR, BETA POLYPEPTIDE (FNRB), FIBRONECTIN RECEPTOR BETA SUBUNIT (FNRB), FLK-1, GA15.3, GA733.2, GALECTIN-3, GAMMA-CATENIN, GAP JUNCTION PROTEIN (26 kDa), GAP JUNCTION PROTEIN (43 kDa), GAP JUNCTION PROTEIN ALPHA-1 (GJA1), GAP JUNCTION PROTEIN BETA-2 (GJB2), GCP1, GELATINASE A, GELATINASE B, GELATINASE (72 kDa), GELATINASE (92 kDa), GLIOSTATIN, GLUCOCORTICOID RECEPTOR INTERACTING PROTEIN 1 (GRIP1), GLUTATHIONE S-TRANSFERASE p, GM-CSF, GRANULOCYTE CHEMOTACTIC PROTEIN 1 (GCP1), GRANULOCYTE-MACROPHAGE-COLONY STIMULATING FACTOR, GROWTH FACTOR RECEPTOR BOUND-7 (GRB-7), GSTp, HAP, HEAT-SHOCK COGNATE PROTEIN 70 (HSC70), HEAT-STABLE ANTIGEN, HEPATOCYTE GROWTH FACTOR (HGF), HEPATOCYTE GROWTH FACTOR RECEPTOR (HGFR), HEPATOCYTE-STIMULATING FACTOR III (HSF III), HER-2, HER2\/NEU, HERMES ANTIGEN, HET, HHM, HUMORAL HYPERCALCEMIA OF MALIGNANCY (HHM), ICERE-1, INT-1, INTERCELLULAR ADHESION MOLECULE-1 (ICAM-1), INTERFERON-GAMMA-INDUCING FACTOR (IGIF), INTERLEUKIN-1 ALPHA (IL-1A), INTERLEUKIN-1 BETA (IL-1B), INTERLEUKIN-11 (IL-11), INTERLEUKIN-17 (IL-17), INTERLEUKIN-18 (IL-18), INTERLEUKIN-6 (IL-6), INTERLEUKIN-8 (IL-8), INVERSELY CORRELATED WITH ESTROGEN RECEPTOR EXPRESSION-1 (ICERE-1), KAI1, KDR, KERATIN 8, KERATIN 18, KERATIN 19, KISS-1, LEUKEMIA INHIBITORY FACTOR (LIF), LIF, LOST IN INFLAMMATORY BREAST CANCER (LIBC), LOT (\u201cLOST ON TRANSFORMATION\u201d), LYMPHOCYTE HOMING RECEPTOR, MACROPHAGE-COLONY STIMULATING FACTOR, MAGE-3, MAMMAGLOBIN, MASPIN, MC56, M-CSF, MDC, MDNCF, MDR, MELANOMA CELL ADHESION MOLECULE (MCAM), MEMBRANE METALLOENDOPEPTIDASE (MME), MEMBRANE-ASSOCIATED NEUTRAL ENDOPEPTIDASE (NEP), CYSTEINE-RICH PROTEIN (MDC), METASTASIN (MTS-1), MLN64, MMP1, MMP2, MMP3, MMP7, MMP9, MMP11, MMP13, MMP14, MMP15, MMP16, MMP17, MOESIN, MONOCYTE ARGININE-SERPIN, MONOCYTE-DERIVED NEUTROPHIL CHEMOTACTIC FACTOR, MONOCYTE-DERIVED PLASMINOGEN ACTIVATOR INHIBITOR, MTS-1, MUC-1, MUC18, MUCIN LIKE CANCER ASSOCIATED ANTIGEN (MCA), MUCIN, MUC-1, MULTIDRUG RESISTANCE PROTEIN 1 (MDR, MDR1), MULTIDRUG RESISTANCE RELATED PROTEIN-1 (MRP, MRP-1), N-CADHERIN, NEP, NEU, NEUTRAL ENDOPEPTIDASE, NEUTROPHIL-ACTIVATING PEPTIDE 1 (NAP1), NM23-H1, NM23-H2, NME1, NME2, NUCLEAR RECEPTOR COACTIVATOR-1 (NCoA-1), NUCLEAR RECEPTOR COACTIVATOR-2 (NCoA-2), NUCLEAR RECEPTOR COACTIVATOR-3 (NCoA-3), NUCLEOSIDE DIPHOSPHATE KINASE A (NDPKA), NUCLEOSIDE DIPHOSPHATE KINASE B (NDPKB), ONCOSTATIN M (OSM), ORNITHINE DECARBOXYLASE (ODC), OSTEOCLAST DIFFERENTIATION FACTOR (ODF), OSTEOCLAST DIFFERENTIATION FACTOR RECEPTOR (ODFR), OSTEONECTIN (OSN, ON), OSTEOPONTIN (OPN), OXYTOCIN RECEPTOR (OXTR), p27\/kip1, p300\/CBP COINTEGRATOR ASSOCIATE PROTEIN (p\/CIP), p53, p9Ka, PAI-1, PAI-2, PARATHYROID ADENOMATOSIS 1 (PRAD1), PARATHYROID HORMONE-LIKE HORMONE (PTHLH), PARATHYROID HORMONE-RELATED PEPTIDE (PTHrP), P-CADHERIN, PD-ECGF, PDGF, PEANUT-REACTIVE URINARY MUCIN (PUM), P-GLYCOPROTEIN (P-GP), PGP-1, PHGS-2, PHS-2, PIP, PLAKOGLOBIN, PLASMINOGEN ACTIVATOR INHIBITOR (TYPE 1), PLASMINOGEN ACTIVATOR INHIBITOR (TYPE 2), PLASMINOGEN ACTIVATOR (TISSUE-TYPE), PLASMINOGEN ACTIVATOR (UROKINASE-TYPE), PLATELET GLYCOPROTEIN IIIa (GP3A), PLAU, PLEOMORPHIC ADENOMA GENE-LIKE 1 (PLAGL1), POLYMORPHIC EPITHELIAL MUCIN (PEM), PRAD1, PROGESTERONE RECEPTOR (PgR), PROGESTERONE RESISTANCE, PROSTAGLANDIN ENDOPEROXIDE SYNTHASE-2, PROSTAGLANDIN G\/H SYNTHASE-2, PROSTAGLANDIN H SYNTHASE-2, pS2, PS6K, PSORIASIN, PTHLH, PTHrP, RAD51, RAD52, RAD54, RAP46, RECEPTOR-ASSOCIATED COACTIVATOR 3 (RAC3), REPRESSOR OF ESTROGEN RECEPTOR ACTIVITY (REA), S100A4, S100A6, S100A7, S6K, SART-1, SCAFFOLD ATTACHMENT FACTOR B (SAF-B), SCATTER FACTOR(SF), SECRETED PHOSPHOPROTEIN-1 (SPP-1), SECRETED PROTEIN, ACIDIC AND RICH IN CYSTEINE (SPARC), STANNICALCIN, STEROID RECEPTOR COACTIVATOR-1 (SRC-1), STEROID RECEPTOR COACTIVATOR-2 (SRC-2), STEROID RECEPTOR COACTIVATOR-3 (SRC-3), STEROID RECEPTOR RNA ACTIVATOR(SRA), STROMELYSIN-1, STROMELYSIN-3, TENASCIN-C (TN-C), TESTES-SPECIFIC PROTEASE 50, THROMBOSPONDIN I, THROMBOSPONDIN II, THYMIDINE PHOSPHORYLASE (TP), THYROID HORMONE RECEPTOR ACTIVATOR MOLECULE 1 (TRAM-1), TIGHT JUNCTION PROTEIN 1 (TJP1), TIMP1, TIMP2, TIMP3, TIMP4, TISSUE FACTOR (TF), TISSUE-TYPE PLASMINOGEN ACTIVATOR, TN-C, TP53, tPA, TRANSCRIPTIONAL INTERMEDIARY FACTOR 2 (TIF2), TREFOIL FACTOR 1 (TFF1), TSG101, TSP-1, TSP1, TSP-2, TSP2, TSP50, TUMOR CELL COLLAGENASE STIMULATING FACTOR (TCSF), TUMOR-ASSOCIATED EPITHELIAL MUCIN, uPA, uPAR, UROKINASE, UROKINASE-TYPE PLASMINOGEN ACTIVATOR, UROKINASE-TYPE PLASMINOGEN ACTIVATOR RECEPTOR (uPAR), UVOMORULIN, VASCULAR ENDOTHELIAL GROWTH FACTOR, VASCULAR ENDOTHELIAL GROWTH FACTOR RECEPTOR-2 (VEGFR2), VASCULAR ENDOTHELIAL GROWTH FACTOR-A, VASCULAR PERMEABILITY FACTOR, VEGFR2, VERY LATE T-CELL ANTIGEN BETA (VLA-BETA), VIMENTIN, VITRONECTIN RECEPTOR ALPHA POLYPEPTIDE (VNRA), VITRONECTIN RECEPTOR, VON WILLEBRAND FACTOR, VPF, VWF, WNT-1, ZAC, ZO-1, and ZONULA OCCLUDENS-1. The genes and\/or gene products can be part of a biosignature for theranosing a cancer.","As an illustration, a treatment can be selected for a subject suffering from Non-Small Cell Lung Cancer. One or more biomarkers, such as, but not limited to, EGFR, excision repair cross-complementation group 1 (ERCC1), p53, Ras, p27, class III beta tubulin, breast cancer gene 1 (BRCA1), breast cancer gene 1 (BRCA2), and ribonucleotide reductase messenger 1 (RRM1), can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Erlotinib, Carboplatin, Paclitaxel, Gefitinib, or a combination thereof.","In another embodiment, a treatment can be selected for a subject suffering from Colorectal Cancer, and a biomarker, such as, but not limited to, K-ras, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Panitumumab, Cetuximab, or a combination thereof.","In another embodiment, a treatment can be selected for a subject suffering from Breast Cancer. One or more biomarkers, such as, but not limited to, HER2, toposiomerase II \u03b1, estrogen receptor, and progesterone receptor, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, trastuzumab, anthracyclines, taxane, methotrexate, fluorouracil, or a combination thereof.","As described, the biosignature used to theranose a cancer can comprise analysis of one or more biomarker, which can be a protein or nucleic acid, including a mRNA or a microRNA. The biomarker can be detected in a bodily fluid and\/or can be detected associated with a vesicle, e.g., as a vesicle antigen or as vesicle payload. In an illustrative example, the biosignature is used to identify a patient as a responder or non-responder to a tyrosine kinase inhibitor. The biomarkers can be one or more of those described in WO\/2010\/121238, entitled \u201cMETHODS AND KITS TO PREDICT THERAPEUTIC OUTCOME OF TYROSINE KINASE INHIBITORS\u201d and filed Apr. 19, 2010; or WO\/2009\/105223, entitled \u201cSYSTEMS AND METHODS OF CANCER STAGING AND TREATMENT\u201d and filed Feb. 19, 2009; both of which applications are incorporated herein by reference in their entirety.","In an aspect, the present invention provides a method of determining whether a subject is likely to respond or not to a tyrosine kinase inhibitor, the method comprising identifying one or more biomarker in a vesicle population in a sample from the subject, wherein differential expression of the one or more biomarker in the sample as compared to a reference indicates that the subject is a responder or non-responder to the tyrosine kinase inhibitor. In an embodiment, the one or more biomarker comprises miR-497, wherein reduced expression of miR-497 indicates that the subject is a responder (i.e., sensitive to the tyrosine kinase inhibitor). In another embodiment, the one or more biomarker comprises one or more of miR-21, miR-23a, miR-23b, and miR-29b, wherein upregulation of the microRNA indicates that the subject is a likely non-responder (i.e., resistant to the tyrosine kinase inhibitor). In some embodiments, the one or more biomarker comprises one or more of hsa-miR-029a, hsa-let-7d, hsa-miR-100, hsa-miR-1260, hsa-miR-025, hsa-let-7i, hsa-miR-146a, hsa-miR-594-Pre, hsa-miR-024, FGFR1, MET, RAB25, EGFR, KIT and VEGFR2. In another embodiment, the one or more biomarker comprises FGF1, HOXC10 or LHFP, wherein higher expression of the biomarker indicates that the subject is a non-responder (i.e., resistant to the tyrosine kinase inhibitor). The method can be used to determine the sensitivity of a cancer to the tyrosine kinase inhibitor, e.g., a non-small cell lung cancer cell, kidney cancer or GIST. The tyrosine kinase inhibitor can be erlotinib, vandetanib, sunitinib and\/or sorafenib, or other inhibitors that operate by a similar mechanism of action. A tyrosine kinase inhibitor includes any agent that inhibits the action of one or more tyrosine kinases in a specific or non-specific fashion. Tyrosine kinase inhibitors include small molecules, antibodies, peptides, or any appropriate entity that directly, indirectly, allosterically, or in any other way inhibits tyrosine residue phosphorylation. Specific examples of tyrosine kinase inhibitors include N-(trifluoromethylphenyl)-5-methylisoxazol-4-carboxamide, 3-[(2,4-dimethylpyrrol-5-yl)methylidenyl)indolin-2-one, 17-(allylamino)-17-demethoxygeldanamycin, 4-(3-chloro-4-fluorophenylamino)-7-methoxy-6-[3-(4-morpholinyl)propoxyl]q-uinazoline, N-(3-ethynylphenyl)-6,7-bis(2-methoxyethoxy)-4-quinazolinamine, BIBX1382, 2,3,9,10,11,12-hexahydro-10-(hydroxymethyl)-10-hydroxy-9-methyl-9,12-epox-y-1H-d{umlaut over (\u03c5)}ndolo[1,2,3-fg:3\u2032,2\u2032,1\u2032-k1]pyrrolo[3,4-i][1,6]benzodiazocin-1-one, SH268, genistein, STI571, CEP2563, 4-(3-chlorophenylamino)-5,6-dimethyl-7H-pyrrolo[2,3-d]pyrimidinemethane sulfonate, 4-(3-bromo-4-hydroxyphenyl)amino-6,7-dimethoxyquinazoline, 4-(4\u2032-hydroxyphenyl)amino-6,7-dimethoxyquinazoline, SU6668, STI571A, N-4-chlorophenyl-4-(4-pyridylmethyl)-1-phthalazinamine, N-[2-(diethylamino)ethyl]-5-[(Z)-(5-fluoro-1,2-dihydro-2-oxo-3H-indol-3-ylidine)methyl]-2,4-dimethyl-1H-pyrrole-3-carboxamide (commonly known as sunitinib), A-[A-[[4-chloro-3 (trifluoromethyl)phenyl] carbamoylamino]phenoxy]-N-methyl-pyridine-2-carboxamide (commonly known as sorafenib), EMD121974, and N-(3-ethynylphenyl)-6,7-bis(2-methoxyethoxy)quinazolin-4-amine (commonly known as erlotinib). In some embodiments, the tyrosine kinase inhibitor has inhibitory activity upon the epidermal growth factor receptor (EGFR), VEGFR, PDGFR beta, and\/or FLT3.","Thus, a treatment can be selected for the subject suffering from a cancer, based on a biosignature identified by the methods of the invention. Accordingly, the biosignature can comprise a presence or level of a circulating biomarker, including a microRNA, a vesicle, or any useful vesicle associated biomarker.","Cardiovascular","Assessing a vesicle can be used in the theranosis of a cardiovascular condition, disorder, or disease. A cardiovascular condition includes, but is not limited to, chronic rheumatic heart disease, hypertensive disease, ischemic heart disease, pulmonary circulatory disease, heart disease, cerebrovascular disease, diseases of arteries, arterioles and capillaries and diseases of veins and lymphatics. A chronic rheumatic heart disease includes, but is not limited to diseases of mitral valve, diseases of aortic valve, diseases of mitral and aortic valves, and diseases of other endocardial structures. A hypertensive disease includes, but is not limited to essential hypertension, hypertension, malignant, hypertension, benign, hypertension, unspecified, hypertensive heart disease, hypertensive renal disease, hypertensive renal disease, unspecified, with renal failure, hypertensive heart and renal disease, hypertension, renovascular, malignant, and hypertension, renovascular benign. An ischemic heart disease includes, but is not limited to acute myocardial infarction, myocardiac infarction, acute, anterolateral, myocardiac infarction, acute, anterior, myocardiac infarction, acute, inferolateral, myocardiac infarction, acute, inferoposterior, myocardiac infarction, acute, other inferior wall, myocardiac infarction, acute, other lateral wall, myocardiac infarction, acute, true posterior, myocardiac infarction, acute, subendocardial, myocardiac infarction, acute, spec, myocardiac infarction, acute, unspecified, postmyocardial infarction syndrome, intermediate coronary syndrome, old myocardial infarction, angina pectoris, angina decubitus, prinzmetal angina, coronary atherosclerosis, aneurysm and dissection of heart, aneurysm of heart wall, aneurysm of coronary vessels, dissection of coronary artery, and unspecified chronic ischemic heart disease.","A pulmonary circulatory disease includes, but is not limited to, diseases of pulmonary circulation, acute pulmonary heart disease, pulmonary embolism, not iatrogenic, chronic pulmonary heart disease, and unspecified chronic pulmonary heart disease. A heart disease includes, but is not limited to acute pericarditis, other and unspecified acute pericarditis, acute nonspecific pericarditis, acute and subacute endocarditis, acute bacterial endocarditis acute myocarditis, other and unspecified acute myocarditis, myocarditis, idiopathic, other diseases of pericardium, other diseases of endocardium, alvular disorder, mitral, valvular disorder, aortic, valvular disorder, tricuspid, valvular disorder, pulmonic, cardiomyopathy, hypertrophic obstructive cardiomyopathy, conduction disorders, atrioventricular block, third degree, atrioventricular block, first degree, atrioventricular block, mobitz ii, atrioventricular block, wenckebach's, bundle branch block, left, bundle branch block, right, sinoatrial heart block, atrioventricular excitation, anomalous, Wolff Parkinson White syndrome, cardiac dysrhythmias, tachycardia, paroxysmal supraventricular, atrial fibrillation and flutter, atrial fibrillation, atrial flutter, ventricular fibrillation and flutter, ventricular fibrillation, cardiac arrest, premature beats, other specified cardiac dysrhythmias, sick sinus syndrome, sinus bradycardia, cardiac dysrhythmia unspecified, gallop rhythm, heart failure, heart failure, congestive, acute pulmonary edema, systolic unspecified heart failure, acute systolic heart failure, chronic systolic heart failure, diastolic unspecified heart failure, diastolic chronic heart failure, combined unspecified heart failure, and cardiomegaly.","A cerebrovascular disease includes, but is not limited to subarachnoid hemorrhage, intracerebral hemorrhage, other and unspecified intracranial hemorrhage, intracranial hemorrhage, occlusion and stenosis of precerebral arteries, occlusion and stenosis of basilar artery, occlusion and stenosis of carotid artery, occlusion and stenosis of vertebral artery, occlusion of cerebral arteries, cerebral thrombosis, cerebral thrombosis without cerebral infarction, cerebral thrombosis with cerebral infarction, cerebral embolism, cerebral embolism without cerebral infarction, cerebral embolism with cerebral infarction, transient cerebral ischemia, basilar artery syndrome, vertebral artery syndrome, subclavian steal syndrome, vertebrobasilar artery syndrome, transient ischemic attack, acute but ill defined cerebrovascular disease, ill defined cerebrovascular disease, cerebral atherosclerosis, other generalized ischemic cerebrovascular disease, hypertensive encephalopathy, cerebral aneurysm nonruptured, cerebral arteritis, moyamoya disease, nonpyogenic thrombosis of intracranial venous sinus, transient global amnesia, late effects of cerebrovascular disease, cognitive deficits, speech and language deficits, unspecified speech and language deficits, aphasia, dysphasia, other speech and language deficits, hemiplegia\/hemiparesis, hemiplegia affecting unspecified side, hemiplegia affecting dominant side, hemiplegia affecting nondominant side, monoplegia of upper limb, monoplegia of lower limb, other paralytic syndrome, other late effects of cerebrovascular disease, apraxia cerebrovascular disease, dysphagia cerebrovascular disease, facial weakness, ataxia, and vertigo.","Diseases of arteries, arterioles and capillaries include, but are not limited to atherosclerosis, atherosclerosis of renal artery, atherosclerosis of native arteries of the extremities, intermittent claudication, atherosclerosis, extremities, without ulceration, atherosclerosis, not heart\/brain, aortic aneurysm, dissection of aorta, abdominal ruptured aortic aneurysm, abdominal, without ruptured aortic aneurysm, unspecified aortic aneurysm, other aneurysm, other peripheral vascular disease, raynaud's syndrome, thromboangiitis obliterans, other arterial dissection, dissection of carotid artery, dissection of iliac artery, dissection of renal artery, dissection of vertebral artery, dissection of other artery, erythromelalgia, unspecified peripheral vascular disease, arterial embolism and thrombosis, polyarteritis nodosa and allied conditions, polyarteritis nodosa, kawasaki disease\/acute febrile mucocutaneous lymph node syndrome, hypersensitivity angiitis, goodpasture's syndrome, lethal midline granuloma, wegener's granulomatosis, giant cell arteritis, thrombotic microangiopathy, takayasu's disease, other disorders of arteries and arterioles, arteriovenous fistula acquired, arteritis unspecified, vasculitis, and vascular non-neoplastic nevus.","Diseases of veins and lymphatics include, but are not limited to, phlebitis and thrombophlebitis, femoral deep vein thrombosis, deep vein thrombosis of other leg veins, phlebitis of other sites, superficial veins of upper extremity, unspecified thrombophlebitis, portal vein thrombosis, other venous embolism and thrombosis, unspecified deep vein thrombosis, proximal deep vein thrombosis, distal deep vein thrombosis, unspecified venous embolism, varicose veins of lower extremities, varicose veins without ulcer, varicose veins without inflammation, varicose veins withoutulcer, inflammation, varicose veins, asymptomatic, hemorrhoids, hemorrhoids, internal without complication, hemorrhoids, external without complication, hemorrhoids, external thrombosed, hemorrhoids, varicose veins of other sites, esophageal varices without bleeding, esophageal varices without bleeding, varicocele, noninfective disorders of lymphatic channels, postmastectomy lymphedema syndrome, hypotension, orthostatic hypotension, iatrogenic hypotension, other disorders of circulatory system, other specified disorders of circulatory system, and unspecified venous insufficiency.","Other examples of cardiac conditions include, without limitation, coronary artery occlusion (e.g., resulting from or associated with lipid\/cholesterol deposition, macrophage\/inflammatory cell recruitment, plaque rupture, thrombosis, platelet deposition, or neointimal proliferation); ischemic syndromes (e.g., resulting from or associated with myocardial infarction, stable angina, unstable angina, coronary artery restenosis or reperfusion injury); cardiomyopathy (e.g., resulting from or associated with an ischemic syndrome, a cardiotoxin, an infection, hypertension, a metabolic disease (such as uremia, beriberi, or glycogen storage disease), radiation, a neuromuscular disease, an infiltrative disease (such as sarcoidosis, hemochromatosis, amyloidosis, Fabry's disease, or Hurler's syndrome), trauma, or an idiopathic cause); arrhythmia or dysrrhythmia (e.g., resulting from or associated with an ischemic syndrome, a cardiotoxin, adriamycin, an infection, hypertension, a metabolic disease, radiation, a neuromuscular disease, an infiltrative disease, trauma, or an idiopathic cause); infection (e.g., caused by a pathogenic agent such as a bacterium, a virus, a fungus, or a parasite); and an inflammatory condition (e.g., associated with myocarditis, pericarditis, endocarditis, immune cardiac rejection, or an inflammatory conditions resulting from one of idiopathic, autoimmune, or a connective tissue disease).","Cardiovascular: Biosignature","A biosignature of a vesicle can be assessed to provide a theranosis for a subject. The biosignature of the vesicle can comprise one or more biomarkers such as, but not limited to, any one or more biomarkers as described herein, such as, but not limited to, those listed in , miR-21, miR-129, miR-212, miR-214, miR-134, and others such as described in US Publication No. 2010\/0010073.","Cardiovascular: Standard of Care","Determining the biosignature of a vesicle, the amount of vesicles, or both, of a sample from a subject suffering from a cardiac condition, disorder, or disease, can be used select a standard of care for the subject. The standard of care may include therapeutic agents or procedures (e.g., angioplasty). Examples of therapeutic agents include, without limitation, angiogenesis promoters (e.g., vascular endothelial growth factor, nitric oxide releasing or generating agents, fibroblast growth factor, platelet derived growth factor, interleukin-6, monocyte chemotactic protein-1, granulocyte-macrophage colony stimulating factor, transforming growth factor-.beta.), anti-thrombotic agents (e.g., aspirin, heparin, PPACK, enoxaprin, hirudin), anticoagulants, antibiotics, antiplatelet agents, thrombolytics (e.g., tissue plasminogen activator), antiproliferatives, antiinflammatories, agents that inhibit hyperplasia, agents that inhibit restenosis, smooth muscle cell inhibitors, growth factors, growth factor inhibitors, cell adhesion inhibitors, chemotherapeutic agents, and combinations thereof.","For example, detection of one or more microRNAs biomarkers, such as miR-21, miR-129, miR-212, miR-214, miR-134 or a combination thereof from vesicles can be used to characterize a cardiac hypertrophy and\/or heart failure, which provides a theranosis for the cardiac hypertrophy. The theranosis can include selecting a therapy such as adminstering angiogenesis promoters. Other examples of treatments include those for treating abnormal cholesterol and\/or triglyceride levels in the blood, such as listed in Table 12.",{"@attributes":{"id":"p-1019","num":"1021"},"tables":{"@attributes":{"id":"TABLE-US-00012","num":"00012"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"322pt","align":"center"}},"thead":{"row":{"entry":"TABLE 12"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Examples of Classes of Drugs for Treatment of Cardiovascular Conditions"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"189pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Class","Mechanism of Action","Examples"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Statins","Competitive inhibitors of HMG-CoA reductase","Atorvastatin, Simvastatin,"]},{"entry":[{},{},"Pravastatin, Fluvastatin,"]},{"entry":[{},{},"Rosuvastatin, Lovastatin,"]},{"entry":[{},{},"Pitavastatin, Cerivastatin"]},{"entry":[{},{},"(withdrawn)"]},{"entry":["Fibrates","PPAR\u03b1 activators","Fenofibrate, Bezafibrate,"]},{"entry":[{},{},"Gemfibrozil, clofibrate,"]},{"entry":[{},{},"ciprofibrate"]},{"entry":["Cholesterol","May inhibit NCP1L1 in gut","Ezetimibe"]},{"entry":"Absorption"},{"entry":"Inhibitors"},{"entry":["Nicotinic","Inhibits cholesterol and triglyceride synthesis, exact mechanism","Niacin"]},{"entry":["Acid","unknown"]},{"entry":"Derivatives"},{"entry":["Bile Acid","Interrupt the enterohepatic circulation of bile acids","Colesevelam,"]},{"entry":["Sequestrants",{},"Cholestyramine,"]},{"entry":[{},{},"Colestimide, Colestipol"]},{"entry":["Cholesteryl","Inhibit cholesteryl ester transfer protein, a plasma protein that","JTT-705, CETi-1,"]},{"entry":["Ester Transfer","mediates the exchange of cholesteryl esters from antiatherogenic","Torcetrapib"]},{"entry":["Protein","HDL to proatherogenic apoliprotein B-containing lipoproteins"]},{"entry":"Inhibitors"},{"entry":["Reverse Lipid","Stimulate reverse lipid transport, a four-step process form","ETC-216, ETC-588, ETC-"]},{"entry":["Transport","removing excess cholesterol and other lipids from the walls of","642, ETC-1001, ESP-1552,"]},{"entry":["Pathway","arteries and other tissues","ESP-24232"]},{"entry":"Activators"},{"entry":["Antioxidants\/","Inhibit vascular inflammation and reduce cholesterol levels;","AGI-1067, Probucol"]},{"entry":["Vascular","block oxidant signals that switch on vascular cellular adhesion","(withdrawn)"]},{"entry":["Protectants","molecule (VCAM)-1"]},{"entry":["Acyl-CoA","Inhibit ACAT, which catalyzes cholesterol esterification,","Eflucimibe, Pactimibe,"]},{"entry":["Cholesterol","regulates intracellular free cholesterol, and promotes cholesterol","Avasimibe (withdrawn),"]},{"entry":["Acyltransferase","absorption and assemble of VLDL","SMP-797"]},{"entry":"(ACAT)"},{"entry":"Inhibitors"},{"entry":["Peroxisome","Activate PPARs, e.g., PPAR\u03b1, \u03b3, and possibly \u03b4, which have a","Tesaglitazar, GW-50516,"]},{"entry":["Proliferator","variety of gene regulatory functions","GW-590735, LY-929, LY-"]},{"entry":["Activated",{},"518674, LY-465608, LY-"]},{"entry":["Receptor",{},"818"]},{"entry":"Agonists"},{"entry":["Microsomal","Inhibit MTTP, which catalyze the transport of triglycerides,","Implitapide, CP-346086"]},{"entry":["Triglyceride","cholesteryl ester, and phosphatidylcholine between membranes;"]},{"entry":["Transfer","required for the synthesis of ApoB."]},{"entry":"Protein"},{"entry":"(MTTP)"},{"entry":"Inhibitors"},{"entry":["Squalene","Interfere with cholesterol synthesis by halting the action of liver","TAK-475, ER-119884"]},{"entry":["Synthase","enzymes; may also slow or stop the proliferation of several cell"]},{"entry":["Inhibitors","types that contribute to atherosclerotic plaque formation"]},{"entry":["Lipoprotein","Directly activate lipoprotein lipase, which promotes the","Ibrolipim (NO-1886)"]},{"entry":["Lipase","breakdown of the fat portion of lipoproteins"]},{"entry":"Activators"},{"entry":["Liproprotein","Not yet established","Gembacene"]},{"entry":"(a) Antagonists"},{"entry":["Bile Acid","Inhibit intestinal epithelial uptake of bile acids.","AZD-7806, BARI-1453, S-"]},{"entry":["Reabsorption",{},"8921"]},{"entry":"Inhibitors"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"In one embodiment, a treatment can be selected for a subject suffering from Peripheral Arterial Disease. One or more biomarkers, such as, but not limited to, C-reactive protein (CRP), serum Amylyoid A (SAA), interleukin-6, intracellular adhesion molecule (ICAM), vascular adhesion molecule (VCAM), CD40L, fibrinogen, fibrin D-dimer, fibrinopeptide A, von Willibrand factor, tissue plasminogen activator antigen (t-PA), factor VII, prothrombin fragment 1, oxidized low density lipoprotein (oxLDL), and lipoprotein A, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Atorvastatin, Simvastatin, Rosuvastatin, Pravastatin, Fluvastatin, Lovastatin, or a combination thereof.","In another embodiment, a treatment can be selected for a subject suffering from an arrhythmia. One or more biomarkers, such as, but not limited to, SERCA, AAP, Connexin 40, Connexin 43, ATP-sensitive potassium channel, Kv1.5 channel, and acetylcholine-activated posassium channel, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Disopyramide, Flecamide, Lidocaine, Mexiletine, Moricizine, Procainamide, Propafenone, Quinidine, Tocamide, Acebutolol, Atenolol, Betaxolol, Bisoprolol, Carvedilol, Esmolol, Metoprolol, Nadolol, Propranolol, Sotalol, Timolol, Amiodarone, Azimilide, Bepridil, Dofetilide, Ibutilide, Tedisamil, Diltiazem, Verapamil, Azimilide, Dronedarone, Amiodarone, PM101, ATI-2042, Tedisamil, Nifekalant, Ambasilide, Ersentilide, Trecetilide, Almokalant, D-sotalol, BRL-32872, HMR1556, L768673, Vernakalant, AZD70009, AVE0118, 59947, NIP-141\/142, XEN-D0101\/2, Ranolazine, Pilsicamide, JTV519, Rotigaptide, GAP-134, or a combination thereof.","In another embodiment, a treatment can be selected for a subject suffering from abnormal coagulation. One or more biomarkers, such as, but not limited to, F 1.2, TAT, FPA, beta-throboglobulin, platelet factor 4, soluble P-selectin, IL-6, and CRP can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, aspirin, anticoagulants, ximelagatran, Heparin, Warfarin, or a combination thereof.","In another embodiment, a treatment can be selected for a subject suffering from Premature Atherosclerosis. One or more biomarkers, such as, but not limited to, CRP, NF-kB, IL-1, IL-6, IL-18, Apo-B, Lp-PLA2, Fibrinogen, Hcy, and Hcy-thiolactone can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment.","In yet another embodiment, a treatment can be selected for a subject suffering from Hypertension. One or more biomarkers, such as, but not limited to, Brain natriuretic peptide and N-terminal prohormone BNP, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment.","In another embodiment, a treatment can be selected for a subject suffering from Cardiovascular Disease. One or more biomarkers, such as, but not limited to, an ACE inhibitor or angiotensin can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, lisinopril, candesartan, enalapril, or a combination thereof.","Thus, a treatment can be selected for the subject suffering from a cardiology related condition or cardiovascular condition, based on the biosignature of the subject's vesicle.","Autoimmune","Assessing a vesicle can be used in the theranosis of an autoimmune condition, disorder, or disease. Autoimmune conditions are conditions where a mammal's immune system starts reacting against its own tissues. Such conditions include, without limitation, systemic lupus erythematosus (SLE), discoid lupus, lupus nephritis, sarcoidosis, inflammatory arthritis, including juvenile arthritis, rheumatoid arthritis, psoriatic arthritis, Reiter's syndrome, ankylosing spondylitis, and gouty arthritis, multiple sclerosis, hyper IgE syndrome, polyarteritis nodosa, primary biliary cirrhosis, inflammatory bowel disease, Crohn's disease, celiac's disease (gluten-sensitive enteropathy), autoimmune hepatitis, pernicious anemia, autoimmune hemolytic anemia, psoriasis, scleroderma, myasthenia gravis, autoimmune thrombocytopenic purpura, autoimmune thyroiditis, Grave's disease, Hasimoto's thyroiditis, immune complex disease, chronic fatigue immune dysfunction syndrome (CFIDS), polymyositis and dermatomyositis, cryoglobulinemia, thrombolysis, cardiomyopathy, pemphigus vulgaris, pulmonary interstitial fibrosis, asthma, Churg-Strauss syndrome (allergic granulomatosis), atopic dermatitis, allergic and irritant contact dermatitis, urtecaria, IgE-mediated allergy, atherosclerosis, vasculitis, idiopathic inflammatory myopathies, hemolytic disease, Alzheimer's disease, chronic inflammatory demyelinating polyneuropathy, chagas disease, chronic obstruct pulmonary disease, dermatomyositis, diabetes mellitus type 1, endometriosis, goodpasture's syndrome, graves' disease, guillain-barr\u00e9 syndrome (gbs), Hashimoto's disease, hidradenitis suppurat a, kawasaki disease, iga nephropathy, idiopathic thrombocytopenic purpura, interstitial cystitis, lupus erythematosus i, mixed connect e tissue disease, morphea, myasthenia gravis, narcolepsy, neuromyotonia, pemphigus vulgaris, pernicious anaemia, psoriasis, psoriatic arthritis, polymyositis, primary biliary cirrhosis, rheumatoid arthritis, schizophrenia, scleroderma, sj\u00f6gren's syndrome, stiff person syndrome, temporal arteritis, ulcerat e colitis, vasculitis, vitiligo, Wegener's granulomatosis, and AID.","Autoimmune: Biosignature","A biosignature of a vesicle can be assessed to provide a theranosis for a subject. The biosignature of the vesicle can comprise one or more biomarkers such as, but not limited to, a biomarker such as listed in  for autoimmune disease, or for other autoimmune diseases, such as, but not limited to those listed in .","Autoimmune: Standard of Care","Determining the biosignature of a vesicle, the amount of vesicles, or both, of a sample from a subject suffering from an autoimmune condition, disorder or disease can be used to select a standard of care for the subject. Most autoimmune diseases cannot yet be treated directly, but are treated according to symptoms associated with the condition. The standard of care includes, for example, prescribing corticosteroid drugs, non-steroidal anti-inflammatory drugs (NTHEs) or more powerful immunosuppressant drugs such as cyclophosphamide, methotrexate and azathioprine that suppress the immune response and stop the progression of the disease. Radiation of the lymph nodes and plasmapheresis (a procedure that removes the diseased cells and harmful molecules from the blood circulation) are other ways of treating an autoimmune disease.","Examples of drugs or agents for use in treating autoimmune diseases, which can be selected based on a profiling of a vesicle from a subject, include those in Table 13 for a subject suffering from diabetes, in Table 14 for those suffering from Multiple Sclerosis.",{"@attributes":{"id":"p-1034","num":"1036"},"tables":{"@attributes":{"id":"TABLE-US-00013","num":"00013"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"357pt","align":"center"}},"thead":{"row":{"entry":"TABLE 13"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Example of Classes of Drugs for Treatment of Diabetes"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Class","Mechanism of Action","Examples"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Peroxisome","Target PPAR-gamma or PPAR-gamma and -alpha (see below).","Rosiglitazone, Pioglitazone,"]},{"entry":["Proliferator-","PPAR are nuclear receptors that help regulate glucose and lipid","Balaglitazone, see also"]},{"entry":["Activated","metabolism. Activation of PPAR-gamma improves insulin","others described herein"]},{"entry":["Receptor","sensitivity and thus improves glycemic control."]},{"entry":"(PPAR)"},{"entry":"Agonists"},{"entry":["Dual-Action","Act on both PPAR-gamma and PPAR-alpha. PPAR-alpha","TAK-559, Muraglitazar,"]},{"entry":["Peroxisome","activation has effects on cellular uptake of fatty acids and their","Tesaglitazar, Netoglitazone,"]},{"entry":["Proliferator-","oxidation, and on lipoprotein metabolism. May also act to reduce","see also others described"]},{"entry":["Activated","inflammatory response in vascular endothelial cells.","herein"]},{"entry":"Receptor"},{"entry":"Agonists"},{"entry":["Biguanidines","Complete mechanism is not known. Reduces gluconeogenesis in","Metformin, Metformin GR"]},{"entry":[{},"the liver by inhibiting glucose-6-phosphatase."]},{"entry":["Sulfonylureas","Induce insulin secretion by binding to cellular receptors that","Glimepride,"]},{"entry":[{},"cause membrane depolarization and insulin exocytosis.","Glyburide\/glibenclamide,"]},{"entry":[{},{},"Glipizide, Gliclazide."]},{"entry":[{},{},"Tobutamide"]},{"entry":["Insulin and","Supplements endogenous insulin. Insulin analogs have a variety","Insulin lispro, Insulin aspart,"]},{"entry":["Insulin","of amino acid changes and have altered onset of action and","Insulin glargine, Exubera,"]},{"entry":["Analogs","duration of action, as well as other properties, compared to native","AERx Insulin Diabetes"]},{"entry":["(Injectable,","insulin. Inhaled insulin is absorbed through the alveoli. Spray oral","Management System, HIM-"]},{"entry":["Inhaled, Oral,","insulin is absorbed by the buccal mucosa and intranasal through","2, Oaralin, Insulin detemir,"]},{"entry":["Transdermal,","the nasal mucosa. Transdermal insulin is absorbed through the","Insulin glulisine"]},{"entry":["Intranasal)","skin."]},{"entry":["Meglitinides","Are thought to bind to a nonsulfonylurea beta cell receptor and","Repaglinide, Nateglinide,"]},{"entry":[{},"act to cause insulin secretion by mechanism similar to","Mitiglinide"]},{"entry":[{},"sulfonylureas"]},{"entry":["Alpha-","Inhibit carbohydrate digestion. Act at brush border of intestinal","Acarbose, Miglitol,"]},{"entry":["Glucosidase","epithelium.","Voglibose"]},{"entry":"Inhibitors"},{"entry":["Glucagon-","Diabetic patients may lack native GLP-1, and anlalogs act as","Exenatide, Exenatide LAR,"]},{"entry":["Like","substitutes. GLP-1 is an intestinal peptide hormone that induces","Liraglutide, ZP 10,"]},{"entry":["Peptide(GLP)-","glucose-dependent insulin secretion, controls gastric emptying,","BN51077,"]},{"entry":["1 Analogs","inhibits appetite, and modulates secretion of glucagon and"]},{"entry":[{},"somatostatin."]},{"entry":["Dipeptidyl","Inhibit DPP-IV, a ubiquitous enzyme that cleaves and inactivates","LAF-237, p-32\/98, MK-"]},{"entry":["Peptidase","GLP-1, thus inhibition of DPP-IV increases GLP-1 activity","431, P3298, NVP LAF 237,"]},{"entry":"(DPP)-IV"},{"entry":"Inhibitors"},{"entry":["Pancreatic","Inhibits lipases, thus inhibiting uptake of dietary fat. This causes","Orlistat"]},{"entry":["Lipase","weight loss, improves insulin sensitivity and lowers"]},{"entry":["Inhibitors","hyperglycemia."]},{"entry":["Amylin","Act to augment amylin, which acts with insulin by slowing","Pramlintide"]},{"entry":["Analogs","glucose absorption from the gut and slows after-meal glucose"]},{"entry":[{},"release from liver."]},{"entry":["Dopamine","Thought to act to alleviate abnormal daily variations in central","Bromocriptine"]},{"entry":["D2 receptor","neuroendocrine activity that can contribute to metabolic and"]},{"entry":["agonists","immune system disordered."]},{"entry":["Immunosuppressants","Suppress autoimmune response thought to be implicated in","Daclizumab, NBI 6024,"]},{"entry":[{},"Type I and possibly Type II diabetes. Example: Humanized","TRX-TolerRx, OKT3-"]},{"entry":[{},"monoclonal antibody that recognizes and inhibits the alpha","gamma-1-ala-ala"]},{"entry":[{},"subunit of IL-2 receptors; humanized Mab that binds to T cell"]},{"entry":[{},"CD3 receptor to block function of T-effector cells that attack the"]},{"entry":[{},"body and cause autoimmune disease"]},{"entry":["Insulin-like","Recombinant protein complex of insulin-like growth factor-1 and","Somatomedin-1 binding"]},{"entry":["growth","binding protein-3; regulates the delivery of somatomedin to target","protein 3"]},{"entry":["factor-1","tissues. Reduces insulitis severity and beta cell destruction"]},{"entry":"agonists"},{"entry":["Insulin","Insulin sensitizers, generally orally active","S15261, Dexlipotam, CLX"]},{"entry":["sensitizers",{},"0901, R 483, TAK 654"]},{"entry":["Growth","Mimic the action of native GHRF","TH9507, SOM 230"]},{"entry":"hormone"},{"entry":"releasing"},{"entry":"factor"},{"entry":"agonists"},{"entry":["Glucagon","Inhibit glucagon action, stimulating insulin production and","Liraglutide, NN 2501"]},{"entry":["antagonists","secretion, resulting in lower postprandial glucose levels"]},{"entry":["Diabetes type","Prevents destruction of pancreatic beta cells that occurs in type 1","Q-Vax, Damyd vaccine"]},{"entry":["1 vaccine","diabetes"]},{"entry":["Sodium-","Selectively inhibits the sodium glucose co-transporter, which","T 1095"]},{"entry":["glucose co-","mediates renal reabsorption and intestinal absorption of glucose"]},{"entry":["transporter","to maintain appropriate blood glucose levels."]},{"entry":"inhibitor"},{"entry":["Glycogen","Inhibit glycogen phosphorylase, thus slowing release of glucose","Ingliforib"]},{"entry":"phosphorylase"},{"entry":"inhibitors"},{"entry":["Undefined","Drugs that act in ways beneficial to those with Type I or Type II","FK 614, INGAP Peptide, R"]},{"entry":["mechanisms","Diabetes Mellitus, e.g., by reducing blood glucose and","1439"]},{"entry":[{},"triglyceride levels, whose mechanisms have not been elucidated."]},{"entry":["Antisense","Bind to RNA and cause its destruction, thereby decreasing","ISIS 113715"]},{"entry":["oligonucleotides","protein production from corresponding gene."]},{"entry":["Insulinotropin","Stimulate insulin release","CJC 1131"]},{"entry":"agonists"},{"entry":["Gluconeogenesis","Inhibit gluconeogenesis, thus modulating blood glucose levels","CS 917"]},{"entry":"inhibitors"},{"entry":["Hydroxysteroid","Inhibit hydroxysteroid dehydrogenase, which are responsible for","BVT 3498"]},{"entry":["dehydrogenase","excess glucocorticoid production and hence, visceral obesity"]},{"entry":"inhibitors"},{"entry":["Beta 3","Agonist for beta 3 adrenoceptor, decreases blood glucose and","YM 178, Solabegron,"]},{"entry":["adrenoceptor","suppresses weight gain","N5984,"]},{"entry":"agonist"},{"entry":["Nitric oxide","Decreases effects of NO","NOX 700"]},{"entry":"antagonist"},{"entry":["Carnitine","Inhibits carnitine palmitoyltransferase","ST 1326"]},{"entry":"palmitoyltransferase"},{"entry":"inhibitor"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-1035","num":"1037"},"tables":{"@attributes":{"id":"TABLE-US-00014","num":"00014"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"357pt","align":"center"}},"thead":{"row":{"entry":"TABLE 14"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Classes of Drugs for Treatment of Multiple Sclerosis"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Class","Mechanism of Action","Examples"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Recombinant","IFN-beta has numerous effects on the immune system. Exact","Interferon-beta-1b,"]},{"entry":["interferons","mechanism of action in MS not known","Interferon-beta-1a"]},{"entry":["Altered","Ligands either templated on sequence of myelin basic protein, or","Glatiramer acetate, MBP-"]},{"entry":["peptide","containing randomly arranged amino acids (e.g., ala, lys, glu, tyr)","8298, Tiplimotide, AG-284"]},{"entry":["ligands","whose structure resembles myelin basic protein, which is thought"]},{"entry":[{},"to be an antigen that plays a role in MS. Bind to the T-cell"]},{"entry":[{},"receptor but do not activate the T-cell because are not presented"]},{"entry":[{},"by an antigen-presenting cell."]},{"entry":["Chemotherapeutic","Immunosuppressive. MS is thought to be an autoimmune disease,","Mitoxantrone,"]},{"entry":["agents","so chemotherapeutics that suppress immunity improve MS","Methotrexate,"]},{"entry":[{},{},"Cyclophosphamide"]},{"entry":["Immunosuppressants","Act via a variety of mechanisms to dampen immune response.","Azathioprine,"]},{"entry":[{},{},"Teriflunomide, Oral"]},{"entry":[{},{},"Cladribine"]},{"entry":["Corticosteroids","Induce T-cell death and may up-regulate expression of adhesion","Methylprednisolone"]},{"entry":[{},"molecules in endothelial cells lining the walls of cerebral vessels,"]},{"entry":[{},"as well as decreasing CNS inflammation."]},{"entry":["Monoclonal","Bind to specific targets in the autoimmune cascade that produces","Natalizumab, Daclizumab,"]},{"entry":["Antibodies","MS, e.g., bind to activated T-cells","Altemtuzumab, BMS-"]},{"entry":[{},{},"188667, E-6040, Rituximab,"]},{"entry":[{},{},"M1 MAbs, ABT 874, T-"]},{"entry":[{},{},"0047"]},{"entry":["Chemokine","Prevent chemokines from binding to specific chemokine","BX-471, MLN-3897, MLN-"]},{"entry":["Receptor","receptors involved in the attraction of immune cells into the CNS","1202"]},{"entry":["Antagonists","of multiple sclerosis patients, and inhibiting immune cell"]},{"entry":[{},"migration into the CNS"]},{"entry":["AMPA","AMPA receptors bind glutamate, an excitatory neurotransmitter,","E-2007"]},{"entry":["Receptor","which is released in excessive quantities in MS. AMPA"]},{"entry":["Antagonists","antagonists suppresses the damage caused by the glutamate"]},{"entry":["Recombinant","GGF is associated with the promotion and survival of","Recombinant Human GGF2"]},{"entry":["Human Glial","oligodendrocytes, which myelinate neurons of the CNS. rhGGF"]},{"entry":["Growth","may help myelinate oligodendrocytes and protect the myelin"]},{"entry":["Factor (GGF)","sheath."]},{"entry":["T-cell","Mimic the part of the receptor in T cells that attack myelin","NeuroVax"]},{"entry":["Receptor","sheath, which activates regulatory T cells to decrease pathogenic"]},{"entry":["Vaccine","T-cells."]},{"entry":["Oral","Various effects on the immune response that can modulate the","Simvastatin, FTY-720, Oral"]},{"entry":["Immunomodulators","process of MS","Glatiramer Acetate, FTY-"]},{"entry":[{},{},"720, Pirfenidone,"]},{"entry":[{},{},"Laquinimod"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"In one embodiment, detection of miR-326 from a vesicle can be used to characterize multiple sclerosis, and one or more treatments selected from Table 14 can be selected for the subject. In another embodiment, the theranosis can include selecting a therapy such as interferon \u03b2-1b and interferon \u03b2-1a.","In another embodiment, a treatment can be selected for a subject suffering from Rheumatoid arthritis. One or more biomarkers, such as, but not limited to, 677CC\/1298AA MTHFR, 677CT\/1298AC MTHFR, 677CT MTHFR, G80AA RFC-1, 3435TT MDR1 (ABCB1), 3435TT ABCB1, AMPD1\/ATIC\/ITPA, IL1-RN3, HLA-DRB103, CRP, HLA-D4, HLA DRB-1, anti-citrulline epitope containing peptides, anti-A1\/RA33, Erythrocyte sedimentation rate (ESR), C-reactive protein (CRP), SAA (serum amyloid-associated protein), rheumatoid factor, IL-1, TNF, IL-6, IL-8, IL-1Ra, Hyaluronic acid, Aggrecan, Glc-Gal-PYD, osteoprotegerin, RNAKL, carilage oligomeric matrix protein (COMP), and calprotectin, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Methotrexate, infliximab, adalimumab'etanercept, sulfasalazine, or a combination thereof.","Thus, a treatment can be selected for the subject suffering from an autoimmune condition, based on the biosignature of the subject's vesicle","Infectious Diseases","Assessing a vesicle can be used in the theranosis of an infectious disease such as a bacterial, viral or other infectious condition or disease. An infectious or parasitic disease can arise from bacterial, viral, fungal, or other parasitic infection. For example, the disease or condition may be Whipple's Disease, Prion Disease, cirrhosis, methicillin-resistant , HIV, hepatitis, syphilis, meningitis, malaria, tuberculosis, or influenza.","An infectious or parasitic disease includes, but is not limited to, intestinal infectious diseases, tuberculosis, zoonotic bacterial diseases, other bacterial diseases, human immunodeficiency virus hiv infection, poliomyelitis and other non arthropod borne viral diseases of central nervous system, viral diseases accompanied by exanthem, arthropod borne viral diseases, other diseases due to viruses and chlamydiae, rickettsioses and other arthropod borne diseases, syphilis and other venereal diseases, other spirochetal diseases, mycoses, helminthiases, other infectious and parasitic diseases, and late effects of infectious and parasitic diseases. Intestinal infectious diseases include, but are not limited to cholera, typhoid and paratyphoid fevers, gastroenteritis, shigellosis, shigellosisunspecified, staphylococcal food poisoning, amoebiasis, acute amoebic dysentery without mention of abscess, chronic intestinal amoebiasis without mention of abscess, amoebic nondysenteric colitis, amoebic liver abscess, amoebic lung abscess, amoebic brain abscess, amoebic skin ulceration, amoebic infection of other sites, unspecified amoebiasis, balantidiasis, giardiasis, coccidiosis, intestinal trichomoniasis, cryptosporidiosis, cyclosporiasisunspecifiedified protozoal intestinal disease, intestinal infections due to other organisms, enteritis due to rotavirus, enteritis due to other viral enteritis, intestinal infection due to other organism not elsewhere classified, ill defined intestinal infections, colitis enteritis and gastroenteritis of presumed infectious origin.","A human immunodeficiency virus infection includes, but is not limited to human immunodeficiency virus infection with specified conditions, human immunodeficiency virus infection causing other specified, and other human immunodeficiency virus infection.","A poliomyelitis and other non arthropod borne viral diseases of central nervous system include, but are not limited to acute poliomyelitis, slow virus infection of central nervous system, kuru, creutzfeld jakob disease, meningitis due to enterovirus, other enterovirus diseases of central nervous system, and other non arthropod borne viral diseases of central nervous system. Viral diseases accompanied by exanthem include, but are not limited to smallpox, cowpox and paravaccinia, chickenpox, herpes zoster, herpes simplex, genital herpes, herpetic gingivostomatitis, herpetic disease, uncomplicated, measles, rubella, other viral exanthemata, fifth disease, unspecified viral exanthems, roseola infantum, other human herpesvirus encephalitis, other human herpesvirus infections, other poxvirus infections, other orthopoxvirus infections, monkeypox, other parapoxvirus infections, bovine stomatitis, sealpox, yatapoxvirus infections, tanapox, yaba monkey tumor virus, other poxvirus infections, and unspecified poxvirus infections.","Arthropod borne viral diseases include, but are not limited to yellow fever, dengue fever, mosquito borne viral encephalitis, encephalitis, mosquitounspecified, tick borne viral encephalitis, viral encephalitis transmitted by other and unspecified arthropods, arthropod borne hemorrhagic fever, ebolaunspecified, other arthropod borne viral diseases, and unspecified west nile virus.","Other diseases due to viruses and chlamydiae include, but are not limited to viral hepatitis, hepatitis a with hepatic coma, hepatitis a without coma, hepatitis b with hepatic coma, hepatitis b without coma, acute, other specified viral hepatitis with mention of hepatic coma, other specified viral hepatitis without mention of hepatic coma, unspecified viral hepatitis c, viral hepatitis c without hepatic coma, viral hepatitis c with hepatic coma, hepatitis, viral, rabies, mumps, mumps, uncomplicated, ornithosis, specific diseases due to coxsackie virus, herpangina, hand, foot, mouth disease, mononucleosis, trachoma, other diseases of conjunctiva due to viruses and chlamydiae, other diseases due to viruses and chlamydiae, molluscum contagiosum, warts, all sites, condyloma acuminata, sweating fever, cat scratch disease, foot and mouth disease, cmv disease, viral infection in conditions classified elsewhere and of unspecified site, rhinovirus, hpv, and respiratory syncytial virus. Rickettsioses and other arthropod borne diseases include, but are not limited to louse borne epidemic typhus, other typhus, tick borne rickettsioses, rocky mountain spotted fever, other rickettsioses, malaria, leishmaniasis, trypa omiasis, relapsing fever, other arthropod borne diseases, other specified arthropod borne diseases, lyme disease, and babesiosis.","A viral host includes, but is not limited to Adenovirus, Astrovirus, Avian influenza virus, Coxsackievirus, Dengue virus, Ebola virus, Echovirus, Enteric adenovirus, Enterovirus, Hantaviruses, Hepatitis A virus, Hepatitis B virus, Hepatitis C virus, Hepatitis D virus, Hepatitis E virus, Herpes simplex virus (HSV), Human cytomegalovirus, Human immunodeficiency virus (HIV), Human papillomavirus (HPV), Influenza virus, Japanese encephalitis virus (JEV), Lassa virus, Marburg virus, Measles virus, Mumps virus, Norovirus, Parainfluenza virus, Poliovirus, Rabies virus, Respiratory syncytial virus, Rotavirus, Rubella virus, SARS coronavirus, Tick-borne encephalitis virus (TBEV), Variola virus, West Nile virus, and Yellow fever virus. A fungal host includes, but is not limited to . A parasitic host includes, but is not limited to , and ","A bacterial host includes, but is not limited to , and ","Zoonotic bacterial diseases includes, but is not limited to plague, bubonic plague, tularemia, anthrax, brucellosis, glanders, melioidosis, rat bite fever, listeriosis, infection, and pasteurellosis. Other bacterial diseases include, but are not limited to leprosy, diseases due to other mycobacteria, diphtheria, whooping cough, sore throat and scarlatina, strep throat, scarlet fever, erysipelas, meningococcal meningitis, tetanus, septicaemia, pneumococcal septicemia, septicemia, gram negativeunspecified, septicemia, and actinomycotic infections.","Tuberculosis includes, but is not limited to primary tuberculous infection, pulmonary tuberculosis, tuberculosis of meninges and central nervous system, tuberculosis of intestines, peritoneum, and mesenteric glands, tuberculosis of bones and joints, tuberculosis of vertebral column, pott's disease, tuberculosis of genitourinary system, tuberculosis of other organs, erythema nodosum with hypersensitivity reaction in tuberculosis, bazin disease, tuberculosis of peripheral lymph nodes, scrofula, and miliary tuberculosis.","Syphilis and other venereal diseases include, but are not limited to congenital syphilis, early syphilis, symptomatic, syphilis, primary, genital, early syphilis, latent, cardiovascular syphilis, neurosyphilis, other forms of late syphilis, with symptoms, late syphilis, latent, other and unspecified syphilis, gonococcal infections, gonorrhoea, acute, lower gu tract, gonococcal conjunctivitis, and nongonococcal urethritis. Other spirochetal diseases include, but are not limited to leptospirosis, Vincent's angina, yaws, and pinta. Mycoses include, but are not limited to dermatophytosis, dermatophytosis of scalp\/beard, onychomycosis, dermatophytosis of hand, tinea cruris, tinea pedis, tinea corporis, dermatomycosis, other and unspecified, tinea versicolor, dermatomycosisunspecified, candidiasis, moniliasis, oral, moniliasis, vulva\/vagina, monilial balanitis, moniliasis, skin\/nails, coccidioidomycosis, histoplasmosis, infectionunspecified, blastomycotic infection, other mycoses, and opportunistic mycoses.","Helminthiases include, but are not limited to schistosomiasis bilharziasis, other trematode infections, echinococcosis, other cestode infection, trichi is, filarial infection and dracontiasis, ancylostomiasis and necatoriasis, other intestinal helminthiases, , anisakiasis, strongyloidiasis, trichuriasis, enterobiasis, capillariasis, trichostrongyliasis, other and unspecified helminthiases, and unspecified intestinal parasitism. Other infectious and parasitic diseases include, but are not limited to toxoplasmosis, toxoplasmosisunspecified, trichomoniasis, urogenital trichomoniasis, trichomonal vaginitis, trichomoniasis, urethritis, pediculosis and phthirus infestation, pediculosis, head lice, pediculosis, body lice, pediculosis, pubic lice, pediculosisunspecified, acariasis, scabies, chiggers, sarcoidosis, ainhum, behcet's syndrome, pneumocystosis, psorospermiasis, and sarcosporidiosis. Late effects of infectious and parasitic diseases include, but are not limited to late effects of tuberculosis, and late effects of polio.","Infectious Disease: Biosignature","A biosignature of a vesicle can be assessed to provide a theranosis for a subject. The biosignature of the vesicle can comprise one or more biomarkers such as, but not limited to, any one or more biomarkers as described herein, such as, but not limited to, those listed in  for infection diseases, and .","In some embodiments, an infectious disease can be characterized by detecting a component of a pathogen, such as a virus, bacteria, or other infectious agent, in a vesicle. For example, the component can be ABC transporters (), ABC transporters (), AMA-1 (Apical membrane antigen 1), ATPase, Aac(6\u2032)-Aph(2\u2033) enzyme, Ace (Accessory cholera enterotoxin), Acf (Accessory colonization factor), Acr (\u03b1-crystallin) protein, AhpC and AhpD, Amyloid-\u03b2, AroC, Attachment glycoprotein (G) (Respiratory syncytial virus), Autolysin (N-acetylmuramoyl-L-alanine amidase), BacA, BmpA (P39), Botulinum neurotoxins, BvgA, -S, and -R, BvrR-BvrS, C4BP(C4b-binding protein), C5a peptidase, CAMP factor (cohemolysin), CBP (Choline binding protein), CME type \u03b2-lactamase, CSP (Circumsporozoite protein), CT (cholera toxin), CTX-M metallo-\u03b2-lactamase, CagA (cytotoxin-associated antigen), Capsid protein (C) (Dengue virus), Capsid protein (C) (Japanese encephalitis virus), Capsid protein (C) (Tick-borne encephalitis virus), Capsid protein (C) (West Nile virus), Capsid protein (C) (Yellow fever virus), Capsid protein (Astrovirus), Capsid protein (Coxsackievirus), Capsid protein (Echovirus), Capsid protein (Enterovirus), Capsid protein (Hepatitis A virus), Capsid protein (Poliovirus), Capsid protein (Rotavirus), Catechol siderophore ABC transporter, Com-1, CrmB (Cytokine response modifier), Cytolysin, D-Ala-D-Lac ligase, DHFR (Dihydrofolate reductase), DHPS (Dihydropteroate synthetase), DbpA (Decorin-binding protein A), Diphtheria toxin, Dot\/Icm complex, E1 and E2 proteins (Rubella virus), E1A protein (Adenovirus), E1A protein (Enteric adenovirus), E1B protein (Adenovirus), E1B protein (Enteric adenovirus), E2 early transcription region 2, E3 protein (Adenovirus), E4 protein (Adenovirus), E6 early transcription region 6, E7 early transcription region 7, EF (Edema factor), ESAT-6 and CFP-10, Elastase (), Env, Envelope glycoprotein (E) (Dengue virus), Envelope glycoprotein (E) (Japanese encephalitis virus), Envelope glycoprotein (E) (Tick-borne encephalitis virus), Envelope glycoprotein (E) (West Nile virus), Envelope glycoprotein (E) (Yellow fever virus), Esp (Enterococcal surface protein), Esp (Type III System-Secreted Proteins), F1 capsule (F1 antigen), FH (Factor H), FHA (Filamentous hemagglutinin), Falcipain 1\/2, Fiber protein (Adenovirus), Fiber protein (Enteric adenovirus), Fibronectin binding protein II (Protein F\/sfbII) (), Fibronectin binding protein (), Fibronetin binding protein (FBP54) (), Fimbrial protein, Flagellin (FlaB and -A) (), Flagellin (H-antigen) (), Flagellin (H-antigen) (), Flagellin (), FopA (43 kDa lipoprotein), Fusion protein (F) (Mumps virus), Fusion protein (F) (Parainfluenza virus), Fusion protein (F) (Respiratory syncytial virus), G6PD (Glucose-6-phosphate dehydrogenase), GES (Guiana extended-spectrum \u03b2-lactamase), GTP cyclohydrolase, Gag, Glycoprotein (G) (Rabies virus), Glycoprotein (GP) (Ebola virus), Glycoprotein (GP) (Lassa virus), Glycoprotein (GP) (Marburg virus), Glycoproteins (Gn\/Gc) (Hantaviruses), HMW (Cytadherence accessory protein), HRP2 (Histidine-rich protein 2), Hemagglutinin (Avian influenza virus), Hemagglutinin (Influenza virus), Hemagglutinin (Measles virus), Hemagglutinin (Variola virus), Hemagglutinin-esterase glycoprotein (HE), Hemagglutinin-neuraminidase (HN) (Mumps virus), Hemagglutinin-neuramimidate (HN) (Paraninfluenza virus), Hemolysin (Vvh), Hexon protein (Adenovirus), Hexon protein (Enteric adenovirus), Hsp60 (Heat shock protein 60), Hyaluronate lyase, Hyaluronidase, IMP metallo-\u03b2-lactamase (), IMP metallo-\u03b2-lactamase (), IcsA and IcsB, IgA protease (), IgA1 protease (), IgG and IgM for HSV 1\/2, InhA, Intimin, InvA (), Invasin (), Invasin (), IpaA, -B, -C, -D and -H, KPC metallo-\u03b2-lactamase, KatG, L protein (Lassa virus), L1 late transcription region 1, LF (Lethal factor), LSA1 (Liver-stage antigen 1), LT (heat labile toxin), LcrV (V antigen), LigA and LigB, Lipoprotein, M protein, MSP (Merozoite surface protein), Matrix protein (M) (Rabies virus), Matrix protein (M) (Respiratory syncytial virus), Matrix protein (Avian influenza virus), Matrix protein (Influenza virus), MexAB-OprM, MexCD-OprJ, MexEF-OprN, MexXY-OprM, Mip (Macrophage infectivity potentiator), NSE (Neuron-specific enolase), Nef, Neuraminidase (Avian influenza virus), Neuraminidase (Influenza virus), Neuraminidase (), Non-structural protein (NS) (Respiratory syncytial virus), Non-structural protein 1 (NS1) (Dengue virus), Non-structural protein 1 (NS1) (Japanese encephalitis virus), Non-structural protein 1 (NS1) (Tick-borne encephalitis virus), Non-structural protein 1 (NS1) (West Nile virus), Non-structural protein 1 (NS1) (Yellow fever virus), Non-structural protein 2A (NS2A) (Dengue virus), Non-structural protein 2A (NS2A) (Japanese encephalitis virus), Non-structural protein 2A (NS2A) (Tick-borne encephalitis virus), Non-structural protein 2A (NS2A) (West Nile virus), Non-structural protein 2A (NS2A) (Yellow fever virus), Non-structural protein 2B (NS2B) (Dengue virus), Non-structural protein 2B (NS2B) (Japanese encephalitis virus), Non-structural protein 2B (NS2B) (Tick-borne encephalitis virus), Non-structural protein 2B (NS2B) (West Nile virus), Non-structural protein 2B (NS2B) (Yellow fever virus), Non-structural protein 3 (NS3) (Dengue virus), Non-structural protein 3 (NS3) (Japanese encephalitis virus), Non-structural protein 3 (NS3) (Tick-borne encephalitis virus), Non-structural protein 3 (NS3) (West Nile virus), Non-structural protein 3 (NS3) (Yellow fever virus), Non-structural protein 4 (Rotavirus), Non-structural protein 4A (NS4A) (Dengue virus), Non-structural protein 4A (NS4A) (Japanese encephalitis virus), Non-structural protein 4A (NS4A) (Tick-borne encephalitis virus), Non-structural protein 4A (NS4A) (West Nile virus), Non-structural protein 4A (NS4A) (Yellow fever virus), Non-structural protein 4B (NS4B) (Dengue virus), Non-structural protein 4B (NS4B) (Japanese encephalitis virus), Non-structural protein 4B (NS4B) (Tick-borne encephalitis virus), Non-structural protein 4B (NS4B) (West Nile virus), Non-structural protein 4B (NS4B) (Yellow fever virus), Non-structural protein 5 (NS5) (Dengue virus), Non-structural protein 5 (NS5) (Japanese encephalitis virus), Non-structural protein 5 (NS5) (Tick-borne encephalitis virus), Non-structural protein 5 (NS5) (West Nile virus), Non-structural protein 5 (NS5) (Yellow fever virus), Non-structural proteins (Avian influenza virus), Non-structural proteins (Influenza virus), Nucleocapsid (Hantaviruses), Nucleocapsid (Measles virus), Nucleocapsid (Parainfluenza virus), Nucleocapsid (SARS coronavirus), Nucleoprotein (N) (Rabies virus), Nucleoprotein (NP) (Respiratory syncytial virus), Nucleoprotein (major nucleoprotein) (Marburg virus), Nucleoprotein (Avian influenza virus), Nucleoprotein (Ebola virus), Nucleoprotein (Influenza virus), Nucleoprotein (Lassa virus), ORF 1 (Hepatitis E virus), ORF2 (Hepatitis E virus), ORF3 (Hepatitis E virus), OXA metallo-\u03b2-lactamase (), OXA metallo-\u03b2-lactamase (), OmpA and OmpB (), OmpL1 (), OmpQ (Outer membrane porin protein) (), OmpS (), Opacity factor, OprD, Osp (Outer surface protein), Outer membrane proteins (), Outer membrane proteins (), P1 adhesin, P30 adhesin, PA (Protective antigen), PBP (Penicillin-binding protein), PCRMP 1-4 (Cysteine repeat modular proteins), PER metallo-\u03b2-lactamase, Pat1, Peptidoglycan (murein) hydrolase, Pertactin (p69), Pertussis toxin, PfEMP1 (erythrocyte membrane protein-1), Phosphoprotein (P) (Respiratory syncytial virus), Phosphoprotein (Measles virus), Pla (plasminogen activator), Plasminogen-binding protein, Pld, Pneumolysin, Pol, Poly-D-glutamic acid capsule, Polymerase (L) (Rabies virus), Porin, Premembrane\/membrane protein (PrM\/M) (Dengue virus), Premembrane\/membrane protein (PrM\/M) (Japanese encephalitis virus), Premembrane\/membrane protein (PrM\/M) (Tick-borne encephalitis virus), Premembrane\/membrane protein (PrM\/M) (West Nile virus), Premembrane\/membrane protein (PrM\/M) (Yellow fever virus), Proteins for two-component regulatory systems (), Proteins for two-component regulatory systems (), Proteins of gB, gC, gD, gH, and gL, PsaA, PspA (Pneumococcal surface protein A), PurE, Pyrogenic exotoxins, RBP 1\/2 (Reticulocyte binding protein 1\/2), RdRp (RNA dependant RNA polymerase) (Norovirus), RdRp (RNA dependent RNA polymerase) (Astrovirus), RdRp (RNA dependent RNA polymerase) (SARS coronavirus), Rev, RfbE, RibD and RibE, Rmp, S-layer protein, S100B (S100 protein \u03b2 chain), SHV metallo-\u03b2-lactamase, SIM metallo-\u03b2-lactamase, ST (heat stable toxin), plasmid virulence (SPV) proteins, Serine protease (Astrovirus), ShET1\/2, Shiga toxin (Verotoxin), SipA (Invasion Protein A), SlyA, Small hydrophobic protein, Sop (outer protein), Spike glycoprotein (S), DNase, Streptogramin A acetyltransferase, Streptokinase, Streptolysin O, StxA\/B (Shiga toxin A\/B), SucB (Dihydrolipoamide succinyltransferase) (), SucB (dihydrolipoamide succinyltransferase) (), Syc (Yop chaperones), T protein, TCP (toxin-coregulated pilus), TEM metallo-\u03b2-lactamase, TRAP (Thrombospondin-related anonymous protein), Tat, Tau-protein, TcfA (Tracheal colonization factor), Tir (Translocated intimin receptor), TlyA and TlyC, ToxR (toxin regulatory protein), Tul4 (17 kDa lipoprotein), Type IV pili, Urease (Brucella), Urease (), VEB metallo-\u03b2-lactamase, VETF (Virus early transcription factor), VIM metallo-\u03b2-lactamase (), VIM metallo-\u03b2-lactamase (), VP1 (Norovirus), VP2 (Norovirus), VP24 (Ebola virus), VP24 (Marburg virus), VP30 (minor nucleoprotein) (Ebola virus), VP30 (minor nucleoprotein) (Marburg virus), VP35 (P-like protein) (Ebola virus), VP35 (P-like protein) (Marburg virus), VP40 (Matrix Protein) (Ebola virus), VP40 (Matrix Protein) (Maburg virus), VacA (vacuolating cytotoxin), Vag8 (virulence-activated gene 8), Vif, VirB type IV secretion system, VlsE (35 kDa lipoprotein), Vpr, Vpu\/Vpx, XerD, Yops (Yersinia outermembrane proteins), Ysc (Yop secretion apparatus), Z protein (Lassa virus), Zot (zonula occuldens toxin), gG1 (HSV-1) and gG2 (HSV-2), p41i, p83, and p100, pLDH (Plasmodium lactate dehydrogenase), \u03b1\/\u03b2\/\u03b3 proteins, 120 kDa gene, 16S and 5S rRNA genes (), 16S rRNA (), 16S rRNA (), 16S rRNA (Brucella), 16S rRNA (), 16S rRNA (), 16S rRNA (), 16S rRNA (), 16S rRNA gene (), 16S rRNA gene (), 16S rRNA gene (), 16S rRNA gene (), 16S rRNA gene (), 16S rRNA gene (), 16S-23S rRNA intergenic spacer (), 16S-23S rRNA intergenic spacer (), 17 kDa gene, 18S ssrRNA, 23S rRNA gene (), 23S rRNA gene (), 2C gene, 3\u2032 NCR (Dengue virus), 3\u2032 NCR (Japanese encephalitis virus), 3\u2032 NCR (Tick-borne encephalitis virus), 3\u2032 NCR (West Nile virus), 3\u2032 NCR (Yellow fever virus), 5\u2032 NCR (Coxsackievirus), 5\u2032 NCR (Dengue virus), 5\u2032 NCR (Echovirus), 5\u2032 NCR (Enterovirus), 5\u2032 NCR (Japanese encephalitis virus), 5\u2032 NCR (Polioviurs), 5\u2032 NCR (Tick-borne encephalitis virus), 5\u2032 NCR (West Nile virus), 5\u2032 NCR (Yellow fever virus), 56 kDa gene, A13L gene, ARE1 gene, ATF2 gene, B12R gene, B6R gene, B8R gene, C gene (Dengue virus), C gene (Japanese encephalitis virus), C gene (Tick-borne encephalitis virus), C gene (West Nile virus), C gene (Yellow fever virus), C3L gene, CDR 1\/2 genes, E gene (Dengue virus), E gene (Japanese encephalitis virus), E gene (Tick-borne encephalitis virus), E gene (West Nile virus), E gene (Yellow fever virus), E1 and E2 genes, E1A gene (Adenovirus), E1A gene (Enteric adenovirus), E1B gene (Adenovirus), E1B gene (Enteric adenovirus), E2 gene, E3 gene (Adenovirus), E3L gene, E4 gene (Adenovirus), E6 gene, E7 gene, ERG genes, ESAT-6 and CFP-10 genes, F gene (Mumps virus), F gene (Parainfluenza virus), F gene (Respiratory syncytial virus), G gene (Rabies virus), G gene (Respiratory syncytial virus), GP gene (Ebola virus), GP gene (Lassa virus), GP gene (Marburg virus), H gene (Measles virus), HA gene (Avian influenza virus), HA gene (Influenza virus), HE gene (SARS Coronavirus), HN gene (Mumps virus), HN gene (Parainfluenza virus), IS100, IS1081, IS1533 (Leptospira interrogans), IS285, IS481 (BP0023), IS6110, IS711 (Brucella), ISFtu, J7R gene, L gene (Lassa virus), L gene (Rabies virus), L segment, L1 gene, LEE (locus of enterocyte effacement), Long control region (LCR), M gene (Rabies virus), M gene (Respiratory syncytial virus), M genes (Avian influenza virus), M genes (Influenza virus), M segment, MDR1 gene, MEC3 gene, N gene (Measles virus), N gene (Rabies virus), N gene (SARS coronavirus), NA gene (Avian influenza virus), NA gene (Influenza virus), NC gene (Parainfluenza virus), NP gene (Avian influenza virus), NP gene (Ebola virus), NP gene (Influenza virus), NP gene (Lassa virus), NP gene (Marburg virus), NP gene (Respiratory syncytial virus), NS gene (Avian influenza virus), NS gene (Influenza virus), NS gene (Respiratory syncytial virus), NS1 gene (Dengue virus), NS1 gene (Japanese encephalitis virus), NS1 gene (Tick-borne encephalitis virus), NS1 gene (West Nile virus), NS1 gene (Yellow fever virus), NS2A gene (Dengue virus), NS2A gene (Japanese encephalitis virus), NS2A gene (Tick-borne encephalitis virus), NS2A gene (West Nile virus), NS2A gene (Yellow fever virus), NS2B gene (Dengue virus), NS2B gene (Japanese encephalitis virus), NS2B gene (Tick-borne encephalitis virus), NS2B gene (West Nile virus), NS2B gene (Yellow fever virus), NS3 gene (Dengue virus), NS3 gene (Japanese encephalitis virus), NS3 gene (Tick-borne encephalitis virus), NS3 gene (West Nile virus), NS3 gene (Yellow fever virus), NS4 gene (Rotavirus), NS4A gene (Dengue virus), NS4A gene (Japanese encephalitis virus), NS4A gene (Tick-borne encephalitis virus), NS4A gene (West Nile virus), NS4A gene (Yellow fever virus), NS4B gene (Dengue virus), NS4B gene (Japanese encephalitis virus), NS4B gene (Tick-borne encephalitis virus), NS4B gene (West Nile virus), NS4B gene (Yellow fever virus), NS5 gene (Dengue virus), NS5 gene (Japanese encephalitis virus), NS5 gene (Tick-borne encephalitis virus), NS5 gene (West Nile virus), NS5 gene (Yellow fever virus), ORF 1a (Astrovirus), ORF1b (Astrovirus), ORF 2 (Astrovirus), ORF1 (Hepatitis E virus), ORF1 (Norovirus), ORF2 (Hepatitis E virus), ORF2 (Norovirus), ORF3 (Hepatitis E virus), ORF3 (Norovirus), P gene (Measles virus), P gene (Respiratory syncytial virus), PDH1 gene, Peptidyltransferase mutations, Plasmids (QpH1, QpRS, QpDG, QpDV), PrM\/M gene (Dengue virus), PrM\/M gene (Japanese encephalitis virus), PrM\/M gene (Tick-borne encephalitis virus), PrM\/M gene (West Nile virus), PrM\/M gene (Yellow fever virus), RdRp gene in ORF 1ab (SARS coronavirus), S gene (SARS coronavirus), S segment, SH gene (Mumps virus), SNP (single nucleotide polymorphism), pathogenicity island (SPI), plasmid virulence (SPV) operon, ShET1\/2 genes, VNTR (variable number tandem repeat) (), VNTR (variable number tandem repeat) (Brucella), VNTR (variable number tandem repeat) (), VNTR (variable number tandem repeat) (), VP24 gene (Ebola virus), VP24 gene (Marburg virus), VP30 gene (Ebola virus), VP30 gene (Marburg virus), VP35 gene (Ebola virus), VP35 gene (Marburg virus), VP40 gene (Ebola virus), VP40 gene (Marburg virus), Z gene (Lassa virus), aac(3) gene, aac(6\u2032) gene, aac(6\u2032)-aph(2\u2033) gene, aad gene, ace gene, acpA gene, agrBDCA locus, ahpC and ahpD genes, arlRS locus, atxA gene, bclA gene, blaCTX-M gene, blaGES gene, blaGIM gene (), blaIMP gene (), blaIMP gene (), blaIMP gene (), blaKPC gene, blaOXA gene (), blaOXA gene (), blaOXA gene (), blaSHV gene, blaSIM gene (), blaSIM gene (), blaTEM gene, blaVIM gene (), blaVIM gene (), blaVIM gene (), bvg locus (bvgA, -S, and -R genes), cagA gene, cap locus (capB, -C, and -A genes) (), cap operon (capB and -C) (), capsid gene (Coxsackievirus), capsid gene (Echovirus), capsid gene (Enterovirus), capsid gene (Hepatitis A virus), capsid gene (Poliovirus), capsid gene (Rotavirus), cme gene, cnt genes, com-1 gene, cppB gene, cps gene, crmB gene, ctx gene, cya gene, cyl gene, eaeA gene, east gene (), env gene, ery gene, esp gene (), esp genes (), fiber gene (Adenovirus), fiber gene (Enteric adenovirus), fimbriae genes, flaB gene (), flaB gene (), flagellin genes, fljA, fljB, and fliC genes, fopA gene, ftsZ gene, gG1 and gG2 gens, gag gene, genes for two-component regulatory systems, genes of gB, gC, gD, gH, and gL, gerX locus (gerXC, -A, and -B genes), glpQ gene, gltA (citrate synthase) gene (), gltA (citrate synthase) gene (), groEL gene (), groEL gene (), groESL gene (), gyrA and gyrB genes (), gyrA gene (), gyrB gene (), hexon gene (Adenovirus), hexon gene (Enteric adenovirus), hin gene, hlyA gene, hmw genes, hspX (Rv2031c) gene, htpAB associated repetitive element (IS1111a), hyl gene, icsA and icsB genes, ileS gene, inhA gene, inv gene (), inv gene (), ipaA, -B, -C, -D and -H genes, katG gene, lef gene, letA gene, lidA gene, lpsB gene, lrgAB locus, luxS gene, lytA gene, lytRS locus, mecA gene, mglA gene, mgrA (rat) gene, mip gene, mtgA gene, mucZ gene, multigene families, mupA gene, nanA and nanB genes, nef gene, omp genes (Brucella), omp genes (), ompA and B gene (), ompQ gene, opa genes, osp genes, p1 gene, p30 gene, pagA gene, pap31 gene, parC and parE genes (), parC gene (), per gene, pilQ gene, ply gene, pmm gene, pol gene, porA and porB genes, prn4 (pertactin) gene, psaA gene, pspA gene, pst1 fragment and HL-1\/HR-1 primers, ptx (promoter region and complete gene), rap 1\/2 genes, rev gene, rpo18 gene, rpoB gene, rpoS gene, rpsL gene, rrf(5S)-rrl(23S) intergenic spacer, rsk gene, rtx gene (), rtxA gene (), sap gene (), sar gene, satA (vatD) and satG (vatE) genes, sca4 gene, secY gene, stx (vt) gene, stxA\/B (stx1\/2) gene, sucB gene, tat gene, tcp gene, tir gene, tox gene, toxR gene, tul4 gene, urease genes, vacA gene, van A-E genes, veb gene, vif gene, viuB gene, vpr gene, vpu\/vpx gene, vvh (Vibrio vulnificus hemolysin) gene, vvpE (Vibrio vulnificus elastase) gene, wboA gene, wzy (O-antigen polymerase) gene, zot gene, \u03b1\/\u03b2\/\u03b3 genes, C-polysaccharide (rhamnose\/N-acetylglucosamine), CPS (capsular polysaccharide), Cyclic \u03b2-1,2 glucan, Hyaluronic acid capsule, LPS (lipopolysaccharide) (), LPS (lipopolysaccharide) (Brucella), LPS (lipopolysaccharide) (), LPS (lipopolysaccharide) (), LPS (lipopolysaccharide) (Vibrio vulnificus), O-antigen (), O-antigen (), O-antigen (), Vi-antigen (), or Catechol siderophore.","Infectious Disease: Standards of Care","Determining the biosignature of a vesicle, the amount of vesicles, or both, of a sample from a subject suffering from an infectious or parasitic disease, disorder or disease can be used to select a standard of care for the subject. An infectious or parasitic disease can be treated according to symptoms associated with the condition. The standard of care includes, for example, treating with one or more antibiotics and antiviral agents.","An antibiotic includes, but not limited to, Amikacin, Gentamicin, Kanamycin, Neomycin, Netilmicin, Streptomycin, Tobramycin, Paromomycin, Geldanamycin, Herbimycin, Loracarbef, Ertapenem, Doripenem, Imipenem\/Cilastatin, Kerosene, Cefadroxil, Cefazolin, Cefalotin or Cefalothin, Cefalexin, Cefaclor, Cefamandole, Cefoxitin, Cefprozil, Cefuroxime, Cefixime, Cefdinir, Cefditoren, Cefoperazone, Cefotaxime, Cefpodoxime, Ceftazidime, Ceftibuten, Ceftizoxime, Ceftriaxone, Cefepime, Ceftobiprole, Teicoplanin, Vancomycin, Azithromycin, Clarithromycin, Dirithromycin, Erythromycin, Roxithromycin, Troleandomycin, Telithromycin, Spectinomycin, Aztreonam, Amoxicillin, Ampicillin, Azlocillin, Carbenicillin, Cloxacillin, Dicloxacillin, Flucloxacillin, Mezlocillin, Meticillin, Nafcillin, Oxacillin, Penicillin, Piperacillin, Ticarcillin, Bacitracin, Colistin, Polymyxin B, Ciprofloxacin, Enoxacin, Gatifloxacin, Levofloxacin, Lomefloxacin, Moxifloxacin, Norfloxacin, Ofloxacin, Trovafloxacin, Grepafloxacin, Sparfloxacin, Temafloxacin, Mafenide, Sulfonamidochrysoidine, Sulfacetamide, Sulfanilimide, Sulfasalazine, Sulfisoxazole, Trimethoprim, Trimethoprim-, Sulfamethoxazole, Demeclocycline, Doxycycline, Minocycline, Oxytetracycline, Tetracycline, Sulfadiazine, Sulfamethizole, Arsphenamine, Chloramphenicol, Clindamycin, Lincomycin, Ethambutol, Fosfomycin, Fusidic acid, Furazolidone, Isoniazid, Linezolid, Metronidazole, Mupirocin, Nitrofurantoin, Platensimycin, Pyrazinamide, Quinupristin or Dalfopristin, Rifampicin, Thiamphenicol, Timidazole, Dapsone, and Clofazimine. Examples of antibiotics are also listed in Table 15.","An antiviral agent includes, but is not limited to Abacavir, Aciclovir, Acyclovir, Adefovir, Amantadine, Amprenavir, Ampligen, Arbidol, Atazanavir, Atripla, Boceprevir, Cidofovir, Combivir, Darunavir, Delavirdine, Didanosine, Docosanol, Edoxudine, Efavirenz, Emtricitabine, Enfuvirtide, Entecavir, Famciclovir, Fomivirsen, Fosamprenavir, Foscarnet, Fosfonet, Ganciclovir, Ibacitabine, Immunovir, Idoxuridine, Imiquimod, Indinavir, Inosine, Interferon type III, Interferon type II, Interferon type I, Lamivudine, Lopinavir, Loviride, Maraviroc, Moroxydine, Nelfinavir, Nevirapine, Nexavir, Oseltamivir, Peginterferon alfa-2a, Penciclovir, Peramivir, Pleconaril, Podophyllotoxin, Raltegravir, Ribavirin, Rimantadine, Ritonavir, Pyramidine, Saquinavir, Stavudine, Tea tree oil, Tenofovir, Tenofovir disoproxil, Tipranavir, Trifluridine, Trizivir, Tromantadine, Truvada, Valaciclovir, Valganciclovir, Vicriviroc, Vidarabine, Viramidine, Zalcitabine, Zanamivir, and Zidovudine.",{"@attributes":{"id":"p-1059","num":"1061"},"tables":{"@attributes":{"id":"TABLE-US-00015","num":"00015"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"301pt","align":"center"}},"thead":{"row":{"entry":"TABLE 15"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Examples of Antibiotic Drugs and their Structure Class"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Structure Class","Examples of Antibiotics within Structure Class"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Amino Acid Derivatives","Azaserine, Bestatin, Cycloserine, 6-diazo-5-oxo-L-norleucine"]},{"entry":["Aminoglycosides","Armastatin, Amikacin, Gentamicin, Hygromicin, Kanamycin, Streptomycin"]},{"entry":["Benzochinoides","Herbimycin"]},{"entry":["Carbapenems","Imipenem, Meropenem"]},{"entry":["Coumarin-glycosides","Novobiocin"]},{"entry":["Fatty Acid Derivatives","Cerulenin"]},{"entry":["Glucosamines","1-deoxynojirimycin"]},{"entry":["Glycopeptides","Bleomycin, Vancomycin"]},{"entry":["Imidazoles","Metroidazole"]},{"entry":["Penicillins","Benzylpenicillin, Benzathine penicillin, Amoxycillin, Piperacillin"]},{"entry":["Macrolides","Amphotericin B, Azithromycin, Erythromycin"]},{"entry":["Nucleosides","Cordycepin, Formycin A, Tubercidin"]},{"entry":["Peptides","Cyclosporin A, Echinomycin, Gramicidin"]},{"entry":["Peptidyl Nucleosides","Blasticidine, Nikkomycin"]},{"entry":["Phenicoles","Chloramphenicol, Thiamphenicol"]},{"entry":["Polyethers","Lasalocid A, Salinomycin"]},{"entry":["Quinolones","8-quinolinol, Cinoxacin, Ofloxacin"]},{"entry":["Steroids","Fusidic Acid"]},{"entry":["Sulphonamides","Sulfamethazine, Sulfadiazine, Trimethoprim"]},{"entry":["Tetracyclins","Oxytetracyclin, Minocycline, Duramycin"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}]}}},"In one embodiment, a subject has an HIV infection. One or more biomarkers, such as, but not limited to p24 antigen, TNF-alpha, TNFR-11, CD3, CD14, CD25, CD27, Fas, FasL, beta2 microglobulin, neopterin, HIV RNA, and HLA-B *5701, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Zidovudine, Didanosine, Zalcitabine, Stavudine, Lamivudine, Saquinavir, Ritonavir, Indinavir, Nevirane, Nelfinavir, Delavirdine, Stavudine, Efavirenz, Etravirine, Enfuvirtide, Darunavir, Abacavir, Amprenavir, Lonavir\/Ritonavirc, Tenofovir, Tipranavir, or a combination thereof.","Thus, a treatment can be selected for the subject suffering from an infectious disease or condition, based on the biosignature of the subject's vesicle.","Neurology","Assessing a vesicle can be used in the theranosis of a neurological disease, such as Multiple Sclerosis (MS), Parkinson's Disease (PD), Alzheimer's Disease (AD) (non-inflammatory and inflammatory), schizophrenia, bipolar disorder, depression, autism, Prion Disease, Pick's disease, dementia, Huntington disease (HD), Down's syndrome, cerebrovascular disease, Rasmussen's encephalitis, viral meningitis, neurospsychiatric systemic lupus erythematosus (NPSLE), amyotrophic lateral sclerosis, Creutzfeldt-Jacob disease, Gerstmann-Straussler-Scheinker disease, transmissible spongiform encephalopathy, ischemic reperfusion damage (e.g. stroke), brain trauma, microbial infection, or chronic fatigue syndrome.","A neurological disorder includes, but is not limited to inflammatory diseases of the central nervous system, hereditary and degenerative diseases of the central nervous system, pain, other headache syndromes, other disorders of the central nervous system, and disorders of the peripheral nervous system. Inflammatory diseases of the central nervous system include, but are not limited to bacterial meningitis, meningitis, , meningitis, bacterial, meningitis due to other organisms, cryptococcal meningitis, meningitis of unspecified cause, encephalitis, myelitis, and encephalomyelitis, postinfectious encephalitis, unspecified encephalitis, intracranial and intraspinal abscess, phlebitis and thrombophlebitis of intracranial venous sinuses, venous sinus thrombosis, intracranial, late effects of intracranial abscess or pyogenic infection, sleep disorders, unspecified organic insomnia, insomnia due to medical condition classified elsewhere, and insomnia due to mental disorder. Hereditary and degenerative diseases of the central nervous system include, but are not limited to cerebral degenerations usually manifest in childhood, leukodystrophy, krabbe disease, pelizaeus merzbacher disease, cerebral lipidoses, tay sacks disease, other cerebral degenerations, alzheimer's, pick's disease, senile degeneration of brain, communicating hydrocephalus, obstructive hydrocephalus, idiopathic normal pressure hydrocephalus, other cerebral degeneration, reye's syndrome, dementia with lewy bodies, mild cognitive impairment, so stated, Parkinson's Disease, parkinsonism, primary, other extrapyramidal disease and abnormal movement disorders, other degenerative diseases of the basal ganglia, olivopontocerebellar atrophy, shydrager syndrome, essential tremor\/familial tremor, myoclonus, lafora's disease, unverricht disease, Huntington's chorea, fragments of torsion dystonia, blepharospasm, other and unspecified extrapyramidal diseases and abnormal movement disorders, other extrapyramidal diseases and abnormal movement disorders, restless legs, serotonin syndrome, spinocerebellar disease, friedreich's ataxia, spinocerebellar ataxia, hereditary spastic paraplegia, primary cerebellar degeneration, other cerebellar ataxia, cerebellar ataxia in diseases classified elsewhere, other spinocerebellar diseases, ataxia telangiectasia, corticostriatal spinal degeneration, unspecified spinocerebellar disease, anterior horn cell disease, motor neuron disease, amyotrophic lateral sclerosis, progressive muscular atrophy, progressive bulbar palsy, pseudobulbar palsy, primary lateral sclerosis, other motor neuron diseases, other diseases of spinal cord, syringomyelia and syringobulbia, disorders of the autonomic nervous system, idiopathic peripheral autonomic neuropathy, unspecified idiopathic peripheral autonomic neuropathy, carotid sinus syndrome, other idiopathic peripheral autonomic neuropathy, peripheral autonomic neuropathy in disorders classified elsewhere, reflex sympathetic dystrophy, autonomic dysreflexia, and unspecified disorder of autonomic nervous system.","Pain includes, but is not limited to, central pain syndrome, acute pain, chronic pain, neoplasm related pain acute chronic and chronic pain syndrome. Other headache syndromes include, but are not limited to cluster headaches and other trigeminal autonomic cephalgias, unspecified cluster headache syndrome, episodic cluster headache, chronic cluster headache, episodic paroxysmal hemicrania, chronic paroxysmal hemicrania, short lasting unilateral neuralgiform headache with conjunctival injection and tearing, other trigeminal autonomic cephalgias, tension type headache, unspecified tension type headache, episodic tension type headache, chronic tension type headache, post traumatic headache, unspecified post traumatic headache, acute post traumatic headache, chronic post traumatic headache, drug induced headache, not elsewhere classified, complicated headache syndromes, hemicrania continua, new daily persistent headache, primary thunderclap headache, other complicated headache syndrome, other specified headache syndromes, hypnic headache, headache associated with sexual activity, primary cough headache, primary exertional headache, and primary stabbing headache.","Other disorders of the central nervous system include, but are not limited to multiple sclerosis, other demyelinating diseases of central nervous system, neuromyelitis optica, schilder's disease, acute myelitis transverse myelitis, hemiplegia, hemiplegia, flaccid, hemiplegia, spastic, infantile cerebral palsy, cerebral palsy, paraplegic, congenital, cerebral palsy, hemiplegic, congenital, cerebral palsy, quadriplegic, other paralytic syndromes, quadraplegia and quadraparesis, paraplegia, diplegia of upper limbs, monoplegia of lower limb, monoplegia of upper limb, unspecified monoplegia, cauda equina syndrome, other specified paralytic syndromes, locked in state, epilepsy, intractable epilepsy, tonic clonic epilepsy without status, epilepsy with status, epilepsy on temporal lobe without status, unspecified epilepsy without status, migraine, classical not intractable migraine, common but not intractable migraine, not intractable cluster headache, unspecified but, not intractable migraine, cataplexy and narcolepsy, narcolepsy without cataplexy, cerebral cysts, anoxic brain damage, pseudotumor cerebri, unspecified encephalopathy, metabolic encephalopathy, compression of brain, cerebral edema, post spinal puncture, post dural puncture headache, cerebrospinal fluid rhinorrhea, and toxic encephalopathy.","Disorders of the peripheral nervous system include, but are not limited to trigeminal nerve disorders, trigeminal neuralgia, facial nerve disorders, bell's palsy, disorders of other cranial nerves, nerve root and plexus disorders, thoracic outlet syndrome, phantom limb, mononeuritis of upper limb and mononeuritis multiplex, carpal tunnel, mononeuritis of lower limb, lesion of sciatic nerve, meralgia paresthetica, other lesion of femoral nerve, lesion of lateral popliteal nerve, lesion of medial popliteal nerve, tarsal tunnel syndrome, lesion of plantar nerve, morton's neuroma, unspecified mononeuritis of lower limb, mononeuritis of unspecified site, hereditary and idiopathic peripheral neuropathy, inflammatory and toxic neuropathy, guillain barye syndrome, poly neuropathy, alcoholic poly neuropathy, myoneural disorders, myasthenia gravis with exacerbation, myasthenia gravis without exacerbation, muscular dystrophies and other myopathies, benign congenital myopathy, central core disease, centronuclear myopathy, myotubular myopathy, nemaline body disease, and hereditary muscular dyst.","A biosignature of a vesicle can be assessed to provide a theranosis for a subject. The biosignature of the vesicle can comprise one or more biomarkers such as, but not limited to, a biomarker such as those disclosed in the following table:","Neurology: Biosignature","A biosignature of a vesicle can be assessed to provide a theranosis for a subject. The biosignature of the vesicle can comprise one or more biomarkers such as, but not limited to, a biomarker such as listed in . The biosignature of the vesicle can comprise one or more biomarkers including, but not limited to, amyloid \u03b2, ICAM-1 (rodent), CGRP (rodent), TIMP-1 (rodent), CLR-1 (rodent), HSP-27 (rodent), FABP (rodent), ATP5B, ATP5H, ATP6V1B, DNM1, NDUFV2, NSF, PDHB, FGF2, ALDH7A1, AGXT2L1, AQP4, PCNT2, FGFR1, FGFR2, FGFR3, AQP4, a mutation of Dysbindin, DAOA\/G30, DISC1, neuregulin-1, IFITM3, SERPINA3, GLS, ALDH7A1, BASP1, OX42, ED9, apolipoprotein D (rodent), miR-7, miR-24, miR-26b, miR-29b, miR-30b, miR-30e, miR-92, miR-195, miR-181b, DISC1, dysbindin, neuregulin-1, seratonin 2a receptor, and NURR1.","Neurology: Standard of Care","Determining the biosignature of a vesicle, the amount of vesicles, or both, of a sample from a subject suffering from a neurological disorder or disease can be used to select a standard of care for the subject. An neurological disorder or disease can be treated according to symptoms associated with the condition. The standard of care can include, for example, a pharmaceutical drug. A pharmaceutical drug includes, but is not limited to aspirin, dipyridamole, naratriptan, apomorphine, donepezil, almotriptan malate, rufinamide, bromfenac, carbatrol, cenestin, tadalafil, clonazepam, entacapone, glatiramer acetate, pemoline, divalproex, difluprednate, zolpidem tartrate, rivastigmine tartrate, dexmethylphenidate, frovatriptan succinate, zinc acetate, sumatriptan, paliperidone, iontocaine, morphine, levetiracetam, lamotrigine, vardenafil, lidocaine, eszopiclone, fospropofol disodium, pregabalin, rizatriptan benzoate, meropenem, Methylphenidate, dihydroergotamine mesylate, Pramipexole, rimabotulinumtoxin B, naltrexone, memantine, rotigotine, gabapentin), hydrocodone, mitoxantrone, armodafinil, oxycodone, pramipexole, samarium 153 lexidronam, interferon beta-1a, dexfenfluramine, eletriptan hydrobromide, galantamine hydrobromide, ropinirole hydrochloride, riluzole, ramelteon, eldepryl, valproic acid, atomoxetine, tolcapone, carbamazepine, topiramate, oxcarbazepine, natalizumab, acetaminophen, tramadol, midazolam, lacosamide, iodixanol, lisdexamfetamine dimesylate, tetrabenazine, sodium oxybate, tizanidine hydrochloride, zolmitriptan, and zonisamide.","Other treatments that can be selected based on a vesicle profile of a subject includes those listed in Table 14, for a subject with Multiple Sclerosis; Table 16, for a subject with Parkinson's Disease; or Table 17, for a subject with depression.",{"@attributes":{"id":"p-1074","num":"1076"},"tables":{"@attributes":{"id":"TABLE-US-00016","num":"00016"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"336pt","align":"center"}},"thead":{"row":{"entry":"TABLE 16"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Classes of Drugs for Treatment of Parkinson's Disease"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"196pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Class","Mechanism of Action","Examples"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Dopamine Precursors","Act as precursors in the synthesis of dopamine, the","Levodopa,"]},{"entry":[{},"neurotransmitter that is depleted in Parkinson's Disease. Usually","Levodopa-"]},{"entry":[{},"administered in combination with an inhibitor of the carboxylase","carbidopa,"]},{"entry":[{},"enzyme that metabolizes levodopa. Some (e.g., Duodopa) are","Levodopa-"]},{"entry":[{},"given by infusion, e.g., intraduodenal infusion","benserazide,"]},{"entry":[{},{},"Etilevodopa,"]},{"entry":[{},{},"Duodopa"]},{"entry":["Dopamine Agonists","Mimic natural dopamine by directly stimulating striatal dopamine","Bromocriptine,"]},{"entry":[{},"receptors. May be subclassed by which of the five known","Cabergoline,"]},{"entry":[{},"dopamine receptor subtypes the drug activates; generally most","Lisuride, Pergolide,"]},{"entry":[{},"effective are those that activate receptors the in the D2 receptor","Pramipexole,"]},{"entry":[{},"family (specifically D2 and D3 receptors). Some are formulated","Ropinirole,"]},{"entry":[{},"for more controlled release or transdermal delivery.","Talipexole,"]},{"entry":[{},{},"Apomorphine,"]},{"entry":[{},{},"Dihydroergocryptine,"]},{"entry":[{},{},"Lisuride,"]},{"entry":[{},{},"Piribedil,"]},{"entry":[{},{},"Talipexole,"]},{"entry":[{},{},"Rotigotin CDS,"]},{"entry":[{},{},"Sumanirole, SLV-"]},{"entry":[{},{},"308"]},{"entry":["COMT Inhibitors","Inhibits COMT, the second major enzyme that metabolized","Entacapone,"]},{"entry":[{},"levodopa.","Tolcapone,"]},{"entry":[{},{},"Entacapone-"]},{"entry":[{},{},"Levodopa-"]},{"entry":[{},{},"Carbidopa fixed"]},{"entry":[{},{},"combination,"]},{"entry":["MAO-B Inhibitors","MAO-B metabolizes dopamine, and inhibitors of MAO-B thus","Selegiline,"]},{"entry":[{},"prolong dopamine's half-life","Rasagiline,"]},{"entry":[{},{},"Safinamide"]},{"entry":["Antiglutamatergic","Block glutamate release. Reduce levodopa-induced dyskinesia","Amantadine,"]},{"entry":["Agents",{},"Budipine,"]},{"entry":[{},{},"Talampanel,"]},{"entry":[{},{},"Zonisamide"]},{"entry":["Anticholinergic","Thought to inhibit excessive cholinergic activity that","Trihexyphenidyl,"]},{"entry":["Agents","accompanies dopamine deficiency","Benztropine,"]},{"entry":[{},{},"Biperiden"]},{"entry":["Mixed Dopaminergic","Act on several neurotransmitter systems, both dopaminergic and","NS-2330, Sarizotan"]},{"entry":["Agents","nondopaminergic."]},{"entry":["Adenosine A2a","Adenosine A2 antagonize dopamine receptors and are found in","Istradefylline"]},{"entry":["antagonists","conjunction with dopamine receptors. Antagonists of these"]},{"entry":[{},"receptors may enhance the activity of dopamine receptors."]},{"entry":["Alpha-2 Adrenergic","Not known.","Yohimbine,"]},{"entry":["Antagonists",{},"Idazoxan,"]},{"entry":[{},{},"Fipamezole"]},{"entry":["Antiapoptotic Agents","Can slow the death of cells associated with the neurodegenerative","CEP-1347, TCH-"]},{"entry":[{},"process of Parkinson's disease.","346"]},{"entry":["Growth Factor","Promote the survival and growth of dopaminergic cells.","GPI-1485, Glial-"]},{"entry":["Stimulators",{},"cell-line-derived"]},{"entry":[{},{},"Neurotrophic"]},{"entry":[{},{},"Factor, SR-57667,"]},{"entry":[{},{},"PYM-50028"]},{"entry":["Cell Replacement","Replace damaged neurons with health neurons.","Spheramine"]},{"entry":"Therapy"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-1075","num":"1077"},"tables":{"@attributes":{"id":"TABLE-US-00017","num":"00017"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"364pt","align":"center"}},"thead":{"row":{"entry":"TABLE 17"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Classes of Drugs for Treatment of Depression"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"182pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"91pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Class","Mechanism of Action","Examples"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Selective","Block presynaptic reuptake of serotonin. Exert little effect on","Escitalopram, Sertraline,"]},{"entry":["Serotonin","norepinephrine or dopamine reuptake. Level of serotonin in","Citalopram, Paroxetine,"]},{"entry":["Reuptake","the synaptic cleft is increased.","Paroxetin, controlled"]},{"entry":["Inhibitor (SSRI)",{},"release, Fluoxetine,"]},{"entry":[{},{},"Fluoxetine weekly,"]},{"entry":[{},{},"Fluvoxamine,"]},{"entry":[{},{},"olanzapine\/fluoxetine"]},{"entry":[{},{},"combination"]},{"entry":["Serotonergic\/","Inhibit both serotonin reuptake and norepinephrine reuptake.","Venlafaxine; Reboxetine,"]},{"entry":["noradrenergic","Different drugs in this class can inhibit each receptor to","Milnacipran, Mirtazapine,"]},{"entry":["agents","different degrees. Do not affect histamine, acetylcholine, and","Nefazodone, Duloxetine"]},{"entry":[{},"adrenergic receptors."]},{"entry":["Serotonergic\/noradrenergic\/","Several different mechanisms. Block norepinephrine,","Bupropion, Maprotiline,"]},{"entry":["dopaminergic","serotonin, and\/or dopamine reuptake. Some have addictive","Mianserin, Trazodone,"]},{"entry":["agents","potential due to dopamine reuptake inhibition.","Dexmethylphenidate,"]},{"entry":[{},{},"Methyphenidate,"]},{"entry":[{},{},"Amineptine"]},{"entry":["Tricyclic","Block synaptic reuptake of serotonin and norepinephrine.","Amitriptyline, Amoxapine,"]},{"entry":["Antidepressants","Have little effect on dopamine. Strong blockers of muscarinic,","Clomipramine,"]},{"entry":[{},"histaminergic H1, and alpha-1-adrenergic receptors.","Desipramine, Doxepin,"]},{"entry":[{},{},"Imipramine, Nortriptyline,"]},{"entry":[{},{},"Protriptyline, Trimipramine"]},{"entry":["Irreversible","Monoamine oxidase (MAO) metabolizes monoamines such as","Isocarboxazid, Phenelzine,"]},{"entry":["Monoamine","serotonin and norepinephrine. MAO inhibitors inhibit MAO,","Tranylcypromine,"]},{"entry":["Oxidase","thus increasing levels of serotonin and norepinephrine.","Transdermal Selegiline"]},{"entry":"Inhibitors"},{"entry":["Reversible","See above. Short acting, reversible inhibitor, inhibits","Moclobemide"]},{"entry":["Monoamine","deamination of serotonin, norepinephrine, and dopamine."]},{"entry":"Oxidase"},{"entry":"Inhibitors"},{"entry":["Serotonergic\/noradrenergic\/","Act to block all of serotonin, norepinephrine, and dopamine","DOV-216303, DOV-21947"]},{"entry":["dopaminergic","reuptake. May have addictive potential due to dopamine"]},{"entry":["reuptake","reuptake inhibition."]},{"entry":"inhibitors"},{"entry":["Noradrenergic\/dopaminergic","Block reuptake of norepinephrine and dopamine","GW-353162"]},{"entry":"agents"},{"entry":["Serotonin","Selective antagonist of one serotonin receptor (the 5-HT","Agomelatine"]},{"entry":["Antagonists","receptor)"]},{"entry":["Serotonin","Partial agonist of the 5-HTreceptor.","Eptapirone, Vilazodone,"]},{"entry":["Agonists",{},"OPC-14523, MKC-242,"]},{"entry":[{},{},"Gepirone ER"]},{"entry":["Substance P","Modify levels of substance P, which is released during acute","Aprepitant, TAK-637, CP-"]},{"entry":["Antagonists","stress.","122721, E6006, R-763OPC-"]},{"entry":[{},{},"GW-597599"]},{"entry":["Beta","Indirectly inhibit norepinephrine reuptake. Also being","SR-58611"]},{"entry":["Adrenoreceptor","investigated for treatment of obesity and diabetes because"]},{"entry":["Agonists","they stimulate lipolysis and thermogenesis."]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},"In one embodiment, a treatment can be selected for a subject suffering from Alzheimer's disease. One or more biomarkers, such as, but not limited to, beta-amyloid protein, amyloid precursor protein (APP), APP670\/671, APP693, APP692, APP715, APP716, APP717, APP723, presenilin 1, presenilin 2, cerebrospinal fluid amyloid beta protein 42 (CSF-Abeta42), cerebrospinal fluid amyloid beta protein 40 (CSF-Abeta40), F2 isoprostane, 4-hydroxynonenal, F4 neuroprostane, and acrolein, can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, Donepezil, Galantamine, Memantine, Rivastigmine, Tacrine, or a combination thereof.","In another embodiment, a treatment can be selected for a subject suffering from Parkinson's Disease. One or more biomarkers, such as, but not limited to, alpha synuclein, PARK7 (DJ-1), S-phase kinase-associated protein 1A (p19A\/SKP1A), Heat shock protein 70 kDa, AMP-regulated phosphoprotein (ARPP-21), vesicular monoamine member 2 (VMAT2), alcohol dehydrogenase 5 (ADH5), aldehyde dehydrogenase 1A1 (ALDH1A1), egle nine homolog 1(EGLN1), proline hydroxylase 2 (PHD2), and hypoxia inducible factor (HIF), can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment, such as, but not limited to, those listed in Table 16.","In another embodiment, a treatment can be selected for a subject suffering from Parkinson's Disease. One or more biomarkers, such as, but not limited to, CRP, TNF, IL-6, S100B, and MMP can be assessed from a vesicle from the subject. Based on one or more characteristics of the one or more biomarkers, the subject can be determined to be a responder or non-responder for a treatment.","Thus, a treatment can be selected for the subject suffering from a neurology-related condition or neurological condition or disease, based on the biosignature of the subject's vesicle.","Biosignature Discovery","The systems and methods provided herein can be used in identifying a novel biosignature of a vesicle, such as one or more novel biomarkers for the diagnosis, prognosis or theranosis of a phenotype. In one embodiment, one or more vesicles can be isolated from a subject with a phenotype and a biosignature of the one or more vesicles determined. The biosignature can be compared to a subject without the phenotype. Differences between the two biosignatures can be determined and used to form a novel biosignature. The novel biosignature can then be used for identifying another subject as having the phenotype or not having the phenotype.","Differences between the biosignature from a subject with a particular phenotype can be compared to the biosignature from a subject without the particular phenotype. The one or more differences can be a difference in any characteristic of the vesicle. For example, the level or amount of vesicles in the sample, the half-life of the vesicle, the circulating half-life of the vesicle, the metabolic half-life of the vesicle, or the activity of the vesicle, or any combination thereof, can differ between the biosignature from the subject with a particular phenotype and the biosignature from the subject without the particular phenotype.","In some embodiments, one or more biomarkers differ between the biosignature from the subject with a particular phenotype and the biosignature from the subject without the particular phenotype. For example, the expression level, presence, absence, mutation, variant, copy number variation, truncation, duplication, modification, molecular association of one or more biomarkers, or any combination thereof, may differ between the biosignature from the subject with a particular phenotype and the biosignature from the subject without the particular phenotype. The biomarker can be any biomarker disclosed herein or that can be used to characterize a biological entity, including a circulating biomarker, such as protein or microRNA, a vesicle, or a component present in a vesicle or on the vesicle, such as any nucleic acid (e.g. RNA or DNA), protein, peptide, polypeptide, antigen, lipid, carbohydrate, or proteoglycan.","In an aspect, the invention provides a method of discovering a novel biosignature comprising comparing the biomarkers between two or more sample groups to identify biomarkers that show a difference between the sample groups. Multiple markers can be assessed in a panel format to potentially improve the performance of individual markers. In some embodiments, the multiple markers are assessed in a multiplex fashion. The ability of the individual markers and groups of markers to distinguish the groups can be assessed using statistical discriminate analysis or classification methods as used herein. Optimal panels of markers can be used as a biosignature to characterize the phenotype under analysis, such as to provide a diagnosis, prognosis or theranosis of a disease or condition. Optimization can be based on various criteria, including without limitation maximizing ROC AUC, accuracy, sensitivity at a certain specificity, or specificity at a certain sensitivity. The panels can include biomarkers from multiple types. For example, the biosignature can comprise vesicle antigens useful for capturing a vesicle population of interest, and the biosignature can further comprise payload markers within the vesicle population, including without limitation microRNAs, mRNAs, or soluble proteins. Optimal combinations can be identified as those vesicle antigens and payload markers with the greatest ROC AUC value when comparing two settings. As another example, the biosignature can be determined by assessing a vesicle population in addition to assessing circulating biomarkers that are not obtained by isolating exosomes, such as circulating proteins and\/or microRNAs.","The phenotype can be any of those listed herein, e.g., in the \u201cPhenotype\u201d section above. For example, the phenotype can be a proliferative disorder such as a cancer or non-malignant growth, a perinatal or pregnancy related condition, an infectious disease, a neurological disorder, a cardiovascular disease, an inflammatory disease, an immune disease, or an autoimmune disease. The cancer includes without limitation lung cancer, non-small cell lung cancerm small cell lung cancer (including small cell carcinoma (oat cell cancer), mixed small cell\/large cell carcinoma, and combined small cell carcinoma), colon cancer, breast cancer, prostate cancer, liver cancer, pancreatic cancer, brain cancer, kidney cancer, ovarian cancer, stomach cancer, melanoma, bone cancer, gastric cancer, breast cancer, glioma, gliobastoma, hepatocellular carcinoma, papillary renal carcinoma, head and neck squamous cell carcinoma, leukemia, lymphoma, myeloma, or other solid tumors.","Any of the types of biomarkers or specific biomarkers described herein can be assessed to discover a novel biosignature. In an embodiment, the biomarkers selected for discovery comprise cell-specific biomarkers as listed herein, including without limitation the genes and microRNA listed in , Tables 9-11 or Tables 27-41. The biomarkers can comprise one or more drug associated target such as a ABCC1, ABCG2, ACE2, ADA, ADH1C, ADH4, AGT, AR, AREG, ASNS, BCL2, BCRP, BDCA1, beta III tubulin, BIRC5, B-RAF, BRCA1, BRCA2, CA2, caveolin, CD20, CD25, CD33, CD52, CDA, CDKN2A, CDKN1A, CDKN1B, CDK2, CDW52, CES2, CK 14, CK 17, CK 5\/6, c-KIT, c-Met, c-Myc, COX-2, Cyclin D1, DCK, DHFR, DNMT1, DNMT3A, DNMT3B, E-Cadherin, ECGF1, EGFR, EML4-ALK fusion, EPHA2, Epiregulin, ER, ERBR2, ERCC1, ERCC3, EREG, ESR1, FLT1, folate receptor, FOLR1, FOLR2, FSHB, FSHPRH1, FSHR, FYN, GART, GNRH1, GNRHR1, GSTP1, HCK, HDAC1, hENT-1, Her2\/Neu, HGF, HIF1A, HIG1, HSP90, HSP90AA1, HSPCA, IGF-1R, IGFRBP, IGFRBP3, IGFRBP4, IGFRBP5, IL13RA1, IL2RA, KDR, Ki67, KIT, K-RAS, LCK, LTB, Lymphotoxin Beta Receptor, LYN, MET, MGMT, MLH1, MMR, MRP1, MS4A1, MSH2, MSH5, Myc, NFKB1, NFKB2, NFKBIA, ODC1, OGFR, p16, p21, p27, p53, p95, PARP-1, PDGFC, PDGFR, PDGFRA, PDGFRB, PGP, PGR, PI3K, POLA, POLA1, PPARG, PPARGC1, PR, PTEN, PTGS2, RAF1, RARA, RRM1, RRM2, RRM2B, RXRB, RXRG, SIK2, SPARC, SRC, SSTR1, SSTR2, SSTR3, SSTR4, SSTR5, Survivin, TK1, TLE3, TNF, TOP1, TOP2A, TOP2B, TS, TXN, TXNRD1, TYMS, VDR, VEGF, VEGFA, VEGFC, VHL, YES1, and ZAP70. The biomarkers can comprise one or more general vesicle marker, one or more cell-specific vesicle marker, and\/or one or more disease-specific vesicle marker.","The biomarkers used for biosignature discovery can comprise include markers commonly associated with vesicles, including without limitation one or more of HSPA8, CD63, Actb, GAPDH, CD9, CD81, ANXA2, HSP90AA1, ENO1, YWHAZ, PDCD6IP, CFL1, SDCBP, PKN2, MSN, MFGE8, EZR, YWHAG, PGK1, EEF1A1, PPIA, GLC1F, GK, ANXA6, ANXA1, ALDOA, ACTG1, TPI1, LAMP2, HSP90AB1, DPP4, YWHAB, TSG101, PFN1, LDHB, HSPA1B, HSPA1A, GSTP1, GNAI2, GDI2, CLTC, ANXA5, YWHAQ, TUBA1A, THBS1, PRDX1, LDHA, LAMP1, CLU, and CD86. The biomarkers can further comprise one or more of CD63, GAPDH, CD9, CD81, ANXA2, ENO1, SDCBP, MSN, MFGE8, EZR, GK, ANXA1, LAMP2, DPP4, TSG101, HSPA1A, GDI2, CLTC, LAMP1, Cd86, ANPEP, TFRC, SLC3A2, RDX, RAP1B, RAB5C, RAB5B, MYH9, ICAM1, FN1, RAB11B, PIGR, LGALS3, ITGB1, EHD1, CLIC1, ATP1A1, ARF1, RAP1A, P4HB, MUC1, KRT10, HLA-A, FLOT1, CD59, C1orf58, BASP1, TACSTD1, and STOM. Other biomarkers can be selected from those disclosed in the ExoCarta database, available at exocarta.ludwig.edu.au, which discloses proteins and RNA molecules identified in exosomes. See also Mathivanan and Simpson, ExoCarta: A compendium of exosomal proteins and RNA. Proteomics. 2009 November 9(21):4997-5000.","The biomarkers used for biosignature discovery can comprise include markers commonly associated with vesicles, including without limitation one or more of A33, a33 n15, AFP, ALA, ALIX, ALP, AnnexinV, APC, ASCA, ASPH (246-260), ASPH (666-680), ASPH (A-10), ASPH (D01P), ASPH (D03), ASPH (G-20), ASPH(H-300), AURKA, AURKB, B7H3, B7H4, BCA-225, BCNP1, BDNF, BRCA, CA125 (MUC16), CA-19-9, C-Bir, CD1.1, CD10, CD174 (Lewis y), CD24, CD44, CD46, CD59 (MEM-43), CD63, CD66e CEA, CD73, CD81, CD9, CDA, CDAC1 1a2, CEA, C-Erb2, C-erbB2, CRMP-2, CRP, CXCL12, CYFRA21-1, DLL4, DR3, EGFR, Epcam, EphA2, EphA2 (H-77), ER, ErbB4, EZH2, FASL, FRT, FRT c.f23, GDF15, GPCR, GPR30, Gro-alpha, HAP, HBD 1, HBD2, HER 3 (ErbB3), HSP, HSP70, hVEGFR2, iC3b, IL 6 Unc, IL-1B, IL6 Unc, IL6R, IL8, IL-8, INSIG-2, KLK2, L1CAM, LAMN, LDH, MACC-1, MAPK4, MART-1, MCP-1, M-CSF, MFG-E8, MIC1, MIF, MIS RII, MMG, MMP26, MMP7, MMP9, MS4A1, MUC1, MUC1 seq1, MUC1 seq11A, MUC17, MUC2, Ncam, NGAL, NPGP\/NPFF2, OPG, OPN, p53, p53, PA2G4, PBP, PCSA, PDGFRB, PGP9.5, PIM1, PR (B), PRL, PSA, PSMA, PSME3, PTEN, R5-CD9 Tube 1, Reg IV, RUNX2, SCRN1, seprase, SERPINB3, SPARC, SPB, SPDEF, SRVN, STAT 3, STEAP1, TF (FL-295), TFF3, TGM2, TIMP-1, TIMP1, TIMP2, TMEM211, TMPRSS2, TNF-alpha, Trail-R2, Trail-R4, TrKB, TROP2, Tsg 101, TWEAK, UNC93A, VEGF A, and YPSMA-1. The biomarkers can include one or more of NSE, TRIM29, CD63, CD151, ASPH, LAMP2, TSPAN1, SNAIL, CD45, CKS1, NSE, FSHR, OPN, FTH1, PGP9, ANNEXIN 1, SPD, CD81, EPCAM, PTH1R, CEA, CYTO 7, CCL2, SPA, KRAS, TWIST1, AURKB, MMP9, P27, MMP1, HLA, HIF, CEACAM, CENPH, BTUB, INTG b4, EGFR, NACC1, CYTO 18, NAP2, CYTO 19, ANNEXIN V, TGM2, ERB2, BRCA1, B7H3, SFTPC, PNT, NCAM, MS4A1, P53, INGA3, MUC2, SPA, OPN, CD63, CD9, MUC1, UNCR3, PAN ADH, HCG, TIMP, PSMA, GPCR, RACK1, PCSA, VEGF, BMP2, CD81, CRP, PRO GRP, B7H3, MUC1, M2PK, CD9, PCSA, and PSMA. The biomarkers can also include one or more of TFF3, MS4A1, EphA2, GAL3, EGFR, N-gal, PCSA, CD63, MUC1, TGM2, CD81, DR3, MACC-1, TrKB, CD24, TIMP-1, A33, CD66 CEA, PRL, MMP9, MMP7, TMEM211, SCRN1, TROP2, TWEAK, CDACC1, UNC93A, APC, C-Erb, CD10, BDNF, FRT, GPR30, P53, SPR, OPN, MUC2, GRO-1, tsg 101 and GDF15. In embodiments, the biomarkers used to discover a biosignature comprise one or more of those shown in -C, A, and\/or A-E.","One of skill will appreciate that any marker disclosed herein or that can be compared between two samples or sample groups of interest can be used to discover a novel biosignature for any given biological setting that can be compared.","The one or more differences can then be used to form a novel biosignature for the particular phenotype, such as the diagnosis of a condition, diagnosis of a stage of a disease or condition, prognosis of a condition, or theranosis of a condition. The novel biosignature can then be used to identify the phenotype in other subjects. The biosignature of a vesicle for a new subject can be determined and compared to the novel signature to determine if the subject has the particular phenotype for which the novel biosignature was identified from.","For example, the biosignature of a subject with cancer can be compared to another subject without cancer. Any differences can be used to form a novel biosignature for the diagnosis of the cancer. In another embodiment, the biosignature of a subject with an advanced stage of cancer can be compared to another subject with a less advanced stage of cancer. Any differences can be used to form a novel biosignature for the classification of the stage of cancer. In yet another embodiment, the biosignature of a subject with an advanced stage of cancer can be compared to another subject with a less advanced stage of cancer. Any differences can be used to form a novel biosignature for the classification of the stage of cancer.","In one embodiment, the phenotype is drug resistance or non-responsiveness to a therapeutic. One or more vesicles can be isolated from a non-responder to a particular treatment and the biosignature of the vesicle determined. The biosignature of the vesicle obtained from the non-responsder can be compared to the biosignature of a vesicle obtained from a responsder. Differences between the biosignature from the non-responder can be compared to the biosignature from the responder. The one or more differences can be a difference in any characteristic of the vesicle. For example, the level or amount of vesicles in the sample, the half-life of the vesicle, the circulating half-life of the vesicle, the metabolic half-life of the vesicle, the activity of the vesicle, or any combination thereof, can differ between the biosignature from the non-responder and the biosignature from the responder.","In some embodiments, one or more biomarkers differ between the biosignature from the non-responder and the biosignature from the responder. For example, the expression level, presence, absence, mutation, variant, copy number variation, truncation, duplication, modification, molecular association of one or more biomarkers, or any combination thereof, may differ between the biosignature from the non-responder and the biosignature from the responder.","In some embodiments, the difference can be in the amount of drug or drug metabolite present in the vesicle. Both the responder and non-responder can be treated with a therapeutic. A comparison between the biosignature from the responder and the biosignature from the non-responder can be performed, the amount of drug or drug metabolite present in the vesicle from the responder differs from the amount of drug or drug metabolite present in the non-responder. The difference can also be in the half-life of the drug or drug metabolite. A difference in the amount or half-life of the drug or drug metabolite can be used to form a novel biosignature for identifying non-responders and responders.","A vesicle useful for methods and compositions described herein can be discovered by taking advantage of its physicochemical characteristics. For example, a vesicle can be discovered by its size, e.g., by filtering biological matter in a known range from 30-120 nm in diameter. Size-based discovery methods, such as differential centrifugation, sucrose gradient centrifugation, or filtration have been used for isolation of a vesicle.","A vesicle can be discovered by its molecular components. Molecular property-based discovery methods include, but are not limited to, immunological isolation using antibodies recognizing molecules associated with vesicle. For example, a surface molecule associated with a vesicle includes, but not limited to, a MHC-II molecule, CD63, CD81, LAMP-1, Rab7 or Rab5.","Various techniques known in the art are applicable for validation and characterization of a vesicle. Techniques useful for validation and characterization of a vesicle includes, but is not limited to, western blot, electron microscopy, immunohistochemistry, immunoelectron microscopy, FACS (Fluorescent activated cell sorting), electrophoresis (1 dimension, 2 dimension), liquid chromatography, mass spectrometry, MALDI-TOF (matrix assisted laser desorption\/ionization-time of flight), ELISA, LC-MS-MS, and nESI (nanoelectrospray ionization). For example U.S. Pat. No. 2009\/0148460 describes use of an ELISA method to characterize a vesicle. U.S. Pat. No. 2009\/0258379 describes isolation of membrane vesicles from biological fluids.","Vesicles can be further analyzed for one or more nucleic acids, lipids, proteins or polypeptides, such as surface proteins or peptides, or proteins or peptides within a vesicle. Candidate peptides can be identified by various techniques including mass spectrometry coupled with purification methods such as liquid chromatography. A peptide can then be isolated and its sequence can be identified by sequencing. A computer program that predicts a sequence based on exact mass of a peptide can also be used to reveal the sequence of a peptide isolated from a vesicle. For example, LTQ-Orbitrap mass spectrometry can be used for high sensitivity and high accuracy peptide sequencing. LTQ-Orbitrap method has been described (Simpson et al, Expert Rev. Proteomics 6:267-283, 2009), which is incorporated herein by reference in its entirety.","Vesicle Compositions","Also provided herein is an isolated vesicle with a particular biosignature. The isolated vesicle can comprise one or more biomarkers or biosignatures specific for specific cell type, or for characterizing a phenotype, such as described above. For example, the isolated vesicle can comprise one or more biomarkers, such as CD63, EpCam, CD81, CD9, PCSA, PSMA, B7H3, TNFR, MFG-E8, Rab, STEAP, 5T4, or CD59. The isolated vesicle can comprise one or more of the following biomarkers: EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR. In one embodiment, the vesicle is EpCam+, CK+, CD45\u2212. The isolated vesicle can have the one or more biomarkers on its surface or within the vesicle. The isolated vesicle can also comprise one or more miRNAs, such as miR-9, miR-629, miR-141, miR-671-3p, miR-491, miR-182, miR-125a-3p, miR-324-5p, miR-148B, or miR-222. In one embodiment, the vesicle comprises one or more miRNAs, such as miR-548c-5p, miR-362-3p, miR-422a, miR-597, miR-429, miR-200a, and miR-200b. In yet another embodiment, the vesicle comprises one or more miRNAs, such as miR-92a-2*, miR-147, miR-574-5p. An isolated vesicle can comprise a biomarker such as CD66, and further comprise one or more biomarkers selected from the group consisting of: EpCam, CD63, or CD9. An isolated vesicle can also comprise a fusion gene or protein, such as TMRSSG2:ERG.","An isolated vesicle can also comprise one or more biomarkers, wherein the expression level of the one or more biomarkers is higher, lower, or the same for an isolated vesicle as compared to an isolated vesicle derived from a normal cell (ie. a cell derived from a subject without a phenotype of interest). For example, an isolated vesicle can comprise one or more biomarkers selected from the group consisting of: B7H3, PSCA, MFG-E8, Rab, STEAP, PSMA, PCSA, 5T4, miR-9, miR-629, miR-141, miR-671-3p, miR-491, miR-182, miR-125a-3p, miR-324-5p, miR-148b, and miR-222, wherein the expression level of the one or more biomarkers is higher for an isolated vesicle as compared those derived from a normal cell. The isolated vesicle can comprise at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 14, 15, 16, 17, 18, or 19 of the biomarkers selected from the group. The isolated vesicle can further comprising one or more biomarkers selected from the group consisting of: EpCam, CD63, CD59, CD81, or CD9.","An isolated vesicle can comprise the biomarkers PCSA, EpCam, CD63, and CD8; the biomarkers PCSA, EpCam, B7H3 and PSMA. An isolated vesicle can comprise the biomarkers miR-9, miR-629, miR-141, miR-671-3p, miR-491, miR-182, miR-125a-3p, miR-324-5p, miR-148b, and miR-222.","A composition comprising an isolated vesicle is also provided herein. The composition can comprise one or more isolated vesicles. For example, the composition can comprise a plurality of vesicles, or one or more populations of vesicles.","The composition can be substantially enriched for vesicles. For example, the composition can be substantially absent of cellular debris, cells, or non-exosomal proteins, peptides, or nucleic acids (such as biological molecules not contained within the vesicles). The cellular debris, cells, or non-exosomal proteins, peptides, or nucleic acids, can be present in a biological sample along with vesicles. A composition can be substantially absent of cellular debris, cells, or non-exosomal proteins, peptides, or nucleic acids (such as biological molecules not contained within the vesicles), can be obtained by any method disclosed herein, such as through the use of one or more binding agents or capture agents for one or more vesicles. The vesicles can comprise at least 30, 40, 50, 60, 70, 80, 90, 95 or 99% of the total composition, by weight or by mass. The vesicles of the composition can be a heterogeneous or homogeneous population of vesicles. For example, a homogeneous population of vesicles comprises vesicles that are homogeneous as to one or more properties or characteristics. For example, the one or more characteristics can be selected from a group consisting of: one or more of the same biomarkers, a substantially similar or identical biosignature, derived from the same cell type, vesicles of a particular size, and a combination thereof.","Thus, in some embodiments, the composition comprises a substantially enriched population of vesicles. The composition can be enriched for a population of vesicles that are at least 30, 40, 50, 60, 70, 80, 90, 95 or 99% homogeneous as to one or more properties or characteristics. For example, the one or more characteristics can be selected from a group consisting of: one or more of the same biomarkers, a substantially similar or identical biosignature, derived from the same cell type, vesicles of a particular size, and a combination thereof. For example, the population of vesicles can be homogeneous by all having a particular biosignature, having the same biomarker, having the same biomarker combination, or derived from the same cell type. In some embodiments, the composition comprises a substantially homogeneous population of vesicles, such as a population with a specific biosignature, derived from a specific cell, or both.","The population of vesicles can comprise one or more of the same biomarkers. The biomarker can be any component such as any nucleic acid (e.g. RNA or DNA), protein, peptide, polypeptide, antigen, lipid, carbohydrate, or proteoglycan. For example, each vesicle in a population can comprise the same or identical one or more biomarkers. In some embodiments, each vesicle comprises the same 1, 2, 3, 4, 5, 6, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 biomarkers. The one or more biomarkers can be selected from .","The vesicle population comprising the same or identical biomarker can refer to each vesicle in the population having the same presence or absence, expression level, mutational state, or modification of the biomarker. For example, an enriched population of vesicle can comprise vesicles wherein each vesicle has the same biomarker present, the same biomarker absent, the same expression level of a biomarker, the same modification of a biomarker, or the same mutation of a biomarker. The same expression level of a biomarker can refer to a quantitative or qualitive measurement, such as the vesicles in the population underexpress, overexpress, or have the same expression level of a biomarker as compared to a reference level.","Alternatively, the same expression level of a biomarker can be a numerical value representing the expression of a biomarker that is similar for each vesicle in a population. For example the copy number of a miRNA, the amount of protein, or the level of mRNA of each vesicle, can be quantitatively similar for each vesicle in a population, such that the numerical amount of each vesicle is \u00b11, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, or 20% from the amount in each other vesicle in the population, as such variations are appropriate.","In some embodiments, the composition comprises a substantially enriched population of vesicles, wherein the vesicles in the enriched population has a substantially similar or identical biosignature. The biosignature can comprise one or more characteristic of the vesicle, such as the level or amount of vesicles, temporal evaluation of the variation in vesicle half-life, circulating vesicle half-life, metabolic half-life of a vesicle, or the activity of a vesicle. The biosignature can also comprise the presence or absence, expression level, mutational state, or modification of a biomarker, such as those described herein.","The biosignature of each vesicle in the population can be at least 30, 40, 50, 60, 70, 80, 90, 95, or 99% identical. In some embodiments, the biosignature of each vesicle is 100% identical. The biosignature of each vesicle in the enriched population can have the same 1, 2, 3, 4, 5, 6, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25, 50, 75 or 100 characteristics. For example, a biosignature of a vesicle in an enriched population can be the presence of a first biomarker, the presence of a second biomarker, and the underexpression of a third biomarker. Another vesicle in the same population can be 100% identical, having the same first and second biomarkers present and underexpression of the third biomarker. Alternatively, a vesicle in the same population can have the same first and second biomarkers, but not have underexpression of the third biomarker.","In some embodiments, the composition comprises a substantially enriched population of vesicles, wherein the vesicles are derived from the same cell type. For example, the vesicles can all be derived from cells of a specific tissue, cells from a specific tumor of interest or a diseased tissue of interest, circulating tumor cells, or cells of maternal or fetal origin. The vesicles can all be derived from tumor cells. The vesicles can all be derived from lung, pancreas, stomach, intestine, bladder, kidney, ovary, testis, skin, colorectal, breast, prostate, brain, esophagus, liver, placenta, or fetal cells.","The composition comprising a substantially enriched population of vesicles can also comprise vesicles are of a particular size. For example, the vesicles can all a diameter of greater than about 10, 20, or nm. They can all have a diameter of about 30-1000 nm, about 30-800 nm, about 30-200 nm, or about 30-100 nm. In some embodiments, the vesicles can all have a diameter of less than about 10,000 nm, 1000 nm, 800 nm, 500 nm, 200 nm, 100 nm or 50 nm.","The population of vesicles homogeneous for one or more characteristics can comprises at least about 30, 40, 50, 60, 70, 80, 90, 95, or 99% of the total vesicle population of the composition. In some embodiments, a composition comprising a substantially enriched population of vesicles comprises at least 2, 3, 4, 5, 10, 20, 25, 50, 100, 250, 500, or 1000 times the concentration of vesicle as compared to a concentration of the vesicle in a biological sample from which the composition was derived. In yet other embodiments, the composition can further comprise a second enriched population of vesicles, wherein the population of vesicles is at least 30% homogeneous as to one or more characteristics, as described herein.","Multiplex analysis can be used to obtain a composition substantially enriched for more than one population of vesicles, such as at least 2, 3, 4, 5, 6, 7, 8, 9, 10 vesicle, populations. Each substantially enriched vesicle population can comprise at least 5, 10, 15, 20, 25, 30, 35, 40, 45, 46, 47, 48, or 49% of the composition, by weight or by mass. In some embodiments, the substantially enriched vesicle population comprises at least about 30, 40, 50, 60, 70, 80, 90, 95, or 99% of the composition, by weight or by mass.","A substantially enriched population of vesicles can be obtained by using one or more methods, processes, or systems as disclosed herein. For example, isolation of a population of vesicles from a sample can be performed by using one or more binding agents for one or more biomarkers of a vesicle, such as using two or more binding agents that target two or more biomarkers of a vesicle. One or more capture agents can be used to obtain a substantially enriched population of vesicles. One or more detection agents can be used to identify a substantially enriched population of vesicles.","In one embodiment, a population of vesicles with a particular biosignature is obtained by using one or more binding agents for the biomarkers of the biosignature. The vesicles can be isolated resulting in a composition comprising a substantially enriched population of vesicles with the particular biosignature. In another embodiment, a population of vesicles with a particular biosignature of interest can be obtained by using one or more binding agents for biomarkers that are not a component of the biosignature of interest. Thus, the binding agents can be used to remove the vesicles that do not have the biosignature of interest and the resulting composition is substantially enriched for the population of vesicles with the particular biosignature of interest. The resulting composition can be substantially absent of the vesicles comprising a biomarker for the binding agent.","Detection System and Kits","Also provided is a detection system configured to determine one or more biosignatures for a vesicle. The detection system can be used to detect a heterogeneous population of vesicles or one or more homogeneous population of vesicles. The detection system can be configured to detect a plurality of vesicles, wherein at least a subset of the plurality of vesicles comprises a different biosignature from another subset of the plurality of vesicles. The detection system detect at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, or 100 different subsets of vesicles, wherein each subset of vesicles comprises a different biosignature. For example, a detection system, such as using one or more methods, processes, and compositions disclosed herein, can be used to detect at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, or 100 different populations of vesicles.","The detection system can be configured to assess at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, 100, 1000, 2500, 5000, 7500, 10,000, 100,000, 150,000, 200,000, 250,000, 300,000, 350,000, 400,000, 450,000, 500,000, 750,000, or 1,000,000 different biomarkers for one or more vesicles. In some embodiments, the one or more biomarkers are selected from , or as disclosed herein. The detection system can be configured to assess a specific population of vesicles, such as vesicles from a specific cell-of-origin, or to assess a plurality of specific populations of vesicles, wherein each population of vesicles has a specific biosignature.","The detection system can be a low density detection system or a high density detection system. For example, a low density detection system can detect up to 1, 2, 3, 4, 5, 6, 7, 8, 9, or 10 different vesicle populations, whereas a high density detection system can detect at least about 15, 20, 25, 50, or 100 different vesicle populations. In another embodiment, a low density detection system can detect up to about 100, 200, 300, 400, or 500 different biomarkers, whereas a high density detection system can detect at least about 750, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9,000, 10,000, 15,000, 20,000, 25,000, 50,000, or 100,000 different biomarkers. In yet another embodiment, a low density detection system can detect up to about 100, 200, 300, 400, or 500 different biosignatures or biomarker combinations, whereas a high density detection system can detect at least about 750, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9,000, 10,000, 15,000, 20,000, 25,000, 50,000, or 100,000 biosignatures or biomarker combinations.","The detection system can comprise a probe that selectively hybridizes to a vesicle. The detection system can comprise a plurality of probes to detect a vesicle. In some embodiments, a plurality of probes is used to detect the amount of vesicles in a heterogeneous population of vesicles. In yet other embodiments, a plurality of probes is used to detect a homogeneous population of vesicles. A plurality of probes can be used to isolate or detect at least two different subsets of vesicles, wherein each subset of vesicles comprises a different biosignature.","A detection system, such as using one or more methods, processes, and compositions disclosed herein, can comprise a plurality of probes configured to detect, or isolate, such as using one or more methods, processes, and compositions disclosed herein at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, or 100 different subsets of vesicles, wherein each subset of vesicles comprises a different biosignature.","For example, a detection system can comprise a plurality of probes configured to detect at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, or 100 different populations of vesicles. The detection system can comprise a plurality of probes configured to selectively hybridize to at least 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, 100, 1000, 2500, 5000, 7500, 10,000, 100,000, 150,000, 200,000, 250,000, 300,000, 350,000, 400,000, 450,000, 500,000, 750,000, or 1,000,000 different biomarkers for one or more vesicles. In some embodiments, the one or more biomarkers are selected from , or as disclosed herein. The plurality of probes can be configured to assess a specific population of vesicles, such as vesicles from a specific cell-of-origin, or to assess a plurality of specific populations of vesicles, wherein each population of vesicles has a specific biosignature.","The detection system can be a low density detection system or a high density detection system comprising probes to detect vesicles. For example, a low density detection system can comprise probes to detect up to 1, 2, 3, 4, 5, 6, 7, 8, 9, or 10 different vesicle populations, whereas a high density detection system can comprise probes to detect at least about 15, 20, 25, 50, or 100 different vesicle populations. In another embodiment, a low density detection system can comprise probes to detect up to about 100, 200, 300, 400, or 500 different biomarkers, whereas a high density detection system can comprise probes to detect at least about 750, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9,000, 10,000, 15,000, 20,000, 25,000, 50,000, or 100,000 different biomarkers. In yet another embodiment, a low density detection system can comprise probes to detect up to about 100, 200, 300, 400, or 500 different biosignatures or biomarker combinations, whereas a high density detection system can comprise probes to detect at least about 750, 1000, 2000, 3000, 4000, 5000, 6000, 7000, 8000, 9,000, 10,000, 15,000, 20,000, 25,000, 50,000, or 100,000 biosignatures or biomarker combinations.","The probes can be specific for detecting a specific vesicle population, for example a vesicle with a particular biosignature, and as described above. A plurality of probes for detecting prostate specific vesicles is also provided. A plurality of probes can comprise probes for detecting one or more of the following biomarkers: CD9, PSCA, TNFR, CD63, MFG-E8, EpCAM, Rab, CD81, STEAP, PCSA, 5T4, EpCAM, PSMA, CD59, CD66, CD24 and B7H3. A plurality of probes for detecting Bcl-XL, ERCC1, Keratin 15, CD81\/TAPA-1, CD9, Epithelial Specific Antigen (ESA), and Mast Cell Chymase can also be provided.","A plurality of probes for detecting one or more miRNAs of a vesicle can comprise probes for detecting one or more of the following miRNAs: miR-9, miR-629, miR-141, miR-671-3p, miR-491, miR-182, miR-125a-3p, miR-324-5p, miR-148b, and miR-222. In another embodiment, the plurality of probes comprises one or more probes for detecting EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR. In some embodiments, the plurality of probes comprises one or more probes for detecting EpCam, CD9, PCSA, CD63, CD81, PSMA, and B7H3. In other embodiments, the plurality of probes comprises one or more probes for detecting EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR. In yet another embodiment, a subset of the plurality of probes are capture agents for one or more of EpCam, CD9, PCSA, CD63, CD81, PSMA, B7H3, PSCA, ICAM, STEAP, and EGFR, and another subset are probes for detecting one or more of CD9, CD63, and CD81. A plurality of probes can also comprises one or more probes for detecting r miR-92a-2*, miR-147, miR-574-5p, or a combination thereof. A plurality of probes can also comprise one or more probes for detecting miR-548c-5p, miR-362-3p, miR-422a, miR-597, miR-429, miR-200a, miR-200b or a combination thereof. A plurality of probes can also comprise one or more probes for detecting EpCam, CK, and CD45. In some embodiments, the one or more probes may be capture agents. In another embodiment, the probes may be detection agents. In yet another embodiment, the plurality of probes comprises capture and detection agents.","The probes, such as capture agents, may be attached to a solid substrate, such as an array or bead. Alternatively, the probes, such as detection agents, are not attached. The detection system may be an array based system, a sequencing system, a PCR-based system, or a bead-based system, such as described above. The detection system can also be a microfluidic device as described above.","The detection system may be part of a kit. Alternatively, the kit may comprise the one or more probe sets or plurality of probes, as described herein. The kit may comprise probes for detecting a vesicle or a plurality of vesicles, such as vesicles in a heterogeneous population. The kit may comprise probes for detecting a homogeneous population of vesicles. For example, the kit may comprise probes for detecting a population of specific cell-of-origin vesicles, or vesicles with the same specific biosignature.","Portfolios","Portfolios of multiplexed markers to guide clinical decisions and disease detection and management can be established such that the combination of biosignatures in the portfolio exhibit improved sensitivity and specificity relative to individual biosignatures or randomly selected combinations of biosignatures. In the context of the instant invention, the sensitivity of the portfolio can be reflected in the fold differences exhibited by a biosignature's expression in the diseased state relative to the normal state. Specificity can be reflected in statistical measurements of the correlation of the signaling of gene expression, for example, with the condition of interest (e.g. standard deviation can be a used as such a measurement). In considering a group of biosignature for inclusion in a portfolio, a small standard deviation in measurements correlates with greater specificity. Other measurements of variation such as correlation coefficients can also be used in this capacity.","When combining biomarkers or biosignatures in this invention In Vitro Diagnostic Multivariate Index Assays (IVDMIAs) guidelines and regulations may apply. IVDMIAs can apply to biosignatures as defined as a set of 2 or more markers composed of any combination of genes, gene alterations, mutations, amplifications, deletions, polymorphisms or methylations, or proteins, peptides, polypeptides or RNA molecules, miRNAs, mRNAs, snoRNAs, hnRNAs or RNA that can be grouped so that information obtained about the set of biosignatures in the group provides a sound basis for making a clinically relevant judgment such as a diagnosis, prognosis, or treatment choice. These sets of biosignatures make up various portfolios of the invention. As with most diagnostic markers, it is often desirable to use the fewest number of markers sufficient to make a correct medical judgment. This prevents a delay in treatment pending further analysis as well inappropriate use of time and resources. Preferably, portfolios are established such that the combination of biosignatures in the portfolio exhibit improved sensitivity and specificity relative to individual biosignatures or randomly selected combinations of biosignatures. In the context of the instant invention, the sensitivity of the portfolio can be reflected in the fold differences exhibited by a biosignature's expression in the diseased state relative to the normal state. Specificity can be reflected in statistical measurements of the correlation of the signaling of gene expression, for example, with the condition of interest. In considering a group of markers in a biosignature for inclusion in a portfolio, standard deviations, variances, co-variances, correlation coefficients, weighted averages, arithmetic sums, means, multiplicative values, weighted or balanced values or any mathematical manipulation of the values of 2 or more markers that can together be used to calculate a value or score that taken as a whole can be shown to produce greater sensitivity, specificity, negative predictive value, positive predictive value or accuracy can also be used in this capacity and are within the scope of this invention.","In another embodiment pattern recognition methods can be used. One example involves comparing biomarker expression profiles for various biomarkers (or biosignature portfolios) to ascribe diagnoses. The expression profiles of each of the biomarker comprising the biosignature portfolio are fixed in a medium such as a computer readable medium.","In one example, a table can be established into which the range of signals (e.g., intensity measurements) indicative of disease or physiological state is input. Actual patient data can then be compared to the values in the table to determine whether the patient samples are normal, benign, diseased, or represent a specific physiological state. In a more sophisticated embodiment, patterns of the expression signals (e.g., fluorescent intensity) are recorded digitally or graphically. In the example of RNA expression patterns from the biomarker portfolios used in conjunction with patient samples are then compared to the expression patterns. Pattern comparison software can then be used to determine whether the patient samples have a pattern indicative of the disease, a given prognosis, a pattern that indicates likeliness to respond to therapy, or a pattern that is indicative of a particular physiological state. The expression profiles of the samples are then compared to the portfolio of a control cell. If the sample expression patterns are consistent with the expression pattern(s) for disease, prognosis, or therapy-related response then (in the absence of countervailing medical considerations) the patient is diagnosed as meeting the conditions that relate to these various circumstances. If the sample expression patterns are consistent with the expression pattern derived from the normal\/control vesicle population then the patient is diagnosed negative for these conditions.","In another exemplary embodiment, a method for establishing biomarker expression portfolios is through the use of optimization algorithms such as the mean variance algorithm widely used in establishing stock portfolios. This method is described in detail in the U.S. Application Publication No. 20030194734, incorporated herein by reference. Alternatively, measured DNA alterations, changes in mRNA, protein, or metabolites to phenotypic readouts of efficacy and toxicity may be modeled and analyzed using algorithms, systems and methods described in U.S. Pat. Nos. 7,089,168, 7,415,359 and U.S. Application Publication Nos. 20080208784, 20040243354, or 20040088116, each of which is herein incorporated by reference in its entirety.","An exemplary process of biosignature portfolio selection and characterization of an unknown is summarized as follows:","(1) Choose baseline class.","(2) Calculate mean, and standard deviation of each biomarker for baseline class samples.","(3) Calculate (X*Standard Deviation+Mean) for each biomarker. This is the baseline reading from which all other samples will be compared. X is a stringency variable with higher values of X being more stringent than lower.","(4) Calculate ratio between each Experimental sample versus baseline reading calculated in step 3.","(5) Transform ratios such that ratios less than 1 are negative (eg. using Log base 10). (Under expressed biomarkers now correctly have negative values necessary for MV optimization).","(6) These transformed ratios are used as inputs in place of the asset returns that are normally used in the software application.","(7) The software will plot the efficient frontier and return an optimized portfolio at any point along the efficient frontier.","(8) Choose a desired return or variance on the efficient frontier.","(9) Calculate the Portfolio's Value for each sample by summing the multiples of each gene's intensity value by the weight generated by the portfolio selection algorithm.","(10) Calculate a boundary value by adding the mean Biosignature Portfolio Value for Baseline groups to the multiple of Y and the Standard Deviation of the Baseline's Biosignature Portfolio Values. Values greater than this boundary value shall be classified as the Experimental Class.","(11) Optionally one can reiterate this process until best prediction.","The process of selecting a biosignature portfolio can also include the application of heuristic rules. Preferably, such rules are formulated based on biology and an understanding of the technology used to produce clinical results. More preferably, they are applied to output from the optimization method. For example, the mean variance method of biosignature portfolio selection can be applied to microarray data for a number of biomarkers differentially expressed in subjects with a specific disease. Output from the method would be an optimized set of biomarkers that could include those that are expressed in vesicles as well as in diseased tissue. If samples used in the testing method are obtained from vesicles and certain biomarkers differentially expressed in instances of disease or physiological state could also be differentially expressed in vesicles, then a heuristic rule can be applied in which a biosignature portfolio is selected from the efficient frontier excluding those that are differentially expressed in vesicles. Of course, the rule can be applied prior to the formation of the efficient frontier by, for example, applying the rule during data pre-selection.","Other statistical, mathematical and computational algorithms for the analysis of linear and non-linear feature subspaces, feature extraction and signal deconvolution in large scale datasets to identify vesicle-derived multiplex analyte profiles for diagnosis, prognosis and therapy selection and\/or characterization of define physiological states can be done using any combination of unsupervised analysis methods, including but not limited to: principal component analysis (PCA) and linear and non-linear independent component analysis (ICA); blind source separation, nongaussinity analysis, natural gradient maximum likelihood estimation; joint-approximate diagonalization; eigenmatrices; Gaussian radical basis function, kernel and polynominal kernel analysis sequential floating forward selection.","Computer Systems","A vesicle can be assayed for molecular features, for example, by determining an amount, presence or absence of one or more biomarkers such as listed . The data generated can be used to produce a biosignature, which can be stored and analyzed by a computer system, such as shown in . The assaying or correlating of the biosignature with one or more phenotypes can also be performed by computer systems, such as by using computer executable logic.","A computer system, such as shown in , can be used to transmit data and results following analysis. Accordingly,  is a block diagram showing a representative example logic device through which results from a vesicle can be analyzed and the analysis reported or generated.  shows a computer system (or digital device)  to receive and store data generated from a vesicle, analyze of the data to generate one or more biosignatures, and produce a report of the one or more biosignatures or phenotype characterization. The computer system can also perform comparisons and analyses of biosignatures generated, and transmit the results. Alternatively, the computer system can receive raw data of vesicle analysis, such as through transmission of the data over a network, and perform the analysis.","The computer system  may be understood as a logical apparatus that can read instructions from media  and\/or network port , which can optionally be connected to server  having fixed media . The system shown in  includes CPU , disk drives , optional input devices such as keyboard  and\/or mouse  and optional monitor . Data communication can be achieved through the indicated communication medium to a server  at a local or a remote location. The communication medium can include any means of transmitting and\/or receiving data. For example, the communication medium can be a network connection, a wireless connection or an interne connection. Such a connection can provide for communication over the World Wide Web. It is envisioned that data relating to the present invention can be transmitted over such networks or connections for reception and\/or review by a party . The receiving party  can be but is not limited to an individual, a health care provider or a health care manager. Thus, the information and data on a test result can be produced anywhere in the world and transmitted to a different location. For example, when an assay is conducted in a differing building, city, state, country, continent or offshore, the information and data on a test result may be generated and cast in a transmittable form as described above. The test result in a transmittable form thus can be imported into the U.S. to receiving party . Accordingly, the present invention also encompasses a method for producing a transmittable form of information on the diagnosis of one or more samples from an individual. The method comprises the steps of (1) determining a diagnosis, prognosis, theranosis or the like from the samples according to methods of the invention; and (2) embodying the result of the determining step into a transmittable form. The transmittable form is the product of the production method. In one embodiment, a computer-readable medium includes a medium suitable for transmission of a result of an analysis of a biological sample, such as biosignatures. The medium can include a result regarding a vesicle, such as a biosignature of a subject, wherein such a result is derived using the methods described herein.","Ex Vivo Harvesting of Vesicles","A vesicle for analysis and determination of a phenotype can also be from ex vivo harvesting. Cells can be cultured and vesicles released from cells of interest in culture either result spontaneously or can be stimulated to release vesicles into the medium. (see for example, Zitvogel, et al. 19984: 594-600; Chaput, et al. 2004172: 2137-214631: 2892-2900; Escudier, et al. 20053: 10; Morse, et al. 20053: 9; Peche, et al. 20066: 1541-1550; Kim, et al. 2005174: 6440-6448, all of which are herein incorporated by reference in their entireties). Cell lines or tissue samples can be grown to 80% confluence before being cultured in fresh DMEM for 72 h. Subsequent vesicle production can be stimulated (see, for example, heat shock treatment of melanoma cells as described by Dressel, et al. 200363: 8212-8220, which is herein incorporated by reference in its entirety). The supernatant can then be harvested and vesicles prepared as described herein.","A vesicle produced ex vivo can, in one example, be cultured from a cell-of-origin or cell line of interest, vesicles can be isolated from the cell culture medium and subsequently labeled with a magnetic label, a fluorescent moiety, a radioisotope, an enzyme, a chemiluminescent probe, a metal particle, a non-metal colloidal particle, a polymeric dye particle, a pigment molecule, a pigment particle, an electrochemically active species, semiconductor nanocrystal or other nanoparticles including quantum dots or gold particles to be reintroduced in vivo as a label for imaging analysis. Ex vivo cultured vesicles can alternatively be used to identify novel biosignatures by setting up culturing conditions for a given cell-of-origin with characteristics of interest, for example a culture of lung cancer cells or cell line with a known EGFR mutation that confers resistant to or susceptibility to gefitinib, then exposing the cell culture to gefitinib, isolating vesicles that arise from the culture and subsequently analyzing them on a discovery array to look for novel antigens or binding agents expressed on the outside of vesicles that could be used as a biosignature to capture this species of vesicle. Additionally, it would be possible to isolate any other biomarkers or biosignatures found within these vesicles for discovery of novel signatures (including but not limited to nucleic acids, proteins, lipids, or combinations thereof) that may have clinical diagnostic, prognostic or therapy related implication.","Cells of interest can also be first isolated and cultured from tissues of interest. For example, human hair follicles in the growing phase, anagen, can be plucked individually from a patient's scalp using sterile equipment and plasticware, taking care not to damage the follicle. Each sample can be transferred to a Petri dish containing sterile PBS for tissue culture. Isolated human anagen hair follicles can be carefully transferred to an individual well of a 24-well plate containing 1 ml of William's E medium. Follicles can be maintained free-floating at 37\u00b0 C. in an atmosphere of 5% COand 95% air in a humidified incubator. Medium can be changed every 3 days, taking care not to damage the follicles. Cells can then be collected and spun down from the media. Vesicles may then be isolated using antigens or cellular binding partners that are specific to such cell-of-origin specific vesicles using methods as previously described. Biomarkers and biosignatures can then be isolated and characterized by methods known to those skilled in the art.","Cells of interest may also be cultured under microgravity or zero-gravity conditions or under a free-fall environment. For example, NASA's bioreactor technology will allow such cells to be grown at much faster rate and in much greater quantities. Vesicles may then be isolated using antigens or cellular binding partners that are specific to such cell-of-origin specific vesicles using methods as previously described.","Rotating wall vessels or RWVersus are a class of bioreactors developed by and for NASA that are designed to grow suspension cultures of cells in a quiescent environment that simulates microgravity can also be used. (see for example, U.S. Pat. Nos. 5,026,650; 5,153,131; 5,153,133; 5,437,998; 5,665,594; 5,702,941; 7,351,584, 5,523,228, 5,104,802, 6,117,674, Schwarz, R P, et al., 14:51-58, 1992; Martin et al., 2004; 22; 80-86, Li et al., 2004; 18; 97-104, Ashammakhi et al., 2006; 9-10: 2693-2711, Zhang et al., 2007; 4: 623-638, Cowger, N L, et al., 64:14-26, 1999, Spaulding, G F, et al., 51:249-251, 1993, Goodwin, T J et al., 202:181-192, 1993; Freed, L E et al., 33:381-385, 1997, Clejan, S. et al, 50:587-597, 1996). Khaoustov, V I, et al., 35:501-509. 1999, each of which is herein incorporated by reference in its entirety).","Alternatively, cells of interest or cell-of-origin specific vesicles that have been isolated may be cultured in a stationary phase plug-flow bioreactor as generally described in U.S. Pat. No. 6,911,201, and U.S. Application Publication Nos. 20050181504, 20050180958, 20050176143 and 20050176137, each of which is herein incorporated by reference in its entirety. Alternatively, cells of interest or cell-origin specific vesicles may also be isolated and cultured as generally described in U.S. Pat. No. 5,486,359.","One embodiment can include the steps of providing a tissue specimen containing cells of interest or cell-origin specific vesicles, adding cells or vesicles from the tissue specimen to a medium which allows, when cultured, for the selective adherence of only the cells of interest or cell-origin specific vesicles to a substrate surface, culturing the specimen-medium mixture, and removing the non-adherent matter from the substrate surface is generally described in U.S. Pat. No. 5,486,359, which is herein incorporated by reference in its entirety.","Prostate cancer cell lines are cultured for 3-4 days in culture media containing 20% FBS (fetal bovine serum) and 1% P\/S\/G. The cells are then pre-spun for 10 minutes at 400\u00d7g at 4\u00b0 C. The supernatant is kept and centrifuged for 20 minutes at 2000\u00d7g at 4. The supernatant containing vesicles can be concentrated using a Millipore Centricon Plus-70 (Cat # UFC710008 Fisher).","The Centricon is pre washed with 30 mls of PBS at 1000\u00d7g for 3 minutes at room temperature. Next, 15-70 mls of the pre-spun cell culture supernatant is poured into the Concentrate Cup and is centrifuged in a Swing Bucket Adapter (Fisher Cat #75-008-144) for 30 minutes at 1000\u00d7g at room temperature.","The flow through in the Collection Cup is poured off. The volume in the Concentrate Cup is brought back up to 60 mls with any additional supernatant. The Concentrate Cup is centrifuged for 30 minutes at 1000\u00d7g at room temperature to concentrate the cell supernatant.","The Concentrate Cup is washed by adding 70 mls of PBS and centrifuged for 30-60 minutes at 1000\u00d7g until approximately 2 mls remains. The vesicles are removed from the filter by inverting the concentrate into the small sample cup and centrifuge for 1 minute at 4\u00b0 C. The volume is brought up to 25 mls with PBS. The vesicles are now concentrated and are added to a 30% Sucrose Cushion.","To make a cushion, 4 mls of Tris\/30% Sucrose\/D2O solution (30g protease-free sucrose, 2.4 g Tris base, 50 ml D2O, adjust pH to 7.4 with 10N NCL drops, adjust volume to 100 mls with D2O, sterilize by passing thru a 0.22-um filter) is loaded to the bottom of a 30 ml V bottom thin walled Ultracentrifuge tube. The diluted 25 mls of concentrated vesicles is gently added above the sucrose cushion without disturbing the interface and is centrifuged for 75 minutes at 100,000\u00d7g at 4\u00b0 C. The \u02dc25 mls above the sucrose cushion is carefully removed with a 10 ml pipet and the \u02dc3.5 mls of vesicles is collected with a fine tip transfer pipet (SAMCO 233) and transferred to a fresh ultracentrifuge tube, where 30 mls PBS is added. The tube is centrifuged for 70 minutes at 100,000\u00d7g at 4\u00b0 C. The supernatant is poured off carefully. The pellet is resuspended in 200 ul PBS and can be stored at 4\u00b0 C. or used for assays. A BCA assay (1:2) can be used to determine protein content and Western blotting or electron micrography can be used to determine vesicle purification.","Vesicles from Vertebral-Cancer of the Prostate (VCaP) and 22Rv1, a human prostate carcinoma cell line, derived from a human prostatic carcinoma xenograft (CWR22R) were collected by ultracentrifugation by first diluting plasma with an equal volume of PBS (1 ml). The diluted fluid was transferred to a 15 ml falcon tube and centrifuged 30 minutes at 2000\u00d7g 4\u00b0 C. The supernatant (\u02dc2 mls) was transferred to an ultracentrifuge tube 5.0 ml PA thinwall tube (Sorvall #03127) and centrifuged at 12,000\u00d7g, 4\u00b0 C. for 45 minutes.","The supernatant (\u02dc2 mls) was transferred to a new 5.0 ml ultracentrifuge tubes and filled to maximum volume with addition of 2.5 mls PBS and centrifuged for 90 minutes at 110,000\u00d7g, 4\u00b0 C. The supernatant was poured off without disturbing the pellet and the pellet resuspended with 1 ml PBS. The tube was filled to maximum volume with addition of 4.5 ml of PBS and centrifuged at 110,000\u00d7g, 4\u00b0 C. for 70 minutes.","The supernatant was poured off without disturbing the pellet and an additional 1 ml of PBS was added to wash the pellet. The volume was increased to maximum volume with the addition of 4.5 mls of PBS and centrifuged at 110,000\u00d7g for 70 minutes at 4\u00b0 C. The supernatant was removed with P-1000 pipette until \u02dc100 \u03bcl of PBS was in the bottom of the tube. The \u02dc90 \u03bcl remaining was removed with P-200 pipette and the pellet collected with the \u02dc10 \u03bcl of PBS remaining by gently pipetting using a P-20 pipette into the microcentrifuge tube. The residual pellet was washed from the bottom of a dry tube with an additional 5 \u03bcl of fresh PBS and collected into microcentrifuge tube and suspended in phosphate buffered saline (PBS) to a concentration of 500 \u03bcg\/ml.","Blood is collected via standard veinpuncture in a 7 ml K2-EDTA tube. The sample is spun at 400g for 10 minutes in a 4\u00b0 C. centrifuge to separate plasma from blood cells (SORVALL Legend RT+ centrifuge). The supernatant (plasma) is transferred by careful pipetting to 15 ml Falcon centrifuge tubes. The plasma is spun at 2,000g for 20 minutes and the supernatant is collected.","For storage, approximately 1 ml of the plasma (supernatant) is aliquoted to a cryovials, placed in dry ice to freeze them and stored in \u221280\u00b0 C. Before vesicle purification, if samples were stored at \u221280\u00b0 C., samples are thawed in a cold water bath for 5 minutes. The samples are mixed end over end by hand to dissipate insoluble material.","In a first prespin, the plasma is diluted with an equal volume of PBS (example, approximately 2 ml of plasma is diluted with 2 ml of PBS). The diluted fluid is transferred to a 15 ml Falcon tube and centrifuged for 30 minutes at 2000\u00d7g at 4\u00b0 C.","For a second prespin, the supernatant (approximately 4 mls) is carefully transferred to a 50 ml Falcon tube and centrifuged at 12,000\u00d7g at 4\u00b0 C. for 45 minutes in a Sorval.","In the isolation step, the supernatant (approximately 2 mls) is carefully transferred to a 5.0 ml ultracentrifuge PA thinwall tube (Sorvall #03127) using a P1000 pipette and filled to maximum volume with an additional 0.5 mls of PBS. The tube is centrifuged for 90 minutes at 110,000\u00d7g at 4\u00b0 C.","In the first wash, the supernatant is poured off without disturbing the pellet. The pellet is resuspended or washed with 1 ml PBS and the tube is filled to maximum volume with an additional 4.5 ml of PBS. The tube is centrifuged at 110,000\u00d7g at 4\u00b0 C. for 70 minutes. A second wash is performed by repeating the same steps.","The vesicles are collected by removing the supernatant with P-1000 pipette until approximately 100 \u03bcl of PBS is in the bottom of the tube. Approximately 90 \u03bcl 1 of the PBS is removed and discarded with P-200 pipette. The pellet and remaining PBS is collected by gentle pipetting using a P-20 pipette. The residual pellet is washed from the bottom of the dry tube with an additional 5 \u03bcl of fresh PBS and collected into a microcentrifuge tube.","This example demonstrates the use of particles coupled to an antibody, where the antibody captures the vesicles. See, e.g., . An antibody, the detector antibody, is directly coupled to a label, and is used to detect a biomarker on the captured vesicle.","First, an antibody-coupled microsphere set is selected (Luminex, Austin, Tex.). The microsphere set can comprise various antibodies, and thus allows multiplexing. The microspheres are resuspended by vortex and sonication for approximately 20 seconds. A Working Microsphere Mixture is prepared by diluting the coupled microsphere stocks to a final concentration of 100 microspheres of each set\/4 in Startblock (Pierce (37538)). 50 \u03bcL of Working Microsphere Mixture is used for each well. Either PBS-1% BSA or PBS-BN (PBS, 1% BSA, 0.05% Azide, pH 7.4) may be used as Assay Buffer.","A 1.2 \u03bcm Millipore filter plate is pre-wet with 100 \u03bcl\/well of PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and aspirated by vacuum manifold. An aliquot of 50 \u03bcl of the Working Microsphere Mixture is dispensed into the appropriate wells of the filter plate (Millipore Multiscreen HTS (MSBVN1250)). A 50 \u03bcl aliquot of standard or sample is dispensed into to the appropriate wells. The filter plate is covered and incubated for 60 minutes at room temperature on a plate shaker. The plate is covered with a sealer, placed on the orbital shaker and set to 900 for 15-30 seconds to re-suspend the beads. Following that the speed is set to 550 for the duration of the incubation.","The supernatant is aspirated by vacuum manifold (less than 5 inches Hg in all aspiration steps). Each well is washed twice with 100 \u03bcl of PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and is aspirated by vacuum manifold. The microspheres are resuspended in 50 \u03bcL of PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))). The PE conjugated detection antibody is diluted to 4 \u03bcg\/mL (or appropriate concentration) in PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))). (Note: 50 \u03bcL of diluted detection antibody is required for each reaction.) A 50 \u03bcl aliquot of the diluted detection antibody is added to each well. The filter plate is covered and incubated for 60 minutes at room temperature on a plate shaker. The filter plate is covered with a sealer, placed on the orbital shaker and set to 900 for 15-30 seconds to re-suspend the beads. Following that the speed is set to 550 for the duration of the incubation. The supernatant is aspirated by vacuum manifold. The wells are washed twice with 100 \u03bcl of PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and aspirated by vacuum manifold. The microspheres are resuspended in 100 \u03bcl of PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))). The microspheres are analyzed on a Luminex analyzer according to the system manual.",{"@attributes":{"id":"p-1174","num":"1176"},"figref":["FIG. 66","FIG. 66A","FIG. 66B","FIG. 66A","FIG. 66B","FIG. 66A"]},"This example demonstrates the use of particles coupled to an antibody, where the antibody captures the vesicles. An antibody, the detector antibody, is biotinylated. A label coupled to streptavidin is used to detect the biomarker.","First, the appropriate antibody-coupled microsphere set is selected (Luminex, Austin, Tex.). The microspheres are resuspended by vortex and sonication for approximately 20 seconds. A Working Microsphere Mixture is prepared by diluting the coupled microsphere stocks to a final concentration of 50 microspheres of each set\/4 in Startblock (Pierce (37538)). (Note: 50 \u03bcl of Working Microsphere Mixture is required for each well.) Beads in Start Block should be blocked for 30 minutes and no more than 1 hour.","A 1.2 \u03bcm Millipore filter plate is pre-wet with 100 \u03bcl\/well of PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and is aspirated by vacuum manifold. A 50 \u03bcl aliquot of the Working Microsphere Mixture is dispensed into the appropriate wells of the filter plate (Millipore Multiscreen HTS (MSBVN1250)). A 50 \u03bcl aliquot of standard or sample is dispensed to the appropriate wells. The filter plate is covered with a seal and is incubated for 60 minutes at room temperature on a plate shaker. The covered filter plate is placed on the orbital shaker and set to 900 for 15-30 seconds to re-suspend the beads. Following that, the speed is set to 550 for the duration of the incubation.","The supernatant is aspirated by a vacuum manifold (less than 5 inches Hg in all aspiration steps). Aspiration can be done with the Pall vacuum manifold. The valve is place in the full off position when the plate is placed on the manifold. To aspirate slowly, the valve is opened to draw the fluid from the wells, which takes approximately 3 seconds for the 100 \u03bcl of sample and beads to be fully aspirated from the well. Once the sample drains, the purge button on the manifold is pressed to release residual vacuum pressure from the plate.","Each well is washed twice with 100 \u03bcl of PBS-1% BSA+Azide (PBS-BN) (Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and is aspirates by vacuum manifold. The microspheres are resuspended in 50 \u03bcl of PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032)))","The biotinylated detection antibody is diluted to 4 \u03bcg\/mL in PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))). (Note: 50 \u03bcl of diluted detection antibody is required for each reaction.) A 50 \u03bcl aliquot of the diluted detection antibody is added to each well.","The filter plate is covered with a sealer and is incubated for 60 minutes at room temperature on a plate shaker. The plate is placed on the orbital shaker and set to 900 for 15-30 seconds to re-suspend the beads. Following that, the speed is set to 550 for the duration of the incubation.","The supernatant is aspirated by vacuum manifold. Aspiration can be done with the Pall vacuum manifold. The valve is place in the full off position when the plate is placed on the manifold. To aspirate slowly, the valve is opened to draw the fluid from the wells, which takes approximately 3 seconds for the 100 ul of sample and beads to be fully aspirated from the well. Once all of the sample is drained, the purge button on the manifold is pressed to release residual vacuum pressure from the plate.","Each well is washed twice with 100 \u03bcl of PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and is aspirated by vacuum manifold. The microspheres are resuspended in 50 \u03bcl of PBS-1% BSA (Sigma (P3688-10PAK+0.05% NaAzide (S8032))).","The streptavidin-R-phycoerythrin reporter (Molecular Probes 1 mg\/ml) is diluted to 4 \u03bcg\/mL in PBS-1% BSA+Azide (PBS-BN). 50 \u03bcl of diluted streptavidin-R-phycoerythrin was used for each reaction. A 50 \u03bcl aliquot of the diluted streptavidin-R-phycoerythrin is added to each well.","The filter plate is covered with a sealer and is incubated for 60 minutes at room temperature on a plate shaker. The plate is placed on the orbital shaker and set to 900 for 15-30 seconds to re-suspend the beads. Following that, the speed is set to 550 for the duration of the incubation.","The supernatant is aspirated by vacuum manifold. Aspiration can be done with the Pall vacuum manifold. The valve is place in the full off position when the plate is placed on the manifold. To aspirate slowly, the valve is opened to draw the fluid from the wells, which takes approximately 3 seconds for the 100 ul of sample and beads to be fully aspirated from the well. Once all of the sample is drained, the purge button on the manifold is pressed to release residual vacuum pressure from the plate.","Each well is washed twice with 100 \u03bcl of PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and is aspirated by vacuum manifold. The microspheres are resuspended in 100 \u03bcl of PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))) and analyzed on the Luminex analyzer according to the system manual.","Antibody Purification Protocol: Antibodies were purified using Protein G resin from Pierce (Protein G spin kit, Product #89979, Pierce, a part of Thermo Scientific, Rockford, Ill.). Micro-chromatography columns made from filtered P-200 tips were used for purification.","One hundred \u03bcl of Protein G resin is loaded with 100 \u03bcl buffer from the Pierce kit to each micro column. After waiting a few minutes to allow the resin to settle down, air pressure is applied with a P-200 Pipettman to drain buffer when needed, ensuring the column is not let to dry. The column is equilibrated with 0.6 ml of Binding Buffer (pH 7.4, 100 mM Phosphate Buffer, 150 mM NaCl; (Pierce, Prod #89979). An antibody is applied to the column (<1 mg of antibody is loaded on the column). The column is washed with 1.5 ml of Binding Buffer. Five tubes (1.5 ml micro centrifuge tubes) are prepared and 10 \u03bcl of neutralization solution (Pierce, Prod #89979) is applied to each tube. The antibody is eluted with the elution buffer from the kit to each of the five tubes, 100 ul for each tube (for a total of 500 \u03bcl). The relative absorbance of each fraction is measured at 280 nm using Nanodrop (Nanodrop 1000 spectrophotometer, Nanodrop, a part of Thermo Scientific, Wilmington, Del.). The fractions with highest OD reading are selected for downstream usage. The samples are dialyzed against 0.25 liters PBS buffer using Pierce Slide-A-Lyzer Dialysis Cassette (Pierce, Product No. 66333, 3 KDa cut off). The buffer is exchanged every 2 hours for minimum three exchanges at 4\u00b0 C. with continuous stirring. The dialyzed samples are then transferred to 1.5 ml microcentifuge tubes, and can be labeled and stored at 4\u00b0 C. (short term) or \u221220\u00b0 C. (long term).","Coupling:","Microspheres are coated with their respective antibodies as listed above according to the following protocol.","The microspheres are protected from prolonged exposure to light throughout this procedure. The stock uncoupled microspheres are resuspended according to the instructions described in the Product Information Sheet provided with the microspheres (xMAP technologies, MicroPlex\u2122 Microspheres, Luminex Corporation, Austin, Tex.). Five \u00d7106 of the stock microspheres are transferred to a USA Scientific 1.5 ml microcentrifuge tube (USA Scientific, Inc., Orlando, Fla.). The stock microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature. The supernatant is removed and the pelleted microspheres are resuspended in 100 \u03bcl of dH2O by vortex and sonication for approximately 20 seconds. The microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature. The supernatant is removed and the washed microspheres are resuspended in 80 \u03bcl of 100 mM Monobasic Sodium Phosphate, pH 6.2 by vortex and sonication (Branson 1510, Branson Ultrasonics Corp., Danbury, Conn.) for approximately 20 seconds. Ten \u03bcl of 50 mg\/ml Sulfo-NHS (Thermo Scientific, Cat#24500) (diluted in dH2O) is added to the microspheres and is mixed gently by vortex. Ten \u03bcl of 50 mg\/ml EDC (Thermo Scientific, Cat#25952-53-8) (diluted in dH2O) is added to the microspheres and gently mixed by vortexing. The microspheres are incubated for 20 minutes at room temperature with gentle mixing by vortex at 10 minute intervals. The activated microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature. The supernatant is removed and the microspheres are resuspended in 250 \u03bcl of 50 mM MES, pH 5.0 (MES, Sigma, Cat# M2933, Sigma-Aldrich, St. Louis, Mo.) by vortex and sonication for approximately 20 seconds. Only PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))) should be used as assay buffer as well as wash buffer. The microspheres are then pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature.","The supernatant is removed and the microspheres are resuspended in 250 \u03bcl of 50 mM MES, pH 5.0 (MES, Sigma, Cat# M2933) by vortex and sonication for approximately 20 seconds. Only PBS-1% BSA+Azide (PBS-BN) ((Sigma (P3688-10PAK+0.05% NaAzide (S8032))) should be used as assay buffer as well as wash buffer. The microspheres are then pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature, thus completing two washes with 50 mM MES, pH 5.0.","The supernatant is removed and the activated and washed microspheres are resuspended in 100 \u03bcl of 50 mM MES, pH 5.0 by vortex and sonication for approximately 20 seconds. Protien in the amount of 125, 25, 5 or 1 \u03bcg is added to the resuspended microspheres. (Note: Titration in the 1 to 125 \u03bcg range can be performed to determine the optimal amount of protein per specific coupling reaction.). The total volume is brought up to 500 \u03bcl with 50 mM MES, pH 5.0. The coupling reaction is mixed by vortex and is incubated for 2 hours with mixing (by rotating on Labquake rotator, Barnstead, Thermo Scientific) at room temperature. The coupled microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature. The supernatant is removed and the pelleted microspheres are resuspended in 5004 of PBS-TBN by vortex and sonication for approximately 20 seconds. (Concentrations can be optimized for specific reagents, assay conditions, level of multiplexing, etc. in use.).","The microspheres are incubated for 30 minutes with mixing (by rotating on Labquake rotator, Barnstead) at room temperature. The coupled microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes at room temperature. The supernatant is removed and the microspheres are resuspended in 1 ml of PBS-TBN by vortex and sonication for approximately 20 seconds. Each time there is the addition of samples, detector antibody or SA-PE the plate is covered with a sealer and light blocker (such as aluminum foil), placed on the orbital shaker and set to 900 for 15-30 seconds to re-suspend the beads. Following that the speed should be set to 550 for the duration of the incubation.","The microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes. The supernatant is removed and the microspheres are resuspended in 1 ml of PBS-TBN by vortex and sonication for approximately 20 seconds. The microspheres are pelleted by microcentrifugation at\u22678000\u00d7g for 1-2 minutes (resulting in a total of two washes with 1 ml PBS-TBN).","Supplies and Equipment:","Pall life sciences Acrodisc, 25 mm syringe filter w\/1.2 um, Versapor membrane (sterile) Part number: 4190; Pierce concentrators 7 ml\/150 K MWCO (molecular weight cut off), Part number: 89922; BD syringe filter, 10 ml, Part number: 305482; Sorvall Legend RT Plus Series Benchtop Centrifuge w 15 ml swinging bucket rotor; PBS, pH 7.4, Sigma cat#P3813-10PAK prepared in Sterile Molecular grade water; Co-polymer 1.7 ml microfuge tubes, USA Scientific, cat#1415-2500. Water used for reagents is Sterile Filtered Molecular grade water (Sigma, cat#W4502). Handling of patient plasma is done in a biosafety hood.","Procedure:","1. Filter procedure for plasma samples\n\n","2. Microvesicle concentration centrifugation protocol\n\n","1. Final volume of concentrated plasma sample\n\n","The samples obtained using methods as described in Example 3 are used in multiplexing assays as described in Examples 4 and 5. The detection antibodies used are CD63, CD9, CD81, B7H3 and EpCam. The capture antibodies used are CD9, PSCA, TNFR, CD63 2X, B7H3, MFG-E8, EpCam 2X, CD63, Rab, CD81, SETAP, PCSA, PSMA, 5T4, Rab IgG (control) and IgG (control), resulting in 100 combinations to be screened ().","Ten prostate cancer patients and 12 normal control patients were screened. Results for the indicated capture and\/or detection antibodies are depicted in  and .  depicts the results of using PCSA capture antibodies (, left graph) or EpCam capture antibodies (, right graph), and detection using one or more detector antibodies. The sensitivity and specificity of the different combinations is depicted in .","Vesicle samples obtained using methods as described in Example 3 are used in multiplexing assays as described in Examples 4 and 5. The detection antibodies used are CD63, CD9, CD81, B7H3 and EpCam. The capture antibodies used are CD9, PSCA, TNFR, CD63 2X, B7H3, MFG-E8, EpCam 2X, CD63, Rab, CD81, STEAP, PCSA, PSMA, 5T4, Rab IgG (control) and IgG (control), resulting in 100 combinations to be screened.","The results are depicted in . The sensitivity of the different combinations is depicted in .","Vesicles isolated as described in Example 2 are used. Approximately 40 ul of the vesicles are incubated with approximately 5 ug (\u02dc50 \u03bcl) of EpCam antibody coated Dynal beads (Invitrogen, Carlsbad, Calif.) and 50 \u03bcl of Starting Block. The vesicles and beads are incubated with shaking for 2 hours at 45\u00b0 C. in a shaking incubator. The tube containing the Dynal beads is placed on the magnetic separator for 1 minute and the supernatant removed. The beads are washed twice and the supernatant removed each time. Wash beads twice, discarding the supernatant each time.","RNA from the bead-bound vesicles of Example 10 was isolated using the Qiagen miRneasy\u2122 kit, (Cat. No. 217061), according to the manufacturer's instructions.","The vesicles are homogenized in QIAzol\u2122 Lysis Reagent (Qiagen Cat. No. 79306). After addition of chloroform, the homogenate is separated into aqueous and organic phases by centrifugation. RNA partitions to the upper, aqueous phase, while DNA partitions to the interphase and proteins to the lower, organic phase or the interphase. The upper, aqueous phase is extracted, and ethanol is added to provide appropriate binding conditions for all RNA molecules from 18 nucleotides (nt) upwards. The sample is then applied to the RNeasy\u2122 Mini spin column, where the total RNA binds to the membrane and phenol and other contaminants are efficiently washed away. High quality RNA is then eluted in RNase-free water.","RNA from the VCAP bead captured vesicles was measured with the Taqman TMPRSS:ERG fusion transcript assay (Kirsten D. Mertz et al. 2007 March; 9(3): 200-206.). RNA from the 22Rv1 bead captured vesicles was measured with the Taqman SPINK1 transcript assay (Scott A. Tomlins et al. 2008 June 13(6):519-528). The GAPDH transcript (control transcript) was also measured for both sets of vesicle RNA.","Higher CT values indicate lower transcript expression. One change in cycle threshold (CT) is equivalent to a 2 fold change, 3 CT difference to a 4 fold change, and so forth, which can be calculated with the following: 2^. This experiment shows a difference in CT of the expression of the fusion transcript TMPRSS:ERG and the equivalent captured with the IgG2 negative control bead (). The same comparison of the SPINK1 transcript in 22RV1 vesicles shows a CT difference of 6.14 for a fold change of 70.5 (). Results with GAPDH were similar.","Blood is collected from subjects (both healthy subjects and subjects with cancer) in EDTA tubes, citrate tubes or in a 10 ml Vacutainer SST plus Blood Collection Tube (BD367985 or BD366643, BD Biosciences). Blood is processed for plasma isolation within 2 h of collection.","Samples are allowed to sit at room temperature for a minimum of 30 min and a max of 2 h. Separation of the clot is accomplished by centrifugation at 1,000-1,300\u00d7g at 4\u00b0 C. for 15-20 min. The serum is removed and dispensed in aliquots of 500 \u03bcl into 500 to 750 \u03bcl cryotubes. Specimens are stored at \u221280\u00b0 C.","At a given sitting, the amount of blood drawn can range from \u02dc20 to \u02dc90 ml. Blood from several EDTA tubes is pooled and transferred to RNase\/DNase-free 50-ml conical tubes (Greiner), and centrifuged at 1,200\u00d7g at room temperature in a Hettich Rotanta 460R benchtop centrifuge for 10 min. Plasma is transferred to a fresh tube, leaving behind a fixed height of 0.5 cm plasma supernatant above the pellet to avoid disturbing the pellet. Plasma is aliquoted, with inversion to mix between each aliquot, and stored at \u221280\u00b0 C.","Four hundred \u03bcl of human plasma or serum is thawed on ice and lysed with an equal volume of 2\u00d7 Denaturing Solution (Ambion). RNA is isolated using the mirVana PARIS kit following the manufacturer's protocol for liquid samples (Ambion), modified such that samples are extracted twice with an equal volume of acid-phenol chloroform (as supplied by the Ambion kit). RNA is eluted with 105 \u03bcl of Ambion elution solution according to the manufacturer's protocol. The average volume of eluate recovered from each column is about 80 \u03bcl.","A scaled-up version of the mirVana PARIS (Ambion) protocol is also used: 10 ml of plasma is thawed on ice, two 5-ml aliquots are transferred to 50-ml tubes, diluted with an equal volume of mirVana PARIS 2\u00d7 Denaturing Solution, mixed thoroughly by vortexing for 30 s and incubated on ice for 5 min. An equal volume (10 ml) of acid\/phenol\/chloroform (Ambion) is then added to each aliquot. The resulting solutions are vortexed for 1 min and spun for 5 min at 8,000 rpm, 20\u00b0 C. in a JA17 rotor. The acid\/phenol\/chloroform extraction is repeated three times. The resulting aqueous volume is mixed thoroughly with 1.25 volumes of 100% molecular-grade ethanol and passed through a mirVana PARIS column in sequential 700 \u03bcl aliquots. The column is washed following the manufacturer's protocol, and RNA is eluted in 105 \u03bcl of elution buffer (95\u00b0 C.). A total of 1.5 \u03bcl of the eluate is quantified by Nanodrop.","A fixed volume of 1.67 \u03bcl of RNA solution from about \u02dc80 \u03bcl-eluate from RNA isolation of a given sample is used as input into the reverse transcription (RT) reaction. For samples in which RNA is isolated from a 400-\u03bcl plasma or serum sample, for example, 1.67 \u03bcl of RNA solution represents the RNA corresponding to (1.67\/80)\u00d7400=8.3 \u03bcl plasma or serum. For generation of standard curves of chemically synthesized RNA oligonucleotides corresponding to known miRNAs, varying dilutions of each oligonucleotide are made in water such that the final input into the RT reaction has a volume of 1.67 \u03bcl. Input RNA is reverse transcribed using the TaqMan miRNA Reverse Transcription Kit and miRNA-specific stem-loop primers (Applied BioSystems) in a small-scale RT reaction comprised of 1.387 \u03bcl of H2O, 0.5 \u03bcl of 10\u00d7 Reverse-Transcription Buffer, 0.063 \u03bcl of RNase-Inhibitor (20 units\/\u03bcl), 0.05 \u03bcl of 100 mM dNTPs with dTTP, 0.33 \u03bcl of Multiscribe Reverse-Transcriptase, and 1.67 \u03bcl of input RNA; components other than the input RNA can be prepared as a larger volume master mix, using a Tetrad2 Peltier Thermal Cycler (BioRad) at 16\u00b0 C. for 30 min, 42\u00b0 C. for 30 min and 85\u00b0 C. for 5 min. Real-time PCR is carried out on an Applied BioSystems 7900HT thermocycler at 95\u00b0 C. for 10 min, followed by 40 cycles of 95\u00b0 C. for 15 s and 60\u00b0 C. for 1 min. Data is analyzed with SDS Relative Quantification Software version 2.2.2 (Applied BioSystems.), with the automatic Ct setting for assigning baseline and threshold for Ct determination.","The protocol can also be modified to include a preamplification step, such as for detecting miRNA. A 1.25-\u03bcl aliquot of undiluted RT product is combined with 3.75 \u03bcl of Preamplification PCR reagents [comprised, per reaction, of 2.5 \u03bcl of TaqMan PreAmp Master Mix (2\u00d7) and 1.25 \u03bcl of 0.2\u00d7 TaqMan miRNA Assay (diluted in TE)] to generate a 5.0-\u03bcl preamplification PCR, which is carried out on a Tetrad2 Peltier Thermal Cycler (BioRad) by heating to 95\u00b0 C. for 10 min, followed by 14 cycles of 95\u00b0 C. for 15 s and 60\u00b0 C. for 4 min. The preamplification PCR product is diluted (by adding 20 \u03bcl of H2O to the 5-\u03bcl preamplification reaction product), following which 2.25 \u03bcl of the diluted material is introduced into the real-time PCR and carried forward as described.","Synthetic single-stranded RNA oligonucleotides corresponding to the mature miRNA sequence (miRBase Release v.10.1) are purchased from Sigma. Synthetic miRNAs are input into the RT reaction over an empirically-derived range of copies to generate standard curves for each of the miRNA TaqMan assays listed above. In general, the lower limit of accurate quantification for each assay is designated based on the minimal number of copies input into an RT reaction that results in a Ct value within the linear range of the standard curve and that is also not equivalent to or higher than a Ct obtained from an RT input of lower copy number. A line is fit to data from each dilution series using Ct values within the linear range, from which y=mln(x)+b equations are derived for quantification of absolute miRNA copies (x) from each sample Ct (y). Absolute copies of miRNA input into the RT reaction are converted to copies of miRNA per microliter plasma (or serum) based on the knowledge that the material input into the RT reaction corresponds to RNA from 2.1% of the total starting volume of plasma [i.e., 1.67 \u03bcl of the total RNA eluate volume (80 \u03bcl on average) is input into the RT reaction]. An example of a synthetic miRNA sequence is for miR-141 which can be obtained commercially such as from Sigma (St. Louis, Mo.).","MicroRNA is extracted from vesicles isolated from patient samples as described herein. See, e.g., Examples 7, 49. Methods for isolation and concentration of vesicles are presented herein. The methods in this Example can also be used to isolate microRNA from patient samples without first isolating vesicles.","Protocol Using Trizol","This protocol uses the QIAzol Lysis Reagent and RNeasy Midi Kit from Qiagen Inc., Valencia Calif. to extract microRNA from concentrated vesicles. The steps of the method comprise:\n\n","Protocol using MagMax","This protocol uses the MagMAX\u2122 RNA Isolation Kit from Applied Biosystems\/Ambion, Austin, Tex. to extract microRNA from concentrated vesicles. The steps of the method comprise:\n\n","MicroRNA levels in a sample can be analyzed using an array format, including both high density and low density arrays. Array analysis can be used to discover differentially expressed in a desired setting, e.g., by analyzing the expression of a plurality of miRs in two samples and performing a statistical analysis to determine which ones are differentially expressed between the samples and can therefore be used in a biosignature. The arrays can also be used to identify a presence or level of one or more microRNAs in a single sample in order to characterize a phenotype by identifying a biosignature in the sample. This Example describes commercially available systems that are used to carry out the methods of the invention.","TaqMan Low Density Array","TaqMan Low Density Array (TLDA) miRNA cards are used to compare expression of miRNA in various sample groups as desired. The miRNA are collected and analyzed using the TaqMan\u00ae MicroRNA Assays and Arrays systems from Applied Biosystems, Foster City, Calif. Applied Biosystems TaqMan\u00ae Human MicroRNA Arrays are used according to the Megaplex\u2122 Pools Quick Reference Card protocol supplied by the manufacturer.","Exiqon mIRCURY LNA microRNA","The Exiqon miRCURY LNA\u2122 Universal RT microRNA PCR Human Panels I and II (Exiqon, Inc, Woburn, Mass.) are used to compare expression of miRNA in various sample groups as desired. The Exiqon 384 well panels include 750 miRs. Samples are normalized to control primers towards synthetic RNA spike-in from Universal cDNA synthesis kit (UniSp6 CP). Results were normalized to inter-plate calibrator probes.","With either system, quality control standards are implemented. Normalized values for each probe across three data sets for each indication are averaged. Probes with an average CV % higher than 20% are not used for analysis. Results are subjected to a paired t-test to find differentially expressed miRs between two sample groups. P-values are corrected with a Benjamini and Hochberg false-discovery rate test. Results are analyzed using GeneSpring software (Agilent Technologies, Inc., Santa Clara, Calif.).","Vesicles were collected by ultracentrifugation from 22Rv1, LNCaP, Vcap and normal plasma (pooled from 16 donors) as described in Examples 1-3. RNA was extracted using the Exiqon miR isolation kit (Cat. Nos. 300110, 300111). Equals amounts of vesicles (30 \u03bcg) were used as determined by BCA assay.","Equal volumes (5 \u03bcl) were put into a reverse-transcription reaction for microRNA. The reverse-transcriptase reactions were diluted in 81 \u03bcl of nuclease-free water and then 9 \u03bcl of this solution was added to each individual miR assay. MiR-629 was found to only be expressed in PCa (prostate cancer) vesicles and was virtually undetectable in normal plasma vesicles. MiR-9 was found to be highly overexpressed (\u02dc704 fold increase over normal as measured by copy number) in all PCa cell lines, and has very low expression in normal plasma vesicles. The top ten differentially expressed miRNAs are shown in .","The bead-bound vesicles of Example 10 were placed in QIAzol\u2122 Lysis Reagent (Qiagen Cat. #79306). An aliquot of 125 fmol of miR-39 was added. The RNA was isolated using the Qiagen miRneasy\u2122 kit, (Cat. #217061), according to the manufacturer's instructions, and eluted in 30 ul RNAse free water.","10 \u03bcl the purified RNA was placed into a pre-amplification reaction for miR-9, miR-141 and miR-629 using a Veriti 96-well thermocycler. A 1:5 dilution of the pre-amplification solution was used to set up a qRT-PCR reaction for miR9 (ABI 4373285), miR-141 (ABI 4373137) and miR-629 (ABI 4380969) as well as miR-39 (ABI 4373455). The results were normalized to the results for each sample.","CD9 coated Dynal beads (Invitrogen, Carlsbad, Calif.) were used instead of EpCam coated beads as in Example 19. Vesicles from prostate cancer patients, LNCaP, or normal purified vesicles were incubated with the CD9 coated beads and the RNA isolated as described in Example 19. The expression of miR-21 and miR-141 was detected by qRT-PCR and the results depicted in .","Six mL of PBS is added to 1 mL of plasma. The sample is then put through a 1.2 micron (\u03bcm) Pall syringe filter directly into a 100 kDa MWCO (Millipore, Billerica, Mass.), 7 ml column with a 150 kDa MWCO (Pierce\u00ae, Rockford, Ill.), 15 ml column with a 100 kDa MWCO (Millipore, Billerica, Mass.), or 20 ml column with a 150 kDa MWCO (Pierce\u00ae, Rockford, Ill.).","The tube is centrifuged for between 60 to 90 minutes until the volume is about 250 \u03bcl. The retentate is collected and PBC added to bring the sample up to 300 \u03bcl. Fifty ul of the sample is then used for further vesicle analysis, such as further described in the examples below.","The vesicle samples obtained using methods as described in Example 21 are used in multiplexing assays as described herein. See, e.g., Examples 31-33. The capture antibodies are CD9, CD63, CD81, PSMA, PCSA, B7H3, and EpCam. The detection antibodies are for biomarkers CD9, CD81, and CD63 or B7H3 and EpCam, as depicted in .","Vesicle samples obtained using methods as described in Example 21, using a 7 mL Pierce\u00ae concentrator with a 150 kDa MWCO (Cat. #89920\/89922) and are used in multiplexing assays as described in herein. See, e.g., Examples 31-33. The capture antibodies are CD9, CD63, CD81, PSMA, PCSA, B7H3, and EpCam. The detection antibodies used are CD63, CD9, and CD81. The results are shown in .","The vesicle samples obtained using methods as described in Example 21, using a 500 \u03bcl column with a 100 kDa MWCO and are used in multiplexing assays as described herein. See, e.g., Examples 31-33. The capture antibodies are CD9, CD63, CD81, PSMA, PCSA, B7H3, and EpCam. The detection antibodies are CD63, CD9, and CD81. The results are shown in . Each figure shows different analysis methods performed using samples from a single patient. In both figures, the graphs depict A) ultracentrifugation purified sample; B) filtered sample C) ultracentrifugation purified sample and 10 ug Vcap and D) filtered sample with 10 ug Vcap.","A variety of filters can be used to remove large debris from the plasma sample as described. Filters ranging in size from 1.2 to 0.8 micron filters provide comparable results, as shown in Tables 18 and 19:",{"@attributes":{"id":"p-1241","num":"1290"},"tables":{"@attributes":{"id":"TABLE-US-00018","num":"00018"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 18"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Filters tested"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"Pore"]},{"entry":["Vendor","Cat No.","Membrane","Size"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["Pall","4190",{},"1.2 uM"]},{"entry":["Millipore","SLAA033SB",{},"0.8 uM"]},{"entry":["Millipore","SLAA033SS",{},"0.8 uM"]},{"entry":["GVS","FJ25ANCCA012CC01",{},"1.2 uM"]},{"entry":["Whatman","6822-1312","GF\/C glass fiber (13 mm)","1.2 uM"]},{"entry":["Whatman","6750-2510","Nylon","1.0 uM"]},{"entry":["Whatman","6781-2510","PES","1.0 uM"]},{"entry":["Whatman","10 462 261","Cellulose Acetate (30 mm)","1.2 uM"]},{"entry":["Whatman","6783-2510","GD glass fiber","1.0 uM"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Plasma samples were filtered and vesicles detected using the biomarkers CD9, PSMA, PCSA, CD63, CD81, B7H3 and EpCam, as indicated in Table 19. Methodology was as described herein. See, e.g., Examples 31-33. Samples were performed in duplicate. The results for the various filters were comparable within each marker.",{"@attributes":{"id":"p-1243","num":"1292"},"tables":{"@attributes":{"id":"TABLE-US-00019","num":"00019"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 19"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"MFI using various filters to isolate vesicles"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"CD9","PSMA","PCSA","CD63","CD81","B7-H3","EpCam"]},{"entry":{"@attributes":{"namest":"1","nameend":"8","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"8"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"21pt","align":"char","char":"."}},{"@attributes":{"colname":"7","colwidth":"28pt","align":"char","char":"."}},{"@attributes":{"colname":"8","colwidth":"28pt","align":"char","char":"."}}],"tbody":{"@attrib
