---
title: Membrane proteins, mechanisms of action and uses thereof
abstract: The invention relates to the atomistic functional understanding of the M2 protein from the influenza A virus. This acid-activated selective proton channel has been the subject of numerous conductance, structural, and computational studies. Previously, little was known at the atomic level about the heart of the functional mechanism of this tetrameric protein, a tetrad of HxxxW residues. The structure of the M2 conductance domain in a lipid bilayer is disclosed and displays the defining features of the native protein that have not been attainable from structures solubilized by detergents. A detailed mechanism for acid activation and proton conductance, involving a strong hydrogen bond between two adjacent histidines and specific interactions with the tryptophan gate, is provided and elucidates many observations on the M2 proton conductance.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=08581584&OS=08581584&RS=08581584
owner: Florida State University Research Foundation
number: 08581584
owner_city: Tallahassee
owner_country: US
publication_date: 20110526
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","GOVERNMENT INTEREST STATEMENT","BACKGROUND","SUMMARY","DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS","Terminology","DESCRIPTION","EXAMPLES","Example 1","Example 2","Example 3","Example 4","Example 5","Example 6"],"p":["This application claims benefit of priority to U.S. Provisional Patent Application No. 61\/348,437 entitled a \u201cAtomistic Mechanism of Influenza A M2 Protein from Structure determined in Lipid Bilayer\u201d filed May 26, 2010, the entire contents and disclosure of which is incorporated herein by reference.","The United States Government has rights in this invention pursuant to funding obtained from the National Institute of Allergy and Infectious Diseases grant AI023007. The spectroscopy was conducted at the National High Magnetic Field Laboratory supported by Cooperative Agreement 0654118 between the NSF Division of Materials Research and the State of Florida.","1. Field of the Invention","The present invention relates generally to viral proteins.","2. Related Art","The M2 protein from the influenza A virus, an acid-activated proton-selective channel, has been the subject of numerous conductance, structural, and computational studies. However, little is known at the atomic level about the heart of the functional mechanism for this tetrameric protein, a His-Trpcluster. The structure of a similar construct of this protein solubilized in detergent micelles as further described in J. R. Schnell, J. J. Chou, 451, 591 (2008), failed to explain the main observations on M2 proton conductance and also generated artifacts that were likely to be a result of the detergent environment used for that structural characterization.","According to one broad aspect, the present invention provides a method comprising the following steps: (a) providing a first solid-state NMR spectrum and a second solid-state NMR spectrum, and (b) detecting a change in resonance of a labeled isotope based on comparing the second solid-state NMR spectrum to the first solid-state NMR spectrum to thereby determine that a drug candidate has bound to a histidine tetrad of a viral protein, wherein second solid-state NMR spectrum is for a first uniformly aligned sample of the viral protein, and wherein the second solid-state spectrum is for a second uniformly aligned sample of the viral protein after being treated with the drug candidate.","Where the meaning of terms departs from the commonly used meaning of the term, applicant intends to utilize the terminology provided below, unless specifically indicated.","For purposes of the present invention, a value or property is \u201cbased\u201d on a particular value, property, the satisfaction of a condition, or other factor, if that value is derived by performing a mathematical calculation or logical decision using that value, property or other factor.","For purposes of the present invention, the term \u201cbind,\u201d the term \u201cbinding\u201d and the term \u201cbound\u201d refers to any type of chemical or physical binding, which includes but is not limited to covalent binding, hydrogen binding, electrostatic binding, biological tethers, transmembrane attachment, cell surface attachment and expression.","For purposes of the present invention, the term \u201cbiological sample\u201d and the term \u201cbiological specimen\u201d refers to either a part or the whole of a human, animal, microbe or plant in vitro or in vivo. The term includes but is not limited to material of human, animal, microbe or plant origin such as human, animal, microbial or plant tissue sections, cell or tissue cultures, suspension of human, animal, microbial or plant cells or isolated parts thereof, human or animal biopsies, blood samples, cell-containing fluids and secretion.","For purposes of the present invention, the term \u201cchannel protein\u201d or \u201cion-channel protein\u201d refers to pore-forming proteins that help establish and control the small voltage gradient across the plasma membrane of cells by allowing the flow of ions down their electrochemical gradient. They are present in the membranes that surround all biological cells. A \u201cproton channel protein\u201d refers to a proton-selective ion channel protein.","For purposes of the present invention, the term \u201cdrug candidate\u201d refers to a naturally occurring or synthetic compound or a combination thereof that is identified to act on a particular target. A target generally refers to a naturally existing cellular or molecular structure involved in the pathology of interest. A good drug candidate exhibits features like increased activity against a chosen target, reduced activity against unrelated targets and improved druglikeness properties. Drug candidates include but are not limited to natural products, plant-derived extracts, microbial metabolites, agents from marine invertebrates, compounds generated through combinatorial chemistry, synthetic or man-made compounds, semi-synthetic derivatives of natural products or small molecules.","For purposes of the present invention, the term \u201cdrug\u201d means any compound intended for use in the diagnosis, cure, mitigation, treatment, or prevention of disease in a human subject or other animals.","For purposes of the present invention, the term \u201cepitope\u201d refers to the smallest part of an antigen moiety recognizable by the combining site of an immunoglobulin.","For purposes of the present invention, the term \u201cexogenous material\u201d refers to material that originates outside the organism of concern or material that may be isolated from a organism, manipulated to any extent externally and then reintroduced into its natural environment or the environment from which it was isolated. Exogenous material includes but is not limited to nucleic acids, proteins, polymeric compounds, particulate matter and artificially synthesized material. For example, \u201cexogenous nucleic acid\u201d refers to any nucleic acid, DNA or RNA or fragments thereof, either single or double stranded, that originates outside of the organism of concern or was isolated from the organism, modified and reintroduced into the organism. Exogenous DNA present in a host cell may be derived from a source organism, cloned into a vector and then introduced into a host cell.","For purposes of the present invention, the term \u201cimmune response\u201d refers to a specific response of the immune system of an animal to antigen or immunogen. Immune response may include the production of antibodies and cellular immunity.","For purposes of the present invention, the term \u201cimmunity\u201d refers to a state of resistance of a subject animal including a human to an infecting organism or substance. It will be understood that an infecting organism or substance is defined broadly and includes parasites, toxic substances, cancer cells and other cells as well as bacteria and viruses. A \u201cTherapeutically Effective Immunization Course\u201d (see below for definition) will produce the immune response.","For purposes of the present invention, the term \u201cimmunization conditions\u201d refers to factors that affect an immune response including the amount and kind of immunogen or adjuvant delivered to a subject animal including a human, method of delivery, number of inoculations, interval of inoculations, the type of subject animal and its condition. \u201cVaccine\u201d refers to pharmaceutical formulations able to induce immunity.","For purposes of the present invention, the term \u201cimmunization dose\u201d refers to the amount of antigen or immunogen needed to precipitate an immune response. This amount will vary with the presence and effectiveness of various adjuvants. This amount will vary with the animal and the antigen, immunogen and\/or adjuvant but will generally be between about 0.1 \u03bcg\/ml or less and about 100 \u03bcg per inoculation. The immunization dose is easily determined by methods well known to those skilled in the art, such as by conducting statistically valid host animal immunization and challenge studies as described: for example, Manual of Clinical Immunology, H. R. Rose and H. Friedman, American Society for Microbiology, Washington, D.C. (1980), tithe entire contents and disclosures of which are incorporated herein by reference. In some instances, several immunization doses including booster doses may administered to provide immunity, and, For purposes of the present invention such a course of treatment is collectively referred to as \u201cTherapeutically Effective Immunization Course\u201d.","For purposes of the present invention, the term \u201cimmunogen\u201d and the term \u201cimmunogenic\u201d refers to a substance or material (including antigens) that is able to induce an immune response alone or in conjunction with an adjuvant. Both natural and synthetic substances may be immunogens. An immunogen is generally a protein, peptide, polysaccharide, nucleoprotein, lipoprotein, synthetic polypeptide, or hapten linked to a protein, peptide, polysaccharide, nucleoprotein, lipoprotein or synthetic polypeptide or other bacterial, viral or protozoal fractions. It will be understood that \u201cimmunogen\u201d or a composition that is \u201cimmunogenic\u201d includes substances (e.g., small peptides) that do not generate an immune response (or generate only a therapeutically ineffective immune response) unless associated with an adjuvant. For purposes of the present invention, such immunogens are referred to as \u201cadjuvant-obligatory\u201d immunogens.","For purposes of the present invention, the term \u201cimmunogenic amount\u201d is an amount of an antigen preparation of interest or amount of a biological toxin that elicits a clinically detectable protective response in an animal.","For purposes of the present invention, the term \u201cisotope\u201d refers to variants of atoms of a particular chemical element, which have differing numbers of neutrons. A number of isotopes may be used in NMR studies which include but are not limited to H, H, He, Li, Li, Be, B, B, C, N, N, O, F, Ne, Na, Mg, Al, Si, P, S, Cl, Cl etc.","For purposes of the present invention, the term \u201clabel\u201d or \u201cmarker\u201d is a modification of a compound (e.g., a ligand or receptor) that enables the user to specifically detect the labeled compound in the presence of unlabeled compounds. For example, one or more atoms within the compound may be labeled with radioactive isotopes. Alternatively, labels may provide antigenic determinants, nucleic acids available for hybridization, altered fluorescence-polarization or altered light-scattering. Still other markers include those that are chromogenic, fluorescent, chemiluminescent or electrochemically detectable. Other methods available to label a ligand or receptor will be readily apparent to those skilled in the art.","For purposes of the present invention, the term \u201cnon-naturally occurring\u201d or \u201cisolated\u201d refers to the component of interest being at least substantially free from at least one other component with which it is naturally associated in nature and as found in nature.","For purposes of the present invention, the term \u201cpeptide-like\u201d refers to short chain peptides as well as proteins, lipoproteins and glycoproteins, but will also, for convenience, include non-proteinaceous molecules, for example, amino acid-containing molecules. In certain embodiments, the peptide-like therapeutic agent may additionally comprise vitamins, steroids, azidothymidine, and free primaquine in addition to other agents. One useful class of peptides is immunomodulators such as interleukins, colony stimulating factors and interferons. Another useful class of proteins is antigens and immunogens such as are used in vaccines.","For purposes of the present invention, the term \u201cpurified\u201d refers to the component in a relatively pure state\u2014e.g. at least about 90% pure, or at least about 95% pure or at least about 98% pure.","For purposes of the present invention, the term \u201csmall molecule\u201d refers to a low molecular weight organic compound which is by definition not a polymer. The term small molecule, especially within the field of pharmacology, is usually restricted to a molecule that also binds with high affinity to a biopolymer such as protein, nucleic acid, or polysaccharide and in addition alters the activity or function of the biopolymer. The upper molecular weight limit for a small molecule may be approximately 800 Daltons which allows for the possibility to rapidly diffuse across cell membranes so that they can reach intracellular sites of action. Small molecules can have a variety of biological functions, serving as cell signaling molecules, as tools in molecular biology, as drugs in medicine, as pesticides in farming, and in many other roles. Small molecules may include but are not limited to compounds that are natural (such as secondary metabolites), artificial (such as antiviral drugs), constituent monomers of biopolymers such as ribo- or deoxyribonucleotides, amino acids, and monosaccharides, very small oligomers such as dinucleotides, peptides such as the antioxidant glutathione, and disaccharides such as sucrose.","For purposes of the present invention, the term \u201cSolid-state NMR\u201d refers to solid state NMR spectroscopy which is a kind of nuclear magnetic resonance (NMR) spectroscopy, characterized by the presence of anisotropic (directionally dependent) interactions. One of skill in the art is aware of the theory and applicability of NMR spectroscopy. Recent innovation within NMR spectroscopy has been within the field of protein NMR, which has become a very important technique in structural biology. A goal of these investigations is to obtain high resolution 3-dimensional structures of the protein, similar to what can be achieved by X-ray crystallography. In contrast to X-ray crystallography, NMR is used primarily on relatively small proteins though technical advances allow ever larger structures to be solved. NMR spectroscopy is advantageous to obtain high resolution information on partially or wholly intrinsically unstructured proteins. It is now a common tool for the determination of Conformation Activity Relationships where the structure before and after interaction with, for example, a drug candidate is compared to its known biochemical activity. Because of the much higher number of atoms present in a protein molecule in comparison with a small organic compound, the basic 1D spectra become crowded with overlapping signals to an extent where direct spectra analysis becomes unreasonable. Therefore, multidimensional (2, 3 or 4D) experiments have been devised to deal with this problem. To facilitate these experiments, it is desirable to isotopically label the protein with C and N because the predominant naturally occurring isotope C is not NMR-active, whereas the nuclear quadrupole moment of the predominant naturally occurring N isotope prevents high resolution information to be obtained from this nitrogen isotope. Aspects of solid state NMR are further described in U.S. Pat. Nos. 7,754,438 and 7,678,546, the contents and disclosure of which is hereby incorporated by reference in its entirety.","For purposes of the present invention, the term \u201cuniformly aligned sample\u201d means a sample in which each molecule in the sample will have the same orientation with respect to the magnetic field axis of the NMR spectrometer. In this way the orientation dependent spin interactions such as anisotropic chemical shifts, dipolar and quadrupolar interactions can be observed. The orientations of the individual atomic sites in the protein reflect the orientation of that particular site with respect to the magnetic field. For membrane proteins the plane of the bilayer and hence the orientation of the membrane protein spanning the membrane can be oriented with the bilayer normal either parallel or perpendicular to the magnetic field.","For purposes of the present invention, the term \u201cviral protein\u201d refers to proteins for viruses which include but are not limited to the following virus families: Retroviridae (for example, human immunodeficiency virus (HIV), human T-cell leukemia viruses; Picornaviridae (for example, polio virus, hepatitis A virus, hepatitis C virus, enteroviruses, human coxsackie viruses, rhinoviruses, echoviruses, foot-and-mouth disease virus); Caliciviridae (such as strains that cause gastroenteritis, including Norwalk virus); Togaviridae (for example, alphaviruses (including chikungunya virus, equine encephalitis viruses, Simliki Forest virus, Sindbis virus, Ross River virus), rubella viruses); Flaviridae (for example, dengue viruses, yellow fever viruses, West Nile virus, St. Louis encephalitis virus, Japanese encephalitis virus, Powassan virus and other encephalitis viruses); Coronaviridae (for example, coronaviruses, severe acute respiratory syndrome (SARS) virus; Rhabdoviridae (for example, vesicular stomatitis viruses, rabies viruses); Filoviridae (for example, Ebola virus, Marburg virus); Paramyxoviridae (for example, influenza viruses, parainfluenza viruses, mumps virus, measles virus, respiratory syncytial virus); Orthomyxoviridae (for example, influenza viruses); Bunyaviridae (for example, Hantaan viruses, Sin Nombre virus, Rift Valley fever virus, bunya viruses, phleboviruses and Nairo viruses); Arenaviridae (such as Lassa fever virus and other hemorrhagic fever viruses, Machupo virus, Junin virus); Reoviridae (e.g., reoviruses, orbiviurses, rotaviruses); Birnaviridae; Hepadnaviridae (hepatitis B virus); Parvoviridae (parvoviruses); Papovaviridae (papilloma viruses, polyoma viruses, BK-virus); Adenoviridae (adenoviruses); Herpesviridae (herpes simplex virus (HSV)-1 and HSV-2; cytomegalovirus; Epstein-Barr virus; varicella zoster virus; and other herpes viruses, including HSV-6); Poxyiridae (variola viruses, vaccinia viruses, pox viruses); and Iridoviridae (such as African swine fever virus); Astroviridae; and unclassified viruses (for example, the etiological agents of spongiform encephalopathies, the agent of delta hepatitis (thought to be a defective satellite of hepatitis B virus).","For purposes of the present invention, the terms \u201cHxxxW quartet\u201d, HxxxW tetrad, \u201cthe histidine tetrad\u201d and \u201cHis-Trpcluster\u201d are all used interchangeably and refer to the novel histidine chemistry that was identified in the M2 protein of Influenza A as described in this disclosure and in Sharma et al. 22; 330 (6003):509-12 (2010). These terms also encompass chemistries in proteins that include but are not limited to viral proteins that are substantially similar to the histidine tetrad in terms of sequence and function.","For purposes of the present invention, the terms \u201cprotein,\u201d \u201cpeptide,\u201d \u201cpolypeptide,\u201d and \u201camino acid sequence\u201d are used interchangeably herein to refer to polymers of amino acid residues of any length. The polymer may be linear or branched, it may comprise modified amino acids or amino acid analogs, and it may be interrupted by chemical moieties other than amino acids. The terms also encompass an amino acid polymer that has been modified naturally or by intervention; for example disulfide bond formation, glycosylation, lipidation, acetylation, phosphorylation, or any other manipulation or modification, such as conjugation with a labeling or bioactive component.","The present invention may be understood more readily by reference to the following detailed description of specific embodiments included herein. Although the present invention has been described with reference to specific details of certain embodiments thereof, it is not intended that such details should be regarded as limitations upon the scope of the invention. The entire text of the references mentioned herein and references cited within them are incorporated herein by reference in their entireties including U.S. Provisional Patent Application No. 61\/348,437 entitled a \u201cAtomistic Mechanism of Influenza A M2 Protein from Structure determined in Lipid Bilayer\u201d filed May 26, 2010.","Influenza A and B viruses cause a highly contagious respiratory disease in humans resulting in approximately 36,000 deaths in the United States annually (Wright, P. F., and Webster, R. G. 2001. Orthomyxoviruses. 1533-79 In: , eds. D. M. Knipe and P. M. Howley, Lippincott Williams & Wilkins, Philadelphia and Prevention. 2005. Background on Influenza on the website of the Center for Disease Control and Prevention).","The M2 protein has a critical role to play in the life cycle of the influenza A virus. It is located in the viral envelope. It allows hydrogen ions to enter the viral particle (virion) from the endosome, thus lowering the pH inside of the virus, which causes dissociation of the viral matrix protein M1 from the ribonucleoprotein RNP. This is a crucial step in uncoating of the virus and exposing its content to the cytoplasm of the host cell. Hence Proton conductance by the M2 protein (with 97 residues per monomer) in influenza A is essential for viral replication as described in M. Takeda, A. Pekosz, K. Shuck, L. H. Pinto, R. A. Lamb, 76, 1391 (2002).","Antiviral drugs amantadine and rimantadine can inhibit the function of the M2 channel and blocks the virus from taking over the host cell. The drug molecule binds to the transmembrane region and sterically blocks the channel. This stops the protons from entering the virion, which subsequently does not disintegrate. However, the M2 gene is susceptible to mutations. A M2 mutation, Ser\u2192Asn, in recent flu seasons and in the recent H1N1 swine flu pandemic renders the viruses resistant to amantadine and rimantadine as described in L. Gubareva et al.; Centers for Disease Control and Prevention (CDC), 58, 1 (2009). As the mutations are relatively frequent this may lead to emergence of resistant strains.","An important aspect of understanding protein function is obtaining accurate structural information at an atomic level. Structural information was previously obtained on the M2 protein when a construct (residues 18-60) was solubilized in detergent micelles as described in J. R. Schnell, J. J. Chou, 451, 591 (2008). However the information obtained failed to explain all the main observations on M2 proton conductance and there were artifacts in the structure that were likely to be resultant of the detergent environment used for the characterization.","Previous structural determinations of M2 focused primarily on its transmembrane (TM) domain, residues 26 to 46 as further described in K. Nishimura, S. Kim, L. Zhang, T. A. Cross, 41, 13170 (2002); J. Hu et al., 92, 4335 (2007); A. L. Stouffer et al., 451, 596 (2008); S. D. Cady, M. Hong, . U.S.A. 105, 1483; S. D. Cady et al., 463, 689 (2010) and R. Acharya et at., . U.S.A. 107, 15075 (2010).","Although the TM domain is capable of conducting protons, residues 47 to 62 following the TM domain are essential for the functional integrity of the channel. Oocyte assays showed that truncations of the post-TM sequence result in reduced conductance as referred to in C. Ma et al., . U.S.A. 106, 12283 (2009). An embodiment of the invention is the structure of the \u201cconductance\u201d domain, consisting of residues 22 to 62, which in liposomes conducts protons at a rate comparable to that of the full-length protein in cell membranes as described in J. A. Mould et al., 275, 8592 (2000) and T. I. Lin, C. Schroeder, 75, 3647 (2001), is amantadine-sensitive (as indicated in ). This structure, solved in uniformly aligned 1,2-dioleoyl-sn-glycero-3-phosphatidylcholine: 1,2-dioleoyl-sn-glycero-3-phosphoethanolamine bilayers by solid-state nuclear magnetic resonance (NMR) at pH 7.5 and 30\u00b0 C., shows striking differences from the structure of a similar construct (residues 18 to 60) solubilized in detergent micelles as described in J. R. Schnell, J. J. Chou, 451, 591 (2008).","In one embodiment, the present invention provides the structure of the M2 conductance domain (residues 22 to 62) in a lipid bilayer, which displays the defining features of the native protein that have not been attainable from structures solubilized by detergents.","In another embodiment, the present invention provides a mechanism by which the tetrameric His-Trpcluster guides protons through the channel by forming and breaking hydrogen bonds between adjacent pairs of histidines and through specific interactions of the histidines with the tryptophan gate.","In one embodiment the present invention relates to atomic structural information of viral proteins and the use of such information to identify and design compounds, such as, but not limited to peptides, antigens, antibodies, small molecule inhibitors or activators and synthetic compounds that when administered affect viral function and may be used in diagnostic, pharmaceutical, immunogenic and immunological compositions.","The structure is a tetramer as seen in the SDS-PAGE gel in , with each monomer comprising two helices (). Residues 26 to 46 form a kinked TM helix, with the N-terminal and C-terminal halves having tilt angles of \u221232\u00b0 and \u221222\u00b0 from the bilayer normal, respectively. The kink in the TM helix occurs around Gly, similar to the kinked TM-domain structure in the presence of amantadine as referred to in J. Hu et al., 92, 4335 (2007). The amphipathic helix (residues 48 to 58) has a tilt angle of 105\u00b0, similar to that observed for the full-length protein as suggested in C. Tian, P. F. Gao, L. H. Pinto, R. A. Lamb, T. A. Cross, 12, 2597 (2003), and resides in the lipid interfacial region (). The turn between the TM and amphipathic helices is tight and rigid, as indicated by substantial anisotropic spin interactions for Leuand Phe(their resonances lie close to the TM and amphipathic helical resonance patterns, respectively; see ). The result is a structural base formed by the four amphipathic helices that stabilizes the tetramer.","The pore formed by the TM-helix bundle is lined by Val, Ser, Gly, His, Trp, Asp, and Arg, which include all of the polar residues of the TM sequence. The pore is sealed by the TM helices () and constricted by Valat the N-terminal entrance and by Trpat the C-terminal exit (). The gating role of Trphas long been recognized as described in Y. Tang, F. Zaitseva, R. A. Lamb, L. H. Pinto, 277, 39880 (2002), and recently Valwas proposed to form a secondary gate as suggested in M. Yi, T. A. Cross, H. X. Zhou, 112, 7977 (2008). An important cavity between Valand Hispresents an amantadine-binding site as described in K. Nishimura, S. Kim, L. Zhang, T. A. Cross, 41, 13170 (2002); S. D. Cady et al., 463, 689 (2010) and M. Yi, T. A. Cross, H. X. Zhou, 112, 7977 (2008). This drug-binding site is eliminated in a structure solubilized in detergent micelles as described in J. R. Schnell, J. J. Chou, 451, 591 (2008), because of a much smaller tilt angle of the TM helices as referred to in T. A. Cross, M. Sharma, M. Yi, H. X. Zhou, 10.1016\/j.tibs.2010.07.005 (2010).","The linewidths of the NMR spectra () are much narrower for the conductance domain than for the TM domain as described in C. Li, H. Qin, F. P. Gao, T. A. Cross, 1768, 3162 (2007), indicating substantially reduced conformational heterogeneity and higher stability (see also ). In the structure determined here, numerous nonpolar residues of the amphipathic helices extend the hydrophobic interactions interlinking the monomers, with their close approach facilitated by the small Gly(). In particular, Pheinteracts with Pheand Leuof an adjacent monomer and Pheinteracts with Leuof another adjacent monomer.","The starting residues of the amphipathic helix, Pheand Phe, are a sequence motif known to signal association of the helix with a lipid bilayer as referred to in T. L. Lau, V. Dua, T. S. Ulmer, 283, 16162 (2008). The burial of the hydrophobic portion of the amphipathic helix in the tight internonomer interface is consistent with hydrogen-deuterium exchange data showing it to be the slowest-exchanging region for the full-length protein in a lipid bilayer as described in C. Tian, P. F. Gao, L. H. Pinto, R. A. Lamb, T. A. Cross, 12, 2597 (2003). Furthermore, the Serhydroxyl here, which in the native protein is a palmitoylated Cysresidue, is located at an appropriate depth in the bilayer (at the level of the glycerol backbone; see ) for tethering the palmitic acid. A third native-like aspect of the amphipathic helix is the outward projection of the charged residues Lys, Arg, Lysand Arg(), which conforms to the \u201cpositive inside rule\u201d such that M2 interacts favorably with negatively charged lipids in native membranes. The C termini of the amphipathic helices are situated to allow for the subsequent residues of the full-length protein to form the tetrameric MI binding domain. Contrary to the lipid interfacial location determined here, the detergent-solubilized structure has the four amphipathic helices forming a bundle in the bulk aqueous solution where the amides fully exchange with deuterium as referred to in J. R. Schnell, J. J. Chou, Nature 451, 591 (2008).","On the external surface at the C terminus of the TM helix, a hydrophobic pocket, with the Aspside chain at the bottom, has been described as a binding site for rimantadine as referred to in J. R. Schnell, J. J. Chou, Nature 451, 591 (2008). In the structure of the invention, the large tilt of the TM helices widens the hydrophobic pocket, which is filled by the side chains of Ileand Phein the amphipathic helix, preventing accessibility to the Aspside chain from the exterior (). Consequently, the formation of a rimantadine-binding site on the protein exterior is likely an artifact of the detergent environment used for that structural characterization.","The heart of acid activation and proton conductance in M2 is the tetrameric His-Trpcluster, referred to here as the HxxxW quartet as referred to in Y. Tang, F. Zaitseva, R. A. Lamb, L. H. Pinto, 277, 39880 (2002), P. Venkataraman, R. A. Lamb, L. H. Pinto, 280, 21463 (2005) and J. Hu et al., . U.S.A. 103, 6865 (2006). The pKa values for the Hisresidues in the TM domain solubilized in a lipid bilayer were determined as 8.2, 8.2, 6.3, and <5.0 as described in J. Hu et al., Proc. Natl. Acad. Sci. U.S.A. 103, 6865 (2006). At pH 7.5 used in an embodiment of this invention, the histidine tetrad is doubly protonated; each of these two protons is shared between the Nof one histidine and the Nof an adjacent histidine, giving rise to substantial downfield N chemical shifts and resonance broadening for the protonated sites as referred to in J. Hu et al., . U.S.A. 103, 6865 (2006), indicative of a strong hydrogen bond as described in X.-j. Song, A. E. McDermott, 39, 537 (2001) and X.-j. Song, C. M. Rienstra, A. E. McDermott, 39, 530 (2001).","The structure of the histidine tetrad as a pair of imidazole-imidazolium dimers is shown in . In each dimer, the shared proton is collinear with the Nand Natoms (); the two imidazole rings are within the confines of the backbones, to be nearly parallel to each other\u2014a less energetically favorable situation than the perpendicular alignment of the rings in imidazole-imidazolium crystals and in computational studies as described in A. Quick, D. J. Williams, 54, 2465 (1976); J. A. Krause, P. W. Baures, D. S. Eggleston, 47, 506 (1991) and W. Tatara, M. J. Wojcik, J. Lindgren, M. Probst, 107, 7827 (2003).","The two Nand two Nsites of the histidine tetrad not involved in the strong hydrogen bonds are protonated and project toward the C-terminal side (). The Nprotons interact with the indoles of the Trpresidues (), and the two Nprotons form hydrogen bonds with their respective histidine backbone carbonyl oxygens (). Therefore, in this \u201chistidine-locked\u201d state of the HxxxW quartet, none of the imidazole N\u2014H protons can be released to the C-terminal pore, resulting in a completely blocked channel. Furthermore, the only imidazole nitrogens available for additional protonation are the sites involved in the strong hydrogen bonds; acceptance and release of protons from the N-terminal pore by these sites, coupled with 90\u00b0 side-chain \u03c72 rotations, would allow the imidazole-imidazolium dimers to exchange partners (). That the NMR data show a symmetric average structure suggests that such exchange occurs on a submillisecond time scale.","The structure of the HxxxW quartet at neutral pH suggests a detailed mechanism for acid activation and proton conductance (). Under acidic conditions in the viral exterior, a hydronium ion in the N-terminal pore attacks one of the imidazole-imidazolium dimers. In the resulting \u201cactivated\u201d state, the triply protonated histidine tetrad is stabilized by a hydrogen bond with water at the newly exposed Nsite on the N-terminal side and an additional cation-n interaction with a Trpresidue at the Nsite on the C-terminal side. Strong cation-\u03c0 interactions between Hisand Trpwere observed by Raman spectroscopy in the TM domain under acidic conditions as described in A. Okada, T. Miura, H. Takeuchi, 40, 6053 (2001).","These interactions protect the protons on the C-terminal side from water access. Conformational fluctuations of the helical backbones [in particular, a change in helix kink around Glyas referred to in M. Yi, T. A. Cross, H, X. Zhou, . U.S.A. 106, 13311 (2009)] and motion of the Trpside chain could lead to occasional breaking of this cation-\u03c0 interaction. In the resulting \u201cconducting\u201d state, the Nproton becomes exposed to water on the C-terminal side, allowing it to be released to the C-terminal pore. Upon proton release, the histidine-locked state is restored, ready for another round of proton uptake from the N-terminal pore and proton release to the C-terminal pore. In each round of proton conductance, the changes among the histidine-locked, activated, and conducting states of the HxxxW quartet can be accomplished by rotations (<45\u00b0 change in \u03c7angle) of the Hisand Trpside chains, which are much smaller than those envisioned previously as suggested in L. H. Pinto et al., . U.S.A. 94, 11301 (1997), and even less than those needed for the imidazole-imidazolium dimers to exchange partners ().","This mechanism is consistent with many M2 proton conductance observations. The permeant proton is shuttled through the pore via the histidine tetrad, at one point being shared between Nand Nof adjacent histidines. No other cations can make use of this mechanism, which explains why M2 is proton-selective. With the permeant proton obligatorily binding to and then unbinding from an internal site (i.e., the histidine tetrad), the proton flux is predicted to saturate at a moderate pH on the N-terminal side as described in M. Yi, T. A. Cross, H, X. Zhou, . U.S.A. 106, 13311 (2009), which is consistent with conductance observations referred to in I. V. Chizhmakov et al., 494, 329 (1996). Such a permeation model also predicts that the transition to saturation occurs at a pH close to the histidine-tetrad pKa for binding or unbinding the permeant proton.","Indeed, the transition is observed to occur around pH 6 as described in I. V. Chizhmakov et al., 494, 329 (1996), close to the third pKa of the histidine tetrad. Without the histidines, as in the H37A mutant, the proton flux would lose pH dependence, as observed previously in P. Venkataraman, R. A. Lamb, L. H. Pinto, 280, 21463 (2005). Another distinguishing feature of the M2 proton channel is its low conductance, at \u02dc100 protons per tetramer per second () as referred to in J. A. Mould et al., 275, 8592 (2000) and T. I. Lin, C. Schroeder, 75, 3647 (2001). Upon acid activation, the HxxxW quartet is primarily in the activated state. Only when the Trpgate opens occasionally to form the conducting state is the proton able to be released to the C-terminal pore, thus explaining the low conductance.","In the mechanism of this invention, the histidine tetrad senses only acidification of the N-terminal side. This provides an explanation for an observation of Chizhmakov et al. (I. V. Chizhmakov et al., 546, 427 (2003)) when a positive voltage is applied to drive protons outward in M2-transformed MEL cells. A step increase in the bathing buffer pH from 6 to 8 (with the intracellular pH held constant at pH 6) produced a brief increase in outward current, as expected for the added driving force by the pH gradient. The brief increase can be attributed to the release of protons from the triply protonated histidine tetrad to the N-terminal pore (as illustrated by the back arrow from the activated state to the histidine-locked state in ) when the pH there is suddenly increased from 6 to 8. With these protons released, the histidine tetrad then becomes doubly protonated and the tryptophan gate becomes closed; however, the outward current quickly decayed to a level lower even than that before the pH increase. In the structure related to the invention, when the pH in the N-terminal side is 8, the HxxxW quartet is stabilized in the histidine-locked state and the Trpgate prevents excess protons in the C-terminal pore from activating the histidine tetrad. Upon removal of the Trpgate (e.g., by a W41A mutation), a substantial outward current would be produced, as was observed in Y. Tang, F. Zaitseva, R. A. Lamb, L. H. Pinto, 277, 39880 (2002).","Another embodiment of the invention relates to the observation that the M2 proton channel can be blocked by Cu (through Hiscoordination) applied extracellularly, but not intracellularly as referred to in Y. Tang, F. Zaitseva, R. A. Lamb, L. H. Pinto, 277, 39880 (2002). Again, the triply protonated histidine tetrad stays predominantly in the activated state, in which the Trpgate blocks Cu access to Hisfrom the C-terminal side. However, the W41A mutation opens that access as described in Y. Tang, F. Zaitseva, R. A. Lamb, L. H. Pinto, 277, 39880 (2002).","The structure of the M2 conductance domain solved in liquid crystalline bilayers is consistent which a mechanism for acid activation and proton conductance shown in . This mechanism takes advantage of conformational flexibility, both in the backbones and in the side chains, which arises in part from the weak interactions that stabilize membrane proteins. M2 appears to use the unique chemistry of the HxxxW quartet to shepherd protons through the channel.","As used herein the terms \u201cnucleotide sequences\u201d and \u201cnucleic acid sequences\u201d refer to deoxyribonucleic acid (DNA) or ribonucleic acid (RNA) sequences, including, without limitation, messenger RNA (mRNA), DNA\/RNA hybrids, or synthetic nucleic acids. The nucleic acid can be single-stranded, or partially or completely double-stranded (duplex). Duplex nucleic acids can be homoduplex or heteroduplex.","As used herein the term \u201ctransgene\u201d may be used to refer to \u201crecombinant\u201d nucleotide sequences that may be associated with the present invention. The term \u201crecombinant\u201d means a nucleotide sequence that has been manipulated \u201cby man\u201d and that does not occur in nature, or is linked to another nucleotide sequence or found in a different arrangement in nature. It is understood that manipulated \u201cby man\u201d means manipulated by some artificial means, including by use of machines, codon optimization, restriction enzymes, etc.","For example, in one embodiment the nucleotide sequences may be mutated such that the activity of the encoded proteins in vivo is abrogated. In another embodiment the nucleotide sequences may be codon optimized: for example, the codons may be optimized for human use. In preferred embodiments the nucleotide sequences of the invention are both mutated to abrogate the normal in vivo function of the encoded proteins, and codon optimized for human use.","As regards codon optimization, the nucleic acid molecules associated with the invention have a nucleotide sequence that may encode the antigens and can be designed to employ codons that are used in the genes of the subject in which the antigen is to be produced. In a preferred embodiment, the codons used are \u201chumanized\u201d codons, i.e., the codons are those that appear frequently in highly expressed human genes as described in Andre et al., \u201cIncreased immune response elicited by DNA vaccination with a synthetic gp120 sequence with optimized codon usage,\u201d 72:1497-1503 (1998). Any suitable method of codon optimization may be used. Such methods, and the selection of such methods, are well known to those of skill in the art. In addition, there are several companies that will optimize codons of sequences, such as Geneart and its website of the same name. Thus, the nucleotide sequences utilized in the invention can readily be codon optimized.","Another embodiment of the invention also encompasses nucleotide sequences that encode proteins, peptides or small molecules or the functional and\/or equivalent variants and derivatives of the proteins, peptides or small molecules that may bind to the histidine tetrad of the M2 protein based on the structural information provided by the invention to affect virus function. For instance, changes in a DNA sequence that do not change the encoded amino acid sequence, as well as those that result in conservative substitutions of amino acid residues, one or a few amino acid deletions or additions, and substitution of amino acid residues by amino acid analogs are those that will not significantly affect properties of the encoded polypeptide. Conservative amino acid substitutions are glycine\/alanine; valine\/isoleucine\/leucine; asparagine\/glutamine; aspartic acid\/glutamic acid; serine\/threonine\/methionine; lysine\/arginine; and phenylalanine\/tyrosine\/tryptophan. In one embodiment, the variants have at least 50%, at least 55%, at least 60%, at least 65%, at least 70%, at least 75%, at least 80%, at least 85%, at least 86%, at least 87%, at least 88%, at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98% or at least 99% homology or identity to the proteins, peptides or small molecules of interest.","For purposes of the present invention, sequence identity or homology is determined by comparing the sequences when aligned so as to maximize overlap and identity while minimizing sequence gaps. In particular, sequence identity may be determined using any of a number of mathematical algorithms. A nonlimiting example of a mathematical algorithm used for comparison of two sequences is the algorithm of Karlin et al., \u201cMethods for assessing the statistical significance of molecular sequence features by using general scoring schemes\u201d 87: 2264-68 (1990), modified as in Karlin et al., \u201cApplications and statistics for multiple high-scoring segments in molecular sequences,\u201d 90: 5873-77 (1993).","Another example of a mathematical algorithm used for comparison of sequences is the algorithm of Myers et al., \u201cOptimal alignments in linear space,\u201d 4: 11-17 (1988). Such an algorithm is incorporated into the ALIGN program (version 2.0) which is part of the GCG sequence alignment software package. When utilizing the ALIGN program for comparing amino acid sequences, a PAM120 weight residue table, a gap length penalty of 12, and a gap penalty of 4 can be used. Yet another useful algorithm for identifying regions of local sequence similarity and alignment is the FASTA algorithm as described in Pearson et al., \u201cImproved tools for biological sequence comparison,\u201d 85: 2444-48 (1988).","Advantageous for use according to the present invention is the WU-BLAST (Washington University BLAST) version 2.0 software. WU-BLAST version 2.0 executable programs for several UNIX platforms can be downloaded from on-line. This program is based on WU-BLAST version 1.4, which in turn is based on the public domain NCBI-BLAST version 1.4 described in Altschul et al., \u201cLocal alignment statistics,\u201d , Doolittle ed., 266: 460-80 (1996); Altschul et al., 215: 403-410 (1990); Gish et al., 3: 266-272 (1993); and Karlin et al., 90: 5873-5877 (1993), the entire contents and disclosures of which are incorporated herein by reference.","The various recombinant nucleotide sequences and polypeptides associated with the invention are made using standard recombinant DNA and cloning techniques. Such techniques are well known to those of skill in the art. See, for example, Sambrook et al., , second edition, volume 1, 2 and 3 (1989).","In certain embodiments, the polypeptides associated with the present invention may be used in vitro (such as using cell-free expression systems) and\/or in cultured cells grown in vitro in order to produce the polypeptides which may then be used for various applications such as in the production of proteinaceous vaccines. For applications where it is desired that the polypeptides be expressed in vivo, for example when the transgenes of the invention are used in DNA or DNA-containing vaccines, any vector that allows for the expression of the polypeptides of the present invention and is safe for use in vivo may be used.","For the polypeptides associated with the present invention to be expressed, the protein coding sequence may be \u201coperably linked\u201d to regulatory or nucleic acid control sequences that direct transcription and translation of the protein. As used herein, a coding sequence and a nucleic acid control sequence or promoter are said to be \u201coperably linked\u201d when they are covalently linked in such a way as to place the expression or transcription and\/or translation of the coding sequence under the influence or control of the nucleic acid control sequence. The \u201cnucleic acid control sequence\u201d can be any nucleic acid element, such as, but not limited to promoters, enhancers, IRES, introns, and other elements described herein that direct the expression of a nucleic acid sequence or coding sequence that is operably linked thereto.","In another embodiment, the invention provides a method of designing or identifying a drug candidate, small molecule inhibitor, an inhibitor compound of an influenza A virus based on computational methods using the structure and its coordinates, wherein the inhibitor binds at least one residue within the histidine tetrad to adversely affect viral function. The design method may be facilitated by using NMR spectra of samples of M2 protein fragments or equivalent constructs thereof. The method may further include the step of optimizing the design of the molecule that may be able to bind to further residues within the structure to affect viral function using methods which include but are not limited to NMR, particularly solid state NMR, X-ray crystallography, synthetic chemistry and combinations thereof.","Another embodiment of the invention is a method of designing an inhibitor compound that may modify the tetrameric His37-Trp41 cluster, referred to here as the HxxxW quartet, or any substructure, based on its conformation to rationally design one or more small molecule inhibitors.","Having described the many embodiments of the present invention in detail, it will be apparent that modifications and variations are possible without departing from the scope of the invention defined in the appended claims. Furthermore, it should be appreciated that all examples in the present disclosure, while illustrating many embodiments of the invention, are provided as non-limiting examples and are, therefore, not to be taken as limiting the various aspects so illustrated.","The description of the present invention is enhanced by the various examples that follow.","Cloning, plasmid construction, expression, and purification. Cloning, plasmid construction, protein expression and purification followed a previously published procedure as described in J. Hu et al., 16, 2153 (2007). Briefly, the DNA corresponding to M2 residues 22-62 of influenza A Udorn\/1972 virus is amplified by PCR and cloned into a modified pET30 vector containing a TEV protease cleavable MBP fusion protein expression system by ligation-independent cloning. The plasmid encoding the MBP-TEVc-M2 (22-62) fusion protein is transformed into strain BL21 (DE3)-RP codon plus for expression. A single colony is picked and inoculated into 3-mL LB media with 100 \u03bcg\/mL ampicillin, and grown overnight at 37\u00b0 C. with shaking. Cells are then collected by centrifugation, washed with M9 media once, and inoculated into 1-L M9 media with stable isotope labels (described below). The culture is grown to an OD=0.6 at 37\u00b0 C. with shaking, and cooled to 30\u00b0 C. Finally expression is induced with 0.4 mM IPTG for 12-16 h at 30\u00b0 C. with shaking.","For purification, cells expressing the fusion protein are collected by centrifugation at 4000 g for 10 min, and then washed once with a buffer containing 20 mM Tris-HCl (pH 8.0). Cooled cells are lysed by a French press in a binding buffer containing 50 mM NaCl and 20 mM Tris-HCl (pH 8.0), and the supernatant is collected after centrifugation at 10,000 g for 20 min. DDM is then added to the supernatant to a final concentration of 0.87% to solubilize the fusion protein in the membrane fraction. The supernatant is allowed to incubate with Ni-NTA agarose resin (Qiagen) while gently shaking. After binding at 4\u00b0 C. overnight, the resin is washed with a binding solution containing 20 mM imidazole followed by elution with a buffer containing 50 mM NaCl, 20 mM Tris-HCl (pH 8.0), 0.087% DDM, and 300 mM imidazole. The fractions containing the fusion protein are pooled and stored at 4\u00b0 C.","To cleave the fusion protein, TEV is added to the freshly purified fusion proteins (2.5-5 mg\/mL) at a mass ratio of 1:10. The TEV cleavage reaction takes place at room temperature for 16-20 h. To purify the M2 conductance domain, the TEV cleavage reaction is stopped by addition of TCA at a final concentration of 6%. The precipitate is collected by slow centrifugation. After washing the pellet with water twice to remove residual TCA, the protein is lyophilized in a vacuum centrifuge. Then 20 mL of methanol is added per liter of culture and mixed gently for several hours at room temperature. To remove the undissolved proteins (MBP and TEV), the solution is centrifuged at 13,000 g for 20 min, and the supernatant is carefully collected. The M2 protein is then lyophilized in a vacuum centrifuge and stored at \u221220\u00b0 C.","The purified protein, after being solubilized in DDM micelles, appears as a band in SDS-PAGE gel at \u02dc22 kDa (), which is close to the molecular weight of the M2 conductance domain as a tetramer.","For uniform N labeling, M9 media is supplemented with 1 g of N-ammonium chloride. For amino-acid specific N labeling, all 20 but the amino acid to be labeled are added per liter of M9 media in the following amounts: 800 mg each of Asp and Glu; 500 mg each of Ala, Val, Leu, and Ile; and 200 mg each of the other amino acids. The amount of the N labeled amino acid is 100 mg per liter of M9 media. Alternatively, to M9 media containing N-ammonium chloride unlabeled amino acids in the amounts listed above are added to prevent their labeling (known as reverse labeling). Labeling efficiency and cross-labeling are checked by comparing the solution NMR heteronuclear single quantum coherence spectra of uniformly labeled, specific amino-acid labeled, and reverse labeled proteins in LPPG detergent micelles.","Uniformly aligned sample preparation. Solid supported lipid-bilayer samples containing the M2 conductance domain are prepared as follows. DOPC and DOPE phospholipids are obtained from Avanti Polar Lipids as chloroform solutions. Aliquots of lipids in chloroform are mixed in a molar ratio of 4:1 (DOPC:DOPE) in a glass vial and thoroughly dried under flow of nitrogen gas to form a thin translucent film 5 mg of peptide in 10 ml TFE:methanol (1:1) is added to a DOPC:DOPE film (\u221275 mg total weight) and vortexed to solubilize the film. Organic solvents are removed by flowing nitrogen gas gently. Residual solvent is removed under vacuum for several hours. The resulting translucent protein-lipid film is hydrated with a small volume (\u22125 ml) of 10 mM Tris-HCl (pH 7.5), vortexed, and mixed in a shaker bath at 37\u00b0 C. for 3 hours. The lipid suspension is then transferred to a 1 kDa MW cutoff dialysis bag. The dialysis bag is placed in 1 L of 5 mM Tris-HCl buffer (pH 7.5) overnight to equilibrate the pH between the M2 liposomes and the buffer and to remove any trace amount of organic solvents. The liposomes are pelleted by ultracentrifugation at 196,000 g. The pellet is agitated at 37\u00b0 C. for 1 h until fluid. This thick fluid is spread onto 40 glass slides (5.7 mm\u00d712 0 mm) (Marienfeld Glassware, Bad Margentheim, Germany) and dehydrated in a 70-75% humidity chamber. The dehydrated slides are rehydrated with 1.5 \u03bcl of 2 mM Tris-HCl buffer (pH 7.5) and then stacked into a rectangular glass cell (New Era Enterprises Inc). The stacked slides are incubated at 43\u00b0 C. for 48 h in a 96% relative humidity (saturated KSO) chamber, resulting in uniformly aligned samples.","For NMR spectroscopy, samples are prepared with N-Leu, N-Ile, N-Val, N-Ala, and N-Phe labeling as well as uniform N labeling and reverse labeling. All NMR experiments are performed on 600 MHz wide bore and 900 MHz ultra wide bore spectrometers at the National High Magnetic Field Laboratory. All spectra are obtained at 30\u00b0 C. and pH 7.5 using home-built low-E static NMR probes as described in P. L. Gor'kov et al., 185, 77 (2007). The polarization inversion spin exchange at magic angle (PISEMA) and SAMPI4 pulse sequences were used for measuring correlation spectra in the N chemical shift and N-H dipolar coupling dimensions (as described in A. Ramamoorthy, S. J. Opella, 4, 387 (1995) and A. A. Nevzorov, S. J. Opella, 164, 182 (2003)). Typical experimental settings were as follows: the RF field strength is 50 kHz for cross polarization and 62.5 KHz for decoupling; the durations of contact pulse and 90\u00b0 pulse were 800 \u03bcs and 4 \u03bcs, respectively; 4000 scans were acquired for each of 16 increments in the dipolar coupling dimension for N chemical shifts >125 ppm and 32 increments for N chemical shifts <125 ppm.","Resonances of amino acids with the lowest frequencies of occurrence in the M2(22-62) sequence were assigned first, based on spectra of specific amino-acid labeled samples and according to the resonance pattern and geometry of helices as referred to in F. M. Marassi, S. J. Opella, 144, 150 (2000) and J. Wang et al., 144, 162 (2000). In particular, N resonances of residues in a helix trace a polarity index slant angle (PISA) wheel analogous to a helical wheel. Due to helical periodicity, resonances of residues at i, i+4, and i+7 positions appear close together on the PISA wheel. Accordingly, two Val resonances were identified in sequential positions on a PISA wheel, and assigned to Valand Val(). The Ile resonance closest to Valis assigned to Ilesince they are i to i+4 related. The next residue along the sequence is also an Ile; a lone Ile resonance is indeed found at a location on the PISA wheel expected for the sequential nearest neighbor and hence assigned to Ile. The resonance pattern of these Val and Ile residues is similar to that found previously for the transmembrane domain (as referred to J. Hu et al, 92, 4335 (2007)), suggesting that the N-terminal half of the transmembrane helix in the conductance domain has similar helix tilt and rotation as in the transmembrane domain. This similarity allowed the assignment of the Leuresonance.","All of the Phe resonances, including Phe, are clustered below 125 ppm in N chemical shifts (), consistent with them being associated with a lipid-interface bound helix, i.e., the amphipathic helix (C. Tian, P. F. Gao, L. H. Pinto, R. A. Lamb, T. A. Cross, Protein Sci. 12, 2597 (2003); N. J. Traaseth et al., Proc. NatL Acad. Sci. USA 104, 14676 (2007) and R. Fu, E. D. Gordon, D. J. Hibbard, M. Cotten, J. Am. Chem. Soc. 131, 10830 (2009)). There is one Leu and one Ile resonance each below 125 ppm (), which are assigned to Leuand Ile, respectively, and are presumed to be part of the PISA wheel for the amphipathic helix. Three Phe resonances are tightly clustered on this PISA wheel, but the fourth Phe resonance, located away from the PISA wheel, is assigned to Phe. The Phe resonance closest to the Leuresonance is assigned to Phe(i+4 to i related); the next nearest Phe resonance is assigned to Phe(i+7 to i related); and the last Phe resonance is assigned to Phe. That the resonances of these residues were clustered on the PISA wheel is an indication that the amphipathic helix extended over them.","After the assignment of 2 Val (out of 2), 3 Ile (out of 6), 2 Leu (out of 7), and 4 Phe (out of 4) residues, the remaining resonances were grouped into either the transmembrane helix or amphipathic helix (according to whether N chemical shifts were >125 ppm). These two sets of resonances were separately assigned by the PIPATH program, which was developed by Asbury et al. (T. Asbury et al, 183, 87 (2006)) for resonance assignment of membrane helices. The resonance assignments of the other residues of the M2 conductance domain are found in . The PISA wheel of the amphipathic helix is consistent with it being tilted at 15\u00b0 with respect to the bilayer surface, as demonstrated by fitting to a simulated PISA wheel for an ideal helix (). That the hydrophobic residues had larger dipolar couplings () than the positively charged residues () defined the C-terminal of the amphipathic helix as being tilted away from the bilayer center. Evidently, the transmembrane helix and the amphipathic helix is connected by a tight turn, since the intervening residue, Phe, is quite rigid, as indicated by its significant dipolar coupling (\u22123 kHz) and anisotropic chemical shift (\u221270 ppm).","Structure calculations. As noted above, the tetrameric state of the M2 conductance domain is demonstrated by SDS-PAGE electrophoresis (). That the tetramer is symmetric is demonstrated by the observation of a single resonance for each residue (). Using the peak positions of these resonances () as restraints, the final tetrameric structure is calculated in four steps. First, a structure for a monomer is generated through simulated annealing. Second, a preliminary structure for the tetramer is assembled by simulated annealing. Third, the tetramer structure is refined by restrained molecular dynamics simulations in a hydrated lipid bilayer. Finally, the sidechains of the HxxxW quartet were optimized by quantum chemical calculations. Below, these steps are described in more detail.","Monomer structure. An M2(22-62) monomer structure is generated using Xplor-NIH (as described in C. D. Schwieters, J. J. Kuszewski, N. Tjandra, G. M. Clore, J. Magn. Reson. 160, 65 (2003)) in torsion-angle space. Starting with an extended conformation, a monomer is equilibrated at 3500 K for 20 ps, with the backbone torsion angles of residues 26-46 and 48-58 restrained to ideal membrane helix values (\u03c6=\u221265\u00b0 and \u03c8=\u221240\u00b0 by flat-well (\u00b130\u00b0) potentials (force constant at 5 kcal\/mol\/rad2). Simulated annealing is then started, with the temperature of the monomer decreasing from 3500 K to 100 K with decrements of 25 K per 2 ps. Restraints to the PISEMA peak positions are enforced for residues 26-59 by flat-well (\u00b110 ppm in chemical shifts and \u00b10.5 kHz in dipolar couplings) harmonic potentials, with the force constants gradually increasing from 0.00006 to 12 kcal\/mol\/ppm2 for chemical shifts and 0.00003 to 6 kcal\/mol\/kHz2 for dipolar couplings. Additionally, the backbone torsion angles of residues 26-46 and 48-58 are again restrained to ideal membrane helix values by flat-well harmonic potentials (force constant at 200 kcal\/mol\/rad2); the distances between i and i+4 backbone hydrogen bonding atoms (carbonyl oxygen and amide nitrogen) of these residues were further restrained with flat-well (\u00b10.3 A) harmonic potentials, with the force constant increasing from 1 to 30 kcal\/mol\/\u212b2. Energy terms for bond angles, improper dihedral angles, and van der Waals interactions are scaled gradually from 0.4 to 1, 0.02 to 4, and 0.1 to 1, respectively (no scaling of the energy term for bond lengths) Finally the monomer is subjected to 2000 steps of energy minimization. The above procedure is repeated to generate 300 monomer structures. The top 10 structures all had well-defined TM and amphipathic helices, with average deviations from PISEMA restraints at 0.2 kHz for dipolar couplings and 3 ppm for chemical shifts. The structure with the least violations of the PISEMA restraints is used below to build a tetramer.","Preliminary tetramer structure. A tetramer is manually built by duplicating the monomer three times and positioning the four copies around a central axis with C4 symmetry. Each copy is oriented such that the polar residues, Ser, His, Trp, Asp, and Argof the transmembrane helix faced the central axis. Starting from the manually built tetramer, 20 structures are generated by running the Xplor-NIH program (C. D. Schwieters, J. J. Kuszewski, N. Tjandra, G. M. Clore, 160, 65 (2003)) following a procedure consisting of five stages: (1) equilibration at 2000 K for 2000 steps; (2) simulated annealing from 2000 K to 20 K with decrements of 10 K per 1000 steps; (3) energy minimization for 500 steps; (4) simulated annealing from 2000 K to 0 K with decrements of 10 K per 1000 steps; and (5) energy minimization for 500 steps. The tetramer is represented by torsion angles in stages (1)-(3) and by Cartesian coordinates in stages (4)-(5). In addition to the energy terms and PISEMA and helix restraints used in monomer building, the following inter-monomer restraints are introduced to maintain the tetrameric state and ensure that the polar residues of the transmembrane helices faced the pore: SerO-L26\u2032 O, HisN-Trp41\u2032 N\u03b5, His37\u2032 N\u03b5-His37\u2032 N, and Arg45 C\u03be Asp44\u2032 C\u03b3 [flat-well (\u00b10.2 \u212b) harmonic potentials centered at 3 \u212b with force constant of 100 kcal\/mol\/\u212b]. The amphipathic helices are also loosely restrained by using distance information obtained by EPR on a similar construct (residues 23-60) as referred to in P. A. Nguyen et al., 47, 9934 (2008)) [flat-well (\u00b12 \u212b) symmetric cubic potentials centered at 17-23 \u212b between C\u03b2 atoms of the same residue on adjacent monomers, for residues 49-57, with force constant ramped from 0.00001 to 10 kcal\/mol\/\u212b]. C4 symmetry is enforced by requiring equal distances from the central axis to each monomer, adapting protocols developed previously (M. Nilges, 17, 297 (1993) and S. I. O'Donoghue, G. F. King, M. Nilges, 8, 193 (1996)) with force constant 1000 kcal\/mol\/\u212b.","The amphipathic helix is connected to the TM helix by a tight turn involving just 2 peptide planes. The PISEMA restraints fixed the peptide planes with high precision with respect to the bilayer normal and significantly limited the choices of the backbone torsion angles. The EPR restraints (as described in P. A. Nguyen et al, 47, 9934 (2008)) further narrowed the conformational space and allowed the orientation of the amphipathic helix in the bilayer plane to be well defined. The 20 tetramer structures produced consisted of a major population with a \u201cclockwise\u201d orientation for the amphipathic helices and a minor population with a \u201ccounterclockwise\u201d orientation. The structure with the least violations of the PISEMA restraints and representing the major population is selected for further refinement, as described next.","Refinement of tetramer structure by restrained molecular dynamics simulations. The preliminary tetramer structure is placed in a DOPC:DOPE bilayer pre-equilibrated in TIP3P waters. The preparation of the bilayer system started with a structure file of a DOPC bilayer (128 lipids per leaflet) downloaded from a CHARMM-GUI (further information is available on the website of CHARMM-GUI) (as described in S. Jo, T. Kim, W. Im, 2, e880 (2007)). From each leaflet, 3 lipids are randomly selected for removal and another 25 are randomly selected for replacement by DOPE, resulting in a bilayer with 200 DOPC and 50 DOPE lipid molecules. The DOPC:DOPE ratio, 4:1, reproduces the composition of the lipid bilayer used for the NMR spectroscopy. The bilayer is solvated by 10985 water molecules (44 water molecules per lipid), and the lipid-water system is further equilibrated for 30 ns by NAMD as referred to in J. C. Phillips et al., 26, 1781 (2005), under constant normal pressure, surface area, and temperature (NPZAT ensemble) at 1 bar, 8425 A2, and 303.15 K, respectively. The surface area is chosen to reproduce the area per lipid, 67.4 A2, observed experimentally for liquid crystalline DOPC bilayers as further described in N. Kucerka et al., 95, 2356 (2008).","After placing the preliminary structure into the DOPC:DOPE bilayer, lipid and water molecules overlapping with the protein molecule are removed, resulting in 152 DOPC lipids, 37 DOPE lipids, and 10870 water molecules. To mimic the pH used for the NMR spectroscopy, two of the Hisresidues are neutral and the other two are protonated. To neutralize the whole system, 10 chloride ions are added.","After 500 steps of energy minimization, the simulation system is equilibrated under constant temperature and pressure conditions for 1 ns. During the energy minimization and equilibration, residues 24-46 and 51-58 are restrained to form ideal membrane helices by harmonic potentials on backbone hydrogen bond distances (force constant at 50 kcal\/mol\/\u212b) and dihedral angles (force constant at 100 kcal\/mol\/rad); C4 symmetry of the tetramer is enforced by applying distance restraints between the corresponding Ca atoms of adjacent monomers (force constant at 50 kcal\/mol\/\u212b). After the equilibration, the PISEMA restraints are gradually introduced by increasing the force constants to 0.02 kcal\/mol\/ppmand 1.0 kcal\/mol\/kHzover a simulation time of 0.5 ns. At this point the helix restraints are removed, and the PISEMA restraint force constants are gradually increased up to 0.9 kcal\/mol\/ppmand 12.0 kcal\/mol\/kHzwhile maintaining the C4-symmetry restraint. In order to stabilize the simulation with the greater PISEMA restraint force constants, the masses of backbone N, HN, and C atoms of residues 26 to 59 are increased to 30 atomic units. This last phase of the simulation lasted 0.1 ns, from which the snapshot with the best fit to the PISEMA data is selected. After fixing the backbone, the sidechains are further refined by continuing the simulation for 1 ns.","The refinement by restrained molecular dynamics simulations is performed by NAMD 2.7 (as described in J. C. Phillips et al., 26, 1781 (2005).) using the CHARMM27 force field with the CMAP correction (A. D. MacKerell et al., 102, 3586 (1998) and A. D. Mackerell, M. Feig, C. L. Brooks, 25, 1400 (2004).). The particle-mesh Ewald summation method is used to treat long-range electrostatic interactions (A. Toukmaji, C. Sagui, J. Board, T. Darden, 113, 10913 (2000)). An integration time step of 1 fs is used with a multiple time-stepping algorithm (T. Schlick et al., 151, 9 (1999)). Bonded interactions were calculated every time step, with short-range non-bonded interactions calculated every second time step, and long range (>12 \u212b) electrostatic interactions calculated every fourth time step. Van der Waals interactions were switched off smoothly between 10 and 12 \u212b. The pair list of non-bonded interactions is updated every 10 steps with a 13.5 \u212b cutoff Pressure is maintained by the Nose-Hoover Langevin piston method at 1 bar with a constant ratio of x-y dimensions of the periodic boundary (G. J. Martyna, D. J. Tobias, M. L. Klein, 101, 4177 (1994) and S. E. Feller, Y. H. Zhang, R. W. Pastor, B. R. Brooks, 103, 4613 (1995)). Temperature is maintained at 303.15 K using Langevin dynamics with 1.0 psdamping coefficient. PISEMA, helix, and C4-symmetry restraints were implemented into NAMD using the TCL force interface.","Sidechain optimization of HxxxW quartet by quantum chemical calculations. Current force fields do not model well the unique structure of the HxxxW quartet. Instead quantum chemical calculations are used to optimize the sidechains of the quartet. The system is comprised of residues His, Leu, Leu, and Trp, with the starting conformation taken from the molecular-dynamics refined structure. The ONIOM protocol (S. Dapprich, I. Komaromi, K. S. Byun, K. Morokuma, M. J. Frisch, -462, 1 (1999)) is used to accommodate the relatively large system for the quantum calculations. Briefly, the system is divided into two layers. The inner layer, comprised of all the Hisand Trpsidechains (excluding C and H atoms), is treated by B3LYP\/6-31G**. The outer layer, comprised of the rest, is treated by the AM1 semiempirical method. The system is optimized with all the backbone heavy atoms fixed. These calculations were done using the Gaussian 03 package (M. J. Frisch et al. (Gaussian, Inc., Wallingford, Conn., 2004)).","The optimized sidechain conformation of the HxxxW quartet is uniquely determined, as optimizations starting from different initial conformations and using somewhat different protocols always result in the same final conformation. This conformation features two strong hydrogen bonds between histidine Nand Natoms. Importantly, calculated NMR chemical shifts of these nuclei are in good agreements with those measured on the M2 transmembrane domain at pH 7 (J. Hu et al., . USA 103, 6865 (2006)).","The final structure of the conductance domain is produced by replacing the HxxxW quartet of the molecule-dynamics refined structure by the ONIOM-optimized conformation and energy-minimizing the rest of the simulation system while imposing the PISEMA and C4-symmetry restraints (the latter force constant now at 100 kcal\/mol\/\u212b). The backbone N chemical shifts and N-H dipolar couplings calculated on the final structure agree very well with the PISEMA data (); the deviations are 6 ppm and 0.4 kHz, well within the respective experimental errors of 10 ppm and 0.5 kHz. In addition, N data obtained from the Hisand Trpsidechains in the M2 \u2122 domain as described in K. Nishimura, S. Kim, L. Zhang, T. A. Cross, 41, 13170 (2002) and R. Witter et al., Proc. WSEAS: ., in press (2010) is consistent with the refined sidechain geometry of the conductance domain described here.","To generate an ensemble of structures, the process starting from the 0.5-ns simulation in which the PISEMA restraints are gradually introduced to the end is repeated. Eight structures are generated, which had average RMSD of 0.6 A calculated over backbone heavy atoms. These structures have been deposited in the Protein Data Bank with accession code 2L0J.","Conductance measurements. The M2 conductance domain is characterized functionally using the liposome proton uptake assay (as described in T. I. Lin, C. Schroeder, 75, 3647 (2001) and J. C. Moffat et al., 94, 434 (2008).). The M2-containing liposomes (145 nm in diameter) are prepared from a solution of 5 pM tetramer and 20 mg\/ml bacterial polar lipids (Avanti Polar Lipids) in an \u201cinternal\u201d buffer (50 mM KCl, 50 mM KHPO, 50 mM KHPO, pH 8.0, 320 mOsm) using a procedure similar to that described under \u201cUniformly aligned sample preparation\u201d. Namely, protein and lipid are cosolubilized in methanol and chloroform, dried to a thin film under a N2 stream and then vacuum, taken up in internal buffer, then thrice frozen, thawed, and sonicated. The liposome suspension is then passed 21 times through a polycarbonate filter (Avestin, Ottawa, Canada) at 50-60\u00b0 C., and evaluated with dynamic light scattering. Conductance measurements are started by diluting the liposomes 100-fold into an \u201cexternal\u201d buffer (165 mM NaCl, 1.67 mM sodium citrate, 0.33 mM citric acid, 320 mOsm, titrated appropriately for the target pH,) at room temperature (21\u00b0 C.). One minute later, the pH is fine tuned by titration. Two minutes thereafter, valinomycin is added (to 30 nM) to allow electrostatic compensation of proton flow and hence proton uptake, and the development of a membrane potential, nominally at \u2212114 mV.","Proton uptake is calculated from the pH change of the external buffer, as measured by an electrode. The pH measurements are calibrated after the vesicles are completely depolarized (using CCCP) by the average of two injections of 30 nEq HCl each. Corrections are made for baseline H leakage into the liposomes prior to valinomycin addition and for valinomycin-induced H leak observed in protein-free liposomes.  presents the time dependence of the proton uptake per tetramer at pH=5.5 (time=0 corresponding to the addition of valinomycin). The proton flux is calculated from the initial slope of the curve. The number of conducting protein tetramers is based on the nominal protein content of the sample, halved as a correction for the inability of proteins to activate when oriented with the N-terminal exposed to the high pH. For measurements with amantadine, liposomes are incubated in 100 pM amantadine overnight, and diluted into the external buffer containing 100 pM amantadine.","Novel screening of drugs against the proven drug target, the M2 protein of Influenza A, may be achieved using solid state NMR spectroscopy and uniformly aligned samples of the M2 protein. Solid state NMR is a technique that permits the observation of M2 protein in a native-like bilayer environment that assures the native conformation of this viral membrane protein. Changes in the observed frequency of the NMR signals reflect the binding site of the drug to the protein. Uniformly aligned samples of membrane proteins meant that each molecule in the sample will have the same orientation with respect to the magnetic field axis of the NMR spectrometer. In this way the orientation dependent spin interactions such as anisotropic chemical shifts, dipolar and quadrupolar interactions can be observed. The orientations of the individual atomic sites in the protein reflect the orientation of that particular site with respect to the magnetic field. For membrane proteins the plane of the bilayer and hence the orientation of the membrane protein spanning the membrane can be oriented with the bilayer normal either parallel or perpendicular to the magnetic field.",{"@attributes":{"id":"p-0133","num":"0132"},"figref":["FIG. 31","FIG. 32"],"sup":"31 ","i":["Science","Science"]},{"@attributes":{"id":"p-0134","num":"0133"},"figref":["FIGS. 1","FIGS. 25-30"],"b":["2","3","4","5","6","7"]},{"@attributes":{"id":"p-0135","num":"0134"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0136","num":"0135"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0137","num":"0136"},"figref":"FIG. 3","sup":["27","34","37","41","44","45 "]},{"@attributes":{"id":"p-0138","num":"0137"},"figref":"FIG. 4","sup":["50 ","50"]},{"@attributes":{"id":"p-0139","num":"0138"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0140","num":"0139"},"figref":"FIG. 6","i":"J. Mol. Graph. ","sup":["21 ","41"]},{"@attributes":{"id":"p-0141","num":"0140"},"figref":"FIG. 7","sup":["58 ","51 ","54"],"i":"Nature "},{"@attributes":{"id":"p-0142","num":"0141"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0143","num":"0142"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0144","num":"0143"},"figref":["FIG. 10","FIG. 9"]},{"@attributes":{"id":"p-0145","num":"0144"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0146","num":"0145"},"figref":"FIGS. 12 and 13"},{"@attributes":{"id":"p-0147","num":"0146"},"figref":"FIG. 12","sup":["37","41 ","37 ","41 ","41 "],"sub":["\u03b41 ","\u03b52","\u03b52 "]},{"@attributes":{"id":"p-0148","num":"0147"},"figref":"FIG. 13","sub":["\u03b41","\u03b52","\u03b41 ","\u03b1","\u03b2","\u03b3"]},{"@attributes":{"id":"p-0149","num":"0148"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0150","num":"0149"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0151","num":"0150"},"figref":"FIGS. 16-19"},{"@attributes":{"id":"p-0152","num":"0151"},"figref":"FIG. 16","sub":"\u03b41 ","sup":["38 ","40 "]},{"@attributes":{"id":"p-0153","num":"0152"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0154","num":"0153"},"figref":["FIG. 18","FIG. 16"]},{"@attributes":{"id":"p-0155","num":"0154"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0156","num":"0155"},"figref":["FIG. 20","FIG. 12"],"sub":["2 ","\u03b1","\u03b2","\u03b3","\u03b1","\u03b2","\u03b3","\u03b1","\u03b2"]},{"@attributes":{"id":"p-0157","num":"0156"},"figref":"FIG. 21","sup":["\u03b41 ","\u03b41 "],"sub":["\u03b52 ","\u03b1","\u03b2","\u03b3","\u03b52 ","\u03b52 "],"i":"Proc. Natl. Acad. Sci. U.S.A. "},{"@attributes":{"id":"p-0158","num":"0157"},"figref":"FIGS. 22 and 23"},{"@attributes":{"id":"p-0159","num":"0158"},"figref":"FIG. 22","sub":"ex","sup":["+","+"]},{"@attributes":{"id":"p-0160","num":"0159"},"figref":["FIG. 23","FIG. 22"],"i":["Proc. Natl. Acad. Sci","Proc. Natl. Acad. Sci"]},{"@attributes":{"id":"p-0161","num":"0160"},"figref":"FIG. 24","i":["Biochemistry ","Nature "]},{"@attributes":{"id":"p-0162","num":"0161"},"figref":["FIGS. 25-30","FIG. 25"],"sup":["15","15","1","15","15"]},{"@attributes":{"id":"p-0163","num":"0162"},"figref":"FIG. 26","sup":["15","15"]},{"@attributes":{"id":"p-0164","num":"0163"},"figref":"FIG. 27","sup":"15"},{"@attributes":{"id":"p-0165","num":"0164"},"figref":"FIG. 28","sup":["29","30","34","52 "]},{"@attributes":{"id":"p-0166","num":"0165"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0167","num":"0166"},"figref":"FIG. 30","sup":["60 ","61 "]},{"@attributes":{"id":"p-0168","num":"0167"},"figref":"FIG. 31","sup":"15"},{"@attributes":{"id":"p-0169","num":"0168"},"figref":"FIG. 32","i":["Science ","Nature, "]},"In the claims, unless specified otherwise, steps of a method may be performed in any order. For example, in a method claim, step (b) may be performed before step (a), unless the language of the claim requires that step (a) be performed prior to step (b).","While the present invention has been disclosed with references to certain embodiments, numerous modification, alterations, and changes to the described embodiments are possible without departing from the sphere and scope of the present invention, as defined in the appended claims. Accordingly, it is intended that the present invention not be limited to the described embodiments, but that it has the full scope defined by the language of the following claims, and equivalents thereof."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":["The accompanying drawings, which are incorporated herein and constitute part of this specification, illustrate exemplary embodiments of the invention, and, together with the general description given above and the detailed description given below, serve to explain the features of the invention.",{"@attributes":{"id":"p-0010","num":"0009"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0011","num":"0010"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0012","num":"0011"},"figref":"FIG. 3","sup":["27","34","37","41","44","45 "]},{"@attributes":{"id":"p-0013","num":"0012"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0014","num":"0013"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 6","sup":["27 ","21 ","41 "]},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 7","sup":["58 ","51 ","54"]},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 8"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":["FIG. 10","FIG. 9"]},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 12","sup":["37","41 ","37 ","41 "]},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 13"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 14","sub":"\u03b1"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 16"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":["FIG. 23","FIG. 22"]},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 24"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 25","sup":["15","15"]},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 26","sup":["15","15"]},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 27","sup":"15"},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 29"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":["FIG. 30","FIG. 29"]},{"@attributes":{"id":"p-0040","num":"0039"},"figref":"FIG. 31"},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 32","i":"Science "}]},"DETDESC":[{},{}]}
