---
title: Ex vivo and in vivo methods and related compositions for regenerating hematopoietic stem cell populations
abstract: Various embodiments provide methods and related compositions for increasing the population size of hematopoietic stem cells (HSCs) in patients that may benefit from reconstitution of stem cells and/or differentiated cells of the blood lineage. The present methods enable the production of HSCs ex vivo and in vivo by reducing latexin expression and/or latexin activity within HSC exposed to various antagonists. Inhibition of latexin expression and/or latexin activity by various antagonists can promote HSC proliferation and/or inhibit HSC apoptosis. Antagonists that can reduce latexin expression and/or latexin activity can be utilized to regenerate endogenous HSCs within patients affected with disorders, diseases, cancers, or therapies for such conditions, that result in the depletion or reduction in HSCs.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09284530&OS=09284530&RS=09284530
owner: University of Kentucky Research Foundation
number: 09284530
owner_city: Lexington
owner_country: US
publication_date: 20120103
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCES","STATEMENT OF GOVERNMENT INTEREST","TECHNICAL FIELD","INCORPORATION-BY-REFERENCE & TEXTS","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","I. Definitions","II. Regulation of Cell Cycling, Apoptosis, and Proliferation of Hematopoietic Stem Cells (HSCs) Mediated by Latexin","III. Methods for Inhibiting Latexin Expression and\/or Latexin Activity","EXAMPLES","Example 1","Mapping Congenic Intervals Containing Latexin","Example 2","Determination of HSC Frequencies by In Vitro Cobblestone Area-Forming Cell (CAFC) Assay","Example 3","In Vivo Determination of HSC Frequencies in Rodents by Competitive Limiting-Dilution Assay","Example 4","Identification of Latexin by Linkage Analysis","Example 5","Validation of the Linkage Analysis and Heritability of Latexin","Example 6","The Effect of Latexin Expression on HSC Replication","Example 7","The Effect of Latexin Expression on HSC Self-Renewal","Example 8","The Effect of Latexin Expression on Apoptosis","Example 9","Microarray Analysis of LSK Cells to Identify Differentially Expressed genes within Consenses Congenic Interval","Example 10","Determination of Lxn mRNA Levels in Hematopoietic Cells","Example 11","Lxn Expression in Undifferentiated and Differentiated Hematopoietic Cell Populations","Example 12","Lxn Protein Levels in Hematopoietic Cells of Congenic and Background Strain Mice","Example 13","Cis-Regulation of Lxn Expression","Example 14","A Decrease in Stem Cell Numbers By the Overexpression of Lxn","Example 15","Inhibition of Latexin by Exemplary SiRNAs"],"p":["This application is a divisional of U.S. application Ser. No. 11\/540,641, filed Oct. 2, 2006, which is a non-provisional application that claims priority under 35 U.S.C. \u00a7119 to U.S. Provisional Application No. 60\/722,755 entitled MANIPULATION OF ADULT STEM CELL POPULATION SIZES BY LATEXIN and filed on Sep. 30, 2005, the entire content of which are hereby incorporated by reference.","This invention has been made with Government support under grant number ROI AG024950, awarded by the National Institute on Aging. The government has certain rights in the invention.","The present invention relates to methods for regenerating adult hematopoietic stem cells (HSCs) and related compositions by reducing\/inhibiting latexin expression and\/or latexin activity.","The material on the accompanying CD-R is hereby incorporated by reference into this application. The accompanying compact disc contains one file, 1028750-000249 HSCS.ST25.txt, which was created on Oct. 2, 2006. The file named 1028750-000249 HSCS.ST25.txt is 5 KB. The file can be accessed using Microsoft Word on a computer that uses Windows OS.","Hematopoiesis is a complex process for producing multiple and distinct lineages of blood-borne cells throughout the life span of an organism. Hematopoietic stem cells (\u201cHSCs\u201d) represent a subset of undifferentiated cells that resides predominantly in the bone marrow of adult mammals. HSCs, as a population, are capable of self-renewal by maintaining a sufficient number of HSCs within an organism's bone marrow as a reservoir of uncommitted cells that can be further differentiated into various types of new blood cells. Such newly generated blood cells emerge from the bone marrow and enter the circulatory system in order to continuously replace mature\/aging circulating blood cell types. The ability of HSCs, as a population, to differentiate and to give rise to cells of multi-lineages is critical for the preservation of an organism.","In order for the maintenance of steady-state hematopoiesis, a balance must be achieved between the rate of self-generation (i.e., for maintaining a steady supply of HSCs) and the rate of differentiation (i.e., for replenishing senescent cells). Hematopoiesis occurs as a developmental continuum in that a given population of HSCs is representative of a heterogeneous mixture of cells, mainly composed of long-term HSCs (\u201cLT-HSCs\u201d) and short-term HSCs (\u201cST-HSCs\u201d). LT-HSCs are stem cells that have the capacity for self-renewal throughout the life span of an organism. However, ST-HSCs exhibit transient self-renewal properties for a limited period of time (e.g., typically less than 8 weeks in a mouse) prior to undergoing full differentiation. HSCs can differentiate into hematopoietic progenitor cells (\u201cHPCs\u201d) that can further differentiate into clonogenic cells, or cells of a single lineage. For example, the differentiation of common lymphoid progenitors (\u201cCLPs\u201d) can produce T lymphocytes (\u201cT cells\u201d), B lymphocytes (\u201cB cells\u201d), and natural killer cells (\u201cNKs\u201d). The differentiation of common myeloid progenitors (\u201cCMPs\u201d) can generate blood cells of other lineages, including erythrocytes, macrophages, granulocytes, and platelets. The maintenance of mature blood-borne cells in the peripheral circulation is critical for various processes, including oxygen delivery and immunological protection.","Bone marrow transplantation (\u201cBMT\u201d) and hematopoietic stem cell transplantation (\u201cHSCT\u201d) can be effective for the treatment of diseases of the blood, diseases of the bone marrow, cancers of the blood, and cancers of the bone marrow, including various types of anemia, leukemia, and immunological disorders. For example, obtaining HSCs by bone marrow harvesting from donors can be technically challenging in that harvesting sufficient material from the bone marrows of donors involves multiple insertions of large needles to obtain sufficient amount of stem cells. Methods for regenerating HSCs can be useful for treating patients affected by various types of disorders, diseases, or cancers that deleteriously affect the number of endogenous HSCs within a patient.","An ex vivo method for producing a renewed population of hematopoietic stem cells (HSCs) is provided, in which a first HSC population from a donor is obtained, the first HSC population is contacted with an antagonist that reduces latexin expression and\/or latexin activity, and the HSC population is cultured under in vitro conditions that can promote cell proliferation and\/or inhibit apoptosis to obtain a second HSC population that comprises a progeny of the first HSC population.","A method for reconstituting a recipient host with a HSC population generated ex vivo is provided, in which a first HSC population from a donor is obtained, the first HSC population is contacted with an antagonist that reduces latexin expression and\/or latexin activity, the HSC population is cultured under in vitro conditions that can promote cell proliferation and\/or inhibit apoptosis to obtain a second HSC population that comprises a progeny of the first HSC population, and the second HSC population is transplanted to the recipient host in need of HSC reconstitution.","A method for reconstituting a recipient host with a HSC population generated in vivo is provided, in which the recipient host is administered with a pharmaceutical composition comprising an antagonist that reduces latexin expression and\/or latexin activity.","Various embodiments provide methods and related compositions useful for reconstituting hematopoietic stem cells (HSCs) within a patient in need of HSC renewal. For example, the present methods are useful for the treatment of patients affected with disorders, diseases, and\/or cancers that deleteriously compromise the number of endogenous HSCs and differentiated forms of such HSCs within patients. Examples of disorders, diseases, and\/or cancers contemplated for treatment by the present methods and compositions include diseases of the blood, diseases of the bone marrow, cancers of the blood, and cancers of the bone marrow, including various types of anemia, leukemia, immunological disorders, thalassemia major, sickle-cell disease, myelodysplastic syndrome, lymphoma, aplastic anemia, and multiple myeloma. Patients that may benefit from treatments that utilize the present methods and related compositions include candidates for bone marrow transplantation (\u201cBMT\u201d) and hematopoietic stem cell transplantation (\u201cHSCT\u201d). Transplantation therapies can be effective for the treatment of various types of leukemia and any disorder\/disease\/cancer for which patients are subjected to radiotherapy and\/or chemotherapy. The renewed or regenerated population of HSCs that can be induced by the present methods and compositions are useful for various applications, including regenerative medicine, tissue engineering, tissue repair, cancer therapy, and gene therapy. As described in the following embodiments, a given HSC population obtained from a donor or within a recipient host (i.e., a patient) can be induced to proliferate and\/or to suppress apoptosis by providing antagonistic compounds\/compositions that can inhibit latexin expression and\/or latexin activity.","As used throughout the specification and the appended claims, the terms listed below have the following meanings, wherein \u201ca\u201d means one or more:","The terms \u201chematopoietic stem cells\u201d (\u201cHSCs\u201d) refer to a heterogeneous mixture of undifferentiated progenitor stem cells, mainly composed of long-term HSCs (\u201cLT-HSCs\u201d) and short-term HSCs (\u201cST-HSCs\u201d). LT-HSCs are undifferentiated stem cells that have the capacity for self-renewal throughout the life span of an organism. ST-HSCs are undifferentiated stem cells that have the capacity for self-renewal for a limited time prior to full differentiation into a specific lineage. For example, HSCs can differentiate into hematopoietic progenitor cells (\u201cHPCs\u201d) that can further differentiate into clonogenic cells, or cells of a single lineage, that represent a subset of a hematopoietic lineage. HSCs include non-embryonic stem cells isolated from post-natal animals. HSCs include \u201cpluripotent adult stem cells\u201d isolated from the bone marrows of mammals that are capable of differentiating into tissues of all three germ or dermal layers, such as the mesoderm, endoderm, and ectoderm. A \u201cpluripotent stem cell\u201d refers to a stem cell that exhibits the capacity to grow into any type of cell that constitutes an organism. A pluripotent cell can be self-renewing and can remain dormant\/quiescent within a tissue. HSCs include \u201cmultipotent stem cells\u201d that exhibit the capacity to grow into any subset of cells that constitute an organism. HSCs include undifferentiated stem cells that exhibit a distinct profile of cell surface markers, including the following combination of markers: Thy-1Sca-1Lineagec-kit; or CD150CD48Sca-1Lineagec-kit; or Thy-1CD150CD48Sca-1Lineagec-kit(Kiel et al., Cell, Vol. 121, p 1109-1121 (2005)). HSCs that express the combination of human counterparts for these markers are preferably contemplated. The following references are herein incorporated by reference in their entirety: Cheshier et al., P.N.A.S., Vol. 96, p 3120-3125 (1999); Eckfeldt et al., Nature, Vol. 6, p 726-737 (2005); Wright et. al., Science, Vol. 94, p 1933-1936 (2001); and Preston et al., J. Clin. Pathol. Vol. 56, p 86-96 (2003).","The term \u201cfirst HSC population\u201d refers to HSCs obtained from a donor, wherein the first HSC population expresses cell surface markers as described in [0036].","The term \u201csecond HSC population\u201d refers to the progeny of the first HSC population produced by in vitro cell proliferation under in vitro conditions, wherein the second HSC population expresses cell surface markers as described in [0036].","The term \u201clatexin\u201d (\u201cLxn\u201d) refers to the Lxn gene, isoforms\/variants of the Lxn gene, and gene products derived from the Lxn gene, including messenger RNA and protein. The sense-strand of a human latexin cDNA is provided as SEQ ID NO:1. A human latexin protein sequence is provided as SEQ ID NO:2. Latexin isoforms\/variants include genes containing exon sequences having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1; genes containing exon sequences having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence identity to SEQ ID NO:1; those sequences encoded by genes containing exon sequences having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1; those sequences encoded by genes containing exon sequences having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence identity to SEQ ID NO:1; those sequences having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:2; and those sequences having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence identity to SEQ ID NO:2. Latexin isoforms\/variants include mammalian homologs listed in Tables 1-3 as described in [0076]. Examples of latexin isoforms\/variants include the carboxypeptidase inhibitor (CARIN) described in U.S. Pat. No. 5,998,373.","The term \u201cantagonist\u201d refers to any compound or composition that can inhibit the expression and\/or the activity of latexin and latexin isoforms, including sequence-specific polynucleotides that can interfere with the transcription of endogenous latexin gene; sequence-specific polynucleotides that can interfere with the translation of latexin mRNA transcripts (e.g., siRNAs, ribozymes); sequence-specific polypeptides that can interfere with the protein stability of latexin, the enzymatic activity of latexin, and\/or the binding activity of latexin with respect to substrates and\/or regulatory proteins; and small molecule compounds that can interfere with the protein stability of latexin, the enzymatic activity of latexin, and\/or the binding activity of latexin. An effective antagonist can promote HSC proliferation by at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, or 100%. An effective antagonist can suppress HSC apoptosis by at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, or 100%.","The present inventors have identified a quantitative trait loci (\u201cQTL\u201d) that confers variation in HSC numbers in young and adult C57BL\/6 (B6) and DBA\/2 (D2) mice. By utilizing reciprocal Chromosome 3 congenic mice derived from these two parental strains, such as B.D Chr3 and D.B Chr3 mice, the present inventors have demonstrated that introgressed D2 alleles confer an increase in HSC population size by enhancing HSC cell proliferation (i.e., HSC self-renewal) and\/or by suppressing HSC apoptosis, as seen in B.D Chr3 congenic mice. In contrast, the B6 alleles conferred a decrease in HSC population size. Differential expression levels conferred by these alleles correlated with the latexin (\u201cLxn\u201d) gene locus. Differential latexin expression levels were shown to inversely correlate with HSC population size, based on experimental data obtained utilizing oligonucleotide arrays (e.g., microarrays), real-time PCR, and Western blots. Ectopic expression of latexin protein from a retroviral construct resulted in a decrease in stem cell population size. Clusters of single nucleotide polymorphisms (SNPs) were identified upstream of the Lxn transcriptional start site. At least two of these clusters appear to be associated with potential binding sites for transcription factors involved in hematorpoietic stem cell regulation. Thus, promoter polymorphisms between B6 and D2 alleles appear to affect Lxn gene expression, and consequently, to influence HSC population size. The results of these experiments are presented in  and are further described in Examples 1-15.","Latexin was identified initially as an antigen expressed within the rat cerebral cortex. Latexin mRNA and protein (29 kDa) have been shown by others to be expressed in neural and non-neural tissues. Latexin exhibits inhibitory activity against rat carboxypeptidase A1 (\u201cCPA1\u201d), carboxypeptidase A2 (\u201cCPA2), and mast-cell CPA (Uratani et al., Biochem. J., Vol 346: 817-826 (2000). CPA and CPB are pancreatic zinc-containing proteolytic enzymes that catalyze the hydrolysis of the carboxyl-terminal peptide bond within polypeptide chains. Carboxypeptidase activities can be regulated either by endogenous protein inhibitors or by enzymatic cleavage of a segment of a propeptide that releases the active form of the carboxypeptidase. Both latexin and Tissue Carboxypeptidase Inhibitor (TCI) are examples of endogenous protein inhibitors of carboxypeptidases. Latexin and TCI contain several potential phosphorylation sites, and both are expressed and localized in the cytosol of a number of tissues, including the brain, the lung, and the digestive tract.",{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 1"},"The mechanism for transcriptional regulation of latexin appears to be related to the unique molecular architecture of the Lxn gene locus, that includes multiple SNPs within potential regulatory regions. The Lxn gene locus is located within an intron of Gfml, a gene that encodes a translational elongation factor in mitochondria. Because Gfml was not differentially expressed amongst the congenic and background strains investigated, the differential transcriptional regulation of the embedded Lxn was unique. The specific promoter and\/or enhancer regions of Lxn appear to be highly gene-specific and to enable differential gene regulation. A potential mechanism for differential gene expression between B6 and D2 alleles is suggested by the identification of clusters of confirmed SNPs sites within the potential promoter region of Lxn that are spatially concordant with two putative binding sites for transcription factors known to regulate stem cells. For example, a cluster of SNPs (SNPs 1-3) in the canonical Lxn promoter region is consistent with a region identified by ESPEER as potentially possessing regulatory activity. The SNP2 is located in the middle of the canonical core binding site in one of two adjacent potential binding sites for the transcription factor CBF1. The CBF1 is associated with the Notch-signaling pathway, known to be play a role in the regulation of stem cell population size. The SNPs 10 and 11 are in close proximity, or reside within potential regulatory regions, identified by ESPERR and TraFaC. The HNF1\/TCF1, a transcription factor with two adjacent binding sites close to SNP11, can mediate events downstream of the canonical Wnt\/beta-catenin-signaling pathway known to be involved in HSC regulation and to interact with the Notch-signaling pathway.","A. Ex Vivo Methods","Various embodiments provide ex vivo methods for producing a renewed population of hematopoietic stem cells (HSCs) in which the method comprises obtaining a first HSC population from a donor, contacting the first HSC population with an antagonist that reduces latexin expression and\/or latexin activity, and culturing the HSC population under in vitro conditions that promote cell proliferation and\/or inhibit apoptosis to obtain a second HSC population that comprises a progeny of the first HSC population.","The first HSC population and the second HSC population can express any of the following combination of cell surface markers characteristic of HSCs: Thy-1Sca-1Lineagec-kit; or CD150CD48Sca-1Lineagec-kit; or Thy-1CD150CD48Sca-1Lineagec-kit.","The first HSC population can be obtained from a biopsy removed from a donor employing techniques known by persons skilled in the art, including the removal of stem cells from the bone marrow of a donor from large bone masses utilizing a large needle intended for bone marrow harvesting. The amount of hematopoietic stem cells needed as the first HSC population for the present method is less than the amount of hematopoietic stem cells needed for a conventional bone marrow transplantation that may require hundreds of needle insertions to obtain sufficient material for direct transplantation into a recipient host. Alternatively, HSCs may be collected by apheresis, a process in which a donor's peripheral blood is withdrawn through a sterile needle and passed through a device that removes white blood cells, and that returns the red blood cells to the donor. The peripheral stem cell yield can be increased with daily subcutaneous injections of granulocyte-colony stimulating factor. The HSCs are preferably obtained from human donors, however, non-human donors are also contemplated, including non-human primates, pigs, cows, horses, cats, and dogs. A purified population of HSCs may be obtained by utilizing various methods known by persons skilled in the art and described in U.S. Pat. No. 5,677,136; and U.S. Patent Publication No. 2006\/0040389, which are incorporated by reference in their entirety.","Suitable antagonists include SiRNAs, ribozymes, anti-sense oligodeoxynucleotides, and small molecule compounds that can reduce latexin expression and\/or latexin activity. Examples of suitable antagonists and methods for producing or identifying such antagonists are described below in Subsection C. A therapeutically effective latexin antagonist can promote HSC proliferation and\/or suppress apoptosis by at least 25%. Suitable antagonists are further defined above in Subsection I.","Contacting the first HSC population with an antagonist that reduces latexin expression and\/or latexin activity can promote cell proliferation and\/or inhibit apoptosis provided that the first HSC population is cultured under suitable in vitro conditions. Latexin expression and\/or latexin activity include intracellular processes that regulate latexin promoter activation, transcriptional activation, transcriptional termination, post-transcriptional processing, translational initiation, translational elongation, translational termination, and post-translational modification. Examples of suitable in vitro conditions are provided in U.S. Patent Publication No. 2005\/0214262; U.S. Patent Publication No. 2006\/0030041; and U.S. Patent Publication No. 2006\/0030041, which are incorporated by reference in their entirety.","Latexin isoforms\/variants suitable for antagonist-mediated targeting are provided above in [0039] in Subsection I.","In another embodiment, a HSC population produced by the present methods is disclosed, in which the HSCs comprise any of the following combination of cell surface markers characteristic of HSCs: Thy-1Sca-1Lineagec-kit; or CD150CD48Sca-1 Lineagec-kit; or Thy-1CD150CD48Sca-1Lineagec-kit.","Various embodiments provide methods for reconstituting a recipient host with a population of hematopoietic stem cells (HSCs) generated ex vivo, in which the method comprises obtaining a first HSC population from a donor, contacting the first HSC population with an antagonist that reduces latexin expression and\/or latexin activity, culturing the HSC population under in vitro conditions that promote cell proliferation and\/or inhibit apoptosis to obtain a second HSC population comprising a progeny of the first HSC population, and providing the second HSC population to the recipient host in need of HSC reconstitution.","The present methods are useful for the treatment of a recipient host affected with a disease of the blood, a disease of the bone marrow, a cancer of the blood, and a cancer of the bone marrow, an immunological disorder, anemia, leukemia, thalassemia major, sickle-cell disease, myelodysplastic syndrome, lymphoma, aplastic anemia, and\/or multiple myeloma. Disorders suitable for treatment by the present methods include disorders that reduce the number of endogenous HSC population, such as genetic disorders that result in sub-optimal number of HSCs, cancers that result in sub-optimal number of HSCs, post-irradiation conditions that result in sub-optimal number of HSCs, and post-chemotherapy conditions that result in sub-optimal number of HSCs.","The HSC population generated ex vivo by the present methods are transferred into a recipient host in need of HSC reconstitution. Such HSC populations that have been exposed to the antagonist, and that have been expanded can be infused intravenously into the blood stream of the recipient host. The antagonist-exposed and pre-expanded HSC population can localize within local niches of a bone marrow after a brief circulation in the blood stream. The antagonist-induced and pre-expanded HSC population may be able to respond to various growth factors produced by endothelial cells within the local bone marrow niches that may contribute to further proliferation and\/or suppression of apoptosis of the antagonist-exposed and pre-expanded HSC population. Reconstitution of HSCs within the recipient host is achieved when the total number of HSCs within the recipient host is at least 25% greater than before transplantation of the antagonist-induced and pre-expanded HSC population.","In one embodiment, the donor is the same organism as the recipient host in that the donor has identical HLA haplotype as the recipient host, and wherein the recipient host is reconstituted with autologous HSCs when provided with the second HSC population. For example, allogenic bone marrow transplantation involves two persons, a (non-diseased) donor and a (diseased) patient-recipient. In autologous transplantion, the risks for developing graft rejection, infection, and Graft Verses Host Disease (\u201cGVHD\u201d) are reduced.","In another embodiment, the donor is not the same organism as the recipient host, and the donor has a HLA haplotype of sufficient similarity to the HLA haplotype of the recipient host, and wherein the recipient host is reconstituted with allogenic HSCs when provided with the second HSC population. For optimal results, the allogeneic HSC donors should have identical human leukocyte antigens (\u201cHLA\u201d) as the recipient host, and the recipient host should receive immunosuppressive medications. For example, an allogenic transplant donor may be a relative (e.g., a sibling) or an unrelated volunteer.","B. In Vivo Methods","Various embodiments provide methods for reconstituting a recipient host with a population of hematopoietic stem cells (HSCs) generated in vivo in which the method comprises administering to the recipient host a pharmaceutical composition comprising an antagonist that reduces latexin expression and\/or latexin activity.","The present methods enable clinicians to promote HSC proliferation within the bone marrow of patients affected by any type of disorder, disease, or cancer that may reduce the supply of endogenous HSCs to less than an optimal number. The recipient host is affected with a disease of the blood, a disease of the bone marrow, a cancer of the blood, and a cancer of the bone marrow, an immunological disorder, anemia, leukemia, thalassemia major, sickle-cell disease, myelodysplastic syndrome, lymphoma, aplastic anemia, and\/or multiple myeloma.","The pharmaceutical composition comprises a therapeutically effective amount of the antagonist. The pharmaceutical composition further comprises a pharmaceutical excipient known by persons skilled in the art. Pharmaceutical formulations for effective delivery of pharmaceutical composition will vary depending on the latexin antagonist and mode of administration. Suitable pharmaceutical carriers are known by persons skilled in the art (Remington's Pharmaceutical Sciences (1989), which is incorporated in entirety). Pharmaceutical compositions can be administered by various methods, including by injection, oral administration, inhalation, transdermal application, or rectal administration. For oral administration, suitable formulations containing a pharmaceutical compound and pharmaceutically-compatible carriers can be delivered in various forms, such as tablets or capsules, liquid solutions, suspensions, emulsions, and the like. For inhalation, suitable formulations containing a pharmaceutical compound and pharmaceutically-compatible carriers can be delivered as aerosol formulations that can be placed into pressurized propellants, such as dichlorodifluoromethane, propane, nitrogen, and the like. For parenteral administration, suitable formulations containing a pharmaceutical compound and pharmaceutically-compatible carriers can be delivered by intra-articular, intra-venous, intra-muscular, intra-dermal, intra-peritoneal, and sub-cutaneous routes.","Pharmaceutical compositions suitable for use include compositions containing active ingredients in an effective amount to achieve its intended purpose. More specifically, a therapeutically-effective amount means an amount effective to promote the proliferation of HSCs and\/or inhibit apoptosis of HSCs in subjects exposed to the present pharmaceutical compositions. Determination of the effective amounts is well within the capability of persons skilled in the art. A dose can be formulated in animal models to achieve a circulating concentration range that includes IC50 value, defined as a dose in which 50% of cells of a culture show an effect due to the test compound. Such information can be used to more accurately determine useful doses in human subjects.","Toxicity and therapeutic efficacy of such compounds can be determined by standard pharmaceutical procedures utilizing cell cultures or experimental animals in order to determine a LD50 value, the dose determined to be lethal to 50% of the exposed population, and to determine a ED50 value, the dose determined to be therapeutically effective in 50% of the exposed population. A dose ratio between toxic effect and therapeutic effect is referred to as the \u201ctherapeutic index,\u201d or it can be expressed as the ratio of the LD50 value over the ED50 value. Compounds that exhibit high therapeutic indices are preferred. The data obtained from cell culture assays and animal studies can be used in formulating a range of effective dosage for human usage. Optimal dosage range includes a ED50 dose with minimal toxicity, although the dosage may vary within this range depending on a given pharmaceutical formulation and route of administration. Dosage administered to a subject should be adjusted according to the age of the subject, the weight of the subject, the manner of administration, and other circumstances unique to each subject.","C. Exemplary Antagonists that Inhibit the Expression and\/or Activity of Latexin and Latexin Isoforms\/Variants","A HSC population obtained from a donor can be induced to proliferate ex vivo under in vitro conditions, or an endogenous HSC population within a patient can be induced to proliferate in situ by exposing the HSC population of interest to various antagonists that can inhibit latexin gene expression and\/or latexin activity.","Suitable compositions that can inhibit the expression and\/or the activity of latexin and latexin variants include sequence-specific polynucleotides that can interfere with the transcription of endogenous latexin gene; sequence-specific polynucleotides that can interfere with the translation of latexin mRNA transcripts (e.g., siRNAs, ribozymes); sequence-specific polypeptides that can interfere with the protein stability of latexin, the enzymatic activity of latexin, and\/or the binding activity of latexin with respect to substrates and\/or regulatory proteins; antibodies that exhibit specificity for latexin; and small molecule compounds that can interfere with the protein stability of latexin, the enzymatic activity of latexin, and\/or the binding activity of latexin. An effective antagonist can promote HSC proliferation by at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95%, or 100%.","1. Sequence-Specific Compounds","a. RNAi Compounds","In one embodiment, various siRNAs that are complementary to human latexin mRNA and mammalian orthologs can be employed by persons skilled in the art to promote HSC proliferation and\/or inhibit HSC apoptosis. By introducing such RNAi compounds to a recipient patient deficient in HSCs, the silencing or inactivation effect of RNAi compounds on latexin gene\/gene products within the HSCs can promote HSC proliferation and\/or inhibit HSC apoptosis. Because introduction of double-stranded RNA (\u201cdsRNA\u201d) that are longer than 30 nucleotides into mammalian cells induces a sequence-nonspecific interferon response, alternative methods for delivery of interfering RNA molecules (\u201cRNAi\u201d) may be suitable. For example, most common form of RNAi molecules are short-interfering RNAs (\u201csiRNAs\u201d) of 21-23 base-pairs that are chemically or enzymatically synthesized, which can be introduced into mammalian host cells by various methods, including transfections. However, unlike fungi, plants, and worms that can replicate siRNAs in vivo, transfection of siRNA produces only transient gene-silencing effect in mammalian cells. As an alternative, DNA vectors encoding precursor-like forms of siRNAs may be used for stable production of siRNAs in vivo in various hosts, including mammalian cells.","In another embodiment, compounds that can promote HSC proliferation and\/or inhibit HSC apoptosis are contemplated, in which the compounds comprise an oligonucleotide that can interact with endogenous messenger RNA encoded by latexin gene or by latexin isoforms having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1 (i.e., a sense-strand of human latexin cDNA). Such RNA oligonucleotide compounds can be single-stranded or double-stranded. Suitable lengths of RNA oligonucleotides include molecules containing 15-20 nucleotides, 20-30 nucleotides, 30-50 nucleotides, 50-75 nucleotides, 75-100 nucleotides, 100-150 nucleotides, 150-200 nucleotides, and 200-300 nucleotides. In another embodiment, the present invention is directed to compounds that can promote HSC proliferation and\/or inhibit HSC apoptosis, the compounds include an anti-sense strand that can hybridize to an endogenously produced messenger RNA, and that can inhibit the translation of the messenger RNA.","Selection of efficient siRNAs is an empirical process, but certain rules governing optimal selection of siRNAs are known. The sequence selected for a siRNA appears to be critical. For example, siRNAs containing sequence motifs, such as AANTT, NANNN, NARNYNN, and NANNYNN, are effective, in which N is any nucleotide, R is a purine, and Y is a pyrimidine. In addition, regions of complementary DNA should have non-repetitive sequences, and should avoid intronic sequences. Suitable siRNAs contain approximately 30-70% GC content, contain even representation of all nucleotides on the anti-sense strand, and do not contain stretches of single nucleotide, especially stretches of Gs. Designing suitable SiRNA molecules is within the scope of persons skilled in the art. The following references are herein incorporated by reference in their entirety: Henschel et. al., Nucleic Acids Research, Vol. 32: 113-120 (2004); Naito et al., Nucleic Acids Research, Vol. 32: 124-129 (2004); Dorsett et al., Vol. 3: 318-329 (2004); and Brummelkamp et al., Nature Reviews, Vol. 3: 781-789 (2003); Pusch et al., Nucleic Acid Research, 31: 6444-6449 (2003); and Chiu et al., RNA 9: 1034-1048 (2003).","Although any region of mRNA can be theoretically targeted, certain sequences that are known binding sites for mRNA-binding proteins should be avoided, including untranslated regions, such as the \u201c5\u2032UTR\u201d and \u201c3\u2032UTR,\u201d start-codons, and exon-exon boundaries. For some mRNA targets, siRNA-directed silencing may be more effective if the siRNA sequence is selected at least 50-100 nucleotides downstream of a start codon, and preferably directed towards the 3\u2032 end of a target mRNA. In addition, the conformation of a mRNA recognition site within an mRNA target is preferably RNAse-H-sensitive, and preferably not within a highly-structured RNA region. These guidelines are generally applicable since the choice of a siRNA depends on the target mRNA sequence, and persons skilled in the art would need to synthesize several siRNAs to validate the efficiency of each. The specificity of a siRNA for a single gene can be ascertained by performing a multiple-genome-sequence alignment, such as a BLAST search of the selected sequence against sequence databases, including \u201cUnigene\u201d libraries associated with National Center for Biotechnology Information (NCBI). Potential off-target silencing by siRNA may be minimized by choosing a siRNA sequence with maximum sequence divergence from a list of genes with partial-sequence identity to the intended mRNA target. General principles for siRNA selection are taught by the following two review articles, which are incorporated by reference (Dorsett and Tuschl, Nature Reviews Vol. 3: 318-329, (2004); Dykxhoorn et al., Nature Reviews Vol. 4: 457-467 (2003)).","Various expression vectors can be constructed to enable stable production of siRNA-like molecules in vivo. For example, RNA-pol II promoters may be operably-linked to a hairpin precursor of a siRNA sequence of interest. RNA-pol II promoters represent a broad range of promoters that enable substantial control over parameters governing RNA expression, such as inducible, constitutive, tissue-specific, or developmentally-regulated RNA expression. Alternatively, RNA-pol III promoters may be used to produce short RNA species that do not activate the interferon pathway. Suitable RNA-pol III promoters include class III promoters that lack essential transcriptional elements downstream of a transcription initiation site, such as U6 and H1 promoters, which may be operably-linked to a siRNA-encoding sequence.","Long-hairpin RNAs, imperfect shRNAs, miRNAs, and siRNAs, can be designed as follows. For example, a sub-sequence of a messenger RNA encoded by latexin gene or by latexin isoforms having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1 (i.e., a sense-strand of human latexin cDNA), can be targeted. For example, an anti-sense strand of shRNA can be designed by selecting a sub-sequence portion of a RNA sequence complementary to endogenous latexin messenger RNA and latexin isoforms having at least about 70% sequence similarity to SEQ ID NO:1","For designing shRNA, the composition and size of the loop and length of the stem of a hairpin duplex should be considered. Suitable stem lengths for efficient silencing include a broad range, including stem lengths of 19-29 nucleotides. Suitable loop lengths for efficient silencing include a broad range, including loop lengths of 4-23 nucleotides. In certain context, hairpin structures with duplexed regions that are longer than 21 nucleotides may promote effective siRNA-directed silencing, regardless of loop sequence and length.","Various gene-delivery vectors that are practiced by persons skilled in the art can be used to introduce the present expression vectors. Examples of viral vectors that may be used to infect HSC cells include: improved adenoviral vectors (Reynold et al., Nature Biotechnology 19: 838-842 (2001)); gene-deleted adenovirus-transposon vectors (Yant et al., Nature Biotechnology 20: 999-1005 (2002)); recombinant adenoviruses (Bilang-Bleuel et al., Proc. Natl. Acad. Sci. U.S.A. 94: 8818-8823 (1997)); the Moloney-murine-leukemia-virus (\u201cMo-MuLV\u201d) based retroviral vectors (Auten et al., Human Gene Therapy 10: 1389-99 (2003)); and poliovirus-replicon-based vectors (Bledsoe et al., Nature Biotechnology 18: 964-969 (2000)). Examples of other suitable viral vectors include: herpes virus, mumps virus, Sindbis virus, vaccinia virus, such as the canary pox virus, and lentivirus. The usage of viral vectors is well known by persons skilled in the art, and for gene therapy uses, viral infection is preferred generally. The following references are incorporated by reference in their entirety: Robbins and Ghizzani, Mol. Med. Today 1:410-417 (1995); Robin et al., Stem Cells 20:514-521 (2002); Chen et al., Immunity 19:525-533 (2003); North et al., Immunity 16: 661-672 (2002); Zhou et al., Nature Medicine 7: 1028-1034 (2001); Ivanova et al., 298: 601-604 (2002); and Santos et. al., Science 298: 597-600 (2002).","b. Chemically Modified Anti-Sense Oligodeoxyribonucleic Acids (\u201cODNs\u201d)","In one embodiment, various oligodeoxyribonucleic acid (ODN) molecules that are complementary to endogenous messenger RNA encoded by latexin gene or by latexin isoforms having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1 (i.e., a sense-strand of human latexin cDNA), can be employed by persons skilled in the art to promote HSC proliferation and\/or to inhibit HSC apoptosis. By introducing such oligodeoxyribonucleic acid (ODN) molecules to a recipient patient deficient in HSCs, the silencing or inactivation effect of RNAi compounds on latexin gene\/gene products within the HSCs can promote HSC proliferation and\/or inhibit HSC apoptosis. Suitable oligodeoxyribonucleic acid molecules (\u201cODNs\u201d) are short polynucleotides of approximately 20 nucleotides in length, that can hybridize with pre-mRNA and mRNA to form RNA-DNA duplexes, which are degraded by ribonuclease H (\u201cRNase H\u201d). Such ODNs can be chemically modified to prevent the action of RNase H, to inhibit translation of mRNA by steric hindrance, to inhibit splicing of pre-mRNAs, and to inhibit transcription by the formation of triple helices. Kurreck, J. et al., Eur. J. Biochem., 270:1628-1644 (2003); Baker et al., J. Biol. Chem., 272:11994-2000 (1997); Lu, Q. L. et al, Nature Med., 9:1009-1014 (2003); and Uil, et al., Nucleic Acids Res., 31:6064-6078 (2003) are incorporated by reference in their entirety.","c. Ribozymes","In one embodiment, various ribozymes containing sequences that are complementary to endogenous messenger RNA encoded by latexin gene or by latexin isoforms having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1 (i.e., a sense-strand of human latexin cDNA), can be employed by persons skilled in the art in order to reduce latexin expression and\/or latexin activity that results in HSC proliferation and\/or HSC apoptosis. Ribozymes, including the \u201chammer-head\u201d ribozyme, are RNA molecules that bind target mRNA by assuming a unique secondary structure when hybridized to target mRNA, which enables catalytic hydrolysis of a phosphodiester bond within in the backbone of target mRNA. Efficient cleavage by a ribozyme requires the presence of divalent ions, such as magnesium, and is also dependent on target RNA structure, and relative proximity between ribozyme and target molecule. RNA-localization signals or RNA chaperones may be used so that low concentrations of ribozymes are sufficiently effective in silencing latexin and latexin isoforms. Ribozymes can be chemically synthesized in vitro, and can be transcribed from expression vectors in vivo. Methods for ribozyme construction and utilization are known by persons skilled in the art. Doudna and Cech, Nature, 418:222-228 (2002); Kuwabara et al, J. Biochem., 132:149-155 (2002); Michienzi and Rossi, Methods Enzymol., 341:581-596 (2001); and Good et al., Gene Ther. 4:45-54, (1997) are herein incorporated by reference.","d. Other Compounds","In another embodiment, suitable antagonists include compounds that can interact with polypeptides having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:2, that can inhibit the activity of the bound polypeptide, and that can promote the proliferation of HSCs and\/or suppress the apoptosis of HSCs. In addition, compounds that can interact with genomic DNA containing exons having at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1 (i.e., a sense-strand of human latexin cDNA) can be employed. In addition, compounds that can interact with RNA transcripts encoded by a gene, in which the exon sequences have at least about 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% sequence similarity to SEQ ID NO:1 (i.e., a sense-strand of human latexin cDNA) can be employed. Various compounds that can inhibit the expression and\/or activity of latexin or latexin isoforms can be either naturally-occurring or synthetically-produced. Large combinatorial libraries of chemical\/biological compounds can be generated by various chemical and biological synthesis methods known in the art. Such combinatorial chemical libraries include: small organic molecule libraries (benzodiazepines, Baum C&EN, Jan. 18, page 33 (1993)); Chen et al., J. Amer. Chem. Soc. 116:2661 (1994)), such as isoprenoids (U.S. Pat. No. 5,569,588), thiazolidinones and metathiazanones (U.S. Pat. No. 5,549,974), pyrrolidines (U.S. Pat. Nos. 5,525,735 and 5,519,134), morpholino compounds (U.S. Pat. No. 5,506,337), and benzodiazepines (U.S. Pat. No. 5,288,514), oligocarbamates (Cho et al., Science 261:1303 (1993)), and peptidyl phosphonates (Campbell et al., J. Org. Chem. 59:658 (1994)). Exemplary combinatorial libraries include: various peptide libraries (U.S. Pat. No. 5,010,175; Furka, Int. J. Pept. Prot. Res. 37:487-493 (1991); Houghton et al., Nature 354:84-88 (1991)); peptoid libraries (PCT Publication No. WO 91\/19735); encoded peptides (PCT Publication No. WO 93\/20242); random bio-oligomers (PCT Publication No. WO 92\/00091); benzodiazepines (U.S. Pat. No. 5,288,514); diversomers such as hydantoins, benzodiazepines and dipeptides (Hobbs et al., Proc. Nat. Acad. Sci. USA 90:6909-6913 (1993)); vinylogous polypeptides (Hagihara et al., J. Amer. Chem. Soc. 114:6568 (1992)); nonpeptidal peptidomimetics with glucose scaffolding (Hirschmann et al., J. Amer. Chem. Soc. 114:9217-9218 (1992)); various nucleic-acid libraries; various peptide-nucleic acid libraries (U.S. Pat. No. 5,539,083); various carbohydrate libraries (Liang et al., Science, 274:1520-1522 (1996); U.S. Pat. No. 5,593,853)); and various antibody libraries (Vaughn et al., Nature Biotechnology, 14(3):309-314 (1996)).","D. Identification of Functionally-Related Isoforms of Latexin","The present methods and compositions can be employed for targeting variants of latexin that have similar properties\/activities within HSCs of many types of mammalian subjects. Functionally-related isoforms of latexin can be identified by searching various genomic databases and conducting multi-genome-wide sequence alignments in order to identify homologous sequences of interest. Related orthologous sequences can be identified by searching composite genomic databases. The breath of a database search is limited by the scope of representative model organisms for which sequence data is available.","Homology can be determined by various methods, including alignments of open-reading-frames (\u201cORFs\u201d) contained in private and\/or public databases. Any suitable mathematical algorithm may be used to determine percent identities and percent similarities between any two sequences being compared. For example, nucleic acid and protein sequences of the present invention can be used as a \u201cquery sequence\u201d to perform a search against sequences deposited within various public databases to identify other family members or evolutionarily-related sequences. Genomic sequences for various organisms are currently available, including fungi, such as the budding yeast, or Saccharomyces cerevisiae; invertebrates, such as Caenorhabditis elegans and Drosophila melangaster; and mammals, such as the mouse, rat, and human. Exemplary databases for identifying orthologs of interest include Genebank, Swiss Protein, EMBL, and National Center for Biotechnology Information (\u201cNCBI\u201d), and many others known in the art. These databases enable a user to set various parameters for a hypothetical search according to the user's preference, or to utilize default settings. Tables 1-3, provided below, lists the accession numbers and gene identification numbers for exemplary mammalian orthologs. Suitable latexin variant that may be targeted for suppression or inhibition include the carboxypeptidase inhibitor (CARIN) described in U.S. Pat. No. 5,998,373.",{"@attributes":{"id":"p-0078","num":"0077"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Gene Bank Accession Number for Exemplary Mouse,"},{"entry":"Human, and Rat Latexin mRNA."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Organism","Accession Number","Gene IDS"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},{"i":"Mus musulus"},"NM-016753","31980631"]},{"entry":[{},"(Mouse)"]},{"entry":[{},{"i":"Rattus norvegicus"},"NM-031655","14269567"]},{"entry":[{},"(Rat)"]},{"entry":[{},{"i":"Homo Sapiens"},"NM-020169","21359932"]},{"entry":[{},"(human)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0079","num":"0078"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 2"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Gene Bank Accession Number for Exemplary Mouse,"},{"entry":"Human, and Rat Latexin Protein."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Organism","Accession Number","Gene IDS"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]},{"entry":[{},{"i":"Mus musulus"},"NP-058033","31980632"]},{"entry":[{},"(Mouse)"]},{"entry":[{},{"i":"Rattus norvegicus"},"NP-113843","14269568"]},{"entry":[{},"(Rat)"]},{"entry":[{},{"i":"Homo Sapiens"},"NP-064554","21359933"]},{"entry":[{},"(human)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}}},{"@attributes":{"id":"p-0080","num":"0079"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Mammalian Homologs for Latexin"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Accession",{},"Percentage",{}]},{"entry":["Organism","Number","Gene IDS","Identity","Gene Information"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{"i":"Mus musulus"},"AK032170.1","26327996","99.91",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"male olfactory brain"]},{"entry":[{},{},{},{},"cDNA, RIKEN full-"]},{"entry":[{},{},{},{},"length enriched library,"]},{"entry":[{},{},{},{},"clone: 6430407E02"]},{"entry":[{},{},{},{},"product: Latexin, full"]},{"entry":[{},{},{},{},"insert sequence"]},{"entry":[{"i":"Mus musulus"},"AK149981.1","74211713","100.00",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"marrow macrophage"]},{"entry":[{},{},{},{},"cDNA, RIKEN full-"]},{"entry":[{},{},{},{},"length enriched library,"]},{"entry":[{},{},{},{},"clone: G530111O19"]},{"entry":[{},{},{},{},"product: Latexin, full"]},{"entry":[{},{},{},{},"insert sequence"]},{"entry":[{"i":"Mus musulus"},"D88769.1","1669620","99.50",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"for Latexin, complete"]},{"entry":[{},{},{},{},"cds"]},{"entry":[{"i":"Mus musulus"},"AC124190.4","23499687","100.00",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"clone RP23-267M9"]},{"entry":[{"i":"Mus musulus"},"AC124190.4","23499687","100.00","from 3, complete"]},{"entry":["(Mouse)",{},{},{},"sequence"]},{"entry":[{"i":"Mus musulus"},"AC124190.4","23499687","100.00"]},{"entry":"(Mouse)"},{"entry":[{"i":"Mus musulus"},"AC124190.4","23499687","100.00"]},{"entry":"(Mouse)"},{"entry":[{"i":"Mus musulus"},"AC124190.4","23499687","100.00"]},{"entry":"(Mouse)"},{"entry":[{"i":"Mus musulus"},"AC124190.4","23499687","100.00"]},{"entry":"(Mouse)"},{"entry":[{"i":"Mus musulus"},"AK018305.1","12857946","99.58",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"neonate cerebellum"]},{"entry":[{},{},{},{},"cDNA, RIKEN full-"]},{"entry":[{},{},{},{},"length enriched library,"]},{"entry":[{},{},{},{},"clone: 6530401A10"]},{"entry":[{},{},{},{},"product: Latexin, full"]},{"entry":[{},{},{},{},"insert sequence"]},{"entry":[{"i":"Mus musulus"},"AK198791.1","56022968","100.00",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"clone: Y1G0129D08,"]},{"entry":[{},{},{},{},"strand: minus,"]},{"entry":[{},{},{},{},"reference: ENSEMBL:"]},{"entry":[{},{},{},{},"Mouse-Transcript-"]},{"entry":[{},{},{},{},"ENST: ENSMUST00000058981,"]},{"entry":[{},{},{},{},"based on"]},{"entry":[{},{},{},{},"BLAT search"]},{"entry":[{"i":"Mus musulus"},"AK187030.1","56011207","99.52",{"i":"Mus musculus "}]},{"entry":["(Mouse)",{},{},{},"clone: Y0G0139J09,"]},{"entry":[{},{},{},{},"strand: minus,"]},{"entry":[{},{},{},{},"reference: ENSEMBL:"]},{"entry":[{},{},{},{},"Mouse-Transcript-"]},{"entry":[{},{},{},{},"ENST: ENSMUST00000058981,"]},{"entry":[{},{},{},{},"based on"]},{"entry":[{},{},{},{},"BLAT search"]},{"entry":[{"i":"Rattus"},"Y18435.2","6066618","94.33",{"i":"Rattus norvegicus"}]},{"entry":[{"i":"norvegicus"},{},{},{},"Latexin gene, exons 1"]},{"entry":["(Rat)",{},{},{},"to 6"]},{"entry":[{"i":"Rattus"},"Y18435.2","6066618","96.92"]},{"entry":{"i":"norvegicus"}},{"entry":"(Rat)"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"The following examples are offered by way of illustration and not by way of limitation. Examples 1 provides a map of congenic intervals defined in the reciprocal strains. Example 2 provides an in vitro method for determining HSC frequencies by cobblestone area-forming cell (CAFC) assay. Example 3 provides an in vivo method for determining HSC frequencies by a competitive limiting-dilution assay. Example 4 provides the identification of latexin by a genome-wide linkage analysis that correlates three QTL with HSC population size. Example 5 provides data that validates the linkage analysis and heritability of latexin. Example 6 provides the effect of latexin on HSC Replication. Example 7 provides the effect of latexin on HSC Self-renewal. Example 8 provides the effect of latexin expression on apoptosis. Example 9 provides microarray analysis of LSK cells. Example 10 provides Lxn mRNA levels in hematopoietic cells. Example 11 provides Lxn expression in undifferentiated and differentiated hematopoietic cell populations. Example 12 provides Lxn protein levels in hematopoietic cells of congenic and background strain mice. Example 13 provides Cis-Regulation of Lxn expression. Example 14 provides a decrease in stem cell numbers by the overexpression of Lxn.",{"@attributes":{"id":"p-0082","num":"0081"},"figref":"FIG. 2","sup":"b "},"The Chr 3 congenic mice were generated by well-established methods known in the art. Mouse strains congenic for the Chr 3 QTL were generated by crossing the genomic interval harboring the QTL using D2, as the donor strain, and B6, as the recipient strain (symbolized as B.D Chr3), and vice versa (D.B Chr3). The two reciprocal congenic strains were derived by a \u2018speed congenic\u2019 approach involving at least eight backcrosses with the respective background strains. Both congenic lines were homozygous within their respective congenic intervals and genotyped exclusively for background strain alleles using 100 microsatellite markers scattered throughout the non-congenic genome. Congenic mice were genotyped every 6 months through genetic marker-based polymerase chain reaction. All primers for simple sequence repeat (SSR) element markers were bought from Research Genetics (Huntsville, Ala.). Two strains of Chr 3 congenic mice were generated and maintained: B.D Chr3 (14 cM\u02dc33 cM) congenics with D2 QTL being introgressed onto the B6 background, and their reciprocal congenic strains D.B Chr3 (19 cM\u02dc60 cM) with B6 QTL on a D2 background. B6.SJL (Ptprc[CD 45.1]) mice from Charles River laboratories (Frederick, Md.) were used as transplantation recipients in competitive repopulation experiments. Female mice (6 to 10 weeks old) were housed in the animal facilities of the University of Kentucky under pathogen-free conditions according to NIH-mandated guidelines for animal welfare. They were fed with acidified water and food ad libitum.","To quantify primitive hematopoietic cells derived from bone marrow in B6, D2, and 26 BXD RI strains, an in vitro cobblestone area-forming cell (CAFC) assay was performed. In this assay, each cobblestone area represents a population formed by clonal amplification of a single stem cell or a progenitor cell, and distinct populations arise in a chronological continuum. For determining the relative chronological age of a hematopoietic cell, the latency period is determined by measuring the duration before clonal expansion by a primitive hematopoietic cell. A longer latency period indicates a more primitive hematopoietic cell. Cobblestone areas were counted after 35 days of culture, a late time-point. Primitive hematopoietic cells measured at this time-point correlate strongly with long-term repopulating stem cells in vivo.","Whole bone marrow cells and sorted LSK cells, described below in Example 5 and , were evaluated in a cobblestone area-forming cell (CAFC) assay. In brief, a confluent monolayer of FBMD-1 stromal cells was established in 96-well tissue culture-treated plates (Costar, Cambridge, Mass.). After 7 to 10 days, wells were seeded either with unfractionated marrow at a dose of 81,000, 27,000, 9,000, 3,000, 1,000, or 333 cells per well, or with sorted LSK cells at a dose of 1, 3, 10, or 30 cells per well using an automated cell deposition unit. Twenty or 60 replicate wells per cell number were evaluated in experiments with unfractioned or sorted cells, respectively. The cells were cultured in Iscove's Modified Dulbecco Medium (IMDM), containing 20% horse serum, 80 U\/mL penicillin, 80 mg\/mL streptomycin (all from Life Technologies), 10\u03b2-mecaptoethanol, and 10M hydrocortisone (both Sigma, St. Louis, Mo.). Individual wells were screened at day 7, 14, 21, 28 and 35 for the presence of a cobblestone area, defined as a colony of at least 5 small, non-refractile cells growing underneath the stroma. A longer latency period before the appearance of cobblestones correlates with a more primitive state of the stem cell. Thus, the most primitive HSCs show cobblestones at day 35, whereas colonies that appear earlier are derived from more committed progenitor cells (HPCs). Frequencies of CAFCs were calculated by using maximum likelihood analysis and are equal to 1 divided by the number of cells yielding 37% negative wells. The frequencies of CAFC day35 in BXD RI mouse strains were measured, and were used to perform genome-wide searches for linked loci within a GeneNetwork database.","A limiting-dilution analysis in competitively repopulated host animals was performed over a long term (20 week) period in order to functionally identify and to quantify HSCs in vivo. Graded numbers of bone marrow cells from B.D Chr3 congenic or B6 mice were first admixed with a fixed dose of competitor cells, and were transplanted into groups of lethally-irradiated mice. Lymphomyeloid reconstitution by either B.D Chr3 or B6-derived cells was assessed in the peripheral blood cells 20 weeks after transplantation. In limiting-dilution competitive repopulation assay, graded numbers (6,000; 20,000; and 60,000) of B.D Chr3 congenic or B6 \u201ctest\u201d cells (CD45.2) were admixed with a radio-protective dose (2\u00d710) of competitor cells (CD45.1), and were injected intravenously into lethally irradiated (900 Gy) CD45.1 recipient mice.","Recipients were bled from the retro-orbital sinus 20 weeks after transplantation. Peripheral blood cell counts were performed on anesthetized mice bled from the retro-orbital venous plexus. Circulating leukocyte, erythrocyte, and platelet counts were measured by analysis of 40 ul blood using a System 9118 Hematology Series Cell Counter (Biochem Immunosystems, Allentown, Pa. Erythrocytes were depleted by hypotonic lysis using NHCl. The leukocytes were stained in triplicate with fluorescein isothiocyanate (FITC)-conjugated anti-CD45.2 monoclonal antibody (mAb) (clone ALI4A2) and phycoerythrin (PE)-conjugated mAbs (Becton-Dickinson-PharMingen, San Diego, Calif.) specific for either B (anti-CD45R\/B220; clone RA3-6B2) or T lymphocytes (anti-Thy-1.2; clone 30H12), or granulocytes (anti-Ly6G\/Gr-1; clone RB6-8C5), and macrophages (anti-CD11 b\/Mac-1; clone M1\/70). Samples were analyzed using a FACScan instrument (Becton-Dickinson Immunocytometry Systems, San Jose, Calif.). The frequencies of long-term HSCs were calculated from the proportions of negative recipients in each cell dose group by using L-Calc software (StemCell Technologies Inc., Vancouver, BC). In each cell dose group, <5% of the circulating B, T, and myeloid cells were regenerated from CD45.2 stem cells. The results obtained from two independent experiments showing the effect of D2 alleles on the number of long-term repopulating HSCs in B.D Chr3 mice are provided below in  in Example 5.",{"@attributes":{"id":"p-0088","num":"0087"},"figref":["FIG. 3","FIG. 3"]},"To validate the results from genome-wide linkage analysis described in Example 4, CAFC day 35 assays were performed on whole bone marrow cells and LSK cells.  illustrates the CAFC day 35 frequency observed for femur cells (whole bone marrow cells derived from reciprocal congenic strains) of Chr3 congenic mice when compared to background mice. In , introgression of D2 alleles onto the B6 background in the QTL region caused nearly a 2-fold increase in CAFC day35 number (P=0.003) representing the number of HSCs in B.D Chr3 congenic mice, whereas the introgression of B6 alleles onto the D2 background caused more than a 50% decrease (P=0.02). The values are shown as mean\u00b1SEM (n>9). The absolute number of CAFC day35 per femur (\u00b1SEM) for all strains as follows: B66: 397\u00b159; B.D Chr3: 724\u00b173; D2: 1978\u00b1684; D.B Chr3; 1532\u00b1286. Moreover, numbers for CAFC day 7 and day 21, which represent hematopoietic progenitor cells (HPCs) at different stages, and the cell counts for peripheral blood leukocytes, erythrocytes, and platelets showed no differences among congenic and background strains. This suggested that the Chr 3 QTL specifically regulates HSC population size.",{"@attributes":{"id":"p-0090","num":"0089"},"figref":["FIG. 4B","FIG. 4B","FIG. 4A"]},{"@attributes":{"id":"p-0091","num":"0090"},"figref":["FIG. 4C","FIG. 4C"]},"A long-term, limiting-dilution analysis in competitively repopulated animal hosts was performed, as further described above in Example 3. Graded numbers of bone marrow cells from B.D Chr3 congenic or B6 mice were admixed with a fixed dose (2\u00d710) of competitor cells and transplanted into groups of lethally-irradiated mice irradiated to ablate their endogenous hematopoietic system. The competitor cells and recipient mice have a CD45.1 surface marker, whereas B.D Chr3 and B6 mice carry the CD45.2 marker on virtually all hematopoietic cells. This genetic marker is not presently available on the D2 background that also bears CD45.2, which precludes analogous experiments with the reciprocal congenic strain. Twenty weeks after transplantation, lymphomyeloid reconstitution by either B.D Chr3 or B6-derived cells was assessed in the peripheral blood cells 20 weeks after transplantation for the presence of CD45.2 cells by flow cytometry.","Together, these three independent studies support the conclusion that a Chr 3 QTL in the congenic intervals confers the phenotype (HSC population regulation), and confirms the results obtained by linkage analysis. In the context of these congenics, the QTL accounts for a significant portion of the natural variation of this parameter between the progenitor strains, despite \u2018suggestive\u2019 linkage of the phenotype with two other QTLs (Chr 5 and Chr18) and with the presence of several other QTLs that approach \u2018suggestive\u2019 linkage illustrated in .","The effect of latexin expression on the proliferative activity of various primitive cells was assessed using two independent approaches. First, in vivo BrdU incorporation was used to determine cells in the S phase of the cell cycle. Mice were injected intraperitoneally with BrdU (1 mg per kg body weight) and sacrificed 90 minutes later. Bone marrow LSK cells were identified as described above. Analysis of BrdU incorporation in LSK was performed using BrdU Flow Kit (Pharmingen, San Diego, Calif.) according to the manufacturer's instruction. Second, the fraction of cells killed by a 1-hour in vitro incubation with hydroxyurea (HU) was measured. Hydroxyurea (HU) is an agent that blocks DNA synthesis and stops cell division by inhibiting ribonucleotide reductase. Briefly, all bone marrow cell suspensions were diluted to a concentration of 1\u00d710cells in 1 mL. Hydroxyurea (Sigma, St Louis, Mo.; 200 ug\/mL, a total volume of 10 ul) was added to one sample, and incubated with the control sample for 1 hour at 33\u00b0 C. The standard CAFC assay was then used to assay both HU-treated and control samples. The fraction of cells killed by HU was calculated by dividing the CAFC frequency in the HU-treated sample by the control value.",{"@attributes":{"id":"p-0095","num":"0094"},"figref":["FIG. 5A","FIG. 5A"]},"To determine the effect of QTL on cell cycle kinetics, the CAFC day7 progenitor cells killed by hydroxyurea (HU) in the reciprocal congenic strains, B6, and D2 mice were measured.  illustrates the effect of Chr 3 QTL on the cell-cycle kinetics of the progeny of HSCs, also referred to as hematopoietic progenitor cells (HPCs). Proliferation of HPCs measured by CAFC day7 cells killed by hydroxyurea (HU) in reciprocal congenic and B6, D2 strains. Data are shown as the average (\u00b1SD) of 3 independent experiments (n\u22679). * indicates the comparison between B6 and D2, and ** indicates the comparison between congenics and their respective background mice. P values are shown in the figure. The results showed that the D2 alleles in and around the Chr 3 QTL in a B6 background quantitatively can reproduce the higher killing rate observed in D2 mice. Similarly, B6 alleles on a D2 background reproduced the entire variation in this trait between B6 and D2 mice. These results underscore the effect of the Chr 3 QTL on cell-cycle kinetics of HSC and their immediate progeny. Interestingly, progenitor cell numbers and differential blood cell counts showed no differences among the congenic and background strains, showing that compensatory mechanisms during downstream amplification of differentiating cells obviated the variation in the final cell outputs in the multiple blood cell lineages.","The self-renewal capacity of B.D Chr3 congenic HSCs was tested in serial transplant experiments in which B.D Chr3 congenic HSCs and B6 marrow cells were co-transferred to recipients so that both types of cells competed for engraftment. In serial bone marrow transplantation, B.D Chr3 congenic or B6 \u201ctest\u201d cells were injected into the primary recipients along with equal number of competitor cells (1\u00d710). Sixteen weeks after transplant, marrows were harvested from the primary recipients, and were transplanted into secondary recipients. An identical regimen was repeated twice more, culminating in engrafted quaternary hosts. At each transplant, the peripheral blood, bone marrow, and bone marrow LSK cells were analyzed. The competitive advantage and self-renewal capacities were analyzed by comparing the percentage of B.D Chr3 congenic-derived or B6-derived cells at each round of transplantation.",{"@attributes":{"id":"p-0098","num":"0097"},"figref":["FIG. 5C","FIG. 5C"],"sup":"th "},"To determine whether allelic differences in the Chr 3 congenic interval affected steady-state apoptosis in HSC population, apoptosis was assessed by Annexin V-staining of bone marrow LSK cells.",{"@attributes":{"id":"p-0100","num":"0099"},"figref":"FIG. 5D"},{"@attributes":{"id":"p-0101","num":"0100"},"figref":["FIG. 5E","FIG. 5E"]},"Introgression of D2 alleles around the Chr 3 QTL, in the B.D Chr3 congenic, resulted in an apoptotic LSK subpopulation (0.6%\u00b10.1%) similar to that of D2 mice, which suggests that the congenic interval conferred the entire phenotype. Surprisingly, analysis of marrow from the reciprocal congenic strain revealed a very low (0.3%\u00b10.03%) apoptotic LSK population. Thus, B6 alleles at and around the QTL do not confer the B6 phenotype onto D2 HSCs. In fact, apoptosis was significantly (P=0.02) less than in the D2 background strain. This result may be due to genes in the mutually exclusive congenic intervals of the reciprocal strains, or genes in the D2 background having a strong negative effect on HSC apoptosis. The results suggest that the QTL might act through a mechanism influencing apoptosis in HSCs.","Oligonucleotide arrays of LSK cells were performed by the Microarray Facility Center at the University of Kentucky. Briefly, total RNA were extracted from at least 550,000 sorted LSK cells from a minimum of 40 mice using RNeasy\u00ae Kit (QIAGEN, Valencia, Calif.). The extracted RNA was reverse transcribed into cDNA, which was subsequently used to synthesize biotin-labeled cRNA. The labeled cRNA was then fragmented and hybridized to the Mouse Genome U74Av2 chips (MGUAv2, Affymetrix, Santa Clara, Calif.). The genechips were then washed, stained, and subsequently scanned for quantification of gene expression. Three independent biological samples were obtained for each strain and each was run on an individual chip. The gene expression levels were compared among all 4 strains using one way-ANOVA with a statistical cutoff of P<0.05. The genes that were differentially expressed were screened for genomic location and function of the gene products.","Oligonucleotide arrays were utilized to screen for genes in the consensus congenic interval whose differential expression may account for the observed phenotypic variations. LSK cells were sorted from bone marrow of the reciprocal congenic mice, and the two background strains mice. Large independently sorted biological samples for total RNA extraction were used to avoid representational skewing of low abundance mRNAs that may accompany amplification by PCR. Triplicate RNA samples, each independently obtained from at least 550,000 LSK cells sorted and pooled from a minimum of 40 mice, were collected from the four strains. Each RNA sample (4 strains\u00d73 replicates=12 in all) was hybridized to an individual microarray chip (MGU Av2 Affymetrix). Results obtained from the triplicate chips for each strain were analyzed using Affymetrix Microarray Suite\u2122 software, and were compared using one-way ANOVA for each congenic-background strain pair. The number of genes differentially expressed between B.D Chr3 and B6 strains was 96 (P<0.05) and between D.B Chr3 and D2 strains was 84 (P<0.05). Because congenic intervals represent less than 1% of the entire genome, these numbers are commensurate with the 940 differentially expressed genes measured between B6 and D2 LSK cells in these experiments.","Table 4 is a list of genes differentially expressed in reciprocal Chr 3 congenic and B6, D2 HSCs, as shown below. In Table 4, only 17 genes on 11 different chromosomes were mutually inclusive in comparisons between the reciprocal congenics and their respective background strains. Of 17 differentially expressed genes, the D2 and B6 alleles had opposite effects on transcription in 9\/17. 4\/17 were mapped to Chr 3, but only \u00bc was located in the consensus congenic interval, near the marker of highest linkage in the mapping (D3Mit5). The gene was identified to be latexin (Lxn), whose expression was upregulated by B6 alleles and down-regulated by D2 alleles, a pattern suggested by the linkage analysis.",{"@attributes":{"id":"p-0106","num":"0105"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"441pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Genes differentially expressed in reciprocal Chr 3 congenic and B6, D2 HSCs."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"133pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"126pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},"Chromosomal","Entrez",{}]},{"entry":["Probe Set","Gene title","Gene symbol","location (Mb)","gene ID","1. Molecular function"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["101209_at","Fc receptor, IgE, high affinity I, alpha","Fcer1a","\u20021 (173)","14125","Receptor activity; IgE binding"]},{"entry":[{},"polypeptide"]},{"entry":["102260_at","Growth factor independent 1B","Gfi1b","2 (28)","14582","ATP binding"]},{"entry":["96065_at","Latexin","Lxn","3 (67)","17035","Enzyme inhibitor activity"]},{"entry":[{},{},{},{},{},"Metalloendopeptidase inhibitor activity"]},{"entry":["92770_at*","S100 calcium binding protein A6 (calcyclin)","S100a6","3 (91)","20200","Calcium ion binding; Protein binding"]},{"entry":[{},{},{},{},{},"Growth factor activity"]},{"entry":["98600_at","S100 calcium binding protein A11","S100a11","3 (93)","20195","Cytokine activity; Calcium ion binding"]},{"entry":[{},"(calizzarin)"]},{"entry":["103257_at","RIKEN cDNA 4930577M16 gene","4930577 M16Rik","\u20023 (121)","99887","Unknown"]},{"entry":[{},"(transmembrane protein 56)","(Tmem56)"]},{"entry":["94559_at","General transcription factor III A","Gtf3a","\u20025 (145)","66596","DNA binding"]},{"entry":["102011_at","RIKEN cDNA 2610507L03 gene","2610507L03Rik","7 (23)","72140","Unknown"]},{"entry":["94286_at","RIKEN cDNA 9130011J15 gene","9130011J15 Rik","8 (71)","66818","Unknown"]},{"entry":["93498_s_at","Amyloid beta (A4) precursor-like protein 2","Aplp2","9 (31)","11804","DNA binding; Protein binding"]},{"entry":[{},{},{},{},{},"Serine-type endopeptidase inhibitor"]},{"entry":[{},{},{},{},{},"activity"]},{"entry":["101026_at*","Pituitary tumor-transforming 1","Pttg1","11 (43)\u2002","30939","Cysteine protease inhibitor activity"]},{"entry":["16128_at#","MYB binding protein (P160) 1a","Mybbp1a","11 (72)\u2002","18432","Protein binding Transcriptional repressor"]},{"entry":[{},{},{},{},{},"activity"]},{"entry":["93134_at","Neuronal pentraxin 1","Nptx1","11 (119)","18164","Calcium ion binding"]},{"entry":["100391_at","Mitogen activated protein kinase 8","Mapk8","14 (29)\u2002","26419","Kinase activity; Transferase activity"]},{"entry":["100343_at*","Tubulin, alpha 1","Tuba1","15 (99)\u2002","22142","GTPase activity; GTP binding"]},{"entry":[{},{},{},{},{},"Structural constituent of cytoskeleton"]},{"entry":["103282_at*","RAS, guanyl releasing protein 2\/\/","Rasgrp2\/\/","19 (6)\u2003","19395 \/\/","Guanyl-nucleotide exchange factor activity"]},{"entry":[{},"similar to calcium and DAG-regulated","LOC381240",{},"381240","Calcium ion binding; Diacylglycerol"]},{"entry":[{},"guanine nucleotide exchange factor I",{},{},{},"binding"]},{"entry":[{},{},{},{},{},"Kinase activity"]},{"entry":["160391_at","Fatty acid desaturase 1","Fads1","19 (9)\u2003","76267","Linoleoyl-CoA desaturase activity"]},{"entry":[{},{},{},{},{},"Oxidoreductase activity"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":"Total RNA isolated from LSK cells in B.D Chr3, D.B Chr3 congenic, and B6, D2 mice was reverse transcribed into cDNA and subsequently used for the synthesis of cRNA, cRNA was labeled with biotin and hybridized onto Affymetrix Mouse Genome U74Av2 chip. Gene expression in LSK cells was quantified and averaged from 3 independent experiments. The gene expression levels were compared between congenic and repective background mice using one way-ANOVA with a statistical cutoff of p < 0.05. 17 genes are differentially expressed among all 4 strains."},{"entry":"#indicates the genes whose expression is up-regulated by the B6 alleles (high expression in D.B Chr3 and B6 cells) but down-regulated by the D2 alleles (low expression in B.D Chr3 and D2 cells), whereas"},{"entry":"*indicate the genes whose expression is upregulated by the D2 alleles but down-regulated by the B6 alleles."}]}}]}}},{"@attributes":{"id":"p-0107","num":"0106"},"figref":["FIG. 6A","FIG. 6A","FIG. 6A"],"sup":"5"},"Lxn expression levels were determined along the differentiation pathway in the hematopoietic system.  illustrates quantitation of Lxn transcripts by real-time PCR in peripheral blood leukocytes, whole bone marrow, Lin-negative bone marrow cells, and LSK stem cells. In , the same number of peripheral blood leukocytes, whole bone marrow, Lin-negative bone marrow cells, and LSK stem cells in each of the four strains were evaluated. Two points are noteworthy from the compiled results of three independent experiments. First, overall levels of Lxn expression per cell in each of the populations increase in concert with the content of primitive hematopoietic cells. Second, Lxn expression per cell in each of the cell populations accurately reflects the presence of B6 (higher expression) or D2 (lower expression) alleles in the two congenic and background strains. Lxn expression in differentiated peripheral blood leukocytes (PB), bone marrow nucleated cells and primitive hematopoietic cell-enriched population, including lineage-negative cells (Lin\u2212) and HSCs (LSK cells). Lxn mRNA level was measured by quantitative real-time PCR and shown as mean (\u00b11 SD) (n=12).","Quantitative real-time PCR was performed as follows. Identical numbers (200,000) of cells were used for total RNA extraction in each type of cell population (peripheral blood, bone marrow, Lin-negative and LSK cells) for each mouse strain. Isolated total RNA was reverse transcribed into cDNA using random hexamers in a TaqMan\u00ae reverse transcription solution (PN N8080234) and stored at \u221220\u00b0 C. In real-time PCR reactions, primer and probe mix for Lxn were purchased from Applied Biosystems (Foster city, CA, USA). TaqMan\u00ae rodent glyceraldehyde-3-phosphate dehydrogenase (GAPDH) (PN 4308313) served as an endogenous control to normalize Lxn expression. PCR reactions were set up according to manufacturer's instructions using TaqMan\u00ae universal PCR master mix (PN 4304437). Analyses of gene expression were performed in single reporter assays in an ABI PRISM 7700 sequence detection system (PE Biosystems, Foster city, CA, USA).","A rabbit anti-mouse LXN antibody was generated from the LXN-specific amino acid sequence CKHNSRLPKEGQAE at the carboxyl terminus. Because Lxn transcripts were present at reasonable levels in unfractionated bone marrow and in Lin-negative cells described above in  of Example 11, the cell lysates derived from these cells were employed for Western blot analysis.",{"@attributes":{"id":"p-0111","num":"0110"},"figref":["FIG. 6C","FIG. 6C"]},{"@attributes":{"id":"p-0112","num":"0111"},"figref":"FIG. 6D"},"Western blots were performed as follows. Cell samples were lysed at a concentration of 2\u00d710cells\/ml in protein lysis buffer containing: 10 mM Tris pH7.5, 50 mM NaCl, 30 mM sodium pyrophosphate, 50 mM NaF, 5 \u03bcM ZnCland 1% Triton X-100, 2.8 ug\/ml aprotinin (Sigma), 1 mM phenylmethylsulfonyl fluoride (Sigma), 1 mM sodium vanadate (NaVO) 1 ug\/ml pepstatin, and 1 \u03bcg\/ml leupeptin (Oncogene Research, MA, USA). Lysate was incubated on ice for 30 min, and then centrifuged at 15,000\u00d7g for 10 minutes to remove debris. The resulting supernatant was then aliquoted and stored at \u221280\u00b0 C. For Western blot, protein lysates were thawed and mixed with running buffer and a reducing agent (Novex, San Diego, Calif., USA, per manufacturer's instructions) and heated at 95\u00b0 C. for 5 minutes. Samples were then analyzed by denaturing PAGE (Novex, 10% bis-Tris gel) using the equivalent of 4\u00d710cells per lane. Following electrophoresis, samples were electro-transferred onto immunobilon-P membranes (Millipore, Bedford, Mass., USA), which were subsequently blocked and probed with polyclonal rabbit anti-Lxn Ig-G antibody at a 1:3000 dilution. This antibody was generated from the Lxn-specific amino acid sequence CKHNSRLPKEGQAE at the carboxyl terminus, and was produced by Bethyl Laboratories, Inc (Montgomery, Tex.). Primary antibodies were detected using alkaline phosphatase-conjugated secondary antibodies (Santa Cruz Biotechnology) and electro-chemifluorescent (ECF) reagent (Pharmacia Biotech) according to the manufacturer's instructions. Blots were visualized using a Molecular Dynamics STORM 860 system and Imagequant Software. Following the detection and quantification of anti-Lxn antibody, immunobilon-P membrane was sequentially stripped in 40% methanol and the buffer containing 100 mm \u03b2-mercaptoethanol, 2% sodium dodecyl sulfate and 62.4 mM Tris-HCl to remove ECF reaction product and antibodies, respectively. The stripped membrane was re-probed with anti-actin antibody (Sigma) at 1:500,000 dilution and detected as described previously.","Because Lxn expression was detected and quantified in LSK cells of the 30 BXD RI strains in the GeneNetwork database, a linkage analysis to search for a QTL that could modify Lxn expression was performed. When modifiers of Lxn expression were queried using such an approach, suggestive linkage (LRS>9.6) was obtained to a genomic interval spanning from 56.9 Mb to 66.8 Mb on Chr 3, very close to the genomic locations of D3Mit5 and Lxn, a finding consistent with cis-regulation. Moreover, the negative additive effects of all markers in this genomic interval suggest that the B6 allele increases Lxn expression, which corroborates our results from microarray, real-time PCR, and Western blot analyses. Therefore, these results not only suggest the existence of a regulatory element in the upstream region of the Lxn sequence, but also corroborate D3Mit5 as a marker, presumably the primary one, for a QTL directly or indirectly responsible for the phenotypes conferred by the Chr 3 QTL.",{"@attributes":{"id":"p-0115","num":"0114"},"figref":["FIG. 7A","FIG. 7A"]},{"@attributes":{"id":"p-0116","num":"0115"},"figref":"FIG. 7B"},"Thus, differential methylation status in the regulatory region of Lxn may be involved in the differential expression of Lxn. The results also suggest that Lxn may be a potential \u201ctumor progenitor gene\u201d which plays an occult role in early stages of cancer development. Taken together, each of these possible pathways may be synergistically or independently involved in the regulation of cycling, apoptosis, and\/or self-renewal of HSCs with entrained consequences for its population size.","Because D2 and B.D Chr3 congenic mice have lower levels of Lxn expression and higher stem cell numbers, Lxn expression in both strains were enhanced in both strains to test the effect of Lxn overexpression on stem cell numbers.",{"@attributes":{"id":"p-0119","num":"0118"},"figref":["FIG. 8A","FIG. 8A"]},{"@attributes":{"id":"p-0120","num":"0119"},"figref":"FIG. 8B"},{"@attributes":{"id":"p-0121","num":"0120"},"figref":"FIG. 8C"},{"@attributes":{"id":"p-0122","num":"0121"},"figref":"FIG. 8D"},"In , the frequency of primitive stem cells was significantly reduced (P<0.05) in the population infected with the Lxn-GFP vector relative to cells infected with the GFP control vector. Approximately a 3-fold decrease in HSC numbers in Lxn-overexpressing cells reflects precisely the difference between two parental B6 and D2 strains. Therefore, these results further confirm Lxn as the primary quantitative gene negatively regulating HSC numbers.","Retroviral vectors were constructed as follows. The retroviral vector, Sfbeta 91, served as a control and the backbone for cloning of Lxn cDNA. It contained the 5\u2032-long terminal repeat (LTR) derived from myeloproliferative sarcoma virus (MPSV) and a 3\u2032-LTR derived from spleen focus forming virus (SFFV). The internal ribosomal entry site sequence derived from the encephalomyocarditis virus was used for simultaneous translation of gene insert and the gene for enhanced green fluorescent protein (GFP). The Lxn cDNA sequence was cloned upstream of the IRES of the Sfbeta91 vector (MPSV-IRES-GFP-SFFV) to create a recombinant Lxn-carrying vector (MPSV-Lxn-IRES-GFP-SFFV). Production of high-titer helper-free retrovirus was carried out by standard procedure in ectotropic Phoenix packaging cells.","Infection of primary murine bone marrow cells were infected as follows. Primary mouse bone marrow cells were transduced as previously described with modifications specified below. Briefly, bone marrow cells were harvested from mice treated 4 days previously with 150 mg\/kg body weight 5-fluorouracil (5-FU) (Sigma, St. Louis, Mo.) and cultured for 24 hours in Dulbecco's modified eagle's medium (DMEM) supplemented with 10% fetal bovine serum (FBS), 1% penicillin\/streptomycin (Gibco Technologies, Carlsbad, Calif.), 50 ng\/mL recombinant mouse stem cell factor (mSCF), 10 ng\/mL mouse interleukin 6 (mIL-6), and 10 ng\/mL mouse interleukin 3 (mIL-3) (R&D Systems, Minneapolis, Minn.). The cells were then harvested and slowly spread on the top surface of the membrane of a Transwell insert (Corning Incorporated Life Sciences, Acton, Mass.) at a density of 2\u00d710cells per well. The viral supernatants were added to the Transwell inserts along with 4 \u03bcg\/ml of polybrene and were cultured with cells for further 48 hours. The viral supernatant was changed 3 times during this period of time. The recovered cells were then transplanted into sub-lethally-irradiated (600 Rad) recipient mice at a dose of at least 1\u00d710cells per mouse. Around 12 weeks after transplantation, retrovirally-transduced, i.e. GFP positive, bone marrow cells were flow cytometrically sorted from primary recipient mice using a FACSVantage (Becton-Dickinson) and used for CAFC assay and limiting-dilution competitive repopulation assay.","Functional analysis of retrovirally-transduced cells were performed as follows. Identical numbers of bone marrow cells transduced with either the GFP control vector or the Lxn-GFP vector were sorted from the primary recipient mice. The CAFC and limiting-dilution competitive repopulation assays of these GFP positive cells was performed as described above. The HSC frequencies (CAFC day35 and LTRC) were calculated and compared to measure the effects of Lxn overexpression on HSC number.",{"@attributes":{"id":"p-0127","num":"0126"},"figref":["FIG. 9A","FIG. 9B"],"sup":"5 "},"Quantitative real-time PCR was performed in the following manner. Identical numbers (200,000) of cells were used for total RNA extraction in cells transfected with 3 SiRNAs and negative control RNAs. Isolated total RNA was reverse transcribed into cDNA using random hexamers in a TaqMan\u00ae reverse transcription solution (PN N8080234) and stored at \u221220\u00b0 C. Quantitative real-time PCR analyses of selected genes were performed in single reporter assays with an ABI PRISM 7700 sequence detection system (PE Biosystems, Foster city, CA, USA). PCR reactions were set up according to manufacturer's instructions using TaqMan\u00ae universal PCR master mix (PN 4304437), and the primer and probe mixes for each tested gene were purchased from Applied Biosystems (Foster city, CA, USA). TaqMan\u00ae rodent glyceraldehyde-3-phosphate dehydrogenase (GAPDH (PN 4308313) served as an endogenous control to normalize gene expression. The analysis of gene expression was performed by the relative standard curve method. In brief, standard curves were prepared for the GAPDH control and for each of tested genes, and the amount of GAPDH control and test gene were determined from the standard curve. A normalized test gene value was determined relative to GAPDH expression. Each biological sample was obtained from at least 5 mice, and each sample was run with at least 5 replicates within an experiment. Three replicate biological experiments were performed in each strain.","It will be appreciated that, although specific embodiments of the invention have been described herein for purposes of illustration, various modifications may be made without departing from the spirit and the scope of the invention. Accordingly, the invention is not limited except as by the appended claims."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF DRAWINGS","p":[{"@attributes":{"id":"p-0012","num":"0011"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0013","num":"0012"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0014","num":"0013"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0015","num":"0014"},"figref":"FIG. 4A"},{"@attributes":{"id":"p-0016","num":"0015"},"figref":"FIG. 4B"},{"@attributes":{"id":"p-0017","num":"0016"},"figref":"FIG. 4C"},{"@attributes":{"id":"p-0018","num":"0017"},"figref":"FIG. 5A"},{"@attributes":{"id":"p-0019","num":"0018"},"figref":"FIG. 5B"},{"@attributes":{"id":"p-0020","num":"0019"},"figref":"FIG. 5C"},{"@attributes":{"id":"p-0021","num":"0020"},"figref":"FIG. 5D"},{"@attributes":{"id":"p-0022","num":"0021"},"figref":"FIG. 5E"},{"@attributes":{"id":"p-0023","num":"0022"},"figref":"FIG. 6A"},{"@attributes":{"id":"p-0024","num":"0023"},"figref":"FIG. 6B"},{"@attributes":{"id":"p-0025","num":"0024"},"figref":"FIG. 6C"},{"@attributes":{"id":"p-0026","num":"0025"},"figref":"FIG. 6D"},{"@attributes":{"id":"p-0027","num":"0026"},"figref":"FIG. 7A"},{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 7B"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 8A"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 8B"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 8C"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 8D"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 9A"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 9B"}]},"DETDESC":[{},{}]}
