---
title: Systems for induction of gene expression and protein depletion in yeast
abstract: A system allows for rapid and specific induction of individual genes in eukaryotic cells using a chimeric transcriptional activator that is responsive to hormone inducer. Upon addition of the hormone, cytoplasmic transcriptional activator localizes to the nucleus and subsequently binds to promoters containing sequences that bind to its DNA-binding domain. Genetic modifications allow for rapid and specific degradation of a targeted protein upon addition of hormone by means of a regulated degron method that utilizes a protease variant. This system is useful for discovering new compounds by high throughput screening when introducing compound libraries to these protein-depleted cells.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09212359&OS=09212359&RS=09212359
owner: 
number: 09212359
owner_city: 
owner_country: 
publication_date: 20120615
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS-REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","SEQUENCE LISTING","FIELD OF THE INVENTION","BACKGROUND","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","Definitions","Embodiments","EXPERIMENTAL RESULTS","Example 1","Example 2","Quantitation of GEV Localization","Example 3","Kinetics of GEV-Mediated Gene Induction","Example 4","GEV-Mediated Gene Induction Shows a Graded Response","Example 5","The Connection Between GEV Expression and Cell Growth","Example 6","Example 7","Example 8","Using GEV Overexpression to Probe Kinetics of Transcriptional Regulatory Networks","Example 9","Example 10","Characterization of ZEV and 4ZEV Activity","REFERENCES"],"p":["This application claims benefit of U.S. Provisional Applications No. 61\/500,367, filed Jun. 23, 2011, No. 61\/608,788, filed Mar. 9, 2012, and No. 61\/618,248, filed Mar. 30, 2012, which are hereby incorporated in their entirety by reference.","This invention was made with government support under Grant GM046406 and Grant GM071508 awarded by the National Institutes of Health. The government has certain rights in this invention.","The instant application contains a Sequence Listing which has been submitted in ASCII format via EFS-Web and is hereby incorporated by reference in its entirety. Said ASCII copy, created on Jun. 13, 2012, is named PR-20776.txt and is 19,939 bytes in size.","The invention relates to the control of protein abundance in eukaryotes. More specifically, the invention relates to induction of targeted protein and to proteasomal degradation of specific protein.","Previous studies of the dynamics and functional relationships of genes and the networks they comprise followed gene expression after a well-defined perturbation to a culture growing in steady state. Traditionally, such perturbations involved a pulse of preferential carbon source, heat or nutrients (Rand B2006). However, when one wants to perturb only a single element of the system, such as a transcription factor, use of the traditional perturbation methods result in side effects on physiology and gene expression that can be deleterious to the experiment. What is needed to resolve this problem is finding specific ways of turning on (or off) only a single transcription factor.","Significant effort in the prior art had been placed on achieving tightly controlled gene expression systems across a spectrum of organisms, guided by the transformative discoveries of gratuitous inducers of expression of the lactose operon genes in . The significant desirable features of a useful induction system include (1) a complete lack of target gene activation in the absence of inducer, (2) rapid induction of a target gene in the presence of inducer, and (3) the availability of a truly gratuitous inducer. A gratuitous inducer should interact with no other regulatory system and should be essentially inert with respect to physiology.","A common approach in mammalian cells is a direct adaptation of bacterial systems: tetracycline-inducible tetR-based systems (Let al. 1990). By fusing activating domains (e.g., VP16) to tetR and expressing this chimeric transcription factor, transcription of a target gene downstream of a minimally active promoter with the appropriate operator sites can be maintained at high levels in the absence of inducer. TetR-VP16, also called tTA (tetracycline transactivator), unbinds DNA in the presence of tetracycline, thereby repressing transcription. This configuration is referred to as tet-off. Tet-off has been used for studying the effects of depletion of a target gene, but the kinetics of transcriptional shut off do not result in rapid elimination of the protein products of the regulated genes, as many proteins have long half-lives. Therefore, a better method for rapid depletion of a target gene is needed.","The most widely used overexpression system in has utilized GAL4-mediated induction of targets, which are placed downstream of promoters containing 17-mer UASgal sites (such as P, P, P), CGG-N-CCG (SEQ ID NO: 1) (Get al. 1985). The UASgal motif is recognized by dimerized GAL4 proteins (Het al. 2008; Met al. 1992). Deletion of GAL1 has made galactose overexpression of GAL-driven targets nearly gratuitous (Het al. 1989). Since GAL genes are catabolite repressed, activation required that cells be grown in relatively poor, non-glucose carbon sources prior to the galactose pulse.","A more convenient system was developed that avoids the necessity of switching the carbon source. A chimeric transcriptional activator called GEV (Let al. 1993) contains the DNA binding domain of Gal4p, the hormone binding domain (HBD) of the human estrogen receptor (hER), and the highly electronegative portion of herpes simplex virus protein VP16, which confers strong transcriptional activity (Set al. 1988). Though GEV binds DNA via a GAL4 DNA binding domain, it is not subject to inhibition or repression by glucose, making it feasible to conduct induction and overexpression experiments in standard glucose-containing media simply by the addition of the inducer \u03b2-estradiol. The presence of the HBD provides a simple on\/off switch for GEV activity; steroid receptor binding of ligand results in a conformational change of the receptor and its subsequent disassociation from the Hsp90 chaperone complex (Pand T1997).","Previously, improvements over the original GEV system were made by placing GEV expression under the control of low-strength constitutive promoters to reduce errant expression of target genes in the absence of \u03b2-estradiol (Gand P2000; Vet al. 2009). Another GEV-based induction system implemented an autocatalytic approach to rapidly increase GEV production in the presence of \u03b2-estradiol. In this case, GEV was placed under the control of the GAL1 promoter, which contains 4 UASgal sites (Qet al. 2007). However, the prior art only envisioned GEV for induction of gene expression.","Genetically modified yeast cells were created by a novel method involving transformation with a DNA construct to modify any target gene so that when expressed, a degradation signal (a dormant N-degron) is placed at the amino-terminus of the protein. Following transformation, the selectable drug marker used for the transformation was eliminated. The recipient cells were also modified to contain a hormone-responsive transcription factor and a protease-encoding gene induced by this transcription factor. Introduction of hormone into the cell medium results in activation of the transcription factor with minimal effects on the physiology and transcriptome of the cells except for target-specific degradation and downstream effects. The protease cleaves the dormant N-degron to initiate N-end rule degradation of the target protein.","The novel cell system allows rapid, specific target protein degradation in glucose grown cells of yeast such as . Additionally, the transcription factor, GEV, also rapidly induces single genes in a hybrid diploid yeast (e.g., ). The N-degron\/GEV method extends to yeast, generally and, more generally extends to eukaryotes.","One aspect of the invention is a method of targeted protein depletion in a eukaryotic cell. The eukaryotic cell comprises (a) a target gene having a modified degron sequence at the initiator codon of the open reading frame, the target gene being capable of expressing a target protein; (b) a protease-encoding gene that expresses a protease that is capable of hydrolyzing the target protein to yield a protein having an N-end rule destabilizing amino acid at the amino terminus; and (c) a transcriptional activator comprising a DNA-binding domain and a hormone-binding domain, wherein, when bound by a hormone, the transcriptional activator induces expression of the protease-encoding gene. The method comprises exposing the eukaryotic cell to a hormone by contacting the cell with an effective amount of a hormone that binds to the hormone binding domain of the transcriptional activator, inducing expression of the protease by the protease-encoding gene. The protease cleaves the target protein to expose the N-end rule destabilizing amino acid and the target protein is degraded.","In one embodiment of this method, the DNA-binding domain is a GAL4 DNA binding domain, a zinc finger domain, or a DNA-binding domain of a transcriptional activator that is not from yeast.","In another embodiment of the method the hormone-binding domain is from the human estradiol receptor, the human glucocorticoid receptor or the human mineralocorticoid receptor.","In yet another embodiment of the method the transcriptional activator is GEV, the hormone is \u03b2-estradiol, and the protease-encoding gene encodes the tobacco etch virus (TEV) protease.","In still another embodiment of the method the protease-encoding gene is driven by a synthetic promoter operably linked to a modified TEV protease-coding region, a spliceosome subunit p14-peptide coding region, a terminator, and a selectable marker.","In another embodiment of the method the protease is TEV, HRV 2A or HRV 3C.","A related aspect of the invention is a eukaryotic cell comprising: (a) a target gene having a modified degron sequence at the initiator codon of the open reading frame, the target gene being capable of expressing a target protein, (b) a protease-encoding gene that expresses a protease capable of hydrolyzing the target protein to yield a cleaved target protein having an N-end rule destabilizing amino acid at the amino terminus, and (c) a transcriptional activator comprising a hormone-binding domain, wherein when bound by the hormone, the transcriptional activator induces expression of the protease-encoding gene.","In one embodiment, the cell's transcriptional activator is GEV, ZEV or 4ZEV, and the hormone is \u03b2-estradiol.","In yet another embodiment, the invention comprises modifying the chimeric transcriptional activator so that another DNA binding domain substitutes for the GAL4 DNA binding domain. Additionally or alternatively the transcriptional activator can be modified to replace the hER2 receptor binding domain with that of another receptor so that instead of requiring \u03b2-estradiol, an alternative ligand would bind and serve to activate the transcription factor.","Another related aspect of the invention is a transcriptional activator that is functional in a eukaryotic cell. The activator comprises a DNA-binding domain, an activation domain, and a hormone-binding domain wherein, when bound by a hormone the transcriptional activator induces transcription. The DNA-binding domain of the transcriptional activator is from a non-yeast transcription factor or from a customized zinc finger domain that recognizes DNA sequences not found in yeast. The hormone-binding domain is from the hormone-binding domain of a eukaryotic steroid hormone receptor. The activation domain is from a polypeptide sequence that comprises an acidic region. In one embodiment, the transcriptional activator is ZEV or 4ZEV.","A further aspect of the invention is a method of targeted protein induction in a eukaryotic cell. The cell comprises (a) a gene encoding a chimeric transcriptional activator operably linked to a constitutive promoter and (b) a target gene that expresses a target protein. The transcriptional activator comprises a hormone-binding domain, an activation domain, and a DNA-binding domain. The DNA-binding domain is from a transcription factor other than yeast or comprises a zinc finger domain that is engineered or rationally designed. The target gene is operably linked to a promoter that comprises a DNA sequence recognized by the DNA binding domain of the transcriptional activator. According to the method, the cell is exposed to an effective amount of a hormone that binds to the hormone binding domain. The-hormone-bound transcriptional activator induces the target gene to express the target protein.","A related aspect of the invention is a eukaryotic cell comprising: a gene encoding a chimeric transcriptional activator and a target gene that expresses a target protein. The gene encoding the transcriptional activator is operably linked to a constitutive promoter. The transcriptional activator comprises a hormone-binding domain, an activation domain, and a DNA-binding domain, wherein the DNA binding domain is from a transcription factor other than yeast or comprises a zinc finger domain that is engineered or rationally designed. The target gene is operably linked to a promoter that comprises a DNA sequence recognized by the DNA binding domain of the transcriptional activator.","In yet another related aspect, the invention features a method of constructing a targeted protein depletion system in a yeast cell transformed with a plasmid or chromosomally-integrating form of bacteriophage lambda CRE recombinase expressed by an inducible promoter. The method comprises (a) constructing a protease cassette comprising a yeast native promoter operably linked to a modified protease-coding region, a spliceosome subunit p14 peptide coding region, a CYC1 terminator, and a first selectable marker; (b) integrating the protease cassette into a chromosome of the yeast cell; (c) constructing a modified degron\/marker cassette comprising modified degron-coding sequence and a second selectable marker, the marker being surrounded by a removable (loxP or other CRE recombinase removable) segment; (d) integrating the modified degron\/marker cassette at the initiator methionine codon of the ORF of a gene coding for a target protein; and (e) inducing the CRE recombinase with an inducer, wherein the second selectable marker is removed.","A further related aspect of the invention features a method for high throughput screening for compounds that are candidates to structurally disrupt or metabolically inhibit a target protein having the capability to suppress growth of a eukaryotic cell. The method uses eukaryotic host cells comprising (a) a gene encoding a transcriptional activator operably linked to a constitutive promoter and (b) a target gene. The transcriptional activator comprises a hormone-binding domain, an activation domain, and a DNA-binding domain, wherein the DNA binding domain is from a transcription factor other than yeast or comprises a zinc finger domain that is engineered or rationally designed. The target gene is operably linked to a promoter that comprises a DNA sequence recognized by the DNA binding domain of the transcriptional activator. The target gene is capable of expressing a target protein that is an enzyme or structural protein that suppresses cell growth at high concentration. The method comprises: (a) contacting the cells with the hormone in an amount that provides a constant level of the target protein that results in a growth deficiency of the host cells; (b) exposing the host cells to a library of the candidate compounds; and (c) determining which compounds afford a growth advantage to the host cells. The compounds that afford the growth advantage are the candidates to inhibit or disrupt the target protein.","The terms \u201cpeptide,\u201d \u201cpolypeptide,\u201d and \u201cprotein\u201d are used interchangeably herein, and refer to a polymeric form of amino acids of any length.","By \u201cconstruct\u201d is meant a recombinant nucleic acid, generally recombinant DNA, which has been generated for the purpose of the expression of a specific nucleotide sequence(s), or is to be used in the construction of other recombinant nucleotide sequences.","\u201cOperably linked\u201d refers to a juxtaposition wherein the components so described are in a relationship permitting them to function in their intended manner. For instance, a promoter is operably linked to a coding sequence if the promoter affects its transcription or expression.","The term \u201ctransformation\u201d refers to a permanent or transient genetic change induced in a cell following introduction of new nucleic acid (i.e., DNA exogenous to the cell). In a eukaryotic cell, a permanent genetic change is generally achieved by introduction of the DNA into the genome of the cell.","The N-degron is an intracellular degradation signal whose essential determinant is a specific (\u201cdestabilizing\u201d) N-terminal amino acid residue of a substrate protein. A set of N-degrons containing different destabilizing residues is manifested as the N-end rule, which relates the in vivo half-life of a protein to the identity of its N-terminal residue. The fundamental principles of the N-end rule, and the proteolytic pathway that implements it, are well established in the literature (see, e.g., Bachmair et al., Science 234: 179 (1986); Varshaysky. Cell 69: 725 (1992), U.S. Pat. Nos. 5,122,463; 5,132,213; 5,093,242 and 5,196,321) the disclosures of which are incorporated herein by reference in their entirety. A dormant N-degron is one that must be cleaved or degraded in order that the destabilizing residue becomes the N-terminal amino acid residue.","As used herein, TDeg'F is used synonomously with NDeg.","An estradiol-responsive transcription factor is one that has a hormone binding domain that binds to estradiol to activate the transcription factor.","The term \u201cdegrade\u201d and \u201cdegradation\u201d as used herein refer to proteolytic degradation as may be facilitated by a component of the N-end rule proteolytic pathway. Such a degradation is meant to describe the targeted degradation of a specific \u201ctarget gene polypeptide.\u201d","The description of induction and degradation systems herein make reference to a modified target gene driven by its native promoter. Genes included in expression vectors must be driven by a nucleotide sequence comprising a regulatory element, for example, a promoter. Several types of promoters are well known in the transformation arts, as are other regulatory elements that can be used alone or in combination with promoters. As used herein, \u201cpromoter\u201d includes reference to a region of DNA upstream from the start of transcription and involved in recognition and binding of RNA polymerase and other proteins to initiate transcription.","A nucleic acid is hybridizable to another nucleic acid, such as a cDNA, genomic DNA, or RNA, when a single stranded form of the nucleic acid can anneal to the other nucleic acid under the appropriate conditions of temperature and solution ionic strength. Hybridization and washing conditions are well known and exemplified in Sambrook, J., Fritsch, E. F. and Maniatis, T. Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor (1989), and Sambrook, J. and Russell, W., Molecular Cloning: A Laboratory Manual, Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor (2001).","The chimeric transcription factor used to induce gene expression has a hormone-binding domain from a steroid receptor. When the hormone-binding domain is bound by hormone, the transcription factor localizes to the cell nucleus. Thus the method includes a step of exposing the cell to an effective amount of a hormone that binds to the hormone binding domain of the transcription factor. The effective amount of hormone refers to an amount that results in observable transcription of the target genes above background levels.","The invention relates to the regulation of protein abundance in eukaryotic cells. It is exemplified herein with novel yeast cells having a GEV system developed using a protrophic, HAP1+S288c strain of yeast. (All strains described herein are shown in Table 1, below).","One aspect of the invention relates to targeted protein depletion in yeast. The proteins of interest have a modified degron sequence attached thereto. Generally, the modified degron sequence is at the initiator codon of the open reading frame (e.g., methionine in yeast). In an alternative embodiment, the modified degron sequence is at a site internal to and in frame with the open reading frame such that the target protein retains native or near-native activity. Upon adding hormone inducer to the medium, a protease is induced which cleaves the N-degron, leaving an N-end rule destabilizing amino acid exposed at the N-terminus so that the proteins are rapidly degraded by the process known as N-end rule degradation. In this aspect of the invention, a protease (e.g., TEV) is induced by a transcription factor (e.g., GEV) and the protease targets the proteins of interest so that the proteins are rapidly degraded by N-end rule degradation.","The inducible protein depletion method allows rapid, specific target protein degradation simply by making the system hormone-inducible using a modular transcriptional activator that has a hormone-binding domain as well as the DNA-binding domain and activator domain. As exemplified in the examples herein, the hormone was \u03b2-estradiol and the transcriptional activator was GEV. As exemplified herein, GEV induced the tobacco etch virus (TEV) protease which unmasks an N-degron to enable N-end rule degradation of a target gene driven by its native promoter and modified to contain a dormant N-degron. Thus the inducible protein depletion method has an advantage over the tet-off system, of resulting in a rapid elimination of the protein products of the regulated genes.","Another aspect of the invention relates to a chimeric transcription factor. One chimeric transcription factor for use in the method of the invention is the transcriptional activator GEV, which has three modular domains from different sources that function in hormone binding, DNA binding and activation. Transcriptional activators with alternative modular domains are also contemplated.","The contemplated transcriptional activators are useful to get effective target specificity. By replacing GEV's GAL4 DNA binding domain with a DNA binding domain from a non-yeast transcription factor (such as LexA) or from custom-designed zinc fingers that recognize DNA sequences not found in , the number of non-target genes induced or repressed will be lower than with GEV.","When constructing similar transcriptional activators with alternative components, the most specific activator will have a DNA binding domain that has high target DNA specificity and low enough affinity for binding DNA that the chimeric protein only binds to a single sequence of interest. For example, the DNA binding domain can be any other endogenous, rationally designed or engineered zinc finger domain (e.g., zinc fingers from endogenous mouse transcription factor zif268, 4-fingered rationally designed zinc finger 4Z, or any other engineered Cys2His2 zinc finger domain). Two embodiments of the engineered zinc finger domain are the ZEV and the 4ZEV transcription factors. In these embodiments, the DNA binding domain of GEV is replaced with an alternative DNA binding domain that recognizes a sequence that is either absent from or rare within the yeast genome. Zif268, the DNA-binding domain of ZEV, recognizes the sequence GCGTGGGCG (SEQ ID NO: 2) as well as many previously described similar sequences. 4ZEV was designed to recognize the 12-basepair sequence GCGGCGGAGGAG (SEQ ID NO: 3) which is not found in the yeast genome. Specificity determination predicts this zinc finger would function with all binding sites included in GCG(G\/C)(C\/G)(G\/T)G(A\/C)G(G\/A\/C)(A\/C)(G\/T) (SEQ ID NO: 4). In practice, the target sites for ZEV or 4ZEV are arranged as 3 dimer site targets (6-monomeric targets) with one or two bases between each monomeric target to allow for efficient assembly around the DNA double helix.","Since the transcription factors have a modular DNA-binding domain derived from other organisms or have a completely synthetic DNA-binding domain, the transcription factor and the inventive method are useful with respect to any eukaryotic cell.","With respect to other alternative modular domains of the transcription factor, the invention contemplates replacing the hormone binding domain so that a variety of hormones will work with the inventive system. In this case the hormone binding domain is replaced by the binding domain of any hormone receptor such that hormone binding results in nuclear localization of the transcription factor. For example the binding domains of the glucocorticoid or mineralocorticoid receptors may be used.","Replacement of the VP16 activating domain with other activating domains such as a protein sequence that provides an acidic region is also contemplated (e.g., the acidic regions of GAL4 or GCN4).","The ACT1 promoter used for driving GEV expression in the examples is a relatively strong promoter. While this is an advantage for some uses, a diversity of promoter strengths should be considered when constructing a system similar to what has been herein described but with alternative components. The diversity of promoter strengths is an important consideration, since very strong induction appears, from the data shown, to be at least partly responsible for the modest impairment of growth seen under full induction. This diversity of promoter strengths can be generated by isolating mutant promoters, or by using weaker natural promoters.","The contemplated invention includes a variety of promoters to express the GEV, ZEV or other hormone-activated transcriptional activator. Any promoter native to the organism in which the inventive system is placed (or that works by virtue of similarity in the original organism) is appropriate. The promoter should be constitutive, expressing the activator at an appropriate level such that, upon hormone treatment, the desired graded response of activation is achieved. In addition to ACT1 in yeast, other examples that exemplify this concept include the TDH3, ADH1, or TEF promoter or, in mammalian cells, the cytomegalovirus promoter (CMV).","The inventive protein depletion system contemplates use of an alternative protease. In this respect, what is contemplated is use of any protease with a specific recognition sequence that is not present in any of the native proteins in the organism used in the depletion system. For example, the inventive protein depletion system is also contemplated with use of proteases 2A or 3C, encoded by the genome of human rhinovirus (HRV 2A, HRV 3C).","Another manner of constructing the system of the invention is by adapting other nuclear receptors that work by the same mechanism as the one in our GEV system, so that independent induction of two target genes can be accomplished. This will be particularly useful if one wants to use the system to provide pulses of protein. With two independent inducing systems one can induce synthesis of a target protein by adding one inducer, and then destroy, at will, the protein by inducing TEV with the other inducer.","The transformed cells can provide rapid depletion of single or multiple cellular proteins. This system is readily adaptable to the study of essential genes (genes that are lethal if deleted). Through rapid depletion, the cell's transcriptional response to the loss of these genes can be assayed over a time course using gene expression microarrays. Depletion via the degron method mimics the effect of application of potential inhibitors of a target polypeptide. Studies are conducted in relevant physiological conditions in the presence of normal or near normal expression of the target (before application of inhibitors). This enables determination of the immediate and\/or long term effects of inhibition (depletion) to assess target and off-target physiological effects. This is especially useful if genetic deletion of the target leads to inviability. Likewise, transient induction can be assessed, starting from no expression, or normal expression (e.g., diploid cells with one normal and one hybrid transcription factor inducible (promoter-modified) target present).","Both induction and degradation are \u201ctunable\u201d, facilitating comparative two-strain screens. Induction level adjustment is convenient for \u201crescue\u201d screens, where overexpression would be deleterious to growth, or would affect another assay marker and compounds that inhibit the target return the cells to a normal state (i.e. growth or normal marker expression). When induction is undesirable but a two-strain comparative screen is indicated, partial depletion can be accomplished to enable comparison to cells with normal expression of the target.","The rescue screen is useful when the target protein is an enzyme or structural protein that, when overexpressed, is harmful to the cell (e.g., tubulin). The target of this screen is not necessarily native to the screening organism. For example, a mammalian target could be screened in yeast, as long as the harm (toxicity) of the screening cells is related to the manner the target functions in its natural host. In such a case, a target gene construct is prepared having a promoter responsive to the hormone-inducible activator and this is introduced into a chromosome of the cells of the screening organism. Growth studies are conducted to determine the level of hormone necessary to achieve a constant level of target protein that leads to a growth deficiency of the host organism. These host cells displaying the growth deficiency are then exposed to a compound library and a determination is made of which compound(s) afford a growth advantage to these host cells. The compounds that afford a growth advantage are candidates for enzymatic inhibitors or structural disruptors of the target protein.","Depletion of a protein involved in the production, processing or consumption of a valuable metabolite or protein can be accomplished in a near gratuitous fashion. Application of the depletion inducer at a strategic phase of batch fermentation can lead to increased yield of the desired metabolite or protein.","Thus this system has applications in synthetic biology, pharmaceutical and agricultural industries, and in basic biomedical research. One application is in bioengineering for biologic or compound synthesis. This system can be integrated as a module into engineered circuits. As used herein, a circuit is a programmed set of genes being expressed, or proteins that are degraded. Rapid, gratuitous degradation of a single protein is used in the construction of more complex, synthetic circuits that are orthogonal to the cell's wild type machinery. For example, the modified yeast cells are used as a vehicle for producing large amounts of a particular chemical, since achieving specific levels of particular proteins is essential for achieving maximal efficiency. The GEV\/TEV system allows a particular target protein to be tuned down without affecting the other cellular machinery.","The cells of the invention are also useful for high throughput screening (HTS) of compound libraries by exposing these compound libraries to the protein-depleted cells by methods well known to those skilled in the art. HTS can, for example, be used for discovery of new pharmaceutical or agricultural compounds. The cells used in this manner behave essentially as wild type capable of growing in normal, non-physiologically-changing medium. They allow for target identification.","For some drug targets, constitutive overexpression or elimination of a target protein is not possible because this would produce deleterious consequences for the cell. Thus, a screen of the type that uses a control strain and a modified strain (or cell line) for testing compounds would not be possible. The protein induction\/depletion system of this invention overcomes that problem. With this system it is possible to transiently overexpress or deplete a target. In doing so, one skilled in the art can 1) determine the appropriate duration and concentration of exposure to the hormone used for induction or depletion, so that the target protein concentration is sufficiently changed for the purpose of the assay, 2) expose cells to the library of compounds, 3) determine the appropriate time for cells to recover from the perturbation (e.g. after dilution with non-hormone containing medium) and 4) determine the growth difference, or other assay-specific change, conferred by a compound, that distinguishes the treated control cells from the hormone responsive cells.","The following examples set forth the general procedures involved in practicing the present invention. To the extent that specific materials are mentioned, it is merely for purposes of illustration and is not intended to limit the invention.","Materials And Methods","Strains:","Prototrophic HAP1+ yeast strains were constructed that can be readily modified to either (1) induce expression of a single gene or (2) rapidly degrade a single target protein (Table 1, below). These strains contain an integrated copy of GEV driven by an ACT1 promoter and are gal4\u0394 gal1\u0394.","For induction\/overexpression studies, GEV-containing strains of either mating type: DBY12020 (MATa) and DBY12021 (MAT\u03b1) were constructed. These strains can be transformed with a KanMX-Plinear fragment (Liko et al., 2006) in front of any non-essential gene to make that gene inducible by \u03b2-estradiol.","For protein degradation studies, DBY12132 and DBY12200 were constructed, which contain both GEV and the TEV protease. In DBY12132 and DBY12200, P-GEV is at the LEU2 and CAN1 locus, respectively. By modifying a target gene in these strains to contain a degron sequence recognized by TEV at its amino terminus, a gene can be expressed under its native promoter and degraded in the presence of \u03b2-estradiol.","Media and Growth Conditions:","For microscopy and flow cytometry experiments, cells were grown in low-fluorescence media (LFM). LFM is chemostat minimal medium (Bet al. 2008) lacking riboflavin and folic acid, and contains, in addition to glucose (20 g\/L), 5 g\/L (NH)SO, 1 g\/L KHPO, 0.5 g\/L MgSO, 0.1 g\/L NaCl, 0.1 g\/L CaCl, 0.5 mg\/L HBO, 0.04 mg\/l CuSO, 0.1 mg\/l KI, 0.2 mg\/l FeCl, 0.4 mg\/l MnSO, 0.4 mg\/l MnSO, 0.2 mg\/l NaMoO, 0.4 mg\/l ZnSO, 2 ng\/1 biotin, 0.4 mg\/l calcium pantothenate, 2 mg\/l inositol, 0.4 mg\/l niacin, 0.2 mg\/l para-aminobenzoic acid, 0.4 mg\/l pyridoxine HCl, 0.4 mg\/l thiamine.","For induction experiments, yeast were cultured in 500 mL chemostats (Sixfors; Bottmingen, Switzerland) under phosphate-limitation (20 mg\/L potassium phosphate) and pulsed with 1 \u03bcM \u03b2-estradiol (Tocris Biosciences; Missouri). For the MET4 induction experiment, phosphate-limited media was supplemented with 200 mg\/L methionine. Culture volume was maintained at 300 mL. 12-18 hours of batch phase growth prior to pump turn-on was initiated from a 1\/60 dilution of cells grown overnight in the same medium. Cultures were grown at 30\u00b0 C., stirred at 400 RPM, and aerated at 5 L\/min with filtered humidified air.","For all other experiments, cells were grown on rich medium (YPD; 2% peptone, 1% yeast extract).","Microscopy:","Fluorescence was measured with an epifluorescence Nikon Eclipse-TI inverted microscope using a Nikon 40\u00d7\/0.95NA DIC Plan Apo objective. The microscope is equipped with a PerfectFocus system (Nikon) for maintaining the correct focal plane, a Clara CCD Camera (Andor DR328G) for recording fluorescence emission, and a TI-S-ER motorized stage with encoders to ensure near perfect return to marked stage positions (Nikon MEC56100). GEV-GFP emission was visualized at 525 nm (50-nm bandwidth) upon excitation at 470 nm (40-nm bandwidth) (Chroma #49002_Nikon ETGFP filter cube). HTB2-mCherry emission was visualized at 620 nm (60-nm bandwidth) upon excitation at 545 nm (30-nm bandwidth) (Chroma #96364_Nikon ET-DSRed filter cube). Image acquisition was automatically controlled using NIS Elements software.","For Fluorescence in situ hybridization (FISH), images were acquired with an Olympus IX81 inverted fluorescence microscope as described in Silverman et al. (2010).","GEV-GFP Localization Experiment:","Overnight cultures of DBY11415 (MATa\/MAT\u03b1 HTB2-mCherry-caURA3\/HTB2 his3\/HIS3, (gal1+P):: loxP\/GAL1, gal4::LEU2\/GAL4, HAP1+\/hap1\u2212, leu2\u03940::P-GEV-GFP-KanMX\/LEU2) were grown in LFM overnight to saturation. Overnight cultures were diluted 1:50 in fresh LFM media to a Klett of \u02dc10. The Klett was monitored, and at Klett 40 cells were loaded into 96-well glass-bottomed optical plates for microscopy (Nunc #265300). Each well in the 96-well plate was incubated with 30 ul of concanavalin A solution (2 mg\/ml concanavalin A MP Biomedicals #150710 dissolved in 5 mM MnCl5 mM CaClpH 6.5) to fix cells to the bottom of the plate. After a 5 minute incubation, excess concanavalin A was aspirated off, and 4 \u03bcL of cell culture+91 \u03bcL of fresh LFM media+1 \u03bcL of 100% ethanol were added to each well. Cells were allowed to settle to the bottom of each well while stage positions were marked on the microscope using ambient room light.","Cells were imaged immediately prior to and immediately after addition of 4 \u03bcL \u03b2-estradiol to each well. Each experiment was done in triplicate and two images were taken per each well of the 96-well plate, resulting in 6 image frames per concentration of \u03b2-estradiol. Images were acquired every 170 seconds during induction of GEV-GFP localization.","GEV-GFP Washout Experiment:","Growth of strain DBY11415 was performed as described above for the GEV induction experiments. Cells were maintained in a flow cell (Bioptechs FCS2) during microscopy. Cells were adhered to the bottom of a 40 mm round coverslip by first incubating the coverslip with concanavalin A (as above) and then the coverslip was assembled into the FCS2 chamber. Cells were maintained at room temperature during the course of the experiment (25\u00b0 C.). The Klett of the culture was 110 when cells were loaded into the flow cell. Two peristaltic pumps (INSTECH P720) were used to perfuse media to cells during microscopy. One pump perfused LFM medium and the other perfused LFM media with 1 \u03bcM \u03b2-estradiol. The pump flow rates were automatically controlled by computer (Bioptechs Perfusion Controller) and by switching between pumps media was exchanged within the cell chamber in \u02dc2 minutes. GEV nuclear localization was first induced with LFM+1 uM \u03b2-estradiol for 50 minutes. The pump flow rates were then switched and images were acquired every 300 s to monitor delocalization of GEV-GFP. Conditions for imaging GFP and mCherry were identical to the induction experiment except that a 1.5\u00d7 optovar was inserted into the optical path to magnify images from 40\u00d7 to 60\u00d7.","Microscopy Image Processing:","Image processing was performed by preprocessing the images in ImageJ and then using a custom written Matlab script to extract the intensities in the nuclei relative to the cytoplasm. Briefly, the image processing was done as follows: Fluorescence images were background subtracted using the rolling ball background subtraction command in ImageJ using a ball radius of 50 pixels. The method has been previously described (S1983). Images were examined by eye in ImageJ and out of focus images were removed before further analysis.","These preprocessed images were then analyzed using a custom-written Matlab script. In brief, the code first identifies cells using various thresholding commands on the GFP images to create a \u201cmask\u201d of the image (an image where pixels inside of cells are identified as 1 and pixels outside of cells are identified as 0). The user is then given an opportunity to correct any errors in this mask by hand. The code then uses similar thresholding commands on the mCherry image to identify cell nuclei and create a mask for the nuclei. Using these two masks the script then extracts intensity information from the GFP image for each cell identified in the cell mask, using the nucleus mask to differentiate between the cytoplasm and the nucleus. A particle tracking routine (track.m) is then used to align values for cells and nuclei between images taken at different time points. Finally, the various parameters are exported as cell by time matrices where each row represents a single cell identified throughout the time course. The localization score in all figures is determined for each cell by first averaging the top 5% of pixel intensities and then dividing this number by the average pixel intensity over the entire cell (Cet al. 2008).","Flow Cytometry:","Cell fluorescence was measured with a BD LSRII Multi-Laser Analyzer with HTS. \u201cHigh-fluorescence\u201d in single cells was determined from the distribution of fluorescent intensities in cells lacking GFP (the distribution is determined from the fluorescent values of 100,000 cells). The cutoff for \u201chigh-fluorescence\u201d was set at the top 0.2% of the distribution (GFP signal=190 au).","Fluorescence In Situ Hybridization:","FISH probe design and experiments were performed as described by Silverman et al. (2010). DBY12040 was grown to early-log phase and pulsed with 1 \u03bcM \u03b2-estradiol. Cells were fixed prior to \u03b2-estradiol addition to the culture, and at 2 and 8 minutes () following \u03b2-estradiol addition. The Cy3-labeled DNA probes used for FISH are listed in Table 2, below.","Growth Curves:","100 \u03bcL of cells+media were grown on flat-bottom 96-well plates (Costar #3631). Awas measured every 15 minutes in a Synergy HT (Biotek; Vermont) microplate reader. Cells were grown at 30\u00b0 C. with medium shaking. Growth rates were computed with spline-fits using the grofit R package (Ket al. 2010).","Microarrays and Functional Annotation:","RNA from cells was extracted, labeled, and hybridized to Agilent expression microarrays as described by (Brauer et al., 2008) with slight modifications. Briefly, RNA was extracted from frozen cells with a standard acid-phenol method and then cleaned with RNeasy (Qiagen). Cleaned mRNA was then converted to cDNA and subsequently converted to labeled cRNA with the Agilent Quick-Amp Labeling Kit (Part No. 5190-0424). Microarray hybridizations were performed for 17 hours at 65\u00b0 C. and rotated on a rotisserie at 20 RPM. Microarrays were washed and then scanned with Agilent Feature Extractor Software version 9.5. Flagged features marked as unreliable were marked N\/A. Raw green- and red-channel intensities were floored to a value of 350 (i.e., intensities below 350 were set equal to 350). If the numerator (red-channel, sample RNA) for a given measurement was floored, the denominator (green-channel, reference RNA) for that measurement was set to the average green channel intensity for that gene. The purpose of this smoothing process is to make genes with negligible red-channel signal intensity less sensitive to small fluctuations in the reference intensity. After flooring of the data, the log 2 ratios were computed.","For time series experiments, 5 mL of cells were taken per culture per time point. Reference RNA was taken from DBY12001 grown in a phosphate-limited chemostat at a dilution rate of 0.18 h. Enrichment for gene ontology (GO) terms was determined using GO-term finder (Boyle et al., 2004).","Construction of the Degron System:","This consists of two parts, each of which ends up inserted into the chromosome. One of these is the protease cassette, which contains the GAL1 promoter, a modified TEV protease-coding region, the spliceosome subunit p14 peptide, a CYC1 terminator, and a selectable (hygromycin) marker, all integrated to replace the genomic GAL1 coding sequence. The details of this construction are described in detail below. The second part is a simple way to insert the degron\/marker cassette at any promoter of interest. To this end, a generic template was constructed for PCR that enables the placement of the modified degron\/marker cassette at the initiator methionine codon of any open reading frame; the details are also to be found below.","Modification of Target Genes to Make them TEV-Sensitive:","Since native proteins are not sensitive to the TEV protease, a generic system was devised to make the required modifications of any gene routine. The overall idea is to generate a native promoter-driven degron-modified gene. A particular new feature of this method is the inclusion of a removable (CRE-LOX) segment that results in loss of the selectable marker. The details of this construction, and its application to the specific cases of MET4 and MET31 are given below.","Construction of can1\u0394::P-GEV Strains:","The \u02dc2,240 bp P-GEV construct was amplified from the pAGL plasmid (Veatch et al., 2009) using oligonucleotides containing 56-59 bases of sequence identity to the flanking region of the CAN1 gene (ACT1-GEV_to_CAN1 FOR and ACT1-GEV_to_CAN1 REV). This PCR product was then transformed into either MATa or MATa wild type prototrophic yeast cells using a standard protocol.","Western Blots:","Total protein was extracted from \u02dc10 mL aliquots of cultures at \u02dc2\u00d710cells\/mL using YPX extraction kit (Protein Discovery, Knoxville, Tenn.) with the addition of protease inhibitors (Roche tablets, cat#11836170001). Proteins were separated using Invitrogen NuPage precast 10% Bis-Tris gels and transferred to Invitrolon PVDF (Invitrogen cat#LC2005) according to the manufacturer's protocols. Primary antibody for the \u03b2-Myc epitope was mouse anti-human c-Myc from BD Pharmingen (cat#51-1485GR). Secondary antibody was goat anti-mouse-HRP (cat#115-035-003) from Jackson ImmunoResearch. Primary antibody dilution was typically 1\/1000 and secondary antibodies were used at 1\/10,000 dilution. Immunoreactive peptides were detected by incubation with ECL detection reagent (Amersham\/GE RPN2132) and imaging on HyBlot CL autoradiography film (Denville Scientific, Metuchen, N.J.). As a loading control, blots were probed with mouse monoclonal anti-\u03b2-Actin antibody loading control cat#ab8224, Abcam, Cambridge, Mass. ( 1\/600 or 1\/2000). Secondary antibody was as above, goat anti-mouse-HRP. Protein levels were quantified with the \u201cGel Analysis\u201d feature in ImageJ and normalized to the t=0 levels in each time course.","GEV Strain Constructions:","Primers are listed in Table 3, below. DBY12021 was constructed through a series of crosses and sporulations. First, UCC1909 was crossed with DBY12013 (ura3\u03940 leu2\u03940 lys2-1280 HAP1+Mat\u03b1) and sporulated to generate DBY11191 (ura3\u03940 leu2\u03940::P-GEV-NatMX met15::LYS2 lys2-hap1-Mata). RSMx2-10a was then crossed with DBY12017 ((Pgal10+gal1)::loxP P-CRE-phleo HAP1+Mat\u03b1) and sporulated to generate DBY12018 ((P+gal1)::loxP leu2\u03940::P-GEV-NatMX HAP1+Mata) and DBY12019 ((P+gal1)::loxP leu2\u03940::P-GEV-NatMX HAP1+Mat\u03b1). A gal4::LEU2 linear DNA fragment was PCR amplified from UCC1864 and transformed into DBY12018 and DBY12019 to generate the protrophic strains DBY12020 ((P+gal1)::loxP leu2\u03940::P-GEV-NatMX gal4\u0394::LEU2 HAP1+Mata) and DBY12021 ((P+gal1)::loxP leu2\u03940::P-GEV-NatMX gal4\u0394::LEU2 HAP1+Mat\u03b1).","The C-terminus-tagged GEV-GFP reporter was generated by homologous recombination of GFP-KanMX from pFA6-GFP-KanMX (Sheff and Thorn, 2004) into DBY12021. The primers used for making the amplified GFP-KanMX are GEVGFP-F and GEVGFP-R.","The C-terminus-tagged HTB2-mCherry fusion was generated by homologous recombination. mCherry was first digested from pFA6a-mCherry-HIS3MX6 (pKT355, with PacI and AscI. pKT175 was cut with PacI and AscI to remove YECitrine. The digested mCherry was ligated into the pKT175 backbone. The primers used for making the mCherry-CaUra3 PCR fragment for transformation are HTB2-F and HTB2-R (Table 3, below).","The strains DBY12027, DBY12040, and DBY12086 were made by homologous recombination of KanMX-PDNA into DBY12021. KanMX-PDNA was PCR amplified from DBY11408 and with extra homology to insert directly upstream of the target ORF.","Insertion of TEV Protease at the GAL1 Locus:","We deleted the chromosomal GAL1 ORF, replacing it with the TEV protease. To do this, two overlapping PCR products were created. The first PCR product was generated using oligonucleotides homologous to 1) the GAL1 promoter and to 2) the CYC1 terminator, using pCT271 (which contains P-TEV-CYC1 (Taxis et al. 2009)) as a template. The second PCR product was generated using one oligonucleotide comprising 40 nt at the end of the CYC1 terminator sequence (for overlap with the first PCR product) followed by the TEF promoter sequence, paired with a second oligonucleotide comprising 40 nt downstream of the GAL1 ORF followed by TEF terminator sequences, using pAG32 as a template (Goldstein and McCusker, 1999). The two PCR products were transformed into strain DBY11389, selecting for hygromycin B (Calbiochem) resistance, to generate the gal1\u0394::TEV-HphMX strain. For some experiments, this TEV allele was modified to replace the HphMX cassette with the URA3 gene. To do this, two oligonucleotides were used to PCR URA3 from the pAG26 template (Goldstein and McCusker, 1999): 1) containing the CYC1 terminator sequence in the TEV construct and the URA3 promoter present in pAG26 and 2) containing the URA3 terminator and a sequence 3\u2032 of the gal1\u0394::TEV-HphMX insertion. This PCR product was transformed into strain DBY12096, resulting in strain DBY12132. All oligonucleotides are listed in Table 3, below.","Construction of a Generic Plasmid Template for NDeg-Gene Specific PCR Product Generation:","Plasmid pCT251 (Taxis et al., 2009) was used as a template for PCR to generate a fragment with 40 bp of homology to a region preceding the EcoRI site of YCplac33, an ATG, the coding sequence of the degron cassette starting with the first spacer sequence (encoding SISTL . . . (SEQ ID NO: 5)) and ending with the ATG at the terminus of the last spacer region ( . . . ENSSM (SEQ ID NO: 6)) followed by 40 bp of homology to the YCplac33 sequence region following the EcoRI site (\u201coverlap tipi to ycplac F\u201d; \u201coverlap tipi to ycplac R\u201d). YCplac33 was cleaved with EcoRI and was used, in conjunction with the PCR fragment above, to transform a ura3 auxotroph of yeast (DBY12015) to Ura prototrophy. The resulting plasmid (pRB3324) was reconstructed by homologous recombination in yeast, and recovered into by standard methods (Ma et al., 1987).","The spacer region after the SF3b coding sequence of the degron cassette derived from pCT251 contains two EcoRI sites. To replace this sequence with the loxP-flanked KanMX marker we used plasmid pUG6 (Guldener et al., 1996) as a template to generate two PCR products (necessary because of the identity of the loxP sites). The first product was generated by using an oligonucleotide (\u201cN-deg F+lox F\u201d) that contains 40 nt of homology preceding the first (most amino-terminal-encoding) EcoRI site of pRB3324 and 20 nt of homology to the 5\u2032 loxP site of pUG6, in conjunction with an oligonucleotide (KanMX r1) homologous to a sequence of KanMX near the 3\u2032 end. The second product was generated using an oligonucleotide (KanMX f1) homologous to a sequence near the 5\u2032 coding sequence of KanMX in conjunction with an oligonucleotide (\u201cN-deg R lox R\u201d) containing 40 nt of homology to the second (most carboxyl-encoding) EcoRI site of pRB3324 and 20 nt of homology to the 3\u2032 loxP site of pUG6. Plasmid pRB3324 was cut with EcoRI and used with the two oligonucleotides described above to transform yeast cells selecting for kanamycin (G418 sulfate; Cellgro) resistance. The resulting plasmid, pRB3326, recovered by transformation of the yeast DNA into , is a generic template for generating PCR products for insertion at the initiator methionine of any yeast gene.  shows the NDeg sequence that results from Cre-mediated loxP recombination of such a construct at any gene.","NDeg-MET4 Construction:","To generate chromosomal NDeg-MET4, the following oligonucleotides were used: 1) an oligonucleotide (MET4-Tipi-F) comprising 40 nucleotides of homology to the sequence upstream, and including, the ATG of MET4, and 20 nucleotides encoding the first spacer amino acids at the start of the degron construct of pRB3326; and 2) an oligonucleotide (MET4-Tipi-R) comprising 40 nucleotides of homology to the sequence downstream of the ATG (including an additional ATG) and 20 nucleotides of the loxP-modified spacer sequence at the end of the degron construct. This PCR product was used to transform a diploid (DBY12021\/DBY12032) to kanamycin resistance. After recovery and verification of the clone containing this construct, we transformed the strain with pSH65 (Gueldener et al., 2002) containing a GAL1-promoter driven Cre recombinase, selecting for resistance to phleomycin (Invivogen). The strain was grown overnight in galactose-containing medium and plated for single colonies on YPD medium. The resulting colonies were screened for kanamycin resistance and the sensitive clones were PCR-verified to have lost the loxP-KanMX cassette. The strain was sporulated and haploids selected for epitope tagging.","Wild type and the corresponding NDeg-MET4 strains were transformed with a PCR product to insert 13-Myc epitopes at the MET4 carboxyl terminus pFA6a-13Myc-KanMX6 (Longtine et al., 1998) was used as a template for oligonucleotides (MET4-Myc-F and MET4-Myc-R) containing sequences adjacent to the stop codon of MET4 and in-frame 13Myc sequence (5\u2032 primer) or TEF terminator sequences (3\u2032 primer). The resulting PCR fragment was transformed into cells selecting for kanamycin resistance. Standard genetic crosses were performed to generate DBY12055 and DBY11440.","NDeg-MET31 Construction:","MET31 was modified in a similar fashion to that of MET4. Oligonucleotides for MET3\u2032-specific NDeg amplification were Met31-TIPI-F and Met31-TIPI-R. The PCR product was used to transform a diploid to Kanamycin resistance. The KanMX sequence was eliminated as above. After sporulation both wild-type MET31 and NDeg-MET31 haploid strains were recovered. Carboxy-terminal 13-Myc tags were appended to the sequence of each variant as above. Diploids were generated to contain GEV, TEV and each of the MET31 variants (DBY12234, NDeg-MET31-13Myc; DBY12235, MET31-13Myc) for protein extraction.","\u03b2-Galactosidase Activity Assay:","Assays were performed essentially as described in (Liu et al., 1999) with minor modifications. 5 mL of cells transformed with pCM64-GAL1 were grown to midlog phase in SC-URA medium at 30\u00b0 C., then incubated at indicated temperatures for the indicated times with or without (a) 5 \u03bcL of ethanol (vehicle control), (b) 2 \u03bcL of 2.5 mM \u03b2-estradiol (Sigma) dissolved in ethanol (final concentration of 1 \u03bcM), (c) 5 \u03bcL of 1 mM DOC (deoxycorticosterone; Sigma) dissolved in ethanol (final concentration of 1 \u03bcM), or (d) 5 \u03bcL of 10 mM DOC dissolved in ethanol (final concentration of 10 \u03bcM). Cells (1 mL) were harvested by centrifugation, washed once with sterile HO, and resuspended in 700 \u03bcl of Z buffer (100 mM sodium phosphate, pH 7.0, 10 mM KCl, 1 mM MgSO, 50 mM \u03b2-mercaptoethanol). 50 \u03bcl of chloroform and 50 \u03bcl of 0.1% SDS were added, and the cell suspension was vortex-mixed at top speed for 30 seconds followed by addition of 200 \u03bcl of 4 mg\/ml ONPG (ortho-nitrophenyl-\u03b2-galactoside; Sigma) was added to each sample and incubated at room temperature until pale yellow color appeared. 350 \u03bcl of 1 M NaCOwas added to stop the reactions. After clarification by centrifugation, the supernatant was diluted, and Awas measured within the linear response range. Specific \u03b2-galactosidase activity was calculated as follows: (A\u00d71000)\/(A\u00d7(volume of cells examined in ml)\u00d7(time of incubation in minutes)) and reported in Miller units.","pCM64-GAL1 Plasmid:","pCM64-GAL1 (University of Texas Health Science Center at Houston; K. Morano) was constructed by inserting the promoter region of GAL1 in front of truncated lacZ into the plasmid pCM64, which was derived from pLG669Z (Guarente and Ptashne, 1981).","Engineered Cys2His2 Zinc Fingers:","The creation of the ZEV system was accomplished by replacing the gal4 DNA binding domain of the original GEV system with a set of Cys2His2 Zinc fingers. The ZEV system utilizes the zif268 zinc fingers. The 4ZEV system uses the four zinc finger protein rationally designed below.",{"@attributes":{"id":"p-0139","num":"0138"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"357pt","align":"center"}},"thead":{"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Sequences\u2003Used\u2003In\u2003Constructing\u2003ZEV\u2003And\u20034ZEV"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"357pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Zif268:"},{"entry":"GTRPYACPVESCDRRFSRSDELTRHIRIHTGQKPFQCRICMRNFSRSDHLTTHIRTH"},{"entry":"TGEKPFACDICGRKFARSDERKRHTKIHTG\u2003(SEQ\u2003ID\u2003NO:\u20037)"},{"entry":{}},{"entry":"4Z:"},{"entry":"GTRPYACPVESCDRRFSRHANLTRHIRIHTGQKPFQCRICMRNFSRNANLVRHIRT"},{"entry":"HTGSQKPFQCRICMRNFSRKADLRRHIRTHTGEKPFACDICGRKFARKGDLKRHT"},{"entry":"KIHTG\u2003(SEQ\u2003ID\u2003NO:\u20038)"},{"entry":{}},{"entry":"Optimal\u2003Zif\u2003individual\u2003target:\u2003GCGTGGGCG\u2003(SEQ\u2003ID\u2003NO:\u20032)"},{"entry":{}},{"entry":"Alternative\u2003Zif\u2003targets\u2003with\u2003reduced\u2003affinity:"},{"entry":"GCGTGGGG\u2003(SEQ\u2003ID\u2003NO:\u20039)-2-fold\u2003reduction\u2003in\u2003affinity\u2003in\u2003comparison\u2003to\u2003optimal"},{"entry":"GCGTGGGG\u2003(SEQ\u2003ID\u2003NO:\u200310)-5-fold\u2003reduction\u2003in\u2003affinity\u2003in\u2003comparison\u2003to\u2003optimal"},{"entry":"GCGTGGGC\u2003(SEQ\u2003ID\u2003NO:\u200311)-10-fold\u2003reduction\u2003in\u2003affinity\u2003in\u2003comparison\u2003to\u2003optimal"},{"entry":"GCGTGGGG\u2003(SEQ\u2003ID\u2003NO:\u200312)-20+ fold\u2003reduction\u2003in\u2003affinity\u2003in\u2003comparison\u2003to\u2003optimal"},{"entry":"(Affinity\u2003differences\u2003from\u2003Miller\u2003J\u2003and\u2003Pabo\u2003CO\u2003(2001)\u2003,\u2003313,\u2003309-315)."},{"entry":{}},{"entry":"Zif\u2003in\u2003vivo\u2003target:"},{"entry":{"u":["GCGTGGGCG","GCGTGGGCG","GCGTGGGCG","GCGTGGGCGgGCGTGGGCG","GCGT"]}},{"entry":{"u":"GGGCG"}},{"entry":{}},{"entry":"4Z\u2003designed\u2003optimal\u2003target:\u2003GCGGCGGAGGAG\u2003(SEQ\u2003ID\u2003NO:\u20033)"},{"entry":{}},{"entry":"4Z\u2003in\u2003vivo\u2003target:"},{"entry":{"u":["GCGGCGGAGGAG","GCGGCGGAGGAG","GCGGCGGAGGAG","GCGGCGGAGG"]}},{"entry":{"u":["AG","GCGGCGGAGGAG","GCGGCGGAGGAG"]}},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}}},"ZEV and 4ZEV Systems:","To test the optimal zinc finger \u2212EV fusion and binding site complement, three linkers between the zinc fingers and the estrogen receptor were tested (5, 10 and 15 amino acids). These constructs were tested for their activity with a mCherry reporter when binding sites were positions, spacing and orientations. Ultimately it was found that the 5 amino acid linker used in conjunction with three pairs of binding sites placed head to tail, with a single base separating each site, was the most active combination. All further binding sites and zinc finger constructs were built according to this template.","Construction of GFP Reporters:","GFP reporter plasmids were constructed to test the ability of the new ZEV and 4ZEV transcription factors to activate a target gene in the presence of the hormone \u03b2-estradiol. Reporter plasmids were constructed using the CEN plasmid pRS416 (Sikorski and Heiter 1989), which contains a URA3 selectable marker, as a template. A DNA fragment containing a modified GAL1 promoter fused to GFP (Gal1pr-GFP) was cloned into pRS416 between the NheI and AscI restriction sites. The GAL1 promoter was modified so that the 3 primary Gal4p (and also GEV) DNA binding sites were removed and replaced with DNA sequences containing 3 pairs of GEV, ZEV, or 4ZEV binding sites between the XbaI and NotI restriction sites ().","The GEV System:","Two isogenic strains of opposite mating type were constructed to use for studies of gene induction\/overexpression (DBY12020, DBY12021) studies in (see Materials and Methods). These strains contain an integrated copy of P-GEV, so that GEV is produced constitutively and at relatively high levels. A gene placed immediately downstream of a GAL-regulated promoter will only be expressed after addition of \u03b2-estradiol to the growth medium ().","Importantly, though GEV binds DNA via a Gal4p DNA binding domain, it is not subject to inhibition or repression by glucose, making feasible induction and overexpression experiments in standard glucose-containing media simply by the addition of the inducer \u03b2-estradiol.","A chromosomally integrated C-terminus-tagged GEV-GFP reporter was constructed, and its localization was measured following pulses of \u03b2-estradiol () in single cells, using a microfluidic device (Hersen et al.; 2008; McClean et al., 2011). GEV localization level was proportional to the amount of added \u03b2-estradiol for a range of concentrations. At the higher doses (100 nM to 10 \u03bcM), nuclear localization signal is seen emerging from the background about 6-8 minutes following hormone addition. The initial rate of localization achieved its maximum at concentrations at or above 1 \u03bcM \u03b2-estradiol.","To determine the residence time of activated GEV after removal of \u03b2-estradiol, cells were grown in a flow chamber. For the first 50 minutes, medium containing 1 \u03bcM \u03b2-estradiol was flowed over the cells. Then the medium inflow was switched (, black line) to a medium lacking \u03b2-estradiol, resulting in an approximately exponential reduction in nuclear fluorescence with time. This demonstrated that nuclear half-life of GEV is about 40 minutes (, C).","To investigate the kinetics of target gene induction by GEV, the GAL1 promoter (P) was inserted in front of various reporter genes. First, a strain was constructed where Pis driving transcription of the CBF1 gene at its native chromosomal locus. Upon GEV activation, the level of CBF1 transcript reaches \u02dc70% of its maximal value by 5 minutes following addition of \u03b2-estradiol (). Using Fluorescence in situ Hybridization (FISH), both the cytoplasmic transcripts and nuclear transcription sites of CBF1 were observable, as shown in the image from , which was taken 8 minutes following \u03b2-estradiol to the culture. A P-GFP reporter was used to find the point at which GEV becomes saturated by inducer. The percentage of induced cells is determined (see Materials and Methods) over time and the results indicate that GEV is saturated between 100 nM and 1 \u03bcM \u03b2-estradiol, as the induction kinetics are very similar between these two concentrations ().","The generally accepted model for the mechanism of nuclear receptors predicts that at sub-saturating concentrations of inducer, one should observe a steady-state response that is graded, as opposed to switch-like (Takahashi and Pryciak, 2008). Data from flow cytometry support this model for GEV induction of the galactose promoter-driven GFP construct. At concentrations below 1 nM \u03b2-estradiol, the average intensity is near the background after twelve hours of incubation (). At 10 nM the average intensity is increased but well below what is achieved at higher inducer concentrations (); the significant point is that all the cells exposed to 10 nM \u03b2-estradiol exhibit significant fluorescence, as shown by the shift of the entire distribution to the right (). At 100 nM, the average intensity induction is near maximum, and the distribution of cells resembles closely that found for higher, completely saturating concentrations of inducer.","The ACT1 promoter is known to be a relatively strong promoter, thus full induction in the experimental system should result in overexpression of most GEV targets. Despite the deletion of the GAL1 locus, the predicted strong GEV activation achieved at high concentrations of \u03b2-estradiol resulted in a slowing of growth (). This is not due to \u03b2-estradiol toxicity because the slow growth does not occur in a strain lacking the GEV system. This could well be due to \u201csquelching\u201d (GILL and PTASHNE 1988), whereby strong transcriptional activators repress off-target genes through titration of the RNA polymerase machinery, and thus reduce the cell's capacity for growth. At lower concentrations of \u03b2-estradiol (10 nM), the GEV-containing cells no longer show the growth defect (), but are able to induce the P-GFP reporter essentially in all the cells nonetheless ().","In order to determine whether physiologically significant expression is induced at intermediate (10 nM) levels of induction, where no growth consequence is observed, an experiment was undertaken in which a lethal defect might be complemented in a GEV-dependent manner. TPS2 encodes an enzyme that converts trehalose-6-phosphate to trehalose. Deletion of TPS2 causes a heat-sensitive growth phenotype, likely due to the build-up of trehalose-6-phosphate (Det al. 1993). By placing TPS2 downstream of P, cells become heat sensitive in the absence of \u03b2-estradiol (). Growth is restored to wild type levels in the presence of 10 nM \u03b2-estradiol, and the resulting growth is indistinguishable from wild type (). This result suggests that maximal induction is not likely to be necessary in order to achieve physiological levels, and therefore that intermediate levels of inducer suffice to avoid the growth-inhibiting side effects.","GEV Induction in Chemostats:","Since yeast are extremely sensitive to slight changes in the extracellular environment, the ideal growth setting for induction experiments is in the chemostat. Briefly, a chemostat works by flowing fresh media into a growth vessel, and culture medium is diluted from the vessel at a rate determined by the experimenter. This experimental setup allows for the growth of a culture of cells containing the GEV machinery and Pfused to any gene of interest to steady state. In the absence of \u03b2-estradiol, the cells are effectively deletion mutants of the gene of interest. Adding \u03b2-estradiol to such cultures, strongly induces the gene of interest so that the physiological consequences can be followed. For the particular case of genes encoding transcription factors, the consequence to be followed is the genome-wide transcriptional response of the cells. The goal is to infer the immediate targets of the factors and their relative strengths of response. For those that respond most quickly, the further inference of causation is strengthened.","GEV Induction in a Steady-State Culture is Nearly Gratuitous:","For this approach to be effective, GEV needs to be nearly gratuitous, meaning that it doesn't have a large effect on the transcriptional landscape of the cell. We can define a large effect as one that results in strong increases\/decreases in many genes such that it becomes difficult to separate out changes due to overexpressing the target gene and background. By growing a strain (DBY12021) that contains GEV in a chemostat with the appropriate GAL machinery removed, we find that GEV is nearly gratuitous (). In DBY12021, the GAL1 gene is deleted, as is the GAL10 promoter. Two hours following saturating (1 \u03bcM) \u03b2-estradiol addition, 105 genes decrease 2-fold (Table 3, below) and 94 genes increase 2-fold (Table 2) (). This accounts for about 3% of all genes. Using GO-term finder to find enriched processes within these groups, repressed genes are most enriched for glucose catabolic processes (corrected p-value=8.49\u00d710) and activated genes are most enriched for oxidation-reduction processes (corrected p-value=1.73\u00d710).","GEV induction results in strong activation of GAL2 and GAL7 and moderate induction of GAL3, GAL5, and GAL80 (). Surprisingly, despite the removal of the GAL10 promoter by loxP in DBY12021, we observe slight induction of GAL10 over the course of the experiment.","We found two genes (YPL067C and YPL066W) that are divergently transcribed in response to GEV induction (). These genes are reminiscent of GAL1 and GAL10, which are divergently transcribed in response to GAL4 activation. YPL066W and YPL067C are separated by a 385-nucleotide region. This region contains a single UASsequence. This sequence is conserved upstream of the YPL066W and YPL067C orthologs in and . Despite the conservation of the GAL4 binding site, these genes have no known role in galactose metabolism and are of unknown function.","The ten genes mostly strongly repressed at 2 hours following GEV activation are shown in . We could not find any significant correlation with annotated functions these genes might hold in common.","Large-scale ChIP-chip studies have defined extensive transcription factor-DNA binding maps, yielding a highly informative, though static, view of genetic regulation. Using the GEV system to induce rapid and high-level expression of a single transcription factor allows observation of the dynamics of activation or repression of each of its targets.","The MET4 gene encodes a well-studied, strong transcriptional activator of the sulfur metabolic genes in and served to exemplify that the inventive system serves as a powerful tool for observing genetic regulation. A strain containing P-MET4 was grown to steady state in a phosphate-limited chemostat containing high levels of extracellular methionine, a condition in which Met4p is supposed to be less active. By hierarchically clustering the transcriptional profile of the sulfur metabolic genes as a time series up to 90 minutes following MET4 induction, we were able to observe that the kinetics of MET4-target induction can vary from gene to gene ().","The targets fell into three categories based on their kinetic profiles: strongly induced (group 1), weakly induced (group 2), and uninduced (group 3). The MUP1 and MUP3 genes, which encode the high- and low-affinity methionine permeases, respectively, and SAM1 and SAM2, which encode enzymes that catalyze the synthesis of S-adenosylmethionine (AdoMet), are strongly induced group 1 genes. AdoMet is the primary source of methyl donor groups in the cell.","The MET31 and MET32 genes encode homologous zinc-finger proteins (46% identical), which aid in the recruitment of Met4p to many sulfur metabolic gene promoters. Unexpectedly, while MET32 is weakly induced (group 2), we do not detect any induction of MET31. While met31\u0394 and met32\u0394 mutant cells are able to produce their own methionine when grown on minimal medium, met31\u0394met32\u0394 double-mutants are methionine auxotrophs. Despite the purported redundancy of these factors, we were able to observe differential regulation in response to MET4 overexpression.","GEV-Mediated Induction of TEV Results in Rapid Degradation of N-Degron-Tagged Proteins:","MET4 and MET31 genes were chosen for use in a proof of principle experiment to exemplify the protein depletion method (). We constructed Ndeg-modified MET4 and MET31 alleles in separate strains containing GEV and P-TEV (see Materials and Methods). Cells were grown in rich medium (YPD) and treated with 1 \u03bcM\u03b2-estradiol at \u02dc2\u00d710cells\/mL. Western blots were performed on proteins extracted at several time points after induction (). Treatment of cultures with 1 \u03bcM \u03b2-estradiol had no effect on levels of Met4p-13Myc or Met31p-13Myc lacking NDeg (, B). However, 1 \u03bcM \u03b2-estradiol treatment of NDeg-MET4 or NDeg-MET31 cells showed rapid reduction of the NDeg-tagged proteins (, B). By performing a best fit of the data from 10, 20, and 40-minute timepoints to a power-law, we find that the half-life of NDeg-Met4p-13Myc is 16 minutes and that of NDeg-Met31p-13Myc is 13.75 minutes ().","A met4 null allele has a growth defect and produces small colonies on YPD plates (Hickman et al., 2011). Therefore a diploid cell was used for construction of NDeg-MET4. Sporulation of this diploid followed by dissection of the resulting tetrads showed that the spore colonies had the same wild type size, confirming that the modified gene was functional.","Met4p is known to be ubiquitinated (Rouillon et al., 2000) and is, as a result, unstable. Three clear bands were observed in our Met4p Western blots, consistent with the presence of non-ubiquitinated Met4p in addition to heavier ubiquitinated forms (). When MET4 is tagged with NDeg, both the ubiquitinated and non-modified Met4p proteins are targeted for degradation by TEV ().","Finally, the phenotypic effect of \u03b2-estradiol treatment of NDeg-MET4 containing cells was assayed. As expected only cells treated with \u03b2-estradiol displayed a methionine auxotrophy (met) phenotype (), as Met4p is required for methionine synthesis (Masselot and Robichon-Szulmajster, 1975).","The ZEV and 4ZEV transcriptional activators have the same hormone binding domain and activator domain as GEV, but each has a different DNA binding domain. ZEV has the DNA binding domain of transcription factor zif268 and 4ZEV has a rationally designed four zinc fingers domain.","The performance of these protein transcription factors are shown in .  shows the reporter plasmids used for characterizing the different expression systems. By placing the binding sites between restriction sites, new binding sites can be cut-and-pasted with relative ease. Thus, input\/output relationships can be easily tested for different DNA binding sequences. Importantly, each of the three systems show a graded response, ), and the use of non-yeast DNA binding domains allows for more production of GFP (FIG. A,B), presumably through a reduction in the errant transcriptional responses associated with the GEV system. Importantly, the ZEV and 4ZEV systems eliminate the expression changes associated with the GEV system (FIG. A,B). Nonetheless, each system strongly induces GFP (), and each system induces GFP transcripts >50-fold by 15 minutes following hormone addition (). Finally, by reducing the background expression of these new systems, the growth defect associated with GEV is eliminated (). This makes the new systems ideal for a variety of applications where 1) remaining inert with respect to the cell's overall physiology and 2) reducing background expression is of importance.","Discussion","This system demonstrates that the hormone-inducible transcriptional activator is an efficient switch that induces target genes to near maximal levels within minutes following hormone addition to the culture. A single-cell visualization method was used to establish that GEV activation occurs by its rapid nuclear localization in the presence of \u03b2-estradiol. The system demonstrated that GEV overexpression results in minimal errant induction of genes in a \u0394gal1 background, making it ideal for overexpression studies in ","With the endogenous galactose utilization machinery deleted, the results indicated that such a system (a) can be nearly gratuitous, as indicated by genome-wide profiling; (b) is fast acting, requiring only several minutes to activate transcription; and (c) exhibits low translational noise.","GEV induces or represses relatively few genes in a \u0394gal10\u0394gal1\u0394gal4 background. A 3-hour time course of DBY12021 (leu2\u03940::pACT1-GEV-NatMX, (Pgal10-1+Gal1)::loxP, Gal4::LEU2, HAP1+) following a pulse with \u03b2-estradiol showed that GEV-mediated activation by estradiol affects relatively few genes in the genome, less than 2% of the entire genome according to measurement by K-means clustering of time course data.","The examples show that the GEV system, configured in the manner of the invention, satisfies the requirements of an effective gratuitous induction system. It can be used to both rapidly induce a target gene in every cell in the culture while conferring minimal induction of off-target genes. GEV provides efficient induction. The range of transcription rates in measured in the literature ranges from 0.7-2 kb\/min (Mand S2005; Zet al. 2008), the upper bound of which has been measured from a P-driven gene (Mand S2005). If a gene is 2 kb in length and is downstream of P, as in the case of P-MET4, generating a full transcript should take 1 minute. By 2.5 minutes, detectable amounts of P-driven transcript were observed (). By 2.5 minutes, therefore, the gene has gone through several rounds of transcription. By 5 minutes, the amount of induction is near-maximal.","In the localization experiments herein, GEV-GFP is detected in the nucleus at \u02dc6-8 min. following addition of \u03b2-estradiol to the media. If nuclear localization takes \u02dc6-8 min., it would appear surprising that full-length transcripts are measured at 2.5 minutes following \u03b2-estradiol addition to the media. There are at least two potential explanations. First, it can be accounted for if the GEV-GFP fusion has slightly slower kinetics of nuclear localization than GEV. Second, it can be accounted for if the kinetics of GEV-GFP reflects the kinetics of GEV without the fluorescent tag, but measurement of localization is too insensitive to detect the first few GEV-GFP molecules that become localized to the nucleus and activate transcription.","Opportunities for Dynamic Studies of Regulation:","One of the most significant advantages of the system is the potential for following the consequences of the induction of a specific protein dynamically. This is clearest in the case of a transcriptional regulator like Met4p. The induction kinetics allow us to distinguish primary from secondary targets, and the cascade of consequences can be followed as time series that can shed light on the sequence of events under a variety of physiological conditions.","Through overexpression of MET4, the kinetics of induction of the sulfur metabolic genes was distinguishable (). The kinetics of target induction can depend strongly on the choice of nutrient limitation (i.e., phosphate versus methionine limitation). ChIP-chip studies can show the presence or absence of transcription factor-DNA interactions. They fail, however, to capture the strength and kinetics of induction of a particular gene in response to a single factor, something that is readily measurable via GEV-mediated overexpression of particular transcription factors.","By finding motifs in the promoters of genes that rapidly change expression in response to overexpression of a particular transcription factor, we can potentially generate position weight matrices (PWMs) for factors with unknown DNA binding motifs. Another potential application is to refine known PWMs of better-studied factors by complementing previous in vitro and context-specific ChIP-chip data with in vivo measurements that look for the presence of direct regulation through GEV-mediated overexpression.","Upon removing \u03b2-estradiol from the media, GEV-GFP begins to delocalize within minutes (). In the presence of \u03b2-estradiol, cytoplasmic GEV binds \u03b2-estradiol and enters the nucleus at a rate k. Nucleus-localized GEV is either bound to or unbound from DNA. Unbound GEV diffuses throughout the nucleus and exits the nucleus at a rate k. Since kapproaches 0 in the absence of \u03b2-estradiol, we can immediately measure unbound GEV leaving the nucleus at rate k. However, the true rate of nuclear delocalization of GEV includes the unbinding of GEV from DNA, which can be much slower.","Given that the half-life of GEV nuclear localization is \u02dc40 minutes and that protein half-lives can be as large as days, removing \u03b2-estradiol from the media is not an effective way to quickly deplete the levels of a target protein. Similarly, methods that rely on blocking transcription upon addition of inducer to cells are not effective for rapidly depleting target proteins. These facts were the motivation to prepare a construct to drive the expression of TEV by GEV, which allows for rapidly turning off target proteins containing an N-degron sequence. This sequence can readily be added to genes via homologous recombination. Therefore, we now have developed a method for rapidly shutting off the levels of single genes expressed from their native promoters. Upon \u03b2-estradiol addition and subsequent depletion of the target gene by TEV, one can study the cell's transient physiological response (such as the transcriptional profile) to losing both essential and non-essential genes. These types of studies assist to rigorously define the transcriptional regulatory network of essential genes in ","Various modifications and variations of the invention in addition to those shown and described herein will be apparent to those skilled in the art without departing from the scope and spirit of the invention and fall within the scope of the appended claims. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. All publications and patents mentioned in the above specification are incorporated in their entirety by reference.",{"@attributes":{"id":"p-0184","num":"0183"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"266pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Strain list."},{"entry":"All strains are s288c. HAP1+ indicates a repaired HAP1 allele"},{"entry":"(Hickman and Winston, 2007) and hap1\u2212 indicates the partial"},{"entry":"loss-of-function s288c allele (Gaisne et al., 1999; Hickman and Winston, 2007)."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"182pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Strain","Genotype","Source"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["DBY11389","Mata, ura3\u03940","This Study"]},{"entry":["DBY11408","MAT\u03b1, his3\u03941 leu2\u03940 lys2\u03940 ura3\u03940 gal10::KanMX","Deletion"]},{"entry":[{},{},"Collection"]},{"entry":["DBY11415","MATa\/MAT\u03b1, (P+ gal1)::loxP\/GAL1, leu2\u03940::P-","This Study"]},{"entry":[{},"GEV-GFP-KanMX\/LEU2, gal4\u0394::LEU2\/GAL4, HTB2-"]},{"entry":[{},"mCherry-CaUra3, his-\/HIS, leu-\/LEU, hap1\u2212\/HAP1+"]},{"entry":["DBY12020","MATa, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, HAP1+"]},{"entry":["DBY12021","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, HAP1+"]},{"entry":["DBY12027","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, KanMX-Pgal1-MET4, HAP1+"]},{"entry":["DBY12039","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, ybr032w\u0394::P-GFP-NatMX, HAP1+"]},{"entry":["DBY12040","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, KanMX-P-CBF1, HAP1+"]},{"entry":["DBY12055","Mata, NDeg-MET4-13Myc-KanMX, gal4::LEU2,","This Study"]},{"entry":[{},"gal1::TEV-HphMX, leu2\u03940::P-GEV-NatMX, HAP1+"]},{"entry":["DBY12086","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, KanMX-P-TPS2, HAP1+"]},{"entry":["DBY12099","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, KanMX-Pgal1-MET4, met6\u0394::HphMX4"]},{"entry":[{},"HAP1+"]},{"entry":["DBY12100","MAT\u03b1, (P+ gal1)::loxP, leu2\u03940::P-GEV-NatMX,","This Study"]},{"entry":[{},"gal4\u0394::LEU2, ura3\u0394::HphMX4, HAP1+"]},{"entry":["DBY12132","MAT\u03b1, gal1::TEV::URA3, ura3\u03940, leu2\u03940::P-GEV-","This Study"]},{"entry":[{},"NatMX, gal4::LEU2, HAP1+"]},{"entry":["DBY12168","MATa\/MAT\u03b1, \u0394gal4::LEU2\/GAL4, (P+","This Study"]},{"entry":[{},"gal1)::loxP\/GAL1-mCherry-NatMX,"]},{"entry":[{},"leu2\u03940::P-GEV-GFP-KanMX\/LEU2, HAP1+\/HAP1+"]},{"entry":["DBY11440","MATa\/MAT\u03b1, MET4-13Myc-KanMX\/MET4,","This Study"]},{"entry":[{},"ura3\u03940\/URA3, gal1::TEV::URA3\/GAL1,"]},{"entry":[{},"gal4::LEU2\/GAL4, leu2\u03940::P-GEV-KanMX\/LEU2,"]},{"entry":[{},"hap1\u2212\/HAP1+"]},{"entry":["DBY12032","MATa, leu2\u03940","This Study"]},{"entry":["DBY12194","MAT\u03b1, NDeg-MET31, gal1::TEV-URA3,","This Study"]},{"entry":[{},"fcy1::CreEDB78"]},{"entry":["DBY12197","MATa, MET31-13Myc-KanMX, can1::P-GEV, lys2,","This Study"]},{"entry":[{},"gal1::TEV-URA3"]},{"entry":["DBY12199","MATa, NDeg-MET31-13Myc-KanMX, gal1::TEV-","This Study"]},{"entry":[{},"URA3, fcy1::CreEDB78"]},{"entry":["DBY12200","MAT\u03b1, gal1::TEV-URA3, can1::P-GEV, gal4::LEU2","This Study"]},{"entry":["DBY12234","DBY12199 \u00d7 DBY12200","This Study"]},{"entry":["DBY12235","DBY12197 \u00d7 DBY12194","This Study"]},{"entry":["DBY11378","MATa, his3delta1, ura3delta0, ade2delta::hisG,","Dan"]},{"entry":[{},"lys2delta0, met15delta::LYS2, distal ChrIVR-MET15,","Gottschling"]},{"entry":[{},"RAD52-eGFP-KANMX, leu2delta0::pACT1-GEV-NAT","UCC1909"]},{"entry":["DBY11376","MATa, his3delta1, leu2delta0, ura3delta0,","Dan"]},{"entry":[{},"ade2delta::hisG, lys2delta0, met15delta::ADE2, distal-","Gottschling"]},{"entry":[{},"chrIV::MET15, gal4:LEU2, pRPT1-GEV-Nat","UCC1864"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0185","num":"0184"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"308pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20032\u2003"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"CBF1\u2003FISH\u2003probes"},{"entry":"The\u2003thymidines\u2003surrounded\u2003by\u2003asterisks\u2003are\u2003amino-allyl-Cy3\u2003dye\u2003labeled."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"210pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SEQ\u2003ID"]},{"entry":["Probe\u2003ID","Probe\u2003Sequence","NO:"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["cbf1-15","*T*CT\u2003CTT\u2003TGC\u2003TTT\u2003T*T*C\u2003CTG\u2003GCC\u2003TCG\u2003CTG\u2003*T*AG\u2003TTC","15"]},{"entry":[{},"TGC\u2003TCC\u2003*T*CG\u2003TTA\u2003TAG\u2003CCT\u2003C*T*T",{}]},{"entry":{}},{"entry":["cbf1-31","*T*TC\u2003GCG\u2003TTC\u2003GTT\u2003*T*GA\u2003TTA\u2003GAT\u2003GTG\u2003*T*GG\u2003GCA","16"]},{"entry":[{},"TCA\u2003GAA\u2003C*T*A\u2003TCT\u2003TCC\u2003TGG\u2003CCC\u2003*T*GG",{}]},{"entry":{}},{"entry":["cbf1-48","*T*TC\u2003TGG\u2003CCC\u2003ACG\u2003A*T*C\u2003GTG\u2003TTC\u2003TGC\u2003CAA\u2003*T*GG","17"]},{"entry":[{},"TAT\u2003GAG\u2003CAT\u2003G*T*C\u2003GTT\u2003TTT\u2003AGA\u2003T*T*G",{}]},{"entry":{}},{"entry":["cbf1-57","*T*TA\u2003CTT\u2003GAT\u2003TCT\u2003C*T*C\u2003ACG\u2003GGC\u2003AGG\u2003AGG\u2003*T*CG","18"]},{"entry":[{},"CTT\u2003AAA\u2003ACG\u2003T*T*G\u2003ATT\u2003GCA\u2003GTG\u2003T*T*G",{}]},{"entry":{}},{"entry":["cbf1-68","*T*CC\u2003CAG\u2003TTC\u2003TTC\u2003C*T*G\u2003CAG\u2003TTT\u2003CTC\u2003A*T*T\u2003TGC","19"]},{"entry":[{},"ACT\u2003GGC\u2003TAA\u2003*T*TG\u2003CGA\u2003TGC\u2003GTT\u2003T*T*G"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0186","num":"0185"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"280pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20033\u2003"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Primers\u2003used\u2003in\u2003this\u2003study."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"168pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SEQ\u2003ID"]},{"entry":["Primer\u2003ID","Primer\u2003Sequence","NO:"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["GEVGFP-F",{"i":"GATGTTTACCGATGCCCTTGGAATTGACGAGTACGGTG"},"20"]},{"entry":[{},{"i":"GGggtgacggtgctggttta"},{}]},{"entry":{}},{"entry":["GEVGFP-R","TTGACCTCTAATCATGCGGCCGCTCTAGAACTAGTG","21"]},{"entry":[{},"GATCtcgatgaattcgagctcg",{}]},{"entry":{}},{"entry":["HTB2-F","tactagggctgttaccaaatactcctcctctactcaagcc","22"]},{"entry":[{},"GGTGACGGTGCTGGTTTA",{}]},{"entry":{}},{"entry":["HTB2-R","aaaagaaaacatgactaaatcacaatacctagtgagtgac","23"]},{"entry":[{},"TCGATGAATTCGAGCTCG",{}]},{"entry":{}},{"entry":["MET4-Pgal1-F","TGCTGTATGAGTCGCCTTCGTGGGACTGCTCCTGCT","24"]},{"entry":[{},"TCATTATAGTTTTTTCTCCTTGACGTTAAAG",{}]},{"entry":{}},{"entry":["MET4-Pgal1-R","TCTCGTCAATAAAGCGCACTTCTGATAAGCACTTTT","25"]},{"entry":[{},"ATTCGCAATTAAGAACTAAAAGATATAGAGTGC",{}]},{"entry":{}},{"entry":["CBF1-Pgal1-F","CCTCAGTAGAAAGCTTATTATTATTTGCCAGAGAGT","26"]},{"entry":[{},"TCATTATAGTTTTTTCTCCTTGACGTTAAAG",{}]},{"entry":{}},{"entry":["CBF1-Pgal1-R","AGTGCTTAAAATATAATACGGTTTTCTACACTTTTAT","27"]},{"entry":[{},"TAAGCAATTAAGAACTAAAAGATATAGAGTGC",{}]},{"entry":{}},{"entry":["TPS2-Pgal1-F","AAACCAAAATAACACTGCCTGTCACTATTTCTGTGC","28"]},{"entry":[{},"CGAAGCAATTAAGAACTAAAAGAT",{}]},{"entry":{}},{"entry":["TPS2-Pgal1-R","TCTCTTCTTTGGAGAATTGTCTTGGGCAGTGGTGGTC","29"]},{"entry":[{},"ATTATAGTTTTTTCTCCTTGACG",{}]},{"entry":{}},{"entry":["MET4-Tipi-F","AAAGCGCACTTCTGATAAGCACTTTTATTCCTTTTTT","30"]},{"entry":[{},"TCCACTGTGAACGATGTCTATTACTTCTTTGTACAAG",{}]},{"entry":{}},{"entry":["MET4-Tipi-R","CCGTGCTGTATGAGTCGCCTTCGTGGGACTGCTCCT","31"]},{"entry":[{},"GCTTCATTAATAACTTCGTATAGCATAC",{}]},{"entry":{}},{"entry":["MET4-Myc-F","AAAAAGCTGAAAGCTTAAAGAAGCAAATTTTTGAG","32"]},{"entry":[{},"AAGGTTCAGAAAGAACGGATCCCCGGGTTAATTAA",{}]},{"entry":{}},{"entry":["MET4-Myc-R","ATGCACGTATATATATATATATATATAATTAAACTG","33"]},{"entry":[{},"TATAGTCTGTTATTGAATTCGAGCTCGTTTAAAC",{}]},{"entry":{}},{"entry":["MET31-Tipi-F","GTTGGGCTCAATATACACAGTCGATAGTCTATATGT","34"]},{"entry":[{},"GCATATGTCTATTACTTCTTTGTACAAG",{}]},{"entry":{}},{"entry":["MET31-Tipi-R","GAGAAATATTTCATCTACATTCATGTCTTGCGCCAG","35"]},{"entry":[{},"TTTCATTAATAACTTCGTATAGCATAC",{}]},{"entry":{}},{"entry":["overlap\u2003tipi\u2003to","TAACAATTTCACACAGGAAACAGCTATGACCATGAT","36"]},{"entry":["ycplac\u2003F","TACGTCTATTACTTCTTTGTAC",{}]},{"entry":{}},{"entry":["overlap\u2003tipi\u2003to","CAGGGTTTTCCCAGTCACGACGTTGTAAAACGACGG","37"]},{"entry":["ycplac\u2003R","CCAGCATCGATGAATTCTCTGTCGTCC",{}]},{"entry":{}},{"entry":["N-deg\u2003F\u2003+ lox\u2003F","GCTGGTTATGTTCCTATTCGAACTCCAGCTCATATG","38"]},{"entry":[{},"GATATAACTTCGTATAATGTATGC",{}]},{"entry":{}},{"entry":["N-deg\u2003R\u2003lox\u2003R","CGGCCGCATAGGCCACTAGTGGATCTGATATCATCG","39"]},{"entry":[{},"ATAATAACTTCGTATAGCATAC",{}]},{"entry":{}},{"entry":["ATG\u2003plus\u2003tipi","ATGTCTATTACTTCTTTGTACAAG","40"]},{"entry":["start\u2003F\u2003fuse",{},{}]},{"entry":{}},{"entry":["lox\u2003rev\u2003fuse","CATTAATAACTTCGTATAGCATAC","41"]},{"entry":{}},{"entry":["KanMX\u2003f1","GACTCACGTTTCGAGGCCGCG","42"]},{"entry":{}},{"entry":["KanMX\u2003r1","TGGTATCGGTCTGCGATTCCGA","43"]},{"entry":{}},{"entry":["cyc\u2003term\u2003with","CGCTCGAAGGCTTTAATTTGCGGCCGGTACGAAGCT","44"]},{"entry":["hph\u2003overlap","TCAGAGCTTGCCTTGTCCCCGCCGG",{}]},{"entry":{}},{"entry":["gal1p\u2003from","AGTACGGATTAGAAGCCGCCGAGCGGG","45"]},{"entry":["pct271",{},{}]},{"entry":{}},{"entry":["cyc\u2003term\u2003rev\u2003F","CTGAAGCTTCGTACCGGCCGC","46"]},{"entry":{}},{"entry":["o-GIL\u2003104-extR1","TGAGAAGTTGTTCTGAACAAAGTAAAAAAAAGAAG","47"]},{"entry":[{},"TATACTAATACGACTCACTATAGGG",{}]},{"entry":{}},{"entry":["cyc\u2003term\u2003F-ura3","TGGGACGCTCGAAGGCTTTAATTTGCGGCCGGTACG","48"]},{"entry":["prom\u2003F","AAGCTTCGATTCGGTAATCTCCGAACAG",{}]},{"entry":{}},{"entry":["gal1\u2003end\u2003R-ura3","TGAGAAGTTGTTCTGAACAAAGTAAAAAAAAGAAG","49"]},{"entry":["term\u2003R","TATACTTAAGGGTAATAACTGATATAATTAAATTG",{}]},{"entry":{}},{"entry":["ACT1-","GCACAAATTAGCAGAAAGAAGAGTGGTTGCGAACA","50"]},{"entry":["GEV_to_CAN1","GAGTAAACCGAATCAGGGAATCCCGGCGCGCCGCC",{}]},{"entry":["FOR","TCTACCTTGCAGACC",{}]},{"entry":{}},{"entry":["ACT1-","CGGTGTATGACTTATGAGGGTGAGAATGCGAAATG","51"]},{"entry":["GEV_to_CAN1","GCGTGGAAATGTGATCAAAGGGGATCCTACCCACC",{}]},{"entry":["REV","GTACTCGTCAATTCC"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0187","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["B, E. I., S. A. W, J. G, H. J, D. Bet al., 2004 GO::TermFinder-open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes. Bioinformatics 20: 3710-3715.","B, M. J., C. H, E. M. A, R. R, J. C. Met al., 2008 Coordination of growth rate, cell cycle, stress response, and metabolic activity in yeast. Molecular Biology of the Cell 19: 352-367.","C, L., C. K. Dand M. B. E2008 Frequency-modulated nuclear localization bursts coordinate gene regulation. Nature 455: 485-U416.","D, C., N. B, W. B, P. J, T. Bet al., 1993 Disruption of Tps2, the Gene Encoding the 100-Kda Subunit of the Trehalose-6-Phosphate Synthase Phosphatase Complex in -, Causes Accumulation of Trehalose-6-Phosphate and Loss of Trehalose-6-Phosphate Phosphatase-Activity. European Journal of Biochemistry 212: 315-323.","G, C. Y., and J. L. P2000 Tightly regulated, beta-estradiol dose-dependent expression system for yeast. Biotechniques 29: 1226-1231.","G, M., B, A. M., V, J., H, C. J. (1999). A \u2018natural\u2019 mutation in strains derived from S288c affects the complex regulatory gene HAP1 (CYP1). Curr Genet. 36, 195-200.","G, G., and M. P1988 Negative Effect of the Transcriptional Activator Ga14. Nature 334: 721-724.","G, E., S. M. Vand M. P1985 Specific DNA-Binding of Gal4, a Positive Regulatory Protein of Yeast. Cell 40: 767-774.","G, A. L., and J. H. M1999 Three new dominant drug resistance cassettes for gene disruption in . Yeast 15: 1541-1553.","G, L., P, M. (1981). Fusion of --Lacz to the Cytochrome-C Gene of -. P Natl Acad Sci-Biol 78, 2199-2203.","G, U., J. H, G. J. K, D. Voss and J. H. H2002 A second set of loxP marker cassettes for Cre-mediated multiple gene knockouts in budding yeast. Nucleic Acids Res 30: e23.","G, U., S. H, T. F, J. Band J. H. H1996 A new efficient gene disruption cassette for repeated use in budding yeast. Nucleic Acids Res 24: 2519-2524.","H, P., M, M. N., M, L., R, S. (2008). Signal Processing by the HOG MAP kinase pathway. P Natl Acad Sci USA 105, 7165-7170.","H, M. J. W, F (2007). Heme Levels Switch the Function of Hap1 of between Transcriptional Activator and Transcriptional Repressor. Molecular and Cellular Biology, 27, 7414-7424.","H, M. J., P, A. A., H-S, O., S, S. J., MI, R. S., L, T. A., B, D. (2011). Complex co-regulation of sulfur and phospholipid metabolism reflects the importance of methylation in the growth of yeast. In preparation.","H, M. Q., M. X. F, S. H, C. L, D. W. Set al., 2008 Structural basis for dimerization in DNA recognition by Ga14. Structure 16: 1019-1026.","H, P., J. F, M. Jand R. A. S1989 Galactose as a Gratuitous Induced of Gal Gene-Expression in Yeasts Growing on Glucose. Gene 83: 57-64.","K, M., G. H, H. L-F, J. Land M. K2010 grofit: Fitting Biological Growth Curves with R. Journal of Statistical Software 33: 1-21.","L, M. A., S. B. B, T. Sand A. J. L1990 Conversion of the Lac Repressor into an Allosterically Regulated Transcriptional Activator for Mammalian-Cells. Molecular and Cellular Biology 10: 3343-3356.","L, D., S, M. G., P, C. L., H, W. (2006). Using the yeast gene deletion collection to customize gene expression. Biotechniques 40, 728-+.","L, X. D., M, K. A., T, D. J. (1999). The yeast Hsp110 family member, SSE1, is an Hsp90 cochaperone. J Biol Chem 274, 26654-26660.","L, M. S., A. MK3, D. J. D, N. G. S, A. Wet al., 1998 Additional modules for versatile and economical PCR-based gene deletion and modification in . Yeast 14: 953-961.","L, J. F., B. Hand D. P1993 Fusion of Ga14-Vp16 to a Steroid-Binding Domain Provides a Tool for Gratuitous Induction of Galactose-Responsive Genes in Yeast. Gene 131: 129-134.","L, C., M. J. Band D. B2009 Slow Growth Induces Heat-Shock Resistance in Normal and Respiratory-deficient Yeast. Molecular Biology of the Cell 20: 891-903.","M, H., S. K, P. J. Sand D. B1987 Plasmid construction by homologous recombination in yeast. Gene 58: 201-216.","M, R., M. C, M. Pand S. C. H1992 DNA Recognition by Gal4\u2014Structure of a Protein DNA Complex. Nature 356: 408-414.","M, P. B., and K. S2005 Distinction and relationship between elongation rate and processivity of RNA polymerase II in vivo. Mol Cell 17: 831-840.","M, M., R-S, H. (1975). Methionine Biosynthesis in . Molecular and General Genetics 139, 121-132.","MC, M. N., H, P., R, S. (2011). Measuring In Vivo Signaling Kinetics in a Mitogen-Activated Kinase Pathway Using Dynamic Input Stimulation. Methods Mol Biol 734.","P, W. B., and D. O. T1997 Steroid receptor interactions with heat shock protein and immunophilin chaperones. Endocrine Reviews 18: 306-360.","Q, M. J., D. M, M. A-R, A. Cand S. C2007 An improved system for estradiol-dependent regulation of gene expression in yeast. Microbial Cell Factories 6: \u2014.","R, M., and D. B2006 Transcriptional response of steady-state yeast cultures to transient perturbations in carbon source. Proceedings of the National Academy of Sciences of the United States of America 103: 389-394.","R, A., B, R., P, E. E., T, M., T, D. (2000). Feedback-regulated degradation of the transcriptional activator Met4 is triggered by the SCFMet30 complex. Embo J 19, 282-294.","S, I., J. M, S. Tand M. P1988 Ga14-Vp16 Is an Unusually Potent Transcriptional Activator. Nature 335: 563-564.","S, M. A., T, K. S. (2004). Optimized cassettes for fluorescent protein tagging in . Yeast 21, 661-670.","SH, (1989). A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in . Genetics 122, 19-27.","S, S. J., P, A. A., S, N., P, L., B, R., T, S. Y., Z, D., G, S. J., L, D. R., S, R. H., B, D. (2010). Metabolic cycling in single yeast cells from unsynchronized steady-state populations limited on glucose or phosphate. P Natl Acad Sci USA 107, 6946-6951.","S, S. R., 1983 Biomedical Image-Processing. Computer 16: 22-34.","T, S., P, P. M. (2008). Membrane localization of scaffold proteins promotes graded signaling in the yeast MAP kinase cascade. Curr Biol 18, 1184-1191.","T, C., G. S, R. Sand M. K2009 Efficient protein depletion by genetically controlled deprotection of a dormant N-degron. Mol Syst Biol 5: 267.","V, J. R., M. A. MM, Z. W. Nand D. E. G2009 Mitochondrial Dysfunction Leads to Nuclear Genome Instability via an Iron-Sulfur Cluster Defect. Cell 137: 1247-1258.","Z, D., D. R. Land R. H. S2008 Single-RNA counting reveals alternative modes of gene expression in yeast. Nature Structural & Molecular Biology 15: 1263-1271."]}}],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 2","smallcaps":"AI "},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 3","sub":["GAL1","GAL1","GAL1"]},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 5","sub":"GAL1"},{"@attributes":{"id":"p-0033","num":"0032"},"figref":"FIG. 6"},{"@attributes":{"id":"p-0034","num":"0033"},"figref":"FIG. 7"},{"@attributes":{"id":"p-0035","num":"0034"},"figref":"FIG. 8","sub":"GAL1"},{"@attributes":{"id":"p-0036","num":"0035"},"figref":"FIG. 9","b":["9","9"]},{"@attributes":{"id":"p-0037","num":"0036"},"figref":"FIG. 10"},{"@attributes":{"id":"p-0038","num":"0037"},"figref":"FIG. 11"},{"@attributes":{"id":"p-0039","num":"0038"},"figref":"FIG. 12","sub":"GAL1"},{"@attributes":{"id":"p-0040","num":"0039"},"figref":["FIG. 13","FIG. 11"]},{"@attributes":{"id":"p-0041","num":"0040"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0042","num":"0041"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0043","num":"0042"},"figref":["FIG. 16","FIG. 6"]},{"@attributes":{"id":"p-0044","num":"0043"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0045","num":"0044"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0046","num":"0045"},"figref":"FIG. 19"},{"@attributes":{"id":"p-0047","num":"0046"},"figref":"FIG. 20"},{"@attributes":{"id":"p-0048","num":"0047"},"figref":"FIG. 21"},{"@attributes":{"id":"p-0049","num":"0048"},"figref":"FIG. 22"}]},"DETDESC":[{},{}]}
