---
title: Single nucleotide polymorphisms associated with bull fertility
abstract: Single nucleotide polymorphic sites of the bovine MAP1B, PPP1R11, and DDX4 genes are associated with improved bull fertility as measured by e.g. sire conception rates. Nucleic acid molecules, arrays, kits, methods of genotyping and marker-assisted bovine breeding methods based on these SNPs are disclosed.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09102985&OS=09102985&RS=09102985
owner: Wisconsin Alumni Research Foundation
number: 09102985
owner_city: Madison
owner_country: US
publication_date: 20130313
---

{"@attributes":{"id":"description"},"GOVINT":[{},{}],"heading":["STATEMENT OF GOVERNMENT INTEREST","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","Materials and Methods","REFERENCES"],"p":["This invention was made with government support under 12-CRHF-0-6055 awarded by the USDA\/NIFA. The government has certain rights in the invention.","The present invention relates to compositions and methods for testing and selecting cattle using molecular genetic methods by assaying for the presence of at least one genetic marker indicative of increased bull fertility. Specifically, genetic variations in the MAP1B and PPP1R11 genes are tested and used for selecting cattle animals with improved blastocyst or fertilization rates, or both.","The dairy cattle genome has been significantly restructured over the past 30 years due to intensive breeding effort selecting for production traits, including high quality milk and high and sustained productivity. However, while those efforts led to dramatic improvement of productivity, there has been significant reproductive deterioration in high-producing dairy cows, which in turn has caused substantial economic loss in the dairy cattle industry (Lucy, 2007, Fertility in high-producing dairy cows: reasons for decline and corrective strategies for sustainable improvement. 64 237-254). Key factors contributing to decreasing fertility of dairy cow are low fertilization rates and decreased embryonic survival.","Fertility is a complex trait that comprises developmental stages such as combining sperm and egg to form a zygote, compaction of embryo cells to form a morula, establishment of the blastocyst, attachment of the embryo to the uterus, and fetal development (Amann and DeJarnette, 2012). This complexity makes accurate prediction of successful pregnancy difficult, as aberrant development of sperm, oocyte, embryo, or fetus all would lead to conception failure. Conception rate in dairy cattle is about 40%, and only 50% of the fertilized eggs produce viable embryos (Santos et al., 2004). The decline in reproductive performance in cattle over the past few decades (Dobson et al., 2007) has been ascribed primarily to fertilization failure and early embryonic loss (Santos et al., 2004).","Previous studies have shown that genetic makeup of an individual plays crucial roles in embryonic development and reproductive success (Weigel, 2006; Shook, 2006). Although a male and female parent each contributes half of its genetic material to the new zygote and both are necessary for embryo development, it is not obvious whether or not this contribution is equally important to pregnancy success. For example, it is well established that the paternal genome supports growth of extra-embryonic tissues while the maternal genome fosters development of the embryo proper (Barton et al., 1984). After fertilization, the development of an embryo is controlled by maternal genomic information that is accumulated during oogenesis (Telford et al., 1990). It is only at the 8-cell stage in the bovine embryo that the embryonic genome activates and the embryo switches to transcribing its own RNA (Memili and First, 2000).","Despite that most breeding schemes in cattle are focused on the selection of elite bulls using progeny testing or genomic selection, and that some semen traits (e.g., sperm motility and percentage of abnormal sperm) show moderate to high heritabilities (Druet et al., 2009), most fertility studies in cattle have focused on the maternal contribution, and the paternal contribution to reproductive performance has not been thoroughly investigated, and only a few studies have been reported in the literature (Feugang et al., 2009; Khatib et al., 2010; Pe\u00f1agaricano et al., 2012). Therefore, characterization of bull fertility markers is both feasible and highly desirable, and the deployment of these markers in cattle breeding would lead to improved reproductive performance in cattle.","A recent comparative genomics study has characterized many genes involved in the control of spermatogenesis that were highly conserved from fly to human (Bonilla and Xu, 2008). Some of these genes were reported to be crucial for human fertility. However, it is not known whether or not these spermatogenesis genes play important roles in the fertility of bulls.","The present inventor carried out an association analysis between highly conserved spermatogenesis genes and sire conception rate (SCR) as a measure of bull fertility, with the objective that significant polymorphisms associated with bull fertility can be used as genetic markers in breeding programs aimed at improving reproductive performance in cattle.","Specifically, an association analysis is performed between highly conserved spermatogenesis genes and SCR in US Holstein populations as a measure of bull fertility. Sequence analysis revealed 24 single nucleotide polymorphisms (SNPs) in 9 genes in the bull population using the pooled DNA sequencing approach. These 9 genes were selected for their high level of sequence conservation between flies and humans. Overall, the 24 SNPs were tested for association with SCR in a population of 1,988 bulls. Three SNPs located in the MAP1B gene, one SNP in the PPP1R11 gene and one SNP in the DDX4 gene showed significant associations with SCR. Nucleotide probes based upon these SNPs are found to be useful for genetic testing of bull animals for improved fertilization rate.","Accordingly, in one embodiment, the present invention provides an isolated oligo- or poly-nucleotide molecule consisting of\n\n","In one embodiment, the nucleotide molecule of the present invention comprises at least about 15 contiguous nucleotides adjacent to its respective position (hereinafter the \u201cSNP position\u201d) of the respective figure. In one embodiment, the nucleic acid molecule of the present invention comprises at least about 20 contiguous nucleotides adjacent to the respective SNP position. In one embodiment, the oligonucleotide molecule of the present invention consists of not more than about 100 nucleotides. In one embodiment, the oligonucleotide molecule of the present invention consists of not more than about 50 nucleotides. In one embodiment, the SNP position of the nucleotide molecule of the present invention near or at the center of the molecule; alternatively, the SNP position is at the 3\u2032-end of the oligonucleotide molecule.","Also provided herein is an array of nucleic acid molecules, comprising the isolated oligonucleotide molecule of the present invention, supported on a substrate. The substrate may be any suitable medium, known and readily available to one of ordinary skills in the art, and the array may be addressable.","The present invention further provides a kit comprising an isolated oligonucleotide molecule of the present invention, and a suitable container.","In another embodiment, the present invention provides a method for detecting single nucleotide polymorphism (SNP) in a gene listed in Table 1 below in a bovine cell, the method comprising optionally isolating an DNA from the bovine cell, determining the identity of a nucleotide on the gene of the cell at a SNP position identified in Table 1 below, and comparing the identity to the preferred nucleotide identity at a corresponding position in Table 1.","In one embodiment, the bovine cell may be an adult cell, an embryo cell, a sperm, an egg, a fertilized egg, or a zygote. The identity of the nucleotide may be determined by many methods known and readily available to those ordinarily skilled in the art, such as but not limited to sequencing a nucleic acid molecule comprising a suitable portion of the gene of the cell comprising a respective SNP position, or by hybridizing a suitable probe to a nucleic acid preparation from the cell, which probe may be suitably labeled e.g. fluorescently or radioactively.","The nucleic acid molecule may be isolated from the cell via a large variety of methods, known and readily available to an ordinarily skilled artisan, such as amplification by the polymerase chain reaction (PCR) of genomic DNA of the cell, or when appropriate, by RT-PCR of the mRNA of the cell.","In preferred embodiment, both copies of the gene in a diploid genome are genotyped according to the method of the present invention.","The identity of the nucleotide may be determined based on genotypes of the parent of the cell, genotypes of the daughter of the cell, or both, through genetic analysis methods well-known to those skilled in the art.","A method is further provided for determining whether an individual bovine animal is suitable as a gamete donor for natural mating, artificial insemination or in vitro fertilization, the method comprising detecting the SNP according to the above method of the present invention, and excluding as gamete donor an individual which does not have the preferred allele identity at the respective SNP position as described in Table 1.","In one embodiment, the individual is excluded as a gamete donor if the individual, whose genotype is not homozygous of the preferred allele with regard to the respective SNP position.","The present invention additionally provides a method of selecting a bovine embryo for planting in a uterus, the method comprising genotyping the embryo according to the present invention, while preserving the viability of the embryo, and excluding from planting an embryo which does not have the preferred allele identity at the respective SNP position as described in Table 1.","In another embodiment, the present invention further provides a method for selectively breeding cattle using a multiple ovulation and embryo transfer procedure (MOET), the method comprising superovulating a female animal, collecting eggs from said superovulated female, in vitro fertilizing said eggs from a male animal which has at least one preferred allele identity at the respective SNP position as described in Table 1, implanting said fertilized eggs into other females allowing for an embryo to develop.","In another embodiment, the method for selectively breeding cattle using MOET may further comprise a step of genotyping the male animal according to the method of the present invention.","In another embodiment, the method for selectively breeding cattle using MOET may further comprise a step of genotyping the developing embryo, and allowing pregnancy to proceed only if the genotype of the embryo comprises at least one preferred allele identity at the respective SNP position as described in Table 1.","The present inventor has found that the spermatogenesis genes MAP1B, PPP1R11 and DDX4 showed significant associations with SCR. Table 1 below summarizes the SNPs demonstrated to be significantly associated with sire conception rates according to the present invention.",{"@attributes":{"id":"p-0033","num":"0038"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Genetic markers significantly associated with sire conception rate"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Nucleotide","Nucleotide"]},{"entry":[{},{},"Identity of","Identity of"]},{"entry":[{},{},"Preferred","Polymorph"]},{"entry":["SNP ID","Location\/Position","Allele","in GenBank"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["SNP1.MAP1B","Position 3066 of FIG. 1","A","A"]},{"entry":["SNP3.MAP1B","Position 3323 of FIG. 1","T","T"]},{"entry":["SNP5.MAP1B","Position 87071 of FIG. 3","C","C"]},{"entry":["SNP1.PPP1R11","Position 112 of FIG. 4","G","T"]},{"entry":["SNP1.DDX4","Position 61646 of FIG. 6","A","A"]},{"entry":["SNP2.DDX4","Position 34239 of FIG. 5","G","G"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},"Three SNPs in MAP1B, in low to moderate linkage disequilibrium (LD), were significantly associated with SCR. After correction for multiple testing, only one SNP in intron 5, SNP 5, showed the most significant association with SCR.","The MAP1B gene belongs to the microtubule-associated protein family and is known to affect neuronal development such as axon growth (Tymanskyj et al., 2012), development of dendritic spine and synaptic maturation (Tortosa et al., 2011), and regulation of the interaction between microtubules and actin microfilaments for axonal development (Montenegro-Venegas et al., 2010). Recent reports on the expression of MAP1B in the male reproductive tract in both rat and human (Queir\u00f3z et al., 2006) and in testis of fruit fly and mouse (Bonilla and Xu, 2008) suggest important functions of this gene in the regulation of male fertility. The finding by the present inventor, disclosed herein, that the MAP1B gene is associated with SCR, supports the conclusion that MAP1B plays a role in male fertility across a wide range of species.","The present inventor also found that a SNP in the 5\u2032UTR of PPP1R11 was associated with SCR in the bull population examined in this study. This is the first report of association between male fertility in cattle and PPP1R11, which is consistent with previous reports on the roles of this gene in spermatogenesis in mouse and human. For example, the different isoforms of PPP1R11 (also known as TCTEX5) were found to be expressed in most mouse tissues with high expression in testis, epididymis, and in the head and tail regions of spermatozoa (Han et al., 2007). In a subsequent study, it was shown that mutations in the long transcript of PPP1R11 were associated with normal sperm function (Han et al., 2008). The authors concluded that PPP1R11 plays important roles in sperm motility and spermatogenesis. A recent study reported that an isoform of protein phosphatase 1 (PP1\u03b32), which has an essential role in spermatogenesis, forms a complex with PPP1R11 in the testis (Cheng et al., 2009). Given that PP1\u03b32 is regulated by PPP1R11, these results further support the idea that PPP1R11 has important functions in spermatogenesis.","The spermatogenesis genes investigated in this study were selected from a pool of genes whose expression is highly conserved in testis of both fruit fly and mouse (Bonilla and Xu, 2008). The protein sequence identities between cattle and human and between cattle and fly are 91% and 32%, respectively for MAP1B and 99% and 47%, respectively for PPP1R11. As such, the association of these genes with bull fertility testifies to the usefulness of the comparative genomics approach in selecting candidate male fertility genes.","To further explore involvement of male fertility genes identified in this study in female fertility, we tested the association of the SNPs in MAP1B, with fertilization and embryo survival rates using data from the IVF system. MAP1B genotypes of the cows, from which oocytes were extracted and used for fertilization and embryo culture, were significantly associated with differential fertilization rate and embryo survival rate. Recently, the expression of MAP1B was found to be downregulated in follicular cystic follicles compared to normal follicles, suggesting that alteration in MAP1B expression may be involved in reproduction failure in cattle (Choe et al., 2010). The instant disclosure again demonstrates the significance of both parental genomes to embryonic development and fertility.","Accordingly, the present invention provides nucleic acid-based genetic markers for identifying bovine animals, especially bulls, with superior fertility, specifically, sire conception rate as a measure of male fertility. In general, for use as markers, isolated oligonucleotide or polynucleotide molecules, or isolated nucleic acid fragments, preferably DNA fragments, as used. Such markers will be of at least 10 nucleotides (nt), preferably at least 11, 12, or 15 nt, usually at least 20 nt, often at least 50 nt. Such small DNA fragments are useful as primers for the polymerase chain reaction (PCR), and probes for hybridization screening, etc.","In one embodiment, the isolated nucleic acid molecule comprises not more than 1,500 nt, or not more than 1000 nt, or not more than 900 nt, or not more than 800 nt, or not more than 700 nt, or not more than 600 nt, or not more than 500 nt, or not more than 400 nt, or not more than 300 nt, or not more than 200 nt., or not more than 150 nt., or not more than 100 nt., or not more than 75 nt.","In the context of the present invention, the term \u201cisolated\u201d refers to a nucleic acid molecule purified to some degree from endogenous materials with which the nucleic acid molecule may naturally occur or exist. At the least, the term \u201cisolated\u201d refers to a nucleic acid molecule separated from chromatin or other protein or components of the genomic DNA. Preferably, the isolated oligonucleic acid molecule or polynucleic acid molecule of the present invention comprises a fragment that is shorter than that which is naturally occurring.","In the context of the present invention, the provided sequences also encompass the complementary sequence corresponding to any of the provided polymorphisms. Where appropriate, and in order to provide an unambiguous identification of the specific site of a polymorphism, the numbering of the original nucleic sequences in the GenBank may be used; alternatively, the numbering may simply refer to the specific sequence in the Sequence Listing accompanying this disclosure.","The term primer refers to a single-stranded oligonucleotide capable of acting as a point of initiation of template-directed DNA synthesis under appropriate conditions (i.e., in the presence of four different nucleoside triphosphates and an agent for polymerization, such as, DNA or RNA polymerase or reverse transcriptase) in an appropriate buffer and at a suitable temperature. The appropriate length of a primer depends on the intended use of the primer but typically ranges from 15 to 30 nucleotides. Short primer molecules generally require cooler temperatures to form sufficiently stable hybrid complexes with the template. A primer need not reflect the exact sequence of the template but must be sufficiently complementary to hybridize with a template. The term primer site, or priming site, refers to the area of the target DNA to which a primer hybridizes. The term primer pair means a set of primers including a 5\u2032 upstream primer that hybridizes with the 5\u2032 end of the DNA sequence to be amplified and a 3\u2032, downstream primer that hybridizes with the complement of the 3\u2032 end of the sequence to be amplified. One of these two primers is often referred to as the \u201cforward primer,\u201d while the other the \u201creverse primer.\u201d","The term \u201cprobe\u201d or \u201chybridization probe\u201d denotes a defined nucleic acid segment (or nucleotide analog segment) which can be used to identify by hybridization a specific polynucleotide sequence present in samples, said nucleic acid segment comprising a nucleotide sequence complementary of the specific polynucleotide sequence to be identified. \u201cProbes\u201d or \u201chybridization probes\u201d are nucleic acids capable of binding in a base-specific manner to a complementary strand of nucleic acid.","An objective of the present invention is to determine which embodiment of the polymorphisms a specific sample of DNA has. For example, it is desirable to determine whether the nucleotide at a particular position is A or G. An oligonucleotide probe can be used for such purpose. Preferably, the oligonucleotide probe will have a detectable label, and contains an A at the corresponding position. Experimental conditions can be chosen such that if the sample DNA contains an A, they hybridization signal can be detected because the probe hybridizes to the corresponding complementary DNA strand in the sample, while if the sample DNA contains a G, no hybridization signal is detected.","Similarly, PCR primers and conditions can be devised, whereby the oligonucleotide is used as one of the PCR primers, for analyzing nucleic acids for the presence of a specific sequence. These may be direct amplification of the genomic DNA, or RT-PCR amplification of the mRNA transcript of the gene of interest. The use of the polymerase chain reaction is described in Saiki et al. (1985) Science 230:1350-1354. Amplification may be used to determine whether a polymorphism is present, by using a primer that is specific for the polymorphism. Alternatively, various methods are known in the art that utilize oligonucleotide ligation as a means of detecting polymorphisms, for examples see Riley et al (1990) Nucleic Acids Res. 18:2887-2890; and Delahunty et al (1996) Am. J. Hum. Genet. 58:1239-1246. The detection method may also be based on direct DNA sequencing, or hybridization, or a combination thereof. Where large amounts of DNA are available, genomic DNA is used directly. Alternatively, the region of interest is cloned into a suitable vector and grown in sufficient quantity for analysis. The nucleic acid may be amplified by PCR, to provide sufficient amounts for analysis.","Hybridization may be performed in solution, or such hybridization may be performed when either the oligonucleotide probe or the target polynucleotide is covalently or noncovalently affixed to a solid support. Attachment may be mediated, for example, by antibody-antigen interactions, poly-L-Lys, streptavidin or avidin-biotin, salt bridges, hydrophobic interactions, chemical linkages, UV cross-linking baking, etc. Oligonucleotides may be synthesized directly on the solid support or attached to the solid support subsequent to synthesis. Solid-supports suitable for use in detection methods of the invention include substrates made of silicon, glass, plastic, paper and the like, which may be formed, for example, into wells (as in 96-well plates), slides, sheets, membranes, fibers, chips, dishes, and beads. The solid support may be treated, coated or derivatized to facilitate the immobilization of the allele-specific oligonucleotide or target nucleic acid. For screening purposes, hybridization probes of the polymorphic sequences may be used where both forms are present, either in separate reactions, spatially separated on a solid phase matrix, or labeled such that they can be distinguished from each other.","Hybridization may also be performed with nucleic acid arrays and subarrays such as described in WO 95\/11995. The arrays would contain a battery of allele-specific oligonucleotides representing each of the polymorphic sites. One or both polymorphic forms may be present in the array. Usually such an array will include at least 2 different polymorphic sequences, i.e. polymorphisms located at unique positions within the locus, and may include all of the provided polymorphisms. Arrays of interest may further comprise sequences, including polymorphisms, of other genetic sequences, particularly other sequences of interest. The oligonucleotide sequence on the array will usually be at least about 12 nt in length, may be the length of the provided polymorphic sequences, or may extend into the flanking regions to generate fragments of 100 to 200 nt in length. For examples of arrays, see Ramsay (1998) Nat. Biotech. 16:4044; Hacia et al. (1996) Nature Genetics 14:441-447; Lockhart et al. (1996) Nature Biotechnol. 14:1675-1680; and De Risi et al. (1996) Nature Genetics 14:457-460.","The identity of polymorphisms may also be determined using a mismatch detection technique, including but not limited to the RNase protection method using riboprobes (Winter et al., Proc. Natl. Acad. Sci. USA 82:7575, 1985; Meyers et al., Science 230:1242, 1985) and proteins which recognize nucleotide mismatches, such as the mutS protein (Modrich, P. Ann Rev. Genet. 25:229-253, 1991). Alternatively, variant alleles can be identified by single strand conformation polymorphism (SSCP) analysis (Orita et al., Genomics 5:874-879, 1989; Humphries et al., in Molecular Diagnosis of Genetic Diseases, R. Elles, ed., pp. 321-340, 1996) or denaturing gradient gel electrophoresis (DGGE) (Wartell et al., Nucl. Acids Res. 18:2699-2706, 1990; Sheffield et al., Proc. Natl. Acad. Sci. USA 86:232-236, 1989).","A polymerase-mediated primer extension method may also be used to identify the polymorphism(s). Several such methods have been described in the patent and scientific literature and include the \u201cGenetic Bit Analysis\u201d method (WO92\/15712) and the ligase\/polymerase mediated genetic bit analysis (U.S. Pat. No. 5,679,524). Related methods are disclosed in WO91\/02087, WO90\/09455, WO95\/17676, U.S. Pat. Nos. 5,302,509, and 5,945,283. Extended primers containing a polymorphism may be detected by mass spectrometry as described in U.S. Pat. No. 5,605,798. Another primer extension method is allele-specific PCR (Ruao et al., Nucl. Acids Res. 17:8392, 1989; Ruao et al., Nucl. Acids Res. 19, 6877-6882, 1991; WO 93\/22456; Turki et al., J. Clin. Invest. 95:1635-1641, 1995). In addition, multiple polymorphic sites may be investigated by simultaneously amplifying multiple regions of the nucleic acid using sets of allele-specific primers as described in Wallace et al. (WO 89\/10414).","A detectable label may be included in an amplification reaction. Suitable labels include fluorochromes, e.g. fluorescein isothiocyanate (FITC), rhodamine, Texas Red, phycoerythrin, allophycocyanin, 6-carboxyfluorescein (6-FAM), 2\u2032,7\u2032-dimethoxy-4\u2032,5\u2032-dichloro-6-carboxyfluorescein (JOE), 6-carboxy-X-rhodamine (ROX), 6-carboxy-2\u2032,4\u2032,7\u2032,4,7-hexachlorofluorescein (HEX), 5-carboxyfluorescein (5-FAM) or N,N,N\u2032,N\u2032-tetramethyl-6-carboxyrhodamine (TAMRA), radioactive labels, e.g. P, S, H; etc. The label may be a two stage system, where the amplified DNA is conjugated to biotin, haptens, etc. having a high affinity binding partner, e.g. avidin, specific antibodies, etc., where the binding partner is conjugated to a detectable label. The label may be conjugated to one or both of the primers. Alternatively, the pool of nucleotides used in the amplification is labeled, so as to incorporate the label into the amplification product.","It is readily recognized by those ordinarily skilled in the art that in order to maximize the signal to noise ratio, in probe hybridization detection procedure, the polymorphic site should at the center of the probe fragment used, whereby a mismatch has a maximum effect on destabilizing the hybrid molecule; and in a PCR detection procedure, the polymorphic site should be placed at the very 3\u2032-end of the primer, whereby a mismatch has the maximum effect on preventing a chain elongation reaction by the DNA polymerase. The location of nucleotides in a polynucleotide with respect to the center of the polynucleotide are described herein in the following manner. When a polynucleotide has an odd number of nucleotides, the nucleotide at an equal distance from the 3\u2032 and 5\u2032 ends of the polynucleotide is considered to be \u201cat the center\u201d of the polynucleotide, and any nucleotide immediately adjacent to the nucleotide at the center, or the nucleotide at the center itself is considered to be \u201cwithin 1 nucleotide of the center.\u201d With an odd number of nucleotides in a polynucleotide any of the five nucleotides positions in the middle of the polynucleotide would be considered to be within 2 nucleotides of the center, and so on. When a polynucleotide has an even number of nucleotides, there would be a bond and not a nucleotide at the center of the polynucleotide. Thus, either of the two central nucleotides would be considered to be \u201cwithin 1 nucleotide of the center\u201d and any of the four nucleotides in the middle of the polynucleotide would be considered to be \u201cwithin 2 nucleotides of the center,\u201d and so on.","In some embodiments, a composition contains two or more differently labeled oligonucleotides for simultaneously probing the identity of nucleotides or nucleotide pairs at two or more polymorphic sites. It is also contemplated that primer compositions may contain two or more sets of allele-specific primer pairs to allow simultaneous targeting and amplification of two or more regions containing a polymorphic site.","Alternatively, the relevant portion of the gene of the sample of interest may be amplified via PCR and directly sequenced, and the sequence be compared to the wild type sequence shown in the figures. It is readily recognized that, other than those disclosed specifically herein, numerous primers can be devised to achieve the objectives. PCR and sequencing techniques are well known in the art and reagents and equipments are readily available commercially.","Alternatively, an invasive signal amplification assay, as described in e.g. U.S. Pat. No. 5,422,253 and Lyamichev et al., 2000, Biochemistry 39:9523-9532, both incorporated herein by reference in their entirety, may be used for detecting the SNP of interest. This assay takes advantage of enzymes such as the 5\u2032 nuclease activity of a DNA polymerase or the gene 6 product from bacteriophage T7 in their ability to cleave polynucleotide molecules by recognizing specific structures instead of specific sequences. A single-stranded target molecule is annealed to a pilot oligonucleotide such that the 5\u2032 end of the pilot forms a duplex with the target molecule. If the 3\u2032 end of the pilot oligonucleotide does not pair with the target, a 3\u2032 arm is formed. When exposed to a cleavage agent such as a DNA polymerase having a 5\u2032 nuclease activity or the gene 6 product from bacteriophage T7, the target molecule is cleaved in the 5\u2032 region, one nucleotide into the duplex adjacent to the unpaired region of the target. If a cut in a double-stranded molecule is required, the double-stranded molecule is denatured. Because this unpaired 3\u2032 arm can be as short as one nucleotide, this assay can be used for detecting a single-nucleotide difference, e.g. in the context of SNP detection. The pilot oligonucleotide is designed such that it pairs perfectly with one allele, but has a 3\u2032, single nucleotide mismatch with another allele. Cleavage only occurs if there is a mismatch between the target molecule and the pilot. To achieve signal amplification, the above invasive reaction is modified such that cleavage occurs on the pilot oligonucleotide. Two oligonucleotides are annealed in an adjacent manner to the target molecule. The resulting adjacent duplexes overlaps by at least one nucleotide to create an efficient substrate, called the overlapping substrate, for the 5\u2032 nucleases. The 5\u2032 end of the downstream oligonucleotide, also called the probe, contains an unpaired region termed the 5\u2032 arm (Lyamichev et al., 1993260:778-783.) or flap (Harrington and Lieber, 199413: 1235-1246) that is not required for the enzyme activity; however, very long arms can inhibit cleavage (Lyamichev et al., 1993260:778-783). Specific cleavage of the probe, termed invasive cleavage (Lyamichev et al., 199917 292-296; Kwiatkowski et al., 19994, 353-364.), occurs at the position defined by the 3\u2032 end of the upstream oligonucleotide, which displaces or \u201cinvades\u201d the probe. If the overlap between the adjacent oligonucleotides is only one nucleotide, cleavage takes place between the first two base pairs of the probe, thus releasing its 5\u2032 arm and one nucleotide of the base paired region (Lyamichev et al., 199996: 6143-6148, and Kaiser et al., 1999274:21387-21394). If the upstream oligonucleotide and the probe are present in large molar excess over the target nucleic acid, and invasive cleavage is carried out near the melting temperature of the probe, a cut probe can rapidly dissociate, and an intact probe will anneal to the target more frequently than will a cut probe, thus initiating a new cycle of cleavage. This allows multiple probes to be cut for each target molecule under isothermal conditions, resulting in linear signal amplification with respect to target concentration and time (Lyamichev et al., 199917: 292-296).","DNA markers have several advantages; segregation is easy to measure and is unambiguous, and DNA markers are co-dominant, i.e., heterozygous and homozygous animals can be distinctively identified. Once a marker system is established selection decisions could be made very easily, since DNA markers can be assayed any time after a blood sample can be collected from the individual infant animal, or even earlier by testing embryos in vitro if very early embryos are collected. The use of marker assisted genetic selection will greatly facilitate and speed up cattle breeding problems. For example, a modification of the multiple ovulation and embryo transfer (MOET) procedure can be used with genetic marker technology. Specifically, females are superovulated, eggs are collected, in vitro fertilized using semen from superior males and implanted into other females allowing for use of the superior genetics of the female (as well as the male) without having to wait for her to give birth to one calf at a time. Developing blastomeres at the 4-8 cell stage may be assayed for presence of the marker, and selection decisions made accordingly.","In one embodiment of the invention an assay is provided for detection of presence of a desirable genotype using the markers.","The term \u201cgenotype\u201d as used herein refers to the identity of the alleles present in an individual or a sample. In the context of the present invention a genotype preferably refers to the description of the polymorphic alleles present in an individual or a sample. The term \u201cgenotyping\u201d a sample or an individual for a polymorphic marker refers to determining the specific allele or the specific nucleotide carried by an individual at a polymorphic marker.","The present invention is suitable for identifying a bovine, including a young or adult bovine animal, an embryo, a semen sample, an egg, a fertilized egg, or a zygote, or other cell or tissue sample therefrom, to determine whether said bovine possesses the desired genotypes of the present invention, some of which are indicative of improved reproduction traits.","Further provided is a method for genotyping the bovine genes identified in Table 1, comprising determining for the two copies of the gene in a diploid genome present the identity of the nucleotide pair at the relevant SNP position (see below).","One embodiment of a genotyping method of the invention involves examining both copies of the gene, or a fragment thereof, to identify the nucleotide pair at the polymorphic site in the two copies to assign a genotype to the individual. In some embodiments, \u201cexamining a gene\u201d may include examining one or more of: DNA containing the gene, mRNA transcripts thereof, or cDNA copies thereof. As will be readily understood by the skilled artisan, the two \u201ccopies\u201d of a gene, mRNA or cDNA, or fragment thereof in an individual may be the same allele or may be different alleles. In another embodiment, a genotyping method of the invention comprises determining the identity of the nucleotide pair at the polymorphic site.","The present invention further provides a kit for genotyping a bovine sample, the kit comprising in a container a nucleic acid molecule, as described above, designed for detecting the polymorphism, and optionally at least another component for carrying out such detection. Preferably, a kit comprises at least two oligonucleotides packaged in the same or separate containers. The kit may also contain other components such as hybridization buffer (where the oligonucleotides are to be used as a probe) packaged in a separate container. Alternatively, where the oligonucleotides are to be used to amplify a target region, the kit may contain, preferably packaged in separate containers, a polymerase and a reaction buffer optimized for primer extension mediated by the polymerase, such as PCR.","In one embodiment the present invention provides a breeding method whereby genotyping as described above is conducted on a bovine animal, especially a bull, and based on the results, certain cattle are either selected or dropped out of the breeding program.","Through use of the linked marker loci, procedures termed \u201cmarker assisted selection\u201d (MAS) may be used for genetic improvement within a breeding nucleus; or \u201cmarker assisted introgression\u201d for transferring useful alleles from a resource population to a breeding nucleus (Soller 1990; Soller 1994).","A method is further provided for determining whether an individual bovine animal is suitable as a gamete donor for natural mating, artificial insemination or in vitro fertilization, the method comprising determining the identity of one or more SNPs according to the present invention using a method of the present invention, and excluding as gamete donor an individual which does not have the preferred allele identity at the respective SNP position as described in Table 1.","Specifically, an individual bovine animal, especially a bull, is excluded as a gamete donor, if its genome does not have at least:\n\n","In one embodiment, the individual is excluded as a gamete donor if the genotype of the individual is not homozygous of the preferred allele with regard to the respective SNP position.","The present invention additionally provides a method of selecting a bovine embryo for planting in a uterus, the method comprising genotyping the embryo according to the present invention, while preserving the viability of the embryo, and excluding from planting an embryo which does not have the preferred allele identity at the respective SNP position as described in Table 1.","In another embodiment, the present invention further provides a method for selectively breeding cattle using a multiple ovulation and embryo transfer procedure (MOET), the method comprising superovulating a female animal, collecting eggs from said superovulated female, in vitro fertilizing said eggs from a male animal which has at least one preferred allele identity at the respective SNP position as described in Table 1 and above, implanting said fertilized eggs into other females allowing for an embryo to develop.","In another embodiment, the method for selectively breeding cattle using MOET may further comprise a step of genotyping the male animal according to the method of the present invention.","In another embodiment, the method for selectively breeding cattle using MOET may further comprise a step of genotyping the developing embryo, and allowing pregnancy to proceed only if the genotype of the embryo comprises at least one preferred allele identity at the respective SNP position as described in Table 1.","The following examples are intended to illustrate preferred embodiments of the invention and should not be interpreted to limit the scope of the invention as defined in the claims.","Associations of candidate fertility genes examined were carried out in two experiments. In the first part, single nucleotide polymorphisms (SNPs) in the spermatogenesis genes were tested for associations with sire conception rate (SCR) in a large bull population. In the second part, genes found significantly associated with SCR were tested for association with female fertility traits (fertilization and blastocyst rates). Male fertility genes that play roles in female fertility can be used to improve reproductive performance in cattle using genetic information from both males and females in breeding schemes.","Gene Selection, SNP Identification, and Genotyping for Bull Fertility","A total of 58 spermatogenesis genes, with conserved testicular expression from fly to human, were reported in Bonilla and Xu (2008). Of those genes, only 22 were annotated in the bovine genome. For SNP identification and genotyping, genomic DNA was extracted from semen samples of 268 Holstein bulls (Genex Cooperative\/CRI, Shawano, Wis.) using standard phenol\/chloroform protocols. One DNA pool was constructed from 20 random semen samples with equal amounts of DNA. The DNA pool was amplified using primers designed in the 22 candidate genes to amplify 5\u2032 untranslated regions (UTRs), exons, introns, and 3\u2032 UTRs. The PCR products were sequenced, and SNPs were identified by visually inspecting sequence traces. PCR amplification and sequencing were performed as described in Khatib et al. (2008). Table 2 shows the primer sets used to amplify the nine candidate spermatogenesis genes found to be polymorphic in the bull population.",{"@attributes":{"id":"p-0076","num":"0087"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"308pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20032"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Primers\u2003used\u2003to\u2003amplify\u2003the\u2003nine\u2003spermatogenic\u2003genes"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"21pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"SEQ",{},"SEQ",{}]},{"entry":[{},{},"ID",{},"ID","Product"]},{"entry":["Gene","Forward\u2003Primer","NO:","Reverse\u2003Primer","NO:","Size"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["DCUN1D1","ATACCCTTAGGCAGTTAG","\u20037","AATTGTAAACCCTGAGAC","\u20038","536"]},{"entry":{}},{"entry":["DDX4(1)","AAACACGGAACAGAGGGT","\u20039","AGGCAGGATTAGCAAGTATG","10","404"]},{"entry":{}},{"entry":["DDX4(2)","AACCAAGTGGCTGGGATG","11","CAGACTCAAATGCGACAA","12","354"]},{"entry":{}},{"entry":["DNAI1(1)","CGGTAAGTGAGCAGCATC","13","ACTGAAGCCTTTGCCCTA","14","495"]},{"entry":{}},{"entry":["DNAI1(2)","CCCAGTGCTCCAAATCCT","15","ATGGCTCATCTTGTCTTCAGTA","16","413"]},{"entry":{}},{"entry":["DNAI1(3)","CGTGACTGGGTTTAGGAT","17","CTGGTGGCTGCTGTCTAT","18","602"]},{"entry":{}},{"entry":["GAPDHS(1)","CCAGGAAACGGCATCACC","19","ACACGCAGCAGGGCAACT","20","414"]},{"entry":{}},{"entry":["GAPDHS(2)","GTGAAGGCCAGGGACTATGA","21","ACATGAACAAGAGGGCTGCT","22","541"]},{"entry":{}},{"entry":["GSTM3","TTCTCTTCCCTGCAAGTCGT","23","TGAGAACAGCTGCCATCATC","24","664"]},{"entry":{}},{"entry":["MAP1B(1)","CCATTTCCTAAGGCACAG","25","TTCCGCCATCTTCCTACA","26","504"]},{"entry":{}},{"entry":["MAP1B(2)","CTTATGGTCGTGATTATGAA","27","AAGGCTAACACTGCTGGT","28","502"]},{"entry":{}},{"entry":["MAP1B(3)","GGCTGTGACATACCTACC","29","CAGACCTTCCCTACTTATT","30","645"]},{"entry":{}},{"entry":["PPP1R11(1)","CACATTACGGCGGAACTA","31","ATCCCAAGCAGTATCACCTA","32","591"]},{"entry":{}},{"entry":["PPP1R11(2)","ACCTGTTCTATCTCCTCCCA","33","GTCACCTACCCACCTTGC","34","543"]},{"entry":{}},{"entry":["SPATA20","TTGGAGAAGAAACCCACCAG","35","CCTCACAAGCAAGGCTAAGG","36","459"]},{"entry":{}},{"entry":["UBC","TCGCTCAGTCGTGTCTTAC","37","TCAACCAACGCCTAATGT","38","420"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},"A total of 24 SNPs located in the nine spermatogenic genes were genotyped in the Genex population (268 animals) by MALDI-TOF MS (GeneSeek Inc., Lincoln, Nebr.).","Imputation of SNPs for Validation of Significant SNPs Found in Genex Population in a Larger Bull Population Obtained from the USDA","Solely for the purpose of validating the conclusions drawn from the results using the Genex population, and to increase the sample size and improve the statistical power of the study, the 24 SNPs identified in the 9 candidate genes and genotyped in 268 bulls (from here forward, the reference population) were imputed in a total of 1,720 bulls (from here forward, the imputed population) so that a final dataset of 1,988 bulls with genotypic data was generated for subsequent statistical analyses. Bulls in the reference and imputed populations have been previously genotyped with the Illumina BovineSNP50 Bead Chip, and hence shared SNP were used to infer the genotypes of the unshared SNP in the imputed population. Genotypes of the reference (n=268) and the imputed populations (n=1,720) for the 50K SNP Chip were provided by Genex Cooperative\/CRI (Shawano, Wis.) and the Animal Improvement Programs Laboratory of the United States Department of Agriculture (AIPL, USDA; Beltsville, Md.), respectively. SNP with minor allele frequencies below 5% were removed. After data editing, 38,265 SNP spanning the entire bovine genome were available in both populations for the imputation process.","Imputation of SNPs was carried out for each candidate gene separately. In each case, a total of 100 SNP on each side of the gene were used to infer the genotypes of the ungenotyped SNP Imputation was performed using the population-based haplotype clustering algorithm of Scheet and Stephens (2006), which was implemented via the fastPHASE version 1.2 software using the default settings for all parameters (University of Washington TechTransfer Digital Ventures Program, Seattle, Wash.).","Phenotypic Data for Bull Fertility","The 1,988 bulls genotyped with the 50K SNP Chip were evaluated for sire conception rate (SCR), a phenotypic evaluation of bull fertility provided to dairy producers by AIPL-USDA as described in Pe\u00f1agaricano et al. (2012). Briefly, SCR is the expected difference in conception rate of a sire compared with the mean of all other evaluated sires (Kuhn and Hutchison, 2008; Kuhn et al., 2008).","In this study, SCR values ranged from \u221210.66% to +6.80%, and the number of breedings per bull ranged from 303 to 111,402. SCR data were obtained from seven consecutive evaluations provided by AIPL-USDA between August 2008 and December 2010. For bulls with multiple evaluations, the most recent SCR evaluation was used in the analysis.","Statistical Analysis for Bull Fertility","The association between each SNP and SCR was evaluated using the following mixed linear model,\n\nSCR=\u03bc+EVAL+\u03b2SNP+sire\n\nwhere \u03bc is the general mean, EVALis the fixed effect of the jAIPL-USDA SCR evaluation (j=1, 2, . . . , 7), SNPis the number of copies of one allele of the SNP (corresponding to 0, 1 or 2 copies) carried by the ianimal (i=1, 2, . . . , 1988), \u03b2 is the regression coefficient for the SNP considered (also known as the allele substitution effect), sirerepresents the random additive genetic effect of the lsire (l=1, 2, . . . , 246) of the ianimal, and erepresents the random residual for each observation. To detect possible deviations from the additive model, associations between genotype and SCR were evaluated using SNP as a categorical variable.\n","Random effects were assumed to follow the multivariate normal distribution,",{"@attributes":{"id":"p-0087","num":"0098"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mrow":[{"mo":["(",")"],"mrow":{"mrow":{"mtable":{"mtr":[{"mtd":{"mi":"s"}},{"mtd":{"mi":"\u025b"}}]},"mo":"|","msubsup":{"mi":["\u03c3","s"],"mn":"2"}},"mo":",","msubsup":{"mi":["\u03c3","\u025b"],"mn":"2"}}},{"mi":"N","mo":"\u2061","mrow":{"mo":["[","]"],"mrow":{"mn":"0","mo":",","mrow":{"mo":["(",")"],"mtable":{"mtr":[{"mtd":[{"mrow":{"mi":"A","mo":["\u2062","\u2062"],"mstyle":{"mspace":{"@attributes":{"width":"0.3em","height":"0.3ex"}}},"msubsup":{"mi":["\u03c3","s"],"mn":"2"}}},{"mn":"0"}]},{"mtd":[{"mn":"0"},{"mrow":{"msup":{"mi":"W","mrow":{"mo":"-","mn":"1"}},"mo":"\u2062","msubsup":{"mi":["\u03c3","\u025b"],"mn":"2"}}}]}]}}}}}],"mo":"~"}}},"br":{},"sub":["s","\u03b5"],"sup":["2 ","2 "]},"Phenotypic and Genotypic Data for Cow Fertility","The most significant SNP for SCR (rs109423562 located in MAP1B) was further investigated for association analysis with fertilization and blastocyst rates\u2014the main cow fertility traits\u2014using an IVF system. The procedures of in vitro fertilization and subsequent embryo culture were described in Khatib et al. (2008). To generate fertilization and blastocyst rate data, a total of 6,282 in vitro fertilizations were performed, and a total of 4,207 embryos were produced using oocytes from 359 ovaries collected from 359 Holstein cows and semen samples from 12 Holstein bulls. For 74 ovaries, oocytes were fertilized by two different bulls each. Fertilization rate was calculated as the number of cleaved embryos at Day 2 post-fertilization divided by the total number of fertilized oocytes collected from one ovary. Blastocyst rate was calculated as the number of embryos that reached the blastocyst stage (Day 8) and appeared normal out of the total number of embryos produced.","The 359 ovaries were genotyped for SNP rs109423562 (G\/A) using PCR-RFLP. A 171 bp fragment was amplified using the primers 5\u2032-GCAGCTCTTTTAGGAGTGTTAGCGTCTGAT-3\u2032 (SEQ ID NO: 39) (forward) and 5\u2032-CTCACAGAGGGCATTTGACA-3\u2032 (SEQ ID NO: 40) (reverse). The PCR product was then digested by the restriction enzyme HinfI and electrophoresed on a 2.0% agarose gel. Allele G was cut while allele A was uncut.","Statistical Analysis for Cow Fertility","Association between SNP rs109423562 (G\/A) in MAP1B and fertilization and blastocyst rates were analyzed using the following mixed linear model,\n\n=\u03bc+ovary+sire+SNP\n\nwhere yrepresents the fertilization or blastocyst rate of oocyte k from ovary i fertilized with semen from bull j, \u03bc represents a general mean for the trait considered, ovaryrepresents the random effect of the individual ovary from which oocytes were harvested, sirerepresents the random effect of the sire used in the fertilization, SNPrepresents the fixed effect of the ovary genotype for the SNP considered, and erepresents the residuals, assumed normal, independent and identically distributed with mean 0 an variance I\u03c3. Ovaries and bulls were assumed uncorrelated with variance structures I\u03c3and I\u03c3, respectively. Association between the SNP and fertilization or blastocyst rate was tested again using a likelihood ratio test by comparing with a reduced model without the SNP effect. All the statistical analyses were performed using the pedigreemm package (Vazquez et al. 2010) of the R language\/environment (R Development Core Team 2009).\n\nResults\n","SNP Identification and Association of Candidate Genes with SCR","Sequencing analysis revealed 24 SNPs in 9 spermatogenesis genes (DCUN1D1, DNAI1, DDX4, GAPDHS, GSTM3, MAP1B, PPP1R11, SPATA20 and UBC) in the bull population using the pooled DNA sequencing approach. All 24 SNPs located in 9 candidate genes were tested for association with sire conception rate first in Genex population and then in a larger population of 1,988 bulls for validation.","Association Analysis in Genex Population","SNPs in the genes MAP1B, PPP1R11, and DDX4 are associated with SCR","For MAP1B gene, a SNP C\/T in intron 5 at position 9331992 (University of Maryland bovine version 3.1; UMD3.1) showed significant association with sire conception rate. Primers used to amplify the gene and SNP location are shown in . Frequency of allele T was 25% and frequency of allele C was 75%. The allele substitution effect was \u22120.15\u00b10.01 (p-value=0.01) so that allele C is favorable for SCR.","For PPP1R11, one SNP (T\/G) in the 5\u2032UTR region at position 28710268 (UMD3.1) was significantly associated with SCR in the Genex population, with allele G associated with increased SCR.  shows SNP location and primers used to identify the SNP.","For DDX4, two SNPs located at positions 23382814 (SNP1; A\/G) and 23410221 (SNP2; G\/A) were identified in the gene. For SNP locations see . The two SNPs were in almost complete linkage disequilibrium, so they have the same allele frequencies in the bull population examined. Genotype AA of SNP1 has 0.783 units of SCR versus \u22120.214 SCR units for GG genotype (P-value=0.05). Similarly, GG genotype of SNP2 has 0.749 units of SCR versus \u22120.494 for AA genotype.","Association of Spermatogenesis Genes in the Combined Genex and USDA Populations","Three SNPs located in MAP1B and one SNP in PPP1R11 showed significant associations with SCR (Table 3). The SNP with the most significant association with SCR is SNP5.MAP1B located in intron 5 with an allele substitution effect of \u22120.24 and a P-value of 0.001. The other two significant SNPs were SNP1.MAP1B and SNP3.MAP1B, both located in intron 1 with an allele substitution effect of 0.15 and P-values of 0.025 and 0.039, respectively. After Bonferroni correction for multiple testing, only SNP5.MAP1B remained significant (P-value=0.024). Pair-wise linkage disequilibrium (LD) tests of MAP1B SNPs showed a moderate LD (r=0.38) between SNP1.MAP1B and SNP3.MAP1B and SNP5.MAP1B. The LD between SNP3.MAP1B and SNP5.MAP1B was relatively low (r=0.14).","SNP1.PPP1R11 located in the 5\u2032UTR region of PPP1R11 showed significant association with SCR with an allele substitution effect of 0.15 and a P-value of 0.046.",{"@attributes":{"id":"p-0103","num":"0114"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 3"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Genetic markers significantly associated with sire conception rate"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Genotype","Allele substitution",{}]},{"entry":["SNP ID","Gene","(N)","effect \u00b1 SE","P-value"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["SNP1.MAP1B","MAP1B","GG (463)","0.15 \u00b1 0.01","0.025"]},{"entry":[{},{},"GA (987)"]},{"entry":[{},{},"AA (538)"]},{"entry":["SNP3.MAP1B","MAP1B","CC (972)","0.15 \u00b1 0.01","0.039"]},{"entry":[{},{},"CT (847)"]},{"entry":[{},{},"TT (169)"]},{"entry":["SNP5.MAP1B","MAP1B","CC (1081)","\u22120.24 \u00b1 0.01\u2002","0.001"]},{"entry":[{},{},"TC (763)"]},{"entry":[{},{},"TT (144)"]},{"entry":["SNP1.PPP1R11","PPP1R11","TT (1033)","0.15 \u00b1 0.01","0.046"]},{"entry":[{},{},"TG (821)"]},{"entry":[{},{},"GG (134)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},"Association of MAP1B with Fertilization Rate and Embryo Survival Rate","SNP5.MAP1B located in intron 5 of MAP1B showed the most significant association with sire conception rate in the bull population analyzed above. To characterize its impact on female fertility, we tested the association of this SNP with fertilization rate and blastocyst rate in the IVF system. SNP5.MAP1B showed significant associations with both fertilization rate (P-value=0.027) and blastocyst rate (P-value=0.029) (Table 4). Oocytes collected from genotype CT cows showed the lowest fertilization rate (59.9%) compared with that from CC (66.4%) and TT (66.3%) cows (Table 4). For blastocyst rate, the CT genotype again showed the lowest rate (27.1%) while homozygous CC and TT individuals showed blastocyst rates of 31.0% and 41.8%, respectively (Table 4).",{"@attributes":{"id":"p-0106","num":"0117"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE 4"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Association between SNP5 of MAP1B gene and"},{"entry":"fertilization rate and blastocyst rate"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Fertilization rate","Blastocyst rate"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"49pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Genotype (N)","Estimate \u00b1 SE","P-value","Estimate \u00b1 SE","P-value"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":["CC (321)","0.664 \u00b1 0.03","0.027","0.310 \u00b1 0.02","0.029"]},{"entry":["CT (84)","0.599 \u00b1 0.03",{},"0.271 \u00b1 0.03"]},{"entry":["TT (28)","0.663 \u00b1 0.04",{},"0.418 \u00b1 0.05"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0107","num":"0000"},"ul":{"@attributes":{"id":"ul0005","list-style":"none"},"li":["Amann, R. P., J. M. Dejarnette. 2012 Impact of genomic selection of AI dairy sires on their likely utilization and methods to estimate fertility: A paradigm shift. Theriogenology 77:795-817.","Barton, S. C., M. A. Surani, and M. L. Norris. 1984. Role of paternal and maternal genomes in mouse development. Nature. 311:374-376.","Bonilla, E., and E. Y. Xu. 2008. Identification and characterization of novel mammalian spermatogenic genes conserved from fly to human. Mol. Hum. Reprod. 14:137-142.","Cheng, L., S. Pilder, A. C. Nairn, S. Ramdas, and S. Vijayaraghavan. 2009. PP1gamma2 and PPP1R11 are parts of a multimeric complex in developing testicular germ cells in which their steady state levels are reciprocally related. PLoS One. 4:e4861.","Choe, C., Y. W. Cho, C. W. Kim, D. S. Son, J. Han, and D. Kang. 2010. Identification of differentially expressed genes in bovine follicular cystic ovaries. Korean J. Physiol. Pharmacol. 14:265-272.","Dobson, H., R. Smith, M. Royal, Ch. Knight, and I. Sheldon. 2007. The high-producing dairy cow and its reproductive performance. Reprod Domest Anim 42 Suppl 2:17-23.","Druet, T., S. Fritz, E. Sellem, B. Basso, O. G\u00e9rard, L. Salas-Cortes, P. Humblot, X. Druart, and A. Eggen. 2009. Estimation of genetic parameters and genome scan for 15 semen characteristics traits of Holstein bulls. J. Anim Breed. Genet. 126:269-277.","Feugang, J. M., A. Kaya, G. P. Page, L. Chen, T. Mehta, K. Hirani, L. Nazareth, E. Topper, R. Gibbs, and E. Memili 2009. Two-stage genome-wide association study identifies integrin beta 5 as having potential role in bull fertility. BMC Genomics. 10:176.","Han, Y. B., H. L. Feng, C. K. Cheung, P. M. Lam, C. C. Wang, and C. J. Haines. 2007. Expression of a novel T-complex testis expressed 5 (Tctex5) in mouse testis, epididymis, and spermatozoa. Mol. Reprod. Dev. 74:1132-1140.","Han, Y., X. X. Song, H. L. Feng, C. K. Cheung, P. M. Lam, C. C. Wang, and C. J. Haines. 2008. Mutations of t-complex testis expressed gene 5 transcripts in the testis of sterile t-haplotype mutant mouse. Asian J Androl. 10:219-226.","Imumorin, I. G., E. H. Kim, Y. M. Lee, D. J. De Koning, J. A. van Arendonk, M. De Donato, J. F. Taylor, and J. J. Kim. 2011. Genome Scan for Parent-of-Origin QTL Effects on Bovine Growth and Carcass Traits. Front. Genet. 2:44.","Khatib, H., R. L. Monson, W. Huang, R. Khatib, V. Schutzkus, H. Khateeb, and J. J. Parrish. 2010. Short communication: Validation of in vitro fertility genes in a Holstein bull population. J. Dairy. Sci. 93:2244-2249.","Khatib, H., W. Huang, X. Wang, A. H. Tran, A. B. Bindrim, V. Schutzkus, R. L. Monson, and B. S. Yandell. 2009. Single gene and gene interaction effects on fertilization and embryonic survival rates in cattle. J. Dairy Sci. 92:2238-2247.","Khatib, H., R. L. Monson, V. Schutzkus, D. M. Kohl, G. J. M. Rosa, and J. J. Rutledge. 2008. Mutations in the STAT5A gene are associated with embryonic survival and milk composition in cattle. J. Dairy Sci. 91:784-793.","Kuhn, M. T., and J. L. Hutchison. 2008. Prediction of dairy bull fertility from field data: use of multiple services and identification and utilization of factors affecting bull fertility. J. Dairy Sci. 91:2481-2492.","Kuhn, M. T., J. L. Hutchison, and H. D. Norman. 2008. Modeling nuisance variables for prediction of service sire fertility. J. Dairy Sci. 91:2823-2835.","Magee, D. A., D. P. Berry, E. W. Berkowicz, K. M. Sikora, D. J. Howard, M. P. Mullen, R. D. Evans, C. Spillane, and D. E. MacHugh. 2011. Single nucleotide polymorphisms within the bovine DLK1-DIO3 imprinted domain are associated with economically important production traits in cattle. J. Hered. 102:94-101.","Magee, D. A., K. M. Sikora, E. W. Berkowicz, D. P. Berry, D. J. Howard, M. P. Mullen, R. D. Evans, C. Spillane, and D. E. MacHugh. 2010. DNA sequence polymorphisms in a panel of eight candidate bovine imprinted genes and their association with performance traits in Irish Holstein-Friesian cattle. BMC Genet. 11:93.","Memili, E., and N. L. First. 2000. Zygotic and embryonic gene expression in cow: A review of timing and mechanisms of early gene expression as compared with other species. Zygote. 8:87-96.","Montenegro-Venegas, C., E. Tortosa, S. Rosso, D. Peretti, F. Bollati, M. Bisbal, I. Jausoro, J. Avila, A. C\u00e1ceres, and C. Gonzalez-Billault. 2010. MAP1B regulates axonal development by modulating Rho-GTPase Rac1 activity. Mol. Biol. Cell. 21:3518-3528.","Pe\u00f1agaricano, F., K. A. Weigel, and K. Khatib. 2012. Genome-wide association study identifies candidate markers for bull fertility in Holstein dairy cattle. Anim Genet. (In press).","Queir\u00f3z, D. B., A. M. Silva, G. Guti\u00e9rrez-Ospina, C. S. Porto, G. Grossman, P. Petrusz, and M. C. Avellar. 2006. Cells positive for microtubule-associated protein 1B (MAP 1B) are present along rat and human efferent ductules and epididymis. Cell Tissue Res. 325:125-133.","R Development Core Team. 2009. R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria.","Santos, J. E. P., W. W. Thatcher, R. C. Chebel, R. L. A. Cerri, and K. N. Galvao. 2004. The effect of embryonic death rates in cattle on the efficacy of estrus synchronization programs. Anim Reprod. Sci. 83:513-535.","Scheet, P., and M. Stephens. 2006. A fast and flexible statistical model for large-scale population genotype data: Applications to inferring missing genotypes and haplotypic phase. Am. J. Hum. Genet. 78:629-644.","Shook G. E. 2006. Major advances in determining appropriate selection goals. J. Dairy Sci. 89:1349-1361.","Telford, N. A., A. J. Watson, and G. A. Schultz. 1990. Transition from maternal to embryonic control in early mammalian development: A comparison of several species. Mol. Reprod. Dev. 26:90-100.","Tortosa, E., C. Montenegro-Venegas, M. Benoist, S. H\u00e4rtel, C. Gonz\u00e1lez-Billault, J. A. Esteban, and J. Avila. 2011. Microtubule-associated protein 1B (MAP1B) is required for dendritic spine development and synaptic maturation. J. Biol. Chem. 286:40638-40648.","Tymanskyj, S. R., T. M. Scales, and P. R. Gordon-Weeks. 2012. MAP1B enhances microtubule assembly rates and axon extension rates in developing neurons. Mol. Cell Neurosci. 49:110-119.","Vazquez, A. I., D. M. Bates, G. J. M. Rosa, D. Gianola, and K. A. Weigel. 2010. Technical Note: An R package for fitting generalized linear mixed models in animal breeding. J. Anim Sci. 88:497-504.","Weigel, K. A. 2006. Prospects for improving reproductive performance through genetic selection. Anim Reprod. Sci. 96: 323-330."]}}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0026","num":"0031"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0027","num":"0032"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0028","num":"0033"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0029","num":"0034"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0030","num":"0035"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0031","num":"0036"},"figref":"FIG. 6"}]},"DETDESC":[{},{}]}
