---
title: Optimization of gene expression analysis using immobilized capture probes
abstract: Disclosed are methods of multiplexed analysis of oligonucleotides in a sample, including a method of preventing a significant reduction in duplexes detectable in a hybridization assay involving (i) selecting probe lengths for sets of oligonucleotide probes, wherein probes include different subsequences such that at least one subsequence is complementary to a subsequence in a cognate target; wherein probes for longer cognate targets are longer in length than probes for shorter cognate targets, (ii) selecting, for each set of probes, a density of oligonucleotides probes attached per unit area on a solid phase carrier which is below a limit at which the significant reduction in detectable duplexes is predicated to take place, (iii) producing the probes and affixing them to different solid phase carriers at the selected density, and (iv) annealing targets to the probes, wherein signal intensities of probes and targets of different lengths are about the same.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09637777&OS=09637777&RS=09637777
owner: BioArray Solutions, Ltd.
number: 09637777
owner_city: Warren
owner_country: US
publication_date: 20140421
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["RELATED APPLICATIONS","GOVERNMENT INTEREST","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION","Example I","Effect of Probe and Transcript Length on Capture Efficiency","Example II","Determination of the Absolute Number of Fluorophores Present Per Particle","Example III","Generic Protocol for Rapid Expression Monitoring","Example IV","Analysis of Kanamycin mRNA (Using Protocol of Example III)","Example IVA","mpmp-RT Design and Transcript Labeling","Example IVB","Transcript Length and Improved RT Design","Example IVC","Optimization of Assay in Titration of Model mRNA","Example V","Optimization of Reverse Transcription of Model mRNA","Example VI","Spiking Experiments in Total Human RNA Background: Specificity","Example VII","Illustrative Applications","Example VIII","Multiplexed Expression Monitoring: Cytokine mRNA Panel","Example IX","Analysis of Highly Homologous mRNA Sequences in Maize Zein Gene Family"],"p":["This application is a continuation of U.S. application Ser. No. 12\/480,215, filed Jun. 8, 2009, now U.S. Pat. No. 8,795,960 issued on Aug. 5, 2014, which is a continuation of U.S. application Ser. No. 10\/974,036 filed Oct. 26, 2004, now U.S. Pat. No. 7,563,569 issued on Jul. 21, 2009, which claims priority to U.S. Provisional Application Ser. No. 60\/516,611, filed Oct. 28, 2003, and 60\/544,533, filed Feb. 14, 2004, the contents of each application being incorporated by reference in its entirety.","Agencies of the United States government may have certain rights in this application, as certain work was performed under a DARPA contract.","Gene Expression Analysis\u2014","Fundamental biological processes such as cell cycle progression, cell differentiation and cell death are associated with variations in gene expression patterns which therefore provide a means of monitoring these processes on a molecular level. Gene expression patterns can be affected by exposure to therapeutic agents, and they are thus useful molecular indicators of efficacy of new drugs and validation of drug targets. At present, gene expression analysis plays an increasingly important role in connection with target discovery.","Gene expression analysis also offers a systematic molecular approach to the analysis of multigenic traits. In the context of plant molecular biology and molecular agriculture, expression patterns of designated genes and their temporal evolution are finding increasing application to guide \u201cbreeding\u201d of desirable properties such as the rate of growth or ripening of fruits or vegetables.","Changes in expression levels also are indicators of the status and progression of pathogenesis. Thus, the under-expression of functional tumor suppressor genes and\/or over-expression of oncogenes or protooncogenes is known to be associated with the presence and progression of various cancers. Specific genes have been identified whose expression patterns undergo characteristic variations in the early stages of immune response to inflammation or exposure to pathogenic agents including common viruses such as HSV or CMV as well as biochemical warfare agents such as anthrax. Contrary to the expression of protein markers such as antibodies, gene expression occurs at the earliest stages of immune response, thereby offering the possibility of early and specific therapeutic intervention.","Accordingly, the rapid quantitative analysis of expression levels of specific genes (\u201cmessages\u201d) and their evolution in time following exposure to infectious agents\u2014or following treatment\u2014holds significant promise as a tool to advance the molecular diagnosis of disease. However, as elaborated in the present invention, standard methods of quantitative gene expression analysis produce data of uncertain quality. Further, as a reliable and practical tool of molecular diagnostics, gene expression analysis, and specifically multiplexed expression monitoring (herein also referred to in abbreviation as \u201cmEM\u201d), must be simple in protocol, quick to complete, flexible in accommodating selected sets of genes, reliable in controlling cross-reactivity and ensuring specificity, capable of attaining requisite levels of sensitivity while performing quantitative determinations of message abundance over a dynamic range of three to four orders of magnitude and convenient to use.","These attributes generally do not apply to current methods. That is, while gene expression analysis has become a standard methodology of target discovery, its use as a diagnostic methodology, particularly in expression monitoring, requiring the quantitative determination of cDNA levels in the target mixture as a measure of the levels of expression of the corresponding mRNAs, has been limited by the lack of flexible and reliable assay designs ensuring rapid, reliable and quantitative multiplexed molecular diagnosis.","Spatially Encoded Arrays: In-Situ Synthesis and \u201cSpotting\u201d\u2014","The practical utility of gene expression analysis is greatly enhanced when it is implemented using parallel assay formats that permit the concurrent (\u201cmultiplexed\u201d) analysis of multiple analytes in a single reaction. In a commonly practiced format (see, e.g., U. Maskos, E. M. Southern, 20, 1679-1684 (1992); S. P. A. Fodor, et al., 251, 767-773 (1991)), the determination of gene expression levels is performed by providing an array of oligonucleotide capture probes\u2014or, in some cases, cDNA molecules\u2014disposed on a planar substrate, and contacting the array\u2014under specific conditions permitting formation of probe-target complexes\u2014with a solution containing nucleic acid samples of interest; these can include mRNAs extracted from a particular tissue, or cDNAs produced from the mRNAs by reverse transcription (RT). Following completion of the step of complex formation (\u201chybridization\u201d), unbound target molecules are removed, and intensities are recorded from each position within the array, these intensities reflecting the amount of captured target. The intensity pattern is analyzed to obtain information regarding the abundance of mRNAs expressed in the sample. This \u201cmultiplexed\u201d assay format is gaining increasing acceptance in the analysis of nucleic acids as well as proteins in molecular medicine and biomedical research.","Lack of Flexibility, Reproducibility and Reliability\u2014","However, spatially encoded probe arrays generally are not well suited to quantitative expression analysis of designated sets of genes. Thus, in-situ photochemical oligonucleotide synthesis does not provide a flexible, open design format given the time and cost involved in customizing arrays. As a result, \u201cspotted\u201d, or printed arrays, which provide flexibility in the selection of probes, have been preferred in applications requiring the use of only a limited gene set. However, \u201cspotting\u201d continues to face substantial technical challenges akin to those encountered by the standard \u201cstrip\u201d assay format of clinical diagnostics, which generally is unsuitable for quantitative analysis. Poor reproducibility, relating to the non-uniformity of coverage, and uncertain configuration and accessibility of immobilized probes within individual spots, remains a significant concern. In addition, these arrays require expensive confocal laser scanning instrumentation to suppress substantial \u201cbackground\u201d intensities, and further require statistical analysis even at the early stages of subsequent data processing to account for non-uniform probe coverage and heterogeneity. Another concern is the comparatively large footprint of spotted arrays and the correspondingly large quantities of reagent consumed. Finally, scale-up of production to levels required for large-scale diagnostic use will be complex and economically unfavorable compared to batch processes such as those available for the preferred embodiment of the present invention in the form of planar arrays of encoded microparticles.","In addition to limited sensitivity, other problems with array-based diagnostics include limited ability to detect genes expressed in widely varying copy number (from 1 or 2 copies per cell to \u02dc10copies per cell). Thus, what is needed is an assay method which avoids these problems by maximizing detection sensitivity, minimizing cross-reactivity and permitting detection over a wide dynamic range of transcript copies.","Lack of Specificity\u2014","the most prevalent methods of the prior art rely on multiplexed probe-target hybridization as the single step of quantitative determination of, and discrimination between multiple target sequences. Hybridization is sometimes lacking in specificity in a multiplexed format of analysis (see discussion in U.S. application Ser. No. 10\/271,602, entitled: \u201cMultiplexed Analysis of Polymorphic Loci by Concurrent Interrogation and Enzyme-Mediated Detection,\u201d filed Oct. 15, 2002). To enhance specificity, some formats of multiplexed hybridization employ long probes in spotted arrays, e.g. Agilent EP 1207209 discloses probes of preferred length 10 to 30, and preferably about 25. These may help to offset the random obstruction and limited accessibility of capture sequences in spotted probes. That is, probe-target complex formation in spotted arrays generally will not involve the full length, but rather randomly accessible subsequences of the probe. However, as disclosed herein, the use of long probes in a solid phase format generally will be counterproductive. Furthermore, the lack of specificity remains a source of concern: as shown herein, cross-hybridization generally will distort intensity patterns, thereby precluding quantitative analysis unless careful primer and probe designs are employed, using, for example the methods of a co-pending application (U.S. application Ser. No. 10\/892,514, \u201cConcurrent Optimization in Selection of Primer and Capture Probe Sets for Nucleic Acid Analysis,\u201d filed Jul. 15, 2004) and performing careful analysis taking into account the molecular interactions between non-cognate probes and targets.","Differential Gene Expression (\u201cTranscript Profiling\u201d)\u2014","Given these difficulties of standard methods of the art, and the potential for serious uncertainty and error in the quantitative determination of absolute expression levels, the format usually preferred in practice is differential expression analysis. This format characterizes differences in expression patterns between normal tissue or cells vs diseased or otherwise altered tissue or cells, or differences between normal (\u201cwild-type\u201d) vs transgenic plants. In accordance with a commonly practiced approach, a set of cDNA clones is \u201cspotted\u201d onto a planar substrate to form the probe array which is then contacted with DNA from normal and altered sources. DNA from the two sources is differentially labeled to permit the recording of patterns formed by probe-target hybridization in two color channels and thus permitting the determination of expression ratios in normal and altered samples (see, e.g., U.S. Pat. No. 6,110,426 (Stanford University)). The system of two-color fluorescent detection is cumbersome, requiring careful calibration of the laser scanning instrumentation generally required to read spotted or other spatially encoded probe arrays\u2014and as well as separate scans for each of the two color channels. These disadvantages are overcome by the subtractive method of differential gene expression disclosed herein which requires only a single detection color.","Complex Protocols\u2014","In a commonly practiced approach to multiplexed expression profiling, mRNA molecules in a sample of interest are first reverse transcribed to produce corresponding cDNAs and are then placed in contact with an array of oligonucleotide capture probes formed by spotting or by in-situ synthesis. Lockhart et al. (U.S. Pat. No. 6,410,229) invoke a complex protocol to produce cRNA wherein mRNA is reverse transcribed to cDNA, which is in turn transcribed to cRNA under heavy labeling\u2014of one in eight dNTPs on average\u2014and detected on an array of synthesized oligonucleotide probes using a secondary \u201cdecoration\u201d step. Such a laborious, error-prone and expensive process not only greatly increases the complexity of the method but greatly contributes to the uncertainty of final determinations of message abundance, for example by producing non-linear amplification.","A preferred method of the prior art for multiplexed expression analysis is the use either of randomly placed short reverse transcription (RT) primers to convert a set of RNAs into a heterogeneous population of cDNAs or the use of a universal RT primer directed against the polyA tail of the mRNA to produce full-length cDNAs. While these methods obviate the need for design of sequence-specific RT primers, both have significant disadvantages in quantitative expression monitoring.","Randomly placed RT primers will produce a representative population of cDNAs, that is, one in which each cDNA is represented with equal frequency, only in the limit of infinitely long mRNA molecules. The analysis of a designated set of short mRNAs by random priming generally will produce cDNAs of widely varying lengths for each type of mRNA in the mixture, and this in turn will introduce potentially significant bias in the quantitative determination of cDNA concentration, given that short cDNAs will more readily anneal to immobilized capture probes than will long cDNAs, as elaborated in the present invention. Further, the production of full-length cDNAs, if in fact full-length RT is successful, provides a large sequence space for potential cross-reactivity between probes and primers, making the results inherently difficult to interpret and hence unreliable.","The Role of Target and Probe Configurations\u2014","DNA in solution has been shown to display the characteristics of polymers governed by chain entropy (see Larson et al., \u201cHydrodynamics of a DNA molecule in a flow field,\u201d Physical Review E 55:1794-97 (1997)). Especially single-stranded (ss) DNA is quite flexible, a fact which manifests itself in a short persistence length of the order of only a few nucleotides (nt) under most experimentally relevant conditions, considerably smaller than that of double stranded DNA (Marko J F, Siggia E D, \u201cFluctuations and supercoiling of DNA,\u201d 22:265, 506-(1994)). Capture of ssDNA to immobilized probes thus involves considerable restriction of the molecules' conformational freedom. At the same time if duplex formation is to occur, immobilized probes used in solid phase formats of nucleic acid analysis must accommodate invading target strands by elastic deformation. Conformational adjustments in target and probe molecules, considered as polymers, heretofore have not been appreciated in designing assays for nucleic acid analysis.","In view of the foregoing considerations, it will be desirable to have flexible, rapid, sensitive and specific methods, compositions and assay protocols particularly for diagnostic applications of gene expression analysis\u2014herein also referred to as multiplexed expression monitoring (mEM). The present invention discloses such methods and compositions, specifically methods and compositions for rapid, customizable, multiplexed assay designs and protocols for multiplexed expression monitoring, preferably implemented in the format of random encoded array detection for multianalyte molecular analysis. A co-pending application discloses methods by which to select optimized sets of desirable conversion probes (e.g. RT primers) and detection probes (e.g., probes for hybridization-mediated target capture) to further enhance the level of reliability (see U.S. application Ser. No. 10\/892,514 \u201cConcurrent Optimization in Selection of Primer and Capture Probe Sets for Nucleic Acid Analysis,\u201d filed Jul. 15, 2004).","Described herein are methods of multiplexed analysis of oligonucleotides in a sample, including: methods of probe and target \u201cengineering\u201d, as well as methods of assay signal analysis relating to the modulation of the probe-target affinity constant, K by a variety of factors including the elastic properties of target strands and layers of immobilized (\u201cgrafted\u201d) probes; and assay methodologies relating to: the tuning of assay signal intensities including dynamic range compression and on-chip signal amplification; the combination of hybridization-mediated and elongation-mediated detection for the quantitative determination of abundance of messages displaying a high degree of sequence similarity, including, for example, the simultaneous determination of the relative expression levels, and identification of the specific class of, untranslated AU-rich subsequences located near the 3\u2032 terminus of mRNA; and a new method of subtractive differential gene expression analysis which, requires only a single color label.","Specifically, disclosed are methods, designs and compositions relating to:\n\n","For optimizing the specificity of detection, the sequence specificity in multiplexed reverse transcription and detection is optimized by appropriate selection of primers and corresponding probes, as described in co-pending U.S. application Ser. No. 10\/892,514, filed Jul. 15, 2003, entitled \u201cConcurrent Optimization in Selection of Primer and Capture Probe Sets for Nucleic Acid Analysis,\u201d incorporated by reference, and also referred to herein for convenience as \u201capplication Ser. No. 10\/892,514.\u201d","Use of these methods of optimizing sensitivity and specificity permits the rapid, quantitative concurrent analysis of a designated set of genes by way of a reverse transcription of the given set of mRNAs to cDNAs and detection of these cDNAs by capture to a set of matching oligonucleotide probes, preferably on the basis of a simple protocol as disclosed herein, preferably obviating the need for a separate target amplification step, thereby simplifying the protocol and reducing the time to completion of the assay. The methods, protocols and designs described herein are particularly useful for a parallel format of multiplexed nucleic acid analysis, specifically quantitative analysis of expression patterns of a designated set of genes, the set of designated genes typically comprising between 2 and 100 different mRNAs (\u201cmessages\u201d), and more typically between 10 and 30 messages, the process herein referred to as multiplexed expression monitoring (mEM). The methods, protocols and designs herein can be used advantageously in conjunction with the READ\u2122 format of multiplexed expression monitoring, as described in U.S. application Ser. No. 10\/204,799, filed Aug. 23, 2002, entitled: \u201cMultianalyte molecular analysis using application-specific random particle arrays,\u201d hereby incorporated by reference.","The utility and advantages of the various methods, designs and compositions are set forth in detail below. A description of the drawings follows, which aid in understanding the inventions set forth herein.","Disclosed are methods, protocols and designs, including systematic procedures for enhancing the reliability of the process of determining levels of concentration (\u201cabundance\u201d) of multiple nucleic acid analytes by capture to anchored oligonucleotide probes, specifically including the concurrent (\u201cmultiplexed\u201d) analysis of the expression levels of a designated set of genes. More specifically, disclosed are methods for the optimization of sensitivity, specificity and dynamic range of multiplexed gene expression analysis, and further, assay protocols including a subtractive format of performing differential expression analysis using only a single detection color. Also introduced is an explicit phenomenological description of the interaction of targets with anchored probes in order to evaluate the actual affinity constant governing this process. A preferred embodiment of forming planar arrays of capture probes displayed on color-encoded microparticles, without recourse to target amplification as in the case of a cytokine reference panel described herein, may permit completion of quantitative multiplexed expression monitoring in as little as three hours or less, from sample collection to data analysis (). These methods and designs are herein illustrated by application to a variety of problems involving the capture of target nucleic acid strands to a layer of immobilized oligonucleotide probes.","I Optimizing Sensitivity and Dynamic Range: Modulation of Probe-Target Affinity","I.1 Sequence-Specific Affinity Governing Hybridization Complex (\u201cDuplex\u201d) Formation\u2014","The standard analysis of the hybridization-mediated formation of a complex (\u201cannealing\u201d) of two oligonucleotides invokes the law of mass action to relate the concentration of complexed (\u201cbound\u201d) probes and targets, c=[TP], to the concentration of uncomplexed (\u201cunbound\u201d, \u201cfree\u201d) probes, herein preferably displayed on encoded beads, p=[P], and the concentration of uncomplexed targets, t=[T], as follows:\n\n[\n\nor\n\n\n","In analogy to the common practice of computing \u201cmelting temperatures\u201d, the (sequence-dependent) affinity constant is computed using a phenomenological \u201cnearest-neighbor\u201d (NN) model to represent the interaction between adjacent base pairs formed within the probe-target complex for given experimental conditions including salt concentration and temperature. The free energy of duplex formation, also referred to herein as \u201cbinding energy\u201d or \u201ccondensation energy\u201d, is computed in the form:\n\n\u0394+\u03a3)\n\nwhere \u0394Hand \u0394Srepresent enthalpy and entropy, respectively. The condition \u0394G=defines the \u201cmelting temperature\u201d, T, widely used in the field to estimate the stability of a duplex.\n","In accordance with standard thermodynamics, the (sequence-specific) affinity constant, K, is computed from the expression\n\nexp(\u2212\u0394)\n\nwherein Krepresents a constant and k denotes the Boltzmann constant.\n","Given an affinity constant, and given initial concentrations of probe, [P], and target, [T], the equilibrium concentration of probe-target complex, [TP], is obtained as a function of initial target concentration [T].","Using this standard model, melting temperatures and affinity constants were calculated for complexes formed by a 175 nt DNA target and seven different DNA oligonucleotide probes varying in length from 15 nt to 35 nt at a temperature of 55\u00b0 C. and a salt concentrations of 2M. Target and probe sequences are shown below in Table I-1.",{"@attributes":{"id":"p-0073","num":"0088"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"203pt","align":"left"}}],"thead":{"row":[{"entry":"TABLE\u2003I-1"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}},{"entry":["Seq\u2003ID","Sequence"]},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["Target\u2003175-","AG\u2003GGT\u2003AAA\u2003ATT\u2003AAG\u2003CAC\u2003AGT\u2003GGA\u2003AGA\u2003ATT\u2003TCA\u2003TTC"]},{"entry":["mer","TGT\u2003TCT\u2003CAG\u2003TTT\u2003TCC\u2003TGG\u2003ATT\u2003ATG\u2003CCT\u2003GGC\u2003ACC"]},{"entry":["SEQ\u2003ID\u2003NO.","ATT\u2003AAA\u2003"]},{"entry":["1",{"u":"GAT\u2003GAA\u2003TA"}]},{"entry":[{},"CAA\u2003CTA\u2003GAA\u2003GAG\u2003GTA\u2003AGA\u2003AAC\u2003TAT\u2003GTG\u2003AAA\u2003ACT"]},{"entry":[{},"TTT\u2003TG"]},{"entry":{}},{"entry":["Target\u200390-","T\u2003CAG\u2003TTT\u2003TCC\u2003TGG\u2003ATT\u2003ATG\u2003CCT\u2003GGC\u2003ACC\u2003ATT\u2003AAA"]},{"entry":["mer",{"u":"GAA\u2003AAT\u2003ATC\u2003ATC\u2003TTT\u2003GGT\u2003GTT\u2003TCC\u2003TAT\u2003GAT\u2003GAA"}]},{"entry":["SEQ\u2003ID\u2003NO.",{"u":"TA"}]},{"entry":["2",{}]},{"entry":{}},{"entry":["Target\u200340-","C\u2003ACC\u2003ATT\u2003AAA\u2003"]},{"entry":["mer",{"u":"TAT\u2003GAT"}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["3",{}]},{"entry":{}},{"entry":["Target\u200325-","GAA\u2003AAT\u2003ATC\u2003ATC\u2003TTT\u2003GGT\u2003GTT\u2003TCC\u2003T"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["4",{}]},{"entry":{}},{"entry":["Probe\u200315-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["5",{}]},{"entry":{}},{"entry":["Probe\u200317-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA\u2003CC"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["6",{}]},{"entry":{}},{"entry":["Probe\u200319-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA\u2003CCA\u2003C"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["7",{}]},{"entry":{}},{"entry":["Probe\u200321-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA\u2003CCA\u2003CAA"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["8",{}]},{"entry":{}},{"entry":["Probe\u200325-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA\u2003CCA\u2003CAA\u2003AGG\u2003A"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["9",{}]},{"entry":{}},{"entry":["Probe\u200330-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA\u2003CCA\u2003CAA\u2003AGG\u2003ATA\u2003CTA"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":["10",{}]},{"entry":{}},{"entry":["Probe\u200335-","CTT\u2003TTA\u2003TAG\u2003TAG\u2003AAA\u2003CCA\u2003CAA\u2003AGG\u2003ATA\u2003CTA\u2003CTT\u2003AT"]},{"entry":["mer",{}]},{"entry":["SEQ\u2003ID\u2003NO.",{}]},{"entry":"11"},{"entry":{"@attributes":{"namest":"1","nameend":"2","align":"center","rowsep":"1"}}}]}}}},"br":[{},{},{}],"sub":["0","min","min","0","min","min ","min","min"],"sup":["5","10","5","3","\u221212 "],"in-line-formulae":[{},{}],"i":["T]","=[PT]","\/K[P]","=[PT]","\/M\u00d7"]},{"@attributes":{"id":"p-0074","num":"0089"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE I-2"},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}},{"entry":["Probe Length","Melting Temperature, \u00b0 C.","Affinity Constant (\/M)"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"1","colwidth":"56pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"84pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"77pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["15","48.4","5.382 \u00d7 10"]},{"entry":["17","56.1","3.536 \u00d7 10"]},{"entry":["19","61.3","1.129 \u00d7 10"]},{"entry":["21","64.9","1.712 \u00d7 10"]},{"entry":["25","71.1","1.116 \u00d7 10"]},{"entry":["30","74.0","2.717 \u00d7 10"]},{"entry":["35","76.2","7.823 \u00d7 10"]},{"entry":{"@attributes":{"namest":"1","nameend":"3","align":"center","rowsep":"1"}}}]}}]}},"br":{}},"As described below, the size and configuration of the target as well as the size, configuration and arrangement of substrate-anchored probes have a substantial effect on probe-target interaction which leads to substantial deviations of actual probe-target affinities from those predicted by the NN model.","The adverse role of steric effects (\u201chindrance\u201d) in the capture of target analytes to immobilized probes, and specifically the importance of probe accessibility, have been known in the art; see e.g., Guisan, J. M. in \u201cImmobilization of Enzymes and Cells,\u201d Gordon F. Bickerstaff, Humana Press, Totowa, N.J., pp. 261-275 (1997). Thus, empirical strategies of enhancing capture efficiency by introducing spacers of preferred length in order to alleviate constraints related to probe \u201cpacking\u201d have been described; see e.g., Southern E. et al., Nat. Genet. (suppl.) 21, 5-9 (1999). However, in contrast to the known methods, the methods disclosed herein establish the fundamental interconnection between certain properties of target and probe layer as the foundation of a systematic design process guiding the optimization of probe-target interaction. Probe layer compressibility is identified as a property to be maximized in order to facilitate penetration of the target, or portions of the target, into the layer in the course of duplex formation. More generally, the design criteria herein reflect the nature and magnitude of effects of length, grafting density and electrostatic charge of substrate-anchored probes, length and configuration of target, and selection of the location of the capture subsequence relative to the target's 5\u2032 terminus on capture efficiency and hence assay signal. Conversely, to permit the correct determination of target abundances, methods are disclosed to determine the re-normalized constants governing probe-target interaction.","Disclosed are methods, designs and design rules relating to the selection of sizes, configurations and arrangements of anchored capture probes, sizes and configurations of target including the selection of capture subsequences and the selection of array compositions and protocols, in order to modulate probe-target capture efficiencies and to optimize assay sensitivity, specificity and dynamic range.","In order to establish design criteria, the nature and magnitude of effects of length, grafting density and charge of substrate-anchored probes as well as size and configuration of target, or designated subsequences of target, on capture efficiency and hence assay signal, are disclosed. Relevant experiments were performed in accordance with the Random Encoded Array Detection (READ\u2122) format of multianalyte molecular analysis in which probes are displayed on color-coded polymer microparticles (\u201cbeads\u201d), and beads are arranged in a planar array on a silicon chip. See U.S. application Ser. No. 10\/204,799, filed Aug. 23, 2002, entitled: \u201cMultianalyte molecular analysis using application-specific random particle arrays,\u201d hereby incorporated by reference. Probes preferably are \u201cend-grafted\u201d to beads by way of a covalent linkage at the 5\u2032 terminus. The analysis of experiments performed on synthetic model DNA targets as well as model cDNAs generated by reverse transcription from a 1,200 nt Kanamycin mRNA (Promega), establishes a critical role of target and probe configurations in the interaction of targets with an immobilized set of probes, even when the target strands of interest are of such relatively modest size.","I.2.1 Synthetic Model Targets\u2014","Binding Isotherms were Recorded Over a Wide Range of concentration of labeled synthetic DNA targets varying from 25 nt to 175 nt in length, and over a range of capture probe lengths varying from 15 nt to 35 nt (see Table I-1 and Example I).","Target Length Dependence\u2014","To investigate the dependence of probe-target capture efficiency on the length of the target strand, four fluorescently end-labeled synthetic DNA targets, 25 nt, 40 nt, 90 nt and 175 nt in length (see Table I-1), all containing a common subsequence, were permitted to hybridize to a 19 nt capture probe displayed on color-coded beads of 3.2 \u03bcm diameter and arranged in a planar array in accordance with the READ format. Representative binding curves, reveal a significant dependence on target length, L. As illustrated in , the longer the target, the lower the signal intensity attained at any given target concentration below saturation; here, the intensity is normalized, for each curve, to that attained at saturation.","Estimates of the experimental affinity constants, K*, and the number densities of available capture probes, [P]=p, were obtained by fitting each profile to the law of mass action; results are summarized in . To compute affinities, the signal intensity, I, is herein taken to be proportional to the product of the number of captured targets per bead, c, and the number of fluorophores per target, n, that is, I\u02dcnc; interconversion between I and c is facilitated by reference to a calibration curve, described in Example II in conjunction with Table I-3 and . Typical observed affinity constants are of the order of K*=10\/M where target length is about equal to probe length, an order of magnitude lower than those predicted by the NN model (Table I-2). Typical values of p, the number of occupied sites at saturation, are of the order of 10per bead.","Under typical experimental conditions of interest in the context of gene expression analysis, the size of the target will exceed that of the probe, and each captured target will thus occlude more than a single probe; accordingly, saturation will reflect the capture of a limiting number, N, of targets to a bead of finite area, A. A lower limit of Nis obtained by assuming that the bead surface is decorated with captured targets assuming a \u201crelaxed\u201d configuration in which a target's characteristic size is set by its radius of gyration, R\u02dca L, \u03bd denoting a characteristic exponent with numerical value \u03bd=\u00bd for an ideal chain and \u03bd=\u2157 for a self-excluding chain in a good solvent in 3 dimensions (deGennes, \u201cScaling Concepts in Polymer Physics\u201d, Cornell University Press, 1979). Accordingly, for the smallest target, N\u02dcA\/R, or N\u02dc1\/L. Identifying pwith the number, N, of targets captured per bead at saturation yields, for example for the smallest target (L=25 nt), an average molecular area of A\u02dc4\u03c0 (1.6 \u03bcm)\/8*10\u02dc4*10\u212b, a value comparable to that obtained for A\u02dc\u03c0R\u02dc6.5*10\u212bwhen using an (experimental) estimate of R\u22439 L\u224345 \u212b (Tinland et al, Macromolecules 30, 5763 (1997)). For the 175 nt target, comparison of the corresponding two values yields A\u22431.6*10\u212b<A\u22434.5*10\u212b. These comparisons suggest that, at saturation, either the larger target molecules are not in their relaxed, but in a more compact configuration, or that they are no longer isolated but are substantially \u201coverlapping,\u201d that is, interpenetrating.","When plotted at a fixed target concentration as a function of target length, L, the signal intensity displays a 1\/Ldependence (), with 3\/2\u2266x\u22662, as target length is varied from L=25 nt to L=175 nt, and target concentration, at each length, is varied over three orders of magnitude from 0.1 nM to 100 nM. Notwithstanding the fact that all targets hybridize to the 19 nt probe via the same 19 nt subsequence (Table I-1), implying identical \u201ccondensation\u201d energies of duplex formation, the increase in target length is seen to result in a substantial reduction in signal intensity. Thus, for given length of capture probe, the longer the target, the less favorable the formation of the duplex and the lower the effective affinity.","The power-law dependence of the effective affinity governing probe-target hybridization provides a means of tuning the capture efficiency in accordance with the length of specific target strands. This is a particularly useful design criterion in applications such as expression monitoring permitting the control of cDNA lengths by placement of sequence-specific reverse transcription (RT) primers. As discussed herein in greater detail, rare messages preferably are converted to short cDNAs to maximize capture efficiency.","Probe Length Dependence\u2014","A complete set of binding curves such as those shown for the 19 nt probe in  was generated using a set of capture probes varying in length from 15 nt to 35 nt. The binding curves for the 175 nt target are shown in  along with fits to the law of mass action, assuming, as stated above, I\u02dcnc, nrepresenting the (average) number of fluorescent labels per molecule. For this set, fits yield values of the affinity constant of the order of K*\u22435*10\/M, approximately a factor of 20 lower than those predicted by the NN model (see Table I-2). The dependence of signal intensity, at a fixed concentration of targets of length 25 nt, 90 nt and 175 nt, is shown as a function of increasing probe length in . The intensity profiles for short probe lengths display the expected increase, although smaller than that predicted by the NN model; however, for all four target lengths, the profiles peak or level off at a probe length of approximately 30 nt. This is entirely unexpected from the point of view of the NN model. Instead, as discussed herein below, these results suggest that the capture of target to immobilized probes requires elastic deformation of not only the incoming target strands but also of the layer of capture probes.","I.2.2 Kanamycin mRNA: Selection of Transcript Length and Placement of Capture Sequence","It is further shown that, as with synthetic targets, the reduction in length, L, of cDNAs, herein also referred to as \u201ctranscripts,\u201d obtained by reverse transcription, produces a systematic and significant enhancement in the assay signal of the shorter transcript over that attained from the longer transcript given the same mRNA concentration. As illustrated herein for a 1,200 nt Kanamycin mRNA (Promega), cDNA products varying in length from \u02dc1,000 nt to \u02dc50 nt were produced by selecting suitable RT primers (Example III). Placement of the capture subsequence near the 5\u2032 end of the cDNA is shown to produce an additional enhancement. Accordingly, capture probes preferably were designed to match subsequences located in close proximity to the transcript's 5\u2032 end (see ). Both enhancements reflect the importance of configurational contributions to the free energy governing the interaction of targets with anchored probes. As a result of these effects, the sequence-dependent affinity, K, is reduced to an effective affinity, K*(L)<K, with significant implications for the design of anchored capture probes as well as transcripts, particularly when the fraction of available substrate surface covered by adsorbed target exceeds a characteristic value, \u03b3*=c*\/c.","Multiple Primer Multiple Probe (mpmp)-RT Protocol\u2014","In some cases, multiple reverse transcription (RT) primers were employed () so as to allow for the possibility of producing multiple cDNA transcripts from a single mRNA template by way of displacing a shorter cDNA incorporating a first RT primer placed in close proximity to the mRNA's 3\u2032 end, by a longer cDNA transcript incorporating a second RT primer placed farther from the mRNA's 3\u2032 end. For each cDNA, one or more capture probes\u2014here of length 19 nt\u2014were provided (Example IV). An embodiment for multiplexed expression monitoring invokes the READ format, for example in the version illustrated in .","I.2.2A Effect of Reduction in Transcript Length\u2014","Guided by the results of titrations on model compounds, as described in Sect. I.2.1, it was established that a reduction in transcript length does indeed yield a substantial improvement in assay signal.","A series of RT reactions, performed on Kanamycin mRNA over a range of initial concentrations in accordance with an mpmp-RT design and assay protocol (Example IV), produced the titration curves shown in . At each mRNA concentration, ranging from 36 nM to 560 pM, the signal recorded for the 150 nt transcript exceeds that recorded for the 1,000 nt transcript, notwithstanding the fact that the number, n, of fluorophores for the 1000 nt transcript exceeds that for the 150 nt transcript.","For example, I\/I\u02dc3, at the target concentration corresponding to 1.13 nM. The experimental observation of an enhancement of \u02dc3, for example near the cross-over concentration (see \u201cbreak points\u201d indicated in ) is in accordance with the enhancement anticipated from the reduction in transcript length, L. That is, the expected enhancement arising from the reduction in L from 1,000 nt to 150 nt would be given by \u02dc(1000\/150)( 3\/15), the first factor relating to length reduction, as discussed in Sect. I.2.1 for the model targets (with 3\/2\u2266x\u22662), and the second factor reflecting the fact that the 150-mer, at the chosen linear labeling densities, n\u02dc3 and n\u02dc15. Setting x= 3\/2, this estimate yields an enhancement of \u02dc3.5, comparable to the experimental observation.","Similarly, a reduction of transcript length from 1,000 nt to 50 nt results in an enhancement of \u02dc( 1000\/50)( 1\/15)\u02dc6, the first factor relating to length reduction (with x= 3\/2) and the second factor reflecting the fact that the 50-mer, at the chosen labeling densities, would contain, on average, only a single label.","Linearized Adsorption Isotherm Representation\u2014","Further insight is gained by representing the titration curves in the form of a linearized adsorption isotherm representation which directly follows from the law of mass action. For the reaction P (probe)+T (target)<->C (probe-target complex), mass action implies the relation c=Kpt, where c, p and t denote the respective concentrations and K denotes the affinity constant. Setting p=c\u2212p, t=c\u2212t, where pand trespectively represent initial probe and target concentrations, yields c=K(c\u2212p)(c\u2212t) and, provided that c<<t, as in the experiments reported here, c=K(p\u2212c)tor c=p\u2212c\/K t. Displaying titration results in the latter form\u2014assuming, as before, that the signal, I, is proportional to c, I\u02dcnc, ndenoting the number of fluorophores per transcript\u2014highlights the linear dependence of c on (c\/Kt) and permits the determination of p, from the intercept, and K, from the slope. Specifically, abrupt changes in slope signal a cross-over between regimes, as discussed in the text.",{"@attributes":{"id":"p-0093","num":"0108"},"figref":"FIG. 10A"},"As summarized in Table I-4, at the cross-over\u2014observed for all transcripts at a",{"@attributes":{"id":"p-0095","num":"0110"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE I-4"},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}},{"entry":[{},{},{},{},"Fractional"]},{"entry":["cDNA","K [M]","K [M]","Crossover","Coverage at"]},{"entry":["Length","(Dilute","(Concentrated","Conc.","Crossover"]},{"entry":["(nt)","regime)","regime)","[nM]","[\u03b8]"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"56pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"49pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["1000","2 \u00d7 10","1 \u00d7 10","~1","0.2"]},{"entry":["150","2 \u00d7 10","1 \u00d7 10","~1","0.2"]},{"entry":["50","5 \u00d7 10","2 \u00d7 10","~1","0.5"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}},"br":{},"sub":"0"},"The cross-over to this regime occurs at low values of coverage, \u03b8, as may be seen from the following argument. Transformation of the linearized adsorption isotherm representation to the standard form of the Langmuir isotherm, 1\/({1+1\/K t}=c\/p, displays the fraction of occupied probes, c\/p=\u03b8; as discussed below, is more precisely viewed as the ratio of the number of probes occupied at trelative to the number occupied at saturation. Specifically, extrapolating from the concentrated regime into the cross-over regime shows that, for the examples in , K t<<1 and hence 1\/K t=p\/c. Using the estimates obtained above for the effective affinity constants in the concentrated regime, the estimated fraction of occupied sites, \u03b8*=c*\/p, at the cross-over is \u02dc0.2 for the 150 nt and the 1,000 nt transcripts. That is, the larger transcripts start to interact at a fractional occupancy of available bead-displayed probes of 20%.",{"@attributes":{"id":"p-0097","num":"0112"},"figref":"FIG. 11","sup":"y"},"I.2.2B Effect of Capture Probe Placement: Terminal Capture Sequences\u2014","It is also disclosed herein that the effective affinity governing capture efficiency and hence assay signal and sensitivity is enhanced by locating capture subsequences near the 5\u2032 end of long transcripts, as illustrated in , depicting the relative alignment of RT primers as well as internal and terminal probes relative to the 1,200 nt Kanamycin mRNA.  displays the comparison of titration results obtained for the capture of a 500 nt transcript to two different (sets of) 19-mer probes, one (set) directed to a subsequence located near the 5\u2032-end of the transcript, the other directed to a subsequence located in the interior of the transcript. The use of the \u201cterminal\u201d capture probe leads to an enhancement by a factor of \u02dc1.5 in assay signal over that recorded with \u201cinternal\u201d probe. Transforming these results in accordance with the adsorption isotherm format () indicates the effect of placing the capture subsequence near the transcript's 5\u2032 terminus to have an effect on the isotherms analogous to that produced by length reduction. This is consistent with the view that capture of the terminal subsequence is equivalent to capture of a shorter target, requiring less configurational adjustment in probe layer as well as incoming target, and thereby reducing chain entropy-mediated repulsive effects, as elaborated below.","The results disclosed so far imply that the quantitative determination of message abundance requires a careful analysis of the effective affinities governing the interaction between targets and anchored probes.","I.3 Empirical Design Rules\u2014","A priori knowledge of the sequence of transcripts to be detected in \u201cdiagnostic\u201d expression profiling permits the design of capture probes directed against specific target subsequences in order to enhance sensitivity, preferably selecting terminal capture probes, modulate the dynamic range by selecting the operating regime to be above or below c*, and to optimize specificity, methods and designs for which are described in greater detail in application Ser. No. 10\/892,514.","The following empirical design rules are useful in guiding the optimization of probe-target interaction. These rules also indicate the need for corresponding corrections in the analysis of signal intensity patterns, as further discussed in Sect. II.\n\n","To account for the observations presented in Sect. I, and to provide a basis for the refinement of design rules into a systematic design process guiding the selection of optimal probe layer and target configurations, the present invention discloses a phenomenological model for the capture of single-stranded (ss) DNA or RNA targets to a layer of end-grafted probes, each such probe designed to be complementary to a designated \u201ccapture\u201d subsequence within the cognate target. Specifically, this model views the formation of a duplex between a capture probe and a designated target subsequence as an adsorption process which requires the penetration of a portion of the target into the probe layer. This involves an elastic deformation of the layer as well as the confinement of (a portion of) the target which will be accompanied by a loss of configurational entropy. The formation of anchored probe-target complexes is thus viewed herein as a grafting process which mediates the transformation of the end-grafted probe \u201cmonolayer\u201d into a probe-target \u201cbilayer\u201d.","Polyelectrolyte Brush\u2014","In one way, the model presented herein is thus informed by the process of polyelectrolyte adsorption to a deformable substrate, this substrate displaying the characteristics of a polyelectrolyte \u201cbrush\u201d, or, under certain conditions, that of a polymer \u201cbrush,\u201d composed of end-grafted probes (; Pincus, Macromolecules 24, 2912-2919 (1991)\u2014incorporated by reference; see also: Fleer et al, Sect.4 in: \u201cPolymers at Interfaces\u201d, Chapman Hall, 1993). In a layer of end-grafted probes at lateral density \u03c3, the characteristic separation, d, between adjacent probes, \u03c3\u2212d, and the characteristic size, \u03be, of each probe in a relaxed or expanded (\u201cmushroom\u201d) configuration, are interrelated: as long as \u03be<<d, individual \u201cmushroom\u201d configurations are unconstrained by their neighbors; however, when probe chains start to overlap, \u201cmushroom\u201d configurations become constrained, and probes will adopt increasingly \u201cstretched\u201d configurations, thereby transforming the probe layer into a \u201cbrush\u201d in which chain ends tend to be displaced toward the free surface (Fleer et al, op. cit.; Milner, Witten & Cates, Macromolecules 21, 2610-2619 (1988)).","As described herein, the high charge density realized within a layer of anchored oligonucleotide probes permits operation under a variety of external conditions, with the possibility of realizing a variety of probe layer configurations. These are determined primarily by the probe grafting density, \u03c3, and by the effective linear charge density, f, 0<f<1, reflecting the degree of dissociation, \u03b1, of probes within the layer in response to solution conditions, especially pH, temperature and salt concentration, C.","For example, denoting by k the dissociation constant for the solution reaction AH\u21c4A+H, \u03b1:=[A]\/[AH] is given in terms of k and [H] in the form \u03b1=1\/{1+[H]\/k}; generally [H]>[H]and \u03b1<\u03b1, and f=f(\u03b1) or, more precisely, f=f(k, C). When the salt concentration, Cin the bulk solution is low, counterions are retained in order to maintain electroneutrality in the interior of the brush at the expense of a loss of entropy of mixing. Under the action of the corresponding osmotic pressure, chains are expected to be fully elongated, regardless of grafting density. Conversely, at sufficiently high bulk salt concentration, excess mobile co-ions and counterions can penetrate into the brush and screen electrostatic interactions within the brush; as the osmotic pressure associated with the trapped counterions is diminished, the appearance of relaxed chain configurations\u2014and a corresponding reduction in layer thickness\u2014are expected. Under the high salt concentrations, in the range of \u02dc100 mM to \u02dc2M, frequently realized in conventional hybridization experiments, a collapsed state can result in which counterions are no longer distributed throughout the layer but are associated with anchored probe chains (or probe-target duplexes).","Interfacial Film of Short Amphiphiles\u2014","In another way, the model herein is informed by the process of adsorption of solutes, say proteins, to monomolecular (\u201cLangmuir\u201d) films composed of amphiphilic molecules such as phospholipids, surfactants or certain peptides adsorbed at an air-water or oil-water interface. Insertion of solutes into such a film requires local film compression, mediated by changes in chain packing and configuration, in a manner analogous to that produced by lateral compression. As a function of grafting density, the interplay of orientational and configurational degrees of freedom can produce a variety of phases; for present purposes, phases, or coexistence regions of high lateral compressibility are of principal interest. While the following discussion employs the language of polymer theory, it is understood that any extensions or refinements likely possible for layers of short probe chains by reference to the known phase behavior of interface-adsorbed amphiphilic (\u201cLangmuir\u201d) films also are included herein.","The phenomenological model is to elucidate the critical role played by elastic effects arising from distortions in target and probe layer configurations required for duplex formation between targets and probes, particularly when either targets or probes are immobilized. Further, it is to provide a basis for the refinement of the empirical design rules delineating optimal \u201coperating regimes\u201d for target capture to immobilized probe layers and for the completion of assay protocols. For example, such protocols may call for target-mediated, polymerase-catalyzed probe elongation, as illustrated below in connection with a method of signal amplification which will require penetration into the probe layer of additional assay constituents including enzymes.","II.1.1 Probe Layer Deformation and Target Confinement: Renormalization of Affinity Constant","A (portion of a) target penetrating into a layer of end-grafted probes will increase the local segment concentration and will generate a corresponding osmotic pressure; in addition, the incoming target also will induce an elastic deformation of the layer which is mediated by chain elongation (\u201cstretching\u201d), as illustrated in . The osmotic pressure and elastic energy of chain elongation act to repel the incoming target, and thus provide a repulsive contribution, G, to the free energy of duplex formation. It is this repulsive free energy which contributes to the entropic stabilization of colloidal suspensions; however, while in that instance, optimal grafting layer configurations are those which minimize interpenetration of chains on colloidal particles coming into contact, the present objective in optimizing capture probe layer configurations is to facilitate target strand penetration into the layer.","At very low grafting density, for example, in the limit d\u02dc\u03c3>>R, isolated probes assume a relaxed (\u201cmushroom\u201d) configuration of size R\u02dcaP, \u03bd=\u2157, and target capture will proceed in the absence of the constraints imposed by local chain \u201cpacking\u201d; however, the maximal number of targets captured will be small and the corresponding assay signal low. Conversely, at high grafting density, for example such that d\u02dc\u03c3\u2266\u03be<<R, particularly under conditions producing full chain elongation, the number of available capture probes will be high, but the lateral compressibility of the layer will be low and target capture will be inefficient and the assay signal low; here, \u03bedenotes a characteristic target \u201cblob\u201d size in a partially elongated target. Accordingly, to optimize target capture to a layer of immobilized probes, the grafting density is optimized so as to provide the highest possible number of probes per unit area without substantially reducing compressibility. For example, given an actual target of which a portion of size T is to participate in duplex formation, the optimal grafting density can be found by providing a synthetic target of size T and determining\u2014under fixed external conditions\u2014the assay signal reflecting fraction of captured target as a function of increasing grafting density until a plateau or peak in the resulting profile is obtained. \u201cIndirect\u201d probe anchoring, for example to a flexible \u201cbackbone\u201d which is in turn attached to the solid phase, also can alleviate constraints. See U.S. application Ser. No. 10\/947,095, filed Sep. 22, 2004, entitled: \u201cSurface Immobilized Polyelectrolyte with Multiple Functional Groups Capable of Covalent Bonding to Biomolecules,\u201d incorporated by reference.","Targets, or portions of targets, in order to make contact with the capture sequence, must adjust to the local configuration of the probe layer or the already formed composite probe-target layer (see , ). The resulting confinement of target strands and corresponding loss of configurational entropy\u2014even in the dilute regime\u2014represents a repulsive contribution, G, to the free energy of duplex formation. The degree of confinement imposed on ssDNA or RNA, will depend on the specific unconstrained (\u201crelaxed\u201d) configuration assumed by these polyelectrolytes under conditions prevailing in solution\u2014even without the considering the possibility of sequence-specific interactions (\u201cfolding\u201d), a complex phase. behavior is expected (see e.g., Schiessel & Pincus, Macromolecules 31, 7953-7959 (1998)). For purposes of illustration: penetration of a portion of target of length T and, assuming a Gaussian coil configuration, of size R\u02dcaT, \u03bd=\u2157, into a probe layer of local grafting density, \u03c3, will require an elastic energy of target deformation G\u02dc(R\/\u03c3)\u02dcaT\/\u03c3. That is, the larger the portion of target penetrating into the layer relative to the characteristic distance between adjacent probes, d\u02dc\u03c3, the more difficult the requisite deformation of the target.","The sequence-dependent \u201ccondensation\u201d energy, G, which favors the formation of probe-target pairs must be balanced against these repulsive contributions to the free energy, G=G+G; accordingly, the free energy governing probe-target complex formation has the form G\u02dcG\u2212G. An immediate consequence of this form of the free energy is a \u201crenormalization\u201d of the sequence-dependent affinity constant, K, to an effective affinity constant, K*<K. As long as G<G, condensation will still occur, but with a smaller net gain in free energy, \u2212\u0394G*=\u2212\u0394G+G, >\u2212\u0394G, and a correspondingly smaller effective condensation energy implies a smaller effective affinity constant,\n\n*\u02dcexp(\u2212\u0394)<\u02dcexp(\u2212);\n\nas well as a lower \u201cmelting temperature\u201d, T*<T, wherein T*is determined from the condition \u0394G(T*)=\u0394G*(T*)=0 and Tis determined from the condition \u0394G(T)=0. Substantial corrections to the sequence-specific values must be anticipated, in fact, elastic effects can suppress duplex formation altogether.\n","One method of assessing effective affinity constants is the empirical method, described herein, of performing isotherm measurements using probe payers of defined configuration and synthetic targets comprised of one target containing only the subsequence of interest of length T, and additional targets containing the subsequence of length T embedded in a total sequence of length L>T. Ignoring excluded volume effects, the probe layer configuration is determined, for given probe length, P, by grafting density, \u03c3, and effective linear charge density, f, 0<f<1, the latter in turn reflecting experimental conditions, especially salt, pH and temperature, realized in bulk solution. From these isotherm measurements, values for the effective affinity constant in various regimes of target concentration are readily extracted.","Another method of assessing effective affinity constants, complementary to the empirical method, is that of invoking a phenomenological model of probe-target capture to account for the effects of elastic and electrostatic interactions.","II.1.2 Design Considerations","Probe Layer Configuration: Preferred Grating Density\u2014","For given grafting density, \u03c3, overlap between adjacent chains in a \u201cmushroom\u201d configuration begins to occur when the transverse displacement of probe chains, s, is comparable to d, that is, s\u02dcaP\u2245d, P denoting probe length and a denoting a monomer or segment size. With \u03bd=\u00bd, the condition becomes aP\u02dcd\u02dc1\/\u03c3 and hence P\u02dc1\/\u03c3a. Given a preferred length, P, for the capture probe of interest, the grafting density therefore preferably is adjusted such that \u03c3<1\/aP.","Considering target penetration to increase segment density in a manner equivalent to that of an increase in probe grafting density, suggests a modification of this rule. Given a preferred length, P, for the capture probe of interest, and anticipating penetration of a portion of target occupying at least the same footprint as the probe, select a preferred grafting density such that \u03c3=g\u03c3<g\/aP, \u00bd<g<1; for example, with g=\u00bd, that is, T=P (a situation realized to good approximation in the case of terminal capture, ), select \u03c3<\u00bdaP in order to accommodate the anticipated insertion of target.","Free Energy of Probe Layer: Osmotic Pressure and Elastic Deformation\u2014","The penetration of a target strand, or a portion thereof, into a brush of end-grafted probes leads to an increase in local segment density, \u03c6. For a brush of area Aand thickness D=D(\u03c3) containing nchains, \u03c6\u02dcS\/AD(\u03c3)\u02dc(n\/A)P\/D(\u03c3), P representing the number of segments per chain; hence, \u03c6\u02dc\u03c3P\/D(\u03c3). An increase in \u03c6 leads to an increase in the osmotic pressure, II\u02dc\u03c6denoting a characteristic exponent, and to a decrease in the layer compressibility, \u03c7:=(1\/\u03c6)\u2202\u03c6\/\u2202II. Introduction of each additional segment also leads to elastic deformation. For example, in a brush composed of strings of \u201cblobs\u201d (), elastic deformation reduces the characteristic \u201cblob\u201d size, \u03be, with a corresponding cost in free energy arising from the requisite stretching of chain segments and the concomitant increase in brush thickness, D=D(\u03c3). Assuming each blob to contain Psegments, \u03be\u2243aP, yields P\u2243\u03be\/a; if each probe chain of length P contains P\/Pblobs spanning the thickness of the brush, D\u2243(P\/P)\u03be\u02dcaP \u03be and, with \u03be\u02dc\u03c3, D\u02dcaP \u03c3. That is, an increase in grafting density leads to an increase in layer thickness as a result of chain elongation. This type of scaling relation arises very generally from the balance of a repulsive contribution (e.g. excluded volume, electrostatic interactions) and the attractive contribution of chain elasticity.","Control of Grafting Density\u2014","Unless limited by the lateral density of adsorption sites provided on solid phase carrier surfaces, the grafting density realized in the formation of the probe layer by covalent end-grafting reflects the balance between a characteristic adsorption (\u201cbinding\u201d) energy (per probe) and repulsive interactions such as the elastic deformation of the growing probe layer required to accommodate an additional probe. That is, the grafting density defines a characteristic area per chain, A\u02dcd\u02dc1\/\u03c3. In this case, grafting density reflects the conditions pertinent to the covalent functionalization of solid phase carriers, notably the concentration of probe and the conditions of incubation.","The experimental observation of a maximal capture efficiency at typical values of P\u224330 suggests a characteristic \u201cfootprint\u201d, \u03be, per chain. Using p\u22436*10() as an estimate of the maximal number of targets (of size L=25 nt) accommodated per bead (of 3.2 \u03bcm diameter), and assuming each of these targets to be hybridized to one probe equal in size to the captured target, the average molecular area is estimated to be A\u02dc\u03c0(1.6 \u03bcm)\/2*6*10\u02dc0.65*10\u212bfollowing target capture, or twice that value prior to target capture, the latter corresponding to a probe grafting density \u03c3=1\/A\u22437.5*10\/cm. This suggests a picture of a \u201cself-limiting\u201d grafting process producing\u2014at least under conditions applied in the production of solid phase carriers used in the experiments cited here\u2014a layer in which end-grafted probes are no longer in their relaxed configuration but assume a partially elongated configuration; partial elongation would be consistent with a configuration in the form of an elongated string of \u201cblobs\u201d of characteristic radius \u03be\u02dc(1.25*10A\/\u03c0)\u02dc20 \u212b<R\u22439 L\u224350 \u212b (Tinland et al, op. cit.), Rdenoting the radius of gyration of an unconstrained probe chain in solution. That is, in a brush produced by a \u201cself-limiting\u201d grafting process, \u03c3\u2243\u03be.","As discussed herein, high grafting densities, particularly those realized in typical conditions of in-situ synthesis of oligonucleotide probes (Lipshutz, R. J. et al., Nat. Genet. (suppl.), 21, 20-24 (1999); Shchepinov, M. S. et al., Nucleic Acids Research 25, 1155-1161 (1997)) generally may be unfavorable. Spotting of probes generally will not produce end-grafted layers but rather more complex \u201ccrumpled\u201d layers (Netz & Joanny, Macromolecules 32, 9013-9025 (1999)) in which molecules may be attached to the solid phase at multiple (random) sites, leaving only a small portion of probe sequences\u2014unknown a priori and highly variable from spot to spot\u2014accessible to the target. Control of grafting densities may be difficult to achieve in this situation.","Preset values of a lower than that attained in the \u201cself-limiting\u201d case are realized, for example, by introducing an intermediate step into the process of microparticle functionalization. Specifically, introduction of a bifunctional modifier in the form of a functionalized polymer such as bifunctional polyethyleneglycol (\u201cPEG\u201d) molecules of adjustable molecular weight, biotin-binding proteins like NeutrAvidin, Streptavidin or Avidin, and any other heterofunctional polymeric linkers of known molecular size sets an upper limit on the probe grafting density, which is now determined by the size of the modifier and its lateral \u201cpacking\u201d at the bead surface (). In the embodiment using the READ format, in a first step, the modifier is covalently attached to a color-encoded microparticle (\u201cbead\u201d), and, in a second step, the modifier is functionalized by covalent attachment of the capture probe, preferably way of a 5\u2032 modification introducing a functional group such as amine or biotin using standard conjugation chemistry.","Target Strand Confinement: Dilute and Concentrated Regimes of Adsorption\u2014","The discussion of the elastic response of the probe layer to target insertion suggests that elastic deformations of the composite probe-target layer give rise to the observed cross-over between dilute and concentrated regimes in the adsorption isotherms (), delineated by the locus c*(L) for which the limited available data suggest c*\u02dc1\/L().","In the limit of small targets, the principal effect of capture will be that of increasing the segment density within the probe layer, as discussed above, suggesting the cross-over to reflect the transition of the probe layer, or more generally, the layer formed by capture probes of characteristic size \u03be^<\u03beand already captured targets of characteristic size \u03be^<\u03be, into a regime of lower compressibility. That is, the cross-over occurs when n*\u03be^+n*\u03be^\u02dc\u03b7*A, hence \u03b7*\u02dc(n*\/A)\u03be^+(n*\/A)\u03be^\u02dcp\u03be^+c*\u03be^and c*\u02dc(\u03b7*\u2212p\u03be^)\/\u03be^. In the special case \u03be^\u2243\u03be^\u2243\u03be^, c*+p\u02dc\u03b7*\/\u03be^, or, assuming \u03be^\u02dcL, 0\u2266y\u22661, c*+p\u02dc\u03b7*\/L; in he special case n*=n*=n*, \u03b7*\u02dc(n*\/A)\u03be^or c*=(n*\/A)\u02dc\u03b7*\/\u03be^, where \u03be^represents the footprint of the probe target duplex; here, as before, 0\u2266\u03b7*\u22661. This limit may be realized either by providing a short target, not a generally available design in practice, or by placing the designated target sequence in proximity to the target's 5\u2032 end. The latter possibility is illustrated herein in connection with .","In contrast, in the limit of large targets, in exact analogy to the \u201cself-limiting\u201d grafting process of producing the grafted probe layer, the cross-over reflects the incipient overlap (\u201ccrowding\u201d) of target strands in the growing layer of captured targets of (overall) size L and characteristic \u201cfootprint\u201d \u03be; target overlap occurs when n*\u03be\u02dc\u03b7*A, 0\u2266\u03b7*\u22661, implying c*\u02dcn*\/A\u02dc\u03b7*\/\u03be\u02dc1\/L where \u03b7*Arepresents the fraction of the available area covered by captured target.","Adjusting Grafting Density to Allow for Target Penetration, Refined","The expression derived for the second case represents a design rule which may be applied to optimize the grafting density of the probe layer so as to ensure realization of the dilute regime in accordance with the boundary delineated in :\n\n","Typical values of grafting densities described herein in relation to the preferred embodiment of the invention, namely 10per bead of 3.2 \u03bcm diameter (or \u02dc3*10\/cm) correspond to high intralayer volume charge densities, zC. For example, for an oligonucleotide of length P=20, assuming a corresponding probe layer thickness D\u02dc50 A, C\u224310\/(\u03c0(3\u00b72)D)\u02dc10 mM for the concentration of probe chains, and thus yielding a corresponding value of fC=200 mM, f=20, for the local concentration of charges associated with (fully dissociated) backbone phosphate groups.","In electrochemical equilibrium, the concentrations of cations and (poly)anions present in the interior of the probe layer and in bulk solution are interrelated in accordance with the condition CC=CC. Electroneutrality requires, within the probe layer, C+fC=C, and in bulk solution, C=C=C. Accordingly, the concentration of cations within the layer, for given negative charge fC, can substantially exceed the concentration of cations in bulk solutions:\n\n\u00bd(1+{1+(4)}\n","For example, in the limit C\/fC<<1, C+\u02dcfC>>C. That is, counterions are retained within the brush even in the presence of a large gradient in ion concentration; in fact, they are distributed throughout an effective volume, V, which is smaller than the volume, V, of the brush by the finite volume occupied by the probe chains, V\u02dcV (1\u2212\u03c6).","The corresponding Debye screening length, \u03be\u02dc1\/\u03ba, associated with the backbone charge, fC, per chain, is obtained from the expression \u03ba=4\u03c01fC, 1=e\/\u03b5T denoting the Bjerrum length, and C=P\/dD. Balancing the repulsive contribution arising from the osmotic pressure \u03a0=fCT generated by counterions trapped within the brush with chain elasticity, fCT=kD\/d, with an elastic constant k=T\/aP, yields D\u2243faP, independent of grafting density, so that \u03be\u2243d(a\/4\u03c01f). This scale is set by the mean separation, d, between chains, and hence the grafting density. In the limit \u03be\u2266D, chains are elongated for any degree of charging, f>0, producing the maximal brush thickness independent of grafting density. Provided that the grafting density is sufficiently low so as to accommodate penetration of incoming target, capture to such a layer in the configuration of a \u201cbed of nails\u201d can proceed without significant elastic distortion of the probe layer. The return to partial chain elongation in accordance with the \u201cblob\u201d configuration is achieved by addition of free co- and counterions at sufficient concentration so as to ensure that the Debye screening length \u03baassociated with these free ions is comparable to \u03beso that \u03be\u03ba\u22671. For such a screened brush, the internal configuration, while qualitatively resembling that of the semidilute polymer brush composed of a string of \u201cblobs\u201d, will respond to conditions maintained in bulk solution in order to maintain electrochemical equilibrium.","Confining Duplex Formation to Interior of Charged Probe Layer\u2014","In this case, while exposed to a salt concentration of only 1 mM in solution, generally considered to preclude duplex formation (Primrose, \u201cPrinciples of Genome Analysis\u201d, Blackwell Science, 1995), the target, once it has penetrated into the probe layer, actually encounters a far higher local salt concentration and conditions of electrostatic screening that are favorable to duplex formation. That is, the probe layer provides a local chemical environment permitting probe-target hybridization under nominal conditions of extreme stringency in the bulk solution which counteract the formation of secondary structures in ssDNA or RNA and prevent reannealing of dsDNA in bulk while permitting (local) duplex formation within the probe layer. This scenario preferably is realized in accordance with the rule:\n\n","Given a sequence, or sequences, of interest, specifically a set of mRNA messages, proceed as follows, applying design rules as appropriate:",{"@attributes":{"id":"p-0131","num":"0162"},"tables":{"@attributes":{"id":"TABLE-US-00004","num":"00004"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"224pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"S","Target Subsequence of Interest"]},{"entry":[{},"L","Target Length (number of nucleotides);"]},{"entry":[{},"C","Target Abundance;"]},{"entry":[{},"ampC","Target Abundance following Amplification"]},{"entry":[{},"S","Primer Sequence"]},{"entry":[{},"S","Capture Sequence (i.e., target subsequence to be analyzed by"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"capture to probe)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u03bb","Linear Labeling Density"]},{"entry":[{},"P","Probe Length (number of nucleotides);"]},{"entry":[{},"\u03c3","Probe Grafting Density"]},{"entry":[{},"C","Salt Concentration"]},{"entry":[{},"C*","Target Concentration at Cross-over"]},{"entry":[{},"L*","L( C*);"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SelectTargetLength(C, C*, S);","\/* By placing primer, select"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"target length"]},{"entry":[{},"in accordance with given or"]},{"entry":[{},"anticipated target abundance"]},{"entry":[{},"*\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"{"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"IF(C LOW)","RETURN( L < L* );","\/* ensure operation in dilute"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"regime *\/"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"IF(C HIGH)","RETURN( L > L* );","\/* ensure operation in conc"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"regime *\/"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SelectCaptureSequence (ProbeSeq);","\/* The optimization of"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"primer and probe"]},{"entry":[{},"sequences preferably is"]},{"entry":[{},"performed concurrently (see"]},{"entry":[{},"co-pending application)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"{"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"RETURN(S= TerminalCapture Sequence( ) );"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SelectFinalTargetAbundance(L, L*, C);","\/* For given initial message"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"abundance, select target"]},{"entry":[{},"amplification conditions to"]},{"entry":[{},"establish operating regime *\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"{"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"IF( L > L*)"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"IF(C LOW)","RETURN( ampC \u2266 C*);","\/*"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"dilute"]},{"entry":[{},"regime"]},{"entry":[{},"*\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"IF(C HIGH)","RETURN( ampC > C*);","\/* conc"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"217pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"regime"]},{"entry":[{},"*\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"ELSE IF( L < L* )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"IF ( C LOW )","RETURN( ampC > C*); \/*"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"best to operate in conc"]},{"entry":[{},"regime *\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"IF( C HIGH )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"IF( C < C* )","RETURN( ampC \u2266 C*);"]},{"entry":[{},"ELSE","RETURN( ampC = C);"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"210pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SelectLabelingDensity(L, ampC);","\/* NOTE: if m'plex RT or"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"m'plex amp,"]},{"entry":[{},"\u03bb will be identical for all"]},{"entry":[{},"targets *\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"{"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\/* for long targets: operate"]},{"entry":[{},"in dilute regime, select high"]},{"entry":[{},"labeling density *\/"]},{"entry":[{},"\/* for long targets at high"]},{"entry":[{},"abundance: select low"]},{"entry":[{},"labeling density *\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"RETURN(\u03bb);"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"OptimizeTargetConfiguration(L, \u03bb, C, S, S, S )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"119pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"IF( C Fixed )","L = SelectTargetLength(C, C*, S);"]},{"entry":[{},"ELSE IF( L Fixed )","ampC ="]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"140pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"SelectFinalTargetAbundance(L, L*,"]},{"entry":[{},"C);"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u03bb = SelectLabelingDensity( );"]},{"entry":[{},"S= SelectCaptureSequence (ProbeSeq);"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"OptimizeProbeLayerConfiguration( )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"P = SelectProbeLength( );","\/* maximize Kwhile"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"minimizing cross-"]},{"entry":[{},"hybridization *\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"119pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"\u03c3 = AdjustGrafting Density(P, L);","\/* the longer the probe, the"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"154pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"105pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"lower \u03c3, allowing for"]},{"entry":[{},"insertion of target of known"]},{"entry":[{},"length *\/"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"OptimizeRepresentation( )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SelectTypeRedundancy( );"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"OptimizeReactionConditions( )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"SelectIonicStrength( );"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"main( )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"259pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"FOR( each Target in Designated Set )"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"{"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"OptimizeTargetConfiguration( );"]},{"entry":[{},"OptimizeProbeLayerConfiguration( );"]},{"entry":[{},"OptimizeRepresentation( );"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"224pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"OptimizeReactionConditions( );"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"245pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}},"br":{}},{"@attributes":{"id":"p-0132","num":"0163"},"tables":{"@attributes":{"id":"TABLE-US-00005","num":"00005"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"182pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"S","Capture Sequence (i.e., target subsequence to be analyzed by"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"capture to probe)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"21pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"P","Probe Length (number of nucleotides);"]},{"entry":[{},"C","Salt Concentration"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"EvalEffectiveFreeEnergy(S, P, C, pH);"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"\u0394G= EvalTargetElasticFreeEnergy(TargetConfig,"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ProbeLayerConfig);"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"\u0394G= EvalProbeLayerElasticFreeEnergy(TargetConfig,"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ProbeLayerConfig);"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"Return(\u0394G = \u0394G+ \u0394G\u2212 \u0394G);"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"EvalCondensationFreeEnergy(S, P, C, pH, T);"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Return( \u0394G= SumNNBasePairInteractions(S, P, C, pH,"]},{"entry":[{},"T) );"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},"main( )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"FOR( each Target in Designated Set )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u0394G= EvalCondensationFreeEnergy(S, P, C, pH, T);"]},{"entry":[{},"\u0394G = EvalEffectiveFreeEnergy(\u0394G, TargetConfig,"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"ProbeLayerConfig);"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"168pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"K = Kexp(\u2212\u0394G\/kT)"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"182pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"}"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]}}]}},"br":{}},{"@attributes":{"id":"p-0133","num":"0164"},"tables":{"@attributes":{"id":"TABLE-US-00006","num":"00006"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"175pt","align":"left"}}],"thead":{"row":{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"aI:","Array of Assay Signal Intensities"]},{"entry":[{},"aK:","Array of Affinity Constants"]},{"entry":[{},"aS:","Array of Designated Target Subsequences"]},{"entry":[{},"aC:","Array of Target Concentrations"]},{"entry":[{},"aP:","Array of Probes"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"147pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"56pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"EvalEffectiveAffinityConstant(aK, aS, aP)","\/* See II.2.2 *\/"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"{"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"FOR( j=0; j \u2266Number of Targets in Designated Set; j++)"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"56pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"161pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"\u0394G= EvalCondensationFreeEnergy(aS(j), aP(j),"]},{"entry":[{},"C, pH, T);"]},{"entry":[{},"\u0394G = EvalEffectiveFreeEnergy(\u0394G, TargetConfig,"]},{"entry":[{},"ProbeLayerConfig);"]},{"entry":[{},"aK(j) = Kexp(\u2212\u0394G\/kT)"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":{"entry":[{},"}"]}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"\/* NOTE: evaluation of effective affinities generally will have to include"},{"entry":"coaffinities *\/"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"main( )"]},{"entry":[{},"{"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"175pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"RecordAssaySignal(N, aI);"]},{"entry":[{},"EvalEffectiveAffinityConstant(aK, aS, aP, C, pH, T);"]},{"entry":[{},"CorrectAssaySignal(aI, aK);"]},{"entry":[{},"EvalTargetConcentration(aI, aC);"]}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"2"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"203pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"}"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"1","align":"center","rowsep":"1"}}]}]}}]}},"br":{}},"This section discloses several methodologies relating to optimization of sensitivity, dynamic range and assay specificity, particularly pertaining to the multiplexed analysis of abundances of highly homologous messages, and further discloses a design strategy for subtractive differential gene expression analysis using only a single detection color.","III.1 Tuning of Signal Intensities","In nucleic acid analysis, target analyte concentration can vary over a wide range. Thus, multiplexed expression monitoring generally will encounter a range of message abundance from low, corresponding to one or two mRNA copies per cell, to high, corresponding to 10copies per cell or more. The requisite dynamic range of decades for the simultaneous detection of signals from the weakest and the strongest transcripts will exceed the capabilities of many cameras and recording devices. The modulation of probe-target affinities as well as certain methods of array composition provide the means to tune the signal intensity in accordance with known or anticipated message abundance.","III.1.1 Optimization of Array Composition: Operation in Dilute Vs Concentrated Regime","The selection of RT primers for producing cDNA transcripts of desired length from an mRNA subsequence of interest, and the selection of 5\u2032-terminal target subsequences for capture, in accordance with the considerations elaborated herein, permit the modulation of probe-target affinity and thus the control of the dynamic range of assay signals indicating target capture.","Selection of Transcript Length\u2014","In the simplest case of an assay design calling only for reverse transcription, but not amplification, the concentration of cDNAs reflects the abundance of mRNAs in the original sample; that is, the target abundance is given. Then, a judicious choice of transcript length, and\/or the placement of capture subsequences, permit the maximization of detection sensitivity and the simultaneous \u201ccompression\u201d of signal dynamic range by way of tuning the effective affinity constant.","To compensate for the low abundance of transcripts representing rare messages, a short transcript length is preferably selected in order to realize the highest possible effective affinity constant and to maximize the assay signal produced by hybridization of these transcripts to anchored probes. This will ensure maximization of the detection sensitivity: Conversely, to compensate for the high abundance of transcripts representing common messages, a long transcript length is preferably selected in order to realize the lowest possible effective affinity constant and to minimize the assay signal produced by hybridization of common transcripts to anchored probes. This will ensure the (approximate) \u201cequalization\u201d of assay signals from rare and abundant messages.","Tuning of Transcript Abundance\u2014","More generally, a situation may arise in which the selection of the optimal transcript length is subject to additional constraints. For example, as herein discussed, in the case of analyzing closely homologous sequences, the subsequences near the 5\u2032 termini of many or all targets in a given sample may be identical, and identification of a specific target may require preparation of a longer than otherwise desirable cDNA. Then, for given length, L, the target abundance, t, preferably will be selected (for example by one or more rounds of differential amplification, see below) so as to ensure, for rare message, operation below c* and\/or, for abundant message, operation above c*.","Placement of Capture Subsequence\u2014","Another method of enhancing the sensitivity of detection of transcripts present in low copy number is to provide capture probes directed to a target subsequence located near the 5\u2032 end of transcripts, rather than to subsequences located in the central portion of transcripts. As discussed in Section I, the central portions of the target tend to be less accessible, and require a greater degree of probe layer distortion, than do the terminal portions of the target, with a correspondingly lower effective affinity constant in the former situation.","By any available method, the preferred design aims to realize one of the following configurations.",{"@attributes":{"id":"p-0142","num":"0173"},"tables":{"@attributes":{"id":"TABLE-US-00007","num":"00007"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"3"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"84pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"70pt","align":"left"}}],"thead":{"row":[{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]},{"entry":[{},"Short Transcript","Long Transcript"]},{"entry":[{},"(L < L*)","(L \u2267 L*)"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"2","align":"center","rowsep":"1"}}]}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"offset","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"1","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Rare Message","high K*","high K*"]},{"entry":[{},"Abundant Message","low or high K*","low K*"]},{"entry":[{},{"@attributes":{"namest":"offset","nameend":"3","align":"center","rowsep":"1"}}]}]}}]}},"br":{},"figref":"FIG. 11"},"The corresponding design procedure is summarized in Section II.2 as part of the Assay Design Optimization procedure within the functions: SelectFinalTargetAbundance (L, L*, C), SelectTargetLength (C, C*, S) and SelectCaptureSequence (ProbeSeq).","III.1.2 Control of Array Composition: Carrier Redundancy","Dynamic range and detection sensitivity can be further optimized by matching the number of probes of a given type to the anticipated concentration of the specific targets. Specifically, in the preferred READ format of the invention, the number of probes is readily adjusted by simply adjusting the number of microarticles (\u201cbeads\u201d) of particular type, a quantity also referred to herein as redundancy. A design rule for specifying the selection of optimal relative abundances of beads of different types is provided.","Ekins (U.S. Pat. No. 5,807,755) discusses a related method of designing spotted arrays of receptors to perform receptor-ligand binding assays. This method of the art requires that the concentration of receptors be significantly smaller than the concentration of ligand. As discussed below, this situation corresponds to a limiting case of the theoretical description presented below in which both [P]and the number, N, of beads are small. However, Ekins neither contemplates the regime of high receptor concentration nor the related methods for dynamic range compression disclosed herein. Furthermore, Ekins does not contemplate the use of random encoded arrays of particles for receptor-ligand interaction analysis, nor does he contemplate the variation of the relative abundances of beads\/probes of different type as a means to establish desirable assay conditions.","The reaction of interest is the complexation in solution of target molecules (which include, for example, ligands T) with receptor molecules P (which can be probes) displayed on solid phase carriers, such as color encoded beads, to form reversible complexes P\u00b7T. This reaction is governed by the law of mass action and has an affinity constant, K Thus, for the case of a single receptor binding a single ligand:",{"@attributes":{"id":"p-0147","num":"0178"},"chemistry":{"@attributes":{"id":"CHEM-US-00001","num":"00001"},"img":{"@attributes":{"id":"EMI-C00001","he":"3.73mm","wi":"21.93mm","file":"US09637777-20170502-C00001.TIF","alt":"embedded image","img-content":"chem","img-format":"tif"}}},"br":{}},{"@attributes":{"id":"p-0148","num":"0179"},"maths":{"@attributes":{"id":"MATH-US-00001","num":"00001"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"K","mo":"=","mfrac":{"mrow":[{"mo":["[","]"],"mi":"PT"},{"mrow":[{"mo":["[","]"],"mi":"P"},{"mo":["[","]"],"mi":"T"}],"mo":"\u2061"}]}}}},"br":{},"sub":["0 ","0","0 ","0"]},"At any instant, the concentration of complexed molecules on the surface is [PT](c) molecules\/bead. The number of uncomplexed receptor sites, [T](t), is given by (p\u2212c). The number of ligand molecules available for reaction at any time is the difference between the initial number of ligands and the number of molecules of ligand already complexed. In an array of Nbeads, all having receptor molecules of type P, the total number of complexes formed is equal to cN. Thus, in an analyte solution of volume V, the number of available ligand molecules is given by VNt\u2212Nc; where Ndenotes Avogadro's number. The law of mass action can be rewritten to include known variables in the form:",{"@attributes":{"id":"p-0150","num":"0181"},"maths":{"@attributes":{"id":"MATH-US-00002","num":"00002"},"math":{"@attributes":{"overflow":"scroll"},"mrow":{"mi":"K","mo":"=","mfrac":{"mi":"c","mrow":{"mrow":[{"mo":["(",")"],"mrow":{"msub":{"mi":"p","mn":"0"},"mo":"-","mi":"c"}},{"mo":["(",")"],"mrow":{"msub":{"mi":"t","mn":"0"},"mo":"-","mfrac":{"mrow":{"msub":{"mi":["N","B"]},"mo":"\u2062","mi":"c"},"msub":{"mi":["VN","A"]}}}}],"mo":"\u2062"}}}}},"br":[{},{}],"sub":["0","0","B","0","A"],"img":{"@attributes":{"id":"CUSTOM-CHARACTER-00001","he":"3.56mm","wi":"10.92mm","file":"US09637777-20170502-P00001.TIF","alt":"custom character","img-content":"character","img-format":"tif"}}},"The number of receptor sites available for reaction far exceeds the number of ligand molecules available. Under these circumstances, if a few more beads were added to an equilibrated system, some of the complexed ligand molecules would have to dissociate and redistribute themselves onto the newly-added beads to reattain equilibrium. In effect, the limiting situation is c=tVN\/N. Thus, for a given concentration of ligand molecules, the number of complexes displayed per bead, and thus the corresponding fluorescence intensity, is inversely proportional to the number of beads, c\u221d1\/N.","Introducing dimensionless variables, Y=c\/p, X=Kt, and C=KpN\/N\/V, the equation for K can be rewritten in the form Y\/(1\u2212Y)=(X\u2212CY).  shows the variation of fractional occupancy, Y, with C, which is directly proportional to the number of beads and X, the nondimensionalized ligand concentration. For lower number of beads, Y is independent of C. This situation is equivalent to situation (a) above. Nondimensionally, when X>>C, Y\u2192X\/(1+X) and is independent of C. Further, for X>>1, Y\u21921, which indicates that high ligand concentration and large values of the affinity constant ensure that the beads reach full occupancy. For larger values of C, Y decreases monotonically with C. With respect to situation (b) above, the limiting case is Y=X\/C.","Sensitivity of Detection\u2014","Control of the number of beads of a given type within a random encoded array provides a preferred means for producing signal intensities within desired limits. In the simplest case of single ligands binding to single receptors, maximum occupancy is obtained by reducing the number of beads below the knee of the curves in , given by C=1+X.","Dynamic Range Compression\u2014","As discussed earlier, in a multiplexed assay, often there is a large disparity in the concentrations of individual ligands to be detected. To accommodate within the dynamic range of a given detector the wide range of signals corresponding to this range in analyte concentration, it generally will be desirable that the number of beads of each type in a multiplexed reaction be adjusted according to the respective expected analyte concentrations. Specifically, it will be desirable that weak signals, produced by analytes present in low concentration, be enhanced so as to be detectable and that, at the same time, strong signals, produced by analytes present in high concentration, be reduced so as not to exceed the saturation limit of the detection system.","The equalization of specific signal intensities provided by dynamic range compression is particularly desirable when:","a) concentrations of ligands in an analyte solution are known (or anticipated) to vary widely.","b) binding affinitities of some ligands are known (or anticipated) to be very weak.","c) receptor density for some bead types is known (or anticipated) to be low.","For example, in a 2 ligand-2 receptor system, with ligand concentrations, t>>tit is desirable that the corresponding relative abundances of beads displaying cognate receptors be adjusted in accordance with the condition N>>N. Such reasoning is readily extended to assays involving a multianalyte solution containing a large number of ligands that is placed in contact with an array of beads containing corresponding cognate receptors.","Therefore, an array design rule for purposes of compositional optimization entails the following steps:","Select a desirable number of fluorophores or complexed molecules con beads of each type of interest.",{"@attributes":{"id":"p-0158","num":"0000"},"ul":{"@attributes":{"id":"ul0018","list-style":"none"},"li":["1. Set Yfor each receptor-ligand pair on the basis of known or anticipated values of p.","2. Calculate Xas a product of analyte concentrations and affinity constants.","3. Calculate C=X\/Y\u22121\/(1\u2212Y) for each receptor-ligand pair.","4. Calculate the desired number of beads of each type from N=CVN\/pK.\n\nAn Experimental Demonstration\u2014\n"]}},"As described herein, the effective affinity constants can display a substantial length-dependent variation: for example, in the case of Kanamycin, K(L=50 nt)\/K(L=1000 nt)\u02dc10 in the concentrated regime. An example of the dramatic effect of the combination of transcript length selection and bead redundancy on assay signal intensity is illustrated in , produced in accordance with the protocols of Example V but using \u02dc3,000 beads for detection of the Kanamycin cDNA, present at 10,000 femtomoles in a reaction volume of 20 ul, and using \u02dc100 beads for detection of the IL-8 cDNA, present at 2 femtomoles in a reaction volume of 20 ul.","As depicted in , notwithstanding the fact that, in the fifth and seventh pairs of ratios shown in that figure (counting from the left), the 50 nt and the 1,000 nt Kanamycin transcripts are present at an identical abundance of 1,000 femtomole, the respective signal intensities recorded are seen to differ by more than an order of magnitude. Further, as depicted in , the Kanamycin cDNA, present at approximately 5,000-fold excess over the IL-8 cDNA, produces only an approximately 20-fold higher signal intensity, directly demonstrating dynamic range compression.","Without correction for the substantially differing effective affinity constants of the two transcripts, the analysis of the experimental data would lead to a substantial error in message abundance.","Entanglement\u2014","This particular example illustrates a further effect on signal intensity of captured target which arises from entanglement of target strands in solution. That is, target strands in solution begin to overlap at a certain threshold, t*, in target concentration. For a target containing L nucleotides and assuming a Gaussian coil configuration, the corresponding target concentration is simply t*\u2243L\/R\u02dcaL, or, with \u03bd=\u2157, t*\u02dcL, implying, for the target volume fraction, \u03a6*\u02dcL. For targets of appreciable length, \u03a6* can be quite small: \u03a6*(L=1,000)=0.004. In the example, with a\u22435 A, L=1,000, yields a radius of gyration, R\u22439L\u22439*33 A \u2243300 A and a molecular volume, V=(4\/3)\u03c0R\u2243300*10A; with 10fmoles=10molecules, the volume occupied by target is V\u22430.3 \u03bcl and hence \u03a6=0.3\/20\u22430.015>\u03a6*. That is, in the example, the capture efficiency of the 1,000 nt Kanamycin transcript would be expected to be further diminished by target entanglement.","As necessary, an additional measure would be to perform multiple concurrent multiple probe, multiple primer-RT reactions to permit different degrees of initial mRNA dilution. Products would be pooled to perform detection in a single multiplexed reaction.","III.1.3 Differential Amplification\u2014","Because it is governed by an affinity constant that approaches the sequence-dependent affinity constant, K, the dilute regime of operation generally will be the preferred regime of operation for detection of low-abundance messages. This is so particularly when the design of short cDNAs is difficult or impossible, as discussed herein in connection with the analysis of sets of closely homologous sequences. RT-PCR protocols may devised which limit PCR cycles to a small number, say 3-4, in order to bring the concentration of the lowest-abundance transcripts to the detectable range corresponding to the dilute regime.","Given the reduction in affinity constants in the concentrated regime, transcript amplification to concentrations exceeding the cross-over concentration will yield diminishing returns. That is, for a target of any given length, target amplification may produce a relatively smaller increase in signal in accordance with the length-dependent effective affinities governing transcript capture, particularly in the concentrated regime. Specifically, if high abundance transcripts are amplified into the regime of saturation, additional amplification will not translate into any additional gain in capture and hence detected signal. Unless taken into account in the assay design and the analysis of assay signals, this \u201csaturation\u201d effect can seriously distort the quantitative determination of target concentration.","However, if properly taken into account on the basis of the methods of the present invention, this scenario therefore lends itself to dynamic range compression by differential amplification in which the signal of low abundance messages is enhanced relative to that of high abundance messages undergoing the same number of amplification cycles and in the same multiplexed target amplification reaction.","Pools\u2014","More generally, it may be desirable to equalize the concentrations of transcripts from high and low abundance messages\u2014regardless of target length\u2014within a preset narrow range of concentration. In this instance, it will be useful to split targets into two or more sets undergoing separate multiplexed target amplification reactions in order to be able to subject high abundance messages to a small number of amplification cycles while and to subject low abundance messages to a higher number of amplification cycles.","III.1.4 Labeling Density\u2014","Operation in the dilute regime requires detection of a small number of captured transcripts, and this is facilitated by a high rate of incorporation of labeled dNTPs. In Examples described herein, a typical labeling density of 1:64 is achieved by a molar ratio of one labeled dCTP per eight unlabeled dCTPs. For a 150 nt transcript, this ratio implies n\u02dc3, and correspondingly lower numbers for the shorter transcripts present in the mixture. In addition, more label can be added per unit length by adding more than one type of labeled dNTP during reverse transcription. For example, one can use biotin-dATP and biotin-dCTP both in a particular reaction mixture, which generates more label per unit length than either one alone. In an experiment (not shown) labeled biotin-dATP at a ratio of 1:6.25 relative to unlabeled dATP was added as a reagent in a reverse transcription reaction. Comparing to end-labeled cDNA controls, there were about 20 labeled nucleotides present on a 1,000 nucleotide (\u201cnt\u201d) Kanamycin cDNA.","More generally, differential labeling also provides a further method of equalizing the signal intensities produced by capture of transcripts differing in concentration. Preferably, this is accomplished by adjusting the number of labels incorporated into sets of transcripts in accordance with the respective known or anticipated levels of abundance as well as length. Preferably, a higher density of labeled dNTPs will be ensured in transcripts exceeding the length limit associated with the cross-over into the concentrated regime. In this instance, a higher labeling density will increase detection sensitivity by compensating for the lower effective affinities of such longer transcripts of which fewer will be captured to anchored probes as discussed herein. The calculation must of course take into account the fact that the average total number of labels per target is proportional to target length.","To accomplish differential labeling of transcripts, RT reactions can be carried out by separating the mRNA sample into two or more aliquots in different tubes (reaction chambers) such that, for example, in one reaction, only short transcripts are generated and in another, only long transcripts are generated and adjusting in each RT reaction the ratio of the labeled dNTPs to unlabeled dNTPs i.e., the higher the ratio, the more label included in the transcript.","III.2 Elongation-Mediated Sequence Specific Signal Amplification\u2014","Sensitivity and Specificity\u2014","Results obtained to date using these assay designs to produce short, labeled cDNAs demonstrate sensitivity sufficient to detect\u2014without recourse to mRNA or cDNA amplification but taking advantage of a novel signal amplification method\u2014labeled Kanamycin cDNA fragments, 50 nt-70 nt in length, at the level of one femtomole of material in a total reaction volume of 10 \u03bcl ().","As set forth in Example VI and , \u201cspiking\u201d experiments can be performed to further evaluate the level of specificity attainable in detecting a specific mRNA in the complex environment typical of a clinical human sample.","Novel Signal Amplification Method\u2014","To attain higher sensitivity, a method of (post-assay) signal amplification is disclosed which invokes sequence-specific probe elongation and subsequent decoration with a fluorescent probe to produce an enhancement in signal by an order of magnitude subsequent to cDNA capture. This elongation-mediated process () takes only a few minutes and can be employed selectively, for example for low abundance messages, in conjunction RT labeling of cDNAs or exclusively, for all messages.","In elongation, the 5\u2032 end of the transcript hybridized to the probe is elongated only if there is a perfect match to the probe in this region. See U.S. application Ser. No. 10\/271,602, filed Oct. 15, 2002, entitled; \u201cMultiplexed Analysis of Polymorphic Loci by Concurrent Interrogation and Enzyme-Mediated Detection,\u201d incorporated by reference.","First, Kanamycin mRNA (here, in a range of concentrations from 1 to 32 fmoles per 20 \u03bcl) is labeled, for example by incorporating Cy3-labeled dCTPs into the cDNA during the RT reaction. The labeled cDNA is captured to immobilized capture probes as described in connection with Examples III, IV and V and . To enhance the signal produced by the captured target, a probe elongation reaction is performed in-situ (\u201con chip\u201d) using biotinylated dCTPs (\u201cBio-14-dCTP\u201d). The resulting biotinylated elongation product is then \u201cdecorated\u201d by exposure to a Streptavidin-Phycoerythrin conjugate, producing substantially enhanced fluorescence from the Phycoerythrin tags (see Example II).","In fact, as shown in , the reaction is quantitative, producing a 10-fold enhancement over a wide range of concentrations, and thus permitting quantitative determination of message abundance at increased sensitivity, readily-permitting the resolution of two-fold changes in intensity over the entire dynamic range in signal of \u02dc3 decades.","Under assay protocols described herein in various Examples, and using an embodiment in accordance with the READ format, the signal produced by capture of 50 nt-70 nt transcripts was readily detected without target amplification (but with signal amplification, as described herein)\u2014at a level of signal to (uncorrected) background of 2:1\u2014at a cDNA concentration of approximately 0.1 fmole per 10 \u03bcl of sample. This is sufficient for the detection of mRNA present at a frequency of 10-30 copies per cell, assuming the collection of mRNA from 10Peripheral Blood Mononucleocytes per ml, as assumed in standard protocols (Lockhart, D. J., Dong, H., Byrne, M. C., Follettie, M. T., Gallo, M. V., Chee, M. S., et al., Nature Biotechnology 14: 1675-1680 (1996)).","III.3 Optimizing Specificity of Detection","The interaction of multiple transcripts with a set of immobilized sequence-specific detection probes is governed by a multiplicity of competing reaction equilibria and a corresponding set of co-affinities. These measure the strength of the interaction between a given probe in the set with all available target subsequences, and between any target subsequence and the set of detection probes. Interactions of a given target with any but its \u201ccognate\u201d capture probe has the potential to generate unwanted interference in the multiconstituent probe-target reaction kinetics and equilibria.","III.3.1. Optimizing Primer and Probe Selection","The risk of cross-reaction increases with transcript length and also increases with the number of transcripts in the reaction because the conditional probability of encountering a second subsequence which approximates a given first (\u201ccognate\u201d) subsequence increases with the total length of available target sequence. To enhance specificity of capture, several references of the prior art describe a strategy of \u201cmulti-dentate\u201d capture using two or more probes directed to each anticipated target. However, in a multiplexed format of quantitative analysis, this strategy generally is not advisable, given that it not only increases the complexity of the probe array design but also increases the risk of cross-reactivity with each added probe.","In order to minimize cross-reactivity, it is therefore preferable to produce short transcripts by judicious placement of sequence-specific RT primers close to the 3\u2032 end of the mRNA. Other aspects of assay design relating to certain entropic effects described herein likewise lead to this preference. Accordingly, the assay design techniques described herein are practiced by optimizing the selection of sequence specific RT primers as well as sequence-specific detection probes, preferably in accordance with the methods of the co-pending Application Ser. No. 60\/487,451, supra.","The methods of the present invention take advantage of the a priori knowledge of the sequences and anticipated levels of abundance of the designated mRNAs of interest to select and place RT primers in specific regions of each mRNA in order to control the length and degree of labeling of the cDNA produced in the RT reaction. In some cases, it will be advantageous to place multiple RT primers on one or several of the mRNAs in the designated set and to analyze the corresponding cDNAs using multiple probes directed against different subsequences of these cDNAs. This is referred to herein as \u201cMultiple Primer Multiple Probe\u201d (mpmp) design, as described in the co-pending Application 60\/487,451, supra. In some situations, it will be advantageous to perform the further step of amplifying the reverse transcripts prior to detection.","These methods of the invention relating to optimization of specificity are useful in numerous applications, exemplified by those in Example VII. They also were applied to the multiplexed analysis of a set of cytokine genes, described in detail in Example VIII and related .","III.3.2. Enhancing Specificity by MultiProbe Detection","Combining hMAP and eMAP\u2014","Another assay format of the invention is useful to detect members of gene families where the members of the families have subsequences, in relatively close proximity, of both: (i) significant differences in sequence, such as an insert of 3- or more nucleotides in some members, and (ii) substantial sequence homology, but with minor differences such as single nucleotide polymorphisms (SNPs). Because of the substantial sequence similarity, such sequences can be difficult to distinguish with a conventional hybridization assay given the substantial cross-hybridization.","To solve the problems posed by cross-hybridization, and reduce the cost, the members of the family can be discriminated, and respective abundances determined, by performing a combination of elongation and hybridization in a dual assay format, in which some probes hybridize to the transcripts representing regions with large differences, and other probes hybridize to the transcripts representing regions with small differences, wherein only the latter transcripts are detected using an elongation reaction. By a particular analysis of the results, the family members can be detected. That is, small differences between otherwise homologous sequences preferably are detected by performing a sequence-specific elongation reaction, thereby ensuring identification of members of a gene family while simultaneously using either the elongation reaction itself for the quantitative determination of message abundances (see III.2) or combining elongation with hybridization to ensure discrimination and quantitation.","In the simplest example, one has a family of members having one region of significant sequence differences (a section of 3 added bases) and one region with one SNP. Using the format described above, one would use four beads and two different transcript labels. As illustrated in , one bead has probe hPattached (hybridizing to region P, which contains the added three bases), another coded bead has hPprobe attached (hybridizing to corresponding region P, which does not contain the 3 added bases). A third bead has probe ePattached (hybridizing to region eP, which has normal allele, and the fourth bead has probe ePattached (hybridizing to corresponding region eP, which has a variant allele). The 5\u2032 terminal end of each transcript is labeled with a first color (\u201cred\u201d) by using an appropriately labeled primer during reverse transcription. If a transcript hybridized by the eP1 or eP2 probes is elongated following hybridization, the elongation product is labeled by using extending nucleotides (dNTP or ddNTP) labeled with a second color (\u201cgreen\u201d).","Following hybridization of a sample, one can analyze the array. Where red appears on beads hPor hP, this indicates that the presence of to region Por P, respectively, in the transcript. Where the transcript on the ePbead is elongated, as detected from the green label, this indicates capture of the ePnormal (\u201cwild type\u201d) allele, and where the ePbead displays green, this indicates capture of the ePvariant allele. Accordingly, one can readily detect the presence of transcripts with both regions, using only one elongation reaction, by analyzing patterns of hybridization and elongation. Families of mRNAs with more complex patterns of differences could be analyzed in the same manner, using the appropriate numbers of encoded beads and hybridization and elongation reactions.","III.3.2A. Concurrently Determining Expression Levels and Class of AU-Rich mRNAs","Messenger RNA (mRNA) turnover is involved in the transient response to infection and stress. In mammalian cells, most mRNAs undergo poly(A) shortening as the initial step in their decay. Adenylate uridylate (AU)-rich elements in 3\u2032-untranslated regions (UTR) of mRNA is involved in effectively destabilizing mRNA molecules. Many mRNAs containing an AU-rich element (ARE) are highly expressed in disease states, and may function in selectively boosting or inhibiting gene expression during disease response. The core pentameric sequence of the ARE motif is AUUUA. AREs may contain several copies of dispersed AUUUA motifs, often coupled with nearby U-rich sequences or U stretches. A number of classes of AREs are currently known.","The method herein permits discriminating among the classes of AREs associated with particular unique mRNA subsequences, using probes which can detect the different unique subsequences but which can be labeled with a dye of one color (as opposed to needing multiple colors), and also of determining relative expression levels of unique mRNA subsequences associated with AREs. In this method, one first attaches several of types of probes to encoded beads, where each beads' encoding correlates with the probe-type attached. The probes are selected to hybridize to cDNA regions which are complementary to unique mRNA subsequences upstream of AREs and poly A tails. Samples of mRNA are reverse transcribed to cDNA using primers selected so as to reverse transcribe the ARE as well as the unique mRNA susequence upstream, and the transcripts are labeled and contacted with the probes on the beads under hybridizing conditions.","Following hybridization, as a step in quantitating the relative gene expression, one takes an assay image to show the labeled transcript associated with each encoded bead, and provide an overall image of the labeled transcript in the array. As a step in discriminating among ARE classes, the probes on the beads which have hybridized with a cDNA are elongated under conditions whereby the newly elongated product (which is attached to an encoded bead) will include a portion corresponding to the ARE. This is done by adding all four types of dNTPs in large excess, so that a relatively long probe elongation can take place. An assay image is then recorded for identification of the probe\/transcript type on different beads.","The transcript is then denatured from the elongated probe, for example by heating, and the bead\/probe is contacted, in sequence, with labeled probes of one sequence, from a library of probes complementary to various classes of AREs. These \u201cARE probes\u201d can all be labeled with the same dye, because they are used in succession, rather than being added to the same assay mixture. Upon decoding, following hybridizing the ARE probes, the ARE class which is associated with each bead, and therefore each unique gene sequence, can be determined. The process is shown schematically in .","The relative expression level of the unique gene sequences in vivo can be determined at various points in time, based on the relative signal from the labeled transcripts as determined at such points in time. Such a determination can be useful in monitoring whether certain gene sequences associated with AREs, and thus often with disease conditions, are up or down regulated over time.","III.3.2B. Discrimination of Closely Homologous Sequences: Inbred Strains of Maize","Certain applications such as those discussed herein in greater detail call for the detection of specific targets within an ensemble of hundreds or thousands of targets displaying substantial sequence homology with the target(s) of interest. These circumstances generally will require a degree of sequence-specificity beyond that afforded by hybridization. Certain aspects relating to the selection of suitable primer and probe sets are discussed in detail in co-pending provisional application Ser. No. 60\/487,451, supra. Here we disclose several specific array designs and assay protocols which invoke combinations of sequence-specific sequence conversion by reverse transcription and\/or amplification as well as multiplexed detection by hybridization (hMAP) and\/or elongation (eMAP). Several specific instances are now described to illustrate these assay designs and methodologies of the present invention.","Interrogation of Elongation Products Using Hybridization Probes\u2014","Another assay format of the invention is useful to detect closely homologous members of gene families by a sequence of elongation-mediated detection to discriminate a first subset of genes from a second subset of genes, only the first subset being capable of forming an elongation product which may be detected by incorporating therein a detection label of a first color. Members within the first set may then be further discriminated by the identification of a specific subsequence in the elongation product, this identification involving a hybridization probe modified with a detection label of a second color. Details of this method, previously disclosed in connection with \u201cphasing\u201d of polymorphisms are described in pending U.S. application Ser. No. 10\/271,602, filed Oct. 15, 2002, entitled: \u201cMultiplexed Analysis of Polymorphic Loci by Concurrent Interrogation and Enzyme-Mediated Detection,\u201d and are further described in Example IX with reference to  (the DNA sequence in  is SEQ ID NO. 12; the he DNA sequence in  is SEQ ID NO. 13).","III.4 Subtractive Differential Analysis Using Single Color Detection","In one particular assay format of the invention, subtractive hybridization is used to determine differential expression of different mRNAs (). This is useful, for example, in diagnosis of certain diseases and conditions, where corresponding mRNA levels that differ between diseased and healthy subjects. In this assay format, designated mRNAs are extracted from healthy (\u201cnormal\u201d, N) and diseased (\u201cvariant\u201d, V) subjects and are equalized to ensure equal mRNA concentrations in both samples. This is accomplished, for example, by inclusion of common reference mRNAs in both samples.","In both samples, mRNAs are first reverse transcribed to produce sense cDNAs, respectively denoted cDNAand cDNA. The RT primer used for reverse transcription of one, but not the other sample, is modified with a tag permitting subsequent strand selection. Following reverse transcription, the sample containing the tagged primer, say the normal sample, is transcribed to produce ccDNA, that is, a strand of DNA that is complementary to cDNA; the latter is enzymatically digested.","Next, cDNAand ccDNAare combined under conditions permitting the annealing of these mutually complementary single strands to form a duplex. This step removes (\u201csubtracts\u201d) that amount of DNA that is equal in both samples. Underexpression of one or more designated genes in the V-sample leaves the corresponding excess in the N-sample, and conversely, overexpression of one or more designated genes in the V-sample leaves the corresponding excess in the V-sample. The excess of single stranded DNA is detected using pairs of encoded \u201csense\u201d and \u201cantisense\u201d probes, one matching cDNAthe other matching ccDNA. Preferably, sets of sense and anti-sense probes are displayed on encoded microparticles (\u201cbeads\u201d) forming a random encoded array.","The combined sample is placed in contact with the set of sense and antisense probes and hybridized transcripts are detected, for example, by recording from the set of beads fluorescence signals produced by captured transcripts which may be fluorescently labeled by incorporation of fluorescent RT primers or by incorporation of labeled dNTPs. For each pair of sense and antisense probes, the difference in the intensities indicates the sign and amount of the excess in the corresponding transcript. Significantly, in contrast to standard methods of ratio analysis, only a single color is required here.","IV. Generic Disclosure","Random Encoded Array Detection (READ)\u2014","The method of multiplexed quantitative detection preferably employs an array of oligonucleotide probes displayed on encoded microparticles (\u201cbeads\u201d) which, upon decoding, identify the particular probe displayed on each type of encoded bead. Preferably, sets of encoded beads are arranged in the form of a random planar array of encoded microparticles on a planar substrate permitting examination and analysis by microscopy. Intensity is monitored to indicate the quantity of target bound per bead. The labels associated with encoded beads and the labels associated with the transcripts bound to the probes in the array are preferably fluorescent, and can be distinguished using filters which permit discrimination among different hues. This assay format is explained in further detail in U.S. application Ser. No. 10\/204,799, filed Aug. 23, 2002, entitled: \u201cMultianalyte molecular analysis using application-specific random particle arrays,\u201d hereby incorporated by reference.","Libraries of Probe-Functionalized Encoded Microparticles (\u201cBeads\u201d)\u2014","The particles to which the probes are attached may be composed of, for example, plastics, ceramics, glass, polystyrene, methylstyrene, acrylic polymers, paramagnetic materials, thoria sol, carbon graphite, titanium dioxide, latex or cross-linked dextrans such as sepharose, cellulose, nylon, cross-linked micelles and Teflon. (See, e.g., \u201cMicrosphere Detection Guide\u201d from Bangs Laboratories, Fishers, Ind.). The particles need not be spherical and may be porous. The particle sizes may range from nanometers (e.g., 100 nm) to millimeters (e.g., 1 mm), with particles from about 0.2 micron to about 200 microns being preferred, with particles from about 0.5 to about 5 microns being more preferred.","Particles are encoded so as to be correlated with the sequence-specific bead-displayed probes that are placed on the surface of the particles by a chemically or physically distinguishable characteristic, for example fluorescence, uniquely identifying the particle. Chemical, optical, or physical characteristics may be provided, for example, by staining beads with sets of optically distinguishable tags, such as those containing one or more fluorophore or chromophore dyes spectrally distinguishable by excitation wavelength, emission wavelength, excited-state lifetime or emission intensity. The optically distinguishable tags may be used to stain beads in specified ratios, as disclosed, for example, in Fulwyler, U.S. Pat. No. 4,717,655. Staining may also be accomplished by swelling particles in accordance with methods known to those skilled in the art, (See, e.g., Molday, Dreyer, Rembaum & Yen, J. Mol Biol 64, 75-88 (1975); L. Bangs, \u201cUniform latex Particles, Seragen Diagnostics, 1984). Using these techniques, up to twelve types of beads were encoded by swelling and bulk staining with two colors, each individually in four intensity levels, and mixed in four nominal molar ratios. Alternatively, the methods of combinatorial color encoding described in International Application No. PCT\/US 98\/10719 (incorporated herein by reference) may be used to endow the bead arrays with optically distinguishable tags.","Probes\u2014","A set of sequence-specific probes, known as a \u201ccapture probe set\u201d, is used in the assay. Each member of a capture probe set is designed\u2014preferably using methods of the co-pending provisional application entitled \u201cHybridization-Mediated Analysis of Polymorphisms (hMAP),\u201d filed May 17, 2004, Ser. No. 10\/847,046\u2014to have a unique complementary region with one \u201ccognate\u201d cDNA target molecule. As explained above, the length of the complementary region of each member of a capture probe set may be different in order to tailor the binding affinity.","These oligonucleotide probes may be synthesized to include, at the 5\u2032 end, a biotinylated TEG spacer for attachment to microparticles functionalized by attachment of Neutravidin, or an aminated TEG spacer (Synthegen TX) for covalent attachment to the functionalized surface of particles, using carboxylated beads and an EDAC reaction.","Reverse Transcription\u2014","The total RNA used for these assays is isolated and reverse transcribed to cDNA, and the cDNA molecules are added in the presence of a solution containing dNTPs, or ddNTPS, and DNA polymerase to elongate the cDNA on those probes on which the 5\u2032 end of the target and the complementary sequence on the probe are perfectly matched. The dNTP\/ddNTP mixture contains at least one labeled dNTP or ddNTP, in order to incorporate fluorescent label in the elongated target. The cDNA target molecules of the assay are fluorescently labeled as described herein, and the density of the fluorescently labeling (e.g., the degree of incorporation of fluorescently labeled dNTPs) of the cDNA target molecules may vary, depending on whether the expression level of the corresponding mRNA is expected to be high or low. In addition, the region the probe binds to on the transcript affects the hybridization pattern; i.e., it is easier for probes to bind to the ends. Details are described in several Examples below.","Methods of Array Assembly\u2014","To produce a custom array containing a specific probe combination, the encoded, probe-decorated beads are pooled together and assembled into arrays. Many different methods of assembling arrays are possible, including a technique known as LEAPS\u2122 (Light-Controlled Electrokinetic Assembly of Particles Near Surfaces, described in U.S. Pat. No. 6,251,691 which is hereby incorporated by reference). In LEAPS\u2122, the bead arrays are prepared by first providing a planar electrode that is substantially parallel to a second planar electrode (in a \u201csandwich\u201d configuration), with the two electrodes being separated by a gap, where in the gap is a polarizable liquid medium, such as an electrolyte solution. The surface or the interior of the second planar electrode is patterned to create areas of lowered impedance. The beads are then introduced into the gap. When an AC voltage is applied to the gap, the beads form a random encoded array on the second electrode, in accordance with the patterning, or, in the alternative, in accordance with an illumination pattern on the second electrode. The resulting arrays can exhibit a very high feature density. Alternative methods of assembly of particle arrays are described in U.S. application Ser. No. 10\/192,352, filed Jul. 9, 2002, entitled: \u201cArrays of Microparticles and Methods of Preparation Thereof.\u201d","Decoding Image\u2014","In an assay of the invention, the population of particles is encoded with a distinct chemical or physical characteristic that allows the type of particle to be determined before and after the assay. For decoding, a decoding image of the assembled array is taken, prior to the assay or subsequent to the assay, to record the spatial distribution of encoded particles in the array and hence the spatial distribution of the members of the capture probe set.","Optical Signatures and Assay Images\u2014","To facilitate detection of captured targets, cDNA molecules are fluorescently labeled by incorporation, during reverse transcription, of labeled dNTPs at a preset molar ratio, the total amount of incorporated dNTP varying with the length of the (reverse) transcript. Instead of, or in addition to, hybridization-mediated capture, the assays of the invention also include elongation-mediated detection; cDNA molecules are added in the presence of a solution containing dNTPs, or ddNTPS, and DNA polymerase to elongate the cDNA on those probes whose 3\u2032 end is complementary to the captured target. The dNTP\/ddNTP mixture contains at least one labeled dNTP or ddNTP, in order to incorporate fluorescent label in the elongated probe.","The labels associated with the encoded beads and the labels associated with the transcripts bound to the probes in the array are preferably fluorescent, and can be distinguished using filter combinations which permit discrimination among different excitation and emission wavelengths and hence combinations of base colors that are combined in multiple combinations. In accordance with the preferred embodiment of READ, beads are assembled into planar arrays that can be readily examined and analyzed using, for example, a microscope. The intensity of an optical signature produced in the course of capturing and analyzing targets is monitored to indicate the quantity of captured target.","Recording of Decoding and Assay Images\u2014","A fluorescence microscope is used to decode particles in the array and to detect assay signals from the array of probe-captured cDNA molecules. The fluorescence filter sets in the decoder are designed to distinguish fluorescence produced by encoding dyes used to stain particles, whereas other filter sets are designed to distinguish assay signals produced by the dyes associated with the transcripts\/amplicons. A CCD camera may be incorporated into the system for recording of decoding and assay images. The assay image is analyzed to determine the identity of each of the captured targets by correlating the spatial distribution of signals in the assay image with the spatial distribution of the corresponding encoded particles in the array.","Assay\u2014","Either prior to, or subsequent to decoding, the array of encoded particles is exposed to the cDNA target molecules under conditions permitting capture to particle-displayed probes. After a reaction time, the array of encoded particles is washed with 1\u00d7TMAC to remove remaining free and weakly annealed cDNA target molecules. Instead of or in addition to hybridization assays, the assays of the invention include elongation-based detection.","An assay image of the array is then taken to record the optical signal of the probe-cDNA complexes of the array. Because each type of particle is uniquely associated with a sequence-specific probe, combination of the assay image with the decoding image, recorded, for example, prior to performing the assay, permits the identification of annealed cDNA molecules whose respective abundances\u2014relating directly to the abundances of the corresponding original mRNA messages\u2014are determined from the fluorescence intensities of each type of particle.","The examples below provide further details regarding the making and using of the invention.","Synthetic DNA polynucleotide targets varying in length from 25-mers to 175-mers, were synthesized (by IDT, Madison, Wis.), and each of the larger targets contained the smaller target as an interior subsequence. All the targets were labeled with Cy5 fluorescent label at the 5\u2032 end. Amine-modified (5\u2032 end) oligonucleotide probes, varying in length from 15 nt to 35 nt, were also synthesized (IDT, Madison, Wis.). The detailed sequence information is shown in Table I-1.","The probes were covalently linked to encoded tosylated microparticles using an EDAC reaction, as is well known in the art. A precalculated amount of each of the synthetic targets was taken from a 10 \u03bcM stock solution of the target in de-ionized water, and was diluted with 1\u00d7TMAC (4.5 M tetramethyl ammonium chloride, 75 mM Tris pH 8.0, 3 mM EDTA, 0.15% SDS) to a desired final concentration. One or more of the probe types listed in TableI-1 were functionalized with fluorescent microparticles and were then assembled into planar arrays on silicon substrates. Twenty microliters of the synthetic target was added to the substrate surface and the substrate was placed in a 55\u00b0 C. heater for 20 minutes. The slide was then removed from the heater and the target solution was aspirated. The substrate was washed thrice with 1\u00d7TMAC at room temperature. Following this, 10 \u03bcl of 1\u00d7TMAC was placed on the substrate surface, covered with a glass cover-slip and the fluorescence intensity of the array was recorded.  show the results obtained from these hybridization experiments.","Experiments were performed with commercially available QuantiBRITE\u2122 PE Phycoerythrin Fluorescence Quantitation kit from Becton-Dickinson, Franklin Lakes, N.J. The kit consists of 6.6 \u03bcm polymer beads, conjugated with known number of Phycoerythrin (PE) molecules on the surface. For quantitative analysis of the fluorescent intensity associated with the beads, random planar arrays of the beads were assembled on the surface of a silicon wafer. The fluorescent intensity from the PE fluorophores on the particle surface was then monitored as a function of varying number of surface conjugated PE fluorophores (data supplied by manufacturer) using a standard fluorescent microscope fitted with an appropriate fluorescence filter and a CCD camera. In this study, a Nikon Eclipse E-600FN epifluorescence microscope equipped with 150 W xenon-arc lamp was used for measurements. A Nikon 20\u00d70.75 NA air objective, and a R&B PE Filter cube (Chroma Technology Corp., Battleboro, Vt.) was used for the measurements. Images were recorded with a cooled 16 bit CCD camera (Apogee Instruments Inc.). The exposure\/integration time for the experiment was 500 ms. User interfaced programs for collection and analysis of images were developed using MATLAB\u2122 which was run on a PC. The results are shown in , from which it can be seen that \u02dc100 PE molecules\/particle (i.e. IPE molecules\/\u03bcm) can be detected using this system.","The fluorescent properties of R-phycoerythrin and 2 common CY dyes are compared in the following Table I-3.",{"@attributes":{"id":"p-0217","num":"0251"},"tables":{"@attributes":{"id":"TABLE-US-00008","num":"00008"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"center"}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"center"}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"center"}}],"thead":{"row":[{"entry":"TABLE I-3"},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":[{},{},{},{},"QY for",{}]},{"entry":[{},"Abs. Max.","Em. Max.","Ext. Coeff.","protein","Mol. Wt."]},{"entry":["Name","(nm)","(nm)","(Mcm)","conjugates","(dye)"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":{}}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"char","char":"."}},{"@attributes":{"colname":"5","colwidth":"35pt","align":"char","char":"."}},{"@attributes":{"colname":"6","colwidth":"35pt","align":"char","char":"."}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["R-phyco-","480","578","1,960,000","0.82","240,000"]},{"entry":["erythrin","546"]},{"entry":[{},"565"]},{"entry":["Cy3","550","570","150,000","0.16","766"]},{"entry":["Cy5","649","670","250,000","0.28","792"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},"Hence one PE molecule is equivalent to \u02dc60 Cy3 molecules or \u02dc20 Cy5 molecules. Accordingly, the anticipated detection threshold for the Cy3 is \u02dc60 molecules\/umand for Cy5\u02dc20 molecules\/um. A 2 um particle has a surface area of \u02dc12.5 umand would hence need 750 molecules of Cy3\/particle for detection and 250 molecules of Cy5\/particle for detection. The corresponding numbers for a 3 micron particle are 1700 for Cy3 and 600 for Cy5. Hence, a conservative estimate of the detection sensitivity using Cy dyes (for 2-3 micron particles) is \u02dc1000 fluorophores\/particle.","In the same way as discussed above the slope of the curve can also be used as an approximate conversion factor (when using dyes other than PE) for converting recorded raw intensities back to number of molecules\/umand with the knowledge of the bead size, then to the number of fluorophores\/bead.","A typical experimental protocol for multiplexed expression monitoring is as follows. A protocol establishing optimized conditions in accordance with the methods of the present invention is described below. The entire protocol including signal amplification in accordance with the methods of the present invention is completed in less than three hours (see ).","Step 1\u2014Total RNA is isolated from a blood or tissue sample using Qiagen silica-gel-membrane technology. DNA oligonucleotides with a sequence complementary to that of mRNAs of interest are added to the preparation to prime the reverse transcription of the targeted mRNAs into cDNAs.\n\nStep 2\u2014The solution containing mRNAs is heated to 65\u00b0 C., typically for a period of 5 minutes, to facilitate annealing of primers to denatured mRNAs, following which the solution is gradually cooled to room temperature at a typical rate of 2\u00b0 C.\/min. Reverse transcriptase (for example Superscript III, Contech) along with fluorescently labeled dNTPs (at a typical molar ratio of 1:8, labeled to unlabeled dCTP) are added to initiate the RT reaction. After synthesis of labeled cDNAs, RNA templates are digested using RNase.\n\nStep 3\u2014Fluorescently labeled cDNAs are permitted to anneal, in 1\u00d7TMAC buffer at 50\u00b0 C. for 30 minutes, to arrays of color-encoded microparticles displaying DNA oligonucleotide capture probes on silicon chips () in accordance with the READ format. Hybridization was followed by three consecutive steps of washing in 1\u00d7TMAC buffer, each step requiring only the exchange of buffer.\n","As necessary, signal amplification in accordance with the methods of the present invention may be performed as described herein.","Capture probe sequences are designed to be complementary to the 3\u2032 regions of individual cDNAs in the mixture. The optimization of capture probe sequences for use in the multiplexed analysis of cDNAs is described in greater detail in the co-pending application Ser. No. 10\/892,514 entitled: \u201cConcurrent Optimization in Selection of Primer and Capture Probe Sets for Nucleic Acid Analysis,\u201d filed Jul. 15, 2003. Arrays are prepared as described herein. Step 4\u2014The resulting pattern of fluorescence is recorded in the form of a fluorescence image by instant imaging (typically using integration times less than 1 second) on an automated Array Imaging System as described in greater detail in U.S. Provisional application Ser. No. 10\/714,203 entitled: \u201cAnalysis, Secure Access to, and Transmission of Array Images\u201d","filed Nov. 14, 2003. Manually operated fluorescence microscopy also may be used. From the assay image quantitative intensities are determined by analysis of the assay image as described herein and described in greater detail in the Ser. No. 10\/714,203.","An mpmp-RT design comprising six Cy3-modified RT primers and multiple microparticle-displayed capture probes was used, in a single reaction for each of a series of solutions of successively lower Kanamycin concentrations, in accordance with a 1:2 serial dilution. A mixture of fragments varying from 79 nt to 150 nt in size, incorporating into each fragment Cy-3 modified dCTP at an average molar ratio of 1:16 of labeled to unlabeled dCTP and hence at an average labeling density of 1:64, was produced.","Using an mpmp-RT design comprising either one or two Cy3-modified RT primers and microparticle-displayed capture probes, RT reactions were performed on each of a series of Kanamycin mRNA solutions of successively lower concentrations, spanning a range from 25 nM to \u02dc50 pM. Specifically, three combinations of RT primers and capture probes were tested to produce and analyze cDNA fragments of 70 nt and\/or 50 nt in size. The Cy3 labeling density of the transcripts was also doubled\u2014from 1:64 to 1:32\u2014by incorporating into each fragment Cy-3 modified dCTP at an average molar ratio of 1:8 of labeled to unlabeled dCTP. Using Cy3-labeled RT primers, each 50 nt transcript will on average contain 2-3 Cy3 labels.","Having established target configurational entropy as a critical factor affecting the sensitivity of cDNA detection, it was then confirmed in several assay designs that a further reduction in transcript length from 150 nt to \u02dc50 nt, along with a doubling of the Cy3 labeling density of transcripts obtained from a 1,200 nt Kanamycin model mRNA, produced a further enhancement in assay signal by the anticipated factor of \u02dc5, corresponding to a detection limit of \u02dc50 pM.","Significantly, closely comparable results\u2014including the critical role of target entropy\u2014were obtained with a mixture of 8 unknown mRNAs into which the Kanamycin mRNA was \u201cspiked\u201d at molar ratios varying from \u02dc1:12 to \u02dc1:6,200, respectively, corresponding to Kanamycin concentrations of 25 pM and 50 pM and an mRNA \u201cbackground\u201d of 300 nM. The results of these model assays indicate sufficient sensitivity and specificity to detect a specific message in the presence of other mRNA molecules at an abundance as low as \u02dc3-5 copies per cell.","To test the predictions in Example III, namely that a further reduction in transcript length from \u02dc150 nt to \u02dc50 nt would produce a further enhancement in assay signal, mpmp-RT reactions were designed to generate 50 nt and\/or 70 nt transcripts. Having demonstrated the enhancement in assay signal arising from the use of \u201c5\u2032-end-directed\u201d capture probes (see Example III), capture probes were designed so as to target a subsequence near the transcript's 5\u2032 terminus.","Optimization of Assay Protocol\u2014","In order to further improve assay sensitivity and dynamic range further, assay conditions were optimized. Specifically, RT primer concentrations in the Kanamycin mRNA titrations were reduced 25-fold (from 50 \u03bcM to 2 \u03bcM) and hybridization time was reduced by half (from 30 min to 15 min at 50\u00b0 C.).","This protocol modification not only avoids saturation of the detector at the highest target concentration of \u02dc500 pM () but also reduces the background signal contributed by non-specific adsorption of fluorescently labeled RT primers and dCTPs remaining in the solution, thereby contributing to an extension in the dynamic range of the assay. A two-fold improvement was observed in assay sensitivity.","To further improve upon assay performance of the mpmp-RT design reported in Example III, the Reverse Transcription (RT) protocol was optimized for 50 nt kanamycin transcripts\u2014the best performer\u2014by performing RT reactions under stringent temperature control. Using a programmable temperature profile in a thermocycler, the improved protocol for RT reactions in conjunction with stringent RT primer annealing and transcription conditions, an enhancement of fluorescence signal intensities by a factor of 2-3 was obtained ().","Specifically, RT reactions, configured as described in Example III, were performed in a thermocycler (Perkin-Elmer) ti implementing the following temperature profile:",{"@attributes":{"id":"p-0232","num":"0000"},"ul":{"@attributes":{"id":"ul0019","list-style":"none"},"li":{"@attributes":{"id":"ul0019-0001","num":"0000"},"ul":{"@attributes":{"id":"ul0020","list-style":"none"},"li":["RNA denaturation: 5 min at 65\u00b0 C.;","Annealing: 30 min at 450\u00b0 C.;","Annealing: 20 min at 38\u00b0 C.;","SuperScript III heat inactivation: 5 min at 85\u00b0 C.; and","Hold at 4\u00b0 C."]}}}},"Hybridization conditions were: incubation for 15 minutes at 50\u00b0 C. in 1\u00d7TMAC, followed by 3 subsequent wash steps with the same buffer, each simply involving exchange of the 20 \u03bcl volume in contact with BeadChips by fresh buffer.","This 2-step protocol enforcing stringent RT conditions produced an enhancement in the specific fluorescence signal while leaving non-specific background signal comparable to that obtained earlier (\u201cProtocol 2\u201d), thus improving the signal to noise ratio of the assay about 2-fold.","To further evaluate the level of specificity attainable in detecting a specific mRNA in the complex environment typical of a clinical human sample enriched with multiple RNA messages, an additional series of \u201cspiking\u201d experiments were performed by replacing the background of unknown total RNA of bacterial origin by total RNA from Human Placenta (Ambion). Total Human Placental RNA more realistically simulates conditions typically encountered in the determination of expression patterns of particular RNA species such as human interleukins and other cytokines in clinical samples.","Aliquots of Kanamycin mRNA, ranging in concentration from \u02dc12.5 nM to \u02dc50 pM, were spiked into solutions of total Human Placental RNA diluted to 100 ng\/ul, corresponding to a concentration of \u02dc300 nM. That is, the molar ratios of specific to non-specific mRNA ranged from 1:24 to 1:6,200. At each of eight ratios\u2014including a no-target control\u2014an RT reaction was performed separately under optimized assay conditions.","The results () follow the trend previously observed in the absence of total RNA. Thus, for a transcript of length 50 nt, spiked into a total RNA of human origin, the non-specific signal arising from the capture of fluorescently labeled cDNAs produced by randomly primed reverse transcription was insignificant compared to the specific signal generated by the capture of the entropically favored 50 nt Kanamycin cDNA. The lowest detected target level, at a molar ratio of \u02dc1:6,200, corresponds to a concentration of \u02dc50 pM of the specific mRNA, equivalent to approximately hundreds of copies per cell. Thus, this assay design attains a sensitivity and specificity comparable to that of commercially available expression profiling protocols (Lockhart et al, (1996)) not only in a mixture of eight unknown RNA in-vitro transcripts, but also in a complex environment using a real processed human sample.","Given the critical importance of specificity in multiplexed gene expression profiling, the previously reported Kanamycin \u201cspiking\u201d experiments to a pool of human placental RNAs was extended in order to simulate conditions relevant to clinical samples. The results are essentially identical in terms of specificity and sensitivity to those previously reported for spiking of in-vitro transcribed RNAs of bacterial origin, suggesting that the combination of producing short RT transcripts, directing capture probes to regions near the transcript's 5\u2032-end and performing RT and hybridization under stringent conditions enhances specificity. Randomly primed RT transcripts generally will exceed the length of specific RT transcripts, providing the latter with a significantly entropic advantage in capture to immobilized probes.","The critical role of target entropy was again apparent under the optimized RT conditions. Thus, the biphasic plots in  again indicate a cross-over from a dilute regime characterized by a higher affinity constant to a \u201cconcentrated\u201d regime with lower affinity constant. As previously discussed, effective affinity constants in the concentrated regime, reflecting the \u201ccrowding\u201d of targets, are strongly transcript-length dependent. Indeed, slopes of the adsorption isotherms in the concentrated regime are substantially identical for the 50 nt transcripts produced under two different RT reaction protocols (). In contrast, in the dilute regime, the isotherm of the 50 nt unspiked transcript prepared by the stringent Protocol 3 displays a slope that is smaller by a factor of \u02dc2.5 than that of the isotherm of the 50 nt unspiked transcript prepared under the less-stringent Protocol 2, indicating a correspondingly higher value for the affinity constant under improved RT conditions.","The assay formats described herein can be used for diagnosis and can, in certain cases, be used in connection with providing treatment.","Leukemia\u2014","For example, International Application No. WO 03\/008552 describes diagnosis of mixed lineage leukemia (MLL), acute lymphoblastic leukemia (ALL), and acute myellgenous leukemia (AML) according to the gene expression profile. These assay formats can also be used to analyze expression profiles of other genes, such as for Her-2, which is analyzed prior to administration of Herceptin\u2122. The gene expression profile could also be useful in deciding on organ transplantation, or in diagnosing an infectious agent. The effect of a drug on a target could also be analyzed based on the expression profile. The presence of certain polymorphisms in cytokines, which can indicate susceptibility to disease or the likelihood of graft rejection, also can be analyzed with the format described herein. Other examples for the application of the methods of the invention include such the analysis of the host response to exposure to infectious and\/or pathogenic agents, manifesting itself in a change of expression patterns of a set of designated genes","ADME Panel\u2014","Adverse drug reactions have been cited as being responsible for over 100,000 deaths and 2 million hospitalizations in one year in the USA. Individual genetic variation is responsible for a significant proportion of this. However, the indirect method of detecting genetic variation as a result of drug therapies is to monitor gene expression levels of the specific biomarkers.","The described methodology in Example 1 can be expanded to drug metabolism-associated genetic markers with approximately 200 genes that regulate drug metabolism. These important markers are available in flexible, customizable ADME (absorption-distribution-metabolism-excretion\/elimination) panels. The first ADME panel is based on cytochrome P450, a super-family of 60 genes that govern many drug-metabolizing enzymes.","The new standard in multiplexed gene expression monitoring using BeadChips offers unprecedented accuracy, sensitivity and specificity. For instance, hMAP method followed by eMAP (elongation reaction) was applied to discrimination of closely related sequences of cytochrome P 450 gene family, namely, CYP 450 2B1 and 2B2. The established methodology on BeadChips allows to specifically measure 2-fold changes in gene expression levels of 96% homologous sequences in a highly multiplexed assay format.","Preparation of Nine (9) Human Cytokine In-Vitro Transcripts\u2014","To initiate the development of a custom BeadChip for multiplexed gene expression profiling of a clinically relevant panel of markers, we have designed a control system of nine (9) human cytokine mRNA targets, listed in Table III-1.","Full-length cDNA clones of seven cytokines (IL-2, -4, -6, -8, -10, TNF-\u03b1 and IFN-\u03b3) and two endogenous controls (GAPDH, Ubiquitin) were characterized by sequencing and recovered in the form of plasmid DNAs containing specific cytokine cDNA inserts in pCMV6 vector (OriGene Technologies, MD). PCR primers to the cloning vector sequence were designed to amplify all cDNAs with a standard primer pair, thus eliminating the substantial cost of target-specific PCR amplification. Positioning of the Forward PCR primer upstream of the T7 promoter sequence\u2014located next to the cloning site of every cytokine insert (cDNA)\u2014enables T7 in-vitro transcription of only the specific cDNA sequence located at the 5\u2032-end of the target of interest. Following in-vitro transcription (MegaScript, Ambion), templates were characterized for purity in agarose gel using SybrGreen staining; DNA concentrations were determined by optical absorption following 200-fold dilution.","Next, a multiplexed RT reaction was performed using a set of nine gene-specific RT primers to produce a pool of nine Cy3-labeled cDNAs, according to the optimized protocol we developed for Kanamycin. Specifically, we applied our empirical design rules (see below) to select RT primers so as to produce cDNAs 50 nt to 70 nt in length while minimizing cross-hybridization. This pool of cDNAs was placed, without any purification, onto a BeadChip containing eleven types of encoded beads displaying specific capture probes designed for the set of seven cytokine cDNAs as well as two endogenous positive controls and two negative controls, namely a oligo-C18 and Kanamycin.","First results based on the empirical design rules for primer\/probe selection demonstrated the ability of Random Encoded Array Detection (READ) format of multiplexed analysis to determine expression levels of multiple designated cytokine genes. However, two mRNA targets in 9-plex assay were detected with the signal intensity close to the marginal threshold of unspecific background signal, as a result of cross-reactive binding of the corresponding RT primers to other mRNA targets in a complex sample pool. These results indicated an urgent need in the further optimization of primer\/probe design rules involving user-friendly computational tools based on the mathematical algorithms which we disclosed above.","Using the second version of our design rules for RT primer and capture probe selection, we have re-designed 11 sets of capture probes with the corresponding reverse transcription primers specific for each mRNA of interest (Table III-1). To increase specificity of hybridization reactions between RT primers and targets, we also extended length of primer sequences to \u02dc20 nucleotides in length. Based on calculated melting temperatures for the re-designed RT primers and capture probes, we performed the RT reaction with a higher stringency than earlier, using a 2-step profile, starting with RNA denaturation at 70\u00b0 C. for 5 min, followed by primer annealing and extension at 52\u00b0 C. for 60 min. On chip hybridization was performed at 57\u00b0 C.\u2014an average Tm of the nine re-designed probes.","Next, a multiplexed RT reaction was performed on 9 in vitro transcribed RNAs, containing 32 femtomoles of each message, using a set of nine gene-specific RT primers to produce a pool of nine Cy3-labeled cDNAs in accordance with the 2-step temperature incubation protocol we optimized as discussed above. Specifically, we applied our computational design rules (see Report IV) to select RT primers so as to produce cDNAs from 60 nt to 200 nt in length while minimizing cross-hybridization (see above).","This pool of directly labeled Cy3-cDNAs, containing 16 femtomoles of each added mRNA, was placed, without any purification, onto a BeadChip containing eleven types of encoded beads displaying specific capture probes designed for the set of seven cytokine cDNAs as well as two endogenous positive controls and two negative controls, namely a oligo-C18 and Kanamycin. The results presented in  demonstrate multiplexed reproducible detection of six cytokine cDNAs, IL-6 having been omitted from the RT reaction to provide an indication of the low level of non-specific hybridization. The signal to noise ratios were reproducible within the range from 3.5 to (see Table III-2, included in ), that confirms statistical significance of signal output for every message detected. BeadChips included \u02dc300 beads for each of the cDNAs\u2014this redundancy provides an added level of reliability.",{"@attributes":{"id":"p-0252","num":"0290"},"tables":{"@attributes":{"id":"TABLE-US-00009","num":"00009"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","pgwide":"1","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"329pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u2003III-1"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Set\u2003of\u20039\u2003human\u2003Cytokine\u2003cDNA\u2003Clones\u2003for\u2003Multiplexed\u2003Analysis:"},{"entry":"Designs\u2003of\u2003Reverse\u2003Transcription\u2003Primers\u2003and\u2003Capture\u2003Probes"},{"entry":"of\u2003the\u2003Analytes."}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"6"},"colspec":[{"@attributes":{"colname":"1","colwidth":"14pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"42pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"70pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"77pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"98pt","align":"left"}},{"@attributes":{"colname":"6","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"Access-","Sample",{},{},"Bead"]},{"entry":["No","ion\u2003No.","Description","RT\u2003primer","Capture\u2003Probe","Code"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}},{"entry":["\u20031","NM_00",{"i":"Homo\u2003sapiens"},"ATTGGGCGTC","ATGTTGAAGCCAT","G5B"]},{"entry":[{},"0206","interleukin\u20032","AGAATTGTCG","CATTACCATTC",{}]},{"entry":[{},{},"receptor,","20-mer,\u200362.0C","25-mer,\u200362.6C",{}]},{"entry":[{},{},"gamma","SEQ\u2003ID\u2003NO.\u200354","SEQ\u2003ID\u2003NO.\u200355",{}]},{"entry":[{},{},"(IL2RG),",{},{},{}]},{"entry":[{},{},"mRNA",{},{},{}]},{"entry":{}},{"entry":["\u20032","NM_15",{"i":"Homo\u2003sapiens"},"GGACGAGGAC","TGTCCTGCTGTCAC","G5C"]},{"entry":[{},"2899","interleukin\u20034","GAGGAGGT","CAAGAG",{}]},{"entry":[{},{},"induced\u20031","18-mer,","20-mer,\u2003Tm\u2003= 62.7C",{}]},{"entry":[{},{},"(IL4I1),","Tm\u2003= 63.6C","SEQ\u2003ID\u2003NO.\u200357",{}]},{"entry":[{},{},"transcript","SEQ\u2003ID\u2003NO.\u200356",{},{}]},{"entry":[{},{},"variant\u20031,",{},{},{}]},{"entry":[{},{},"mRNA",{},{},{}]},{"entry":{}},{"entry":["\u20033","NM_00",{"i":"Homo\u2003sapiens"},"GCTAATGGGA","CAGTGTGTGTAGA","G5D"]},{"entry":[{},"0565","interleukin\u20036","ACCGGGC","GAGCCGG",{}]},{"entry":[{},{},"receptor","17-mer,","20-mer,\u2003Tm\u2003= 63.1C",{}]},{"entry":[{},{},"(IL6R),","Tm\u2003= 61.5C","SEQ\u2003ID\u2003NO.\u200359",{}]},{"entry":[{},{},"mRNA","SEQ\u2003ID\u2003NO.\u200358",{},{}]},{"entry":{}},{"entry":["\u20034","NM_00",{"i":"Homo\u2003sapiens"},"TCTTTAGCACT","GTGTAGGCACTGA","G5E"]},{"entry":[{},"0584","interleukin\u20038","CCTTGGCAAA","GGACGG",{}]},{"entry":[{},{},"(IL8),\u2003mRNA","21-mer,\u200360.8C","22-mer,\u200364.3",{}]},{"entry":[{},{},{},"SEQ\u2003ID\u2003NO.\u200360","SEQ\u2003ID\u2003NO.\u200361",{}]},{"entry":{}},{"entry":["\u20035","NM_00",{"i":"Homo\u2003sapiens"},"ATGAGCGTCT","ATGCTGCCGTGCC","G5F"]},{"entry":[{},"1558","interleukin\u200310","GAGCCAAGA","TCGTAG",{}]},{"entry":[{},{},"receptor,\u2003alpha","19-mer,","19-mer,\u2003Tm\u2003= 66.1C",{}]},{"entry":[{},{},"(IL10RA),","Tm\u2003= 62.0C","SEQ\u2003ID\u2003NO.\u200363",{}]},{"entry":[{},{},"mRNA","SEQ\u2003ID\u2003NO.\u200362",{},{}]},{"entry":{}},{"entry":["\u20036","NM_00",{"i":"Homo\u2003sapiens"},"TCATAGTATTC","CAGGTGGCATTTA","G3B"]},{"entry":[{},"1066","tumor\u2003necrosis","TCTGAGCCGG","CACCCTACG\u200322-",{}]},{"entry":[{},{},"factor\u2003receptor","19-mer,\u200359.4C","mer,\u200364.3C",{}]},{"entry":[{},{},"superfamily,","SEQ\u2003ID\u2003NO.\u200364","SEQ\u2003ID\u2003NO.\u200365",{}]},{"entry":[{},{},"member\u20031B.",{},{},{}]},{"entry":[{},{},"TNFRSF1B,",{},{},{}]},{"entry":[{},{},"mRNA",{},{},{}]},{"entry":{}},{"entry":["\u20037","NM_01",{"i":"Homo\u2003sapiens"},"GTCTTGCCGGT","GCAGGATCCTGGT","G3C"]},{"entry":[{},"8955,","ubiquitin\u2003B","AAGGGTT","ATCCGCTA",{}]},{"entry":[{},"Internal\u2003","(UBB),","18-mer,","21-mer,\u2003Tm\u2003= 64.4C",{}]},{"entry":[{},"control","mRNA","Tm\u2003= 60.4C","SEQ\u2003ID\u2003NO.\u200367",{}]},{"entry":[{},{},{},"SEQ\u2003ID\u2003NO.\u200366",{},{}]},{"entry":{}},{"entry":["\u20038","NM_00",{"i":"Homo\u2003sapiens"},"ACGGTGCCAT","GGAGTCAACGGAT","G3D"]},{"entry":[{},"2046,","glycer-","GGAATTTGC","TTGGTCGT",{}]},{"entry":[{},"Internal","aldehyde-3-","19-mer,","21-mer,\u2003Tm\u2003= 63.6C",{}]},{"entry":[{},"control","phosphate","Tm\u2003= 62.8C","SEQ\u2003ID\u2003NO.\u200369",{}]},{"entry":[{},{},"dehydrogenase","SEQ\u2003ID\u2003NO.\u200368",{},{}]},{"entry":[{},{},"(GAPD),",{},{},{}]},{"entry":[{},{},"mRNA",{},{},{}]},{"entry":{}},{"entry":["\u20039","NM_00",{"i":"Homo\u2003sapiens"},"GTGTAGGCAC","GCATGGCTCTCCT","G3E"]},{"entry":[{},"0416","interferon","TGAGGACGG","CTTTCTCC",{}]},{"entry":[{},{},"gamma","19-mer,\u2003Tm\u2003= 63C","21-mer,\u2003Tm\u2003= 63.5C",{}]},{"entry":[{},{},"receptor\u20031","SEQ\u2003ID\u2003NO.\u200370","SEQ\u2003ID\u2003NO.\u200371",{}]},{"entry":[{},{},"(IFNGR1),",{},{},{}]},{"entry":[{},{},"mRNA",{},{},{}]},{"entry":{}},{"entry":["10","Neg","Control\u2003for","none","Oligo-C18","G2A"]},{"entry":[{},"control","unspecific",{},{},{}]},{"entry":[{},{},"binding\u2003of",{},{},{}]},{"entry":[{},{},"nucleic\u2003acids",{},{},{}]},{"entry":{}},{"entry":["11","Neg","Kanamycin","none","TACAAGCTTGGGC","G2B"]},{"entry":[{},"control,","mRNA",{},"GTGTCTC",{}]},{"entry":[{},"Non-","not\u2003present\u2003in",{},"20-mer,\u2003Tm\u2003= 63.4C",{}]},{"entry":[{},"human","a\u2003multiplexed",{},"SEQ\u2003ID\u2003NO.\u200372",{}]},{"entry":[{},{},"mix"]},{"entry":{"@attributes":{"namest":"1","nameend":"6","align":"center","rowsep":"1"}}}]}}]}}},"In the two inbred maize lines B73 and BSSS53, certain mRNA sequences of the zein gene display a degree of 95% to 99% homology over the entire 945 nt of the sequence.  illustrates the placement of capture and elongation probes to target specific mutations (highlighted in red) for detection of seven highly expressed mRNA sequences in the inbred maize line BSSS53.","The task of detecting these sequences and estimating their respective expression levels with current methods is a very laborious process, requiring of sequencing large sets of clones. A combination of elongation-mediated and hybridization-mediated detection methodologies is useful in discriminating between highly homologous sequences of mRNAs, while simultaneously determining respective abundances of these messages in a highly parallel format of analysis. The detection assay was performed as follows.","First, the RT reaction was performed on the processed total RNA samples using specific RT primer (highlighted in yellow) to convert mRNAs of interest into Cy3-labeled cDNAs. Seven cDNA targets were hybridized on a BeadChip to a perfectly matched capture\/elongation probe. The probes are designed such that the 3\u2032-end of each probe aligns with each unique polymorphic position in the targets. The matched hybridized probes were elongated using TAMRA-labeled dCTP. Therefore, elongated probes would emit a fluorescent signal.","A more complicated case of sequence discrimination, involving two sequences having a common mutation, but only one having a second specific mutation is illustrated in . Specifically, genes 16 and 31 have the same mutation T (replacing C), that discriminates them from all the other sequences in multiple sequence alignment (not shown). Gene 31 is detected using a second specific capture\/elongation probe to discriminate a unique mutation C (replacing G). However, gene 16 does have another specific mutation which permits its identification in a pool of 7 closely homologous sequences by a \u201cphasing\u201d design. As depicted in detail in , in order to ensure discrimination, this design calls for three steps; steps 1 and 2 occur simultaneously.","Step 1: Probe 16, with T at the 3\u2032-end, was immobilized on bead type 1 and placed under annealing conditions in contact with a pool of 7 amplified gene transcripts. Elongation following hybridization discriminated two genes, 16 and 31, from the other sequences in the pool, as detected by the TMRA fluorescence from beads carrying the probes. Simultaneously, probe 31, with C at the 3\u2032-end, was immobilized on another bead type and placed in hybridizing conditions with a pool of 7 amplified gene transcripts. An elongation reaction followed hybridization, and gene 31 was detected by TMRA fluorescence from a particular encoded bead type.\n\nStep 2: The next stage of the assay is removal of the target  from the elongated probe 16, by a denaturation reaction at 95\u00b0 C.\n\nStep 3: The single-stranded elongated probe 16 is then hybridized with a short Cy5-labeled detection probe 16 at the melting temperature of the duplex formation (Tm=49\u00b0 C.) using a matched probe with C in the middle of the sequence. If hybridization at the indicated melting temperature (Tm) occurs, and therefore Cy5 fluorescence is detected on beads of type 1, this indicates that gene 16 is present in the pool. Thus, in this design, a TMRA signal recorded from the bead type carrying probe 31 confirms the presence of gene31 and a TMRA signal recorded with subsequent Cy5 signal from the bead type carrying probe 16 confirms the presence of gene 16.\n","It should be understood that the terms, expressions and examples used herein are exemplary only and not limiting and that the scope of the invention is defined only in the claims which follow, and includes all equivalents of the subject matter of the claims. All steps in method claims can be performed in any order, including that set forth in the claims, unless otherwise stated in the claims."],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"BRIEF DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0024","num":"0039"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0025","num":"0040"},"figref":["FIG. 2","FIG. 1"]},{"@attributes":{"id":"p-0026","num":"0041"},"figref":"FIG. 3A"},{"@attributes":{"id":"p-0027","num":"0042"},"figref":["FIG. 3B","FIG. 3A"],"sub":"o"},{"@attributes":{"id":"p-0028","num":"0043"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0029","num":"0044"},"figref":"FIG. 5"},{"@attributes":{"id":"p-0030","num":"0045"},"figref":"FIG. 6A"},{"@attributes":{"id":"p-0031","num":"0046"},"figref":["FIG. 6B","FIG. 6A"],"sub":"o"},{"@attributes":{"id":"p-0032","num":"0047"},"figref":"FIGS. 7A, 7B, 7C"},{"@attributes":{"id":"p-0033","num":"0048"},"figref":"FIG. 8A"},{"@attributes":{"id":"p-0034","num":"0049"},"figref":"FIG. 8B"},{"@attributes":{"id":"p-0035","num":"0050"},"figref":"FIG. 9"},{"@attributes":{"id":"p-0036","num":"0051"},"figref":["FIG. 10A","FIG. 8"]},{"@attributes":{"id":"p-0037","num":"0052"},"figref":"FIG. 10B"},{"@attributes":{"id":"p-0038","num":"0053"},"figref":"FIG. 10C"},{"@attributes":{"id":"p-0039","num":"0054"},"figref":["FIG. 11","FIG. 10"]},{"@attributes":{"id":"p-0040","num":"0055"},"figref":"FIG. 12A"},{"@attributes":{"id":"p-0041","num":"0056"},"figref":"FIG. 12B"},{"@attributes":{"id":"p-0042","num":"0057"},"figref":["FIG. 13","FIG. 12"]},{"@attributes":{"id":"p-0043","num":"0058"},"figref":"FIG. 14"},{"@attributes":{"id":"p-0044","num":"0059"},"figref":"FIG. 15"},{"@attributes":{"id":"p-0045","num":"0060"},"figref":"FIG. 16A"},{"@attributes":{"id":"p-0046","num":"0061"},"figref":"FIG. 16B"},{"@attributes":{"id":"p-0047","num":"0062"},"figref":"FIG. 17"},{"@attributes":{"id":"p-0048","num":"0063"},"figref":"FIG. 18"},{"@attributes":{"id":"p-0049","num":"0064"},"figref":"FIG. 19A"},{"@attributes":{"id":"p-0050","num":"0065"},"figref":"FIG. 19B"},{"@attributes":{"id":"p-0051","num":"0066"},"figref":["FIG. 19C","FIG. 19B"]},{"@attributes":{"id":"p-0052","num":"0067"},"figref":"FIG. 20A"},{"@attributes":{"id":"p-0053","num":"0068"},"figref":"FIG. 20B"},{"@attributes":{"id":"p-0054","num":"0069"},"figref":["FIG. 21","FIG. 19"]},{"@attributes":{"id":"p-0055","num":"0070"},"figref":"FIG. 22"},{"@attributes":{"id":"p-0056","num":"0071"},"figref":["FIG. 23","FIG. 19"]},{"@attributes":{"id":"p-0057","num":"0072"},"figref":"FIG. 24A"},{"@attributes":{"id":"p-0058","num":"0073"},"figref":["FIG. 24B","FIG. 24A"]},{"@attributes":{"id":"p-0059","num":"0074"},"figref":"FIG. 25A"},{"@attributes":{"id":"p-0060","num":"0075"},"figref":"FIG. 25B"},{"@attributes":{"id":"p-0061","num":"0076"},"figref":["FIG. 25C","FIG. 25A","FIG. 25B"]},{"@attributes":{"id":"p-0062","num":"0077"},"figref":"FIG. 26"},{"@attributes":{"id":"p-0063","num":"0078"},"figref":"FIG. 27"},{"@attributes":{"id":"p-0064","num":"0079"},"figref":"FIG. 28"},{"@attributes":{"id":"p-0065","num":"0080"},"figref":["FIG. 29","FIG. 28"]},{"@attributes":{"id":"p-0066","num":"0081"},"figref":"FIG. 30"}]},"DETDESC":[{},{}]}
