---
title: Methods and compositions for improved fertilization and embryonic survival
abstract: Single nucleotide polymorphic sites of the bovine HSP genes are associated with improved fertilization rate and/or improved embryo survival rate in cattle. Nucleic acid molecules, kits, methods of genotyping and marker assisted bovine breeding methods based on these SNPs are disclosed.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=09322068&OS=09322068&RS=09322068
owner: Wisconsin Alumni Research Foundation
number: 09322068
owner_city: Madison
owner_country: US
publication_date: 20140619
---

{"@attributes":{"id":"description"},"RELAPP":[{},{}],"heading":["CROSS REFERENCE TO RELATED APPLICATIONS","STATEMENT OF GOVERNMENT INTEREST","FIELD OF THE INVENTION","BACKGROUND OF THE INVENTION","SUMMARY OF THE INVENTION","DETAILED DESCRIPTION OF THE INVENTION","EXAMPLES","REFERENCES CITED"],"p":["This is a divisional application of U.S. application Ser. No. 13\/563,750 filed on Aug. 1, 2012 claiming priority to U.S. Patent Application No. 61\/514,030, filed on Aug. 2, 2011, the entire disclosure of which is incorporated herein by reference.","This invention was made with government support under 09-CRHF-0-6055 awarded by the USDA\/NIFA. The government has certain rights in the invention.","The present invention relates to a method of genetic testing for improved fertilization rate and embryonic survival rate in animals, especially dairy cattle, based on single nucleotide polymorphisms (SNPs) in genes encoding the heat shock proteins (HSP).","Poor reproductive performance, in particular in high-producing dairy cows, is a major problem on dairy farms throughout the world and has been identified as the single most important problem in dairy herd management in many countries (Royal et al., 2000; Dobson et al., 2008). In addition to direct financial losses, infertility can result in increased management complexity, for example, an inability to achieve a compact calving pattern, which is of critical importance in maximizing milk production from grazed grass in seasonal production systems.","This is of particular interest in cattle because of declining fertility over the past few decades (Dobson et al., 2007; Leroy et al., 2008). Furthermore, recent studies have shown that low fertilization rates and embryonic loss seem to be the main factors in dairy cattle infertility (Santos et al., 2004; Morris and Diskin, 2008).","While reproductive performance is influenced by a large number of factors, low fertilization rate and early embryonic loss are the primary factors contributing to poor reproductive performance in dairy cattle (Santos et al., 2004; Morris and Diskin, 2008). Enormous efforts, such as animal breeding and artificial insemination, have been and continue to be invested in ensuring adequate fertility in the cattle herd. Typically, artificial insemination in dairy cattle is successful only 30-35% of the time. The reasons for this are not clear. However, it is understood that both biological and environmental factors affect fertility rate. Some environmental factors such as high temperature, and lack of precipitation can cause stress in cattle and can drop the fertility rate to 10-15%. Commercial artificial insemination operations often shut down in July and August due to the drop in fertility caused by the hot, dry weather.","Genetics is also a prominent factor in fertility, and accounts for about one-third of the decline in pregnancy rate of dairy cows (Shook, 2006). In particular, identifying highly fertile bulls has been a time-consuming and expensive process. It can take 5-10 years of tracking the attempts of artificial insemination using semen from a bull before it can be certified as a quality bull.","Marker-assisted selection, on the other hand, can lower the high cost and reduce the extended time commitment of progeny testing currently used to improve sires, since young bull progeny could be evaluated immediately after birth or even prior to birth for the presence\/absence of the marker, and young bulls that are determined by genetic testing to have undesirable markers would never be progeny tested.","There is thus a need for a method of genetically evaluating the bulls, as well as the cows, e.g., by genetic testing, to enable a quick and accurate evaluation of its fertility as well as the survival rate of embryos conceived therefrom.","Heat shock proteins (HSPs) are among the most highly conserved proteins in nature and have been found in all organisms studied from bacteria to humans (Becker and Craig, 1994). The structure and roles of HSPs as molecular chaperones in the folding, transport, and assembly of proteins as well as in protecting the cell under different stress conditions have been extensively studied and reported in the scientific literature (Becker and Craig, 1994; Nollen and Morimoto, 2002; Qiu et al., 2006). The fact that HSPs are essential in the folding, stability, and cellular localization of newly synthesized proteins implies key roles of these proteins in apoptosis, cell differentiation, and regulation of the embryo cell cycle (Luft and Dix, 1999; Lanneau et al., 2007). Strong evidence which has accumulated on the expression of HSPs during spermatogenesis, oogenesis, and embryogenesis suggests they have important functions in fertilization and during the pre-implantation period (Neuer et al., 1999). Mouse embryos cultured with monoclonal antibodies to HSPs have been found to display a significantly reduced blastocyst rate (Neuer et al., 1998). Al-Katanani and Hansen (2002) reported that the addition of antibodies for the induced form of HSP70 reduced blastocyst rate of cattle embryos, suggesting that HSP70 is involved in proper embryonic development. Matwee and colleagues (2001) showed that fertilization\u2014measured as the number of permatozoa tightly bound to a zona pellucida\u2014and embryo development in cattle were significantly affected by the presence of different concentrations of anti-HSP70.","Although there is strong evidence in the literature on the roles of HSPs in early embryonic development, most of the studies have focused on the mouse and only a few HSP genes have been studied in cattle embryos. No evidence existed that any polymorphism in HSPs, if existed, is related to dairy cattle fertility in any way.","The present inventor investigated the expression profiles of HSPs and their splice variants in bovine embryos (degenerates vs. blastocysts), and also analyzed the association of these profiles with fertility traits. Some splice variants showed differential expression between degenerates and blastocysts while others were not expressed at all in embryos, implying different functions of these transcripts in embryonic development. Among the HSPs investigated, DNAJC15, DNAJC19, DNAJC24 and DNAJC27, all of the HSP40 family, had the most significant expression differential. They were further investigated for association with fertility and development traits. Single nucleotide polymorphisms (SNP) in DNAJC15 and DNAJC27 were found to be associated with blastocyst rate and fertilization rate, respectively.","Accordingly, the present invention provides an isolated nucleic acid molecule comprising at least one polymorphic site selected from the group consisting of positions 85146, 85161, 85216, 85292, and 85300 of the nucleic acid sequence shown in  (SEQ ID NO: 1) (part of the DNAJC15 gene), and at least 10 contiguous nucleotides of the sequence shown in , wherein position 85146 is guanine, position 85161 is guanine, position 85216 is adenosine, position 85292 is cytosine, or position 85300 is guanine; or an isolated nucleic acid molecule comprising a polymorphic position selected from the group consisting of positions 35728, 36016, and 38867 of the nucleic acid sequence shown in  (SEQ ID NO: 2) (part of the DNAJC27 gene), and at least 10 contiguous nucleotides of the sequence shown in , wherein position 35728 is guanine, position 36016 is guanine, or position 38867 is guanine. It is recognized that SEQ ID NO: 1 is already known, and the nucleic acid molecule therefore does not encompass one that consists of SEQ ID NO: 1.","Preferably, the nucleic acid molecule which comprises at least 15, more preferably at least 20, still more preferably at least 25, contiguous bases of SEQ ID NO: 1 or SEQ ID NO: 2 adjacent to the polymorphic site. In one embodiment, the isolated nucleic acid molecule comprises not more than 1,500 nt, preferably not more than 1000 nt, more preferably not more than 900 nt, more preferably not more than 800 nt, more preferably not more than 700 nt, preferably not more than 600 nt, more preferably not more than 500 nt, preferably not more than 400 nt, more preferably not more than 300 nt, more preferably not more than 150 nt., preferably not more than 100 nt., still more preferably not more than 50 nt.","The nucleic acid molecule preferably contains the polymorphic site which is within 4 nucleotides of the center of the nucleic acid molecule. Preferably, the polymorphic site is at the center of the nucleic acid molecule.","In another embodiment, the nucleic acid molecule contains the polymorphic site which is at the 3\u2032-end of the nucleic acid molecule.","In another embodiment, the nucleic acid molecule contains the polymorphic site which is at the 5\u2032-end of the nucleic acid molecule.","The present invention also provides an array of nucleic acid molecules comprising at least two nucleic acid molecules described above.","The present invention further provides a kit comprising a nucleic acid molecule described above, and a suitable container.","Also provided is a method for detecting single nucleotide polymorphism (SNP) in the bovine DNAJC15 or DNAJC27 genes, wherein the DNAJC15 gene is partially shown in , and the DNAJC27 gene is partially shown in , the method comprising optionally isolating a nucleic acid sample from the bovine animal or a tissue sample therefrom, determining the identity of a nucleotide of at least one position selected from the group consisting of a first polymorphic position comprising positions 85146, 85161, 85216, 85292, and 85300 of SEQ ID NO: 1, and a second polymorphic position comprising positions 35728, 36016, and 38867 of SEQ ID NO: 2, and comparing the nucleotide identity of the position respectively to the nucleotide identity at a corresponding position of SEQ ID NO: 1 or SEQ ID NO: 2.","Also provided is a method for genotyping a bovine cell, comprising obtaining a nucleic acid sample from said cell and determining the identity of a nucleotide of at least one position selected from the group consisting of positions 85146, 85161, 85216, 85292, and 85300 of SEQ ID NO: 1, and positions 35728, 36016, and 38867 of SEQ ID NO: 2. In one embodiment, the bovine cell is an adult cell, an embryo cell, a sperm, an egg, a fertilized egg, or a zygote. In one embodiment, both copies of the gene in the cell are genotyped.","The present invention further provides a method for progeny testing of cattle, the method comprising collecting a nucleic acid sample from said progeny, and genotyping said nucleic sample.","In another embodiment, a method is provided for selectively breeding cattle using a multiple ovulation and embryo transfer procedure (MOET), the method comprising superovulating a female animal, collecting eggs from said superovulated female, in vitro fertilizing said eggs from a suitable male animal, implanting said fertilized eggs into other females allowing for an embryo to develop, and genotyping said developing embryo, and terminating pregnancy if the developing embryo does not have at least one polymorph selected from the group consisting of a first polymorph selected from the group consisting of guanine at position 85146, guanine at position 85161, adenosine at position 85216, cytosine at position 85292, guanine at position 85300 of SEQ ID NO: 1; and a second polymorph selected from the group consisting of guanine at position 35728, guanine at position 36016, and guanine at position 38867 of SEQ ID NO: 2.","In one embodiment, pregnancy is terminated if the developing embryo does not have either the first polymorph or the second polymorph.","In another embodiment, pregnancy is terminated if the developing embryo is not homozygous in the first polymorph and the second polymorph.","Further provided is a method for selecting a cattle as a breeder, wherein the cattle is genotyped according to the present invention, and the animal is only selected for breeding purpose if it comprises at least a first polymorph or a second polymorph, wherein the first polymorph is selected from guanine at position 85146, guanine at position 85161, adenosine at position 85216, cytosine at position 85292, guanine at position 85300 of SEQ ID NO: 1; and the second polymorph is selected from the group consisting of guanine at position 35728, guanine at position 36016, and guanine at position 38867 of SEQ ID NO: 2. In one embodiment, the cattle animal is selected only if it comprises both the first and second polymorphs. In another embodiment, the cattle animal is selected only if it is homozygous with regard to both the first and second polymorphs.","The present inventor has constructed an in-vitro fertilization (IVF) system which has the advantages of a unified environment and well-isolated components of the embryonic development process, and used this system to characterize genetic factors involved in embryonic loss, and hence infertility, and to associate candidate genes and pathways with fertilization rate and embryonic survival at both the genomic and gene expression levels (Khatib et al., 2008a,b; Khatib et al., 2009; Huang et al., 2010a,b).","As indicated above, HSPs are among the first proteins produced during embryonic development (Neuer et al., 1999). The present inventor hypothesized that determining expression patterns of these genes in developed vs. arrested embryos could lead to the identification of specific genes involved in reproductive success. The high structural and functional conservation of HSPs during evolution suggests crucial roles of these proteins in fertilization, embryo development, and thus fertility in cattle. Also, some HSPs are considered housekeeping genes that are essential for many cell functions.","In efforts leading to the present invention, the expression levels of HSP genes in biological replicate bovine embryo pools that differ in their morphology and developmental statuses were evaluated, and associations of polymorphisms with fertility traits were examined. The HSP genes were found to be differentially expressed between blastocysts and degenerate embryos.","SNPs in two HSP genes were found to be associated with fertility traits. Specifically, a total of 17 candidate genes were chosen based on differential expression results observed in a previous microarray analysis of bovine embryos. To test whether or not these genes have roles in early development of cattle embryos, their expression levels were quantified and compared between blastocysts and degenerates. All 17 genes showed expression differences that ranged between 1.5- and 7.6-fold between the embryo groups. Two of these bovine HSP genes, DNAJC15 and DNAJC27, both of the HSP40 gene family, have several polymorphic positions, and these polymorphisms are associated with fertilization and blastocyst rates. Furthermore, all SNPs in each of DNAJC15 and DNAJC27 genes showed high linkage disequilibrium with each other.","The term \u201cpolymorphic\u201d or \u201cpolymorphism\u201d as used herein refers to the occurrence of two or more alternative genomic sequences or alleles between or among different genomes or individuals. \u201cPolymorphic\u201d refers to the condition in which two or more variants of a specific genomic sequence can be found in a population. A \u201cpolymorphic site\u201d is the locus at which the variation occurs. Polymorphisms generally have at least two alleles, each occurring at a significant frequency in a selected population. A polymorphic locus may be as small as one base pair. The first identified allelic form is arbitrarily designated as the reference form, and other allelic forms are designated as alternative or variant alleles. The allelic form occurring most frequently in a selected population is sometimes referred to as the wild type form. Diploid organisms may be homozygous or heterozygous for allelic forms. A biallelic polymorphism has two forms, and a triallelic polymorphism has three forms, and so on.","Polymorphisms may provide functional differences in the genetic sequence, through changes in the encoded polypeptide, changes in mRNA stability, binding of transcriptional and translation factors to the DNA or RNA, and the like. Polymorphisms are also used to detect genetic linkage to phenotypic variation.","One type of polymorphism, single nucleotide polymorphisms (SNPs), has gained wide use for the detection of genetic linkage recently. SNPs are generally biallelic systems, that is, there are two alleles that an individual may have for any particular SNP marker. In the instant case, the SNPs are used for determining the genotypes of the two HSP genes, which are found to have strong correlation to fertilization rate and embryonic survival.","The provided sequences also encompass the complementary sequence corresponding to any of the provided polymorphisms. In order to provide an unambiguous identification of the specific site of a polymorphism, the numbering of the original sequence in the GenBank is shown in  and is used throughout this disclosure.","The present invention provides nucleic acid based genetic markers for identifying bovine animals with superior breeding (such as fertility and embryo survival rates) traits. In general, for use as markers, nucleic acid fragments, preferably DNA fragments, may be as short as 7 nucleotides (nt), but may preferably be at least 12 nt, 15 nt, usually at least 20 nt, often at least 50 nt. Such small DNA fragments are useful as primers for the polymerase chain reaction (PCR), and\/or probes for hybridization screening, etc.","The term primer refers to a single-stranded oligonucleotide capable of acting as a point of initiation of template-directed DNA synthesis under appropriate conditions (i.e., in the presence of four different nucleoside triphosphates and an agent for polymerization, such as, DNA or RNA polymerase or reverse transcriptase) in an appropriate buffer and at a suitable temperature. The appropriate length of a primer depends on the intended use of the primer but typically ranges from 15 to 30 nucleotides. Short primer molecules generally require cooler temperatures to form sufficiently stable hybrid complexes with the template. A primer need not reflect the exact sequence of the template but must be sufficiently complementary to hybridize with the template. The term primer site, or priming site, refers to the area of the target DNA to which a primer hybridizes. The term primer pair means a set of primers including a 5\u2032 upstream primer that hybridizes with the 5\u2032 end of the DNA sequence to be amplified and a 3\u2032, downstream primer that hybridizes with the complement of the 3\u2032 end of the sequence to be amplified.","The term \u201cprobe\u201d or \u201chybridization probe\u201d denotes a defined nucleic acid segment (or nucleotide analog segment) which can be used to identify by hybridizing to a specific polynucleotide sequence present in samples, said nucleic acid segment comprising a nucleotide sequence complementary of the specific polynucleotide sequence to be identified. \u201cProbes\u201d or \u201chybridization probes\u201d are nucleic acids capable of binding in a base-specific manner to a complementary strand of nucleic acid.","An objective of the present invention is SNP genotyping, that is, to determine which embodiment of the SNP polymorphisms a specific sample of DNA has. For example, it is desirable to determine whether the nucleotide at a particular position is A or C. Many references describe genotyping methods, such as Chen et al., \u201cSingle nucleotide polymorphism genotyping: biochemistry, protocol, cost and throughput\u201d, Pharmacogenomics J. 2003; 3(2):77-96; Kwok et al., \u201cDetection of single nucleotide polymorphisms\u201d, Curr Issues Mol. Biol. April 2003; 5(2):43-60; Shi, \u201cTechnologies for individual genotyping: detection of genetic polymorphisms in drug targets and disease genes\u201d, Am J. Pharmacogenomics. 2002; 2(3):197-205; and Kwok, \u201cMethods for genotyping single nucleotide polymorphisms\u201d, Annu Rev Genomics Hum Genet. 2001; 2:235-58. Exemplary techniques for high-throughput SNP genotyping are described in Marnellos, \u201cHigh-throughput SNP analysis for genetic association studies\u201d, Curr Opin Drug Discov Devel. May 2003; 6(3):317-21. Common SNP genotyping methods include, but are not limited to, TaqMan assays and modifications thereof such as Molecular Beacon assays, SNPlex platforms, Bio-Plex system, CEQ and SNPstream systems, Molecular Inversion Probe array technology, BeadArray Technologies (e.g., Illumina GoldenGate and Infinium assays), single stranded conformation polymorphism assays (SSCP), molecular beacon assays, nucleic acid arrays, allele-specific primer extension, allele-specific PCR, arrayed primer extension, homogeneous primer extension assays, primer extension with detection by mass spectrometry, pyrosequencing, multiplex primer extension sorted on genetic arrays, ligation with rolling circle amplification, homogeneous ligation, OLA (U.S. Pat. No. 4,988,167), multiplex ligation reaction sorted on genetic arrays, restriction-fragment length polymorphism, single base extension-tag assays, and the Invader assay. Such methods may be used in combination with detection mechanisms such as, for example, luminescence or chemiluminescence detection, fluorescence detection, time-resolved fluorescence detection, fluorescence resonance energy transfer, fluorescence polarization, mass spectrometry, and electrical detection.","The sequence neighboring the SNP site can be used to design SNP detection reagents such as oligonucleotide probes, which may optionally be implemented in a kit format. Preferably, the oligonucleotide probe will have a detectable label, and contains for example an A at the corresponding position. Experimental conditions can be chosen such that if the sample DNA contains an A, a hybridization signal can be detected because the probe hybridizes to the corresponding complementary DNA strand in the sample, while if the sample DNA contains a G, no hybridization signal is detected.","Similarly, PCR primers and conditions can be devised, whereby the oligonucleotide is used as one of the PCR primers, for analyzing nucleic acids for the presence of a specific sequence. These may be direct amplification of the genomic DNA, or RT-PCR amplification of the mRNA transcript of the target gene. The use of the polymerase chain reaction is described in Saiki et al. (1985) Science 230:1350-1354. Amplification may be used to determine whether a polymorphism is present, by using a primer that is specific for the polymorphism. Alternatively, various methods are known in the art that utilize oligonucleotide ligation as a means of detecting polymorphisms, for examples see Riley et al (1990) Nucleic Acids Res. 18:2887-2890; and Delahunty et al (1996) Am. J. Hum. Genet. 58:1239-1246. The detection method may also be based on direct DNA sequencing, or hybridization, or a combination thereof. Where large amounts of DNA are available, genomic DNA is used directly. Alternatively, the region of interest is cloned into a suitable vector and grown in sufficient quantity for analysis. The nucleic acid may be amplified by PCR, to provide sufficient amounts for analysis.","Hybridization may be performed in solution, or such hybridization may be performed when either the oligonucleotide probe or the target polynucleotide is covalently or noncovalently affixed to a solid support. Attachment may be mediated, for example, by antibody-antigen interactions, poly-L-Lys, streptavidin or avidin-biotin, salt bridges, hydrophobic interactions, chemical linkages, UV cross-linking baking, etc. Oligonucleotides may be synthesized directly on the solid support or attached to the solid support subsequent to synthesis. Solid-supports suitable for use in detection methods of the invention include substrates made of silicon, glass, plastic, paper and the like, which may be formed, for example, into wells (as in 96-well plates), slides, sheets, membranes, fibers, chips, dishes, and beads. The solid support may be treated, coated or derivatized to facilitate the immobilization of the allele-specific oligonucleotide or target nucleic acid. For screening purposes, hybridization probes of the polymorphic sequences may be used where both forms are present, either in separate reactions, spatially separated on a solid phase matrix, or labeled such that they can be distinguished from each other.","Hybridization may also be performed with nucleic acid arrays and subarrays such as described in WO 95\/11995. The arrays would contain a battery of allele-specific oligonucleotides representing each of the polymorphic sites. One or both polymorphic forms may be present in the array, for example the polymorphism of a SNP position may be represented by either, or both, of the listed nucleotides. Usually such an array will include at least 2 different polymorphic sequences, i.e. polymorphisms located at unique positions within the locus, and may include all of the provided polymorphisms. Arrays of interest may further comprise sequences, including polymorphisms, of other genetic sequences, particularly other sequences of interest. The oligonucleotide sequence on the array will usually be at least about 12 nt in length, may be the length of the provided polymorphic sequences, or may extend into the flanking regions to generate fragments of 100 to 200 nt in length. For examples of arrays, see Ramsay (1998) Nat. Biotech. 16:4044; Hacia et al. (1996) Nature Genetics 14:441-447; Lockhart et al. (1996) Nature Biotechnol. 14:1675-1680; and De Risi et al. (1996) Nature Genetics 14:457-460.","The identity of polymorphisms may also be determined using a mismatch detection technique, including but not limited to the RNase protection method using riboprobes (Winter et al., Proc. Natl. Acad. Sci. USA 82:7575, 1985; Meyers et al., Science 230:1242, 1985) and proteins which recognize nucleotide mismatches, such as the mutS protein (Modrich, P. Ann. Rev. Genet. 25:229-253, 1991). Alternatively, variant alleles can be identified by single strand conformation polymorphism (SSCP) analysis discussed above (Orita et al., Genomics 5:874-879, 1989; Humphries et al., in Molecular Diagnosis of Genetic Diseases, R. Elles, ed., pp. 321-340, 1996) or denaturing gradient gel electrophoresis (DGGE) (Wartell et al., Nucl. Acids Res. 18:2699-2706, 1990; Sheffield et al., Proc. Natl. Acad. Sci. USA 86:232-236, 1989).","A polymerase-mediated primer extension method may also be used to identify the polymorphism(s). Several such methods have been described in the patent and scientific literature and include the \u201cGenetic Bit Analysis\u201d method (WO92\/15712) and the ligase\/polymerase mediated genetic bit analysis (U.S. Pat. No. 5,679,524). Related methods are disclosed in WO91\/02087, WO90\/09455, WO95\/17676, U.S. Pat. Nos. 5,302,509, and 5,945,283. Extended primers containing a polymorphism may be detected by mass spectrometry as described in U.S. Pat. No. 5,605,798. Another primer extension method is allele-specific PCR (Ruao et al., Nucl. Acids Res. 17:8392, 1989; Ruao et al., Nucl. Acids Res. 19, 6877-6882, 1991; WO 93\/22456; Turki et al., J. Clin. Invest. 95:1635-1641, 1995). In addition, multiple polymorphic sites may be investigated by simultaneously amplifying multiple regions of the nucleic acid using sets of allele-specific primers as described in Wallace et al. (WO 89\/10414).","A detectable label may be included in an amplification reaction. Suitable labels include fluorochromes, e.g. fluorescein isothiocyanate (FITC), rhodamine, Texas Red, phycoerythrin, allophycocyanin, 6-carboxyfluorescein (6-FAM), 2\u2032,7\u2032-dimethoxy-4\u2032,5\u2032-dichloro-6-carboxyfluorescein (JOE),6-carboxy-X-rhodamine (ROX), 6-carboxy-2\u2032,4\u2032,7\u2032,4,7-hexachlorofluorescein (HEX), 5-carboxyfluorescein (5-FAM) or N,N,N\u2032,N\u2032-tetramethyl-6-carboxyrhodamine (TAMRA), radioactive labels, e.g. P, S, H; etc. The label may be a two stage system, where the amplified DNA is conjugated to biotin, haptens, etc. having a high affinity binding partner, e.g. avidin, specific antibodies, etc., where the binding partner is conjugated to a detectable label. The label may be conjugated to one or both of the primers. Alternatively, the pool of nucleotides used in the amplification is labeled, so as to incorporate the label into the amplification product.","It is readily recognized by those ordinarily skilled in the art that in order to maximize the signal to noise ratio, in probe hybridization detection procedure, the polymorphic site should at the center of the probe fragment used, whereby a mismatch has a maximum effect on destabilizing the hybrid molecule; and in a PCR detection procedure, the polymorphic site should be placed at the very 3\u2032-end of the primer, whereby a mismatch has the maximum effect on preventing a chain elongation reaction by the DNA polymerase. The location of nucleotides in a polynucleotide with respect to the center of the polynucleotide are described herein in the following manner. When a polynucleotide has an odd number of nucleotides, the nucleotide at an equal distance from the 3\u2032 and 5\u2032 ends of the polynucleotide is considered to be \u201cat the center\u201d of the polynucleotide, and any nucleotide immediately adjacent to the nucleotide at the center, or the nucleotide at the center itself is considered to be \u201cwithin 1 nucleotide of the center.\u201d With an odd number of nucleotides in a polynucleotide any of the five nucleotides positions in the middle of the polynucleotide would be considered to be within 2 nucleotides of the center, and so on. When a polynucleotide has an even number of nucleotides, there would be a bond and not a nucleotide at the center of the polynucleotide. Thus, either of the two central nucleotides would be considered to be \u201cwithin 1 nucleotide of the center\u201d and any of the four nucleotides in the middle of the polynucleotide would be considered to be \u201cwithin 2 nucleotides of the center,\u201d and so on.","In some embodiments, a composition contains two or more differently labeled oligonucleotides for simultaneously probing the identity of nucleotides or nucleotide pairs at two or more polymorphic sites. It is also contemplated that primer compositions may contain two or more sets of allele-specific primer pairs to allow simultaneous targeting and amplification of two or more regions containing a polymorphic site.","Alternatively, the relevant portion of the target gene of the sample of interest may be amplified via PCR and directly sequenced, and the sequence be compared to the wild type sequence. It is readily recognized that, other than those specifically disclosed herein, numerous primers can be devised to achieve the objectives. PCR and sequencing techniques are well known in the art and reagents and equipments are readily available commercially.","The so-called next generation sequencing, and high-throughput sequencing methods, may also be sued. For example, Massively Parallel Signature Sequencing (MPSS); Polony sequencing, pyro sequencing, SOLiD sequencing; ion semiconductor sequencing, and DNA nanoball sequencing systems are well-known to those skilled in the art.","DNA markers have several advantages; segregation is easy to measure and is unambiguous, and DNA markers are co-dominant, i.e., heterozygous and homozygous animals can be distinctively identified. Once a marker system is established selection decisions could be made very easily, since DNA markers can be assayed any time after a sample is collected from the individual animal, or even earlier by testing embryos in vitro if very early embryos are collected. The use of marker assisted genetic selection will greatly facilitate and speed up cattle breeding problems. For example, a modification of the multiple ovulation and embryo transfer (MOET) procedure can be used with genetic marker technology. Specifically, females are superovulated, eggs are collected, in vitro fertilized using semen from superior males and implanted into other females allowing for use of the superior genetics of the female (as well as the male) without having to wait for her to give birth to one calf at a time. Developing blastomeres at the 4-8 cell stage may be assayed for presence of the marker, and selection decisions made accordingly.","In one embodiment of the invention an assay is provided for detection of presence of a desirable genotype using the markers.","The term \u201cgenotype\u201d as used herein refers to the identity of the alleles present in an individual or a sample. In the context of the present invention a genotype preferably refers to the description of the polymorphic alleles present in an individual or a sample. The term \u201cgenotyping\u201d a sample or an individual for a polymorphic marker refers to determining the specific allele or the specific nucleotide carried by an individual at a polymorphic marker.","The present invention is suitable for identifying a bovine, including a young or adult bovine animal, an embryo, a semen sample, an egg, a fertilized egg, or a zygote, or other cell or tissue sample therefrom, to determine whether said bovine possesses the desired genotypes of the present invention, some of which are indicative of improved fertilization rate and embryonic survival.","Further provided is a method for genotyping the bovine HSP gene, comprising determining for the two copies of the HSP gene present the identity of the nucleotide pair at positions 25402 and 19069.","One embodiment of a genotyping method of the invention involves examining both copies of the relevant HSP gene, or a fragment thereof, to identify the nucleotide pair at the polymorphic site in the two copies to assign a genotype to the individual. In some embodiments, \u201cexamining a gene\u201d may include examining one or more of: DNA containing the gene, mRNA transcripts thereof, or cDNA copies thereof. As will be readily understood by the skilled artisan, the two \u201ccopies\u201d of a gene, mRNA or cDNA, or fragment thereof in an individual may be the same allele or may be different alleles. In another embodiment, a genotyping method of the invention comprises determining the identity of the nucleotide pair at the polymorphic site.","The present invention further provides a kit for genotyping a bovine sample, the kit comprising in a container a nucleic acid molecule, as described above, designed for detecting the polymorphism, and optionally at least another component for carrying out such detection. Preferably, a kit comprises at least two oligonucleotides packaged in the same or separate containers. The kit may also contain other components such as hybridization buffer (where the oligonucleotides are to be used as a probe) packaged in a separate container. Alternatively, where the oligonucleotides are to be used to amplify a target region, the kit may contain, preferably packaged in separate containers, a polymerase and a reaction buffer optimized for primer extension mediated by the polymerase, such as PCR.","In one embodiment the present invention provides a breeding method whereby genotyping as described above is conducted on bovine embryos, and based on the results, certain cattle are either selected or dropped out of the breeding program.","Through use of the linked marker loci, procedures termed \u201cmarker assisted selection\u201d (MAS) may be used for genetic improvement within a breeding nucleus; or \u201cmarker assisted introgression\u201d for transferring useful alleles from a resource population to a breeding nucleus (Soller 1990; Soller 1994).","Testing of Roles of 17 HSP Genes","The roles of 17 HSP genes in early embryonic development and fertility in cattle at both genomic and gene expression levels were investigated. These genes include 8 HSP40 genes, six HSP70 genes, one HSPP, and 2 HSP10 genes. (see Table 1) These genes were chosen based on differential expression results observed in a previous microarray analysis of bovine embryos; and their expression levels were quantified and compared between blastocysts and degenerates to test whether or not these genes have roles in early development of cattle embryos.","All 17 genes showed expression differences that ranged between 1.5- and 7.6-fold between the embryo groups. All HSP40 family genes were found to be upregulated in degenerate embryos compared to blastocysts. Although the specific functions of HSP40s have not been reported in cattle, studies from other species have shown that these genes play important roles in protecting cells under stress conditions (Lanneau et al., 2007). It is well established that ATP hydrolysis is necessary for protein folding activity of HSP70s and that HSP40s stimulate ATPase activity and stabilize the interaction of HSP70s with their substrates (Qiu et al., 2006). Gotoh and colleagues (2004) reported that HSP40 combines with HSP70 to act as chaperones to protect cells from apoptosis. Degenerate embryos in the IVF system are growth-arrested embryos and seem to undergo partial apoptosis. Research has shown that HSPs are upregulated to prevent apoptosis triggered by different stimuli by interacting with key factors of the apoptotic signaling pathways (Lanneau et al., 2007). Therefore, it is feasible that upregulation of HSP40 genes observed in degenerates is a response to stress on these embryos so they can maintain protein homeostasis.","The qRT-PCR analysis revealed moderate fold change expression of HSPE1, HSPH1, HSB1, and six HSP70 genes (HSPA2, HSPA4, HSPA5, HSPA8, HSPA9 and HSPA14). HSPA2, HSPA4, HSPA5, and HSPA8 showed higher expression in blastocysts than degenerates, whereas expression levels of HSPA9 and HSPA14 were higher in degenerate embryos. Although the specific functions of most HSP70s have not been reported in cattle, studies in human and mouse have indicated important roles for these genes in early embryogenesis. It has been reported that Hspa2 is essential for normal spermatogenesis and for growth and survival of cancer cells and that Hspa8 knockout mice are not viable because of the housekeeping functions of this gene (Daugaard et al., 2007). Human HSPA5 is involved in the folding and transport of proteins into the endoplasmic reticulum and Hspa5 knockout mice embryos die at day 3.5, and therefore it, too, is considered a housekeeping gene (Daugaard et al., 2007). Furthermore, Matwee et al. (2001) have reported a reduced blastocyst rate and an increased apoptosis of embryos cultured in the presence of anti-HSP70. Thus, upregulation and downregulation of HSPs in bovine embryos observed in the present study imply that these genes play vital roles in early embryo development.","Given that alternative splicing is a major source of phenotypic complexity in mammals (Wang et al., 2008), a search was conducted of the Ensembl annotation of the bovine genome and found that only 4 out of 17 HSP candidate genes have annotated splice variants. In contrast, for the 17 human and mouse counterparts, 15 and 13 genes, respectively, have 2 or more splice variants. The low number of alternative transcripts identified in cattle is presumably due to the incomplete annotation of the bovine genome. Indeed, in a previous study, thousands of alternative splicing events were characterized in blastocysts and degenerative embryos, and some of those events were found to be associated with the developmental status of embryos (Huang and Khatib, 2010).","In order to explore the effects of the different splice variants of HSPs on embryo development, expression levels of the identified variants were estimated and compared between blastocysts and degenerate embryos. The results clearly show that different splice variants display different expression levels which imply different functions in embryo development. For example, while the long variant of DNAJC24 was highly expressed in degenerate embryos compared to blastocysts, the short variant of this gene was found to be lowly expressed in both embryo groups. Also, for DNAJC5, one transcript could not be detected using qRT-PCR. These results are consistent with human studies in which only some of the splice variants of genes were found to be associated with a disease state (Wang and Cooper, 2007). Collectively, these results testify to the importance of alternative splicing as a key regulator of phenotypic variation in mammals.","To better understand the involvement of the HSP genes in fertility traits, 4 genes that showed the most significant expression differences between embryo groups were tested for SNP association.","Recently, an IVF experimental system in cattle has been developed in our laboratory (see Methods) aimed at identifying genetic factors affecting fertilization and embryo development. The effectiveness of this system in identifying genes and pathways associated with developmental and fertility traits has been demonstrated in several studies (Khatib et al., 2008a,b; Khatib et al., 2009; Driver et al., 2009; Wang et al., 2009; Huang et al., 2010a).","Analysis of DNAJC15 and DNAJC27 SNP revealed significant associations with blastocyst and fertilization rates, respectively. These results are of particular significance because these 2 genes showed a remarkable difference in expression between blastocysts and degenerate embryos, providing strong evidence for the involvement of these genes in embryo production and development in cattle. The results also provide further evidence for the roles of HSPs in the fertilization process and blastocyst rates observed in mice and cattle in previous studies. Neuer et al. (1998) found that the presence of anti-HSP60 monoclonal antibodies had significant effects on blastocyst rates of mouse embryos. Matwee et al. (2001) reported that the addition of anti-HSP70 monoclonal antibodies to the culture medium reduced the mean number of spermatozoa bound to zona pellucida by about 50% to that of controls. Also, the authors reported that anti-HSP70 significantly reduced the number of bovine embryos that reached the blastocyst stage (Matwee et al., 2001). A recent study by Rosenkrans et al. (2010) reported an association between SNP in the promoter region of HSP70 and calving percentages in Brahman cows. Although a small sample size was used in the association analysis, the results point to an important role of HSP70 in cow fertility.","The identification of specific HSP genes contributing to embryo survival can be a unique opportunity to improve protection of IVF embryos from different toxic conditions and to enhance pregnancy rates in cattle (Hansen, 2007). HSP genes investigated in this study were found to be differentially expressed between blastocysts and degenerate embryos and SNP in 2 genes were associated with fertility traits. Blastocysts and degenerate embryos have distinct morphological and developmental features. As such, the present study provides a set of candidate genetic markers for pre-implantation embryo development. However, it has been reported that relatively small changes (decrease or increase) in expression levels of heat shock proteins can result in growth abnormalities and cell death (Nollen and Morimoto, 2002). Also, the assessment of the expression of alternative transcript isoforms in the embryos examined in this study testifies to the importance of inclusion of this approach in studying gene expression.","The present invention accordingly provides novel cattle genotyping, selective cattle breeding and related methods, based on the discovery that certain HSP40 polymorphism confers improved fertilization and embryonic survival rate to cattle.","The following examples are intended to illustrate preferred embodiments of the invention and should not be interpreted to limit the scope of the invention as defined in the claims.","Materials and Methods","In-Vitro Maturation, Fertilization, and Embryo Culture","In this study, fertilization and embryo production were performed in 2 different experiments. The first experiment is a comparison of expression profiles of 17 HSPs between 2 populations of embryos differing in their morphology, and the second experiment is a genomic association analysis of genes\u2014found to be differentially-expressed in the first experiment\u2014with fertilization and blastocyst rates.","Experiment I: Expression Analysis of HSP Genes in Cattle Embryos","Ovaries were obtained from a local abattoir and processed for IVF using standard protocol (Huang et al., 2010b; Khatib et al., 2008a). Briefly, oocytes were aspirated from 2-6 mm follicles and washed in Tyrode's albumin lactate pyruvate (TALP)-Hepes after which they underwent maturation in supplemented M199 media (Khatib et al., 2008a). Incubation was completed at 5% COin air at 39\u00b0 C. and high humidity for 22-24 hours at which time oocytes were then washed in TALP-Hepes and moved into fresh IVF-TALP (Biowhittaker, Walkersburg, Md.). Semen samples from 2 different bulls were used for fertilization; sperm underwent Percoll separation (45-90% gradient) and were adjusted to a final concentration of 1\u00d710\/ml (Parrish et al., 1995). Fertilization was marked as day 0 of development and completed by combining sperm, heparin, and PHE with the oocytes in 44 \u03bcl drops of media (Khatib et al., 2008a). Incubation was then continued for 22-24 hours after which the putative zygotes were washed in TALP-Hepes, denuded of remaining cumulus cell complexes, and placed into supplemented synthetic oviductal fluid (Biowhittaker) and returned to incubation (Khatib et al., 2008a).","Morphological Assessment","Putative zygotes were cultured over an 8-day period during which assessments of their developmental progress were made. By day 5 of development, a bovine embryo should attain approximately 16-32 cells and show signs of cellular compaction deeming it as morula. Embryos failing to show these characteristics were assumed to have been arrested in development and were excluded from analysis. Morulas were then continued in culture until day 8. By this time an embryo should show evidence of a fluid filled cavity (blastocoele), which gives rise to the differentiation of the inner cell mass and the trophectoderm, qualifying it as a blastocyst. For expression analysis, there were 2 populations of embryos collected. The first consisted of embryos that attained a compacted morula status by day 5 but failed to form a blastocoele by day 8, referred as degenerate embryos. The second consisted of embryos that developed into blastocysts by day 8. Embryos from each morphological group were collected in pools of 20 and preserved in RNALater (Ambion, Austin, Tex.). Two bulls were used with 2 sets of biological replicate pools (total of 4 biological replicates) from each to prevent maternal crossover.","Quantitative Real-Time PCR (qRT-PCR)","A previous study in our laboratory using microarray expression analysis has revealed that many genes are differentially expressed between blastocysts and degenerate embryos (Huang et al., 2010b). Seventeen HSP genes that showed 1.5-fold or higher differences in expression between the embryo groups described in the study of Huang et al. (2010b) were chosen for validation of differential expression and further investigation in new sets of blastocysts and degenerate embryo pools using qRT-PCR analysis. Primers (Table 1) were designed to amplify fragments spanning more than one exon to exclude the possibility of genomic DNA contamination in the qRT-PCR reactions using the Beacon Designer software (Premier Biosoft International, Palo Alto, Calif.). Total RNA was extracted from pools of embryos using RNaqueous Micro (Ambion) and quality controlled using a RNA6000 PicoChip (Agilent Technologies, CA). Messenger RNA was amplified using MessageAmp II (Ambion, Austin, Tex.), followed by cDNA synthesis using iScript (Bio-Rad, Hercules, Calif.) according to manufacturers' instructions. Reactions of qRT-PCR were run on a DNA Engine\u2014Opticon 2 Detection System (MJ Research, Watertown, Mass.) using iQ SYBR Green Supermix kit (Bio-Rad Laboratories, CA). Each sample was run in quadruplicate and all 4 expression results were averaged. Analysis of gene expression levels was conducted using the 2method (Livak and Schmittgen, 2001). The selection of the housekeeping gene GAPDH as an endogenous control was as described in Huang et al. (2010b).","Experiment II: Association Analysis of Differentially-Expressed HSPs with Fertilization and Blastocyst Rates","In order to further investigate the involvement in fertility of genes (DNAJC15, DNAJC19, DNAJC24, and DNAJC27) that showed the highest fold change in expression in degenerate embryos vs. blastocysts, an association analysis between SNPs in these genes and 2 main fertility traits: fertilization rate and blastocyst rate was performed. Using the IVF experimental system, several genes were found to be significantly associated with variation in fertilization and embryonic survival rates (Khatib et al., 2008a,b; Khatib et al., 2009).","Phenotypic Data. In order to generate phenotypic data for association analysis, a total of 6,893 in-vitro fertilizations were performed using oocytes from 399 ovaries (obtained from 399 Holstein cows) and semen samples from 12 Holstein bulls. For 92 ovaries, oocytes were fertilized by two different bulls each. Fertilization rate was calculated as the number of cleaved embryos 48 h postfertilization out of the total number of oocytes exposed to sperm. Blastocyst rate was calculated as the number of embryos that reached blastocyst stage out of the total embryos cultured by day 8. asdf","Genotyping. DNA was extracted from ovaries (n=399) using standard phenol\/chloroform protocols. The DNA concentrations were measured using a spectrophotometer (Ultraspec 2100; Amersham Biosciences). For polymorphism identification, 3 DNA pools were constructed from 20 different ovary\/cow samples to contain 25 ng of DNA from each sample. DNA Pools were amplified with different sets of primers designed from the coding and 5\u2032 and 3\u2032 UTR regions of the 4 candidate genes (Table 3). Amplification, sequencing of PCR products, and SNP identification were as described in Khatib et al. (2008a,b). SNP identified in DNAJC15, DNAJC19, DNAJC24, and DNAJC27 were genotyped for the 399 ovary\/cow samples at GeneSeek (Lincoln, Nebr.).","Statistical Analysis","For expression analysis, normalized gene expression values (\u0394Ct) were analyzed using a general linear model including the fixed effects of the bull, the type of embryo (blastocyst or degenerate), and the random effect of the pool. Association between the normalized gene expression and the type of embryo was tested using a likelihood ratio test by comparing this model to a reduced model without the embryo effect. The mean and the range of the fold change for each gene were calculated as 2using the estimated \u0394\u0394Ct value\u00b1standard error.","For genomic association analysis between SNP and fertilization and blastocyst rates, the following mixed linear model was used,\n\n+SNP\n\nwhere yrepresents in turn, the fertilization or survival rate of oocytes k from ovary i fertilized with semen from bull j; \u03bc represents a general mean for the trait considered, orepresents the random effect of the individual ovary from which oocytes were harvested; brepresents the random effect of the sire used in the fertilization; SNPrepresents the fixed effect of the genotypic class for the SNP considered; and erepresents the residuals, assumed normal, independent and identically distributed with mean 0 an variance I\u03c3. Ovaries and bulls were assumed uncorrelated with variance structures I\u03c3and I\u03c3, respectively. Association between the SNP and fertilization or blastocyst rates was tested using a likelihood ratio test by comparing the full model to a reduced model without the SNP effect. All analyses were performed using the lme4 package of R language\/environment (R Development Core Team, 2009).\n","Results","Expression Profiling of HSP Genes in Embryos","Expression differences between degenerate embryos and blastocysts were estimated for 17 HSP genes in 4 sets of biological replicates using qRT-PCR (). A range of 1.5- to 7.6-fold difference in expression was observed between embryo groups. Interestingly, all HSP40 gene family members were found to be upregulated in degenerate embryos compared to blastocysts (). For example, DNAJC15 (P<0.0001), DNAJC19 (P<0.0001), DNAJC24 (P=0.002), and DNAJC27 (P=0.0098) showed an average of 7.6-, 4.8-, 3.3-, 4.33-fold differences in expression, respectively. In contrast, only 2 members of HSP70 family (HSPA14 and HSPA9) showed upregulation in degenerates, whereas 4 HSP70 genes (HSPA2, HSPA4, HSPA5, and HSPA8) showed higher expression levels in blastocysts than in degenerates ().","Differential Expression of Alternatively-Spliced Transcripts","Splice variants of HSP genes were identified using the genebuild procedure of Ensembl (http:\/\/uswest.ensembl.org\/). In this procedure, annotation of transcripts is based on mRNA and protein sequences deposited in public scientific databases. Ensembl search revealed one transcript per gene for 13 genes, 2 transcripts each for DNAJB12 and DNAJC19, and 3 transcripts each for DNAJC5 and DNAJC24. Expression levels of splice variants of DNAJC5, DNAJC19, DNAJC24, and DNAJB12 () were estimated in 4 sets of biological replicates of blastocysts and degenerate embryos using qRT-PCR. Transcripts DNAJC5-1049 and DNAJC5-667 () showed 2.53- and 1.67-fold higher expression, respectively, in degenerate embryos compared to blastocysts while the DNAJC5-725 transcript was not detected in any of the embryo groups. For DNAJC19, only transcript DNAJC19-536 was detected in embryos. For DNAJC24, DNAJC24-2172 transcript showed a 3.22-fold higher expression in degenerate embryos compared to blastocysts, whereas transcript DNAJC24-628 was lowly expressed in all embryo samples. DNAJC24-1893 transcript could not be detected in embryos. The sequences of the 2 DNAJB12 transcripts were overlapping so that only transcript DNAB12-1220 could be amplified using transcript-specific primers. DNAB12-1220 showed a 1.95-fold higher expression in degenerate embryos vs. blastocysts.","Association of HSP Polymorphisms with Fertilization and Blastocyst Rates","SNP in genes DNAJC15, DNAJC19, DNAJC24, and DNAJC27 were tested for association with fertilization and blastocyst rates. These genes showed the highest fold differences in expression between embryo groups () and the highest statistical significance in the general linear model analysis (P<0.01). Using the pooled DNA sequencing approach, 5 SNP were identified in the 3\u2032UTR of DNAJC15, one SNP in exon 5 of DNAJC19, one SNP and one 4-bp deletion in the 3\u2032 UTR of DNAJC24, and 3 SNP in the 3\u2032 UTR of DNAJC27. All SNP in each of DNAJC15 and DNAJC27 genes showed high linkage disequilibrium with each other (). Polymorphisms in DNAJC19 and DNAJC24 did not show significant associations with fertility traits. Table 2 shows fertilization and blastocysts rates for the genotypic classes of the DNAJC15 and DNAJC27 SNP. For DNAJC27 SNP36016, oocytes collected from genotype GG ovaries showed a 69.3% fertilization rate vs. 62.2% for oocytes collected from CC ovaries (P=0.034). For DNAJC15 SNP85146, the blastocyst rate of embryos produced from GG dams was 40.1% vs. 31.0% and 28.1% for embryos produced from AA and AG dams, respectively (Table 2).","While the invention has been described in connection with one or more embodiments, it should be understood that the invention is not limited to those embodiments, and the description is intended to cover all alternatives, modifications, and equivalents, as may be included within the spirit and scope of the appended claims. All referenced cited herein are further incorporated by reference in their entirety.",{"@attributes":{"id":"p-0100","num":"0000"},"ul":{"@attributes":{"id":"ul0001","list-style":"none"},"li":["Becker, J., and E. A. Craig. 1994. Heat-shock proteins as molecular chaperones. Eur. J. Biochem. 219:11-23.","Carlborg, O., and C. S. Haley. 2004. Epistasis: too often neglected in complex trait studies? Nat. Rev. Genet. 5:618-625.","Cobanoglu, O., I. Zaitoun, Y. M. Chang, G. E. Shook, and H. Khatib. 2006. Effects of the signal transducer and activator of transcription 1 (STAT1) gene on milk production traits in Holstein dairy cattle. J. Dairy Sci. 89:4433-4437.","Darnell, J. E. 1997. STATs and gene regulation. Science 277:1630-1635.","Daugaard, M., M. Rohde, and M. J\u00e4\u00e4ttel\u00e4. 2007. The heat shock protein 70 family: Highly homologous proteins with overlapping and distinct functions. FEBS Lett. 581:3702-3710.","Dobson, H., R. Smith, M. Royal, Ch. Knight, and I. Sheldon. 2007. The high-producing dairy cow and its reproductive performance. Reprod Domest Anim. 42 Suppl 2:17-23.","Dobson, H., S. L. Walker, M. J. Morris, J. E. Routly, and R. F. Smith. 2008. Why is it getting more difficult to successfully artificially inseminate dairy cows? Animal 2:1104-1111.","Driver, A. M., W. Huang, S. Gajic, R. L. Monson, G. J. Rosa, and H. Khatib. 2009. Effects of the progesterone receptor variants on fertility traits in cattle. J. Dairy Sci. 92:4082-4085.","Hansen, P. J. 2007. To be or not to be\u2014determinants of embryonic survival following heat shock. Theriogenology 68 Suppl 1:S40-8.","Hombr\u00eda, J. C., and S. Brown. 2002. The fertile field of Jak\/STAT signalling. Curr. Biol. 12:R569-R575.","Huang, W., B. W. Kirkpatrick, G. J. Rosa, and H. Khatib. 2010a. A genome-wide association study using selective DNA pooling identifies candidate markers for fertility in Holstein cattle. Anim. Genet. 41:570-578.","Huang, W., B. S. Yandell, and H. Khatib. 2010b. Transcriptomic profiling of bovine IVF embryos revealed candidate genes and pathways involved in early embryonic development. BMC Genomics 11:23.","Huang, W., and H. Khatib. 2010. Comparison of transcriptomic landscapes of bovine embryos using RNA-Seq. BMC Genomics 11:711.","Kisseleva, T., S. Bhattacharya, J. Braunstein and C. W. Schindler. 2002. Signaling through the JAK\/STAT pathway, recent advances and future challenges. Gene 285:1-24.","Khatib, H., R. L. Monson, V. Schutzkus, D. M. Kohl, G. J. Rosa, and J. J. Rutledge. 2008a. Mutations in the STATSA gene are associated with embryonic survival and milk composition in cattle. J. Dairy Sci. 91:784-793.","Khatib, H., C. Maltecca, R. L. Monson, V. Schutzkus, X. Wang, and J. J. Rutledge. 2008b. The fibroblast growth factor 2 gene is associated with embryonic mortality in cattle. J. Anim. Sci. 86:2063-2067.","Khatib, H., W. Huang, X. Wang, A. H. Tran, A. B. Bindrim, V. Schutzkus, R. L. Monson, and B. S. Yandell. 2009. Single gene and gene interaction effects on fertilization and embryonic survival rates in cattle. J. Dairy Sci. 92:2238-2247.","Kodama, H., K. Fukuda, J. Pan, S. Makino, A. Baba, S. Hori, and S. Ogawa. 1997. Leukemia inhibitory factor, a potent cardiac hypertrophic cytokine, activates the JAK\/STAT pathway in rat cardiomyocytes. Circ. Res. 81:656-663.","Lanneau, D., A. de Thonel, S. Maurel, C. Didelot, and C. Garrido. 2007. Apoptosis versus cell differentiation: role of heat shock proteins HSP90, HSP70 and HSP27. Prion 1:53-60.","Leroy, J. L., G. Opsomer, A. Van Soom, I. G. Goovaerts, and P. E. Bols. 2008. Reduced fertility in high-yielding dairy cows: are the oocyte and embryo in danger? Part I. The importance of negative energy balance and altered corpus luteum function to the reduction of oocyte and embryo quality in high-yielding dairy cows. Reprod Domest Anim. 43:612-622.","Livak, K. J., and T. D. Schmittgen. 2001. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods 25:402-408.","Luft, J. C., and D. J. Dix. 1999. Hsp70 expression and function during embryogenesis. Cell Stress Chaperones 4:162-170.","Maj, T., and A. Chelmonska-Soyta. 2007. Pleiotropy and redundancy of STAT proteins in early pregnancy. Reprod. Domest. Anim. 42:343-353.","McCullagh, P., and J. A. Nelder. 1989. Generalized Linear Models, 2nd edn. Chapman and Hall, London.","Morris, D., and M. Diskin. 2008. Effect of progesterone on embryo survival. Animal 2:1112-1119.","Neuer, A., C. Mele, H. C. Liu, Z. Rosenwaks, and S. S. Witkin. 1998. Monoclonal antibodies to mammalian heat shock proteins impair mouse embryo development in vitro. Hum. Reprod. 13:987-990.","Neuer, A., S. D. Spandorfer, P. Giraldo, J. Jeremias, S. Dieterle, I. Korneeva, H. C. Liu, Z. Rosenwaks, and S. S. Witkin. 1999. Heat shock protein expression during gametogenesis and embryogenesis. Infect. Dis. Obstet. Gynecol. 7:10-6.","Nollen, E. A., and R. I. Morimoto. 2002. Chaperoning signaling pathways: molecular chaperones as stress-sensing \u2018heat shock\u2019 proteins. J. Cell Sci. 115:2809-2816.","Parrish, J. J., A. Krogenaes, and J. L. Susko-Parrish. 1995. Effect of bovine sperm separation by either swim-up or Percoll method on success of in vitro fertilization and early embryonic development. Theriogenology 44:859-869.","Qiu, X. B., Y. M. Shao, S. Miao, and L. Wang. 2006. The diversity of the DnaJ\/Hsp40 family, the crucial partners for Hsp70 chaperones. Cell Mol. Life. Sci. 63:2560-2570.","R Development Core Team. 2008. R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria. (www.R-project.org).","Royal, M., G. E. Mann, and A. P. Flint. 2000. Strategies for reversing the trend towards subfertility in dairy cattle. Vet. J. 160:53-60.","Santos, J. E. P., W. W. Thatcher, R. C. Chebel, R. L. A. Cerri, and K. N. Galvao. 2004. The effect of embryonic death rates in cattle on the efficacy of estrus synchronization programs. Anim. Reprod. Sci. 83:513-535.","Shook G. E. 2006. Major advances in determining appropriate selection goals. J. Dairy Sci. 89:1349-1361.","Soller, M. (1990) Genetic mapping of the bovine genome using DNA-level markers with particular attention to loci affecting quantitative traits of economic importance. J. Dairy Sci. 73:2628-2646.","Soller, M. (1994) Marker-assisted selection, an overview. Anim. Biotech. 5:193-208.","Takeda, K., K. Noguchi, W. Shi, T. Tanaka, M. Matsumoto, N. Yoshida, T. Kishimoto, and S. Akira. 1997. Targeted disruption of the mouse Stat3 gene leads to early embryonic lethality. Proc. Natl. Acad. Sci. USA. 94:3801-3804.","Teglund, S., C. McKay, E. Schuetz, J. M. van Deursen, D. Stravopodis, D. Wang, M. Brown, S. Bodner, G. Grosveld and J. N. Ihle. 1998. StatSa and Stat5b proteins have essential and nonessential, or redundant, roles in cytokine responses. Cell 93:841-850.","Truchet, S., M. Chebrout, C. Djediat, J. Wietzerbin, and P. Debey. 2004. Presence of permanently activated signal transducers and activators of transcription in nuclear interchromatin granules of unstimulated mouse oocytes and preimplantation embryos. Biol. Reprod. 71:1330-1339.","VanRaden, P. M., A. H. Sanders, M. E. Tooker, R. H. Miller, H. D. Norman, M. T. Kuhn, and G. R. Wiggans. 2004. Development of a national genetic evaluation for cow fertility. J. Dairy Sci. 87:2285-2292.","Wang, G. S., and T. A. Cooper. 2007. Splicing in disease: disruption of the splicing code and the decoding machinery. Nat. Rev. Genet. 8:749-761.","Wang, X., V. Schutzkus, W. Huang, G. J. Rosa, and H. Khatib. 2009. Analysis of segregation distortion and association of the bovine FGF2 with fertilization rate and early embryonic survival. Anim. Genet. 40:722-728.","Wang, E. T., R. Sandberg, S. Luo, I. Khrebtukova, L. Zhang, C. Mayr, S. F. Kingsmore, G. P. Schroth, and C. B. Burge. 2008. Alternative isoform regulation in human tissue transcriptomes. Nature 456:470-476."]}},{"@attributes":{"id":"p-0101","num":"0142"},"tables":{"@attributes":{"id":"TABLE-US-00001","num":"00001"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20031"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Primers\u2003for\u2003total\u2003expression\u2003and\u2003alternative\u2003"},{"entry":"splicing\u2003analyses\u2003of\u2003HSP\u2003genes"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},{},{},"ampli-"]},{"entry":[{},{},"forward\u2003","reverse\u2003","con\u2003\u2003"]},{"entry":[{},{},"primer\u2003","primer\u2003","size"]},{"entry":[{},"Family","5\u2032-3\u2032","5\u2032-3\u2032","(bp)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Gene"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["GAPDH",{},"TGCCCAGAAT","AGGTCAGATC","134"]},{"entry":[{},{},"ATCATCCC","CACAACAG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20033)","(SEQ\u2003ID\u2003NO:\u20034)",{}]},{"entry":{}},{"entry":["DNAJC5","HSP40","CTACGACAAG","GGCAGCAGC","149"]},{"entry":[{},{},"TACGGCTCAC","AACAGTAGC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20035)","(SEQ\u2003ID\u2003NO:\u20036)",{}]},{"entry":{}},{"entry":["DNAJC15","HSP40","AGGTCGCTA","GACTTGCTTC","137"]},{"entry":[{},{},"CGCATTTCAG","TCGCCTACTC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20037)","(SEQ\u2003ID\u2003NO:\u20038)",{}]},{"entry":{}},{"entry":["DNAJC19","HSP40","GGACTGACCA","CAAACCCACCT","140"]},{"entry":[{},{},"TTGCTGCTG","CTGTAATAGC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u20039)","(SEQ\u2003ID\u2003NO:\u200310)",{}]},{"entry":{}},{"entry":["DNAJC24","HSP40","GAAATATGGGA","TGTAACTTCTT","134"]},{"entry":[{},{},"CCAGTAGATGC","CTGCTTCATCC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200311)","(SEQ\u2003ID\u2003NO:\u200312)",{}]},{"entry":{}},{"entry":["DNAJC27","HSP40","AACAAGCGGA","TGAAGCAGCA","124"]},{"entry":[{},{},"CACCATTCG","CAGCAAGC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200313)","(SEQ\u2003ID\u2003NO:\u200314)",{}]},{"entry":{}},{"entry":["HSPA5","HSP70","CAACCAACTGT","AAAGGTGACTT","133"]},{"entry":[{},{},"TACCATCAAGG","CAATCTGTGG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200315)","(SEQ\u2003ID\u2003NO:\u200316)",{}]},{"entry":{}},{"entry":["HSPA9","HSP70","GACCAACTGC","GATGCCGCCT","113"]},{"entry":[{},{},"CTGCTGATG","GCCTTATG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200317)","(SEQ\u2003ID\u2003NO:\u200318)",{}]},{"entry":{}},{"entry":["HSPA8","HSC70","CGCAGAAGCC","GTTGAGACCAG","115"]},{"entry":[{},{},"TACCTTGG","CAATAGTTCC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200319)","(SEQ\u2003ID\u2003NO:\u200320)",{}]},{"entry":{}},{"entry":["HSPA14","HSP70","AACCTTAGCAC","TGTCAGCACC","101"]},{"entry":[{},{},"AGTACCTAGC","GTTCATCAG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200321)","(SEQ\u2003ID\u2003NO:\u200322)",{}]},{"entry":{}},{"entry":["HSBP1","HSBP","CATGTCCGAC","TTCACTGTCC","115"]},{"entry":[{},{},"CAGATCATTG","AGCTCTTCC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200323)","(SEQ\u2003ID\u2003NO:\u200324)",{}]},{"entry":{}},{"entry":["HSPE1","HSP10","GCAAGCAACG","ACTTTGGTGCC","117"]},{"entry":[{},{},"GTGGTAGC","TCCATATTCTG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200325)","(SEQ\u2003ID\u2003NO:\u200326)",{}]},{"entry":{}},{"entry":["DNAJB1","HSP40","GAGGAGAAGTT","TTAGTACCGC","134"]},{"entry":[{},{},"CAAGGAGATCG","CGCTGCTC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200327)","(SEQ\u2003ID\u2003NO:\u200328)",{}]},{"entry":{}},{"entry":["DNAJB12","HSP40","GCAAACTAGCC","CCTTGTCATCA","141"]},{"entry":[{},{},"CTCAAATTCC","CCGAACTGG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200329)","(SEQ\u2003ID\u2003NO:\u200330)",{}]},{"entry":{}},{"entry":["DNAJC14","HSP40","GTGAATGAGTT","GTATCTGGCAC","114"]},{"entry":[{},{},"TCTGTCCAAGC","TCTTAGGTTCC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200331)","(SEQ\u2003ID\u2003NO:\u200332)",{}]},{"entry":{}},{"entry":["HSPA2","HSP70","ACGCTGTGGAG","TTCCGCCATC","146"]},{"entry":[{},{},"TCCTATACC","TGGTTCCG",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200333)","(SEQ\u2003ID\u2003NO:\u200334)",{}]},{"entry":{}},{"entry":["HSPA4","HSP70","TCCTGCCTTA","CCCAGTGTTG","132"]},{"entry":[{},{},"GAAGAGAAACC","TGTCAAATGC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200335)","(SEQ\u2003ID\u2003NO:\u200336)",{}]},{"entry":{}},{"entry":["HSPH1","HSP100","ATGTTGAGTTG","CCTCCACCGC","141"]},{"entry":[{},{},"CCTATTGAAGC","ATTCTTAGC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200337)","(SEQ\u2003ID\u2003NO:\u200338)",{}]},{"entry":{}}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"left"}},"tbody":{"@attributes":{"valign":"top"},"row":{"entry":"Transcript"}}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"5"},"colspec":[{"@attributes":{"colname":"1","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"28pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"5","colwidth":"28pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":["DNAJC5-",{},"AGGAGACGG","CACGTTCAC","126"]},{"entry":["667",{},"AGTTCTATG","ACCTCAAC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200339)","(SEQ\u2003ID\u2003NO:\u200340)",{}]},{"entry":{}},{"entry":["DNAJC5-",{},"GGCCCTGTT","GGCACAGAC","181"]},{"entry":["725",{},"CATCTTCTG","CCTCTCAT",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200341)","(SEQ\u2003ID\u2003NO:\u200342)",{}]},{"entry":{}},{"entry":["DNAJC5-",{},"GGGTTCAACTA","ACGCCATCTC","79"]},{"entry":["1049",{},"AATCCAGGA","TGTGACTA",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200343)","(SEQ\u2003ID\u2003NO:\u200344)",{}]},{"entry":{}},{"entry":["DNAJC19-",{},"GGATCTCCT","AGCCTTCCC","93"]},{"entry":["475",{},"TATATAGCA","TCCCAGTGA",{}]},{"entry":[{},{},"GCCAAA",{},{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200345)","(SEQ\u2003ID\u2003NO:\u200346)",{}]},{"entry":{}},{"entry":["DNAJC19-",{},"ATGCTCATCG","AGCTGGAACG","161"]},{"entry":["536",{},"GCGAATTATG","CATAAGAGAA",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200347)","(SEQ\u2003ID\u2003NO:\u200348)",{}]},{"entry":{}},{"entry":["DNAJC24-",{},"CTCATTTTAA","GTATCACAAG","176"]},{"entry":["628",{},"TGGAAGATG","AAATCAGT",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200349)","(SEQ\u2003ID\u2003NO:\u200350)",{}]},{"entry":{}},{"entry":["DNAJC24-",{},"CATCCAGAT","GGTCCCATAT","161"]},{"entry":["2172",{},"AAACAGAGT","TTCTTAGAT",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200351)","(SEQ\u2003ID\u2003NO:\u200352)",{}]},{"entry":{}},{"entry":["DNAJC24-",{},"CAAAAGAAAG","TAACTTCTTC","176"]},{"entry":["1893",{},"TATCTCATTC","TGCTTCATC",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200353)","(SEQ\u2003ID\u2003NO:\u200354)",{}]},{"entry":{}},{"entry":["DNAJB12-",{},"ACCGACTGTC","CGGCCTCCAA","127"]},{"entry":["1220",{},"AGAGACTATG","TTCCATTT",{}]},{"entry":[{},{},"(SEQ\u2003ID\u2003NO:\u200355)","(SEQ\u2003ID\u2003NO:\u200356)"]},{"entry":{"@attributes":{"namest":"1","nameend":"5","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0102","num":"0143"},"tables":{"@attributes":{"id":"TABLE-US-00002","num":"00002"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":[{"entry":"TABLE 2"},{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}}]},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":"Blastocyst and fertilization rates for genotypic classes of DNAJC15 and"},{"entry":"DNAJC27 genes"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"63pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"63pt","align":"center"}},{"@attributes":{"colname":"4","colwidth":"28pt","align":"center"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},"genotype",{},{}]},{"entry":["DNAJC15 SNP","(number of ovaries)","blastocyst rate \u00b1 SE","P value"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["SNP 85146(G > A)","AA (202)","0.310 \u00b1 0.025","0.014"]},{"entry":[{},"AG (206)","0.281 \u00b1 0.025"]},{"entry":[{},"GG (53)","0.401 \u00b1 0.040"]},{"entry":["SNP 85161(G > A)","AA (202)","0.309 \u00b1 0.023","0.022"]},{"entry":[{},"AG (206)","0.283 \u00b1 0.024"]},{"entry":[{},"GG (61)","0.390 \u00b1 0.037"]},{"entry":["SNP 85216(A > G)","GG (203)","0.312 \u00b1 0.022","0.059"]},{"entry":[{},"AG (201)","0.282 \u00b1 0.023"]},{"entry":[{},"AA (63)","0.373 \u00b1 0.036"]},{"entry":["SNP 85292(C > A)","AA (204)","0.310 \u00b1 0.023","0.012"]},{"entry":[{},"AC (201)","0.279 \u00b1 0.024"]},{"entry":[{},"CC (62)","0.395 \u00b1 0.037"]},{"entry":["SNP 85300(G > A)","AA (208)","0.312 \u00b1 0.023","0.066"]},{"entry":[{},"AG (201)","0.280 \u00b1 0.023"]},{"entry":[{},"GG (63)","0.369 \u00b1 0.037"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":[{},"genotype","fertilization rate \u00b1"]},{"entry":["DNAJC27 SNP","(number of ovaries)","SE","P value"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["SNP 35728(G > A)","AA (126)","0.621 \u00b1 0.029","0.042"]},{"entry":[{},"AG (257)","0.654 \u00b1 0.025"]},{"entry":[{},"GG (103)","0.691 \u00b1 0.030"]},{"entry":["SNP 36016(C > G)","CC (129)","0.622 \u00b1 0.029","0.034"]},{"entry":[{},"CG (251)","0.653 \u00b1 0.026"]},{"entry":[{},"GG (103)","0.693 \u00b1 0.030"]},{"entry":["SNP 38867(A > G)","AA (133)","0.626 \u00b1 0.028","0.049"]},{"entry":[{},"AG (246)","0.650 \u00b1 0.026"]},{"entry":[{},"GG (104)","0.693 \u00b1 0.030"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}},{"@attributes":{"id":"p-0103","num":"0144"},"tables":{"@attributes":{"id":"TABLE-US-00003","num":"00003"},"table":{"@attributes":{"frame":"none","colsep":"0","rowsep":"0","tabstyle":"monospace"},"tgroup":[{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"1"},"colspec":{"@attributes":{"colname":"1","colwidth":"217pt","align":"center"}},"thead":{"row":{"entry":"TABLE\u20033"}},"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":{"@attributes":{"namest":"1","nameend":"1","align":"center","rowsep":"1"}}},{"entry":"Primers\u2003used\u2003for\u2003SNP\u2003identification,"},{"entry":"product\u2003sizes,\u2003and\u2003GenBank"},{"entry":"accession\u2003numbers\u2003of\u2003amplified\u2003genes"}]}},{"@attributes":{"align":"left","colsep":"0","rowsep":"0","cols":"4"},"colspec":[{"@attributes":{"colname":"1","colwidth":"49pt","align":"left"}},{"@attributes":{"colname":"2","colwidth":"91pt","align":"left"}},{"@attributes":{"colname":"3","colwidth":"35pt","align":"left"}},{"@attributes":{"colname":"4","colwidth":"42pt","align":"left"}}],"tbody":{"@attributes":{"valign":"top"},"row":[{"entry":[{},{},"Ampli-","GenBank"]},{"entry":[{},"Primer\u2003sequence","con","Accession\u2003"]},{"entry":["Primer","(5\u2032-3\u2032)","(bp)","no."]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}},{"entry":["DNAJC15-1","F:\u2003CCGGAGGTCTGCAAA","589","NC_007310"]},{"entry":[{},"TGGG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200357)",{},{}]},{"entry":[{},"R:\u2003AACTGCTCGCCTGGTG",{},{}]},{"entry":[{},"CTGGTC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200358)",{},{}]},{"entry":{}},{"entry":["DNAJC15-2","F:\u2003TCACTGAAAATCAGC","569","NC_007310"]},{"entry":[{},"CAATA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200359)",{},{}]},{"entry":[{},"R:\u2003CGTACAGAAGAGCCCCAT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200360)",{},{}]},{"entry":{}},{"entry":["DNAJC15-3","F:\u2003AATTGCTTTATTACTT","647","NC_007310"]},{"entry":[{},"TAGCGG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200361)",{},{}]},{"entry":[{},"R:\u2003AGGGACCATGTCTGT",{},{}]},{"entry":[{},"TTTGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200362)",{},{}]},{"entry":{}},{"entry":["DNAJC15-4","F:\u2003AAAGTCCCTGTAGAGCT","571","NC_007310"]},{"entry":[{},"TAG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200363)",{},{}]},{"entry":[{},"R:\u2003ATAAAGGCACATCACAA",{},{}]},{"entry":[{},"CTA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200364)",{},{}]},{"entry":{}},{"entry":["DNAJC15-5","F:\u2003TCCTCCTGTCCTAGT","647","NC_007310"]},{"entry":[{},"TCTTG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200365)",{},{}]},{"entry":[{},"R:\u2003TTCATTATGCCCAAA",{},{}]},{"entry":[{},"TCAGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200366)",{},{}]},{"entry":{}},{"entry":["DNAJC15-6","F:\u2003CCATCCACTTCAGAA","620","NC_007310"]},{"entry":[{},"AATTC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200367)",{},{}]},{"entry":[{},"R:\u2003GGGGAAAGATCAGTGC",{},{}]},{"entry":[{},"TAGAGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200368)",{},{}]},{"entry":{}},{"entry":["DNAJC19-1","F:\u2003T\u2003TTTCCGACCTAGTTT","670","NC_007299\u2003"]},{"entry":[{},"ACGG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200369)",{},{}]},{"entry":[{},"R:\u2003ACTTCTACTTCACCAC",{},{}]},{"entry":[{},"AGGGA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200370)",{},{}]},{"entry":{}},{"entry":["DNAJC19-2","F:\u2003AGCCGCATACCTTTA","696","NC_007299"]},{"entry":[{},"CAATG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200371)",{},{}]},{"entry":[{},"R:\u2003ATGGGTCACTTCAGA",{},{}]},{"entry":[{},"TTCCT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200372)",{},{}]},{"entry":{}},{"entry":["DNAJC19-3","F:\u2003TCCCAGAAGAACTGG","496","NC_007299"]},{"entry":[{},"GTTTG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200373)",{},{}]},{"entry":[{},"R:\u2003AGATGGAAGTCCCTG",{},{}]},{"entry":[{},"GCAGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200374)",{},{}]},{"entry":{}},{"entry":["DNAJC19-4","F:\u2003TTTAACTCACTGGAG","597","NC_007299"]},{"entry":[{},"GTAGG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200375)",{},{}]},{"entry":[{},"R:\u2003ACCAAAACAGCAAGT",{},{}]},{"entry":[{},"AGACC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200376)",{},{}]},{"entry":{}},{"entry":["DNAJC19-5","F:\u2003AGGGATTGTTGATAA","611","NC_007299"]},{"entry":[{},"CTGGA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200377)",{},{}]},{"entry":[{},"R:\u2003TTTATACCAACGCT",{},{}]},{"entry":[{},"TTGACT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200378)",{},{}]},{"entry":{}},{"entry":["DNAJC24-1","F:\u2003GCTCGGCTGGAAACTTGA","672","NC_007313"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200379)",{},{}]},{"entry":[{},"R:\u2003CGGTGGATGGCCCTCTAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200380)",{},{}]},{"entry":{}},{"entry":["DNAJC24-2","F:\u2003CCATTTCCTTCTCCA","630","NC_007313"]},{"entry":[{},"CTAGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200381)",{},{}]},{"entry":[{},"R:\u2003CTTTTGATTCCTGCT",{},{}]},{"entry":[{},"TTGAT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200382)",{},{}]},{"entry":{}},{"entry":["DNAJC24-3","F:\u2003CCCTTCTGCTTTGTCC","538","NC_007313"]},{"entry":[{},"ATC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200383)",{},{}]},{"entry":[{},"R:\u2003ACCACATTTCTGGGTTG",{},{}]},{"entry":[{},"CTC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200384)",{},{}]},{"entry":{}},{"entry":["DNAJC24-4","F:\u2003GGAGACTTTTGGCCTA","637","NC_007313"]},{"entry":[{},"GTGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200385)",{},{}]},{"entry":[{},"R:\u2003ATAAAGTTTTCAGGT",{},{}]},{"entry":[{},"GGGAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200386)",{},{}]},{"entry":{}},{"entry":["DNAJC24-5","F:\u2003TAAATAAATTCCCAC","649","NC_007313"]},{"entry":[{},"CTGAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200387)",{},{}]},{"entry":[{},"R:\u2003ACAATAGCCATGTTT",{},{}]},{"entry":[{},"TCTGA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200388)",{},{}]},{"entry":{}},{"entry":["DNAJC24-6","F:\u2003GGCAACTGTAGAAAGG","539","NC_007313"]},{"entry":[{},"ATAG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200389)",{},{}]},{"entry":[{},"R:\u2003TATAAAGAATAAGCA",{},{}]},{"entry":[{},"CCACA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200390)",{},{}]},{"entry":{}},{"entry":["DNAJC24-7","F:\u2003TGGTATTTATTATTGG","531","NC_007313"]},{"entry":[{},"TTTGGAC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200391)",{},{}]},{"entry":[{},"R:\u2003AGGAGAAAGGGATGA",{},{}]},{"entry":[{},"CAAGG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200392)",{},{}]},{"entry":{}},{"entry":["DNAJC27-1","F:\u2003CTCCTCCAGTTCCCT","665","NC_007309"]},{"entry":[{},"ACCC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200393)",{},{}]},{"entry":[{},"R:\u2003A\u2003CAGCCCAGTAAGTT",{},{}]},{"entry":[{},"ATCAGC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200394)",{},{}]},{"entry":{}},{"entry":["DNAJC27-2","F:\u2003TTGAAAACATACCCATA","682","NC_007309"]},{"entry":[{},"TTTGG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200395)",{},{}]},{"entry":[{},"R:\u2003ATCACTAAAAGGAAG",{},{}]},{"entry":[{},"CTCTC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200396)",{},{}]},{"entry":{}},{"entry":["DNAJC27-3","F:\u2003CTTCATCTTGCTCCTA","647","NC_007309"]},{"entry":[{},"CTGTCC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200397)",{},{}]},{"entry":[{},"R:\u2003CAAAGGGTTGGTTCA",{},{}]},{"entry":[{},"CTTCTG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200398)",{},{}]},{"entry":{}},{"entry":["DNAJC27-4","F:\u2003AGGGACAGGGTAGAAGGC","607","NC_007309"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u200399)",{},{}]},{"entry":[{},"R:\u2003CAAACATGGCACCAGAAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003100)",{},{}]},{"entry":{}},{"entry":["DNAJC27-5","F:\u2003GGCAAACGTGATGAAGCC","692","NC_007309"]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003101)",{},{}]},{"entry":[{},"R:\u2003GACCTGGAGCCCAGCAAT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003102)",{},{}]},{"entry":{}},{"entry":["DNAJC27-6","F:\u2003AGAGTAGGATCATAAGC","532","NC_007309"]},{"entry":[{},"CATTT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003103)",{},{}]},{"entry":[{},"R:\u2003TTCGGTGAAGGAGTAG",{},{}]},{"entry":[{},"TGTT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003104)",{},{}]},{"entry":{}},{"entry":["DNAJC27-7","F:\u2003TGCAAGAGGTGTTCTGT","666","NC_007309"]},{"entry":[{},"TAT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003105)",{},{}]},{"entry":[{},"R:\u2003TTTCAGGGGTTCTACT",{},{}]},{"entry":[{},"ATGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003106)",{},{}]},{"entry":{}},{"entry":["DNAJC27-8","F:\u2003ACATAGTAGAACCCCTG","500","NC_007309"]},{"entry":[{},"AAAGT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003107)",{},{}]},{"entry":[{},"R:\u2003ATGATGCTGCAACAAG",{},{}]},{"entry":[{},"GAAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003108)",{},{}]},{"entry":{}},{"entry":["DNAJC27-9","F:\u2003TGCCAAGACAGGTGGG","613","NC_007309"]},{"entry":[{},"AAAT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003109)",{},{}]},{"entry":[{},"R:\u2003CAGGTAGGGTAAGGC",{},{}]},{"entry":[{},"GAATG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003110)",{},{}]},{"entry":{}},{"entry":["DNAJC27-10","F:\u2003ATTCGCCTTACCCTA","644","NC_007309"]},{"entry":[{},"CCTGA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003111)",{},{}]},{"entry":[{},"R:\u2003CTGGGAACTGAGCAAG",{},{}]},{"entry":[{},"ACCTAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003112)",{},{}]},{"entry":{}},{"entry":["DNAJC27-11","F:\u2003CTTAGGTCTTGCTCAG","668","NC_007309"]},{"entry":[{},"TTCC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003113)",{},{}]},{"entry":[{},"R:\u2003CACAACATCTCCAAG",{},{}]},{"entry":[{},"TCCAG",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003114)",{},{}]},{"entry":{}},{"entry":["DNAJC27-12","F:\u2003ACTTGGAGATGTTGTG","565","NC_007309"]},{"entry":[{},"CTGC",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003115)",{},{}]},{"entry":[{},"R:\u2003TATGAACCCCTCTTC",{},{}]},{"entry":[{},"CCTTT",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003116)",{},{}]},{"entry":{}},{"entry":["DNAJC27-13","F:\u2003GTGTCAGTATCTGTCC","681","NC_007309"]},{"entry":[{},"CCTAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003117)",{},{}]},{"entry":[{},"R:\u2003TCTATGATCTCAGTC",{},{}]},{"entry":[{},"GGTAA",{},{}]},{"entry":[{},"(SEQ\u2003ID\u2003NO:\u2003118)"]},{"entry":{"@attributes":{"namest":"1","nameend":"4","align":"center","rowsep":"1"}}}]}}]}}}],"GOVINT":[{},{}],"BRFSUM":[{},{}],"brief-description-of-drawings":[{},{}],"description-of-drawings":{"heading":"DESCRIPTION OF THE DRAWINGS","p":[{"@attributes":{"id":"p-0028","num":"0027"},"figref":"FIG. 1"},{"@attributes":{"id":"p-0029","num":"0028"},"figref":"FIG. 2"},{"@attributes":{"id":"p-0030","num":"0029"},"figref":"FIG. 3"},{"@attributes":{"id":"p-0031","num":"0030"},"figref":"FIG. 4"},{"@attributes":{"id":"p-0032","num":"0031"},"figref":"FIG. 5","sup":"2"}]},"DETDESC":[{},{}]}
